function cif_dict() { this._dict =
[ [ '@comment',
    '##########################################################################' ],
  [ '@comment', ' File:  mmcif_pdbx.dic' ],
  [ '@comment', ' Date:  Thu Oct 27 16:39:28 EDT 2005' ],
  [ '@comment',
    ' Created from files in CVS module dict-mmcif_pdbx.dic unless noted:' ],
  [ '@comment', '        mmcif_pdbx-header.dic   ' ],
  [ '@comment', '        mmcif_pdbx-data.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-1.dic   ' ],
  [ '@comment',
    '        ../dict-mmcif_std/mmcif_std-def-2.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-2.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-3.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-4.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-5.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-6.dic   ' ],
  [ '@comment',
    '        ../dict-mmcif_iims/mmcif_iims-def-2.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-8.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-9.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-10.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-12.dic   ' ],
  [ '@comment', '        mmcif_pdbx-def-13.dic   ' ],
  [ '@comment',
    '##########################################################################' ],
  [ '@comment',
    '##########################################################################' ],
  [ '@comment',
    ' File:  mmcif_pdbx-header.dic                                             ' ],
  [ '@comment', ' ' ],
  [ '@comment',
    '                       PDB Exchange Data Dictionary                 ' ],
  [ '@comment',
    '     This dictionary supplements the contents of the mmCIF data dictionary ' ],
  [ '@comment',
    '     with additional data items that may be required to store all of the   ' ],
  [ '@comment',
    '     information currently included a Protein Data Bank (PDB) format data  ' ],
  [ '@comment',
    '     file.  Additional data items describing internal processing status    ' ],
  [ '@comment',
    '     are also defined here.                                                ' ],
  [ '@comment',
    '                                                                           ' ],
  [ '@comment',
    '     The mmCIF dictionary and this supplement provide the provisional      ' ],
  [ '@comment',
    '     specifications for the data exchange protocol used by the wwPDB:      ' ],
  [ '@comment',
    '     Resource Collaboratory for Structural Bioinformatics (RCSB) PDB, ' ],
  [ '@comment',
    '     Molecular Structure Database (MSD) Group/EBI, PDB Japan (PDBj) at' ],
  [ '@comment',
    '     Osaka University.  This dictionary also includes: data extensions ' ],
  [ '@comment',
    '     for structural genomics projects recommended by International Task ' ],
  [ '@comment',
    '     Force on Data Deposition and Archiving (including protein production), ' ],
  [ '@comment', '     NMR and 3D electron microscopy.' ],
  [ '@comment',
    '                                                                           ' ],
  [ '@comment',
    '                                                                           ' ],
  [ '@comment', '                               Header Section ' ],
  [ '@comment',
    '                                                                           ' ],
  [ '@comment',
    '#############################################################################' ],
  [ 'data_',
    'mmcif_pdbx.dic',
    [ [ 'item_', '_datablock.id', 'mmcif_pdbx.dic' ],
      [ 'item_',
        '_datablock.description',
        '\n     This data block holds the Protein Data Bank Exchange Data dictionary.' ],
      [ 'item_', '_dictionary.title', 'mmcif_pdbx.dic' ],
      [ 'item_', '_dictionary.moniker', 'pdbx' ],
      [ 'item_', '_dictionary.datablock_id', 'mmcif_pdbx.dic' ],
      [ 'item_', '_dictionary.version', '1.021' ],
      [ 'loop_',
        [ '_dictionary_history.version',
          '_dictionary_history.update',
          '_dictionary_history.revision' ],
        [ '0.001',
          '2000-05-23',
          '\n Initial revision based on RCSB local dictionaries\n and EBI database schema.  J. Westbrook and J. Ionides',
          '0.002',
          '2000-06-06',
          ' Changes: (jdw)\n   + Added items in refine_hist and exptl_crystal_grow',
          '0.003',
          '2000-10-16',
          '  Changes: (jdw)\n    + Move enumerations to examples for:\n          _pdbx_nmr_exptl.type           \n          _pdbx_nmr_ensemble.conformer_selection_criteria\n          _pdbx_nmr_software.name                        \n          _struct_keywords.pdbx_keywords                 \n    + Added _atom_site_anisotrop.pdbx_auth_* items   \n    + Added _struct_mon_prot_cis.pdbx_               \n    + Added _struct_conn.pdbx_ptnr3_auth_* items.    ',
          '0.004',
          '2000-10-24',
          ' Changes: (jdw)\n  + Added optional timestamps to data items.  \n  + Repositioned parent items in PDBX_POLY_SEQ_SCHEME.',
          '0.005',
          '2000-10-25',
          ' Changes: (jdw)\n  + drop category PDBX_DATABASE_REMARK',
          '0.006',
          '2000-10-26',
          '\n Changes: (jdw)\n + Added _exptl_crystal_grow.rcsb_pH_range.\n + Added category PDBX_STRUCT_SHEET_HBOND to hold single hydrogen\n   bond registration between beta sheet strands.',
          '0.007',
          '2000-11-09',
          '\n Changes: (jdw)\n + Added mandatory codes for _exptl_crystal_grow.pdbx_details and \n   _exptl_crystal_grow.pdbx_pH_range.\n + Added missing data types and category ids in category \n   pdbx_struct_sheet_hbond.\n + Updated data type for atom name references to atcode.  \n + Updated aliases for _refine.overall_SU_B and _refine.overall_SU_ML.\n + Trimmed line lengths > 80 characters. ',
          '0.008',
          '2001-01-22',
          '\n Changes: (jdw)\n + Added item _pdbx_database_status.dep_release_code_sequence',
          '0.009',
          '2001-03-02',
          '\n Changes: (jdw)\n +  Added missing mandatory codes and reset aliases for\n    _diffrn_radiation.pdbx_wavelenth_list.',
          '0.010',
          '2001-04-18',
          '\n Changes: (jdw)\n +  Added missing _item_type.codes for \n         _refine_hist.number_atoms_solvent\n         _refine_hist.number_atoms_total\n         _refine.overall_SU_B\n         _refine.overall_SU_ML\n         _refine.pdbx_overall_ESU_R\n + Added CCP4 data items output from REFMAC V5.',
          '0.011',
          '2001-06-14',
          '\n Changes: (jdw)\n +  Moved enumerations to examples for _pdbx_nmr_spectrometer.field_strength \n +  _atom_site.pdb_auth_atom_name data type changed from code to atcode.',
          '0.012',
          '2001-06-28',
          '\n Changes: (jdw)\n +  _atom_site_anisotrop.pdbx_auth_* aliases corrected.\n +  adjust regex for date:time type',
          '0.013',
          '2001-07-12',
          '\n Changes: (jdw)\n +  enumerations->examples for _pdbx_nmr_representative.selection_criteria and\n    _pdbx_database_related.db_name.\n +  add regex for int and float ranges',
          '0.014',
          '2002-02-28',
          '\n Changes: (jdw)\n +  Add items for NMR model and PDB insert code to GEOM_BOND, GEOM_ANGLE,\n    GEOM_CONTACT, and GEOM_TORSION categories.',
          '0.015',
          '2002-05-22',
          '\n Changes: (jdw)\n +  Add PDBX_ENTITY_NONPOLY and fix missing data type codes.',
          '0.016',
          '2002-08-05',
          '\n Changes: (jdw)\n +  Add PDBX_AUDIT category.',
          '0.017',
          '2002-10-05',
          '\n Changes: (jdw)\n + Final corrections of recommendations for  NMR and X-ray deposition for\n   structural genomics.  ',
          '0.018',
          '2003-07-07',
          '\n Changes: (jdw)\n +  Integrated protein production data items. ',
          '0.019',
          '2003-09-21',
          '\n Changes: (jdw/hy)\n + Added additional phasing items for PDB_EXTRACT.   ',
          '0.020',
          '2003-10-30',
          '\n Changes: (jdw/zf)\n + Fix cases where category keys are not flagged as mandatory\n + Add category PDBX_AUDIT_AUTHOR',
          '1.000',
          '2004-04-22',
          '\n Changes:  (jdw)\n + New packaging and naming\n + Integrate 3d em definitions\n + Relax enumerations to comply with existing legacy data.\n + Version to 1.000',
          '1.001',
          '2004-05-21',
          '\n Changes:  (jdw)\n + Fixed incorrect category in _pdbx_audit_author.address\n + Added _citation.pdbx_database_id_DOI\n + Fixed placement of _chem_comp.pdbx_align. Moved this item\n   to _chem_comp_atom.pdbx_align',
          '1.002',
          '2004-06-09',
          '\n Changes:  (jdw)\n + Added categories PDBX_DOMAIN, PDBX_SEQUENCE_RANGE, \n   PDBX_FEATURE_ENTRY, PDBX_FEATURE_ASSEMBLY, PDBX_FEATURE_DOMAIN,\n   and PDBX_FEATURE_SEQUENCE_RANGE.',
          '1.003',
          '2004-06-16',
          '\n Changes:  (jdw)\n + Fix miscellaneous unresolved links. ',
          '1.004',
          '2004-08-04',
          '\n Changes:  (jdw)\n + Fix many data type errors in phasing section.\n + Add pdbx_contact_author ',
          '1.005',
          '2004-09-28',
          '\n Changes: (zkf,jdw)\n + Add missing pdbx_contact_author.postal_code item\n + Add missing pdbx_contact_author.city item\n + Add missing pdbx_contact_author.state_province item\n + Add enumerations in pdbx_contact_author.id\n + Modify enumerations in pdbx_contact_author.role',
          '1.006',
          '2004-10-10',
          '\n Changes: (jdw)\n + Update enumerations in cryo-em dictionary. ',
          '1.007',
          '2004-11-02',
          '\n Changes: (jdw)\n + added pdbx_entity_prod_protocol, pdbx_exptl_crystal_grow_sol, \n   pdbx_exptl_crystal_grow_comp, and pdbx_exptl_crystal_cryo_treatment \n   categories\n + relax enumerations in cryo-em dictionary.\n + remove upper-bound for refine.occupancy_max',
          '1.008',
          '2004-11-15',
          '\n Changes: (jdw)\n + fix typo in category save frame name pdbx_exptl_crystal_grow_comp.',
          '1.009',
          '2005-01-13',
          '\n Changes: (zkf)\n + add _pdbx_database_status.status_code_sf item\n + add _pdbx_database_status.date_of_sf_release item\n + add _pdbx_database_status.status_code_mr item\n + add _pdbx_database_status.date_of_mr_release item\n + add _pdbx_database_status.SG_entry item\n + add _entity_poly.pdbx_target_identifier item\n + add pdbx_SG_project category\n + fixed enumeration values for _software.language',
          '1.010',
          '2005-01-25',
          '\n Changes: (jdw)\n  + change mandatory doc for  _pdbx_database_status.SG_entry \n  + fix key in category pdbx_database_PDB_obs_spr',
          '1.011',
          '2005-03-08',
          '\n Changes: (jdw)\n  + Included extensions from mmCIF dictionary...\n Changes: (fzk)\n  + add _pdbx_refine_tls_group.beg_auth_asym_id\n  + add _pdbx_refine_tls_group.end_auth_asym_id\n  + add enumeration values \'SGC\' and \'NPPSFA\' for\n    _pdbx_SG_project.initial_of_center\n  + add enumeration values \'Structural Genomics Consortium\' and\n        \'National Project on Protein Structural and Functional Analyses\'\n        for _pdbx_SG_project.full_name_of_center \n  + fix _item_description.description for _entity_poly.pdbx_target_identifier',
          '1.012',
          '2005-03-29',
          '\n Changes: (kh/jdw)\n  + added preliminary set of extensions for macromolecular powder diffraction\n    experiments, including:\n  _pdbx_exptl_pd (category)\n  _pdbx_exptl_pd.entry.id\n  _pdbx_exptl_pd.spec_preparation_pH\n  _pdbx_exptl_pd.spec_preparation_pH_range\n  _pdbx_exptl_pd.spec_preparation\n  _diffrn_radiation.pdbx_analyzer \n  _refine.pdbx_pd_number_of_powder_patterns\n  _refine.pdbx_pd_number_of_points\n  _refine.pdbx_pd_meas_number_of_points\n  _refine.pdbx_pd_proc_ls_prof_R_factor\n  _refine.pdbx_pd_proc_ls_prof_wR_factor\n  _refine.pdbx_pd_Marquardt_correlation_coeff\n  _refine.pdbx_pd_Fsqrd_R_factor \n  _refine.pdbx_pd_ls_matrix_band_width\n Changes: (fzk)\n  removed \'National Project on Protein Structural and Functional Analyses\' from\n         _pdbx_SG_project.full_name_of_center,\n  added \'National Project on Protein Structural and Functional Analyses\' to\n         _pdbx_SG_project.project_name',
          '1.013',
          '2005-04-05',
          '\n Changes: (jdw)\n + Temporarily remove definitions from Image CIF dictionary\n + Added mandatory code for _cell.reciprocal_angle_beta\n + Added mandatory code for _pdbx_entity_nonpoly.entity_id \n + Fixed category attribute for  _reflns_shell.pdbx_chi_squared\n + Changed mandatory code for _software.citation_id ',
          '1.014',
          '2005-04-08',
          '\n Changes: (jdw)\n + remove _struct_keywords.text from category key',
          '1.015',
          '2005-04-15',
          '\n Changes: (jdw)\n + remove _entity_keywords.text from category key',
          '1.016',
          '2005-06-24',
          '\n Changes: (jdw)\n + aliases repointed for items in category pdbx_database_remark\n + Added the following items:\n  _diffrn_detector.pdbx_frames_total\n  _diffrn_detector.pdbx_collection_time_total\n  _refine.pdbx_ls_sigma_Fsqd\n  _reflns.pdbx_Rmerge_I_all\n  _reflns.pdbx_netI_over_sigmaI\n  _reflns.pdbx_res_netI_over_av_sigmaI_2\n  _reflns.pdbx_res_netI_over_sigmaI_2\n  _reflns_shell.pdbx_netI_over_sigmaI_all\n  _reflns_shell.pdbx_netI_over_sigmaI_obs\n  _struct_biol.pdbx_formula_weight\n  _struct_biol.pdbx_formula_weight_method\n  _struct_biol_gen.pdbx_PDB_order\n  _phasing_MAD.pdbx_number_data_sets\n  _phasing_MAD.pdbx_anom_scat_method\n  _phasing_MIR.pdbx_number_derivatives\n  _phasing_set.pdbx_temp_details\n\n + miscellaneous editorial changes in descriptions and enumerations.',
          '1.017',
          '2005-06-27',
          '\n Changes: (BMcM)\n + added _reflns.threshold_expression and _publ_author.email\n + numerous minor changes to wordings of definitions to align with mmcif\n   version 2.0.09 as released on International Tables G CD-ROM',
          '1.018',
          '2005-07-14',
          '\n Changes: (jdw)\n + Added PDB_SOLN_SCATTER* definitions\n + Added _refln.pdbx_[F,I]_[minus,plus] and \n         _refln.pdbx_[F,I]_[minus,plus]_sigma\n + Extend enumeration for _pdbx_database_status.status_code_sf',
          '1.019',
          '2005-07-21',
          '\n Changes: (jdw)\n + Added _struct.pdbx_formula_weight and  _struct.pdbx_formula_weight_method\n + Added _refln.pdbx_HL_A_iso, _refln.pdbx_HL_B_iso, _refln.pdbx_HL_C_iso,\n          and _refln.pdbx_HL_D_iso \n + Added _phasing_set.pdbx_d_res_low/high _pdbx_phasing_MR.native_data_set_id\n + Added category pdbx_feature_monomer',
          '1.020',
          '2005-08-23',
          '\n Changes: (jdw)\n + Correct typos in pdbx_feature_monomer.\n + Correct typos in examples in categories refine and publ_manuscript_incl',
          '1.021',
          '2005-10-12',
          '\n Changes: (jdw)\n + added categories pdbx_refine_component, pdbx_chem_comp_descriptor,\n   pdbx_chem_comp_identifier, pdbx_chem_comp_import,pdbx_chem_comp_atom_edit,\n   pdbx_chem_comp_bond_edit, and pdbx_chem_comp_audit.\n + added stereochemistry and aromaticity flags to chem_comp_atom and \n   chem_comp_bond.\n + added alternate component and atom nomenclature to chem_comp_atom.  \n + added alternate model coordinates to chem_comp_atom category',
          [ '@comment', '## EOF mmcif_pdbx-header.dic ' ],
          [ '@comment',
            '##########################################################################' ],
          [ '@comment',
            ' File:  mmcif_pdbx-data.dic                                             ' ],
          [ '@comment', ' ' ],
          [ '@comment',
            '               Protein Data Bank Exchange Data Dictionary' ],
          [ '@comment', '                           Data Section ' ],
          [ '@comment', ' ' ],
          [ '@comment',
            '                                                                        ' ],
          [ '@comment',
            '##########################################################################' ],
          [ '@comment', '#################' ],
          [ '@comment', '# SUB_CATEGORY ##' ],
          [ '@comment', '#################' ] ] ],
      [ 'loop_',
        [ '_sub_category.id', '_sub_category.description' ],
        [ 'cartesian_coordinate',
          '              The collection of x, y, and z components of a position specified\n               with reference to a Cartesian (orthogonal angstrom) coordinate\n               system.',
          'cartesian_coordinate_esd',
          '              The collection of estimated standard deviations of the x, y, and\n               z components of a position specified with reference to a\n               Cartesian (orthogonal angstrom) coordinate system.',
          'fractional_coordinate',
          '              The collection of x, y, and z components of a position specified\n               with reference to unit cell directions.',
          'fractional_coordinate_esd',
          '              The collection of estimated standard deviations of the x, y, and\n               z components of a position specified with reference to unit cell\n               directions.',
          'matrix',
          '              The collection of elements of a matrix.',
          'miller_index',
          '              The collection of h, k, and l components of the Miller index of\n               a reflection.',
          'cell_length',
          '              The collection of a, b, and c axis lengths of a unit cell.',
          'cell_length_esd',
          '              The collection of estimated standard deviations of the a, b, and\n               c axis lengths of a unit cell.',
          'cell_angle',
          '              The collection of alpha, beta, and gamma angles of a unit cell.',
          'cell_angle_esd',
          '              The collection of estimated standard deviations of the alpha,\n               beta, and gamma angles of a unit cell.',
          'mm_atom_site_auth_label',
          '              The collection of asym id, atom id, comp id and seq id\n               components of an author\'s alternative specification for\n               a macromolecular atom site.',
          'mm_atom_site_label',
          '              The collection of alt id, asym id, atom id, comp id and seq id\n               components of the label for a macromolecular atom site.',
          'vector',
          '              The collection of elements of a vector.',
          [ '@comment', '########################' ],
          [ '@comment', '# CATEGORY_GROUP_LIST ##' ],
          [ '@comment', '########################' ] ] ],
      [ 'loop_',
        [ '_category_group_list.id',
          '_category_group_list.parent_id',
          '_category_group_list.description' ],
        [ 'inclusive_group',
          '.',
          '              Categories that belong to the macromolecular dictionary.',
          'atom_group',
          'inclusive_group',
          '              Categories that describe the properties of atoms.',
          'audit_group',
          'inclusive_group',
          '              Categories that describe dictionary maintenance and\n               identification.',
          'cell_group',
          'inclusive_group',
          '              Categories that describe the unit cell.',
          'chemical_group',
          'inclusive_group',
          '              Categories that describe chemical properties and nomenclature.',
          'chem_comp_group',
          'inclusive_group',
          '              Categories that describe components of chemical structure.',
          'chem_link_group',
          'inclusive_group',
          '              Categories that describe links between components of\n               chemical structure.',
          'citation_group',
          'inclusive_group',
          '              Categories that provide bibliographic references.',
          'computing_group',
          'inclusive_group',
          '              Categories that describe the computational details of the\n               experiment.',
          'compliance_group',
          'inclusive_group',
          '              Categories that are included in this dictionary specifically to\n               comply with previous dictionaries.',
          'database_group',
          'inclusive_group',
          '              Categories that hold references to entries in databases that\n               contain related information.',
          'diffrn_group',
          'inclusive_group',
          '              Categories that describe details of the diffraction experiment.',
          'em_group',
          'inclusive_group',
          '              Categories that describe 3-dimensional electron microscopy.',
          'entity_group',
          'inclusive_group',
          '              Categories that describe chemical entities.',
          'entry_group',
          'inclusive_group',
          '              Categories that pertain to the entire data block.',
          'exptl_group',
          'inclusive_group',
          '              Categories that hold details of the experimental conditions.',
          'geom_group',
          'inclusive_group',
          '              Categories that hold details of molecular and crystal geometry.',
          'iucr_group',
          'inclusive_group',
          '              Categories that are used for manuscript submission and\n               internal processing by the staff of the International Union of\n               Crystallography.',
          'pdb_group',
          'database_group',
          '              Categories that pertain to the file-format or data-processing\n               codes used by the Protein Data Bank.',
          'phasing_group',
          'inclusive_group',
          '              Categories that describe phasing.',
          'refine_group',
          'inclusive_group',
          '              Categories that describe refinement.',
          'refln_group',
          'inclusive_group',
          '              Categories that describe the details of reflection measurements.',
          'struct_group',
          'inclusive_group',
          '              Categories that contain details about the crystallographic\n               structure.',
          'symmetry_group',
          'inclusive_group',
          '              Categories that describe symmetry information.',
          'valence_group',
          'inclusive_group',
          '              Categories that describe ion valence properties and relationships.',
          'pdbx_group',
          'inclusive_group',
          '             Categories which are part of PDB data exchange protocol.',
          'pdbx_erf_group',
          'inclusive_group',
          '             Categories which are used by RCSB PDB to store derived\n              and computed data. ',
          'ccp4_group',
          'inclusive_group',
          '             Categories from the CCP4 harvest dictionary.',
          'ndb_group',
          'inclusive_group',
          '             Categories which are used by the Nucleic Acid Database.',
          'rcsb_group',
          'inclusive_group',
          '             Categories which are used internally by the RCSB PDB.',
          'solution_scattering_group',
          'inclusive_group',
          '             Categories which describe the details of solution scattering experiments.',
          [ '@comment', '###################' ],
          [ '@comment', '# ITEM_TYPE_LIST ##' ],
          [ '@comment', '###################' ],
          [ '@comment',
            '  The regular expressions defined here are not compliant' ],
          [ '@comment',
            '  with the POSIX 1003.2 standard as they include the' ],
          [ '@comment',
            '  \'\\n\' and \'\\t\' special characters.  These regular expressions' ],
          [ '@comment',
            '  have been tested using version 0.12 of Richard Stallman\'s' ],
          [ '@comment',
            '  GNU regular expression library in POSIX mode.' ],
          [ '@comment',
            '                                                                            ' ],
          [ '@comment',
            ' For some data items, a standard syntax is assumed. The syntax is           ' ],
          [ '@comment',
            '   described for each data item in the dictionary, but is summarized here:  ' ],
          [ '@comment',
            '                                                                            ' ],
          [ '@comment',
            '   Names:     The family name(s) followed by a comma, precedes the first    ' ],
          [ '@comment',
            '              name(s) or initial(s).                                        ' ],
          [ '@comment',
            '                                                                            ' ],
          [ '@comment',
            '   Telephone numbers:                                                       ' ],
          [ '@comment',
            '              The international code is given in brackets and any extension ' ],
          [ '@comment',
            '              number is preceded by \'ext\'.                                  ' ],
          [ '@comment',
            '                                                                            ' ],
          [ '@comment',
            '   Dates:     In the form yyyy-mm-dd.                                       ' ],
          [ '@comment',
            '                                                                            ' ],
          [ '@comment',
            '#############################################################################' ] ] ],
      [ 'loop_',
        [ '_item_type_list.code',
          '_item_type_list.primitive_code',
          '_item_type_list.construct',
          '_item_type_list.detail' ],
        [ 'code',
          'char',
          '[_,.;:"&<>()/\\{}\'`~!@#$%A-Za-z0-9*|+-]*',
          '              code item types/single words ...',
          'ucode',
          'uchar',
          '[_,.;:"&<>()/\\{}\'`~!@#$%A-Za-z0-9*|+-]*',
          '              code item types/single words  (case insensitive) ...',
          'line',
          'char',
          '[][ \\t_(),.;:"&<>/\\{}\'`~!@#$%?+=*A-Za-z0-9|^-]*',
          '              char item types / multi-word items ...',
          'uline',
          'uchar',
          '[][ \\t_(),.;:"&<>/\\{}\'`~!@#$%?+=*A-Za-z0-9|^-]*',
          '              char item types / multi-word items (case insensitive)...',
          'text',
          'char',
          '[][ \\n\\t()_,.;:"&<>/\\{}\'`~!@#$%?+=*A-Za-z0-9|^-]*',
          '              text item types / multi-line text ...',
          'int',
          'numb',
          '-?[0-9]+',
          '              int item types are the subset of numbers that are the negative \n               or positive integers.',
          'float',
          'numb',
          '-?(([0-9]+)[.]?|([0-9]*[.][0-9]+))([(][0-9]+[)])?([eE][+-]?[0-9]+)?',
          '              float item types are the subset of numbers that are the floating\n               numbers.',
          'name',
          'uchar',
          '_[_A-Za-z0-9]+\\.[][_A-Za-z0-9%-]+',
          '              name item types take the form...',
          'idname',
          'uchar',
          '[_A-Za-z0-9]+',
          '              idname item types take the form...',
          'any',
          'char',
          '.*',
          '              A catch all for items that may take any form...',
          'yyyy-mm-dd',
          'char',
          '[0-9]?[0-9]?[0-9][0-9]-[0-9]?[0-9]-[0-9][0-9]',
          '\n               Standard format for CIF dates.',
          'uchar3',
          'uchar',
          '[+]?[A-Za-z0-9][A-Za-z0-9][A-Za-z0-9]',
          '\n               data item for 3 character codes',
          'uchar1',
          'uchar',
          '[+]?[A-Za-z0-9]',
          '\n               data item for 1 character codes',
          'symop',
          'char',
          '([1-9]|[1-9][0-9]|1[0-8][0-9]|19[0-2])(_[1-9][1-9][1-9])?',
          '              symop item types take the form n_klm, where n refers to the\n               symmetry operation that is applied to the coordinates in the\n               ATOM_SITE category identified by _atom_site_label.  It must\n               match a number given in _symmetry_equiv_pos_site_id.\n\n               k, l, and m refer to the translations that are subsequently\n               applied to the symmetry transformed coordinates to generate\n               the atom used.  These translations (x,y,z) are related to\n               (k,l,m) by\n                     k = 5 + x\n                     l = 5 + y\n                     m = 5 + z\n               By adding 5 to the translations, the use of negative numbers\n               is avoided.',
          'atcode',
          'char',
          '[][ _(),.;:"&<>/\\{}\'`~!@#$%?+=*A-Za-z0-9|^-]*',
          '              Character data type for atom names  ...',
          'yyyy-mm-dd:hh:mm',
          'char',
          '[0-9]?[0-9]?[0-9][0-9]-[0-9]?[0-9](-[0-9]?[0-9])?(:[0-9]?[0-9]:[0-9][0-9])?',
          '\n               Standard format for CIF dates with optional time stamp.',
          'fax',
          'uchar',
          '[_,.;:"&<>/\\{}\'`~!@#$%A-Za-z0-9*|+-]*',
          '              code item types/single words  (case insensitive) ...',
          'phone',
          'uchar',
          '[_,.;:"&<>/\\{}\'`~!@#$%A-Za-z0-9*|+-]*',
          '              code item types/single words  (case insensitive) ...',
          'email',
          'uchar',
          '[_,.;:"&<>/\\{}\'`~!@#$%A-Za-z0-9*|+-]*',
          '              code item types/single words  (case insensitive) ...',
          'int-range',
          'numb',
          '-?[0-9]+(--?[0-9]+)?',
          '              int item types are the subset of numbers that are the negative \n               or positive integers with optional range.',
          'float-range',
          'numb',
          '-?(([0-9]+)[.]?|([0-9]*[.][0-9]+))([(][0-9]+[)])?([eE][+-]?[0-9]+)?(--?(([0-9]+)[.]?|([0-9]*[.][0-9]+))([(][0-9]+[)])?([eE][+-]?[0-9]+)?)?',
          '              int item types are the subset of numbers that are the floating\n               numbers.',
          'code30',
          'char',
          '.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?.?',
          'A string value, not allowed to exceed 30 characters.',
          'binary',
          'char',
          '\\n--CIF-BINARY-FORMAT-SECTION--\\n\\\n[][ \\n\\t()_,.;:"&<>/\\{}\'`~!@#$%?+=*A-Za-z0-9|^-]*\\\n\\n--CIF-BINARY-FORMAT-SECTION----',
          '              binary items are presented as MIME-like ascii-encoded\n               sections in an imgCIF.  In a CBF, raw octet streams\n               are used to convey the same information.',
          [ '@comment', '####################' ],
          [ '@comment', '# ITEM_UNITS_LIST ##' ],
          [ '@comment', '####################' ] ] ],
      [ 'loop_',
        [ '_item_units_list.code',
          '_item_units_list.detail',
          [ '@comment', '' ] ],
        [ 'metres',
          'metres (metres)',
          'centimetres',
          'centimetres (metres * 10^( -2))',
          'millimetres',
          'millimetres (metres * 10^( -3))',
          'nanometres',
          'nanometres  (metres * 10^( -9))',
          'angstroms',
          'angstroms   (metres * 10^(-10))',
          'picometres',
          'picometres  (metres * 10^(-12))',
          'femtometres',
          'femtometres (metres * 10^(-15))',
          'reciprocal_metres',
          'reciprocal metres (metres^(-1))',
          'reciprocal_centimetres',
          'reciprocal centimetres ((metres * 10^( -2))^(-1))',
          'reciprocal_millimetres',
          'reciprocal millimetres ((metres * 10^( -3))^(-1))',
          'reciprocal_nanometres',
          'reciprocal nanometres  ((metres * 10^( -9))^(-1))',
          'reciprocal_angstroms',
          'reciprocal angstroms   ((metres * 10^(-10))^(-1))',
          'reciprocal_picometres',
          'reciprocal picometres  ((metres * 10^(-12))^(-1))',
          'nanometres_squared',
          'nanometres squared (metres * 10^( -9))^2',
          'angstroms_squared',
          'angstroms squared  (metres * 10^(-10))^2',
          '8pi2_angstroms_squared',
          '8pi^2 * angstroms squared (metres * 10^(-10))^2',
          'picometres_squared',
          'picometres squared (metres * 10^(-12))^2',
          'nanometres_cubed',
          'nanometres cubed (metres * 10^( -9))^3',
          'angstroms_cubed',
          'angstroms cubed  (metres * 10^(-10))^3',
          'picometres_cubed',
          'picometres cubed (metres * 10^(-12))^3',
          'kilopascals',
          'kilopascals',
          'gigapascals',
          'gigapascals',
          'hours',
          'hours',
          'minutes',
          'minutes',
          'seconds',
          'seconds',
          'microseconds',
          'microseconds',
          'degrees',
          'degrees (of arc)',
          'degrees_per_minute',
          'degrees (of arc) per minute',
          'celsius',
          'degrees (of temperature) Celsius',
          'kelvins',
          'degrees (of temperature) Kelvin',
          'electrons',
          'electrons',
          'electrons_squared',
          'electrons squared',
          'electrons_per_nanometres_cubed',
          ' electrons per nanometres cubed (electrons/(metres * 10^( -9))^(-3))',
          'electrons_per_angstroms_cubed',
          ' electrons per angstroms cubed (electrons/(metres * 10^(-10))^(-3))',
          'electrons_per_picometres_cubed',
          ' electrons per picometres cubed (electrons/(metres * 10^(-12))^(-3))',
          'kilowatts',
          'kilowatts',
          'milliamperes',
          'milliamperes',
          'kilovolts',
          'kilovolts',
          'arbitrary',
          ' arbitrary system of units.',
          'angstroms_degrees',
          'angstroms * degrees',
          'degrees_squared',
          'degrees squared',
          'mg_per_ml',
          'milliliter per milligram',
          'ml_per_min',
          'milliliter per minute',
          'milliliters',
          'liter / 1000',
          'milligrams',
          'grams / 1000',
          'megadaltons',
          'megadaltons',
          'microns_squared',
          'microns squared',
          'microns',
          'microns',
          [ '@comment',
            '     \'millimeters\'            \'millimeters\'' ],
          'electrons_angstrom_squared',
          'electrons square angstrom',
          'electron_volts',
          'electron volts',
          [ '@comment',
            '     \'mg/ml\'                  \'mg per milliliter\'' ],
          'millimolar',
          'millimolar',
          'megagrams_per_cubic_metre',
          'megagrams per cubic metre',
          'pixels_per_millimetre',
          'pixels per millimetre',
          'counts',
          'counts',
          'counts_per_photon',
          'counts per photon' ] ],
      [ 'loop_',
        [ '_item_units_conversion.from_code',
          '_item_units_conversion.to_code',
          '_item_units_conversion.operator',
          '_item_units_conversion.factor',
          [ '@comment', '##' ] ],
        [ 'metres',
          'centimetres',
          '*',
          '1.0E+02',
          'metres',
          'millimetres',
          '*',
          '1.0E+03',
          'metres',
          'nanometres',
          '*',
          '1.0E+09',
          'metres',
          'angstroms',
          '*',
          '1.0E+10',
          'metres',
          'picometres',
          '*',
          '1.0E+12',
          'metres',
          'femtometres',
          '*',
          '1.0E+15',
          'centimetres',
          'millimetres',
          '*',
          '1.0E+01',
          'centimetres',
          'nanometres',
          '*',
          '1.0E+07',
          'centimetres',
          'angstroms',
          '*',
          '1.0E+08',
          'centimetres',
          'picometres',
          '*',
          '1.0E+10',
          'centimetres',
          'femtometres',
          '*',
          '1.0E+13',
          'millimetres',
          'centimetres',
          '*',
          '1.0E-01',
          'millimetres',
          'nanometres',
          '*',
          '1.0E+06',
          'millimetres',
          'angstroms',
          '*',
          '1.0E+07',
          'millimetres',
          'picometres',
          '*',
          '1.0E+09',
          'millimetres',
          'femtometres',
          '*',
          '1.0E+12',
          'nanometres',
          'centimetres',
          '*',
          '1.0E-07',
          'nanometres',
          'millimetres',
          '*',
          '1.0E-06',
          'nanometres',
          'angstroms',
          '*',
          '1.0E+01',
          'nanometres',
          'picometres',
          '*',
          '1.0E+03',
          'nanometres',
          'femtometres',
          '*',
          '1.0E+06',
          'angstroms',
          'centimetres',
          '*',
          '1.0E-08',
          'angstroms',
          'millimetres',
          '*',
          '1.0E-07',
          'angstroms',
          'nanometres',
          '*',
          '1.0E-01',
          'angstroms',
          'picometres',
          '*',
          '1.0E+02',
          'angstroms',
          'femtometres',
          '*',
          '1.0E+05',
          'picometres',
          'centimetres',
          '*',
          '1.0E-10',
          'picometres',
          'millimetres',
          '*',
          '1.0E-09',
          'picometres',
          'nanometres',
          '*',
          '1.0E-03',
          'picometres',
          'angstroms',
          '*',
          '1.0E-02',
          'picometres',
          'femtometres',
          '*',
          '1.0E+03',
          'femtometres',
          'centimetres',
          '*',
          '1.0E-13',
          'femtometres',
          'millimetres',
          '*',
          '1.0E-12',
          'femtometres',
          'nanometres',
          '*',
          '1.0E-06',
          'femtometres',
          'angstroms',
          '*',
          '1.0E-05',
          'femtometres',
          'picometres',
          '*',
          '1.0E-03',
          [ '@comment', '##' ],
          'reciprocal_centimetres',
          'reciprocal_millimetres',
          '*',
          '1.0E-01',
          'reciprocal_centimetres',
          'reciprocal_nanometres',
          '*',
          '1.0E-07',
          'reciprocal_centimetres',
          'reciprocal_angstroms',
          '*',
          '1.0E-08',
          'reciprocal_centimetres',
          'reciprocal_picometres',
          '*',
          '1.0E-10',
          'reciprocal_millimetres',
          'reciprocal_centimetres',
          '*',
          '1.0E+01',
          'reciprocal_millimetres',
          'reciprocal_nanometres',
          '*',
          '1.0E-06',
          'reciprocal_millimetres',
          'reciprocal_angstroms',
          '*',
          '1.0E-07',
          'reciprocal_millimetres',
          'reciprocal_picometres',
          '*',
          '1.0E-09',
          'reciprocal_nanometres',
          'reciprocal_centimetres',
          '*',
          '1.0E+07',
          'reciprocal_nanometres',
          'reciprocal_millimetres',
          '*',
          '1.0E+06',
          'reciprocal_nanometres',
          'reciprocal_angstroms',
          '*',
          '1.0E-01',
          'reciprocal_nanometres',
          'reciprocal_picometres',
          '*',
          '1.0E-03',
          'reciprocal_angstroms',
          'reciprocal_centimetres',
          '*',
          '1.0E+08',
          'reciprocal_angstroms',
          'reciprocal_millimetres',
          '*',
          '1.0E+07',
          'reciprocal_angstroms',
          'reciprocal_nanometres',
          '*',
          '1.0E+01',
          'reciprocal_angstroms',
          'reciprocal_picometres',
          '*',
          '1.0E-02',
          'reciprocal_picometres',
          'reciprocal_centimetres',
          '*',
          '1.0E+10',
          'reciprocal_picometres',
          'reciprocal_millimetres',
          '*',
          '1.0E+09',
          'reciprocal_picometres',
          'reciprocal_nanometres',
          '*',
          '1.0E+03',
          'reciprocal_picometres',
          'reciprocal_angstroms',
          '*',
          '1.0E+01',
          [ '@comment', '##' ],
          'nanometres_squared',
          'angstroms_squared',
          '*',
          '1.0E+02',
          'nanometres_squared',
          'picometres_squared',
          '*',
          '1.0E+06',
          'angstroms_squared',
          'nanometres_squared',
          '*',
          '1.0E-02',
          'angstroms_squared',
          'picometres_squared',
          '*',
          '1.0E+04',
          'angstroms_squared',
          '8pi2_angstroms_squared',
          '*',
          '78.9568',
          'picometres_squared',
          'nanometres_squared',
          '*',
          '1.0E-06',
          'picometres_squared',
          'angstroms_squared',
          '*',
          '1.0E-04',
          [ '@comment', '##' ],
          'nanometres_cubed',
          'angstroms_cubed',
          '*',
          '1.0E+03',
          'nanometres_cubed',
          'picometres_cubed',
          '*',
          '1.0E+09',
          'angstroms_cubed',
          'nanometres_cubed',
          '*',
          '1.0E-03',
          'angstroms_cubed',
          'picometres_cubed',
          '*',
          '1.0E+06',
          'picometres_cubed',
          'nanometres_cubed',
          '*',
          '1.0E-09',
          'picometres_cubed',
          'angstroms_cubed',
          '*',
          '1.0E-06',
          [ '@comment', '##' ],
          'kilopascals',
          'gigapascals',
          '*',
          '1.0E-06',
          'gigapascals',
          'kilopascals',
          '*',
          '1.0E+06',
          [ '@comment', '##' ],
          'hours',
          'minutes',
          '*',
          '6.0E+01',
          'hours',
          'seconds',
          '*',
          '3.6E+03',
          'hours',
          'microseconds',
          '*',
          '3.6E+09',
          'minutes',
          'hours',
          '/',
          '6.0E+01',
          'minutes',
          'seconds',
          '*',
          '6.0E+01',
          'minutes',
          'microseconds',
          '*',
          '6.0E+07',
          'seconds',
          'hours',
          '/',
          '3.6E+03',
          'seconds',
          'minutes',
          '/',
          '6.0E+01',
          'seconds',
          'microseconds',
          '*',
          '1.0E+06',
          'microseconds',
          'hours',
          '/',
          '3.6E+09',
          'microseconds',
          'minutes',
          '/',
          '6.0E+07',
          'microseconds',
          'seconds',
          '/',
          '1.0E+06',
          [ '@comment', '##' ],
          'celsius',
          'kelvins',
          '-',
          '273.0',
          'kelvins',
          'celsius',
          '+',
          '273.0',
          [ '@comment', '##' ],
          'electrons_per_nanometres_cubed',
          'electrons_per_angstroms_cubed',
          '*',
          '1.0E+03',
          'electrons_per_nanometres_cubed',
          'electrons_per_picometres_cubed',
          '*',
          '1.0E+09',
          'electrons_per_angstroms_cubed',
          'electrons_per_nanometres_cubed',
          '*',
          '1.0E-03',
          'electrons_per_angstroms_cubed',
          'electrons_per_picometres_cubed',
          '*',
          '1.0E+06',
          'electrons_per_picometres_cubed',
          'electrons_per_nanometres_cubed',
          '*',
          '1.0E-09',
          'electrons_per_picometres_cubed',
          'electrons_per_angstroms_cubed',
          '*',
          '1.0E-06',
          [ '@comment', '##' ],
          [ '@comment', '## EOF mmcif_pdbx-data.dic' ],
          [ '@comment',
            '##########################################################################' ],
          [ '@comment', ' File:  mmcif_std-def-1.dic' ],
          [ '@comment',
            '                      mmCIF Data Dictionary (standard defintions)' ],
          [ '@comment',
            ' This data dictionary contains the standard mmCIF data definitions.' ],
          [ '@comment', '                           Defintion Section 1' ],
          [ '@comment',
            '##########################################################################' ],
          [ '@comment', '##############' ],
          [ '@comment', '# ATOM_SITE ##' ],
          [ '@comment', '##############' ] ] ],
      [ 'save_',
        'atom_site',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ATOM_SITE category record details about\n               the atom sites in a macromolecular crystal structure, such as\n               the positional coordinates, atomic displacement parameters,\n               magnetic moments and directions.\n\n               The data items for describing anisotropic atomic\n               displacement factors are only used if the corresponding items\n               are not given in the ATOM_SITE_ANISOTROP category.' ],
          [ 'item_', '_category.id', 'atom_site' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_atom_site.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'atom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _atom_site.group_PDB\n    _atom_site.type_symbol\n    _atom_site.label_atom_id\n    _atom_site.label_comp_id\n    _atom_site.label_asym_id\n    _atom_site.label_seq_id\n    _atom_site.label_alt_id\n    _atom_site.Cartn_x\n    _atom_site.Cartn_y\n    _atom_site.Cartn_z\n    _atom_site.occupancy\n    _atom_site.B_iso_or_equiv\n    _atom_site.footnote_id\n    _atom_site.auth_seq_id\n    _atom_site.id\n    ATOM N  N   VAL  A  11  .  25.369  30.691  11.795  1.00  17.93  .  11   1\n    ATOM C  CA  VAL  A  11  .  25.970  31.965  12.332  1.00  17.75  .  11   2\n    ATOM C  C   VAL  A  11  .  25.569  32.010  13.808  1.00  17.83  .  11   3\n    ATOM O  O   VAL  A  11  .  24.735  31.190  14.167  1.00  17.53  .  11   4\n    ATOM C  CB  VAL  A  11  .  25.379  33.146  11.540  1.00  17.66  .  11   5\n    ATOM C  CG1 VAL  A  11  .  25.584  33.034  10.030  1.00  18.86  .  11   6\n    ATOM C  CG2 VAL  A  11  .  23.933  33.309  11.872  1.00  17.12  .  11   7\n    ATOM N  N   THR  A  12  .  26.095  32.930  14.590  1.00  18.97  4  12   8\n    ATOM C  CA  THR  A  12  .  25.734  32.995  16.032  1.00  19.80  4  12   9\n    ATOM C  C   THR  A  12  .  24.695  34.106  16.113  1.00  20.92  4  12  10\n    ATOM O  O   THR  A  12  .  24.869  35.118  15.421  1.00  21.84  4  12  11\n    ATOM C  CB  THR  A  12  .  26.911  33.346  17.018  1.00  20.51  4  12  12\n    ATOM O  OG1 THR  A  12  3  27.946  33.921  16.183  0.50  20.29  4  12  13\n    ATOM O  OG1 THR  A  12  4  27.769  32.142  17.103  0.50  20.59  4  12  14\n    ATOM C  CG2 THR  A  12  3  27.418  32.181  17.878  0.50  20.47  4  12  15\n    ATOM C  CG2 THR  A  12  4  26.489  33.778  18.426  0.50  20.00  4  12  16\n    ATOM N  N   ILE  A  13  .  23.664  33.855  16.884  1.00  22.08  .  13  17\n    ATOM C  CA  ILE  A  13  .  22.623  34.850  17.093  1.00  23.44  .  13  18\n    ATOM C  C   ILE  A  13  .  22.657  35.113  18.610  1.00  25.77  .  13  19\n    ATOM O  O   ILE  A  13  .  23.123  34.250  19.406  1.00  26.28  .  13  20\n    ATOM C  CB  ILE  A  13  .  21.236  34.463  16.492  1.00  22.67  .  13  21\n    ATOM C  CG1 ILE  A  13  .  20.478  33.469  17.371  1.00  22.14  .  13  22\n    ATOM C  CG2 ILE  A  13  .  21.357  33.986  15.016  1.00  21.75  .  13  23\n    # - - - - data truncated for brevity - - - -\n    HETATM C C1 APS  C   .  1   4.171  29.012   7.116  0.58  17.27  1 300  101\n    HETATM C C2 APS  C   .  1   4.949  27.758   6.793  0.58  16.95  1 300  102\n    HETATM O O3 APS  C   .  1   4.800  26.678   7.393  0.58  16.85  1 300  103\n    HETATM N N4 APS  C   .  1   5.930  27.841   5.869  0.58  16.43  1 300  104\n    # - - - - data truncated for brevity - - - -' ] ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[1][1]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[1][1]_esd',
              'associated_esd',
              '_atom_site.aniso_U[1][1]',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][1]',
              'conversion_constant',
              '_atom_site.aniso_U[1][1]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][1]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][1]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[1][1]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_B[1][1].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[1][1]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[1][1]',
              'associated_value',
              '_atom_site.aniso_U[1][1]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][1]_esd',
              'conversion_constant',
              '_atom_site.aniso_U[1][1]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][1]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][1]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[1][2]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[1][2]_esd',
              'associated_esd',
              '_atom_site.aniso_U[1][2]',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][2]',
              'conversion_constant',
              '_atom_site.aniso_U[1][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][2]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[1][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_B[1][2].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[1][2]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[1][2]',
              'associated_value',
              '_atom_site.aniso_U[1][2]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][2]_esd',
              'conversion_constant',
              '_atom_site.aniso_U[1][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][2]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[1][3]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[1][3]_esd',
              'associated_esd',
              '_atom_site.aniso_U[1][3]',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][3]',
              'conversion_constant',
              '_atom_site.aniso_U[1][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[1][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_B[1][3].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[1][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[1][3]',
              'associated_value',
              '_atom_site.aniso_U[1][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_U[1][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[2][2]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[2][2]_esd',
              'associated_esd',
              '_atom_site.aniso_U[2][2]',
              'conversion_constant',
              '_atom_site_anisotrop.U[2][2]',
              'conversion_constant',
              '_atom_site.aniso_U[2][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][2]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[2][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_B[2][2].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[2][2]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[2][2]',
              'associated_value',
              '_atom_site.aniso_U[2][2]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[2][2]_esd',
              'conversion_constant',
              '_atom_site.aniso_U[2][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][2]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[2][3]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[2][3]_esd',
              'associated_esd',
              '_atom_site.aniso_U[2][3]',
              'conversion_constant',
              '_atom_site_anisotrop.U[2][3]',
              'conversion_constant',
              '_atom_site.aniso_U[2][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[2][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_B[2][3].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[2][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[2][3]',
              'associated_value',
              '_atom_site.aniso_U[2][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[2][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_U[2][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[3][3]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[3][3]_esd',
              'associated_esd',
              '_atom_site.aniso_U[3][3]',
              'conversion_constant',
              '_atom_site_anisotrop.U[3][3]',
              'conversion_constant',
              '_atom_site.aniso_U[3][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[3][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[3][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_B[3][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_B[3][3].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_B[3][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_B[3][3]',
              'associated_value',
              '_atom_site.aniso_U[3][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[3][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_U[3][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[3][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[3][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_ratio',
        [ [ 'item_',
            '_item_description.description',
            '              Ratio of the maximum to minimum principal axes of\n               displacement (thermal) ellipsoids.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_ratio' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site_anisotrop.ratio' ],
          [ 'item_', '_item_related.function_code', 'alternate_exclusive' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1.0', '1.0', '1.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[1][1]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[1][1]_esd',
              'associated_esd',
              '_atom_site.aniso_B[1][1]',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][1]',
              'conversion_constant',
              '_atom_site.aniso_B[1][1]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][1]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][1]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[1][1]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_U[1][1].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[1][1]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[1][1]',
              'associated_value',
              '_atom_site.aniso_B[1][1]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][1]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[1][1]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][1]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][1]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[1][2]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[1][2]_esd',
              'associated_esd',
              '_atom_site.aniso_B[1][2]',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][2]',
              'conversion_constant',
              '_atom_site.aniso_B[1][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][2]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[1][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_U[1][2].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[1][2]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[1][2]',
              'associated_value',
              '_atom_site.aniso_B[1][2]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][2]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[1][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][2]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[1][3]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[1][3]_esd',
              'associated_esd',
              '_atom_site.aniso_B[1][3]',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][3]',
              'conversion_constant',
              '_atom_site.aniso_B[1][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[1][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_U[1][3].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[1][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[1][3]',
              'associated_value',
              '_atom_site.aniso_B[1][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[1][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[2][2]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[2][2]_esd',
              'associated_esd',
              '_atom_site.aniso_B[2][2]',
              'conversion_constant',
              '_atom_site_anisotrop.B[2][2]',
              'conversion_constant',
              '_atom_site.aniso_B[2][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][2]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[2][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_U[2][2].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[2][2]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[2][2]',
              'associated_value',
              '_atom_site.aniso_B[2][2]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[2][2]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[2][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][2]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[2][3]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[2][3]_esd',
              'associated_esd',
              '_atom_site.aniso_B[2][3]',
              'conversion_constant',
              '_atom_site_anisotrop.B[2][3]',
              'conversion_constant',
              '_atom_site.aniso_B[2][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[2][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_U[2][3].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[2][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[2][3]',
              'associated_value',
              '_atom_site.aniso_B[2][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[2][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[2][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[3][3]' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[3][3]_esd',
              'associated_esd',
              '_atom_site.aniso_B[3][3]',
              'conversion_constant',
              '_atom_site_anisotrop.B[3][3]',
              'conversion_constant',
              '_atom_site.aniso_B[3][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[3][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[3][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.aniso_U[3][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.aniso_U[3][3].' ],
          [ 'item_', '_item.name', '_atom_site.aniso_U[3][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.aniso_U[3][3]',
              'associated_value',
              '_atom_site.aniso_B[3][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[3][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[3][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[3][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[3][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.attached_hydrogens',
        [ [ 'item_',
            '_item_description.description',
            '              The number of hydrogen atoms attached to the atom at this site\n               excluding any hydrogen atoms for which coordinates (measured or\n               calculated) are given.' ],
          [ 'item_', '_item.name', '_atom_site.attached_hydrogens' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_attached_hydrogens' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '8', '8', '8', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '2',
              'water oxygen',
              '1',
              'hydroxyl oxygen',
              '4',
              'ammonium nitrogen' ] ] ] ],
      [ 'save_',
        '_atom_site.auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              An alternative identifier for _atom_site.label_asym_id that\n               may be provided by an author in order to match the identification\n               used in the publication that describes the structure.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_atom_site.auth_asym_id',
              'atom_site',
              'yes',
              '_geom_angle.atom_site_auth_asym_id_1',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_auth_asym_id_2',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_auth_asym_id_3',
              'geom_angle',
              'no',
              '_geom_bond.atom_site_auth_asym_id_1',
              'geom_bond',
              'no',
              '_geom_bond.atom_site_auth_asym_id_2',
              'geom_bond',
              'no',
              '_geom_contact.atom_site_auth_asym_id_1',
              'geom_contact',
              'no',
              '_geom_contact.atom_site_auth_asym_id_2',
              'geom_contact',
              'no',
              '_geom_hbond.atom_site_auth_asym_id_A',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_auth_asym_id_D',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_auth_asym_id_H',
              'geom_hbond',
              'no',
              '_geom_torsion.atom_site_auth_asym_id_1',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_asym_id_2',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_asym_id_3',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_asym_id_4',
              'geom_torsion',
              'no',
              '_struct_conf.beg_auth_asym_id',
              'struct_conf',
              'no',
              '_struct_conf.end_auth_asym_id',
              'struct_conf',
              'no',
              '_struct_conn.ptnr1_auth_asym_id',
              'struct_conn',
              'no',
              '_struct_conn.ptnr2_auth_asym_id',
              'struct_conn',
              'no',
              '_struct_mon_nucl.auth_asym_id',
              'struct_mon_nucl',
              'no',
              '_struct_mon_prot.auth_asym_id',
              'struct_mon_prot',
              'no',
              '_struct_mon_prot_cis.auth_asym_id',
              'struct_mon_prot_cis',
              'no',
              '_struct_ncs_dom_lim.beg_auth_asym_id',
              'struct_ncs_dom_lim',
              'no',
              '_struct_ncs_dom_lim.end_auth_asym_id',
              'struct_ncs_dom_lim',
              'no',
              '_struct_sheet_range.beg_auth_asym_id',
              'struct_sheet_range',
              'no',
              '_struct_sheet_range.end_auth_asym_id',
              'struct_sheet_range',
              'no',
              '_struct_site_gen.auth_asym_id',
              'struct_site_gen',
              'no' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_geom_angle.atom_site_auth_asym_id_1',
              '_atom_site.auth_asym_id',
              '_geom_angle.atom_site_auth_asym_id_2',
              '_atom_site.auth_asym_id',
              '_geom_angle.atom_site_auth_asym_id_3',
              '_atom_site.auth_asym_id',
              '_geom_bond.atom_site_auth_asym_id_1',
              '_atom_site.auth_asym_id',
              '_geom_bond.atom_site_auth_asym_id_2',
              '_atom_site.auth_asym_id',
              '_geom_contact.atom_site_auth_asym_id_1',
              '_atom_site.auth_asym_id',
              '_geom_contact.atom_site_auth_asym_id_2',
              '_atom_site.auth_asym_id',
              '_geom_hbond.atom_site_auth_asym_id_A',
              '_atom_site.auth_asym_id',
              '_geom_hbond.atom_site_auth_asym_id_D',
              '_atom_site.auth_asym_id',
              '_geom_hbond.atom_site_auth_asym_id_H',
              '_atom_site.auth_asym_id',
              '_geom_torsion.atom_site_auth_asym_id_1',
              '_atom_site.auth_asym_id',
              '_geom_torsion.atom_site_auth_asym_id_2',
              '_atom_site.auth_asym_id',
              '_geom_torsion.atom_site_auth_asym_id_3',
              '_atom_site.auth_asym_id',
              '_geom_torsion.atom_site_auth_asym_id_4',
              '_atom_site.auth_asym_id',
              '_struct_conf.beg_auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_conf.end_auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_conn.ptnr1_auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_conn.ptnr2_auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_mon_nucl.auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_mon_prot.auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_mon_prot_cis.auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_ncs_dom_lim.beg_auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_ncs_dom_lim.end_auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_sheet_range.beg_auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_sheet_range.end_auth_asym_id',
              '_atom_site.auth_asym_id',
              '_struct_site_gen.auth_asym_id',
              '_atom_site.auth_asym_id' ] ],
          [ 'item_', '_item_sub_category.id', 'mm_atom_site_auth_label' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_atom_site.auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              An alternative identifier for _atom_site.label_atom_id that\n               may be provided by an author in order to match the identification\n               used in the publication that describes the structure.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_atom_site.auth_atom_id',
              'atom_site',
              'no',
              '_geom_angle.atom_site_auth_atom_id_1',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_auth_atom_id_2',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_auth_atom_id_3',
              'geom_angle',
              'no',
              '_geom_bond.atom_site_auth_atom_id_1',
              'geom_bond',
              'no',
              '_geom_bond.atom_site_auth_atom_id_2',
              'geom_bond',
              'no',
              '_geom_contact.atom_site_auth_atom_id_1',
              'geom_contact',
              'no',
              '_geom_contact.atom_site_auth_atom_id_2',
              'geom_contact',
              'no',
              '_geom_hbond.atom_site_auth_atom_id_A',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_auth_atom_id_D',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_auth_atom_id_H',
              'geom_hbond',
              'no',
              '_geom_torsion.atom_site_auth_atom_id_1',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_atom_id_2',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_atom_id_3',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_atom_id_4',
              'geom_torsion',
              'no',
              '_struct_conn.ptnr1_auth_atom_id',
              'struct_conn',
              'no',
              '_struct_conn.ptnr2_auth_atom_id',
              'struct_conn',
              'no',
              '_struct_sheet_hbond.range_1_beg_auth_atom_id',
              'struct_sheet_hbond',
              'no',
              '_struct_sheet_hbond.range_1_end_auth_atom_id',
              'struct_sheet_hbond',
              'no',
              '_struct_sheet_hbond.range_2_beg_auth_atom_id',
              'struct_sheet_hbond',
              'no',
              '_struct_sheet_hbond.range_2_end_auth_atom_id',
              'struct_sheet_hbond',
              'no',
              '_struct_site_gen.auth_atom_id',
              'struct_site_gen',
              'no' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_geom_angle.atom_site_auth_atom_id_1',
              '_atom_site.auth_atom_id',
              '_geom_angle.atom_site_auth_atom_id_2',
              '_atom_site.auth_atom_id',
              '_geom_angle.atom_site_auth_atom_id_3',
              '_atom_site.auth_atom_id',
              '_geom_bond.atom_site_auth_atom_id_1',
              '_atom_site.auth_atom_id',
              '_geom_bond.atom_site_auth_atom_id_2',
              '_atom_site.auth_atom_id',
              '_geom_contact.atom_site_auth_atom_id_1',
              '_atom_site.auth_atom_id',
              '_geom_contact.atom_site_auth_atom_id_2',
              '_atom_site.auth_atom_id',
              '_geom_hbond.atom_site_auth_atom_id_A',
              '_atom_site.auth_atom_id',
              '_geom_hbond.atom_site_auth_atom_id_D',
              '_atom_site.auth_atom_id',
              '_geom_hbond.atom_site_auth_atom_id_H',
              '_atom_site.auth_atom_id',
              '_geom_torsion.atom_site_auth_atom_id_1',
              '_atom_site.auth_atom_id',
              '_geom_torsion.atom_site_auth_atom_id_2',
              '_atom_site.auth_atom_id',
              '_geom_torsion.atom_site_auth_atom_id_3',
              '_atom_site.auth_atom_id',
              '_geom_torsion.atom_site_auth_atom_id_4',
              '_atom_site.auth_atom_id',
              '_struct_conn.ptnr1_auth_atom_id',
              '_atom_site.auth_atom_id',
              '_struct_conn.ptnr2_auth_atom_id',
              '_atom_site.auth_atom_id',
              '_struct_sheet_hbond.range_1_beg_auth_atom_id',
              '_atom_site.auth_atom_id',
              '_struct_sheet_hbond.range_1_end_auth_atom_id',
              '_atom_site.auth_atom_id',
              '_struct_sheet_hbond.range_2_beg_auth_atom_id',
              '_atom_site.auth_atom_id',
              '_struct_sheet_hbond.range_2_end_auth_atom_id',
              '_atom_site.auth_atom_id',
              '_struct_site_gen.auth_atom_id',
              '_atom_site.auth_atom_id' ] ],
          [ 'item_', '_item_sub_category.id', 'mm_atom_site_auth_label' ],
          [ 'item_', '_item_type.code', 'atcode' ] ] ],
      [ 'save_',
        '_atom_site.auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              An alternative identifier for _atom_site.label_comp_id that\n               may be provided by an author in order to match the identification\n               used in the publication that describes the structure.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_atom_site.auth_comp_id',
              'atom_site',
              'no',
              '_geom_angle.atom_site_auth_comp_id_1',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_auth_comp_id_2',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_auth_comp_id_3',
              'geom_angle',
              'no',
              '_geom_bond.atom_site_auth_comp_id_1',
              'geom_bond',
              'no',
              '_geom_bond.atom_site_auth_comp_id_2',
              'geom_bond',
              'no',
              '_geom_contact.atom_site_auth_comp_id_1',
              'geom_contact',
              'no',
              '_geom_contact.atom_site_auth_comp_id_2',
              'geom_contact',
              'no',
              '_geom_hbond.atom_site_auth_comp_id_A',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_auth_comp_id_D',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_auth_comp_id_H',
              'geom_hbond',
              'no',
              '_geom_torsion.atom_site_auth_comp_id_1',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_comp_id_2',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_comp_id_3',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_comp_id_4',
              'geom_torsion',
              'no',
              '_struct_conf.beg_auth_comp_id',
              'struct_conf',
              'no',
              '_struct_conf.end_auth_comp_id',
              'struct_conf',
              'no',
              '_struct_conn.ptnr1_auth_comp_id',
              'struct_conn',
              'no',
              '_struct_conn.ptnr2_auth_comp_id',
              'struct_conn',
              'no',
              '_struct_mon_nucl.auth_comp_id',
              'struct_mon_nucl',
              'no',
              '_struct_mon_prot.auth_comp_id',
              'struct_mon_prot',
              'no',
              '_struct_mon_prot_cis.auth_comp_id',
              'struct_mon_prot_cis',
              'no',
              '_struct_ncs_dom_lim.beg_auth_comp_id',
              'struct_ncs_dom_lim',
              'no',
              '_struct_ncs_dom_lim.end_auth_comp_id',
              'struct_ncs_dom_lim',
              'no',
              '_struct_sheet_range.beg_auth_comp_id',
              'struct_sheet_range',
              'no',
              '_struct_sheet_range.end_auth_comp_id',
              'struct_sheet_range',
              'no',
              '_struct_site_gen.auth_comp_id',
              'struct_site_gen',
              'no' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_geom_angle.atom_site_auth_comp_id_1',
              '_atom_site.auth_comp_id',
              '_geom_angle.atom_site_auth_comp_id_2',
              '_atom_site.auth_comp_id',
              '_geom_angle.atom_site_auth_comp_id_3',
              '_atom_site.auth_comp_id',
              '_geom_bond.atom_site_auth_comp_id_1',
              '_atom_site.auth_comp_id',
              '_geom_bond.atom_site_auth_comp_id_2',
              '_atom_site.auth_comp_id',
              '_geom_contact.atom_site_auth_comp_id_1',
              '_atom_site.auth_comp_id',
              '_geom_contact.atom_site_auth_comp_id_2',
              '_atom_site.auth_comp_id',
              '_geom_hbond.atom_site_auth_comp_id_A',
              '_atom_site.auth_comp_id',
              '_geom_hbond.atom_site_auth_comp_id_D',
              '_atom_site.auth_comp_id',
              '_geom_hbond.atom_site_auth_comp_id_H',
              '_atom_site.auth_comp_id',
              '_geom_torsion.atom_site_auth_comp_id_1',
              '_atom_site.auth_comp_id',
              '_geom_torsion.atom_site_auth_comp_id_2',
              '_atom_site.auth_comp_id',
              '_geom_torsion.atom_site_auth_comp_id_3',
              '_atom_site.auth_comp_id',
              '_geom_torsion.atom_site_auth_comp_id_4',
              '_atom_site.auth_comp_id',
              '_struct_conf.beg_auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_conf.end_auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_conn.ptnr1_auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_conn.ptnr2_auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_mon_nucl.auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_mon_prot.auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_mon_prot_cis.auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_ncs_dom_lim.beg_auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_ncs_dom_lim.end_auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_sheet_range.beg_auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_sheet_range.end_auth_comp_id',
              '_atom_site.auth_comp_id',
              '_struct_site_gen.auth_comp_id',
              '_atom_site.auth_comp_id' ] ],
          [ 'item_', '_item_sub_category.id', 'mm_atom_site_auth_label' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_atom_site.auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              An alternative identifier for _atom_site.label_seq_id that\n               may be provided by an author in order to match the identification\n               used in the publication that describes the structure.\n\n               Note that this is not necessarily a number, that the values do\n               not have to be positive, and that the value does not have to\n               correspond to the value of _atom_site.label_seq_id. The value\n               of _atom_site.label_seq_id is required to be a sequential list\n               of positive integers.\n\n               The author may assign values to _atom_site.auth_seq_id in any\n               desired way. For instance, the values may be used to relate\n               this structure to a numbering scheme in a homologous structure,\n               including sequence gaps or insertion codes. Alternatively, a\n               scheme may be used for a truncated polymer that maintains the\n               numbering scheme of the full length polymer. In all cases, the\n               scheme used here must match the scheme used in the publication\n               that describes the structure.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_atom_site.auth_seq_id',
              'atom_site',
              'no',
              '_geom_angle.atom_site_auth_seq_id_1',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_auth_seq_id_2',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_auth_seq_id_3',
              'geom_angle',
              'no',
              '_geom_bond.atom_site_auth_seq_id_1',
              'geom_bond',
              'no',
              '_geom_bond.atom_site_auth_seq_id_2',
              'geom_bond',
              'no',
              '_geom_contact.atom_site_auth_seq_id_1',
              'geom_contact',
              'no',
              '_geom_contact.atom_site_auth_seq_id_2',
              'geom_contact',
              'no',
              '_geom_hbond.atom_site_auth_seq_id_A',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_auth_seq_id_D',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_auth_seq_id_H',
              'geom_hbond',
              'no',
              '_geom_torsion.atom_site_auth_seq_id_1',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_seq_id_2',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_seq_id_3',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_auth_seq_id_4',
              'geom_torsion',
              'no',
              '_struct_conf.beg_auth_seq_id',
              'struct_conf',
              'no',
              '_struct_conf.end_auth_seq_id',
              'struct_conf',
              'no',
              '_struct_conn.ptnr1_auth_seq_id',
              'struct_conn',
              'no',
              '_struct_conn.ptnr2_auth_seq_id',
              'struct_conn',
              'no',
              '_struct_mon_nucl.auth_seq_id',
              'struct_mon_nucl',
              'no',
              '_struct_mon_prot.auth_seq_id',
              'struct_mon_prot',
              'no',
              '_struct_mon_prot_cis.auth_seq_id',
              'struct_mon_prot_cis',
              'no',
              '_struct_ncs_dom_lim.beg_auth_seq_id',
              'struct_ncs_dom_lim',
              'no',
              '_struct_ncs_dom_lim.end_auth_seq_id',
              'struct_ncs_dom_lim',
              'no',
              '_struct_sheet_hbond.range_1_beg_auth_seq_id',
              'struct_sheet_hbond',
              'no',
              '_struct_sheet_hbond.range_1_end_auth_seq_id',
              'struct_sheet_hbond',
              'no',
              '_struct_sheet_hbond.range_2_beg_auth_seq_id',
              'struct_sheet_hbond',
              'no',
              '_struct_sheet_hbond.range_2_end_auth_seq_id',
              'struct_sheet_hbond',
              'no',
              '_struct_sheet_range.beg_auth_seq_id',
              'struct_sheet_range',
              'no',
              '_struct_sheet_range.end_auth_seq_id',
              'struct_sheet_range',
              'no',
              '_struct_site_gen.auth_seq_id',
              'struct_site_gen',
              'no' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_geom_angle.atom_site_auth_seq_id_1',
              '_atom_site.auth_seq_id',
              '_geom_angle.atom_site_auth_seq_id_2',
              '_atom_site.auth_seq_id',
              '_geom_angle.atom_site_auth_seq_id_3',
              '_atom_site.auth_seq_id',
              '_geom_bond.atom_site_auth_seq_id_1',
              '_atom_site.auth_seq_id',
              '_geom_bond.atom_site_auth_seq_id_2',
              '_atom_site.auth_seq_id',
              '_geom_contact.atom_site_auth_seq_id_1',
              '_atom_site.auth_seq_id',
              '_geom_contact.atom_site_auth_seq_id_2',
              '_atom_site.auth_seq_id',
              '_geom_hbond.atom_site_auth_seq_id_A',
              '_atom_site.auth_seq_id',
              '_geom_hbond.atom_site_auth_seq_id_D',
              '_atom_site.auth_seq_id',
              '_geom_hbond.atom_site_auth_seq_id_H',
              '_atom_site.auth_seq_id',
              '_geom_torsion.atom_site_auth_seq_id_1',
              '_atom_site.auth_seq_id',
              '_geom_torsion.atom_site_auth_seq_id_2',
              '_atom_site.auth_seq_id',
              '_geom_torsion.atom_site_auth_seq_id_3',
              '_atom_site.auth_seq_id',
              '_geom_torsion.atom_site_auth_seq_id_4',
              '_atom_site.auth_seq_id',
              '_struct_conf.beg_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_conf.end_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_conn.ptnr1_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_conn.ptnr2_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_mon_nucl.auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_mon_prot.auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_mon_prot_cis.auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_ncs_dom_lim.beg_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_ncs_dom_lim.end_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_sheet_hbond.range_1_beg_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_sheet_hbond.range_1_end_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_sheet_hbond.range_2_beg_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_sheet_hbond.range_2_end_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_sheet_range.beg_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_sheet_range.end_auth_seq_id',
              '_atom_site.auth_seq_id',
              '_struct_site_gen.auth_seq_id',
              '_atom_site.auth_seq_id' ] ],
          [ 'item_', '_item_sub_category.id', 'mm_atom_site_auth_label' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_atom_site.B_equiv_geom_mean',
        [ [ 'item_',
            '_item_description.description',
            '              Equivalent isotropic atomic displacement parameter, B~eq~,\n               in angstroms squared, calculated as the geometric mean of\n               the anisotropic atomic displacement parameters.\n\n               B~eq~ = (B~i~ B~j~ B~k~)^1/3^\n\n               B~n~ = the principal components of the orthogonalized B^ij^\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site.B_equiv_geom_mean' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_B_equiv_geom_mean' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.B_equiv_geom_mean_esd',
              'associated_esd',
              '_atom_site.U_equiv_geom_mean',
              'conversion_constant' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.B_equiv_geom_mean_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.B_equiv_geom_mean.' ],
          [ 'item_', '_item.name', '_atom_site.B_equiv_geom_mean_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.B_equiv_geom_mean',
              'associated_value',
              '_atom_site.U_equiv_geom_mean',
              'conversion_constant' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.B_iso_or_equiv',
        [ [ 'item_',
            '_item_description.description',
            '              Isotropic atomic displacement parameter, or equivalent isotropic\n               atomic displacement parameter, B~eq~, calculated from the\n               anisotropic displacement parameters.\n\n               B~eq~ = (1/3) sum~i~[sum~j~(B^ij^ A~i~ A~j~ a*~i~ a*~j~)]\n\n               A     = the real space cell lengths\n               a*    = the reciprocal space cell lengths\n               B^ij^ = 8 pi^2^ U^ij^\n\n               Ref: Fischer, R. X. & Tillmanns, E. (1988). Acta Cryst. C44,\n                    775-776.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site.B_iso_or_equiv' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_B_iso_or_equiv' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_range.maximum' ],
          [ '@comment', '    _item_range.minimum            .    0.0' ],
          [ '@comment', '                                  0.0   0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.B_iso_or_equiv_esd',
              'associated_esd',
              '_atom_site.U_iso_or_equiv',
              'conversion_constant' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.B_iso_or_equiv_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.B_iso_or_equiv.' ],
          [ 'item_', '_item.name', '_atom_site.B_iso_or_equiv_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.B_iso_or_equiv',
              'associated_value',
              '_atom_site.U_iso_or_equiv_esd',
              'conversion_constant' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.calc_attached_atom',
        [ [ 'item_',
            '_item_description.description',
            '              The _atom_site.id of the atom site to which the\n               \'geometry-calculated\' atom site is attached.' ],
          [ 'item_', '_item.name', '_atom_site.calc_attached_atom' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_calc_attached_atom' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_atom_site.calc_flag',
        [ [ 'item_',
            '_item_description.description',
            '              A standard code to signal whether the site coordinates have been\n               determined from the intensities or calculated from the geometry\n               of surrounding sites, or have been assigned dummy values. The\n               abbreviation \'c\' may be used in place of \'calc\'.' ],
          [ 'item_', '_item.name', '_atom_site.calc_flag' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_calc_flag' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           d' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'd',
              'determined from experimental measurements',
              'calc',
              'calculated from molecular geometry',
              'c',
              'abbreviation for "calc"',
              'dum',
              'dummy site with meaningless coordinates' ] ] ] ],
      [ 'save_',
        '_atom_site.Cartn_x',
        [ [ 'item_',
            '_item_description.description',
            '              The x atom-site coordinate in angstroms specified according to\n               a set of orthogonal Cartesian axes related to the cell axes as\n               specified by the description given in\n               _atom_sites.Cartn_transform_axes.' ],
          [ 'item_', '_item.name', '_atom_site.Cartn_x' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_Cartn_x' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.Cartn_y', '_atom_site.Cartn_z' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.Cartn_x_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_atom_site.Cartn_x_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.Cartn_x.' ],
          [ 'item_', '_item.name', '_atom_site.Cartn_x_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.Cartn_y_esd', '_atom_site.Cartn_z_esd' ] ],
          [ 'item_', '_item_related.related_name', '_atom_site.Cartn_x' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_atom_site.Cartn_y',
        [ [ 'item_',
            '_item_description.description',
            '              The y atom-site coordinate in angstroms specified according to\n               a set of orthogonal Cartesian axes related to the cell axes as\n               specified by the description given in\n               _atom_sites.Cartn_transform_axes.' ],
          [ 'item_', '_item.name', '_atom_site.Cartn_y' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_Cartn_y' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.Cartn_x', '_atom_site.Cartn_z' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.Cartn_y_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_atom_site.Cartn_y_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.Cartn_y.' ],
          [ 'item_', '_item.name', '_atom_site.Cartn_y_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.Cartn_x_esd', '_atom_site.Cartn_z_esd' ] ],
          [ 'item_', '_item_related.related_name', '_atom_site.Cartn_y' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_atom_site.Cartn_z',
        [ [ 'item_',
            '_item_description.description',
            '              The z atom-site coordinate in angstroms specified according to\n               a set of orthogonal Cartesian axes related to the cell axes as\n               specified by the description given in\n               _atom_sites.Cartn_transform_axes.' ],
          [ 'item_', '_item.name', '_atom_site.Cartn_z' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_Cartn_z' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.Cartn_x', '_atom_site.Cartn_y' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.Cartn_z_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_atom_site.Cartn_z_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.Cartn_z.' ],
          [ 'item_', '_item.name', '_atom_site.Cartn_z_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.Cartn_x_esd', '_atom_site.Cartn_y_esd' ] ],
          [ 'item_', '_item_related.related_name', '_atom_site.Cartn_z' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_atom_site.chemical_conn_number',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chemical_conn_atom.number in the\n               CHEMICAL_CONN_ATOM category.' ],
          [ 'item_', '_item.name', '_atom_site.chemical_conn_number' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_chemical_conn_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_atom_site.constraints',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the constraints applied to parameters at this\n               site during refinement. See also _atom_site.refinement_flags\n               and _refine.ls_number_constraints.' ],
          [ 'item_', '_item.name', '_atom_site.constraints' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_constraints' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'pop=1.0-pop(Zn3)' ] ] ],
      [ 'save_',
        '_atom_site.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of this site. See also\n               _atom_site.refinement_flags.' ],
          [ 'item_', '_item.name', '_atom_site.details' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_description' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'Ag/Si disordered' ] ] ],
      [ 'save_',
        '_atom_site.disorder_assembly',
        [ [ 'item_',
            '_item_description.description',
            '              A code which identifies a cluster of atoms that show long-range\n               positional disorder but are locally ordered. Within each such\n               cluster of atoms, _atom_site.disorder_group is used to identify\n               the sites that are simultaneously occupied. This field is only\n               needed if there is more than one cluster of disordered atoms\n               showing independent local order.\n\n               *** This data item would not in general be used in a\n               macromolecular data block. ***' ],
          [ 'item_', '_item.name', '_atom_site.disorder_assembly' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_disorder_assembly' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_atom_site.disorder_group',
        [ [ 'item_',
            '_item_description.description',
            '              A code which identifies a group of positionally disordered atom\n               sites that are locally simultaneously occupied. Atoms that are\n               positionally disordered over two or more sites (e.g. the hydrogen\n               atoms of a methyl group that exists in two orientations) can\n               be assigned to two or more groups. Sites belonging to the same\n               group are simultaneously occupied, but those belonging to\n               different groups are not. A minus prefix (e.g. \'-1\') is used to\n               indicate sites disordered about a special position.\n\n               *** This data item would not in general be used in a\n               macromolecular data block. ***' ],
          [ 'item_', '_item.name', '_atom_site.disorder_group' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_disorder_group' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '.' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_atom_site.footnote_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _atom_site.footnote_id must match an ID\n               specified by _atom_sites_footnote.id in the\n               ATOM_SITES_FOOTNOTE list.' ],
          [ 'item_', '_item.name', '_atom_site.footnote_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_atom_site.fract_x',
        [ [ 'item_',
            '_item_description.description',
            '              The x coordinate of the atom-site position specified as a\n               fraction of _cell.length_a.' ],
          [ 'item_', '_item.name', '_atom_site.fract_x' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_fract_x' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.fract_y', '_atom_site.fract_z' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.fract_x_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_atom_site.fract_x_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.fract_x.' ],
          [ 'item_', '_item.name', '_atom_site.fract_x_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.fract_y_esd', '_atom_site.fract_z_esd' ] ],
          [ 'item_', '_item_related.related_name', '_atom_site.fract_x' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_site.fract_y',
        [ [ 'item_',
            '_item_description.description',
            '              The y coordinate of the atom-site position specified as a\n               fraction of _cell.length_b.' ],
          [ 'item_', '_item.name', '_atom_site.fract_y' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_fract_y' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.fract_x', '_atom_site.fract_z' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.fract_y_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_atom_site.fract_y_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.fract_y.' ],
          [ 'item_', '_item.name', '_atom_site.fract_y_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.fract_x_esd', '_atom_site.fract_z_esd' ] ],
          [ 'item_', '_item_related.related_name', '_atom_site.fract_y' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_site.fract_z',
        [ [ 'item_',
            '_item_description.description',
            '              The z coordinate of the atom-site position specified as a\n               fraction of _cell.length_c.' ],
          [ 'item_', '_item.name', '_atom_site.fract_z' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_fract_z' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.fract_x', '_atom_site.fract_y' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.fract_z_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_atom_site.fract_z_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.fract_z.' ],
          [ 'item_', '_item.name', '_atom_site.fract_z_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_site.fract_x_esd', '_atom_site.fract_y_esd' ] ],
          [ 'item_', '_item_related.related_name', '_atom_site.fract_z' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_site.group_PDB',
        [ [ 'item_',
            '_item_description.description',
            '              The group of atoms to which the atom site belongs. This data\n               item is provided for compatibility with the original Protein\n               Data Bank format, and only for that purpose.' ],
          [ 'item_', '_item.name', '_atom_site.group_PDB' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_enumeration.value' ], [ 'ATOM', 'HETATM' ] ] ] ],
      [ 'save_',
        '_atom_site.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _atom_site.id must uniquely identify a record in the\n               ATOM_SITE list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.\n\n               This data item was introduced to provide compatibility between\n               small-molecule and macromolecular CIFs. In a small-molecule\n               CIF, _atom_site_label is the identifier for the atom. In a\n               macromolecular CIF, the atom identifier is the aggregate of\n               _atom_site.label_alt_id, _atom_site.label_asym_id,\n               _atom_site.label_atom_id, _atom_site.label_comp_id and\n               _atom_site.label_seq_id. For the two types of files to be\n               compatible, a formal identifier for the category had to be\n               introduced that was independent of the different modes of\n               identifying the atoms. For compatibility with older CIFs,\n               _atom_site_label is aliased to _atom_site.id.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_atom_site.id',
              'atom_site',
              'yes',
              '_atom_site_anisotrop.id',
              'atom_site_anisotrop',
              'yes',
              '_geom_angle.atom_site_id_1',
              'geom_angle',
              'yes',
              '_geom_angle.atom_site_id_2',
              'geom_angle',
              'yes',
              '_geom_angle.atom_site_id_3',
              'geom_angle',
              'yes',
              '_geom_bond.atom_site_id_1',
              'geom_bond',
              'yes',
              '_geom_bond.atom_site_id_2',
              'geom_bond',
              'yes',
              '_geom_contact.atom_site_id_1',
              'geom_contact',
              'yes',
              '_geom_contact.atom_site_id_2',
              'geom_contact',
              'yes',
              '_geom_hbond.atom_site_id_A',
              'geom_hbond',
              'yes',
              '_geom_hbond.atom_site_id_D',
              'geom_hbond',
              'yes',
              '_geom_hbond.atom_site_id_H',
              'geom_hbond',
              'yes',
              '_geom_torsion.atom_site_id_1',
              'geom_torsion',
              'yes',
              '_geom_torsion.atom_site_id_2',
              'geom_torsion',
              'yes',
              '_geom_torsion.atom_site_id_3',
              'geom_torsion',
              'yes',
              '_geom_torsion.atom_site_id_4',
              'geom_torsion',
              'yes' ] ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_label' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_site_anisotrop.id',
              '_atom_site.id',
              '_geom_angle.atom_site_id_1',
              '_atom_site.id',
              '_geom_angle.atom_site_id_2',
              '_atom_site.id',
              '_geom_angle.atom_site_id_3',
              '_atom_site.id',
              '_geom_bond.atom_site_id_1',
              '_atom_site.id',
              '_geom_bond.atom_site_id_2',
              '_atom_site.id',
              '_geom_contact.atom_site_id_1',
              '_atom_site.id',
              '_geom_contact.atom_site_id_2',
              '_atom_site.id',
              '_geom_hbond.atom_site_id_A',
              '_atom_site.id',
              '_geom_hbond.atom_site_id_D',
              '_atom_site.id',
              '_geom_hbond.atom_site_id_H',
              '_atom_site.id',
              '_geom_torsion.atom_site_id_1',
              '_atom_site.id',
              '_geom_torsion.atom_site_id_2',
              '_atom_site.id',
              '_geom_torsion.atom_site_id_3',
              '_atom_site.id',
              '_geom_torsion.atom_site_id_4',
              '_atom_site.id' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '5',
              'C12',
              'Ca3g28',
              'Fe3+17',
              'H*251',
              'boron2a',
              'C_a_phe_83_a_0',
              'Zn_Zn_301_A_0' ] ] ] ],
      [ 'save_',
        '_atom_site.label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for this atom site.\n               For further details, see the definition of the ATOM_SITE_ALT\n               category.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_atom_site.label_alt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_sub_category.id', 'mm_atom_site_label' ] ] ],
      [ 'save_',
        '_atom_site.label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for this atom site.\n               For further details, see the definition of the STRUCT_ASYM\n               category.\n\n               This data item is a pointer to _struct_asym.id in the\n               STRUCT_ASYM category.' ],
          [ 'item_', '_item.name', '_atom_site.label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_sub_category.id', 'mm_atom_site_label' ] ] ],
      [ 'save_',
        '_atom_site.label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for this atom site.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_atom_site.label_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_sub_category.id', 'mm_atom_site_label' ] ] ],
      [ 'save_',
        '_atom_site.label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for this atom site.\n\n               This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_atom_site.label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_sub_category.id', 'mm_atom_site_label' ] ] ],
      [ 'save_',
        '_atom_site.label_entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_atom_site.label_entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_atom_site.label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity_poly_seq.num in the\n               ENTITY_POLY_SEQ category.' ],
          [ 'item_', '_item.name', '_atom_site.label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_atom_site.occupancy',
        [ [ 'item_',
            '_item_description.description',
            '              The fraction of the atom type present at this site.\n               The sum of the occupancies of all the atom types at this site\n               may not significantly exceed 1.0 unless it is a dummy site.' ],
          [ 'item_', '_item.name', '_atom_site.occupancy' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_occupancy' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1.0' ],
          [ '@comment', ' JDW 17-Apr-2003  - Unsupportable restriction.' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_range.maximum' ],
          [ '@comment', '    _item_range.minimum           1.0   1.0' ],
          [ '@comment', '                                  1.0   0.0' ],
          [ '@comment', '                                  0.0   0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.occupancy_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_atom_site.occupancy_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.occupancy.' ],
          [ 'item_', '_item.name', '_atom_site.occupancy_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_related.related_name', '_atom_site.occupancy' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', 'save__atom_site.refinement_flags' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              A concatenated series of single-letter codes which indicate the' ],
      [ '@comment',
        '               refinement restraints or constraints applied to this site.' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_atom_site.refinement_flags\'' ],
      [ '@comment', '    _item.category_id             atom_site' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '   _item_aliases.alias_name    \'_atom_site_refinement_flags\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    _item_type.code               ucode' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_enumeration.value' ],
      [ '@comment', '    _item_enumeration.detail      .' ],
      [ '@comment',
        '                                 \'no refinement constraints\'' ],
      [ '@comment', '                                  S' ],
      [ '@comment',
        '                                 \'special-position constraint on site\'' ],
      [ '@comment', '                                  G' ],
      [ '@comment',
        '                                 \'rigid-group refinement of site\'' ],
      [ '@comment', '                                  R' ],
      [ '@comment',
        '                                 \'riding atom site attached to non-riding atom\'' ],
      [ '@comment', '                                  D' ],
      [ '@comment',
        '                                 \'distance or angle restraint on site\'' ],
      [ '@comment', '                                  T' ],
      [ '@comment',
        '                                 \'thermal displacement constraints\'' ],
      [ '@comment', '                                  U' ],
      [ '@comment',
        '                                 \'Uiso or Uij restraint (rigid bond)\'' ],
      [ '@comment', '                                  P' ],
      [ '@comment',
        '                                 \'partial occupancy constraint\'' ],
      [ '@comment', '     save_' ],
      [ 'save_',
        '_atom_site.restraints',
        [ [ 'item_',
            '_item_description.description',
            '              A description of restraints applied to specific parameters at\n               this site during refinement. See also _atom_site.refinement_flags\n               and _refine.ls_number_restraints.' ],
          [ 'item_', '_item.name', '_atom_site.restraints' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_restraints' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'restrained to planar ring' ] ] ],
      [ 'save_',
        '_atom_site.symmetry_multiplicity',
        [ [ 'item_',
            '_item_description.description',
            '              The multiplicity of a site due to the space-group symmetry as is\n               given in International Tables for Crystallography Vol. A (2002).' ],
          [ 'item_', '_item.name', '_atom_site.symmetry_multiplicity' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_symmetry_multiplicity' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '192', '192', '192', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_atom_site.thermal_displace_type',
        [ [ 'item_',
            '_item_description.description',
            '              A standard code used to describe the type of atomic displacement\n               parameters used for the site.' ],
          [ 'item_', '_item.name', '_atom_site.thermal_displace_type' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_thermal_displace_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Uani',
              'anisotropic Uij',
              'Uiso',
              'isotropic U',
              'Uovl',
              'overall U',
              'Umpe',
              'multipole expansion U',
              'Bani',
              'anisotropic Bij',
              'Biso',
              'isotropic B',
              'Bovl',
              'overall B' ] ] ] ],
      [ 'save_',
        '_atom_site.type_symbol',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _atom_type.symbol in the\n               ATOM_TYPE category.' ],
          [ 'item_', '_item.name', '_atom_site.type_symbol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_type_symbol' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_atom_site.U_equiv_geom_mean',
        [ [ 'item_',
            '_item_description.description',
            '              Equivalent isotropic atomic displacement parameter, U~eq~,\n               in angstroms squared, calculated as the geometric mean of\n               the anisotropic atomic displacement parameters.\n\n               U~eq~ = (U~i~ U~j~ U~k~)^1/3^\n\n               U~n~ = the principal components of the orthogonalized U^ij^' ],
          [ 'item_', '_item.name', '_atom_site.U_equiv_geom_mean' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_U_equiv_geom_mean' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '10.0', '10.0', '10.0', '0.0', '0.0', '0.0' ] ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.U_equiv_geom_mean_esd',
              'associated_esd',
              '_atom_site.B_equiv_geom_mean',
              'conversion_constant' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.U_equiv_geom_mean_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.U_equiv_geom_mean.' ],
          [ 'item_', '_item.name', '_atom_site.U_equiv_geom_mean_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.U_equiv_geom_mean',
              'associated_value',
              '_atom_site.B_equiv_geom_mean',
              'conversion_constant' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.U_iso_or_equiv',
        [ [ 'item_',
            '_item_description.description',
            '              Isotropic atomic displacement parameter, or equivalent isotropic\n               atomic  displacement parameter, U~eq~, calculated from\n               anisotropic atomic displacement  parameters.\n\n               U~eq~ = (1/3) sum~i~[sum~j~(U^ij^ A~i~ A~j~ a*~i~ a*~j~)]\n\n               A  = the real space cell lengths\n               a* = the reciprocal space cell lengths\n\n               Ref: Fischer, R. X. & Tillmanns, E. (1988). Acta Cryst. C44,\n                    775-776.' ],
          [ 'item_', '_item.name', '_atom_site.U_iso_or_equiv' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_U_iso_or_equiv' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '10.0', '10.0', '10.0', '0.0', '0.0', '0.0' ] ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.U_iso_or_equiv_esd',
              'associated_esd',
              '_atom_site.B_iso_or_equiv',
              'conversion_constant' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.U_iso_or_equiv_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site.U_iso_or_equiv.' ],
          [ 'item_', '_item.name', '_atom_site.U_iso_or_equiv_esd' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.U_iso_or_equiv',
              'associated_value',
              '_atom_site.B_iso_or_equiv_esd',
              'conversion_constant' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site.Wyckoff_symbol',
        [ [ 'item_',
            '_item_description.description',
            '              The Wyckoff symbol (letter) as listed in the space-group tables\n               of International Tables for Crystallography, Vol. A (2002).' ],
          [ 'item_', '_item.name', '_atom_site.Wyckoff_symbol' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_Wyckoff_symbol' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# ATOM_SITE_ANISOTROP ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'atom_site_anisotrop',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ATOM_SITE_ANISOTROP category record details\n               about anisotropic displacement parameters.\n               If the ATOM_SITE_ANISOTROP category is used for storing these\n               data, the corresponding ATOM_SITE data items are not used.' ],
          [ 'item_', '_category.id', 'atom_site_anisotrop' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_atom_site_anisotrop.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'atom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on NDB structure BDL005 of Holbrook, Dickerson &\n                Kim [Acta Cryst. (1985), B41, 255-262].',
              '\n    loop_\n    _atom_site_anisotrop.id\n    _atom_site_anisotrop.type_symbol\n    _atom_site_anisotrop.U[1][1]\n    _atom_site_anisotrop.U[1][2]\n    _atom_site_anisotrop.U[1][3]\n    _atom_site_anisotrop.U[2][2]\n    _atom_site_anisotrop.U[2][3]\n    _atom_site_anisotrop.U[3][3]\n     1  O   8642   4866   7299   -342   -258  -1427\n     2  C   5174   4871   6243  -1885  -2051  -1377\n     3  C   6202   5020   4395  -1130   -556   -632\n     4  O   4224   4700   5046   1105   -161    345\n     5  C   8684   4688   4171  -1850   -433   -292\n     6  O  11226   5255   3532   -341   2685   1328\n     7  C  10214   2428   5614  -2610  -1940    902\n     8  C   4590   3488   5827    751   -770    986\n     9  N   5014   4434   3447    -17  -1593    539\n    # ----   abbreviated  ----',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[1][1]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_B_11' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[1][1]_esd',
              'associated_esd',
              '_atom_site.aniso_U[1][1]',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][1]',
              'conversion_constant',
              '_atom_site.aniso_B[1][1]',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][1]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][1]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[1][1]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.B[1][1].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[1][1]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[1][1]',
              'associated_value',
              '_atom_site.aniso_U[1][1]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][1]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[1][1]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][1]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][1]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[1][2]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_B_12' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[1][2]_esd',
              'associated_esd',
              '_atom_site.aniso_U[1][2]',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][2]',
              'conversion_constant',
              '_atom_site.aniso_B[1][2]',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][2]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[1][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.B[1][2].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[1][2]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[1][2]',
              'associated_value',
              '_atom_site.aniso_U[1][2]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][2]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[1][2]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][2]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[1][3]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_B_13' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[1][3]_esd',
              'associated_esd',
              '_atom_site.aniso_U[1][3]',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][3]',
              'conversion_constant',
              '_atom_site.aniso_B[1][3]',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[1][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.B[1][3].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[1][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[1][3]',
              'associated_value',
              '_atom_site.aniso_U[1][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[1][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[1][3]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[1][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[2][2]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_B_22' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[2][2]_esd',
              'associated_esd',
              '_atom_site.aniso_U[2][2]',
              'conversion_constant',
              '_atom_site_anisotrop.U[2][2]',
              'conversion_constant',
              '_atom_site.aniso_B[2][2]',
              'alternate_exclusive',
              '_atom_site.aniso_U[2][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][2]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[2][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.B[2][2].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[2][2]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[2][2]',
              'associated_value',
              '_atom_site.aniso_U[2][2]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[2][2]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[2][2]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[2][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][2]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[2][3]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_B_23' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[2][3]_esd',
              'associated_esd',
              '_atom_site.aniso_U[2][3]',
              'conversion_constant',
              '_atom_site_anisotrop.U[2][3]',
              'conversion_constant',
              '_atom_site.aniso_B[2][3]',
              'alternate_exclusive',
              '_atom_site.aniso_U[2][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[2][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.B[2][3].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[2][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[2][3]',
              'associated_value',
              '_atom_site.aniso_U[2][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[2][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[2][3]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[2][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[2][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the anisotropic atomic displacement\n               matrix B, which appears in the structure-factor term as:\n\n               T = exp{-1/4 sum~i~[sum~j~(B^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.\n\n               The IUCr Commission on Nomenclature recommends against the use\n               of B for reporting atomic displacement parameters. U, being\n               directly proportional to B, is preferred.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[3][3]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_B_33' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[3][3]_esd',
              'associated_esd',
              '_atom_site.aniso_U[3][3]',
              'conversion_constant',
              '_atom_site_anisotrop.U[3][3]',
              'conversion_constant',
              '_atom_site.aniso_B[3][3]',
              'alternate_exclusive',
              '_atom_site.aniso_U[3][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[3][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.B[3][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.B[3][3].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.B[3][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.B[3][3]',
              'associated_value',
              '_atom_site.aniso_U[3][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.U[3][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[3][3]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[3][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.U[3][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', '8pi2_angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.ratio',
        [ [ 'item_',
            '_item_description.description',
            '              Ratio of the maximum to minimum principal axes of\n               displacement (thermal) ellipsoids.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.ratio' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_ratio' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1.0', '1.0', '1.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.aniso_ratio' ],
          [ 'item_', '_item_related.function_code', 'alternate_exclusive' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_label' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.type_symbol',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _atom_type.symbol in the\n               ATOM_TYPE category.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.type_symbol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_aniso_type_symbol' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[1][1]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_U_11' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[1][1]_esd',
              'associated_esd',
              '_atom_site.aniso_B[1][1]',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][1]',
              'conversion_constant',
              '_atom_site.aniso_B[1][1]',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][1]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][1]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[1][1]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.U[1][1].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[1][1]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[1][1]',
              'associated_value',
              '_atom_site.aniso_B[1][1]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][1]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[1][1]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][1]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][1]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[1][2]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_U_12' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[1][2]_esd',
              'associated_esd',
              '_atom_site.aniso_B[1][2]',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][2]',
              'conversion_constant',
              '_atom_site.aniso_B[1][2]',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][2]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[1][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.U[1][2].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[1][2]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[1][2]',
              'associated_value',
              '_atom_site.aniso_B[1][2]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][2]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[1][2]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][2]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[1][3]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_U_13' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[1][3]_esd',
              'associated_esd',
              '_atom_site.aniso_B[1][3]',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][3]',
              'conversion_constant',
              '_atom_site.aniso_B[1][3]',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[1][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.U[1][3].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[1][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[1][3]',
              'associated_value',
              '_atom_site.aniso_B[1][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[1][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[1][3]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[1][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[1][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[2][2]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_U_22' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[2][2]_esd',
              'associated_esd',
              '_atom_site.aniso_B[2][2]',
              'conversion_constant',
              '_atom_site_anisotrop.B[2][2]',
              'conversion_constant',
              '_atom_site.aniso_B[2][2]',
              'alternate_exclusive',
              '_atom_site.aniso_U[2][2]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][2]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[2][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.U[2][2].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[2][2]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[2][2]',
              'associated_value',
              '_atom_site.aniso_B[2][2]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[2][2]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[2][2]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[2][2]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][2]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[2][3]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_U_23' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[2][3]_esd',
              'associated_esd',
              '_atom_site.aniso_B[2][3]',
              'conversion_constant',
              '_atom_site_anisotrop.B[2][3]',
              'conversion_constant',
              '_atom_site.aniso_B[2][3]',
              'alternate_exclusive',
              '_atom_site.aniso_U[2][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[2][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.U[2][3].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[2][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[2][3]',
              'associated_value',
              '_atom_site.aniso_B[2][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[2][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[2][3]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[2][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[2][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the standard anisotropic atomic\n               displacement matrix U, which appears in the structure-factor\n               term as:\n\n               T = exp{-2 pi^2^ sum~i~[sum~j~(U^ij^ h~i~ h~j~ a*~i~ a*~j~)]}\n\n               h  = the Miller indices\n               a* = the reciprocal space cell lengths\n\n               These matrix elements may appear with atomic coordinates\n               in the ATOM_SITE category, or they may appear in the separate\n               ATOM_SITE_ANISOTROP category, but they may not appear in both\n               places. Similarly, anisotropic displacements may appear as\n               either B\'s or U\'s, but not as both.\n\n               The unique elements of the real symmetric matrix are\n               entered by row.' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[3][3]' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site_aniso_U_33' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[3][3]_esd',
              'associated_esd',
              '_atom_site.aniso_B[3][3]',
              'conversion_constant',
              '_atom_site_anisotrop.B[3][3]',
              'conversion_constant',
              '_atom_site.aniso_B[3][3]',
              'alternate_exclusive',
              '_atom_site.aniso_U[3][3]',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[3][3]',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.U[3][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _atom_site_anisotrop.U[3][3].' ],
          [ 'item_', '_item.name', '_atom_site_anisotrop.U[3][3]_esd' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site_anisotrop.U[3][3]',
              'associated_value',
              '_atom_site.aniso_B[3][3]_esd',
              'conversion_constant',
              '_atom_site_anisotrop.B[3][3]_esd',
              'conversion_constant',
              '_atom_site.aniso_B[3][3]_esd',
              'alternate_exclusive',
              '_atom_site.aniso_U[3][3]_esd',
              'alternate_exclusive',
              '_atom_site_anisotrop.B[3][3]_esd',
              'alternate_exclusive' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ '@comment', '###############' ],
      [ '@comment', '# ATOM_SITES ##' ],
      [ '@comment', '###############' ],
      [ 'save_',
        'atom_sites',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ATOM_SITES category record details about\n               the crystallographic cell and cell transformations, which are\n               common to all atom sites.' ],
          [ 'item_', '_category.id', 'atom_sites' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_atom_sites.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'atom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _atom_sites.entry_id                   \'5HVP\'\n    _atom_sites.Cartn_transform_axes\n                         \'c along z, astar along x, b along y\'\n    _atom_sites.Cartn_transf_matrix[1][1]  58.39\n    _atom_sites.Cartn_transf_matrix[1][2]   0.00\n    _atom_sites.Cartn_transf_matrix[1][3]   0.00\n    _atom_sites.Cartn_transf_matrix[2][1]   0.00\n    _atom_sites.Cartn_transf_matrix[2][2]  86.70\n    _atom_sites.Cartn_transf_matrix[2][3]   0.00\n    _atom_sites.Cartn_transf_matrix[3][1]   0.00\n    _atom_sites.Cartn_transf_matrix[3][2]   0.00\n    _atom_sites.Cartn_transf_matrix[3][3]  46.27\n    _atom_sites.Cartn_transf_vector[1]      0.00\n    _atom_sites.Cartn_transf_vector[2]      0.00\n    _atom_sites.Cartn_transf_vector[3]      0.00',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_atom_sites.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_atom_sites.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_matrix[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the 3x3 matrix used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.Cartn_transf_vector[].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.Cartn_transf_matrix[1][1]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_matrix_11' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_matrix[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the 3x3 matrix used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.Cartn_transf_vector[].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.Cartn_transf_matrix[1][2]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_matrix_12' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_matrix[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the 3x3 matrix used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.Cartn_transf_vector[].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.Cartn_transf_matrix[1][3]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_matrix_13' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_matrix[2][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][1] element of the 3x3 matrix used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.Cartn_transf_vector[].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.Cartn_transf_matrix[2][1]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_matrix_21' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_matrix[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the 3x3 matrix used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.Cartn_transf_vector[].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.Cartn_transf_matrix[2][2]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_matrix_22' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_matrix[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the 3x3 matrix used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.Cartn_transf_vector[].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.Cartn_transf_matrix[2][3]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_matrix_23' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_matrix[3][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][1] element of the 3x3 matrix used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.Cartn_transf_vector[].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.Cartn_transf_matrix[3][1]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_matrix_31' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_matrix[3][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][2] element of the 3x3 matrix used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.Cartn_transf_vector[].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.Cartn_transf_matrix[3][2]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_matrix_32' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_matrix[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the 3x3 matrix used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.Cartn_transf_vector[].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.Cartn_transf_matrix[3][3]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_matrix_33' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_vector[1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1] element of the three-element vector used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The rotation matrix is defined in\n               _atom_sites.Cartn_transf_matrix[][].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_', '_item.name', '_atom_sites.Cartn_transf_vector[1]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_vector_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_vector[2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2] element of the three-element vector used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The rotation matrix is defined in\n               _atom_sites.Cartn_transf_matrix[][].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_', '_item.name', '_atom_sites.Cartn_transf_vector[2]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_vector_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transf_vector[3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3] element of the three-element vector used to transform\n               fractional coordinates in the ATOM_SITE category to Cartesian\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The rotation matrix is defined in\n               _atom_sites.Cartn_transf_matrix[][].\n\n               |x\'|              |11 12 13| |x|               |1|\n               |y\'|~Cartesian~ = |21 22 23| |y|~fractional~ + |2|\n               |z\'|              |31 32 33| |z|               |3|' ],
          [ 'item_', '_item.name', '_atom_sites.Cartn_transf_vector[3]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_tran_vector_3' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.Cartn_transform_axes',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the relative alignment of the crystal cell\n               axes to the Cartesian orthogonal axes as applied in the\n               transformation matrix _atom_sites.Cartn_transf_matrix[][].' ],
          [ 'item_', '_item.name', '_atom_sites.Cartn_transform_axes' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_Cartn_transform_axes' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'a parallel to x; b in the plane of y and z' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_matrix[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the 3x3 matrix used to transform Cartesian\n               coordinates in the ATOM_SITE category to fractional coordinates\n               in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.fract_transf_vector[].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.fract_transf_matrix[1][1]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_matrix_11' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_matrix[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the 3x3 matrix used to transform Cartesian\n               coordinates in the ATOM_SITE category to fractional coordinates\n               in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.fract_transf_vector[].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.fract_transf_matrix[1][2]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_matrix_12' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_matrix[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the 3x3 matrix used to transform Cartesian\n               coordinates in the ATOM_SITE category to fractional coordinates\n               in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.fract_transf_vector[].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.fract_transf_matrix[1][3]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_matrix_13' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_matrix[2][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][1] element of the 3x3 matrix used to transform Cartesian\n               coordinates in the ATOM_SITE category to fractional coordinates\n               in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.fract_transf_vector[].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.fract_transf_matrix[2][1]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_matrix_21' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_matrix[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the 3x3 matrix used to transform Cartesian\n               coordinates in the ATOM_SITE category to fractional coordinates\n               in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.fract_transf_vector[].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.fract_transf_matrix[2][2]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_matrix_22' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_matrix[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the 3x3 matrix used to transform Cartesian\n               coordinates in the ATOM_SITE category to fractional coordinates\n               in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.fract_transf_vector[].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.fract_transf_matrix[2][3]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_matrix_23' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_matrix[3][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][1] element of the 3x3 matrix used to transform Cartesian\n               coordinates in the ATOM_SITE category to fractional coordinates\n               in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.fract_transf_vector[].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.fract_transf_matrix[3][1]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_matrix_31' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_matrix[3][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][2] element of the 3x3 matrix used to transform Cartesian\n               coordinates in the ATOM_SITE category to fractional coordinates\n               in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.fract_transf_vector[].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.fract_transf_matrix[3][2]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_matrix_32' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_matrix[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the 3x3 matrix used to transform Cartesian\n               coordinates in the ATOM_SITE category to fractional coordinates\n               in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x1 translation is defined in\n               _atom_sites.fract_transf_vector[].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_',
            '_item.name',
            '_atom_sites.fract_transf_matrix[3][3]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_matrix_33' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_vector[1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1] element of the three-element vector used to transform\n               Cartesian coordinates in the ATOM_SITE category to fractional\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x3 rotation is defined in\n               _atom_sites.fract_transf_matrix[][].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_', '_item.name', '_atom_sites.fract_transf_vector[1]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_vector_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_vector[2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2] element of the three-element vector used to transform\n               Cartesian coordinates in the ATOM_SITE category to fractional\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x3 rotation is defined in\n               _atom_sites.fract_transf_matrix[][].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_', '_item.name', '_atom_sites.fract_transf_vector[2]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_vector_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.fract_transf_vector[3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3] element of the three-element vector used to transform\n               Cartesian coordinates in the ATOM_SITE category to fractional\n               coordinates in the same category. The axial alignments of this\n               transformation are described in _atom_sites.Cartn_transform_axes.\n               The 3x3 rotation is defined in\n               _atom_sites.fract_transf_matrix[][].\n\n               |x\'|               |11 12 13| |x|              |1|\n               |y\'|~fractional~ = |21 22 23| |y|~Cartesian~ + |2|\n               |z\'|               |31 32 33| |z|              |3|' ],
          [ 'item_', '_item.name', '_atom_sites.fract_transf_vector[3]' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_fract_tran_vector_3' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_sites.solution_primary',
        [ [ 'item_',
            '_item_description.description',
            '              This code identifies the method used to locate the initial\n               atom sites.\n\n               *** This data item would not in general be used in a\n               macromolecular data block. ***' ],
          [ 'item_', '_item.name', '_atom_sites.solution_primary' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_solution_primary' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'difmap',
              'difference Fourier map',
              'vecmap',
              'real-space vector search',
              'heavy',
              'heavy-atom method',
              'direct',
              'structure-invariant direct methods',
              'geom',
              'inferred from neighbouring sites',
              'disper',
              'anomalous-dispersion techniques',
              'isomor',
              'isomorphous structure methods' ] ] ] ],
      [ 'save_',
        '_atom_sites.solution_secondary',
        [ [ 'item_',
            '_item_description.description',
            '             This code identifies the method used to locate the\n              non-hydrogen-atom sites not found by\n              _atom_sites.solution_primary.\n\n               *** This data item would not in general be used in a\n               macromolecular data block. ***' ],
          [ 'item_', '_item.name', '_atom_sites.solution_secondary' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_solution_secondary' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'difmap',
              'difference Fourier map',
              'vecmap',
              'real-space vector search',
              'heavy',
              'heavy-atom method',
              'direct',
              'structure-invariant direct methods',
              'geom',
              'inferred from neighbouring sites',
              'disper',
              'anomalous-dispersion techniques',
              'isomor',
              'isomorphous structure methods' ] ] ] ],
      [ 'save_',
        '_atom_sites.solution_hydrogens',
        [ [ 'item_',
            '_item_description.description',
            '              This code identifies the method used to locate the\n               hydrogen atoms.\n\n               *** This data item would not in general be used in a\n               macromolecular data block. ***' ],
          [ 'item_', '_item.name', '_atom_sites.solution_hydrogens' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_sites_solution_hydrogens' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'difmap',
              'difference Fourier map',
              'vecmap',
              'real-space vector search',
              'heavy',
              'heavy-atom method',
              'direct',
              'structure-invariant direct methods',
              'geom',
              'inferred from neighbouring sites',
              'disper',
              'anomalous-dispersion techniques',
              'isomor',
              'isomorphous structure methods' ] ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# ATOM_SITES_ALT ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'atom_sites_alt',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ATOM_SITES_ALT category record details\n               about the structural ensembles that should be generated from\n               atom sites or groups of atom sites that are modelled in\n               alternative conformations in this data block.' ],
          [ 'item_', '_category.id', 'atom_sites_alt' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_atom_sites_alt.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'atom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _atom_sites_alt.id\n    _atom_sites_alt.details\n     .\n    ; Atom sites with the alternative ID set to null are not\n      modeled in alternative conformations\n    ;\n     1\n    ; Atom sites with the alternative ID set to 1 have been\n      modeled in alternative conformations with respect to atom\n      sites marked with alternative ID 2. The conformations of\n      amino-acid side chains and solvent atoms with alternative\n      ID set to 1 correlate with the conformation of the\n      inhibitor marked with alternative ID 1. They have been\n      given an occupancy of 0.58 to match the occupancy assigned\n      to the inhibitor.\n    ;\n     2\n    ; Atom sites with the alternative ID set to 2 have been\n      modeled in alternative conformations with respect to atom\n      sites marked with alternative ID 1. The conformations of\n      amino-acid side chains and solvent atoms with alternative\n      ID set to 2 correlate with the conformation of the\n      inhibitor marked with alternative ID 2. They have been\n      given an occupancy of 0.42 to match the occupancy assigned\n      to the inhibitor.\n    ;\n     3\n    ; Atom sites with the alternative ID set to 3 have been\n      modeled in alternative conformations with respect to\n      atoms marked with alternative ID 4. The conformations of\n      amino-acid side chains and solvent atoms with alternative\n      ID set to 3 do not correlate with the conformation of the\n      inhibitor. These atom sites have arbitrarily been given\n      an occupancy of 0.50.\n    ;\n     4\n    ; Atom sites with the alternative ID set to 4 have been\n      modeled in alternative conformations with respect to\n      atoms marked with alternative ID 3. The conformations of\n      amino-acid side chains and solvent atoms with alternative\n      ID set to 4 do not correlate with the conformation of the\n      inhibitor. These atom sites have arbitrarily been given\n      an occupancy of 0.50.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_atom_sites_alt.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the modelling of atoms in\n               alternative conformations.' ],
          [ 'item_', '_item.name', '_atom_sites_alt.details' ],
          [ 'item_', '_item.category_id', 'atom_sites_alt' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_atom_sites_alt.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _atom_sites_alt.id must uniquely identify\n               a record in the ATOM_SITES_ALT list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_atom_sites_alt.id',
              'atom_sites_alt',
              'yes',
              '_atom_site.label_alt_id',
              'atom_site',
              'no',
              '_atom_sites_alt_gen.alt_id',
              'atom_sites_alt_gen',
              'yes',
              '_geom_angle.atom_site_label_alt_id_1',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_label_alt_id_2',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_label_alt_id_3',
              'geom_angle',
              'no',
              '_geom_bond.atom_site_label_alt_id_1',
              'geom_bond',
              'no',
              '_geom_bond.atom_site_label_alt_id_2',
              'geom_bond',
              'no',
              '_geom_contact.atom_site_label_alt_id_1',
              'geom_contact',
              'no',
              '_geom_contact.atom_site_label_alt_id_2',
              'geom_contact',
              'no',
              '_geom_hbond.atom_site_label_alt_id_A',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_label_alt_id_D',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_label_alt_id_H',
              'geom_hbond',
              'no',
              '_geom_torsion.atom_site_label_alt_id_1',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_alt_id_2',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_alt_id_3',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_alt_id_4',
              'geom_torsion',
              'no',
              '_struct_conn.ptnr1_label_alt_id',
              'struct_conn',
              'no',
              '_struct_conn.ptnr2_label_alt_id',
              'struct_conn',
              'no',
              '_struct_mon_nucl.label_alt_id',
              'struct_mon_nucl',
              'yes',
              '_struct_mon_prot.label_alt_id',
              'struct_mon_prot',
              'yes',
              '_struct_mon_prot_cis.label_alt_id',
              'struct_mon_prot_cis',
              'yes',
              '_struct_ncs_dom_lim.beg_label_alt_id',
              'struct_ncs_dom_lim',
              'yes',
              '_struct_ncs_dom_lim.end_label_alt_id',
              'struct_ncs_dom_lim',
              'yes',
              '_struct_site_gen.label_alt_id',
              'struct_site_gen',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_site.label_alt_id',
              '_atom_sites_alt.id',
              '_atom_sites_alt_gen.alt_id',
              '_atom_sites_alt.id',
              '_geom_angle.atom_site_label_alt_id_1',
              '_atom_site.label_alt_id',
              '_geom_angle.atom_site_label_alt_id_2',
              '_atom_site.label_alt_id',
              '_geom_angle.atom_site_label_alt_id_3',
              '_atom_site.label_alt_id',
              '_geom_bond.atom_site_label_alt_id_1',
              '_atom_site.label_alt_id',
              '_geom_bond.atom_site_label_alt_id_2',
              '_atom_site.label_alt_id',
              '_geom_contact.atom_site_label_alt_id_1',
              '_atom_site.label_alt_id',
              '_geom_contact.atom_site_label_alt_id_2',
              '_atom_site.label_alt_id',
              '_geom_hbond.atom_site_label_alt_id_A',
              '_atom_site.label_alt_id',
              '_geom_hbond.atom_site_label_alt_id_D',
              '_atom_site.label_alt_id',
              '_geom_hbond.atom_site_label_alt_id_H',
              '_atom_site.label_alt_id',
              '_geom_torsion.atom_site_label_alt_id_1',
              '_atom_site.label_alt_id',
              '_geom_torsion.atom_site_label_alt_id_2',
              '_atom_site.label_alt_id',
              '_geom_torsion.atom_site_label_alt_id_3',
              '_atom_site.label_alt_id',
              '_geom_torsion.atom_site_label_alt_id_4',
              '_atom_site.label_alt_id',
              '_struct_conn.ptnr1_label_alt_id',
              '_atom_site.label_alt_id',
              '_struct_conn.ptnr2_label_alt_id',
              '_atom_site.label_alt_id',
              '_struct_mon_nucl.label_alt_id',
              '_atom_site.label_alt_id',
              '_struct_mon_prot.label_alt_id',
              '_atom_site.label_alt_id',
              '_struct_mon_prot_cis.label_alt_id',
              '_atom_site.label_alt_id',
              '_struct_ncs_dom_lim.beg_label_alt_id',
              '_atom_site.label_alt_id',
              '_struct_ncs_dom_lim.end_label_alt_id',
              '_atom_site.label_alt_id',
              '_struct_site_gen.label_alt_id',
              '_atom_site.label_alt_id' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'orientation 1', 'molecule abc' ] ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# ATOM_SITES_ALT_ENS ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'atom_sites_alt_ens',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ATOM_SITES_ALT_ENS category record details\n               about the ensemble structure generated from atoms with various\n               alternative conformation IDs.' ],
          [ 'item_', '_category.id', 'atom_sites_alt_ens' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_atom_sites_alt_ens.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'atom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _atom_sites_alt_ens.id\n    _atom_sites_alt_ens.details\n     \'Ensemble 1-A\'\n    ; The inhibitor binds to the enzyme in two, roughly twofold\n      symmetric alternative conformations.\n\n      This conformational ensemble includes the more populated\n      conformation of the inhibitor (ID=1) and the amino-acid\n      side chains and solvent structure that correlate with this\n      inhibitor conformation.\n\n      Also included are one set (ID=3) of side chains with\n      alternative conformations when the conformations are not\n      correlated with the inhibitor conformation.\n    ;\n     \'Ensemble 1-B\'\n    ; The inhibitor binds to the enzyme in two, roughly twofold\n      symmetric alternative conformations.\n\n      This conformational ensemble includes the more populated\n      conformation of the inhibitor (ID=1) and the amino-acid\n      side chains and solvent structure that correlate with\n      this inhibitor conformation.\n\n      Also included are one set (ID=4) of side chains with\n      alternative conformations when the conformations are not\n      correlated with the inhibitor conformation.\n    ;\n     \'Ensemble 2-A\'\n    ; The inhibitor binds to the enzyme in two, roughly twofold\n      symmetric alternative conformations.\n\n      This conformational ensemble includes the less populated\n      conformation of the inhibitor (ID=2) and the amino-acid\n      side chains and solvent structure that correlate with this\n      inhibitor conformation.\n\n      Also included are one set (ID=3) of side chains with\n      alternative conformations when the conformations are not\n      correlated with the inhibitor conformation.\n    ;\n     \'Ensemble 2-B\'\n    ; The inhibitor binds to the enzyme in two, roughly twofold\n      symmetric alternative conformations.\n\n      This conformational ensemble includes the less populated\n      conformation of the inhibitor (ID=2) and the amino-acid\n      side chains and solvent structure that correlate with this\n      inhibitor conformation.\n\n      Also included are one set (ID=4) of side chains with\n      alternative conformations when the conformations are not\n      correlated with the inhibitor conformation.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_atom_sites_alt_ens.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the ensemble structure\n               generated from atoms with various alternative IDs.' ],
          [ 'item_', '_item.name', '_atom_sites_alt_ens.details' ],
          [ 'item_', '_item.category_id', 'atom_sites_alt_ens' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_atom_sites_alt_ens.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _atom_sites_alt_ens.id must uniquely identify a\n               record in the ATOM_SITES_ALT_ENS list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_atom_sites_alt_ens.id',
              'atom_sites_alt_ens',
              'yes',
              '_atom_sites_alt_gen.ens_id',
              'atom_sites_alt_gen',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_sites_alt_gen.ens_id', '_atom_sites_alt_ens.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# ATOM_SITES_ALT_GEN ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'atom_sites_alt_gen',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ATOM_SITES_ALT_GEN category record details\n               about the interpretation of multiple conformations in the\n               structure.' ],
          [ 'item_', '_category.id', 'atom_sites_alt_gen' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_atom_sites_alt_gen.ens_id', '_atom_sites_alt_gen.alt_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'atom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _atom_sites_alt_gen.ens_id\n    _atom_sites_alt_gen.alt_id\n     \'Ensemble 1-A\'  .\n     \'Ensemble 1-A\'  1\n     \'Ensemble 1-A\'  3\n     \'Ensemble 1-B\'  .\n     \'Ensemble 1-B\'  1\n     \'Ensemble 1-B\'  4\n     \'Ensemble 2-A\'  .\n     \'Ensemble 2-A\'  2\n     \'Ensemble 2-A\'  3\n     \'Ensemble 2-B\'  .\n     \'Ensemble 2-B\'  2\n     \'Ensemble 2-B\'  4',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_atom_sites_alt_gen.alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_atom_sites_alt_gen.alt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_atom_sites_alt_gen.ens_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _atom_sites_alt_ens.id in the\n               ATOM_SITES_ALT_ENS category.' ],
          [ 'item_', '_item.name', '_atom_sites_alt_gen.ens_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# ATOM_SITES_FOOTNOTE ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'atom_sites_footnote',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ATOM_SITES_FOOTNOTE category record detailed\n               comments about an atom site or a group of atom sites.' ],
          [ 'item_', '_category.id', 'atom_sites_footnote' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_atom_sites_footnote.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'atom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _atom_sites_footnote.id\n    _atom_sites_footnote.text\n      1\n    ; The inhibitor binds to the enzyme in two alternative\n      orientations. The two orientations have been assigned\n      alternative IDs *1* and *2*.\n    ;\n      2\n    ; Side chains of these residues adopt alternative\n      orientations that correlate with the alternative\n      orientations of the inhibitor.\n      Side chains with alternative ID *1* and occupancy 0.58\n      correlate with inhibitor orientation *1*.\n      Side chains with alternative ID *2* and occupancy 0.42\n      correlate with inhibitor orientation *2*.\n    ;\n      3\n    ; The positions of these water molecules correlate with\n      the alternative orientations of the inhibitor.\n      Water molecules with alternative ID *1* and occupancy 0.58\n      correlate with inhibitor orientation *1*.\n      Water molecules with alternative ID *2* and occupancy 0.42\n      correlate with inhibitor orientation *2*.\n    ;\n      4\n    ; Side chains of these residues adopt alternative\n      orientations that do not correlate with the alternative\n      orientation of the inhibitor.\n    ;\n      5\n    ; The positions of these water molecules correlate with\n      alternative orientations of amino-acid side chains that\n      do not correlate with alternative orientations of the\n      inhibitor.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_atom_sites_footnote.id',
        [ [ 'item_',
            '_item_description.description',
            '              A code that identifies the footnote.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_atom_sites_footnote.id',
              'atom_sites_footnote',
              'yes',
              '_atom_site.footnote_id',
              'atom_site',
              'no' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_site.footnote_id', '_atom_sites_footnote.id' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'a', 'b', '1', '2' ] ] ] ],
      [ 'save_',
        '_atom_sites_footnote.text',
        [ [ 'item_',
            '_item_description.description',
            '              The text of the footnote. Footnotes are used to describe\n               an atom site or a group of atom sites in the ATOM_SITE list.\n\n               For example, footnotes may be used to indicate atoms for which\n               the electron density is very weak, or atoms for which static\n               disorder has been modelled.' ],
          [ 'item_', '_item.name', '_atom_sites_footnote.text' ],
          [ 'item_', '_item.category_id', 'atom_sites_footnote' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '##############' ],
      [ '@comment', '# ATOM_TYPE ##' ],
      [ '@comment', '##############' ],
      [ 'save_',
        'atom_type',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ATOM_TYPE category record details about the\n               properties of the atoms that occupy the atom sites, such as the\n               atomic scattering factors.' ],
          [ 'item_', '_category.id', 'atom_type' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_atom_type.symbol' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'atom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _atom_type.symbol\n    _atom_type.oxidation_number\n    _atom_type.scat_Cromer_Mann_a1\n    _atom_type.scat_Cromer_Mann_a2\n    _atom_type.scat_Cromer_Mann_a3\n    _atom_type.scat_Cromer_Mann_a4\n    _atom_type.scat_Cromer_Mann_b1\n    _atom_type.scat_Cromer_Mann_b2\n    _atom_type.scat_Cromer_Mann_b3\n    _atom_type.scat_Cromer_Mann_b4\n    _atom_type.scat_Cromer_Mann_c\n      C   0   2.31000  20.8439    1.02000   10.2075\n              1.58860   0.568700  0.865000  51.6512      0.21560\n      N   0  12.2126    0.005700  3.13220    9.89330\n              2.01250  28.9975    1.16630    0.582600  -11.529\n      O   0   3.04850  13.2771    2.28680    5.70110\n              1.54630   0.323900  0.867000  32.9089      0.250800\n      S   0   6.90530   1.46790   5.20340   22.2151\n              1.43790   0.253600  1.58630   56.1720      0.866900\n      CL -1  18.2915    0.006600  7.20840    1.17170\n              6.53370  19.5424    2.33860   60.4486    -16.378',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    loop_\n    _atom_type.symbol\n    _atom_type.oxidation_number\n    _atom_type.number_in_cell\n    _atom_type.scat_dispersion_real\n    _atom_type.scat_dispersion_imag\n    _atom_type.scat_source\n     C  0  72  .017  .009  International_Tables_Vol_IV_Table_2.2B\n     H  0  100  0     0    International_Tables_Vol_IV_Table_2.2B\n     O  0  12  .047  .032  International_Tables_Vol_IV_Table_2.2B\n     N  0  4   .029  .018  International_Tables_Vol_IV_Table_2.2B',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_atom_type.analytical_mass_percent',
        [ [ 'item_',
            '_item_description.description',
            '              Mass percentage of this atom type derived from chemical analysis.' ],
          [ 'item_', '_item.name', '_atom_type.analytical_mass_percent' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_analytical_mass_%' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.description',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the atom(s) designated by this atom type. In\n               most cases, this is the element name and oxidation state of\n               a single atom species. For disordered or nonstoichiometric\n               structures it will describe a combination of atom species.' ],
          [ 'item_', '_item.name', '_atom_type.description' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_type_description' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'deuterium', '0.34Fe+0.66Ni' ] ] ] ],
      [ 'save_',
        '_atom_type.number_in_cell',
        [ [ 'item_',
            '_item_description.description',
            '              Total number of atoms of this atom type in the unit cell.' ],
          [ 'item_', '_item.name', '_atom_type.number_in_cell' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_number_in_cell' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_atom_type.oxidation_number',
        [ [ 'item_',
            '_item_description.description',
            '              Formal oxidation state of this atom type in the structure.' ],
          [ 'item_', '_item.name', '_atom_type.oxidation_number' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_oxidation_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '8', '8', '8', '-8', '-8', '-8' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_atom_type.radius_bond',
        [ [ 'item_',
            '_item_description.description',
            '              The effective intramolecular bonding radius in angstroms\n               of this atom type.' ],
          [ 'item_', '_item.name', '_atom_type.radius_bond' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_type_radius_bond' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '5.0', '5.0', '5.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_atom_type.radius_contact',
        [ [ 'item_',
            '_item_description.description',
            '              The effective intermolecular bonding radius in angstroms\n               of this atom type.' ],
          [ 'item_', '_item.name', '_atom_type.radius_contact' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_radius_contact' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '5.0', '5.0', '5.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_atom_type.scat_Cromer_Mann_a1',
        [ [ 'item_',
            '_item_description.description',
            '              The Cromer-Mann scattering-factor coefficient a1 used to\n               calculate the scattering factors for this atom type.\n\n               Ref: International Tables for X-ray Crystallography (1974).\n                   Vol. IV, Table 2.2B\n                or: International Tables for Crystallography  (2004). Vol. C,\n                    Tables 6.1.1.4 and 6.1.1.5.' ],
          [ 'item_', '_item.name', '_atom_type.scat_Cromer_Mann_a1' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_Cromer_Mann_a1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_type.scat_Cromer_Mann_a2',
              '_atom_type.scat_Cromer_Mann_a3',
              '_atom_type.scat_Cromer_Mann_a4',
              '_atom_type.scat_Cromer_Mann_b1',
              '_atom_type.scat_Cromer_Mann_b2',
              '_atom_type.scat_Cromer_Mann_b3',
              '_atom_type.scat_Cromer_Mann_b4',
              '_atom_type.scat_Cromer_Mann_c' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_Cromer_Mann_a2',
        [ [ 'item_',
            '_item_description.description',
            '              The Cromer-Mann scattering-factor coefficient a2 used to\n               calculate the scattering factors for this atom type.\n\n               Ref: International Tables for X-ray Crystallography (1974).\n                   Vol. IV, Table 2.2B\n                or: International Tables for Crystallography  (2004). Vol. C,\n                    Tables 6.1.1.4 and 6.1.1.5.' ],
          [ 'item_', '_item.name', '_atom_type.scat_Cromer_Mann_a2' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_Cromer_Mann_a2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_type.scat_Cromer_Mann_a1',
              '_atom_type.scat_Cromer_Mann_a3',
              '_atom_type.scat_Cromer_Mann_a4',
              '_atom_type.scat_Cromer_Mann_b1',
              '_atom_type.scat_Cromer_Mann_b2',
              '_atom_type.scat_Cromer_Mann_b3',
              '_atom_type.scat_Cromer_Mann_b4',
              '_atom_type.scat_Cromer_Mann_c' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_Cromer_Mann_a3',
        [ [ 'item_',
            '_item_description.description',
            '              The Cromer-Mann scattering-factor coefficient a3 used to\n               calculate the scattering factors for this atom type.\n\n               Ref: International Tables for X-ray Crystallography (1974).\n                   Vol. IV, Table 2.2B\n                or: International Tables for Crystallography  (2004). Vol. C,\n                    Tables 6.1.1.4 and 6.1.1.5.' ],
          [ 'item_', '_item.name', '_atom_type.scat_Cromer_Mann_a3' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_Cromer_Mann_a3' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_type.scat_Cromer_Mann_a1',
              '_atom_type.scat_Cromer_Mann_a2',
              '_atom_type.scat_Cromer_Mann_a4',
              '_atom_type.scat_Cromer_Mann_b1',
              '_atom_type.scat_Cromer_Mann_b2',
              '_atom_type.scat_Cromer_Mann_b3',
              '_atom_type.scat_Cromer_Mann_b4',
              '_atom_type.scat_Cromer_Mann_c' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_Cromer_Mann_a4',
        [ [ 'item_',
            '_item_description.description',
            '              The Cromer-Mann scattering-factor coefficient a4 used to\n               calculate the scattering factors for this atom type.\n\n               Ref: International Tables for X-ray Crystallography (1974).\n                   Vol. IV, Table 2.2B\n                or: International Tables for Crystallography  (2004). Vol. C,\n                    Tables 6.1.1.4 and 6.1.1.5.' ],
          [ 'item_', '_item.name', '_atom_type.scat_Cromer_Mann_a4' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_Cromer_Mann_a4' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_type.scat_Cromer_Mann_a1',
              '_atom_type.scat_Cromer_Mann_a2',
              '_atom_type.scat_Cromer_Mann_a3',
              '_atom_type.scat_Cromer_Mann_b1',
              '_atom_type.scat_Cromer_Mann_b2',
              '_atom_type.scat_Cromer_Mann_b3',
              '_atom_type.scat_Cromer_Mann_b4',
              '_atom_type.scat_Cromer_Mann_c' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_Cromer_Mann_b1',
        [ [ 'item_',
            '_item_description.description',
            '              The Cromer-Mann scattering-factor coefficient b1 used to\n               calculate the scattering factors for this atom type.\n\n               Ref: International Tables for X-ray Crystallography (1974).\n                   Vol. IV, Table 2.2B\n                or: International Tables for Crystallography  (2004). Vol. C,\n                    Tables 6.1.1.4 and 6.1.1.5.' ],
          [ 'item_', '_item.name', '_atom_type.scat_Cromer_Mann_b1' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_Cromer_Mann_b1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_type.scat_Cromer_Mann_a1',
              '_atom_type.scat_Cromer_Mann_a2',
              '_atom_type.scat_Cromer_Mann_a3',
              '_atom_type.scat_Cromer_Mann_a4',
              '_atom_type.scat_Cromer_Mann_b2',
              '_atom_type.scat_Cromer_Mann_b3',
              '_atom_type.scat_Cromer_Mann_b4',
              '_atom_type.scat_Cromer_Mann_c' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_Cromer_Mann_b2',
        [ [ 'item_',
            '_item_description.description',
            '              The Cromer-Mann scattering-factor coefficient b2 used to\n               calculate the scattering factors for this atom type.\n\n               Ref: International Tables for X-ray Crystallography (1974).\n                   Vol. IV, Table 2.2B\n                or: International Tables for Crystallography  (2004). Vol. C,\n                    Tables 6.1.1.4 and 6.1.1.5.' ],
          [ 'item_', '_item.name', '_atom_type.scat_Cromer_Mann_b2' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_Cromer_Mann_b2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_type.scat_Cromer_Mann_a1',
              '_atom_type.scat_Cromer_Mann_a2',
              '_atom_type.scat_Cromer_Mann_a3',
              '_atom_type.scat_Cromer_Mann_a4',
              '_atom_type.scat_Cromer_Mann_b1',
              '_atom_type.scat_Cromer_Mann_b3',
              '_atom_type.scat_Cromer_Mann_b4',
              '_atom_type.scat_Cromer_Mann_c' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_Cromer_Mann_b3',
        [ [ 'item_',
            '_item_description.description',
            '              The Cromer-Mann scattering-factor coefficient b3 used to\n               calculate the scattering factors for this atom type.\n\n               Ref: International Tables for X-ray Crystallography (1974).\n                   Vol. IV, Table 2.2B\n                or: International Tables for Crystallography  (2004). Vol. C,\n                    Tables 6.1.1.4 and 6.1.1.5.' ],
          [ 'item_', '_item.name', '_atom_type.scat_Cromer_Mann_b3' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_Cromer_Mann_b3' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_type.scat_Cromer_Mann_a1',
              '_atom_type.scat_Cromer_Mann_a2',
              '_atom_type.scat_Cromer_Mann_a3',
              '_atom_type.scat_Cromer_Mann_a4',
              '_atom_type.scat_Cromer_Mann_b1',
              '_atom_type.scat_Cromer_Mann_b2',
              '_atom_type.scat_Cromer_Mann_b4',
              '_atom_type.scat_Cromer_Mann_c' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_Cromer_Mann_b4',
        [ [ 'item_',
            '_item_description.description',
            '              The Cromer-Mann scattering-factor coefficient b4 used to\n               calculate the scattering factors for this atom type.\n\n               Ref: International Tables for X-ray Crystallography (1974).\n                   Vol. IV, Table 2.2B\n                or: International Tables for Crystallography  (2004). Vol. C,\n                    Tables 6.1.1.4 and 6.1.1.5.' ],
          [ 'item_', '_item.name', '_atom_type.scat_Cromer_Mann_b4' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_Cromer_Mann_b4' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_type.scat_Cromer_Mann_a1',
              '_atom_type.scat_Cromer_Mann_a2',
              '_atom_type.scat_Cromer_Mann_a3',
              '_atom_type.scat_Cromer_Mann_a4',
              '_atom_type.scat_Cromer_Mann_b1',
              '_atom_type.scat_Cromer_Mann_b2',
              '_atom_type.scat_Cromer_Mann_b3',
              '_atom_type.scat_Cromer_Mann_c' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_Cromer_Mann_c',
        [ [ 'item_',
            '_item_description.description',
            '              The Cromer-Mann scattering-factor coefficient c used to\n               calculate the scattering factors for this atom type.\n\n               Ref: International Tables for X-ray Crystallography (1974).\n                   Vol. IV, Table 2.2B\n                or: International Tables for Crystallography  (2004). Vol. C,\n                    Tables 6.1.1.4 and 6.1.1.5.' ],
          [ 'item_', '_item.name', '_atom_type.scat_Cromer_Mann_c' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_Cromer_Mann_c' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_atom_type.scat_Cromer_Mann_a1',
              '_atom_type.scat_Cromer_Mann_a2',
              '_atom_type.scat_Cromer_Mann_a3',
              '_atom_type.scat_Cromer_Mann_a4',
              '_atom_type.scat_Cromer_Mann_b1',
              '_atom_type.scat_Cromer_Mann_b2',
              '_atom_type.scat_Cromer_Mann_b3',
              '_atom_type.scat_Cromer_Mann_b4' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_dispersion_imag',
        [ [ 'item_',
            '_item_description.description',
            '              The imaginary component of the anomalous-dispersion\n               scattering factor, f\'\', in electrons for this atom type and\n               the radiation identified by _diffrn_radiation_wavelength.id.' ],
          [ 'item_', '_item.name', '_atom_type.scat_dispersion_imag' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_dispersion_imag' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_atom_type.scat_dispersion_real' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_dispersion_real',
        [ [ 'item_',
            '_item_description.description',
            '              The real component of the anomalous-dispersion\n               scattering factor, f\', in electrons for this atom type and\n               the radiation identified by _diffrn_radiation_wavelength.id.' ],
          [ 'item_', '_item.name', '_atom_type.scat_dispersion_real' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_dispersion_real' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_atom_type.scat_dispersion_imag' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_atom_type.scat_length_neutron',
        [ [ 'item_',
            '_item_description.description',
            '              The bound coherent scattering length in femtometres for the\n               atom type at the isotopic composition used for the diffraction\n               experiment.' ],
          [ 'item_', '_item.name', '_atom_type.scat_length_neutron' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_length_neutron' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_units.code', 'femtometres' ] ] ],
      [ 'save_',
        '_atom_type.scat_source',
        [ [ 'item_',
            '_item_description.description',
            '              Reference to the source of the scattering factors or scattering\n               lengths used for this atom type.' ],
          [ 'item_', '_item.name', '_atom_type.scat_source' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_type_scat_source' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'International Tables Vol. IV Table 2.4.6B' ] ] ],
      [ 'save_',
        '_atom_type.scat_versus_stol_list',
        [ [ 'item_',
            '_item_description.description',
            '              A table of scattering factors as a function of sin theta over\n               lambda. This table should be well commented to indicate the\n               items present. Regularly formatted lists are strongly\n               recommended.' ],
          [ 'item_', '_item.name', '_atom_type.scat_versus_stol_list' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_type_scat_versus_stol_list' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_atom_type.symbol',
        [ [ 'item_',
            '_item_description.description',
            '              The code used to identify the atom species (singular or plural)\n               representing this atom type. Normally this code is the element\n               symbol. The code may be composed of any character except\n               an underscore with the additional proviso that digits designate\n               an oxidation state and must be followed by a + or - character.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_atom_type.symbol',
              'atom_type',
              'yes',
              '_atom_site.type_symbol',
              'atom_site',
              'yes',
              '_atom_site_anisotrop.type_symbol',
              'atom_site_anisotrop',
              'yes',
              '_chemical_conn_atom.type_symbol',
              'chemical_conn_atom',
              'yes',
              '_chem_comp_atom.type_symbol',
              'chem_comp_atom',
              'yes',
              '_phasing_MIR_der_site.atom_type_symbol',
              'phasing_MIR_der_site',
              'yes' ] ],
          [ 'item_', '_item_aliases.alias_name', '_atom_type_symbol' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_site.type_symbol',
              '_atom_type.symbol',
              '_atom_site_anisotrop.type_symbol',
              '_atom_type.symbol',
              '_chemical_conn_atom.type_symbol',
              '_atom_type.symbol',
              '_chem_comp_atom.type_symbol',
              '_atom_type.symbol',
              '_phasing_MIR_der_site.atom_type_symbol',
              '_atom_type.symbol' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'C', 'Cu2+', 'H(SDS)', 'dummy', 'FeNi' ] ] ] ],
      [ '@comment', '##########' ],
      [ '@comment', '# AUDIT ##' ],
      [ '@comment', '##########' ],
      [ 'save_',
        'audit',
        [ [ 'item_',
            '_category.description',
            '              Data items in the AUDIT category record details about the\n               creation and subsequent updating of the data block.\n\n               Note that these items apply only to the creation and updating of\n               the data block, and should not be confused with the data items\n               in the JOURNAL category that record different stages in the\n               publication of the material in the data block.' ],
          [ 'item_', '_category.id', 'audit' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_audit.revision_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'audit_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _audit.revision_id                   1\n    _audit.creation_date                \'1992-12-08\'\n\n    _audit.creation_method\n    ; Created by hand from PDB entry 5HVP, from the J. Biol.\n      Chem. paper describing this structure and from\n      laboratory records\n    ;\n\n    _audit.update_record\n    ; 1992-12-09  adjusted to reflect comments from B. McKeever\n      1992-12-10  adjusted to reflect comments from H. Berman\n      1992-12-12  adjusted to reflect comments from K. Watenpaugh\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    _audit.creation_date               1991-03-20\n    _audit.creation_method     from_xtal_archive_file_using_CIFIO\n    _audit.update_record\n    ; 1991-04-09  text and data added by Tony Willis.\n      1991-04-15  rec\'d by co-editor as manuscript HL0007.\n      1991-04-17  adjustments based on first referee report.\n      1991-04-18  adjustments based on second referee report.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_audit.creation_date',
        [ [ 'item_',
            '_item_description.description',
            '              A date that the data block was created. The date format is\n               yyyy-mm-dd.' ],
          [ 'item_', '_item.name', '_audit.creation_date' ],
          [ 'item_', '_item.category_id', 'audit' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_audit_creation_date' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ],
          [ 'item_', '_item_examples.case', '1990-07-12' ] ] ],
      [ 'save_',
        '_audit.creation_method',
        [ [ 'item_',
            '_item_description.description',
            '              A description of how data were entered into the data block.' ],
          [ 'item_', '_item.name', '_audit.creation_method' ],
          [ 'item_', '_item.category_id', 'audit' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_audit_creation_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'spawned by the program QBEE' ] ] ],
      [ 'save_',
        '_audit.revision_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _audit.revision_id must uniquely identify a record\n               in the AUDIT list.' ],
          [ 'item_', '_item.name', '_audit.revision_id' ],
          [ 'item_', '_item.category_id', 'audit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', 'rev1' ] ] ],
      [ 'save_',
        '_audit.update_record',
        [ [ 'item_',
            '_item_description.description',
            '              A record of any changes to the data block. The update format is\n               a date (yyyy-mm-dd) followed by a description of the changes.\n               The latest update entry is added to the bottom of this record.' ],
          [ 'item_', '_item.name', '_audit.update_record' ],
          [ 'item_', '_item.category_id', 'audit' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_audit_update_record' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '1990-07-15   Updated by the Co-editor' ] ] ],
      [ '@comment', '#################' ],
      [ '@comment', '# AUDIT_AUTHOR ##' ],
      [ '@comment', '#################' ],
      [ 'save_',
        'audit_author',
        [ [ 'item_',
            '_category.description',
            '              Data items in the AUDIT_AUTHOR category record details about\n               the author(s) of the data block.' ],
          [ 'item_', '_category.id', 'audit_author' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_audit_author.name' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'audit_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _audit_author.name\n    _audit_author.address\n     \'Fitzgerald, Paula M.D.\'\n    ; Department of Biophysical Chemistry\n      Merck Research Laboratories\n      P. O. Box 2000, Ry80M203\n      Rahway, New Jersey  07065\n      USA\n    ;\n     \'McKeever, Brian M.\'\n    ; Department of Biophysical Chemistry\n      Merck Research Laboratories\n      P. O. Box 2000, Ry80M203\n      Rahway, New Jersey  07065\n      USA\n    ;\n     \'Van Middlesworth, J.F.\'\n    ; Department of Biophysical Chemistry\n      Merck Research Laboratories\n      P. O. Box 2000, Ry80M203\n      Rahway, New Jersey  07065\n      USA\n    ;\n     \'Springer, James P.\'\n    ; Department of Biophysical Chemistry\n      Merck Research Laboratories\n      P. O. Box 2000, Ry80M203\n      Rahway, New Jersey  07065\n      USA\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_audit_author.address',
        [ [ 'item_',
            '_item_description.description',
            '              The address of an author of this data block. If there are\n               multiple authors, _audit_author.address is looped with\n               _audit_author.name.' ],
          [ 'item_', '_item.name', '_audit_author.address' ],
          [ 'item_', '_item.category_id', 'audit_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_audit_author_address' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Department\n                                  Institute\n                                  Street\n                                  City and postcode\n                                  COUNTRY' ] ] ],
      [ 'save_',
        '_audit_author.name',
        [ [ 'item_',
            '_item_description.description',
            '              The name of an author of this data block. If there are multiple\n               authors, _audit_author.name is looped with _audit_author.address.\n               The family name(s), followed by a comma and including any\n               dynastic components, precedes the first name(s) or initial(s).' ],
          [ 'item_', '_item.name', '_audit_author.name' ],
          [ 'item_', '_item.category_id', 'audit_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_audit_author_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Bleary, Percival R.',
              'O\'Neil, F.K.',
              'Van den Bossche, G.',
              'Yang, D.-L.',
              'Simonov, Yu.A' ] ] ] ],
      [ '@comment', '##################' ],
      [ '@comment', '# AUDIT_CONFORM ##' ],
      [ '@comment', '##################' ],
      [ 'save_',
        'audit_conform',
        [ [ 'item_',
            '_category.description',
            '              Data items in the AUDIT_CONFORM category describe the\n               dictionary versions against which the data names appearing in\n               the current data block are conformant.' ],
          [ 'item_', '_category.id', 'audit_conform' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_audit_conform.dict_name', '_audit_conform.dict_version' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'audit_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - any file conforming to the current CIF core dictionary.',
              '\n    _audit_conform.dict_name         cif_core.dic\n    _audit_conform.dict_version      2.3.1\n    _audit_conform.dict_location\n                         ftp://ftp.iucr.org/pub/cif_core.2.3.1.dic',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_audit_conform.dict_location',
        [ [ 'item_',
            '_item_description.description',
            '              A file name or uniform resource locator (URL) for the\n               dictionary to which the current data block conforms.' ],
          [ 'item_', '_item.name', '_audit_conform.dict_location' ],
          [ 'item_', '_item.category_id', 'audit_conform' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_audit_conform_dict_location' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_audit_conform.dict_name',
        [ [ 'item_',
            '_item_description.description',
            '              The string identifying the highest-level dictionary defining\n               data names used in this file.' ],
          [ 'item_', '_item.name', '_audit_conform.dict_name' ],
          [ 'item_', '_item.category_id', 'audit_conform' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_audit_conform_dict_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_audit_conform.dict_version',
        [ [ 'item_',
            '_item_description.description',
            '              The version number of the dictionary to which the current\n               data block conforms.' ],
          [ 'item_', '_item.name', '_audit_conform.dict_version' ],
          [ 'item_', '_item.category_id', 'audit_conform' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_audit_conform_dict_version' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# AUDIT_CONTACT_AUTHOR ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'audit_contact_author',
        [ [ 'item_',
            '_category.description',
            '              Data items in the AUDIT_CONTACT_AUTHOR category record details\n               about the name and address of the author to be contacted\n               concerning the content of this data block.' ],
          [ 'item_', '_category.id', 'audit_contact_author' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_audit_contact_author.name' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'audit_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _audit_contact_author.name    \'Fitzgerald, Paula M.D.\'\n    _audit_contact_author.address\n    ; Department of Biophysical Chemistry\n      Merck Research Laboratories\n      PO Box 2000, Ry80M203\n      Rahway, New Jersey 07065\n      USA\n    ;\n    _audit_contact_author.phone   \'1(908)5945510\'\n    _audit_contact_author.fax     \'1(908)5946645\'\n    _audit_contact_author.email   \'paula_fitzgerald@merck.com\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_audit_contact_author.address',
        [ [ 'item_',
            '_item_description.description',
            '              The mailing address of the author of the data block to whom\n               correspondence should be addressed.' ],
          [ 'item_', '_item.name', '_audit_contact_author.address' ],
          [ 'item_', '_item.category_id', 'audit_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_audit_contact_author_address' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Department\n                                  Institute\n                                  Street\n                                  City and postcode\n                                  COUNTRY' ] ] ],
      [ 'save_',
        '_audit_contact_author.email',
        [ [ 'item_',
            '_item_description.description',
            '              The electronic mail address of the author of the data block to\n               whom correspondence should be addressed, in a form recognizable\n               to international networks. The format of e-mail\n               addresses is given in Section 3.4, Address Specification, of\n               Internet Message Format, RFC 2822, P. Resnick (Editor),\n               Network Standards Group, April 2001.' ],
          [ 'item_', '_item.name', '_audit_contact_author.email' ],
          [ 'item_', '_item.category_id', 'audit_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_audit_contact_author_email' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'name@host.domain.country', 'bm@iucr.org' ] ] ] ],
      [ 'save_',
        '_audit_contact_author.fax',
        [ [ 'item_',
            '_item_description.description',
            '              The facsimile telephone number of the author of the data\n               block to whom correspondence should be addressed.\n\n               The recommended style starts with the international dialing\n               prefix, followed by the area code in parentheses, followed by the\n               local number with no spaces.' ],
          [ 'item_', '_item.name', '_audit_contact_author.fax' ],
          [ 'item_', '_item.category_id', 'audit_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_audit_contact_author_fax' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '12(34)9477334', '12()349477334' ] ] ] ],
      [ 'save_',
        '_audit_contact_author.name',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the author of the data block to whom correspondence\n               should be addressed.\n\n               The family name(s), followed by a comma and including any\n               dynastic components, precedes the first name(s) or initial(s).' ],
          [ 'item_', '_item.name', '_audit_contact_author.name' ],
          [ 'item_', '_item.category_id', 'audit_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_audit_contact_author_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Bleary, Percival R.',
              'O\'Neil, F.K.',
              'Van den Bossche, G.',
              'Yang, D.-L.',
              'Simonov, Yu.A' ] ] ] ],
      [ 'save_',
        '_audit_contact_author.phone',
        [ [ 'item_',
            '_item_description.description',
            '              The telephone number of the author of the data block to whom\n               correspondence should be addressed.\n\n               The recommended style starts with the international dialing\n               prefix, followed by the area code in parentheses, followed by the\n               local number and any extension number prefixed by \'x\',\n               with no spaces.' ],
          [ 'item_', '_item.name', '_audit_contact_author.phone' ],
          [ 'item_', '_item.category_id', 'audit_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_audit_contact_author_phone' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '12(34)9477330', '12()349477330', '12(34)9477330x5543' ] ] ] ],
      [ '@comment', '#########' ],
      [ '@comment', '# CELL ##' ],
      [ '@comment', '#########' ],
      [ 'save_',
        'cell',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CELL category record details about the\n               crystallographic cell parameters.' ],
          [ 'item_', '_category.id', 'cell' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_cell.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'cell_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _cell.entry_id                         \'5HVP\'\n    _cell.length_a                         58.39\n    _cell.length_a_esd                      0.05\n    _cell.length_b                         86.70\n    _cell.length_b_esd                      0.12\n    _cell.length_c                         46.27\n    _cell.length_c_esd                      0.06\n    _cell.angle_alpha                      90.00\n    _cell.angle_beta                       90.00\n    _cell.angle_gamma                      90.00\n    _cell.volume                           234237\n    _cell.details\n    ; The cell parameters were refined every twenty frames during\n      data integration. The cell lengths given are the mean of\n      55 such refinements; the esds given are the root mean\n      square deviations of these 55 observations from that mean.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    _cell.length_a                      5.959\n    _cell.length_a_esd                  0.001\n    _cell.length_b                     14.956\n    _cell.length_b_esd                  0.001\n    _cell.length_c                     19.737\n    _cell.length_c_esd                  0.003\n    _cell.angle_alpha                  90.0\n    _cell.angle_beta                   90.0\n    _cell.angle_gamma                  90.0\n    _cell.volume                       1759.0\n    _cell.volume_esd                      0.3',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_cell.angle_alpha',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell angle alpha of the reported structure in degrees.' ],
          [ 'item_', '_item.name', '_cell.angle_alpha' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_cell_angle_alpha' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '90.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.angle_beta', '_cell.angle_gamma' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.angle_alpha_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cell_angle' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_cell.angle_alpha_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _cell.angle_alpha.' ],
          [ 'item_', '_item.name', '_cell.angle_alpha_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.angle_beta_esd', '_cell.angle_gamma_esd' ] ],
          [ 'item_', '_item_related.related_name', '_cell.angle_alpha' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cell_angle_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_cell.angle_beta',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell angle beta of the reported structure in degrees.' ],
          [ 'item_', '_item.name', '_cell.angle_beta' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_cell_angle_beta' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '90.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.angle_alpha', '_cell.angle_gamma' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_related.related_name', '_cell.angle_beta_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cell_angle' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_cell.angle_beta_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _cell.angle_beta.' ],
          [ 'item_', '_item.name', '_cell.angle_beta_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.angle_alpha_esd', '_cell.angle_gamma_esd' ] ],
          [ 'item_', '_item_related.related_name', '_cell.angle_beta' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cell_angle_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_cell.angle_gamma',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell angle gamma of the reported structure in degrees.' ],
          [ 'item_', '_item.name', '_cell.angle_gamma' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_cell_angle_gamma' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '90.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.angle_alpha', '_cell.angle_beta' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.angle_gamma_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cell_angle' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_cell.angle_gamma_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _cell.angle_gamma.' ],
          [ 'item_', '_item.name', '_cell.angle_gamma_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.angle_alpha_esd', '_cell.angle_beta_esd' ] ],
          [ 'item_', '_item_related.related_name', '_cell.angle_gamma' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cell_angle_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_cell.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_cell.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_cell.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the cell choice, noting\n               possible alternative settings.' ],
          [ 'item_', '_item.name', '_cell.details' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_cell_special_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'pseudo-orthorhombic',
              'standard setting from 45 deg rotation around c' ] ] ] ],
      [ 'save_',
        '_cell.formula_units_Z',
        [ [ 'item_',
            '_item_description.description',
            '              The number of the formula units in the unit cell as specified\n               by _chemical_formula.structural, _chemical_formula.moiety or\n               _chemical_formula.sum.' ],
          [ 'item_', '_item.name', '_cell.formula_units_Z' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_cell_formula_units_Z' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_cell.length_a',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell length a corresponding to the structure reported in\n              angstroms.' ],
          [ 'item_', '_item.name', '_cell.length_a' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_cell_length_a' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.length_b', '_cell.length_c' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_related.related_name', '_cell.length_a_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cell_length' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_cell.length_a_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _cell.length_a.' ],
          [ 'item_', '_item.name', '_cell.length_a_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.length_b_esd', '_cell.length_c_esd' ] ],
          [ 'item_', '_item_related.related_name', '_cell.length_a' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cell_length_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_cell.length_b',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell length b corresponding to the structure reported in\n               angstroms.' ],
          [ 'item_', '_item.name', '_cell.length_b' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_cell_length_b' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.length_a', '_cell.length_c' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_related.related_name', '_cell.length_b_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cell_length' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_cell.length_b_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _cell.length_b.' ],
          [ 'item_', '_item.name', '_cell.length_b_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.length_a_esd', '_cell.length_c_esd' ] ],
          [ 'item_', '_item_related.related_name', '_cell.length_b' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cell_length_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_cell.length_c',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell length c corresponding to the structure reported in\n              angstroms.' ],
          [ 'item_', '_item.name', '_cell.length_c' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_cell_length_c' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.length_a', '_cell.length_b' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_related.related_name', '_cell.length_c_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cell_length' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_cell.length_c_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _cell.length_c.' ],
          [ 'item_', '_item.name', '_cell.length_c_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell.length_a_esd', '_cell.length_b_esd' ] ],
          [ 'item_', '_item_related.related_name', '_cell.length_c' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cell_length_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_cell.volume',
        [ [ 'item_',
            '_item_description.description',
            '              Cell volume V in angstroms cubed.\n\n               V = a b c (1 - cos^2^~alpha~ - cos^2^~beta~ - cos^2^~gamma~\n                          + 2 cos~alpha~ cos~beta~ cos~gamma~)^1/2^\n\n               a     = _cell.length_a\n               b     = _cell.length_b\n               c     = _cell.length_c\n               alpha = _cell.angle_alpha\n               beta  = _cell.angle_beta\n               gamma = _cell.angle_gamma' ],
          [ 'item_', '_item.name', '_cell.volume' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_cell_volume' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_related.related_name', '_cell.volume_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_cubed' ] ] ],
      [ 'save_',
        '_cell.volume_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _cell.volume.' ],
          [ 'item_', '_item.name', '_cell.volume_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_related.related_name', '_cell.volume' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_cubed' ] ] ],
      [ 'save_',
        '_cell.Z_PDB',
        [ [ 'item_',
            '_item_description.description',
            '              The number of the polymeric chains in a unit cell. In the case\n               of heteropolymers, Z is the number of occurrences of the most\n               populous chain.\n\n               This data item is provided for compatibility with the original\n               Protein Data Bank format, and only for that purpose.' ],
          [ 'item_', '_item.name', '_cell.Z_PDB' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# CELL_MEASUREMENT ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'cell_measurement',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CELL_MEASUREMENT category record details\n               about the measurement of the crystallographic cell parameters.' ],
          [ 'item_', '_category.id', 'cell_measurement' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_cell_measurement.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'cell_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _cell_measurement.entry_id             \'5HVP\'\n    _cell_measurement.temp                 293\n    _cell_measurement.temp_esd             3\n    _cell_measurement.theta_min            11\n    _cell_measurement.theta_max            31\n    _cell_measurement.wavelength           1.54',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    _cell_measurement.temp                 293\n    _cell_measurement.reflns_used          25\n    _cell_measurement.theta_min            25\n    _cell_measurement.theta_max            31',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_cell_measurement.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_cell_measurement.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_cell_measurement.pressure',
        [ [ 'item_',
            '_item_description.description',
            '              The pressure in kilopascals at which the unit-cell parameters\n               were measured (not the pressure at which the sample was\n               synthesized).' ],
          [ 'item_', '_item.name', '_cell_measurement.pressure' ],
          [ 'item_', '_item.category_id', 'cell_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_pressure' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell_measurement.pressure_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'kilopascals' ] ] ],
      [ 'save_',
        '_cell_measurement.pressure_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _cell_measurement.pressure.' ],
          [ 'item_', '_item.name', '_cell_measurement.pressure_esd' ],
          [ 'item_', '_item.category_id', 'cell_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell_measurement.pressure' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kilopascals' ] ] ],
      [ 'save_',
        '_cell_measurement.radiation',
        [ [ 'item_',
            '_item_description.description',
            '              Description of the radiation used to measure the unit-cell data.\n               See also _cell_measurement.wavelength.' ],
          [ 'item_', '_item.name', '_cell_measurement.radiation' ],
          [ 'item_', '_item.category_id', 'cell_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_radiation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'neutron', 'Cu K\\a', 'synchrotron' ] ] ] ],
      [ 'save_',
        '_cell_measurement.reflns_used',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of reflections used to determine the unit cell.\n               These reflections may be specified as CELL_MEASUREMENT_REFLN\n               data items.' ],
          [ 'item_', '_item.name', '_cell_measurement.reflns_used' ],
          [ 'item_', '_item.category_id', 'cell_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_reflns_used' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_cell_measurement.temp',
        [ [ 'item_',
            '_item_description.description',
            '              The temperature in kelvins at which the unit-cell parameters\n               were measured (not the temperature of synthesis).' ],
          [ 'item_', '_item.name', '_cell_measurement.temp' ],
          [ 'item_', '_item.category_id', 'cell_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_temperature' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_cell_measurement.temp_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ 'save_',
        '_cell_measurement.temp_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _cell_measurement.temp.' ],
          [ 'item_', '_item.name', '_cell_measurement.temp_esd' ],
          [ 'item_', '_item.category_id', 'cell_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell_measurement.temp' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ 'save_',
        '_cell_measurement.theta_max',
        [ [ 'item_',
            '_item_description.description',
            '              The maximum theta angle of reflections used to measure\n               the unit cell in degrees.' ],
          [ 'item_', '_item.name', '_cell_measurement.theta_max' ],
          [ 'item_', '_item.category_id', 'cell_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_theta_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '90.0', '90.0', '90.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_cell_measurement.theta_min',
        [ [ 'item_',
            '_item_description.description',
            '              The minimum theta angle of reflections used to measure\n               the unit cell in degrees.' ],
          [ 'item_', '_item.name', '_cell_measurement.theta_min' ],
          [ 'item_', '_item.category_id', 'cell_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_theta_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '90.0', '90.0', '90.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_cell_measurement.wavelength',
        [ [ 'item_',
            '_item_description.description',
            '              The wavelength in angstroms of the radiation used to measure\n               the unit cell. If this is not specified, the wavelength is\n               assumed to be that specified in the category\n               DIFFRN_RADIATION_WAVELENGTH.' ],
          [ 'item_', '_item.name', '_cell_measurement.wavelength' ],
          [ 'item_', '_item.category_id', 'cell_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_wavelength' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ '@comment', '###########################' ],
      [ '@comment', '# CELL_MEASUREMENT_REFLN ##' ],
      [ '@comment', '###########################' ],
      [ 'save_',
        'cell_measurement_refln',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CELL_MEASUREMENT_REFLN category record\n               details about the reflections used to determine the\n               crystallographic cell parameters.\n\n               The CELL_MEASUREMENT_REFLN data items would in general be used\n               only for diffractometer data.' ],
          [ 'item_', '_category.id', 'cell_measurement_refln' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_cell_measurement_refln.index_h',
              '_cell_measurement_refln.index_k',
              '_cell_measurement_refln.index_l' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'cell_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - extracted from the CAD-4 listing of Rb~2~S~2~O~6~ at room\n                temperature (unpublished).',
              '\n    loop_\n    _cell_measurement_refln.index_h\n    _cell_measurement_refln.index_k\n    _cell_measurement_refln.index_l\n    _cell_measurement_refln.theta\n       -2    4    1          8.67\n        0    3    2          9.45\n        3    0    2          9.46\n       -3    4    1          8.93\n       -2    1   -2          7.53\n       10    0    0         23.77\n        0   10    0         23.78\n       -5    4    1         11.14\n      # - - - - data truncated for brevity - - - -',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_cell_measurement_refln.index_h',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index h of a reflection used for measurement of the unit\n               cell.' ],
          [ 'item_', '_item.name', '_cell_measurement_refln.index_h' ],
          [ 'item_', '_item.category_id', 'cell_measurement_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_refln_index_h' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell_measurement_refln.index_k',
              '_cell_measurement_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_cell_measurement_refln.index_k',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index k of a reflection used for measurement of the unit\n               cell.' ],
          [ 'item_', '_item.name', '_cell_measurement_refln.index_k' ],
          [ 'item_', '_item.category_id', 'cell_measurement_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_refln_index_k' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell_measurement_refln.index_h',
              '_cell_measurement_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_cell_measurement_refln.index_l',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index l of a reflection used for measurement of the unit\n               cell.' ],
          [ 'item_', '_item.name', '_cell_measurement_refln.index_l' ],
          [ 'item_', '_item.category_id', 'cell_measurement_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_refln_index_l' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_cell_measurement_refln.index_h',
              '_cell_measurement_refln.index_k' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_cell_measurement_refln.theta',
        [ [ 'item_',
            '_item_description.description',
            '              Theta angle for a reflection used for measurement of\n               the unit cell in degrees.' ],
          [ 'item_', '_item.name', '_cell_measurement_refln.theta' ],
          [ 'item_', '_item.category_id', 'cell_measurement_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_cell_measurement_refln_theta' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '90.0', '90.0', '90.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ '@comment', '##############' ],
      [ '@comment', '# CHEM_COMP ##' ],
      [ '@comment', '##############' ],
      [ 'save_',
        'chem_comp',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP category give details about each\n               of the chemical components from which the relevant chemical\n               structures can be constructed, such as name, mass or charge.\n\n               The related categories CHEM_COMP_ATOM, CHEM_COMP_BOND,\n               CHEM_COMP_ANGLE etc. describe the detailed geometry of these\n               chemical components.' ],
          [ 'item_', '_category.id', 'chem_comp' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_chem_comp.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _chem_comp.id\n    _chem_comp.model_source\n    _chem_comp.name\n      phe  \'1987 Protin/Prolsq Ideals file\' phenylalanine\n      val  \'1987 Protin/Prolsq Ideals file\' alanine\n    # - - - - data truncated for brevity - - - -',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp.formula',
        [ [ 'item_',
            '_item_description.description',
            '              The formula for the chemical component. Formulae are written\n               according to the following rules:\n\n               (1) Only recognized element symbols may be used.\n\n               (2) Each element symbol is followed by a \'count\' number. A count\n                  of \'1\' may be omitted.\n\n               (3) A space or parenthesis must separate each cluster of\n                  (element symbol + count), but in general parentheses are\n                  not used.\n\n               (4) The order of elements depends on whether carbon is\n                  present or not. If carbon is present, the order should be:\n                  C, then H, then the other elements in alphabetical order\n                  of their symbol. If carbon is not present, the elements\n                  are listed purely in alphabetic order of their symbol. This\n                  is the \'Hill\' system used by Chemical Abstracts.' ],
          [ 'item_', '_item.name', '_chem_comp.formula' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'C18 H19 N7 O8 S' ] ] ] ],
      [ 'save_',
        '_chem_comp.formula_weight',
        [ [ 'item_',
            '_item_description.description',
            '              Formula mass in daltons of the chemical component.' ],
          [ 'item_', '_item.name', '_chem_comp.formula_weight' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1.0', '1.0', '1.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_chem_comp.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _chem_comp.id must uniquely identify each item in\n               the CHEM_COMP list.\n\n               For protein polymer entities, this is the three-letter code for\n               the amino acid.\n\n               For nucleic acid polymer entities, this is the one-letter code\n               for the base.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chem_comp.id',
              'chem_comp',
              'yes',
              '_atom_site.label_comp_id',
              'atom_site',
              'no',
              '_chem_comp.mon_nstd_parent_comp_id',
              'chem_comp',
              'no',
              '_chem_comp_atom.comp_id',
              'chem_comp_atom',
              'yes',
              '_chem_comp_angle.comp_id',
              'chem_comp_angle',
              'yes',
              '_chem_comp_bond.comp_id',
              'chem_comp_bond',
              'yes',
              '_chem_comp_chir.comp_id',
              'chem_comp_chir',
              'yes',
              '_chem_comp_chir_atom.comp_id',
              'chem_comp_chir_atom',
              'yes',
              '_chem_comp_plane.comp_id',
              'chem_comp_plane',
              'yes',
              '_chem_comp_plane_atom.comp_id',
              'chem_comp_plane_atom',
              'yes',
              '_chem_comp_tor.comp_id',
              'chem_comp_tor',
              'yes',
              '_chem_comp_tor_value.comp_id',
              'chem_comp_tor_value',
              'yes',
              '_entity_poly_seq.mon_id',
              'entity_poly_seq',
              'yes',
              '_geom_angle.atom_site_label_comp_id_1',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_label_comp_id_2',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_label_comp_id_3',
              'geom_angle',
              'no',
              '_geom_bond.atom_site_label_comp_id_1',
              'geom_bond',
              'no',
              '_geom_bond.atom_site_label_comp_id_2',
              'geom_bond',
              'no',
              '_geom_contact.atom_site_label_comp_id_1',
              'geom_contact',
              'no',
              '_geom_contact.atom_site_label_comp_id_2',
              'geom_contact',
              'no',
              '_geom_hbond.atom_site_label_comp_id_A',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_label_comp_id_D',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_label_comp_id_H',
              'geom_hbond',
              'no',
              '_geom_torsion.atom_site_label_comp_id_1',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_comp_id_2',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_comp_id_3',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_comp_id_4',
              'geom_torsion',
              'no',
              '_struct_conf.beg_label_comp_id',
              'struct_conf',
              'yes',
              '_struct_conf.end_label_comp_id',
              'struct_conf',
              'yes',
              '_struct_conn.ptnr1_label_comp_id',
              'struct_conn',
              'yes',
              '_struct_conn.ptnr2_label_comp_id',
              'struct_conn',
              'yes',
              '_struct_mon_nucl.label_comp_id',
              'struct_mon_nucl',
              'yes',
              '_struct_mon_prot.label_comp_id',
              'struct_mon_prot',
              'yes',
              '_struct_mon_prot_cis.label_comp_id',
              'struct_mon_prot_cis',
              'yes',
              '_struct_ncs_dom_lim.beg_label_comp_id',
              'struct_ncs_dom_lim',
              'yes',
              '_struct_ncs_dom_lim.end_label_comp_id',
              'struct_ncs_dom_lim',
              'yes',
              '_struct_ref_seq_dif.db_mon_id',
              'struct_ref_seq_dif',
              'yes',
              '_struct_ref_seq_dif.mon_id',
              'struct_ref_seq_dif',
              'yes',
              '_struct_sheet_range.beg_label_comp_id',
              'struct_sheet_range',
              'yes',
              '_struct_sheet_range.end_label_comp_id',
              'struct_sheet_range',
              'yes',
              '_struct_site_gen.label_comp_id',
              'struct_site_gen',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_site.label_comp_id',
              '_chem_comp.id',
              '_chem_comp.mon_nstd_parent_comp_id',
              '_chem_comp.id',
              '_chem_comp_atom.comp_id',
              '_chem_comp.id',
              '_chem_comp_chir.comp_id',
              '_chem_comp.id',
              '_chem_comp_chir_atom.comp_id',
              '_chem_comp.id',
              '_chem_comp_plane.comp_id',
              '_chem_comp.id',
              '_chem_comp_plane_atom.comp_id',
              '_chem_comp.id',
              '_entity_poly_seq.mon_id',
              '_chem_comp.id',
              '_chem_comp_angle.comp_id',
              '_chem_comp_atom.comp_id',
              '_chem_comp_bond.comp_id',
              '_chem_comp_atom.comp_id',
              '_chem_comp_tor.comp_id',
              '_chem_comp_atom.comp_id',
              '_chem_comp_tor_value.comp_id',
              '_chem_comp_atom.comp_id',
              '_geom_angle.atom_site_label_comp_id_1',
              '_atom_site.label_comp_id',
              '_geom_angle.atom_site_label_comp_id_2',
              '_atom_site.label_comp_id',
              '_geom_angle.atom_site_label_comp_id_3',
              '_atom_site.label_comp_id',
              '_geom_bond.atom_site_label_comp_id_1',
              '_atom_site.label_comp_id',
              '_geom_bond.atom_site_label_comp_id_2',
              '_atom_site.label_comp_id',
              '_geom_contact.atom_site_label_comp_id_1',
              '_atom_site.label_comp_id',
              '_geom_contact.atom_site_label_comp_id_2',
              '_atom_site.label_comp_id',
              '_geom_hbond.atom_site_label_comp_id_A',
              '_atom_site.label_comp_id',
              '_geom_hbond.atom_site_label_comp_id_D',
              '_atom_site.label_comp_id',
              '_geom_hbond.atom_site_label_comp_id_H',
              '_atom_site.label_comp_id',
              '_geom_torsion.atom_site_label_comp_id_1',
              '_atom_site.label_comp_id',
              '_geom_torsion.atom_site_label_comp_id_2',
              '_atom_site.label_comp_id',
              '_geom_torsion.atom_site_label_comp_id_3',
              '_atom_site.label_comp_id',
              '_geom_torsion.atom_site_label_comp_id_4',
              '_atom_site.label_comp_id',
              '_struct_conf.beg_label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_conf.end_label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_conn.ptnr1_label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_conn.ptnr2_label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_mon_nucl.label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_mon_prot.label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_mon_prot_cis.label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_ncs_dom_lim.beg_label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_ncs_dom_lim.end_label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_ref_seq_dif.db_mon_id',
              '_chem_comp.id',
              '_struct_ref_seq_dif.mon_id',
              '_chem_comp.id',
              '_struct_sheet_range.beg_label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_sheet_range.end_label_comp_id',
              '_atom_site.label_comp_id',
              '_struct_site_gen.label_comp_id',
              '_atom_site.label_comp_id' ] ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'ala', 'val', 'A', 'C' ] ] ] ],
      [ 'save_',
        '_chem_comp.model_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the generation of the\n               coordinates for the model of the component.' ],
          [ 'item_', '_item.name', '_chem_comp.model_details' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'geometry idealized but not minimized' ] ] ],
      [ 'save_',
        '_chem_comp.model_erf',
        [ [ 'item_',
            '_item_description.description',
            '              A pointer to an external reference file from which the atomic\n               description of the component is taken.' ],
          [ 'item_', '_item.name', '_chem_comp.model_erf' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_chem_comp.model_source',
        [ [ 'item_',
            '_item_description.description',
            '              The source of the coordinates for the model of the component.' ],
          [ 'item_', '_item.name', '_chem_comp.model_source' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'CSD entry ABCDEF', 'built using Quanta/Charmm' ] ] ] ],
      [ 'save_',
        '_chem_comp.mon_nstd_class',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the class of a nonstandard monomer if the\n               nonstandard monomer represents a modification of a\n               standard monomer.' ],
          [ 'item_', '_item.name', '_chem_comp.mon_nstd_class' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'iodinated base',
              'phosphorylated amino acid',
              'brominated base',
              'modified amino acid',
              'glycosylated amino acid' ] ] ] ],
      [ 'save_',
        '_chem_comp.mon_nstd_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special details of a nonstandard monomer.' ],
          [ 'item_', '_item.name', '_chem_comp.mon_nstd_details' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_chem_comp.mon_nstd_flag',
        [ [ 'item_',
            '_item_description.description',
            '              \'yes\' indicates that this is a \'standard\' monomer, \'no\'\n               indicates that it is \'nonstandard\'. Nonstandard monomers\n               should be described in more detail using the\n               _chem_comp.mon_nstd_parent, _chem_comp.mon_nstd_class and\n               _chem_comp.mon_nstd_details data items.' ],
          [ 'item_', '_item.name', '_chem_comp.mon_nstd_flag' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_', '_item_default.value', 'no' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'the monomer is nonstandard',
              'n',
              'abbreviation for "no"',
              'yes',
              'the monomer is standard',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_chem_comp.mon_nstd_parent',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the parent monomer of the nonstandard monomer,\n               if the nonstandard monomer represents a modification of a\n               standard monomer.' ],
          [ 'item_', '_item.name', '_chem_comp.mon_nstd_parent' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'tyrosine', 'cytosine' ] ] ] ],
      [ 'save_',
        '_chem_comp.mon_nstd_parent_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier for the parent component of the nonstandard\n               component.\n\n               This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_chem_comp.mon_nstd_parent_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_chem_comp.name',
        [ [ 'item_',
            '_item_description.description',
            '              The full name of the component.' ],
          [ 'item_', '_item.name', '_chem_comp.name' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'alanine', 'valine', 'adenine', 'cytosine' ] ] ] ],
      [ 'save_',
        '_chem_comp.number_atoms_all',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of atoms in the component.' ],
          [ 'item_', '_item.name', '_chem_comp.number_atoms_all' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chem_comp.number_atoms_nh',
        [ [ 'item_',
            '_item_description.description',
            '              The number of non-hydrogen atoms in the component.' ],
          [ 'item_', '_item.name', '_chem_comp.number_atoms_nh' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chem_comp.one_letter_code',
        [ [ 'item_',
            '_item_description.description',
            '              For standard polymer components, the one-letter code for\n               the component. If there is not a standard one-letter code\n               for this component, or if this is a non-polymer\n               component, the one-letter code should be given as \'X\'.\n               This code may be preceded by a \'+\' character to indicate\n               that the component is a modification of a standard\n               component.' ],
          [ 'item_', '_item.name', '_chem_comp.one_letter_code' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar1' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ 'A',
              'alanine or adenine',
              'B',
              'ambiguous asparagine/aspartic acid',
              'R',
              'arginine',
              'N',
              'asparagine',
              'D',
              'aspartic acid',
              'C',
              'cysteine or cystine or cytosine',
              'Q',
              'glutamine',
              'E',
              'glutamic acid',
              'Z',
              'ambiguous glutamine/glutamic acid',
              'G',
              'glycine or guanine',
              'H',
              'histidine',
              'I',
              'isoleucine',
              'L',
              'leucine',
              'K',
              'lysine',
              'M',
              'methionine',
              'F',
              'phenylalanine',
              'P',
              'proline',
              'S',
              'serine',
              'T',
              'threonine or thymine',
              'W',
              'tryptophan',
              'Y',
              'tyrosine',
              'V',
              'valine',
              'U',
              'uracil',
              'O',
              'water',
              'X',
              'other' ] ] ] ],
      [ 'save_',
        '_chem_comp.three_letter_code',
        [ [ 'item_',
            '_item_description.description',
            '              For standard polymer components, the three-letter code for\n               the component. If there is not a standard three-letter code\n               for this component, or if this is a non-polymer\n               component, the three-letter code should be given as \'UNK\'.\n               This code may be preceded by a \'+\' character to indicate\n               that the component is a modification of a standard\n               component.' ],
          [ 'item_', '_item.name', '_chem_comp.three_letter_code' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar3' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ 'ALA',
              'alanine',
              'ARG',
              'arginine',
              'ASN',
              'asparagine',
              'ASP',
              'aspartic acid',
              'ASX',
              'ambiguous asparagine/aspartic acid',
              'CYS',
              'cysteine',
              'GLN',
              'glutamine',
              'GLU',
              'glutamic acid',
              'GLY',
              'glycine',
              'GLX',
              'ambiguous glutamine/glutamic acid',
              'HIS',
              'histidine',
              'ILE',
              'isoleucine',
              'LEU',
              'leucine',
              'LYS',
              'lysine',
              'MET',
              'methionine',
              'PHE',
              'phenylalanine',
              'PRO',
              'proline',
              'SER',
              'serine',
              'THR',
              'threonine',
              'TRP',
              'tryptophan',
              'TRY',
              'tyrosine',
              'VAL',
              'valine',
              '1MA',
              '1-methyladenosine',
              '5MC',
              '5-methylcytosine',
              'OMC',
              '2(prime)-O-methylcytodine',
              '1MG',
              '1-methylguanosine',
              '2MG',
              'N(2)-methylguanosine',
              'M2G',
              'N(2)-dimethylguanosine',
              '7MG',
              '7-methylguanosine',
              '0MG',
              '2(prime)-O-methylguanosine',
              'H2U',
              'dihydrouridine',
              '5MU',
              'ribosylthymidine',
              'PSU',
              'pseudouridine',
              'ACE',
              'acetic acid',
              'FOR',
              'formic acid',
              'HOH',
              'water',
              'UNK',
              'other' ] ] ] ],
      [ 'save_',
        '_chem_comp.type',
        [ [ 'item_',
            '_item_description.description',
            '              For standard polymer components, the type of the monomer.\n               Note that monomers that will form polymers are of three types:\n               linking monomers, monomers with some type of N-terminal (or 5\')\n               cap and monomers with some type of C-terminal (or 3\') cap.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chem_comp.type',
              'chem_comp',
              'yes',
              '_chem_comp_link.type_comp_1',
              'chem_comp_link',
              'yes',
              '_chem_comp_link.type_comp_2',
              'chem_comp_link',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_chem_comp_link.type_comp_1',
              '_chem_comp.type',
              '_chem_comp_link.type_comp_2',
              '_chem_comp.type' ] ],
          [ 'item_', '_item_type.code', 'uline' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'D-peptide linking',
              'L-peptide linking',
              'D-peptide NH3 amino terminus',
              'L-peptide NH3 amino terminus',
              'D-peptide COOH carboxy terminus',
              'L-peptide COOH carboxy terminus',
              'DNA linking',
              'RNA linking',
              'DNA OH 5 prime terminus',
              'RNA OH 5 prime terminus',
              'DNA OH 3 prime terminus',
              'RNA OH 3 prime terminus',
              'D-saccharide 1,4 and 1,4 linking',
              'L-saccharide 1,4 and 1,4 linking',
              'D-saccharide 1,4 and 1,6 linking',
              'L-saccharide 1,4 and 1,6 linking',
              'L-saccharide',
              'D-saccharide',
              'saccharide',
              'non-polymer',
              'other' ] ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# CHEM_COMP_ANGLE ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'chem_comp_angle',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_ANGLE category record details about\n               angles in a chemical component. Angles are designated by three\n               atoms, with the second atom forming the vertex of the angle.\n               Target values may be specified as angles in degrees, as a\n               distance between the first and third atoms, or both.' ],
          [ 'item_', '_category.id', 'chem_comp_angle' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_comp_angle.comp_id',
              '_chem_comp_angle.atom_id_1',
              '_chem_comp_angle.atom_id_2',
              '_chem_comp_angle.atom_id_3' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _chem_comp_angle.comp_id\n    _chem_comp_angle.atom_id_1\n    _chem_comp_angle.atom_id_2\n    _chem_comp_angle.atom_id_3\n    _chem_comp_angle.value_angle\n    _chem_comp_angle.value_dist\n      phe  N    CA   C    xxx.xx   x.xx\n      phe  CA   C    O    xxx.xx   x.xx\n      phe  CB   CA   C    xxx.xx   x.xx\n      phe  CB   CA   N    xxx.xx   x.xx\n      phe  CA   CB   CG   xxx.xx   x.xx\n      phe  CB   CG   CD1  xxx.xx   x.xx\n      phe  CB   CG   CD2  xxx.xx   x.xx\n      phe  CD1  CG   CD2  xxx.xx   x.xx\n      phe  CG   CD1  CE1  xxx.xx   x.xx\n      phe  CD1  CE1  CZ   xxx.xx   x.xx\n      phe  CE1  CZ   CE2  xxx.xx   x.xx\n      phe  CZ   CE2  CD2  xxx.xx   x.xx\n      phe  CG   CD2  CE2  xxx.xx   x.xx\n      val  N    CA   C    xxx.xx   x.xx\n      val  CA   C    O    xxx.xx   x.xx\n      val  CB   CA   C    xxx.xx   x.xx\n      val  CB   CA   N    xxx.xx   x.xx\n      val  CA   CB   CG1  xxx.xx   x.xx\n      val  CA   CB   CG2  xxx.xx   x.xx\n      val  CG1  CB   CG2  xxx.xx   x.xx',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp_angle.atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the first of the three atoms that define the angle.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_angle.atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_angle.atom_id_2', '_chem_comp_angle.atom_id_3' ] ] ] ],
      [ 'save_',
        '_chem_comp_angle.atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the second of the three atoms that define the angle.\n               The second atom is taken to be the apex of the angle.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_angle.atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_angle.atom_id_1', '_chem_comp_angle.atom_id_3' ] ] ] ],
      [ 'save_',
        '_chem_comp_angle.atom_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the third of the three atoms that define the angle.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_angle.atom_id_3' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_angle.atom_id_1', '_chem_comp_angle.atom_id_2' ] ] ] ],
      [ 'save_',
        '_chem_comp_angle.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_chem_comp_angle.comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_angle.value_angle',
        [ [ 'item_',
            '_item_description.description',
            '              The value that should be taken as the target value for the angle\n               associated with the specified atoms, expressed in degrees.' ],
          [ 'item_', '_item.name', '_chem_comp_angle.value_angle' ],
          [ 'item_', '_item.category_id', 'chem_comp_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_angle.value_angle_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_chem_comp_angle.value_angle_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_comp_angle.value_angle.' ],
          [ 'item_', '_item.name', '_chem_comp_angle.value_angle_esd' ],
          [ 'item_', '_item.category_id', 'chem_comp_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_angle.value_angle' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_chem_comp_angle.value_dist',
        [ [ 'item_',
            '_item_description.description',
            '              The value that should be taken as the target value for the angle\n               associated with the specified atoms, expressed as the distance\n               between the atoms specified by _chem_comp_angle.atom_id_1 and\n               _chem_comp_angle.atom_id_3.' ],
          [ 'item_', '_item.name', '_chem_comp_angle.value_dist' ],
          [ 'item_', '_item.category_id', 'chem_comp_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_angle.value_dist_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_angle.value_dist_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_comp_angle.value_dist.' ],
          [ 'item_', '_item.name', '_chem_comp_angle.value_dist_esd' ],
          [ 'item_', '_item.category_id', 'chem_comp_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_angle.value_dist' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# CHEM_COMP_ATOM ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'chem_comp_atom',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_ATOM category record details about\n               the atoms in a chemical component. Specifying the atomic\n               coordinates for the components in this category is an\n               alternative to specifying the structure of the component\n               via bonds, angles, planes etc. in the appropriate\n               CHEM_COMP subcategories.' ],
          [ 'item_', '_category.id', 'chem_comp_atom' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_comp_atom.comp_id', '_chem_comp_atom.atom_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _chem_comp_atom.comp_id\n    _chem_comp_atom.atom_id\n    _chem_comp_atom.type_symbol\n    _chem_comp_atom.substruct_code\n    _chem_comp_atom.model_Cartn_x\n    _chem_comp_atom.model_Cartn_y\n    _chem_comp_atom.model_Cartn_z\n      phe  N    N  main   1.20134   0.84658   0.00000\n      phe  CA   C  main   0.00000   0.00000   0.00000\n      phe  C    C  main  -1.25029   0.88107   0.00000\n      phe  O    O  main  -2.18525   0.66029  -0.78409\n      phe  CB   C  side   0.00662  -1.03603   1.11081\n      phe  CG   C  side   0.03254  -0.49711   2.50951\n      phe  CD1  C  side  -1.15813  -0.12084   3.13467\n      phe  CE1  C  side  -1.15720   0.38038   4.42732\n      phe  CZ   C  side   0.05385   0.51332   5.11032\n      phe  CE2  C  side   1.26137   0.11613   4.50975\n      phe  CD2  C  side   1.23668  -0.38351   3.20288\n      val  N    N  main   1.20134   0.84658   0.00000\n      val  CA   C  main   0.00000   0.00000   0.00000\n      val  C    C  main  -1.25029   0.88107   0.00000\n      val  O    O  main  -2.18525   0.66029  -0.78409\n      val  CB   C  side   0.05260  -0.99339   1.17429\n      val  CG1  C  side  -0.13288  -0.31545   2.52668\n      val  CG2  C  side  -0.94265  -2.12930   0.99811',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp_atom.alt_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              An alternative identifier for the atom. This data item would be\n               used in cases where alternative nomenclatures exist for labelling\n               atoms in a group.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.alt_atom_id' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_chem_comp_atom.atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _chem_comp_atom.atom_id must uniquely identify\n               each atom in each monomer in the CHEM_COMP_ATOM list.\n\n               The atom identifiers need not be unique over all atoms in the\n               data block; they need only be unique for each atom in a\n               component.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chem_comp_atom.atom_id',
              'chem_comp_atom',
              'yes',
              '_atom_site.label_atom_id',
              'atom_site',
              'no',
              '_chem_comp_angle.atom_id_1',
              'chem_comp_angle',
              'yes',
              '_chem_comp_angle.atom_id_2',
              'chem_comp_angle',
              'yes',
              '_chem_comp_angle.atom_id_3',
              'chem_comp_angle',
              'yes',
              '_chem_comp_bond.atom_id_1',
              'chem_comp_bond',
              'yes',
              '_chem_comp_bond.atom_id_2',
              'chem_comp_bond',
              'yes',
              '_chem_comp_chir.atom_id',
              'chem_comp_chir',
              'yes',
              '_chem_comp_chir_atom.atom_id',
              'chem_comp_chir_atom',
              'yes',
              '_chem_comp_plane_atom.atom_id',
              'chem_comp_plane_atom',
              'yes',
              '_chem_comp_tor.atom_id_1',
              'chem_comp_tor',
              'yes',
              '_chem_comp_tor.atom_id_2',
              'chem_comp_tor',
              'yes',
              '_chem_comp_tor.atom_id_3',
              'chem_comp_tor',
              'yes',
              '_chem_comp_tor.atom_id_4',
              'chem_comp_tor',
              'yes',
              '_geom_angle.atom_site_label_atom_id_1',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_label_atom_id_2',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_label_atom_id_3',
              'geom_angle',
              'no',
              '_geom_bond.atom_site_label_atom_id_1',
              'geom_bond',
              'no',
              '_geom_bond.atom_site_label_atom_id_2',
              'geom_bond',
              'no',
              '_geom_contact.atom_site_label_atom_id_1',
              'geom_contact',
              'no',
              '_geom_contact.atom_site_label_atom_id_2',
              'geom_contact',
              'no',
              '_geom_hbond.atom_site_label_atom_id_A',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_label_atom_id_D',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_label_atom_id_H',
              'geom_hbond',
              'no',
              '_geom_torsion.atom_site_label_atom_id_1',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_atom_id_2',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_atom_id_3',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_atom_id_4',
              'geom_torsion',
              'no',
              '_struct_conn.ptnr1_label_atom_id',
              'struct_conn',
              'yes',
              '_struct_conn.ptnr2_label_atom_id',
              'struct_conn',
              'yes',
              '_struct_sheet_hbond.range_1_beg_label_atom_id',
              'struct_sheet_hbond',
              'yes',
              '_struct_sheet_hbond.range_1_end_label_atom_id',
              'struct_sheet_hbond',
              'yes',
              '_struct_sheet_hbond.range_2_beg_label_atom_id',
              'struct_sheet_hbond',
              'yes',
              '_struct_sheet_hbond.range_2_end_label_atom_id',
              'struct_sheet_hbond',
              'yes',
              '_struct_site_gen.label_atom_id',
              'struct_site_gen',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_site.label_atom_id',
              '_chem_comp_atom.atom_id',
              '_chem_comp_angle.atom_id_1',
              '_chem_comp_atom.atom_id',
              '_chem_comp_angle.atom_id_2',
              '_chem_comp_atom.atom_id',
              '_chem_comp_angle.atom_id_3',
              '_chem_comp_atom.atom_id',
              '_chem_comp_bond.atom_id_1',
              '_chem_comp_atom.atom_id',
              '_chem_comp_bond.atom_id_2',
              '_chem_comp_atom.atom_id',
              '_chem_comp_chir.atom_id',
              '_chem_comp_atom.atom_id',
              '_chem_comp_chir_atom.atom_id',
              '_chem_comp_atom.atom_id',
              '_chem_comp_plane_atom.atom_id',
              '_chem_comp_atom.atom_id',
              '_chem_comp_tor.atom_id_1',
              '_chem_comp_atom.atom_id',
              '_chem_comp_tor.atom_id_2',
              '_chem_comp_atom.atom_id',
              '_chem_comp_tor.atom_id_3',
              '_chem_comp_atom.atom_id',
              '_chem_comp_tor.atom_id_4',
              '_chem_comp_atom.atom_id',
              '_geom_angle.atom_site_label_atom_id_1',
              '_atom_site.label_atom_id',
              '_geom_angle.atom_site_label_atom_id_2',
              '_atom_site.label_atom_id',
              '_geom_angle.atom_site_label_atom_id_3',
              '_atom_site.label_atom_id',
              '_geom_bond.atom_site_label_atom_id_1',
              '_atom_site.label_atom_id',
              '_geom_bond.atom_site_label_atom_id_2',
              '_atom_site.label_atom_id',
              '_geom_contact.atom_site_label_atom_id_1',
              '_atom_site.label_atom_id',
              '_geom_contact.atom_site_label_atom_id_2',
              '_atom_site.label_atom_id',
              '_geom_hbond.atom_site_label_atom_id_A',
              '_atom_site.label_atom_id',
              '_geom_hbond.atom_site_label_atom_id_D',
              '_atom_site.label_atom_id',
              '_geom_hbond.atom_site_label_atom_id_H',
              '_atom_site.label_atom_id',
              '_geom_torsion.atom_site_label_atom_id_1',
              '_atom_site.label_atom_id',
              '_geom_torsion.atom_site_label_atom_id_2',
              '_atom_site.label_atom_id',
              '_geom_torsion.atom_site_label_atom_id_3',
              '_atom_site.label_atom_id',
              '_geom_torsion.atom_site_label_atom_id_4',
              '_atom_site.label_atom_id',
              '_struct_conn.ptnr1_label_atom_id',
              '_atom_site.label_atom_id',
              '_struct_conn.ptnr2_label_atom_id',
              '_atom_site.label_atom_id',
              '_struct_sheet_hbond.range_1_beg_label_atom_id',
              '_atom_site.label_atom_id',
              '_struct_sheet_hbond.range_1_end_label_atom_id',
              '_atom_site.label_atom_id',
              '_struct_sheet_hbond.range_2_beg_label_atom_id',
              '_atom_site.label_atom_id',
              '_struct_sheet_hbond.range_2_end_label_atom_id',
              '_atom_site.label_atom_id',
              '_struct_site_gen.label_atom_id',
              '_atom_site.label_atom_id' ] ],
          [ 'item_', '_item_type.code', 'atcode' ] ] ],
      [ 'save_',
        '_chem_comp_atom.charge',
        [ [ 'item_',
            '_item_description.description',
            '              The net integer charge assigned to this atom. This is the\n               formal charge assignment normally found in chemical diagrams.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.charge' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '8', '8', '8', '-8', '-8', '-8' ] ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '1', 'for an ammonium nitrogen', '-1', 'for a chloride ion' ] ] ] ],
      [ 'save_',
        '_chem_comp_atom.model_Cartn_x',
        [ [ 'item_',
            '_item_description.description',
            '              The x component of the coordinates for this atom in this\n               component specified as orthogonal angstroms. The choice of\n               reference axis frame for the coordinates is arbitrary.\n\n               The set of coordinates input for the entity here is intended to\n               correspond to the atomic model used to generate restraints for\n               structure refinement, not to atom sites in the ATOM_SITE\n               list.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.model_Cartn_x' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_atom.model_Cartn_y',
              '_chem_comp_atom.model_Cartn_z' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_atom.model_Cartn_x_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_atom.model_Cartn_x_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_comp_atom.model_Cartn_x.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.model_Cartn_x_esd' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_atom.model_Cartn_y_esd',
              '_chem_comp_atom.model_Cartn_z_esd' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_atom.model_Cartn_x' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_atom.model_Cartn_y',
        [ [ 'item_',
            '_item_description.description',
            '              The y component of the coordinates for this atom in this\n               component specified as orthogonal angstroms. The choice of\n               reference axis frame for the coordinates is arbitrary.\n\n               The set of coordinates input for the entity here is intended to\n               correspond to the atomic model used to generate restraints for\n               structure refinement, not to atom sites in the ATOM_SITE\n               list.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.model_Cartn_y' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_atom.model_Cartn_x',
              '_chem_comp_atom.model_Cartn_z' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_atom.model_Cartn_y_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_atom.model_Cartn_y_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_comp_atom.model_Cartn_y.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.model_Cartn_y_esd' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_atom.model_Cartn_x_esd',
              '_chem_comp_atom.model_Cartn_z_esd' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_atom.model_Cartn_y' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_atom.model_Cartn_z',
        [ [ 'item_',
            '_item_description.description',
            '              The z component of the coordinates for this atom in this\n               component specified as orthogonal angstroms. The choice of\n               reference axis frame for the coordinates is arbitrary.\n\n               The set of coordinates input for the entity here is intended to\n               correspond to the atomic model used to generate restraints for\n               structure refinement, not to atom sites in the ATOM_SITE\n               list.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.model_Cartn_z' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_atom.model_Cartn_x',
              '_chem_comp_atom.model_Cartn_y' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_atom.model_Cartn_z_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_atom.model_Cartn_z_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_comp_atom.model_Cartn_z.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.model_Cartn_z_esd' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_atom.model_Cartn_x_esd',
              '_chem_comp_atom.model_Cartn_y_esd' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_atom.model_Cartn_z' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_atom.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_atom.partial_charge',
        [ [ 'item_',
            '_item_description.description',
            '              The partial charge assigned to this atom.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.partial_charge' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_chem_comp_atom.substruct_code',
        [ [ 'item_',
            '_item_description.description',
            '              This data item assigns the atom to a substructure of the\n               component, if appropriate.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.substruct_code' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'main',
              'main chain of an amino acid',
              'side',
              'side chain of an amino acid',
              'base',
              'base of a nucleic acid',
              'phos',
              'phosphate of a nucleic acid',
              'sugar',
              'sugar of a nucleic acid',
              'none',
              'not appropriate for this monomer' ] ] ] ],
      [ '@comment', 'save_chem_comp_atom.type_energy' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              This data item is a pointer to _atom_type_energy.type in the' ],
      [ '@comment', '               ATOM_TYPE_ENERGY category.' ],
      [ '@comment', ';' ],
      [ '@comment', '     save_' ],
      [ 'save_',
        '_chem_comp_atom.type_symbol',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _atom_type.symbol in the\n               ATOM_TYPE category.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.type_symbol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# CHEM_COMP_BOND ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'chem_comp_bond',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_BOND category record details about\n               the bonds between atoms in a chemical component. Target values\n               may be specified as bond orders, as a distance between the two\n               atoms, or both.' ],
          [ 'item_', '_category.id', 'chem_comp_bond' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_comp_bond.comp_id',
              '_chem_comp_bond.atom_id_1',
              '_chem_comp_bond.atom_id_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _chem_comp_bond.comp_id\n    _chem_comp_bond.atom_id_1\n    _chem_comp_bond.atom_id_2\n    _chem_comp_bond.value_order\n      phe  N    CA   sing\n      phe  CA   C    sing\n      phe  C    O    doub\n      phe  CB   CA   sing\n      phe  CB   CG   sing\n      phe  CG   CD1  arom\n      phe  CD1  CE1  arom\n      phe  CE1  CZ   arom\n      phe  CZ   CE2  arom\n      phe  CE2  CD2  arom\n      phe  CD2  CG   arom\n      val  N    CA   sing\n      val  CA   C    sing\n      val  C    O    doub\n      val  CB   CA   sing\n      val  CB   CG1  sing\n      val  CB   CG2  sing',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp_bond.atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the first of the two atoms that define the bond.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_bond.atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_chem_comp_bond.atom_id_2' ] ] ],
      [ 'save_',
        '_chem_comp_bond.atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the second of the two atoms that define the bond.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_bond.atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_chem_comp_bond.atom_id_1' ] ] ],
      [ 'save_',
        '_chem_comp_bond.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_chem_comp_bond.comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_bond.value_order',
        [ [ 'item_',
            '_item_description.description',
            '              The value that should be taken as the target for the chemical\n               bond associated with the specified atoms, expressed as a bond\n               order.' ],
          [ 'item_', '_item.name', '_chem_comp_bond.value_order' ],
          [ 'item_', '_item.category_id', 'chem_comp_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', 'sing' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'sing',
              'single bond',
              'doub',
              'double bond',
              'trip',
              'triple bond',
              'quad',
              'quadruple bond',
              'arom',
              'aromatic bond',
              'poly',
              'polymeric bond',
              'delo',
              'delocalized double bond',
              'pi',
              'pi bond' ] ] ] ],
      [ 'save_',
        '_chem_comp_bond.value_dist',
        [ [ 'item_',
            '_item_description.description',
            '              The value that should be taken as the target for the chemical\n               bond associated with the specified atoms, expressed as a\n               distance.' ],
          [ 'item_', '_item.name', '_chem_comp_bond.value_dist' ],
          [ 'item_', '_item.category_id', 'chem_comp_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_bond.value_dist_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_bond.value_dist_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_comp_bond.value_dist.' ],
          [ 'item_', '_item.name', '_chem_comp_bond.value_dist_esd' ],
          [ 'item_', '_item.category_id', 'chem_comp_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_bond.value_dist' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# CHEM_COMP_CHIR ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'chem_comp_chir',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_CHIR category provide details about\n               the chiral centres in a chemical component. The atoms bonded\n               to the chiral atom are specified in the CHEM_COMP_CHIR_ATOM\n               category.' ],
          [ 'item_', '_category.id', 'chem_comp_chir' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_comp_chir.comp_id', '_chem_comp_chir.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _chem_comp_chir.comp_id\n    _chem_comp_chir.id\n    _chem_comp_chir.atom_id\n      phe  phe1  CA\n      val  val1  CA\n    # - - - - data truncated for brevity - - - -',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp_chir.atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the atom that is a chiral centre.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_chir.atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_chir.atom_config',
        [ [ 'item_',
            '_item_description.description',
            '              The chiral configuration of the atom that is a chiral centre.' ],
          [ 'item_', '_item.name', '_chem_comp_chir.atom_config' ],
          [ 'item_', '_item.category_id', 'chem_comp_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'R',
              'absolute configuration R',
              'S',
              'absolute configuration S' ] ] ] ],
      [ 'save_',
        '_chem_comp_chir.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _chem_comp_chir.id must uniquely identify a record\n               in the CHEM_COMP_CHIR list.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chem_comp_chir.id',
              'chem_comp_chir',
              'yes',
              '_chem_comp_chir_atom.chir_id',
              'chem_comp_chir_atom',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_chem_comp_chir_atom.chir_id', '_chem_comp_chir.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_comp_chir.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_chem_comp_chir.comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_chir.number_atoms_all',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of atoms bonded to the atom specified by\n               _chem_comp_chir.atom_id.' ],
          [ 'item_', '_item.name', '_chem_comp_chir.number_atoms_all' ],
          [ 'item_', '_item.category_id', 'chem_comp_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chem_comp_chir.number_atoms_nh',
        [ [ 'item_',
            '_item_description.description',
            '              The number of non-hydrogen atoms bonded to the atom specified by\n               _chem_comp_chir.atom_id.' ],
          [ 'item_', '_item.name', '_chem_comp_chir.number_atoms_nh' ],
          [ 'item_', '_item.category_id', 'chem_comp_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chem_comp_chir.volume_flag',
        [ [ 'item_',
            '_item_description.description',
            '              A flag to indicate whether a chiral volume should match the\n               standard value in both magnitude and sign, or in magnitude only.' ],
          [ 'item_', '_item.name', '_chem_comp_chir.volume_flag' ],
          [ 'item_', '_item.category_id', 'chem_comp_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'sign',
              'match magnitude and sign',
              'nosign',
              'match magnitude only' ] ] ] ],
      [ 'save_',
        '_chem_comp_chir.volume_three',
        [ [ 'item_',
            '_item_description.description',
            '              The chiral volume, V~c~, for chiral centres that involve a chiral\n               atom bonded to three non-hydrogen atoms and one hydrogen atom.\n\n               V~c~ = V1 * (V2 X V3)\n\n               V1 = the vector distance from the atom specified by\n                    _chem_comp_chir.atom_id to the first atom in the\n                    CHEM_COMP_CHIR_ATOM list\n               V2 = the vector distance from the atom specified by\n                    _chem_comp_chir.atom_id to the second atom in the\n                    CHEM_COMP_CHIR_ATOM list\n               V3 = the vector distance from the atom specified by\n                    _chem_comp_chir.atom_id to the third atom in the\n                    CHEM_COMP_CHIR_ATOM list\n               *  = the vector dot product\n               X  = the vector cross product' ],
          [ 'item_', '_item.name', '_chem_comp_chir.volume_three' ],
          [ 'item_', '_item.category_id', 'chem_comp_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_chir.volume_three_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_cubed' ] ] ],
      [ 'save_',
        '_chem_comp_chir.volume_three_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_comp_chir.volume_three.' ],
          [ 'item_', '_item.name', '_chem_comp_chir.volume_three_esd' ],
          [ 'item_', '_item.category_id', 'chem_comp_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_chir.volume_three' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_cubed' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# CHEM_COMP_CHIR_ATOM ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'chem_comp_chir_atom',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_CHIR_ATOM category enumerate the\n               atoms bonded to a chiral atom within a chemical component.' ],
          [ 'item_', '_category.id', 'chem_comp_chir_atom' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_comp_chir_atom.chir_id',
              '_chem_comp_chir_atom.atom_id',
              '_chem_comp_chir_atom.comp_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _chem_comp_chir_atom.comp_id\n    _chem_comp_chir_atom.chir_id\n    _chem_comp_chir_atom.atom_id\n      phe  1  N\n      phe  1  C\n      phe  1  CB\n      val  1  N\n      val  1  C\n      val  1  CB\n   # - - - - data truncated for brevity - - - -',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp_chir_atom.atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of an atom bonded to the chiral atom.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_chir_atom.atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_chir_atom.chir_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp_chir.id in the\n               CHEM_COMP_CHIR category.' ],
          [ 'item_', '_item.name', '_chem_comp_chir_atom.chir_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_chir_atom.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the\n               CHEM_COMP category.' ],
          [ 'item_', '_item.name', '_chem_comp_chir_atom.comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_chir_atom.dev',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of the position of this atom from the plane defined by\n               all of the atoms in the plane.' ],
          [ 'item_', '_item.name', '_chem_comp_chir_atom.dev' ],
          [ 'item_', '_item.category_id', 'chem_comp_chir_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# CHEM_COMP_LINK ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'chem_comp_link',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_LINK category give details about\n               the links between chemical components.' ],
          [ 'item_', '_category.id', 'chem_comp_link' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_chem_comp_link.link_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'chem_link_group',
              [ '@comment', '     loop_' ],
              [ '@comment', '     _category_examples.detail' ],
              [ '@comment', '     _category_examples.case' ],
              [ '@comment',
                ' # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              [ '@comment', ' ;' ],
              [ '@comment',
                '     Example 1 - from nucleotide external reference dictionary Nucleic' ],
              [ '@comment', '                 Database Project 1997.' ],
              [ '@comment', ' ;' ],
              [ '@comment', ' ;' ],
              [ '@comment',
                '     _chem_comp_link.link_id        ribose_adenine' ],
              [ '@comment', '     _chem_comp_link.type_comp_1    ribose' ],
              [ '@comment', '     _chem_comp_link.type_comp_2    adenine' ],
              [ '@comment', '     _chem_comp_link.details' ],
              [ '@comment', '     ;' ],
              [ '@comment',
                '       Defines the linkage between adenine base and ribose sugar' ],
              [ '@comment', '     ;' ],
              [ '@comment', ' ;' ],
              [ '@comment',
                ' # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp_link.link_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_link.id in the\n               CHEM_LINK category.' ],
          [ 'item_', '_item.name', '_chem_comp_link.link_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_link.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of a link between\n               chemical components in the structure.' ],
          [ 'item_', '_item.name', '_chem_comp_link.details' ],
          [ 'item_', '_item.category_id', 'chem_comp_link' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_chem_comp_link.type_comp_1',
        [ [ 'item_',
            '_item_description.description',
            '              The type of the first of the two components joined by the\n               link.\n\n               This data item is a pointer to _chem_comp.type in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_chem_comp_link.type_comp_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_link.type_comp_2',
        [ [ 'item_',
            '_item_description.description',
            '              The type of the second of the two components joined by the\n               link.\n\n               This data item is a pointer to _chem_comp.type in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_chem_comp_link.type_comp_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# CHEM_COMP_PLANE ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'chem_comp_plane',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_PLANE category provide identifiers\n               for the planes in a chemical component.  The atoms in the plane\n               are specified in the CHEM_COMP_PLANE_ATOM category.' ],
          [ 'item_', '_category.id', 'chem_comp_plane' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_comp_plane.comp_id', '_chem_comp_plane.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _chem_comp_plane.comp_id\n    _chem_comp_plane.id\n      phe  phe1',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp_plane.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _chem_comp_plane.id must uniquely identify a record\n               in the CHEM_COMP_PLANE list.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chem_comp_plane.id',
              'chem_comp_plane',
              'yes',
              '_chem_comp_plane_atom.plane_id',
              'chem_comp_plane_atom',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_chem_comp_plane_atom.plane_id', '_chem_comp_plane.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_comp_plane.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_chem_comp_plane.comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_plane.number_atoms_all',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of atoms in the plane.' ],
          [ 'item_', '_item.name', '_chem_comp_plane.number_atoms_all' ],
          [ 'item_', '_item.category_id', 'chem_comp_plane' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chem_comp_plane.number_atoms_nh',
        [ [ 'item_',
            '_item_description.description',
            '              The number of non-hydrogen atoms in the plane.' ],
          [ 'item_', '_item.name', '_chem_comp_plane.number_atoms_nh' ],
          [ 'item_', '_item.category_id', 'chem_comp_plane' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# CHEM_COMP_PLANE_ATOM ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'chem_comp_plane_atom',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_PLANE_ATOM category enumerate the\n               atoms in a plane within a chemical component.' ],
          [ 'item_', '_category.id', 'chem_comp_plane_atom' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_comp_plane_atom.plane_id',
              '_chem_comp_plane_atom.atom_id',
              '_chem_comp_plane_atom.comp_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _chem_comp_plane_atom.plane_id\n    _chem_comp_plane_atom.comp_id\n    _chem_comp_plane_atom.atom_id\n      phe1  phe  CB\n      phe1  phe  CG\n      phe1  phe  CD1\n      phe1  phe  CE1\n      phe1  phe  CZ\n      phe1  phe  CE2\n      phe1  phe  CD2',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp_plane_atom.atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of an atom involved in the plane.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_plane_atom.atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_plane_atom.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_chem_comp_plane_atom.comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_plane_atom.plane_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp_plane.id in the\n               CHEM_COMP_PLANE category.' ],
          [ 'item_', '_item.name', '_chem_comp_plane_atom.plane_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_plane_atom.dist_esd',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is the standard deviation of the\n               out-of-plane distance for this atom.' ],
          [ 'item_', '_item.name', '_chem_comp_plane_atom.dist_esd' ],
          [ 'item_', '_item.category_id', 'chem_comp_plane_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# CHEM_COMP_TOR ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'chem_comp_tor',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_TOR category record details about\n               the torsion angles in a chemical component. As torsion angles\n               can have more than one target value, the target values are\n               specified in the CHEM_COMP_TOR_VALUE category.' ],
          [ 'item_', '_category.id', 'chem_comp_tor' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_comp_tor.comp_id', '_chem_comp_tor.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _chem_comp_tor.comp_id\n    _chem_comp_tor.id\n    _chem_comp_tor.atom_id_1\n    _chem_comp_tor.atom_id_2\n    _chem_comp_tor.atom_id_3\n    _chem_comp_tor.atom_id_4\n      phe  phe_chi1   N    CA   CB   CG\n      phe  phe_chi2   CA   CB   CG   CD1\n      phe  phe_ring1  CB   CG   CD1  CE1\n      phe  phe_ring2  CB   CG   CD2  CE2\n      phe  phe_ring3  CG   CD1  CE1  CZ\n      phe  phe_ring4  CD1  CE1  CZ   CE2\n      phe  phe_ring5  CE1  CZ   CE2  CD2',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp_tor.atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the first of the four atoms that define the torsion\n               angle.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_tor.atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_tor.atom_id_2',
              '_chem_comp_tor.atom_id_3',
              '_chem_comp_tor.atom_id_4' ] ] ] ],
      [ 'save_',
        '_chem_comp_tor.atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the second of the four atoms that define the torsion\n               angle.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_tor.atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_tor.atom_id_1',
              '_chem_comp_tor.atom_id_3',
              '_chem_comp_tor.atom_id_4' ] ] ] ],
      [ 'save_',
        '_chem_comp_tor.atom_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the third of the four atoms that define the torsion\n               angle.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_tor.atom_id_3' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_tor.atom_id_1',
              '_chem_comp_tor.atom_id_2',
              '_chem_comp_tor.atom_id_4' ] ] ] ],
      [ 'save_',
        '_chem_comp_tor.atom_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the fourth of the four atoms that define the torsion\n               angle.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_tor.atom_id_4' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_tor.atom_id_1',
              '_chem_comp_tor.atom_id_2',
              '_chem_comp_tor.atom_id_3' ] ] ] ],
      [ 'save_',
        '_chem_comp_tor.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _chem_comp_tor.id must uniquely identify a\n               record in the CHEM_COMP_TOR list.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chem_comp_tor.id',
              'chem_comp_tor',
              'yes',
              '_chem_comp_tor_value.tor_id',
              'chem_comp_tor_value',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_chem_comp_tor_value.tor_id', '_chem_comp_tor.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_comp_tor.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_chem_comp_tor.comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# CHEM_COMP_TOR_VALUE ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'chem_comp_tor_value',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_TOR_VALUE category record details\n               about the target values for the torsion angles enumerated in the\n               CHEM_COMP_TOR list. Target values may be specified as angles\n               in degrees, as a distance between the first and fourth atoms, or\n               both.' ],
          [ 'item_', '_category.id', 'chem_comp_tor_value' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_comp_tor_value.tor_id',
              '_chem_comp_tor_value.comp_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _chem_comp_tor_value.tor_id\n    _chem_comp_tor_value.comp_id\n    _chem_comp_tor_value.angle\n    _chem_comp_tor_value.dist\n      phe_chi1   phe  -60.0  2.88\n      phe_chi1   phe  180.0  3.72\n      phe_chi1   phe   60.0  2.88\n      phe_chi2   phe   90.0  3.34\n      phe_chi2   phe  -90.0  3.34\n      phe_ring1  phe  180.0  3.75\n      phe_ring2  phe  180.0  3.75\n      phe_ring3  phe    0.0  2.80\n      phe_ring4  phe    0.0  2.80\n      phe_ring5  phe    0.0  2.80',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_comp_tor_value.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp_atom.comp_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_chem_comp_tor_value.comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_tor_value.tor_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp_tor.id in the\n               CHEM_COMP_TOR category.' ],
          [ 'item_', '_item.name', '_chem_comp_tor_value.tor_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_comp_tor_value.angle',
        [ [ 'item_',
            '_item_description.description',
            '              A value that should be taken as a potential target value for the\n               torsion angle associated with the specified atoms, expressed in\n               degrees.' ],
          [ 'item_', '_item.name', '_chem_comp_tor_value.angle' ],
          [ 'item_', '_item.category_id', 'chem_comp_tor_value' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '-180.0', '-180.0', '-180.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_tor_value.angle_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_chem_comp_tor_value.angle_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_comp_tor_value.angle.' ],
          [ 'item_', '_item.name', '_chem_comp_tor_value.angle_esd' ],
          [ 'item_', '_item.category_id', 'chem_comp_tor_value' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '-180.0', '-180.0', '-180.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_tor_value.angle' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_chem_comp_tor_value.dist',
        [ [ 'item_',
            '_item_description.description',
            '              A value that should be taken as a potential target value for the\n               torsion angle associated with the specified atoms, expressed as\n               the distance between the atoms specified by\n               _chem_comp_tor.atom_id_1 and _chem_comp_tor.atom_id_4 in the\n               referenced record in the CHEM_COMP_TOR list. Note that the\n               torsion angle cannot be fully specified by a distance (for\n               instance, a torsion angle of -60  degree will yield the same\n               distance as a 60 degree angle). However, the distance\n               specification can be useful for refinement in situations\n               in which the angle is already close to the desired value.' ],
          [ 'item_', '_item.name', '_chem_comp_tor_value.dist' ],
          [ 'item_', '_item.category_id', 'chem_comp_tor_value' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_tor_value.dist_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_tor_value.dist_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_comp_tor_value.dist.' ],
          [ 'item_', '_item.name', '_chem_comp_tor_value.dist_esd' ],
          [ 'item_', '_item.category_id', 'chem_comp_tor_value' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_comp_tor_value.dist' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ '@comment', '##############' ],
      [ '@comment', '# CHEM_LINK ##' ],
      [ '@comment', '##############' ],
      [ 'save_',
        'chem_link',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_LINK category give details about\n               the links between chemical components.' ],
          [ 'item_', '_category.id', 'chem_link' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_chem_link.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_link_group' ] ] ] ],
      [ 'save_',
        '_chem_link.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _chem_link.id must uniquely identify each\n               item in the CHEM_LINK list.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chem_link.id',
              'chem_link',
              'yes',
              '_chem_link_angle.link_id',
              'chem_link_angle',
              'yes',
              '_chem_link_bond.link_id',
              'chem_link_bond',
              'yes',
              '_chem_link_chir.link_id',
              'chem_link_chir',
              'yes',
              '_chem_link_plane.link_id',
              'chem_link_plane',
              'yes',
              '_chem_link_tor.link_id',
              'chem_link_tor',
              'yes',
              '_chem_comp_link.link_id',
              'chem_comp_link',
              'yes',
              '_entity_link.link_id',
              'entity_link',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_chem_link_angle.link_id',
              '_chem_link.id',
              '_chem_link_bond.link_id',
              '_chem_link.id',
              '_chem_link_chir.link_id',
              '_chem_link.id',
              '_chem_link_plane.link_id',
              '_chem_link.id',
              '_chem_link_tor.link_id',
              '_chem_link.id',
              '_chem_comp_link.link_id',
              '_chem_link.id',
              '_entity_link.link_id',
              '_chem_link.id' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'peptide', 'oligosaccharide 1,4', 'DNA' ] ] ] ],
      [ 'save_',
        '_chem_link.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of a link between\n               chemical components in the structure.' ],
          [ 'item_', '_item.name', '_chem_link.details' ],
          [ 'item_', '_item.category_id', 'chem_link' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# CHEM_LINK_ANGLE ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'chem_link_angle',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_LINK_ANGLE category record details\n               about angles in a link between chemical components.' ],
          [ 'item_', '_category.id', 'chem_link_angle' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_link_angle.link_id',
              '_chem_link_angle.atom_id_1',
              '_chem_link_angle.atom_id_2',
              '_chem_link_angle.atom_id_3' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_link_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - Engh & Huber parameters [Acta Cryst. (1991), A47,\n                392-400] as interpreted by J. P. Priestle (1995). Consistent\n                Stereochemical Dictionaries for Refinement and Model\n                Building. CCP4 Daresbury Study Weekend,\n                DL-CONF-95-001, ISSN 1358-6254. Warrington: Daresbury\n                Laboratory.',
              '\n    loop_\n    _chem_link_angle.link_id\n    _chem_link_angle.value_angle\n    _chem_link_angle.value_angle_esd\n    _chem_link_angle.atom_id_1\n    _chem_link_angle.atom_1_comp_id\n    _chem_link_angle.atom_id_2\n    _chem_link_angle.atom_2_comp_id\n    _chem_link_angle.atom_id_3\n    _chem_link_angle.atom_3_comp_id\n     PEPTIDE  111.2  2.8  N  1  CA 1  C  1\n     PEPTIDE  120.8  1.7  CA 1  C  1  O  1\n     PEPTIDE  116.2  2.0  CA 1  C  1  N  2\n     PEPTIDE  123.0  1.6  O  1  C  1  N  2\n     PEPTIDE  121.7  1.8  C  1  N  2  CA 2',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_link_angle.atom_1_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether atom 1 is found in the first\n               or the second of the two components connected by the link.' ],
          [ 'item_', '_item.name', '_chem_link_angle.atom_1_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_angle.atom_2_comp_id',
              '_chem_link_angle.atom_3_comp_id' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_angle.atom_2_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether atom 2 is found in the first\n               or the second of the two components connected by the link.' ],
          [ 'item_', '_item.name', '_chem_link_angle.atom_2_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_angle.atom_1_comp_id',
              '_chem_link_angle.atom_3_comp_id' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_angle.atom_3_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether atom 3 is found in the first\n               or the second of the two components connected by the link.' ],
          [ 'item_', '_item.name', '_chem_link_angle.atom_3_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_angle.atom_1_comp_id',
              '_chem_link_angle.atom_2_comp_id' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_angle.atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the first of the three atoms that define the angle.\n\n               An atom with this ID must exist in the component of the type\n               specified by _chem_comp_link.type_comp_1 (or\n               _chem_comp_link.type_comp_2, where the appropriate data item\n               is indicated by the value of _chem_comp_angle.atom_1_comp_id).' ],
          [ 'item_', '_item.name', '_chem_link_angle.atom_id_1' ],
          [ 'item_', '_item.category_id', 'chem_link_angle' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_angle.atom_id_2', '_chem_link_angle.atom_id_3' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_angle.atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the second of the three atoms that define the angle.\n               The second atom is taken to be the apex of the angle.\n\n               An atom with this ID must exist in the component of the type\n               specified by _chem_comp_link.type_comp_1 (or\n               _chem_comp_link.type_comp_2, where the appropriate data item\n               is indicated by the value of _chem_comp_angle.atom_2_comp_id).' ],
          [ 'item_', '_item.name', '_chem_link_angle.atom_id_2' ],
          [ 'item_', '_item.category_id', 'chem_link_angle' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_angle.atom_id_1', '_chem_link_angle.atom_id_3' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_angle.atom_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the third of the three atoms that define the angle.\n\n               An atom with this ID must exist in the component of the type\n               specified by _chem_comp_link.type_comp_1 (or\n               _chem_comp_link.type_comp_2, where the appropriate data item\n              is indicated by the value of _chem_comp_angle.atom_3_comp_id).' ],
          [ 'item_', '_item.name', '_chem_link_angle.atom_id_3' ],
          [ 'item_', '_item.category_id', 'chem_link_angle' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_angle.atom_id_1', '_chem_link_angle.atom_id_2' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_angle.link_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_link.id in the CHEM_LINK\n               category.' ],
          [ 'item_', '_item.name', '_chem_link_angle.link_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_link_angle.value_angle',
        [ [ 'item_',
            '_item_description.description',
            '              The value that should be taken as the target value for the angle\n               associated with the specified atoms, expressed in degrees.' ],
          [ 'item_', '_item.name', '_chem_link_angle.value_angle' ],
          [ 'item_', '_item.category_id', 'chem_link_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_angle.value_angle_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_chem_link_angle.value_angle_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_link_angle.value_angle.' ],
          [ 'item_', '_item.name', '_chem_link_angle.value_angle_esd' ],
          [ 'item_', '_item.category_id', 'chem_link_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_angle.value_angle' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_chem_link_angle.value_dist',
        [ [ 'item_',
            '_item_description.description',
            '              The value that should be taken as the target value for the angle\n               associated with the specified atoms, expressed as the distance\n               between the atoms specified by _chem_comp_angle.atom_id_1 and\n               _chem_comp_angle.atom_id_3.' ],
          [ 'item_', '_item.name', '_chem_link_angle.value_dist' ],
          [ 'item_', '_item.category_id', 'chem_link_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_angle.value_dist_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_link_angle.value_dist_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_comp_angle.value_dist.' ],
          [ 'item_', '_item.name', '_chem_link_angle.value_dist_esd' ],
          [ 'item_', '_item.category_id', 'chem_link_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_angle.value_dist' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# CHEM_LINK_BOND ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'chem_link_bond',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_LINK_BOND category record details about\n               bonds in a link between components in the chemical structure.' ],
          [ 'item_', '_category.id', 'chem_link_bond' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_link_bond.link_id',
              '_chem_link_bond.atom_id_1',
              '_chem_link_bond.atom_id_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_link_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - Engh & Huber parameters [Acta Cryst. (1991), A47,\n                392-400] as interpreted by J. P. Priestle (1995). Consistent\n                Stereochemical Dictionaries for Refinement and Model\n                Building. CCP4 Daresbury Study Weekend,\n                DL-CONF-95-001, ISSN 1358-6254. Warrington: Daresbury\n                Laboratory.',
              '\n    loop_\n    _chem_link_bond.link_id\n    _chem_link_bond.value_dist\n    _chem_link_bond.value_dist_esd\n    _chem_link_bond.atom_id_1\n    _chem_link_bond.atom_1_comp_id\n    _chem_link_bond.atom_id_2\n    _chem_link_bond.atom_2_comp_id\n     PEPTIDE  1.458  0.019  N  1  CA 1\n     PEPTIDE  1.525  0.021  CA 1  C  1\n     PEPTIDE  1.329  0.014  C  1  N  2\n     PEPTIDE  1.231  0.020  C  1  O  1',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chem_link_bond.atom_1_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether atom 1 is found in the first\n               or the second of the two components connected by the link.' ],
          [ 'item_', '_item.name', '_chem_link_bond.atom_1_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_bond.atom_2_comp_id' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_bond.atom_2_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether atom 2 is found in the first\n               or the second of the two chemical components connected by\n               the link.' ],
          [ 'item_', '_item.name', '_chem_link_bond.atom_2_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_chem_link_bond.atom_1_comp_id' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_bond.atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the first of the two atoms that define the bond.\n\n               As this data item does not point to a specific atom in a\n               specific chemical component, it is not a child in the\n               linkage sense.' ],
          [ 'item_', '_item.name', '_chem_link_bond.atom_id_1' ],
          [ 'item_', '_item.category_id', 'chem_link_bond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_bond.atom_id_2' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_bond.atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the second of the two atoms that define the bond.\n\n               As this data item does not point to a specific atom in a\n               specific component, it is not a child in the linkage sense.' ],
          [ 'item_', '_item.name', '_chem_link_bond.atom_id_2' ],
          [ 'item_', '_item.category_id', 'chem_link_bond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_chem_link_bond.atom_id_1' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_bond.link_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_link.id in the CHEM_LINK\n               category.' ],
          [ 'item_', '_item.name', '_chem_link_bond.link_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_link_bond.value_dist',
        [ [ 'item_',
            '_item_description.description',
            '              The value that should be taken as the target for the chemical\n               bond associated with the specified atoms, expressed as a\n               distance.' ],
          [ 'item_', '_item.name', '_chem_link_bond.value_dist' ],
          [ 'item_', '_item.category_id', 'chem_link_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_bond.value_dist_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_link_bond.value_dist_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_link_bond.value_dist.' ],
          [ 'item_', '_item.name', '_chem_link_bond.value_dist_esd' ],
          [ 'item_', '_item.category_id', 'chem_link_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_bond.value_dist' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_link_bond.value_order',
        [ [ 'item_',
            '_item_description.description',
            '              The value that should be taken as the target for the chemical\n               bond associated with the specified atoms, expressed as a bond\n               order.' ],
          [ 'item_', '_item.name', '_chem_link_bond.value_order' ],
          [ 'item_', '_item.category_id', 'chem_link_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', 'sing' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'sing',
              'single bond',
              'doub',
              'double bond',
              'trip',
              'triple bond',
              'quad',
              'quadruple bond',
              'arom',
              'aromatic bond',
              'poly',
              'polymeric bond',
              'delo',
              'delocalized double bond',
              'pi',
              'pi bond' ] ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# CHEM_LINK_CHIR ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'chem_link_chir',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_LINK_CHIR category provide details about\n               the chiral centres in a link between two chemical components.\n               The atoms bonded to the chiral atom are specified in the\n               CHEM_LINK_CHIR_ATOM category.' ],
          [ 'item_', '_category.id', 'chem_link_chir' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_link_chir.link_id', '_chem_link_chir.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_link_group' ] ] ] ],
      [ 'save_',
        '_chem_link_chir.atom_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether the chiral atom is found in the\n               first or the second of the two components connected by the\n               link.' ],
          [ 'item_', '_item.name', '_chem_link_chir.atom_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_chir.atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the atom that is a chiral centre.\n\n               As this data item does not point to a specific atom in a\n               specific chemical component, it is not a child in the linkage\n               sense.' ],
          [ 'item_', '_item.name', '_chem_link_chir.atom_id' ],
          [ 'item_', '_item.category_id', 'chem_link_chir' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_chir.atom_config',
        [ [ 'item_',
            '_item_description.description',
            '              The chiral configuration of the atom that is a chiral centre.' ],
          [ 'item_', '_item.name', '_chem_link_chir.atom_config' ],
          [ 'item_', '_item.category_id', 'chem_link_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'R',
              'absolute configuration R',
              'S',
              'absolute configuration S' ] ] ] ],
      [ 'save_',
        '_chem_link_chir.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _chem_link_chir.id must uniquely identify a record\n               in the CHEM_LINK_CHIR list.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chem_link_chir.id',
              'chem_link_chir',
              'yes',
              '_chem_link_chir_atom.chir_id',
              'chem_link_chir_atom',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_chem_link_chir_atom.chir_id', '_chem_link_chir.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_chir.link_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_link.id in the CHEM_LINK\n               category.' ],
          [ 'item_', '_item.name', '_chem_link_chir.link_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_link_chir.number_atoms_all',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of atoms bonded to the atom specified by\n               _chem_link_chir.atom_id.' ],
          [ 'item_', '_item.name', '_chem_link_chir.number_atoms_all' ],
          [ 'item_', '_item.category_id', 'chem_link_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chem_link_chir.number_atoms_nh',
        [ [ 'item_',
            '_item_description.description',
            '              The number of non-hydrogen atoms bonded to the atom specified by\n               _chem_link_chir.atom_id.' ],
          [ 'item_', '_item.name', '_chem_link_chir.number_atoms_nh' ],
          [ 'item_', '_item.category_id', 'chem_link_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chem_link_chir.volume_flag',
        [ [ 'item_',
            '_item_description.description',
            '              A flag to indicate whether a chiral volume should match the\n               standard value in both magnitude and sign, or in magnitude only.' ],
          [ 'item_', '_item.name', '_chem_link_chir.volume_flag' ],
          [ 'item_', '_item.category_id', 'chem_link_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'sign',
              'match magnitude and sign',
              'nosign',
              'match magnitude only' ] ] ] ],
      [ 'save_',
        '_chem_link_chir.volume_three',
        [ [ 'item_',
            '_item_description.description',
            '              The chiral volume, V(c), for chiral centres that involve a chiral\n               atom bonded to three non-hydrogen atoms and one hydrogen atom.\n\n               V~c~ = V1 * (V2 X V3)\n\n               V1 = the vector distance from the atom specified by\n                    _chem_link_chir.atom_id to the first atom in the\n                    CHEM_LINK_CHIR_ATOM list\n               V2 = the vector distance from the atom specified by\n                    _chem_link_chir.atom_id to the second atom in the\n                    CHEM_LINK_CHIR_ATOM list\n               V3 = the vector distance from the atom specified by\n                    _chem_link_chir.atom_id to the third atom in the\n                    CHEM_LINK_CHIR_ATOM list\n               *  = the vector dot product\n               X  = the vector cross product' ],
          [ 'item_', '_item.name', '_chem_link_chir.volume_three' ],
          [ 'item_', '_item.category_id', 'chem_link_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_chir.volume_three_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms_cubed' ] ] ],
      [ 'save_',
        '_chem_link_chir.volume_three_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_link_chir.volume_three.' ],
          [ 'item_', '_item.name', '_chem_link_chir.volume_three_esd' ],
          [ 'item_', '_item.category_id', 'chem_link_chir' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_chir.volume_three' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_cubed' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# CHEM_LINK_CHIR_ATOM ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'chem_link_chir_atom',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_LINK_CHIR_ATOM category enumerate the\n               atoms bonded to a chiral atom in a link between two\n               chemical components.' ],
          [ 'item_', '_category.id', 'chem_link_chir_atom' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_link_chir_atom.chir_id',
              '_chem_link_chir_atom.atom_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_link_group' ] ] ] ],
      [ 'save_',
        '_chem_link_chir_atom.atom_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether the atom bonded to a chiral\n               atom is found in the first or the second of the two components\n               connected by the link.' ],
          [ 'item_', '_item.name', '_chem_link_chir_atom.atom_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_chir_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_chir_atom.atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of an atom bonded to the chiral atom.\n\n               As this data item does not point to a specific atom in a\n               specific chemical component, it is not a child in the linkage\n               sense.' ],
          [ 'item_', '_item.name', '_chem_link_chir_atom.atom_id' ],
          [ 'item_', '_item.category_id', 'chem_link_chir_atom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_chir_atom.chir_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_link_chir.id in the\n               CHEM_LINK_CHIR category.' ],
          [ 'item_', '_item.name', '_chem_link_chir_atom.chir_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_link_chir_atom.dev',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of the position of this atom from the plane defined by\n               all of the atoms in the plane.' ],
          [ 'item_', '_item.name', '_chem_link_chir_atom.dev' ],
          [ 'item_', '_item.category_id', 'chem_link_chir_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# CHEM_LINK_PLANE ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'chem_link_plane',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_LINK_PLANE category provide identifiers\n               for the planes in a link between two chemical components.\n               The atoms in the plane are specified in the CHEM_LINK_PLANE_ATOM\n               category.' ],
          [ 'item_', '_category.id', 'chem_link_plane' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_link_plane.link_id', '_chem_link_plane.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_link_group' ] ] ] ],
      [ 'save_',
        '_chem_link_plane.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _chem_link_plane.id must uniquely identify a record\n               in the CHEM_LINK_PLANE list.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chem_link_plane.id',
              'chem_link_plane',
              'yes',
              '_chem_link_plane_atom.plane_id',
              'chem_link_plane_atom',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_chem_link_plane_atom.plane_id', '_chem_link_plane.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_plane.link_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_link.id in the CHEM_LINK\n               category.' ],
          [ 'item_', '_item.name', '_chem_link_plane.link_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_link_plane.number_atoms_all',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of atoms in the plane.' ],
          [ 'item_', '_item.name', '_chem_link_plane.number_atoms_all' ],
          [ 'item_', '_item.category_id', 'chem_link_plane' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chem_link_plane.number_atoms_nh',
        [ [ 'item_',
            '_item_description.description',
            '              The number of non-hydrogen atoms in the plane.' ],
          [ 'item_', '_item.name', '_chem_link_plane.number_atoms_nh' ],
          [ 'item_', '_item.category_id', 'chem_link_plane' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# CHEM_LINK_PLANE_ATOM ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'chem_link_plane_atom',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_LINK_PLANE_ATOM category enumerate the\n               atoms in a plane in a link between two chemical components.' ],
          [ 'item_', '_category.id', 'chem_link_plane_atom' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_link_plane_atom.plane_id',
              '_chem_link_plane_atom.atom_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_link_group' ] ] ] ],
      [ 'save_',
        '_chem_link_plane_atom.atom_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether the atom in a plane is found in\n               the first or the second of the two components connected by the\n               link.' ],
          [ 'item_', '_item.name', '_chem_link_plane_atom.atom_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_plane_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_plane_atom.atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of an atom involved in the plane.\n\n               As this data item does not point to a specific atom in a\n               specific chemical component, it is not a child in the linkage\n               sense.' ],
          [ 'item_', '_item.name', '_chem_link_plane_atom.atom_id' ],
          [ 'item_', '_item.category_id', 'chem_link_plane_atom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_plane_atom.plane_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_link_plane.id in the\n               CHEM_LINK_PLANE category.' ],
          [ 'item_', '_item.name', '_chem_link_plane_atom.plane_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '##################' ],
      [ '@comment', '# CHEM_LINK_TOR ##' ],
      [ '@comment', '##################' ],
      [ 'save_',
        'chem_link_tor',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_LINK_TOR category record details about\n               the torsion angles in a link between two chemical components.\n               As torsion angles can have more than one target value, the\n               target values are specified in the CHEM_LINK_TOR_VALUE category.' ],
          [ 'item_', '_category.id', 'chem_link_tor' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chem_link_tor.link_id', '_chem_link_tor.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_link_group' ] ] ] ],
      [ 'save_',
        '_chem_link_tor.atom_1_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether atom 1 is found in the first\n               or the second of the two components connected by the link.' ],
          [ 'item_', '_item.name', '_chem_link_tor.atom_1_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_tor' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_tor.atom_2_comp_id',
              '_chem_link_tor.atom_3_comp_id',
              '_chem_link_tor.atom_4_comp_id' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_tor.atom_2_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether atom 2 is found in the first\n               or the second of the two components connected by the link.' ],
          [ 'item_', '_item.name', '_chem_link_tor.atom_2_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_tor' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_tor.atom_1_comp_id',
              '_chem_link_tor.atom_3_comp_id',
              '_chem_link_tor.atom_4_comp_id' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_tor.atom_3_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether atom 3 is found in the first\n               or the second of the two components connected by the link.' ],
          [ 'item_', '_item.name', '_chem_link_tor.atom_3_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_tor' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_tor.atom_1_comp_id',
              '_chem_link_tor.atom_2_comp_id',
              '_chem_link_tor.atom_4_comp_id' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_tor.atom_4_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item indicates whether atom 4 is found in the first\n               or the second of the two components connected by the link.' ],
          [ 'item_', '_item.name', '_chem_link_tor.atom_4_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_link_tor' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_tor.atom_1_comp_id',
              '_chem_link_tor.atom_2_comp_id',
              '_chem_link_tor.atom_3_comp_id' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '1',
              'the atom is in component 1',
              '2',
              'the atom is in component 2' ] ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_link_tor.atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the first of the four atoms that define the torsion\n               angle.\n\n               As this data item does not point to a specific atom in a\n               specific chemical component, it is not a child in the linkage\n               sense.' ],
          [ 'item_', '_item.name', '_chem_link_tor.atom_id_1' ],
          [ 'item_', '_item.category_id', 'chem_link_tor' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_tor.atom_id_2',
              '_chem_link_tor.atom_id_3',
              '_chem_link_tor.atom_id_4' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_tor.atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the second of the four atoms that define the torsion\n               angle.\n\n               As this data item does not point to a specific atom in a\n               specific chemical component, it is not a child in the linkage\n               sense.' ],
          [ 'item_', '_item.name', '_chem_link_tor.atom_id_2' ],
          [ 'item_', '_item.category_id', 'chem_link_tor' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_tor.atom_id_1',
              '_chem_link_tor.atom_id_3',
              '_chem_link_tor.atom_id_4' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_tor.atom_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the third of the four atoms that define the torsion\n               angle.\n\n               As this data item does not point to a specific atom in a\n               specific chemical component, it is not a child in the linkage\n               sense.' ],
          [ 'item_', '_item.name', '_chem_link_tor.atom_id_3' ],
          [ 'item_', '_item.category_id', 'chem_link_tor' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_tor.atom_id_1',
              '_chem_link_tor.atom_id_2',
              '_chem_link_tor.atom_id_4' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_tor.atom_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              The ID of the fourth of the four atoms that define the torsion\n               angle.\n\n               As this data item does not point to a specific atom in a\n               specific chemical component, it is not a child in the linkage\n               sense.' ],
          [ 'item_', '_item.name', '_chem_link_tor.atom_id_4' ],
          [ 'item_', '_item.category_id', 'chem_link_tor' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_link_tor.atom_id_1',
              '_chem_link_tor.atom_id_2',
              '_chem_link_tor.atom_id_3' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_tor.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _chem_link_tor.id must uniquely identify a\n               record in the CHEM_LINK_TOR list.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chem_link_tor.id',
              'chem_link_tor',
              'yes',
              '_chem_link_tor_value.tor_id',
              'chem_link_tor_value',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_chem_link_tor_value.tor_id', '_chem_link_tor.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_chem_link_tor.link_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_link.id in the CHEM_LINK\n               category.' ],
          [ 'item_', '_item.name', '_chem_link_tor.link_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# CHEM_LINK_TOR_VALUE ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'chem_link_tor_value',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_LINK_TOR_VALUE category record details\n               about the target values for the torsion angles enumerated in the\n               CHEM_LINK_TOR list. Target values may be specified as angles\n               in degrees, as a distance between the first and fourth atoms, or\n               both.' ],
          [ 'item_', '_category.id', 'chem_link_tor_value' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_chem_link_tor_value.tor_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_link_group' ] ] ] ],
      [ 'save_',
        '_chem_link_tor_value.tor_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_link_tor.id in the\n               CHEM_LINK_TOR category.' ],
          [ 'item_', '_item.name', '_chem_link_tor_value.tor_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chem_link_tor_value.angle',
        [ [ 'item_',
            '_item_description.description',
            '              A value that should be taken as a potential target value for the\n               torsion angle associated with the specified atoms, expressed in\n               degrees.' ],
          [ 'item_', '_item.name', '_chem_link_tor_value.angle' ],
          [ 'item_', '_item.category_id', 'chem_link_tor_value' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '-180.0', '-180.0', '-180.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_tor_value.angle_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_chem_link_tor_value.angle_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_link_tor_value.angle.' ],
          [ 'item_', '_item.name', '_chem_link_tor_value.angle_esd' ],
          [ 'item_', '_item.category_id', 'chem_link_tor_value' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '-180.0', '-180.0', '-180.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_tor_value.angle' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_chem_link_tor_value.dist',
        [ [ 'item_',
            '_item_description.description',
            '              A value that should be taken as a potential target value for the\n               torsion angle associated with the specified atoms, expressed as\n               the distance between the atoms specified by\n               _chem_link_tor.atom_id_1 and _chem_link_tor.atom_id_4 in the\n               referenced record in the CHEM_LINK_TOR list. Note that the\n               torsion angle cannot be fully specified by a distance (for\n               instance, a torsion angle of -60 degree will yield the same\n               distance as a 60 degree angle). However, the distance\n               specification can be useful for refinement in situations in\n               which the angle is already close to the desired value.' ],
          [ 'item_', '_item.name', '_chem_link_tor_value.dist' ],
          [ 'item_', '_item.category_id', 'chem_link_tor_value' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_tor_value.dist_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_link_tor_value.dist_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _chem_link_tor_value.dist.' ],
          [ 'item_', '_item.name', '_chem_link_tor_value.dist_esd' ],
          [ 'item_', '_item.category_id', 'chem_link_tor_value' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_chem_link_tor_value.dist' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ '@comment', '#############' ],
      [ '@comment', '# CHEMICAL ##' ],
      [ '@comment', '#############' ],
      [ 'save_',
        'chemical',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEMICAL category would not in general be\n               used in a macromolecular CIF. See instead the ENTITY data\n               items.\n\n               Data items in the CHEMICAL category record details about the\n               composition and chemical properties of the compounds. The\n               formula data items must agree with those that specify the\n               density, unit-cell and Z values.' ],
          [ 'item_', '_category.id', 'chemical' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_chemical.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chemical_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set 9597gaus of Alyea, Ferguson & Kannan\n                [Acta Cryst. (1996), C52, 765-767].',
              '\n    _chemical.entry_id        \'9597gaus\'\n    _chemical.name_systematic\n      trans-bis(tricyclohexylphosphine)tetracarbonylmolybdenum(0)',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chemical.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_chemical.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chemical.compound_source',
        [ [ 'item_',
            '_item_description.description',
            '              Description of the source of the compound under study, or of the\n               parent  molecule if a simple derivative is studied. This includes\n               the place of discovery for minerals or the actual source of a\n               natural product.' ],
          [ 'item_', '_item.name', '_chemical.compound_source' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_compound_source' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'From Norilsk (USSR)',
              'Extracted from the bark of Cinchona Naturalis' ] ] ] ],
      [ 'save_',
        '_chemical.melting_point',
        [ [ 'item_',
            '_item_description.description',
            '              The temperature in kelvins at which the crystalline solid changes\n               to a liquid.' ],
          [ 'item_', '_item.name', '_chemical.melting_point' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_melting_point' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ 'save_',
        '_chemical.name_common',
        [ [ 'item_',
            '_item_description.description',
            '              Trivial name by which the compound is commonly known.' ],
          [ 'item_', '_item.name', '_chemical.name_common' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_chemical_name_common' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', '1-bromoestradiol' ] ] ],
      [ 'save_',
        '_chemical.name_mineral',
        [ [ 'item_',
            '_item_description.description',
            '              Mineral name accepted by the International Mineralogical\n               Association. Use only for natural minerals. See also\n               _chemical.compound_source.' ],
          [ 'item_', '_item.name', '_chemical.name_mineral' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_chemical_name_mineral' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'chalcopyrite' ] ] ],
      [ 'save_',
        '_chemical.name_structure_type',
        [ [ 'item_',
            '_item_description.description',
            '              Commonly used structure-type name. Usually only applied to\n               minerals or inorganic compounds.' ],
          [ 'item_', '_item.name', '_chemical.name_structure_type' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_name_structure_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'perovskite', 'sphalerite', 'A15' ] ] ] ],
      [ 'save_',
        '_chemical.name_systematic',
        [ [ 'item_',
            '_item_description.description',
            '              IUPAC or Chemical Abstracts full name of the compound.' ],
          [ 'item_', '_item.name', '_chemical.name_systematic' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_name_systematic' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '1-bromoestra-1,3,5(10)-triene-3,17\\b-diol' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# CHEMICAL_CONN_ATOM ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'chemical_conn_atom',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEMICAL_CONN_ATOM category would not, in\n               general, be used in a macromolecular CIF. See instead the\n               ENTITY data items.\n\n               Data items in the CHEMICAL_CONN_ATOM and CHEMICAL_CONN_BOND\n               categories record details about the two-dimensional (2D)\n               chemical structure of the molecular species. They allow\n               a 2D chemical diagram to be reconstructed for use in a\n               publication or in a database search for structural and\n               substructural relationships.\n\n               The CHEMICAL_CONN_ATOM data items provide information about the\n               chemical properties of the atoms in the structure. In cases\n               where crystallographic and molecular symmetry elements coincide,\n               they must also contain symmetry-generated atoms, so that the\n               CHEMICAL_CONN_ATOM and CHEMICAL_CONN_BOND data items will always\n               describe a complete chemical entity.' ],
          [ 'item_', '_category.id', 'chemical_conn_atom' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_chemical_conn_atom.number' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chemical_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set DPTD of Yamin, Suwandi, Fun, Sivakumar &\n                bin Shawkataly [Acta Cryst. (1996), C52, 951-953].',
              '\n    loop_\n    _chemical_conn_atom.number\n    _chemical_conn_atom.type_symbol\n    _chemical_conn_atom.display_x\n    _chemical_conn_atom.display_y\n    _chemical_conn_atom.NCA\n    _chemical_conn_atom.NH\n        1   S    .39  .81   1   0\n        2   S    .39  .96   2   0\n        3   N    .14  .88   3   0\n        4   C    .33  .88   3   0\n        5   C    .11  .96   2   2\n        6   C    .03  .96   2   2\n        7   C    .03  .80   2   2\n        8   C    .11  .80   2   2\n        9   S    .54  .81   1   0\n        10  S    .54  .96   2   0\n        11  N    .80  .88   3   0\n        12  C    .60  .88   3   0\n        13  C    .84  .96   2   2\n        14  C    .91  .96   2   2\n        15  C    .91  .80   2   2\n        16  C    .84  .80   2   2',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chemical_conn_atom.charge',
        [ [ 'item_',
            '_item_description.description',
            '              The net integer charge assigned to this atom. This is the\n               formal charge assignment normally found in chemical diagrams.' ],
          [ 'item_', '_item.name', '_chemical_conn_atom.charge' ],
          [ 'item_', '_item.category_id', 'chemical_conn_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_conn_atom_charge' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '8', '8', '8', '-8', '-8', '-8' ] ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '1', 'for an ammonium nitrogen', '-1', 'for a chloride ion' ] ] ] ],
      [ 'save_',
        '_chemical_conn_atom.display_x',
        [ [ 'item_',
            '_item_description.description',
            '              The 2D Cartesian x coordinate of the position of this atom in a\n               recognizable chemical diagram. The coordinate origin is at the\n               lower left corner, the x axis is horizontal and the y axis\n               is vertical. The coordinates must lie in the range 0.0 to 1.0.\n               These coordinates can be obtained from projections of a suitable\n               uncluttered view of the molecular structure.' ],
          [ 'item_', '_item.name', '_chemical_conn_atom.display_x' ],
          [ 'item_', '_item.category_id', 'chemical_conn_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_conn_atom_display_x' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_chemical_conn_atom.display_y' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '1.0', '1.0', '1.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_chemical_conn_atom.display_y',
        [ [ 'item_',
            '_item_description.description',
            '              The 2D Cartesian y coordinate of the position of this atom in a\n               recognizable chemical diagram. The coordinate origin is at the\n               lower left corner, the x axis is horizontal and the y axis\n               is vertical. The coordinates must lie in the range 0.0 to 1.0.\n               These coordinates can be obtained from projections of a suitable\n               uncluttered view of the molecular structure.' ],
          [ 'item_', '_item.name', '_chemical_conn_atom.display_y' ],
          [ 'item_', '_item.category_id', 'chemical_conn_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_conn_atom_display_y' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_chemical_conn_atom.display_x' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '1.0', '1.0', '1.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_chemical_conn_atom.NCA',
        [ [ 'item_',
            '_item_description.description',
            '              The number of connected atoms excluding terminal hydrogen atoms.' ],
          [ 'item_', '_item.name', '_chemical_conn_atom.NCA' ],
          [ 'item_', '_item.category_id', 'chemical_conn_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_conn_atom_NCA' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chemical_conn_atom.NH',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of hydrogen atoms attached to this atom,\n               regardless of whether they are included in the refinement or\n               the ATOM_SITE list. This number is the same as\n               _atom_site.attached_hydrogens only if none of the hydrogen\n               atoms appear in the ATOM_SITE list.' ],
          [ 'item_', '_item.name', '_chemical_conn_atom.NH' ],
          [ 'item_', '_item.category_id', 'chemical_conn_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_chemical_conn_atom_NH' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chemical_conn_atom.number',
        [ [ 'item_',
            '_item_description.description',
            '              The chemical sequence number to be associated with this atom.\n               Within an ATOM_SITE list, this number must match one of\n               the _atom_site.chemical_conn_number values.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_chemical_conn_atom.number',
              'chemical_conn_atom',
              'yes',
              '_atom_site.chemical_conn_number',
              'atom_site',
              'no',
              '_chemical_conn_bond.atom_1',
              'chemical_conn_bond',
              'yes',
              '_chemical_conn_bond.atom_2',
              'chemical_conn_bond',
              'yes' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_conn_atom_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_site.chemical_conn_number',
              '_chemical_conn_atom.number',
              '_chemical_conn_bond.atom_1',
              '_chemical_conn_atom.number',
              '_chemical_conn_bond.atom_2',
              '_chemical_conn_atom.number' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chemical_conn_atom.type_symbol',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _atom_type.symbol in the\n               ATOM_TYPE category.' ],
          [ 'item_', '_item.name', '_chemical_conn_atom.type_symbol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_conn_atom_type_symbol' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# CHEMICAL_CONN_BOND ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'chemical_conn_bond',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEMICAL_CONN_BOND category would not, in\n               general, be used in a macromolecular CIF. See instead the\n               ENTITY data items.\n\n               Data items in the CHEMICAL_CONN_ATOM and CHEMICAL_CONN_BOND\n               categories record details about the two-dimensional (2D)\n               chemical structure of the molecular species. They allow a\n               2D chemical diagram to be reconstructed for use in a\n               publication or in a database search for structural and\n               substructural relationships.\n\n               The CHEMICAL_CONN_BOND data items specify the connections\n               between the atoms in the CHEMICAL_CONN_ATOM list and the nature\n               of the chemical bond between these atoms.' ],
          [ 'item_', '_category.id', 'chemical_conn_bond' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_chemical_conn_bond.atom_1', '_chemical_conn_bond.atom_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chemical_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set DPTD of Yamin, Suwandi, Fun, Sivakumar &\n                bin Shawkataly [Acta Cryst. (1996), C52, 951-953].',
              '\n    loop_\n    _chemical_conn_bond.atom_1\n    _chemical_conn_bond.atom_2\n    _chemical_conn_bond.type\n       4     1     doub     4     3     sing\n       4     2     sing     5     3     sing\n       6     5     sing     7     6     sing\n       8     7     sing     8     3     sing\n       10    2     sing     12    9     doub\n       12    11    sing     12    10    sing\n       13    11    sing     14    13    sing\n       15    14    sing     16    15    sing\n       16    11    sing     17    5     sing\n       18    5     sing     19    6     sing\n       20    6     sing     21    7     sing\n       22    7     sing     23    8     sing\n       24    8     sing     25    13    sing\n       26    13    sing     27    14    sing\n       28    14    sing     29    15    sing\n       30    15    sing     31    16    sing\n       32    16    sing',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chemical_conn_bond.atom_1',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chemical_conn_atom.number in the\n               CHEMICAL_CONN_ATOM category.' ],
          [ 'item_', '_item.name', '_chemical_conn_bond.atom_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_conn_bond_atom_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_chemical_conn_bond.atom_2' ] ] ],
      [ 'save_',
        '_chemical_conn_bond.atom_2',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chemical_conn_atom.number in the\n               CHEMICAL_CONN_ATOM category.' ],
          [ 'item_', '_item.name', '_chemical_conn_bond.atom_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_conn_bond_atom_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_chemical_conn_bond.atom_1' ] ] ],
      [ 'save_',
        '_chemical_conn_bond.type',
        [ [ 'item_',
            '_item_description.description',
            '              The chemical bond type associated with the connection between\n               the two sites _chemical_conn_bond.atom_1 and\n               _chemical_conn_bond.atom_2.' ],
          [ 'item_', '_item.name', '_chemical_conn_bond.type' ],
          [ 'item_', '_item.category_id', 'chemical_conn_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_conn_bond_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', 'sing' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'sing',
              'single bond',
              'doub',
              'double bond',
              'trip',
              'triple bond',
              'quad',
              'quadruple bond',
              'arom',
              'aromatic bond',
              'poly',
              'polymeric bond',
              'delo',
              'delocalized double bond',
              'pi',
              'pi bond' ] ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# CHEMICAL_FORMULA ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'chemical_formula',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEMICAL_FORMULA category would not, in\n               general, be used in a macromolecular CIF. See instead the\n               ENTITY data items.\n\n               Data items in the CHEMICAL_FORMULA category specify the\n               composition and chemical properties of the compound. The formula\n               data items must agree with those that specify the density,\n               unit-cell and Z values.\n\n               The following rules apply to the construction of the data items\n               _chemical_formula.analytical, _chemical_formula.structural and\n               _chemical_formula.sum. For the data item\n               _chemical_formula.moiety, the formula construction is broken up\n               into residues or moieties, i.e. groups of atoms that form a\n               molecular unit or molecular ion. The rules given below apply\n               within each moiety but different requirements apply to the way\n               that moieties are connected (see _chemical_formula.moiety).\n\n               (1) Only recognized element symbols may be used.\n\n               (2) Each element symbol is followed by a \'count\' number. A count\n                  of \'1\' may be omitted.\n\n               (3) A space or parenthesis must separate each cluster of (element\n                  symbol + count).\n\n               (4) Where a group of elements is enclosed in parentheses, the\n                  multiplier for the group must follow the closing parenthesis.\n                  That is, all element and group multipliers are assumed to be\n                  printed as subscripted numbers. (An exception to this rule\n                  exists for _chemical_formula.moiety formulae where pre- and\n                  post-multipliers are permitted for molecular units.)\n\n               (5) Unless the elements are ordered in a manner that corresponds\n                  to their chemical structure, as in\n                  _chemical_formula.structural, the order of the elements within\n                  any group or moiety should be:  C, then H, then the other\n                  elements in alphabetical order of their symbol. This is the\n                  \'Hill\' system used by Chemical Abstracts. This ordering is\n                  used in _chemical_formula.moiety and _chemical_formula.sum.' ],
          [ 'item_', '_category.id', 'chemical_formula' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_chemical_formula.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chemical_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer [(1991).\n                Acta Cryst. C47, 2276-2277].',
              '\n    _chemical_formula.entry_id          \'TOZ\'\n    _chemical_formula.moiety            \'C18 H25 N O3\'\n    _chemical_formula.sum               \'C18 H25 N O3\'\n    _chemical_formula.weight            303.40',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_chemical_formula.analytical',
        [ [ 'item_',
            '_item_description.description',
            '              Formula determined by standard chemical analysis including trace\n               elements. See the CHEMICAL_FORMULA category description for\n               rules for writing chemical formulae. Parentheses are used only\n               for standard uncertainties (estimated standard deviations).' ],
          [ 'item_', '_item.name', '_chemical_formula.analytical' ],
          [ 'item_', '_item.category_id', 'chemical_formula' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_formula_analytical' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'Fe2.45(2)  Ni1.60(3)  S4' ] ] ],
      [ 'save_',
        '_chemical_formula.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_chemical_formula.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_chemical_formula.iupac',
        [ [ 'item_',
            '_item_description.description',
            '              Formula expressed in conformance with IUPAC rules for inorganic\n               and metal-organic compounds where these conflict with the rules\n               for any other CHEMICAL_FORMULA entries. Typically used for\n               formatting a formula in accordance with journal rules. This\n               should appear in the data block in addition to the most\n               appropriate of the other CHEMICAL_FORMULA data names.\n\n               Ref: IUPAC (1990). Nomenclature of Inorganic Chemistry.\n                    Oxford: Blackwell Scientific Publications.' ],
          [ 'item_', '_item.name', '_chemical_formula.iupac' ],
          [ 'item_', '_item.category_id', 'chemical_formula' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_formula_iupac' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '[Co Re (C12 H22 P)2 (C O)6].0.5C H3 O H' ] ] ],
      [ 'save_',
        '_chemical_formula.moiety',
        [ [ 'item_',
            '_item_description.description',
            '              Formula with each discrete bonded residue or ion shown as a\n               separate moiety. See the CHEMICAL_FORMULA category description\n               for rules for writing chemical formulae. In addition to the\n               general formulae requirements, the following rules apply:\n\n                  (1) Moieties are separated by commas \',\'.\n\n                  (2) The order of elements within a moiety follows general rule\n                     (5) in the CHEMICAL_FORMULA category description.\n\n                  (3) Parentheses are not used within moieties but may surround\n                     a moiety. Parentheses may not be nested.\n\n                  (4) Charges should be placed at the end of the moiety. The\n                     charge \'+\' or \'-\' may be preceded by a numerical multiplier\n                     and should be separated from the last (element symbol +\n                     count) by a space. Pre- or post-multipliers may be used for\n                     individual moieties.' ],
          [ 'item_', '_item.name', '_chemical_formula.moiety' ],
          [ 'item_', '_item.category_id', 'chemical_formula' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_formula_moiety' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'C7 H4 Cl Hg N O3 S',
              'C12 H17 N4 O S 1+, C6 H2 N3 O7 1-',
              'C12 H16 N2 O6, 5(H2 O1)',
              '(Cd 2+)3, (C6 N6 Cr 3-)2, 2(H2 O)' ] ] ] ],
      [ 'save_',
        '_chemical_formula.structural',
        [ [ 'item_',
            '_item_description.description',
            '              See the CHEMICAL_FORMULA category description for the rules for\n               writing chemical formulae for inorganics, organometallics, metal\n               complexes etc., in which bonded groups are preserved as\n               discrete entities within parentheses, with post-multipliers as\n               required. The order of the elements should give as much\n               information as possible about the chemical structure.\n               Parentheses may be used and nested as required. This formula\n               should correspond to the structure as actually reported, i.e.\n               trace elements not included in atom-type and atom-site data\n               should not be included in this formula (see also\n               _chemical_formula.analytical).' ],
          [ 'item_', '_item.name', '_chemical_formula.structural' ],
          [ 'item_', '_item.category_id', 'chemical_formula' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_formula_structural' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Ca ((Cl O3)2 O)2 (H2 O)6',
              '(Pt (N H3)2 (C5 H7 N3 O)2) (Cl O4)2' ] ] ] ],
      [ 'save_',
        '_chemical_formula.sum',
        [ [ 'item_',
            '_item_description.description',
            '              See the CHEMICAL_FORMULA category description for the rules\n               for writing chemical formulae in which all discrete bonded\n               residues and ions are summed over the constituent elements,\n               following the ordering given in general rule (5) in the\n               CHEMICAL_FORMULA category description. Parentheses are not\n               normally used.' ],
          [ 'item_', '_item.name', '_chemical_formula.sum' ],
          [ 'item_', '_item.category_id', 'chemical_formula' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_chemical_formula_sum' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'C18 H19 N7 O8 S' ] ] ] ],
      [ 'save_',
        '_chemical_formula.weight',
        [ [ 'item_',
            '_item_description.description',
            '              Formula mass in daltons. This mass should correspond to the\n               formulae given under _chemical_formula.structural,\n               _chemical_formula.moiety or _chemical_formula.sum and,\n               together with the Z value and cell parameters, should\n               yield the density given as _exptl_crystal.density_diffrn.' ],
          [ 'item_', '_item.name', '_chemical_formula.weight' ],
          [ 'item_', '_item.category_id', 'chemical_formula' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_formula_weight' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1.0', '1.0', '1.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_chemical_formula.weight_meas',
        [ [ 'item_',
            '_item_description.description',
            '              Formula mass in daltons measured by a non-diffraction experiment.' ],
          [ 'item_', '_item.name', '_chemical_formula.weight_meas' ],
          [ 'item_', '_item.category_id', 'chemical_formula' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chemical_formula_weight_meas' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1.0', '1.0', '1.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#############' ],
      [ '@comment', '# CITATION ##' ],
      [ '@comment', '#############' ],
      [ 'save_',
        'citation',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CITATION category record details about the\n               literature cited as being relevant to the contents of the data\n               block.' ],
          [ 'item_', '_category.id', 'citation' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_citation.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'citation_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _citation.id\n    _citation.coordinate_linkage\n    _citation.title\n    _citation.country\n    _citation.journal_abbrev\n    _citation.journal_volume\n    _citation.journal_issue\n    _citation.page_first\n    _citation.page_last\n    _citation.year\n    _citation.journal_id_ASTM\n    _citation.journal_id_ISSN\n    _citation.journal_id_CSD\n    _citation.book_title\n    _citation.book_publisher\n    _citation.book_id_ISBN\n    _citation.details\n      primary  yes\n    ; Crystallographic analysis of a complex between human\n      immunodeficiency virus type 1 protease and acetyl-pepstatin\n      at 2.0-Angstroms resolution.\n    ;\n      US  \'J. Biol. Chem.\'  265  .  14209  14219  1990\n      HBCHA3  0021-9258  071  .  .  .\n    ; The publication that directly relates to this coordinate\n      set.\n    ;\n      2  no\n    ; Three-dimensional structure of aspartyl-protease from human\n      immunodeficiency virus HIV-1.\n    ;\n      UK  \'Nature\'  337  .  615  619  1989\n      NATUAS  0028-0836  006  .  .  .\n    ; Determination of the structure of the unliganded enzyme.\n    ;\n      3 no\n    ; Crystallization of the aspartylprotease from human\n      immunodeficiency virus, HIV-1.\n    ;\n      US  \'J. Biol. Chem.\'  264  .  1919  1921  1989\n      HBCHA3  0021-9258  071  .  .  .\n    ; Crystallization of the unliganded enzyme.\n    ;\n      4 no\n    ; Human immunodeficiency virus protease. Bacterial expression\n      and characterization of the purified aspartic protease.\n    ;\n      US  \'J. Biol. Chem.\'  264  .  2307  2312  1989\n      HBCHA3  0021-9258  071  .  .  .\n    ; Expression and purification of the enzyme.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_citation.abstract',
        [ [ 'item_',
            '_item_description.description',
            '              Abstract for the citation. This is used most when the\n               citation is extracted from a bibliographic database that\n               contains full text or abstract information.' ],
          [ 'item_', '_item.name', '_citation.abstract' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_abstract' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_citation.abstract_id_CAS',
        [ [ 'item_',
            '_item_description.description',
            '              The Chemical Abstracts Service (CAS) abstract identifier;\n               relevant for journal articles.' ],
          [ 'item_', '_item.name', '_citation.abstract_id_CAS' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_abstract_id_CAS' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_citation.book_id_ISBN',
        [ [ 'item_',
            '_item_description.description',
            '              The International Standard Book Number (ISBN) code assigned to\n               the book cited; relevant for books or book chapters.' ],
          [ 'item_', '_item.name', '_citation.book_id_ISBN' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_book_id_ISBN' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_citation.book_publisher',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the publisher of the citation; relevant\n               for books or book chapters.' ],
          [ 'item_', '_item.name', '_citation.book_publisher' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_book_publisher' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'John Wiley and Sons' ] ] ],
      [ 'save_',
        '_citation.book_publisher_city',
        [ [ 'item_',
            '_item_description.description',
            '              The location of the publisher of the citation; relevant\n               for books or book chapters.' ],
          [ 'item_', '_item.name', '_citation.book_publisher_city' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_book_publisher_city' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'London' ] ] ],
      [ 'save_',
        '_citation.book_title',
        [ [ 'item_',
            '_item_description.description',
            '              The title of the book in which the citation appeared; relevant\n               for books or book chapters.' ],
          [ 'item_', '_item.name', '_citation.book_title' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_book_title' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_citation.coordinate_linkage',
        [ [ 'item_',
            '_item_description.description',
            '              _citation.coordinate_linkage states whether this citation\n               is concerned with precisely the set of coordinates given in the\n               data block. If, for instance, the publication described the same\n               structure, but the coordinates had undergone further refinement\n               prior to the creation of the data block, the value of this data\n               item would be \'no\'.' ],
          [ 'item_', '_item.name', '_citation.coordinate_linkage' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_coordinate_linkage' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'citation unrelated to current coordinates',
              'n',
              'abbreviation for "no"',
              'yes',
              'citation related to current coordinates',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_citation.country',
        [ [ 'item_',
            '_item_description.description',
            '              The country of publication; relevant for books\n               and book chapters.' ],
          [ 'item_', '_item.name', '_citation.country' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_country' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_citation.database_id_Medline',
        [ [ 'item_',
            '_item_description.description',
            '              Accession number used by Medline to categorize a specific\n               bibliographic entry.' ],
          [ 'item_', '_item.name', '_citation.database_id_Medline' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_database_id_Medline' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '89064067' ] ] ],
      [ 'save_',
        '_citation.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the relationship\n               of the contents of the data block to the literature item cited.' ],
          [ 'item_', '_item.name', '_citation.details' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_special_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '                                 citation relates to this precise\n                                  coordinate set',
              '                                 citation relates to earlier low-resolution\n                                  structure',
              '                                 citation relates to further refinement of\n                                  structure reported in citation 2' ] ] ] ],
      [ 'save_',
        '_citation.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _citation.id must uniquely identify a record in the\n               CITATION list.\n\n               The _citation.id \'primary\' should be used to indicate the\n               citation that the author(s) consider to be the most pertinent to\n               the contents of the data block.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_citation.id',
              'citation',
              'yes',
              '_citation_author.citation_id',
              'citation_author',
              'yes',
              '_citation_editor.citation_id',
              'citation_editor',
              'yes',
              '_software.citation_id',
              'software',
              'no' ] ],
          [ 'item_', '_item_aliases.alias_name', '_citation_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_citation_author.citation_id',
              '_citation.id',
              '_citation_editor.citation_id',
              '_citation.id',
              '_software.citation_id',
              '_citation.id' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'primary', '1', '2' ] ] ] ],
      [ 'save_',
        '_citation.journal_abbrev',
        [ [ 'item_',
            '_item_description.description',
            '              Abbreviated name of the cited journal as given in the\n               Chemical Abstracts Service Source Index.' ],
          [ 'item_', '_item.name', '_citation.journal_abbrev' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_journal_abbrev' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'J. Mol. Biol.' ] ] ],
      [ 'save_',
        '_citation.journal_id_ASTM',
        [ [ 'item_',
            '_item_description.description',
            '              The American Society for Testing and Materials (ASTM) code\n               assigned to the journal cited (also referred to as the CODEN\n               designator of the Chemical Abstracts Service); relevant for\n               journal articles.' ],
          [ 'item_', '_item.name', '_citation.journal_id_ASTM' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_journal_id_ASTM' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_citation.journal_id_CSD',
        [ [ 'item_',
            '_item_description.description',
            '              The Cambridge Structural Database (CSD) code assigned to the\n               journal cited; relevant for journal articles. This is also the\n               system used at the Protein Data Bank (PDB).' ],
          [ 'item_', '_item.name', '_citation.journal_id_CSD' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_journal_id_CSD' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', '0070' ] ] ],
      [ 'save_',
        '_citation.journal_id_ISSN',
        [ [ 'item_',
            '_item_description.description',
            '              The International Standard Serial Number (ISSN) code assigned to\n               the journal cited; relevant for journal articles.' ],
          [ 'item_', '_item.name', '_citation.journal_id_ISSN' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_journal_id_ISSN' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_citation.journal_full',
        [ [ 'item_',
            '_item_description.description',
            '              Full name of the cited journal; relevant for journal articles.' ],
          [ 'item_', '_item.name', '_citation.journal_full' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_journal_full' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'Journal of Molecular Biology' ] ] ],
      [ 'save_',
        '_citation.journal_issue',
        [ [ 'item_',
            '_item_description.description',
            '              Issue number of the journal cited; relevant for journal\n               articles.' ],
          [ 'item_', '_item.name', '_citation.journal_issue' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_journal_issue' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', '2' ] ] ],
      [ 'save_',
        '_citation.journal_volume',
        [ [ 'item_',
            '_item_description.description',
            '              Volume number of the journal cited; relevant for journal\n               articles.' ],
          [ 'item_', '_item.name', '_citation.journal_volume' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_journal_volume' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', '174' ] ] ],
      [ 'save_',
        '_citation.language',
        [ [ 'item_',
            '_item_description.description',
            '              Language in which the cited article is written.' ],
          [ 'item_', '_item.name', '_citation.language' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_language' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'German' ] ] ],
      [ 'save_',
        '_citation.page_first',
        [ [ 'item_',
            '_item_description.description',
            '              The first page of the citation; relevant for journal\n               articles, books and book chapters.' ],
          [ 'item_', '_item.name', '_citation.page_first' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_page_first' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_citation.page_last',
        [ [ 'item_',
            '_item_description.description',
            '              The last page of the citation; relevant for journal\n               articles, books and book chapters.' ],
          [ 'item_', '_item.name', '_citation.page_last' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_page_last' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_citation.title',
        [ [ 'item_',
            '_item_description.description',
            '              The title of the citation; relevant for journal articles, books\n               and book chapters.' ],
          [ 'item_', '_item.name', '_citation.title' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_title' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Structure of diferric duck ovotransferrin\n                                  at 2.35 \\%A resolution.' ] ] ],
      [ 'save_',
        '_citation.year',
        [ [ 'item_',
            '_item_description.description',
            '              The year of the citation; relevant for journal articles, books\n               and book chapters.' ],
          [ 'item_', '_item.name', '_citation.year' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_year' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '1984' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# CITATION_AUTHOR ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'citation_author',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CITATION_AUTHOR category record details\n               about the authors associated with the citations in the\n               CITATION list.' ],
          [ 'item_', '_category.id', 'citation_author' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_citation_author.citation_id', '_citation_author.name' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'citation_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _citation_author.citation_id\n    _citation_author.ordinal\n    _citation_author.name\n      primary  1  \'Fitzgerald, P.M.D.\'\n      primary  2  \'McKeever, B.M.\'\n      primary  3  \'Van Middlesworth, J.F.\'\n      primary  4  \'Springer, J.P.\'\n      primary  5  \'Heimbach, J.C.\'\n      primary  6  \'Leu, C.-T.\'\n      primary  7  \'Herber, W.K.\'\n      primary  8  \'Dixon, R.A.F.\'\n      primary  9  \'Darke, P.L.\'\n      2        1  \'Navia, M.A.\'\n      2        2  \'Fitzgerald, P.M.D.\'\n      2        3  \'McKeever, B.M.\'\n      2        4  \'Leu, C.-T.\'\n      2        5  \'Heimbach, J.C.\'\n      2        6  \'Herber, W.K.\'\n      2        7  \'Sigal, I.S.\'\n      2        8  \'Darke, P.L.\'\n      2        9  \'Springer, J.P.\'\n      3        1  \'McKeever, B.M.\'\n      3        2  \'Navia, M.A.\'\n      3        3  \'Fitzgerald, P.M.D.\'\n      3        4  \'Springer, J.P.\'\n      3        5  \'Leu, C.-T.\'\n      3        6  \'Heimbach, J.C.\'\n      3        7  \'Herber, W.K.\'\n      3        8  \'Sigal, I.S.\'\n      3        9  \'Darke, P.L.\'\n      4        1  \'Darke, P.L.\'\n      4        2  \'Leu, C.-T.\'\n      4        3  \'Davis, L.J.\'\n      4        4  \'Heimbach, J.C.\'\n      4        5  \'Diehl, R.E.\'\n      4        6  \'Hill, W.S.\'\n      4        7  \'Dixon, R.A.F.\'\n      4        8  \'Sigal, I.S.\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_citation_author.citation_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _citation.id in the CITATION\n               category.' ],
          [ 'item_', '_item.name', '_citation_author.citation_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_author_citation_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_citation_author.name',
        [ [ 'item_',
            '_item_description.description',
            '              Name of an author of the citation; relevant for journal\n               articles, books and book chapters.\n\n               The family name(s), followed by a comma and including any\n               dynastic components, precedes the first name(s) or initial(s).' ],
          [ 'item_', '_item.name', '_citation_author.name' ],
          [ 'item_', '_item.category_id', 'citation_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_author_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Bleary, Percival R.',
              'O\'Neil, F.K.',
              'Van den Bossche, G.',
              'Yang, D.-L.',
              'Simonov, Yu.A' ] ] ] ],
      [ 'save_',
        '_citation_author.ordinal',
        [ [ 'item_',
            '_item_description.description',
            '              This data item defines the order of the author\'s name in the\n               list of authors of a citation.' ],
          [ 'item_', '_item.name', '_citation_author.ordinal' ],
          [ 'item_', '_item.category_id', 'citation_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_author_ordinal' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# CITATION_EDITOR ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'citation_editor',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CITATION_EDITOR category record details\n               about the editors associated with the books or book chapters\n               cited in the CITATION list.' ],
          [ 'item_', '_category.id', 'citation_editor' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_citation_editor.citation_id', '_citation_editor.name' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'citation_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - hypothetical example.',
              '\n    loop_\n    _citation_editor.citation_id\n    _citation_editor.name\n      5        \'McKeever, B.M.\'\n      5        \'Navia, M.A.\'\n      5        \'Fitzgerald, P.M.D.\'\n      5        \'Springer, J.P.\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_citation_editor.citation_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _citation.id in the CITATION\n               category.' ],
          [ 'item_', '_item.name', '_citation_editor.citation_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_editor_citation_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_citation_editor.name',
        [ [ 'item_',
            '_item_description.description',
            '              Names of an editor of the citation; relevant for books and\n               book chapters.\n\n               The family name(s), followed by a comma and including any\n               dynastic components, precedes the first name(s) or initial(s).' ],
          [ 'item_', '_item.name', '_citation_editor.name' ],
          [ 'item_', '_item.category_id', 'citation_editor' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_citation_editor_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Bleary, Percival R.',
              'O\'Neil, F.K.',
              'Van den Bossche, G.',
              'Yang, D.-L.',
              'Simonov, Yu.A' ] ] ] ],
      [ 'save_',
        '_citation_editor.ordinal',
        [ [ 'item_',
            '_item_description.description',
            '              This data item defines the order of the editor\'s name in the\n               list of editors of a citation.' ],
          [ 'item_', '_item.name', '_citation_editor.ordinal' ],
          [ 'item_', '_item.category_id', 'citation_editor' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation_editor_ordinal' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '##############' ],
      [ '@comment', '# COMPUTING ##' ],
      [ '@comment', '##############' ],
      [ 'save_',
        'computing',
        [ [ 'item_',
            '_category.description',
            '              Data items in the COMPUTING category record details about the\n               computer programs used in the crystal structure analysis.\n\n               Data items in this category would not, in general, be used in\n               a macromolecular CIF.  The category SOFTWARE, which allows\n               a more detailed description of computer programs and\n               their attributes to be given, would be used instead.' ],
          [ 'item_', '_category.id', 'computing' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_computing.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'computing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - Rodr\\\'iguez-Romera, Ruiz-P\\\'erez & Solans [Acta\n                Cryst. (1996), C52, 1415-1417].',
              '\n    _computing.data_collection      \'CAD-4 (Enraf-Nonius, 1989)\'\n    _computing.cell_refinement      \'CAD-4 (Enraf-Nonius, 1989)\'\n    _computing.data_reduction       \'CFEO (Solans, 1978)\'\n    _computing.structure_solution   \'SHELXS86 (Sheldrick, 1990)\'\n    _computing.structure_refinement \'SHELXL93 (Sheldrick, 1993)\'\n    _computing.molecular_graphics   \'ORTEPII (Johnson, 1976)\'\n    _computing.publication_material \'PARST (Nardelli, 1983)\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_computing.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_computing.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_computing.cell_refinement',
        [ [ 'item_',
            '_item_description.description',
            '              Software used for cell refinement.\n\n               Give the program or package name and a brief reference.' ],
          [ 'item_', '_item.name', '_computing.cell_refinement' ],
          [ 'item_', '_item.category_id', 'computing' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_computing_cell_refinement' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'CAD4 (Enraf-Nonius, 1989)' ] ] ],
      [ 'save_',
        '_computing.data_collection',
        [ [ 'item_',
            '_item_description.description',
            '              Software used for data collection.\n\n               Give the program or package name and a brief reference.' ],
          [ 'item_', '_item.name', '_computing.data_collection' ],
          [ 'item_', '_item.category_id', 'computing' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_computing_data_collection' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'CAD4 (Enraf-Nonius, 1989)' ] ] ],
      [ 'save_',
        '_computing.data_reduction',
        [ [ 'item_',
            '_item_description.description',
            '              Software used for data reduction.\n\n               Give the program or package name and a brief reference.' ],
          [ 'item_', '_item.name', '_computing.data_reduction' ],
          [ 'item_', '_item.category_id', 'computing' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_computing_data_reduction' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'DIFDAT, SORTRF, ADDREF (Hall & Stewart, 1990)' ] ] ],
      [ 'save_',
        '_computing.molecular_graphics',
        [ [ 'item_',
            '_item_description.description',
            '              Software used for molecular graphics.\n\n               Give the program or package name and a brief reference.' ],
          [ 'item_', '_item.name', '_computing.molecular_graphics' ],
          [ 'item_', '_item.category_id', 'computing' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_computing_molecular_graphics' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'FRODO (Jones, 1986), ORTEP (Johnson, 1965)' ] ] ],
      [ 'save_',
        '_computing.publication_material',
        [ [ 'item_',
            '_item_description.description',
            '              Software used for generating material for publication.\n\n               Give the program or package name and a brief reference.' ],
          [ 'item_', '_item.name', '_computing.publication_material' ],
          [ 'item_', '_item.category_id', 'computing' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_computing_publication_material' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_computing.structure_refinement',
        [ [ 'item_',
            '_item_description.description',
            '              Software used for refinement of the structure.\n\n               Give the program or package name and a brief reference.' ],
          [ 'item_', '_item.name', '_computing.structure_refinement' ],
          [ 'item_', '_item.category_id', 'computing' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_computing_structure_refinement' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'SHELX85 (Sheldrick, 1985)', 'X-PLOR (Brunger, 1992)' ] ] ] ],
      [ 'save_',
        '_computing.structure_solution',
        [ [ 'item_',
            '_item_description.description',
            '              Software used for solution of the structure.\n\n               Give the program or package name and a brief reference.' ],
          [ 'item_', '_item.name', '_computing.structure_solution' ],
          [ 'item_', '_item.category_id', 'computing' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_computing_structure_solution' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'SHELX85 (Sheldrick, 1985)' ] ] ],
      [ '@comment', '#############' ],
      [ '@comment', '# DATABASE ##' ],
      [ '@comment', '#############' ],
      [ 'save_',
        'database',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DATABASE category have been superseded by\n               data items in the DATABASE_2 category. They are included\n               here only for compliance with older CIFs.' ],
          [ 'item_', '_category.id', 'database' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_database.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'compliance_group' ] ] ] ],
      [ 'save_',
        '_database.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_database.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', 'save__database.code_CAS' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              The code assigned by Chemical Abstracts.' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_database.code_CAS\'' ],
      [ '@comment', '    _item.category_id             database' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '    _item_aliases.alias_name    \'_database_code_CAS\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_related.related_name' ],
      [ '@comment',
        '    _item_related.function_code \'_database_2.database_id\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment',
        '                                \'_database_2.database_code\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment', '    _item_type.code               line' ],
      [ '@comment', '     save_' ],
      [ '@comment', 'save__database.code_CSD' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              The code assigned by the Cambridge Structural Database' ],
      [ '@comment',
        '               (organic and metal-organic compounds).' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_database.code_CSD\'' ],
      [ '@comment', '    _item.category_id             database' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '    _item_aliases.alias_name    \'_database_code_CSD\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_related.related_name' ],
      [ '@comment',
        '    _item_related.function_code \'_database_2.database_id\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment',
        '                                \'_database_2.database_code\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment', '    _item_type.code               line' ],
      [ '@comment', '     save_' ],
      [ '@comment', 'save__database.code_ICSD' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              The code assigned by the Inorganic Crystal Structure Database.' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_database.code_ICSD\'' ],
      [ '@comment', '    _item.category_id             database' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '    _item_aliases.alias_name    \'_database_code_ICSD\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_related.related_name' ],
      [ '@comment',
        '    _item_related.function_code \'_database_2.database_id\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment',
        '                                \'_database_2.database_code\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment', '    _item_type.code               line' ],
      [ '@comment', '     save_' ],
      [ '@comment', 'save__database.code_MDF' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              The code assigned by the Metals Data File (metal structures).' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_database.code_MDF\'' ],
      [ '@comment', '    _item.category_id             database' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '    _item_aliases.alias_name    \'_database_code_MDF\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_related.related_name' ],
      [ '@comment',
        '    _item_related.function_code \'_database_2.database_id\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment',
        '                                \'_database_2.database_code\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment', '    _item_type.code               line' ],
      [ '@comment', '     save_' ],
      [ '@comment', 'save__database.code_NBS' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              The code assigned by the NBS (NIST) Crystal Data Database' ],
      [ '@comment', '               (lattice parameters).' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_database.code_NBS\'' ],
      [ '@comment', '    _item.category_id             database' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '    _item_aliases.alias_name    \'_database_code_NBS\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_related.related_name' ],
      [ '@comment',
        '    _item_related.function_code \'_database_2.database_id\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment',
        '                                \'_database_2.database_code\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment', '    _item_type.code               line' ],
      [ '@comment', '     save_' ],
      [ '@comment', 'save__database.code_PDF' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              The code assigned by the Powder Diffraction File (JCPDS/ICDD).' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_database.code_PDF\'' ],
      [ '@comment', '    _item.category_id             database' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '    _item_aliases.alias_name    \'_database_code_PDF\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_related.related_name' ],
      [ '@comment',
        '    _item_related.function_code \'_database_2.database_id\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment',
        '                                \'_database_2.database_code\'' ],
      [ '@comment', '                                  replacedby' ],
      [ '@comment', '    _item_type.code               line' ],
      [ '@comment', '     save_' ],
      [ 'save_',
        '_database.journal_ASTM',
        [ [ 'item_',
            '_item_description.description',
            '              The ASTM CODEN designator for a journal as given in the Chemical\n               Source List maintained by the Chemical Abstracts Service.' ],
          [ 'item_', '_item.name', '_database.journal_ASTM' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_database_journal_ASTM' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_database.journal_CSD',
        [ [ 'item_',
            '_item_description.description',
            '              The journal code used in the Cambridge Structural Database.' ],
          [ 'item_', '_item.name', '_database.journal_CSD' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_database_journal_CSD' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '###############' ],
      [ '@comment', '# DATABASE_2 ##' ],
      [ '@comment', '###############' ],
      [ 'save_',
        'database_2',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DATABASE_2 category record details about the\n               database identifiers of the data block.\n\n               These data items are assigned by database managers and should\n               only appear in a data block if they originate from that source.\n\n               The name of this category, DATABASE_2, arose because the\n               category name DATABASE was already in use in the core CIF\n               dictionary, but was used differently from the way it needed\n               to be used in the mmCIF dictionary. Since CIF data names\n               cannot be changed once they have been adopted, a new category\n               had to be created.' ],
          [ 'item_', '_category.id', 'database_2' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_database_2.database_id', '_database_2.database_code' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _database_2.database_id                \'PDB\'\n    _database_2.database_code              \'5HVP\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_database_2.database_id',
        [ [ 'item_',
            '_item_description.description',
            '              An abbreviation that identifies the database.' ],
          [ 'item_', '_item.name', '_database_2.database_id' ],
          [ 'item_', '_item.category_id', 'database_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_database.code_CAS',
              'replaces',
              '_database.code_CSD',
              'replaces',
              '_database.code_ICSD',
              'replaces',
              '_database.code_MDF',
              'replaces',
              '_database.code_NBS',
              'replaces',
              '_database.code_PDF',
              'replaces' ] ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'CAS',
              '                                 Chemical Abstracts',
              'CSD',
              '                                 Cambridge Structural Database\n                                  (organic and metal-organic compounds)',
              'ICSD',
              '                                 Inorganic Crystal Structure Database',
              'MDF',
              '                                 Metals Data File (metal structures)',
              'NDB',
              '                                 Nucleic Acid Database',
              'NBS',
              '                                 NBS (NIST) Crystal Data Database\n                                  (lattice parameters)',
              'PDB',
              '                                 Protein Data Bank',
              'PDF',
              '                                 Powder Diffraction File (JCPDS/ICDD)',
              'RCSB',
              '                         Research Collaboratory for Structural Bioinformatics',
              'EBI',
              '                                 European Bioinformatics Institute' ] ] ] ],
      [ 'save_',
        '_database_2.database_code',
        [ [ 'item_',
            '_item_description.description',
            '              The code assigned by the database identified in\n               _database_2.database_id.' ],
          [ 'item_', '_item.name', '_database_2.database_code' ],
          [ 'item_', '_item.category_id', 'database_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_database.code_CAS',
              'replaces',
              '_database.code_CSD',
              'replaces',
              '_database.code_ICSD',
              'replaces',
              '_database.code_MDF',
              'replaces',
              '_database.code_NBS',
              'replaces',
              '_database.code_PDF',
              'replaces' ] ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1ABC', 'ABCDEF' ] ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# DATABASE_PDB_CAVEAT ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'database_PDB_caveat',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DATABASE_PDB_CAVEAT category record details\n               about features of the data block flagged as \'caveats\' by the\n               Protein Data Bank (PDB).\n\n               These data items are included only for consistency with PDB\n               format files. They should appear in a data block only if that\n               data block was created by reformatting a PDB format file.' ],
          [ 'item_', '_category.id', 'database_PDB_caveat' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_database_PDB_caveat.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdb_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - hypothetical example.',
              '\n    loop_\n    _database_PDB_caveat.id\n    _database_PDB_caveat.text\n    1\n    ; THE CRYSTAL TRANSFORMATION IS IN ERROR BUT IS\n    ;\n    2\n    ; UNCORRECTABLE AT THIS TIME\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_database_PDB_caveat.id',
        [ [ 'item_',
            '_item_description.description',
            '              A unique identifier for the PDB caveat record.' ],
          [ 'item_', '_item.name', '_database_PDB_caveat.id' ],
          [ 'item_', '_item.category_id', 'database_PDB_caveat' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_database_PDB_caveat.text',
        [ [ 'item_',
            '_item_description.description',
            '              The full text of the PDB caveat record.' ],
          [ 'item_', '_item.name', '_database_PDB_caveat.text' ],
          [ 'item_', '_item.category_id', 'database_PDB_caveat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# DATABASE_PDB_MATRIX ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'database_PDB_matrix',
        [ [ 'item_',
            '_category.description',
            '              The DATABASE_PDB_MATRIX category provides placeholders for\n               transformation matrices and vectors used by the Protein Data\n               Bank (PDB).\n\n               These data items are included only for consistency with older\n               PDB format files. They should appear in a data block only if\n               that data block was created by reformatting a PDB format file.' ],
          [ 'item_', '_category.id', 'database_PDB_matrix' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_database_PDB_matrix.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdb_group' ] ] ] ],
      [ 'save_',
        '_database_PDB_matrix.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the PDB ORIGX matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx[1][1]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the PDB ORIGX matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx[1][2]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the PDB ORIGX matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx[1][3]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx[2][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][1] element of the PDB ORIGX matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx[2][1]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the PDB ORIGX matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx[2][2]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the PDB ORIGX matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx[2][3]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx[3][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][1] element of the PDB ORIGX matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx[3][1]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx[3][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][2] element of the PDB ORIGX matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx[3][2]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the PDB ORIGX matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx[3][3]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx_vector[1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1] element of the PDB ORIGX vector.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx_vector[1]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx_vector[2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2] element of the PDB ORIGX vector.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx_vector[2]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.origx_vector[3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3] element of the PDB ORIGX vector.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.origx_vector[3]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the PDB SCALE matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale[1][1]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the PDB SCALE matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale[1][2]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the PDB SCALE matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale[1][3]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale[2][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][1] element of the PDB SCALE matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale[2][1]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the PDB SCALE matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale[2][2]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the PDB SCALE matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale[2][3]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale[3][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][1] element of the PDB SCALE matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale[3][1]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale[3][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][2] element of the PDB SCALE matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale[3][2]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the PDB SCALE matrix.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale[3][3]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1.0' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale_vector[1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1] element of the PDB SCALE vector.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale_vector[1]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale_vector[2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2] element of the PDB SCALE vector.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale_vector[2]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_matrix.scale_vector[3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3] element of the PDB SCALE vector.' ],
          [ 'item_', '_item.name', '_database_PDB_matrix.scale_vector[3]' ],
          [ 'item_', '_item.category_id', 'database_PDB_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# DATABASE_PDB_REMARK ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'database_PDB_remark',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DATABASE_PDB_REMARK category record details\n               about the data block as archived by the Protein Data Bank (PDB).\n\n               Some data appearing in PDB REMARK records can be\n               algorithmically extracted into the appropriate data items\n               in the data block.\n\n               These data items are included only for consistency with older\n               PDB format files. They should appear in a data block only if\n               that data block was created by reformatting a PDB format file.' ],
          [ 'item_', '_category.id', 'database_PDB_remark' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_database_PDB_remark.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdb_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _database_PDB_remark.id\n    _database_PDB_remark.text\n    3\n    ; REFINEMENT. BY THE RESTRAINED LEAST-SQUARES PROCEDURE OF J.\n      KONNERT AND W. HENDRICKSON (PROGRAM *PROLSQ*).  THE R\n      VALUE IS 0.176 FOR 12901 REFLECTIONS IN THE RESOLUTION\n      RANGE 8.0 TO 2.0 ANGSTROMS WITH I .GT. SIGMA(I).\n\n      RMS DEVIATIONS FROM IDEAL VALUES (THE VALUES OF\n      SIGMA, IN PARENTHESES, ARE THE INPUT ESTIMATED\n      STANDARD DEVIATIONS THAT DETERMINE THE RELATIVE\n      WEIGHTS OF THE CORRESPONDING RESTRAINTS)\n      DISTANCE RESTRAINTS (ANGSTROMS)\n      BOND DISTANCE                            0.018(0.020)\n      ANGLE DISTANCE                           0.038(0.030)\n      PLANAR 1-4 DISTANCE                      0.043(0.040)\n      PLANE RESTRAINT (ANGSTROMS)                0.015(0.020)\n      CHIRAL-CENTER RESTRAINT (ANGSTROMS**3)     0.177(0.150)\n      NON-BONDED CONTACT RESTRAINTS (ANGSTROMS)\n      SINGLE TORSION CONTACT                   0.216(0.500)\n      MULTIPLE TORSION CONTACT                 0.207(0.500)\n      POSSIBLE HYDROGEN BOND                   0.245(0.500)\n      CONFORMATIONAL TORSION ANGLE RESTRAINT (DEGREES)\n      PLANAR (OMEGA)                             2.6(3.0)\n      STAGGERED                                 17.4(15.0)\n      ORTHONORMAL                               18.1(20.0)\n    ;\n    4\n    ; THE TWO CHAINS OF THE DIMERIC ENZYME HAS BEEN ASSIGNED THE\n      THE CHAIN INDICATORS *A* AND *B*.\n    ;\n    #  - - - - data truncated for brevity - - - -',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_database_PDB_remark.id',
        [ [ 'item_',
            '_item_description.description',
            '              A unique identifier for the PDB remark record.' ],
          [ 'item_', '_item.name', '_database_PDB_remark.id' ],
          [ 'item_', '_item.category_id', 'database_PDB_remark' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_database_PDB_remark.text',
        [ [ 'item_',
            '_item_description.description',
            '              The full text of the PDB remark record.' ],
          [ 'item_', '_item.name', '_database_PDB_remark.text' ],
          [ 'item_', '_item.category_id', 'database_PDB_remark' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# DATABASE_PDB_REV ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'database_PDB_rev',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DATABASE_PDB_REV category record details\n               about the history of the data block as archived by the Protein\n               Data Bank (PDB).\n\n               These data items are assigned by the PDB database managers and\n               should only appear in a data block if they originate from that\n               source.' ],
          [ 'item_', '_category.id', 'database_PDB_rev' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_database_PDB_rev.num' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdb_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _database_PDB_rev.num\n    _database_PDB_rev.author_name\n    _database_PDB_rev.date\n    _database_PDB_rev.date_original\n    _database_PDB_rev.status\n    _database_PDB_rev.mod_type\n      1  \'Fitzgerald, Paula M.D\' 1991-10-15  1990-04-30\n         \'full release\'  0',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_database_PDB_rev.author_name',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the person responsible for submitting this revision\n               to the PDB.\n\n               The family name(s) followed by a comma precedes the first\n               name(s) or initial(s).' ],
          [ 'item_', '_item.name', '_database_PDB_rev.author_name' ],
          [ 'item_', '_item.category_id', 'database_PDB_rev' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Bleary, Percival R.',
              'O\'Neil, F.K.',
              'Van den Bossche, G.',
              'Yang, D.-L.',
              'Simonov, Yu.A' ] ] ] ],
      [ 'save_',
        '_database_PDB_rev.date',
        [ [ 'item_',
            '_item_description.description',
            '              Date the PDB revision took place. Taken from the REVDAT record.' ],
          [ 'item_', '_item.name', '_database_PDB_rev.date' ],
          [ 'item_', '_item.category_id', 'database_PDB_rev' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_database_PDB_rev.date_original',
        [ [ 'item_',
            '_item_description.description',
            '              Date the entry first entered the PDB database in the form\n               yyyy-mm-dd. Taken from the PDB HEADER record.' ],
          [ 'item_', '_item.name', '_database_PDB_rev.date_original' ],
          [ 'item_', '_item.category_id', 'database_PDB_rev' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ],
          [ 'item_', '_item_examples.case', '1980-08-21' ] ] ],
      [ 'save_',
        '_database_PDB_rev.mod_type',
        [ [ 'item_',
            '_item_description.description',
            '              Taken from the REVDAT record. Refer to the Protein Data Bank\n               format description at\n            http://www.rcsb.org/pdb/docs/format/pdbguide2.2/guide2.2_frame.html\n               for details.' ],
          [ 'item_', '_item.name', '_database_PDB_rev.mod_type' ],
          [ 'item_', '_item.category_id', 'database_PDB_rev' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '0',
              'initial entry',
              '1',
              'all other types of modification',
              '2',
              'modifications to CONECT records',
              '3',
              '                                     modifications affecting the coordinates\n                                      or their transforms (CRYST1, ORIGX, SCALE,\n                                      MTRIX, TVECT, ATOM, HETATM, SIGATM\n                                      records)',
              '4',
              '\n                                   layer 1 to layer 2 revision which may affect\n                                   all record types',
              '5',
              'data uniformity processing' ] ] ] ],
      [ 'save_',
        '_database_PDB_rev.num',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _database_PDB_rev.num must uniquely and\n               sequentially identify a record in the DATABASE_PDB_REV list.\n\n               Note that this item must be a number and that modification\n               numbers are assigned in increasing numerical order.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_database_PDB_rev.num',
              'database_PDB_rev',
              'yes',
              '_database_PDB_rev_record.rev_num',
              'database_PDB_rev_record',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_database_PDB_rev_record.rev_num', '_database_PDB_rev.num' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_database_PDB_rev.replaced_by',
        [ [ 'item_',
            '_item_description.description',
            '              The PDB code for a subsequent PDB entry that replaced the\n               PDB file corresponding to this data block.' ],
          [ 'item_', '_item.name', '_database_PDB_rev.replaced_by' ],
          [ 'item_', '_item.category_id', 'database_PDB_rev' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_database_PDB_rev.replaces',
        [ [ 'item_',
            '_item_description.description',
            '              The PDB code for a previous PDB entry that was replaced by\n               the PDB file corresponding to this data block.' ],
          [ 'item_', '_item.name', '_database_PDB_rev.replaces' ],
          [ 'item_', '_item.category_id', 'database_PDB_rev' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_database_PDB_rev.status',
        [ [ 'item_',
            '_item_description.description',
            '              The status of this revision.' ],
          [ 'item_', '_item.name', '_database_PDB_rev.status' ],
          [ 'item_', '_item.category_id', 'database_PDB_rev' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uline' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'in preparation', 'prerelease', 'full release', 'obsolete' ] ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', '# DATABASE_PDB_REV_RECORD ##' ],
      [ '@comment', '############################' ],
      [ 'save_',
        'database_PDB_rev_record',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DATABASE_PDB_REV_RECORD category record\n               details about specific record types that were changed in a\n               given revision of a PDB entry.\n\n               These data items are assigned by the PDB database managers and\n               should only appear in a data block if they originate from that\n               source.' ],
          [ 'item_', '_category.id', 'database_PDB_rev_record' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_database_PDB_rev_record.rev_num',
              '_database_PDB_rev_record.type' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdb_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - hypothetical example.',
              '\n    loop_\n    _database_PDB_rev_record.rev_num\n    _database_PDB_rev_record.type\n    _database_PDB_rev_record.details\n      1  CONECT\n    ; Error fix - incorrect connection between\n      atoms 2312 and 2317\n    ;\n      2  MATRIX  \'For consistency with 1995-08-04 style-guide\'\n      3  ORIGX   \'Based on new data from author\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_database_PDB_rev_record.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the revision of records in\n               this PDB entry.' ],
          [ 'item_', '_item.name', '_database_PDB_rev_record.details' ],
          [ 'item_', '_item.category_id', 'database_PDB_rev_record' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Based on new data from author',
              'For consistency with 1995-08-04 style-guide',
              'For consistency with structural class' ] ] ] ],
      [ 'save_',
        '_database_PDB_rev_record.rev_num',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _database_PDB_rev.num in the\n               DATABASE_PDB_REV category.' ],
          [ 'item_', '_item.name', '_database_PDB_rev_record.rev_num' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_database_PDB_rev_record.type',
        [ [ 'item_',
            '_item_description.description',
            '              The types of records that were changed in this revision to a\n               PDB entry.' ],
          [ 'item_', '_item.name', '_database_PDB_rev_record.type' ],
          [ 'item_', '_item.category_id', 'database_PDB_rev_record' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'CRYST1', 'SCALE', 'MTRIX', 'ATOM', 'HETATM' ] ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# DATABASE_PDB_TVECT ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'database_PDB_tvect',
        [ [ 'item_',
            '_category.description',
            '              The DATABASE_PDB_TVECT category provides placeholders for\n               the TVECT matrices and vectors used by the Protein Data\n               Bank (PDB).\n\n               These data items are included only for consistency with older\n               PDB format files. They should appear in a data block only if\n               the data block was created by reformatting a PDB format file.' ],
          [ 'item_', '_category.id', 'database_PDB_tvect' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_database_PDB_tvect.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdb_group' ] ] ] ],
      [ 'save_',
        '_database_PDB_tvect.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of this TVECT.' ],
          [ 'item_', '_item.name', '_database_PDB_tvect.details' ],
          [ 'item_', '_item.category_id', 'database_PDB_tvect' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_database_PDB_tvect.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _database_PDB_tvect.id must uniquely identify a\n               record in the DATABASE_PDB_TVECT list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_database_PDB_tvect.id' ],
          [ 'item_', '_item.category_id', 'database_PDB_tvect' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_database_PDB_tvect.vector[1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1] element of the PDB TVECT vector.' ],
          [ 'item_', '_item.name', '_database_PDB_tvect.vector[1]' ],
          [ 'item_', '_item.category_id', 'database_PDB_tvect' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_tvect.vector[2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2] element of the PDB TVECT vector.' ],
          [ 'item_', '_item.name', '_database_PDB_tvect.vector[2]' ],
          [ 'item_', '_item.category_id', 'database_PDB_tvect' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_database_PDB_tvect.vector[3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3] element of the PDB TVECT vector.' ],
          [ 'item_', '_item.name', '_database_PDB_tvect.vector[3]' ],
          [ 'item_', '_item.category_id', 'database_PDB_tvect' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '###########' ],
      [ '@comment', '# DIFFRN ##' ],
      [ '@comment', '###########' ],
      [ 'save_',
        'diffrn',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN category record details about the\n               diffraction data and their measurement.' ],
          [ 'item_', '_category.id', 'diffrn' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_diffrn.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _diffrn.id                            \'Set1\'\n    _diffrn.ambient_temp                   293(3)\n    _diffrn.ambient_environment\n    ; Mother liquor from the reservoir of the vapor diffusion\n      experiment, mounted in room air\n    ;\n    _diffrn.crystal_support\n    ; 0.7 mm glass capillary, sealed with dental wax\n    ;\n    _diffrn.crystal_treatment\n    ; Equilibrated in rotating anode radiation enclosure for\n      18 hours prior to beginning of data collection\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer [(1991).\n                Acta Cryst. C47, 2276-2277].',
              '\n    _diffrn.id                            \'d1\'\n    _diffrn.details\n    ; \\q scan width (1.0 + 0.14tan\\q)\\%, \\q scan rate 1.2\\% per\n       min. Background counts for 5 sec on each side every scan.\n    ;\n\n    _diffrn.ambient_temp                293',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn.ambient_environment',
        [ [ 'item_',
            '_item_description.description',
            '              The gas or liquid surrounding the sample, if not air.' ],
          [ 'item_', '_item.name', '_diffrn.ambient_environment' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_ambient_environment' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', 'save__diffrn.ambient_pressure' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              The mean pressure in kilopascals at which the intensities were' ],
      [ '@comment', '               measured.' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_diffrn.ambient_pressure\'' ],
      [ '@comment', '    _item.category_id             diffrn' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '    _item_aliases.alias_name    \'_diffrn_ambient_pressure\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_range.maximum' ],
      [ '@comment', '    _item_range.minimum            .    0.0' ],
      [ '@comment', '                                  0.0   0.0' ],
      [ '@comment',
        '    _item_related.related_name  \'_diffrn.ambient_pressure_esd\'' ],
      [ '@comment',
        '    _item_related.function_code   associated_esd' ],
      [ '@comment', '    _item_type.code               float' ],
      [ '@comment', '    _item_type_conditions.code    esd' ],
      [ '@comment', '    _item_units.code              kilopascals' ],
      [ '@comment', '     save_' ],
      [ '@comment', 'save__diffrn.ambient_pressure_esd' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              The standard uncertainty (estimated standard deviation)' ],
      [ '@comment', '               of _diffrn.ambient_pressure.' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_diffrn.ambient_pressure_esd\'' ],
      [ '@comment', '    _item.category_id             diffrn' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment', '    _item_default.value           0.0' ],
      [ '@comment',
        '    _item_related.related_name  \'_diffrn.ambient_pressure\'' ],
      [ '@comment',
        '    _item_related.function_code   associated_value' ],
      [ '@comment', '    _item_type.code               float' ],
      [ '@comment', '    _item_units.code              kilopascals' ],
      [ '@comment', '     save_' ],
      [ 'save_',
        '_diffrn.ambient_temp',
        [ [ 'item_',
            '_item_description.description',
            '              The mean temperature in kelvins at which the intensities were\n               measured.' ],
          [ 'item_', '_item.name', '_diffrn.ambient_temp' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_ambient_temperature' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_diffrn.ambient_temp_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ 'save_',
        '_diffrn.ambient_temp_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of temperature control during\n               data collection.' ],
          [ 'item_', '_item.name', '_diffrn.ambient_temp_details' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_diffrn.ambient_temp_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _diffrn.ambient_temp.' ],
          [ 'item_', '_item.name', '_diffrn.ambient_temp_esd' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_related.related_name', '_diffrn.ambient_temp' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ 'save_',
        '_diffrn.crystal_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _exptl_crystal.id in the\n               EXPTL_CRYSTAL category.' ],
          [ 'item_', '_item.name', '_diffrn.crystal_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_crystal_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_diffrn.crystal_support',
        [ [ 'item_',
            '_item_description.description',
            '              The physical device used to support the crystal during data\n               collection.' ],
          [ 'item_', '_item.name', '_diffrn.crystal_support' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'glass capillary', 'quartz capillary', 'fiber', 'metal loop' ] ] ] ],
      [ 'save_',
        '_diffrn.crystal_treatment',
        [ [ 'item_',
            '_item_description.description',
            '              Remarks about how the crystal was treated prior to intensity\n               measurement. Particularly relevant when intensities were\n               measured at low temperature.' ],
          [ 'item_', '_item.name', '_diffrn.crystal_treatment' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_crystal_treatment' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'equilibrated in hutch for 24 hours',
              'flash frozen in liquid nitrogen',
              'slow cooled with direct air stream' ] ] ] ],
      [ 'save_',
        '_diffrn.details',
        [ [ 'item_',
            '_item_description.description',
            '              Special details of the diffraction measurement process. Should\n               include information about source instability, crystal motion,\n               degradation and so on.' ],
          [ 'item_', '_item.name', '_diffrn.details' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_special_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_diffrn.id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item uniquely identifies a set of diffraction\n               data.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_diffrn.id',
              'diffrn',
              'yes',
              '_diffrn_detector.diffrn_id',
              'diffrn_detector',
              'yes',
              '_diffrn_measurement.diffrn_id',
              'diffrn_measurement',
              'yes',
              '_diffrn_orient_matrix.diffrn_id',
              'diffrn_orient_matrix',
              'yes',
              '_diffrn_orient_refln.diffrn_id',
              'diffrn_orient_refln',
              'yes',
              '_diffrn_radiation.diffrn_id',
              'diffrn_radiation',
              'yes',
              '_diffrn_refln.diffrn_id',
              'diffrn_refln',
              'yes',
              '_diffrn_reflns.diffrn_id',
              'diffrn_reflns',
              'yes',
              '_diffrn_source.diffrn_id',
              'diffrn_source',
              'yes',
              '_diffrn_standard_refln.diffrn_id',
              'diffrn_standard_refln',
              'yes',
              '_diffrn_standards.diffrn_id',
              'diffrn_standards',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_diffrn_detector.diffrn_id',
              '_diffrn.id',
              '_diffrn_measurement.diffrn_id',
              '_diffrn.id',
              '_diffrn_orient_matrix.diffrn_id',
              '_diffrn.id',
              '_diffrn_orient_refln.diffrn_id',
              '_diffrn.id',
              '_diffrn_radiation.diffrn_id',
              '_diffrn.id',
              '_diffrn_refln.diffrn_id',
              '_diffrn.id',
              '_diffrn_reflns.diffrn_id',
              '_diffrn.id',
              '_diffrn_source.diffrn_id',
              '_diffrn.id',
              '_diffrn_standard_refln.diffrn_id',
              '_diffrn.id',
              '_diffrn_standards.diffrn_id',
              '_diffrn.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ '@comment', '######################' ],
      [ '@comment', '# DIFFRN_ATTENUATOR ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'diffrn_attenuator',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_ATTENUATOR category record details\n               about the diffraction attenuator scales employed.' ],
          [ 'item_', '_category.id', 'diffrn_attenuator' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_diffrn_attenuator.code' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    _diffrn_attenuator.code        1\n    _diffrn_attenuator.scale       16.976',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_attenuator.code',
        [ [ 'item_',
            '_item_description.description',
            '              A code associated with a particular attenuator setting. This\n               code is referenced by the _diffrn_refln.attenuator_code which is\n               stored with the diffraction data. See _diffrn_attenuator.scale.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_diffrn_attenuator.code', 'diffrn_attenuator', 'yes' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_attenuator_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_diffrn_refln.attenuator_code', '_diffrn_attenuator.code' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_diffrn_attenuator.scale',
        [ [ 'item_',
            '_item_description.description',
            '              The scale factor applied when an intensity measurement is\n               reduced by an attenuator identified by _diffrn_attenuator.code.\n               The measured intensity must be multiplied by this scale to\n                convert it to the same scale as unattenuated intensities.' ],
          [ 'item_', '_item.name', '_diffrn_attenuator.scale' ],
          [ 'item_', '_item.category_id', 'diffrn_attenuator' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_attenuator_scale' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1.0', '1.0', '1.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# DIFFRN_DETECTOR ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'diffrn_detector',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_DETECTOR category describe the\n               detector used to measure the scattered radiation, including\n               any analyser and post-sample collimation.' ],
          [ 'item_', '_category.id', 'diffrn_detector' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_diffrn_detector.diffrn_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _diffrn_detector.diffrn_id             \'d1\'\n    _diffrn_detector.detector              \'multiwire\'\n    _diffrn_detector.type                  \'Siemens\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_detector.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the radiation detector.' ],
          [ 'item_', '_item.name', '_diffrn_detector.details' ],
          [ 'item_', '_item.category_id', 'diffrn_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_detector_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_diffrn_detector.detector',
        [ [ 'item_',
            '_item_description.description',
            '              The general class of the radiation detector.' ],
          [ 'item_', '_item.name', '_diffrn_detector.detector' ],
          [ 'item_', '_item.category_id', 'diffrn_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_radiation_detector',
              'cifdic.c91',
              '1.0',
              '_diffrn_detector',
              'cif_core.dic',
              '2.0' ] ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'photographic film',
              'scintillation counter',
              'CCD plate',
              'BF~3~ counter' ] ] ] ],
      [ 'save_',
        '_diffrn_detector.diffrn_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn.id in the DIFFRN\n               category.' ],
          [ 'item_', '_item.name', '_diffrn_detector.diffrn_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', 'save__diffrn_detector.dtime' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              The deadtime in microseconds of the detectors used to measure' ],
      [ '@comment', '               the diffraction intensities.' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_diffrn_detector.dtime\'' ],
      [ '@comment',
        '    _item.category_id             diffrn_detector' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_aliases.alias_name' ],
      [ '@comment', '    _item_aliases.dictionary' ],
      [ '@comment',
        '    _item_aliases.version       \'_diffrn_radiation_detector_dtime\'' ],
      [ '@comment', '                                  cifdic.c91' ],
      [ '@comment', '                                  1.0' ],
      [ '@comment',
        '                                \'_diffrn_detector_dtime\'' ],
      [ '@comment', '                                  cif_core.dic' ],
      [ '@comment', '                                  2.0' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_range.maximum' ],
      [ '@comment', '    _item_range.minimum            .    0.0' ],
      [ '@comment', '                                  0.0   0.0' ],
      [ '@comment', '    _item_type.code               float' ],
      [ '@comment', '    _item_units.code              microseconds' ],
      [ '@comment', '     save_' ],
      [ 'save_',
        '_diffrn_detector.type',
        [ [ 'item_',
            '_item_description.description',
            '              The make, model or name of the detector device used.' ],
          [ 'item_', '_item.name', '_diffrn_detector.type' ],
          [ 'item_', '_item.category_id', 'diffrn_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_detector_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# DIFFRN_MEASUREMENT ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'diffrn_measurement',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_MEASUREMENT category record details\n               about the device used to orient and/or position the crystal\n               during data measurement and the manner in which the diffraction\n               data were measured.' ],
          [ 'item_', '_category.id', 'diffrn_measurement' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_diffrn_measurement.diffrn_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _diffrn_measurement.diffrn_id          \'d1\'\n    _diffrn_measurement.device             \'3-circle camera\'\n    _diffrn_measurement.device_type        \'Supper model x\'\n    _diffrn_measurement.device_details     \'none\'\n    _diffrn_measurement.method             \'omega scan\'\n    _diffrn_measurement.details\n    ; 440 frames, 0.20 degrees, 150 sec, detector distance 12 cm,\n      detector angle 22.5 degrees\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    _diffrn_measurement.diffrn_id       \'s1\'\n    _diffrn_measurement.device_type\n                              \'Philips PW1100/20 diffractometer\'\n    _diffrn_measurement.method          \\q/2\\q',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_measurement.diffrn_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn.id in the DIFFRN\n               category.' ],
          [ 'item_', '_item.name', '_diffrn_measurement.diffrn_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_diffrn_measurement.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the intensity measurement.' ],
          [ 'item_', '_item.name', '_diffrn_measurement.details' ],
          [ 'item_', '_item.category_id', 'diffrn_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_measurement_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 440 frames, 0.20 degrees, 150 sec, detector\n                                  distance 12 cm, detector angle 22.5 degrees' ] ] ],
      [ 'save_',
        '_diffrn_measurement.device',
        [ [ 'item_',
            '_item_description.description',
            '              The general class of  goniometer or device used to support and\n               orient the specimen.' ],
          [ 'item_', '_item.name', '_diffrn_measurement.device' ],
          [ 'item_', '_item.category_id', 'diffrn_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_measurement_device' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '3-circle camera',
              '4-circle camera',
              'kappa-geometry camera',
              'oscillation camera',
              'precession camera' ] ] ] ],
      [ 'save_',
        '_diffrn_measurement.device_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the device used to measure\n               the diffraction intensities.' ],
          [ 'item_', '_item.name', '_diffrn_measurement.device_details' ],
          [ 'item_', '_item.category_id', 'diffrn_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_measurement_device_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 commercial goniometer modified locally to\n                                 allow for 90\\% \\t arc' ] ] ],
      [ 'save_',
        '_diffrn_measurement.device_type',
        [ [ 'item_',
            '_item_description.description',
            '              The make, model or name of the measurement device\n               (goniometer) used.' ],
          [ 'item_', '_item.name', '_diffrn_measurement.device_type' ],
          [ 'item_', '_item.category_id', 'diffrn_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_measurement_device_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Supper model q',
              'Huber model r',
              'Enraf-Nonius model s',
              'homemade' ] ] ] ],
      [ 'save_',
        '_diffrn_measurement.method',
        [ [ 'item_',
            '_item_description.description',
            '              Method used to measure intensities.' ],
          [ 'item_', '_item.name', '_diffrn_measurement.method' ],
          [ 'item_', '_item.category_id', 'diffrn_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_measurement_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'profile data from theta/2theta scans' ] ] ],
      [ 'save_',
        '_diffrn_measurement.specimen_support',
        [ [ 'item_',
            '_item_description.description',
            '              The physical device used to support the crystal during data\n               collection.' ],
          [ 'item_', '_item.name', '_diffrn_measurement.specimen_support' ],
          [ 'item_', '_item.category_id', 'diffrn_measurement' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_measurement_specimen_support' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'glass capillary', 'quartz capillary', 'fiber', 'metal loop' ] ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# DIFFRN_ORIENT_MATRIX ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'diffrn_orient_matrix',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_ORIENT_MATRIX category record details\n               about the orientation matrix used in the measurement of the\n               diffraction data.' ],
          [ 'item_', '_category.id', 'diffrn_orient_matrix' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_diffrn_orient_matrix.diffrn_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on CAD-4 diffractometer data obtained for\n                Yb(S-C5H4N)2(THF)4.',
              '\n    _diffrn_orient_matrix.diffrn_id    set1\n    _diffrn_orient_matrix.type\n    ; reciprocal axis matrix, multiplies hkl vector to generate\n      diffractometer xyz vector and diffractometer angles\n    ;\n    _diffrn_orient_matrix.UB[1][1]     -0.071479\n    _diffrn_orient_matrix.UB[1][2]      0.020208\n    _diffrn_orient_matrix.UB[1][3]      0.039076\n    _diffrn_orient_matrix.UB[2][1]      0.035372\n    _diffrn_orient_matrix.UB[2][2]      0.056209\n    _diffrn_orient_matrix.UB[2][3]      0.078324\n    _diffrn_orient_matrix.UB[3][1]     -0.007470\n    _diffrn_orient_matrix.UB[3][2]      0.067854\n    _diffrn_orient_matrix.UB[3][3]     -0.017832',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.diffrn_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn.id in the DIFFRN\n               category.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.diffrn_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.type',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the orientation matrix type and how it should\n               be applied to define the orientation of the crystal precisely\n               with respect to the diffractometer axes.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.type' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_matrix_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.UB[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the 3x3 matrix that defines the dimensions\n               of the reciprocal cell and its orientation with respect to the\n               local diffractometer axes. See also _diffrn_orient_matrix.type.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.UB[1][1]' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_matrix_UB_11' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.UB[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the 3x3 matrix that defines the dimensions\n               of the reciprocal cell and its orientation with respect to the\n               local diffractometer axes. See also _diffrn_orient_matrix.type.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.UB[1][2]' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_matrix_UB_12' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.UB[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the 3x3 matrix that defines the dimensions\n               of the reciprocal cell and its orientation with respect to the\n               local diffractometer axes. See also _diffrn_orient_matrix.type.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.UB[1][3]' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_matrix_UB_13' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.UB[2][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][1] element of the 3x3 matrix that defines the dimensions\n               of the reciprocal cell and its orientation with respect to the\n               local diffractometer axes. See also _diffrn_orient_matrix.type.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.UB[2][1]' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_matrix_UB_21' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.UB[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the 3x3 matrix that defines the dimensions\n               of the reciprocal cell and its orientation with respect to the\n               local diffractometer axes. See also _diffrn_orient_matrix.type.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.UB[2][2]' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_matrix_UB_22' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.UB[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the 3x3 matrix that defines the dimensions\n               of the reciprocal cell and its orientation with respect to the\n               local diffractometer axes. See also _diffrn_orient_matrix.type.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.UB[2][3]' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_matrix_UB_23' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.UB[3][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][1] element of the 3x3 matrix that defines the dimensions\n               of the reciprocal cell and its orientation with respect to the\n               local diffractometer axes. See also _diffrn_orient_matrix.type.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.UB[3][1]' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_matrix_UB_31' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.UB[3][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][2] element of the 3x3 matrix that defines the dimensions\n               of the reciprocal cell and its orientation with respect to the\n               local diffractometer axes. See also _diffrn_orient_matrix.type.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.UB[3][2]' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_matrix_UB_32' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_orient_matrix.UB[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the 3x3 matrix that defines the dimensions\n               of the reciprocal cell and its orientation with respect to the\n               local  diffractometer axes. See also _diffrn_orient_matrix.type.' ],
          [ 'item_', '_item.name', '_diffrn_orient_matrix.UB[3][3]' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_matrix' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_matrix_UB_33' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# DIFFRN_ORIENT_REFLN ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'diffrn_orient_refln',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_ORIENT_REFLN category record details\n               about the reflections that define the orientation matrix used in\n               the measurement of the diffraction intensities.' ],
          [ 'item_', '_category.id', 'diffrn_orient_refln' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_diffrn_orient_refln.diffrn_id',
              '_diffrn_orient_refln.index_h',
              '_diffrn_orient_refln.index_k',
              '_diffrn_orient_refln.index_l' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on CAD-4 diffractometer data obtained for\n                Yb(S-C5H4N)2(THF)4.',
              '\n    _diffrn_orient_refln.diffrn_id     myset1\n    _diffrn_orient_refln.index_h       2\n    _diffrn_orient_refln.index_k       0\n    _diffrn_orient_refln.index_l       2\n    _diffrn_orient_refln.angle_chi     -28.45\n    _diffrn_orient_refln.angle_kappa   -11.32\n    _diffrn_orient_refln.angle_omega   5.33\n    _diffrn_orient_refln.angle_phi     101.78\n    _diffrn_orient_refln.angle_psi     0.00\n    _diffrn_orient_refln.angle_theta   10.66\n    # ... data abbreviated  ...',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_orient_refln.angle_chi',
        [ [ 'item_',
            '_item_description.description',
            '              Diffractometer angle chi of a reflection used to\n               define the orientation matrix in degrees. See\n               _diffrn_orient_matrix.UB[][] and the Miller indices\n               in the DIFFRN_ORIENT_REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_orient_refln.angle_chi' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_refln_angle_chi' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_orient_refln.angle_kappa',
              '_diffrn_orient_refln.angle_omega',
              '_diffrn_orient_refln.angle_phi',
              '_diffrn_orient_refln.angle_psi',
              '_diffrn_orient_refln.angle_theta' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_orient_refln.angle_kappa',
        [ [ 'item_',
            '_item_description.description',
            '              Diffractometer angle kappa of a reflection used to\n               define the orientation matrix in degrees. See\n               _diffrn_orient_matrix.UB[][] and the Miller indices\n               in the DIFFRN_ORIENT_REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_orient_refln.angle_kappa' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_refln_angle_kappa' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_orient_refln.angle_chi',
              '_diffrn_orient_refln.angle_omega',
              '_diffrn_orient_refln.angle_phi',
              '_diffrn_orient_refln.angle_psi',
              '_diffrn_orient_refln.angle_theta' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_orient_refln.angle_omega',
        [ [ 'item_',
            '_item_description.description',
            '              Diffractometer angle omega of a reflection used to\n               define the orientation matrix in degrees. See\n               _diffrn_orient_matrix.UB[][] and the Miller indices in\n               the DIFFRN_ORIENT_REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_orient_refln.angle_omega' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_refln_angle_omega' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_orient_refln.angle_chi',
              '_diffrn_orient_refln.angle_kappa',
              '_diffrn_orient_refln.angle_phi',
              '_diffrn_orient_refln.angle_psi',
              '_diffrn_orient_refln.angle_theta' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_orient_refln.angle_phi',
        [ [ 'item_',
            '_item_description.description',
            '              Diffractometer angle phi of a reflection used to\n               define the orientation matrix in degrees. See\n               _diffrn_orient_matrix.UB[][] and the Miller indices\n               in the DIFFRN_ORIENT_REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_orient_refln.angle_phi' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_refln_angle_phi' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_orient_refln.angle_chi',
              '_diffrn_orient_refln.angle_kappa',
              '_diffrn_orient_refln.angle_omega',
              '_diffrn_orient_refln.angle_psi',
              '_diffrn_orient_refln.angle_theta' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_orient_refln.angle_psi',
        [ [ 'item_',
            '_item_description.description',
            '              Diffractometer angle psi of a reflection used to\n               define the orientation matrix in degrees. See\n               _diffrn_orient_matrix.UB[][] and the Miller indices\n               in the DIFFRN_ORIENT_REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_orient_refln.angle_psi' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_refln_angle_psi' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_orient_refln.angle_chi',
              '_diffrn_orient_refln.angle_kappa',
              '_diffrn_orient_refln.angle_omega',
              '_diffrn_orient_refln.angle_phi',
              '_diffrn_orient_refln.angle_theta' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_orient_refln.angle_theta',
        [ [ 'item_',
            '_item_description.description',
            '              Diffractometer angle theta of a reflection used to\n               define the orientation matrix in degrees. See\n               _diffrn_orient_matrix.UB[][] and the Miller indices\n               in the DIFFRN_ORIENT_REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_orient_refln.angle_theta' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_refln_angle_theta' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_orient_refln.angle_chi',
              '_diffrn_orient_refln.angle_kappa',
              '_diffrn_orient_refln.angle_omega',
              '_diffrn_orient_refln.angle_phi',
              '_diffrn_orient_refln.angle_psi' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_orient_refln.diffrn_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn.id in the DIFFRN\n               category.' ],
          [ 'item_', '_item.name', '_diffrn_orient_refln.diffrn_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_diffrn_orient_refln.index_h',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index h of a reflection used to define the orientation\n               matrix.' ],
          [ 'item_', '_item.name', '_diffrn_orient_refln.index_h' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_refln_index_h' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_orient_refln.index_k',
              '_diffrn_orient_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_orient_refln.index_k',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index k of a reflection used to define the orientation\n               matrix.' ],
          [ 'item_', '_item.name', '_diffrn_orient_refln.index_k' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_refln_index_k' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_orient_refln.index_h',
              '_diffrn_orient_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_orient_refln.index_l',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index l of a reflection used to define the orientation\n               matrix.' ],
          [ 'item_', '_item.name', '_diffrn_orient_refln.index_l' ],
          [ 'item_', '_item.category_id', 'diffrn_orient_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_orient_refln_index_l' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_orient_refln.index_h',
              '_diffrn_orient_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# DIFFRN_RADIATION ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'diffrn_radiation',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_RADIATION category describe\n               the radiation used in measuring the diffraction intensities,\n               its collimation and monochromatization before the sample.\n\n               Post-sample treatment of the beam is described by data\n               items in the DIFFRN_DETECTOR category.\n' ],
          [ 'item_', '_category.id', 'diffrn_radiation' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_diffrn_radiation.diffrn_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _diffrn_radiation.diffrn_id          \'set1\'\n\n    _diffrn_radiation.collimation        \'0.3 mm double pinhole\'\n    _diffrn_radiation.monochromator      \'graphite\'\n    _diffrn_radiation.type               \'Cu K\\a\'\n    _diffrn_radiation.wavelength_id       1',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    _diffrn_radiation.wavelength_id    1\n    _diffrn_radiation.type             \'Cu K\\a\'\n    _diffrn_radiation.monochromator    \'graphite\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_radiation.collimation',
        [ [ 'item_',
            '_item_description.description',
            '              The collimation or focusing applied to the radiation.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.collimation' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_collimation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '0.3 mm double-pinhole', '0.5 mm', 'focusing mirrors' ] ] ] ],
      [ 'save_',
        '_diffrn_radiation.diffrn_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn.id in the DIFFRN\n               category.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.diffrn_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_diffrn_radiation.filter_edge',
        [ [ 'item_',
            '_item_description.description',
            '              Absorption edge in angstroms of the radiation filter used.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.filter_edge' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_filter_edge' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_diffrn_radiation.inhomogeneity',
        [ [ 'item_',
            '_item_description.description',
            '              Half-width in millimetres of the incident beam in the\n               direction perpendicular to the diffraction plane.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.inhomogeneity' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_inhomogeneity' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'millimetres' ] ] ],
      [ 'save_',
        '_diffrn_radiation.monochromator',
        [ [ 'item_',
            '_item_description.description',
            '              The method used to obtain monochromatic radiation. If a mono-\n               chromator crystal is used, the material and the indices of the\n               Bragg reflection are specified.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.monochromator' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_monochromator' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Zr filter', 'Ge 220', 'none', 'equatorial mounted graphite' ] ] ] ],
      [ 'save_',
        '_diffrn_radiation.polarisn_norm',
        [ [ 'item_',
            '_item_description.description',
            '              The angle in degrees, as viewed from the specimen, between the\n               perpendicular component of the polarization and the diffraction\n               plane. See _diffrn_radiation.polarisn_ratio.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.polarisn_norm' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_polarisn_norm' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_radiation.polarisn_ratio',
        [ [ 'item_',
            '_item_description.description',
            '              Polarization ratio of the diffraction beam incident on the\n               crystal. This is the ratio of the perpendicularly polarized\n               to the parallel-polarized component of the radiation. The\n               perpendicular component forms an angle of\n               _diffrn_radiation.polarisn_norm to the normal to the\n               diffraction plane of the sample (i.e. the plane containing\n               the incident and reflected beams).' ],
          [ 'item_', '_item.name', '_diffrn_radiation.polarisn_ratio' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_polarisn_ratio' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_radiation.probe',
        [ [ 'item_',
            '_item_description.description',
            '             The nature of the radiation used (i.e. the name of the\n              subatomic particle or the region of the electromagnetic\n              spectrum). It is strongly recommended that this information\n              is given, so that the probe radiation can be simply determined.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.probe' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_probe' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'x-ray', 'neutron', 'electron', 'gamma' ] ] ] ],
      [ 'save_',
        '_diffrn_radiation.type',
        [ [ 'item_',
            '_item_description.description',
            '              The nature of the radiation. This is typically a description\n               of the X-ray wavelength in Siegbahn notation.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.type' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_radiation_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'CuK\\a', 'Cu K\\a~1~', 'Cu K-L~2,3~', 'white-beam' ] ] ] ],
      [ 'save_',
        '_diffrn_radiation.xray_symbol',
        [ [ 'item_',
            '_item_description.description',
            '              The IUPAC symbol for the X-ray wavelength for the probe\n               radiation.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.xray_symbol' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_xray_symbol' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'K-L~3~',
              'K\\a~1~ in older Siegbahn notation',
              'K-L~2~',
              'K\\a~2~ in older Siegbahn notation',
              'K-M~3~',
              'K\\b~1~ in older Siegbahn notation',
              'K-L~2,3~',
              'use where K-L~3~ and K-L~2~ are not resolved' ] ] ] ],
      [ 'save_',
        '_diffrn_radiation.wavelength_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn_radiation_wavelength.id\n               in the DIFFRN_RADIATION_WAVELENGTH category.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.wavelength_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '################################' ],
      [ '@comment', '# DIFFRN_RADIATION_WAVELENGTH ##' ],
      [ '@comment', '################################' ],
      [ 'save_',
        'diffrn_radiation_wavelength',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_RADIATION_WAVELENGTH category\n               describe the wavelength of the radiation used to measure the\n               diffraction intensities. Items may be looped to identify\n               and assign weights to distinct components of a\n               polychromatic beam.' ],
          [ 'item_', '_category.id', 'diffrn_radiation_wavelength' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_diffrn_radiation_wavelength.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _diffrn_radiation_wavelength.id                    1\n    _diffrn_radiation_wavelength.wavelength            1.54\n    _diffrn_radiation_wavelength.wt                    1.0',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_radiation_wavelength.id',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying each value of\n               _diffrn_radiation_wavelength.wavelength.\n               Items in the DIFFRN_RADIATION_WAVELENGTH category are looped\n               when multiple wavelengths are used.\n\n               This code is used to link with the DIFFRN_REFLN category.\n               The _diffrn_refln.wavelength_id codes must match one of\n               the codes defined in this category.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_diffrn_radiation_wavelength.id',
              'diffrn_radiation_wavelength',
              'yes',
              '_diffrn_radiation.wavelength_id',
              'diffrn_radiation',
              'yes',
              '_diffrn_refln.wavelength_id',
              'diffrn_refln',
              'yes',
              '_refln.wavelength_id',
              'refln',
              'yes' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_wavelength_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_diffrn_radiation.wavelength_id',
              '_diffrn_radiation_wavelength.id',
              '_diffrn_refln.wavelength_id',
              '_diffrn_radiation_wavelength.id',
              '_refln.wavelength_id',
              '_diffrn_radiation_wavelength.id' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'x1', 'x2', 'neut' ] ] ] ],
      [ 'save_',
        '_diffrn_radiation_wavelength.wavelength',
        [ [ 'item_',
            '_item_description.description',
            '              The radiation wavelength in angstroms.' ],
          [ 'item_',
            '_item.name',
            '_diffrn_radiation_wavelength.wavelength' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation_wavelength' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_wavelength' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_diffrn_radiation_wavelength.wt',
        [ [ 'item_',
            '_item_description.description',
            '              The relative weight of a wavelength identified by the code\n               _diffrn_radiation_wavelength.id in the list of wavelengths.' ],
          [ 'item_', '_item.name', '_diffrn_radiation_wavelength.wt' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation_wavelength' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation_wavelength_wt' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '1.0', '1.0', '1.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#################' ],
      [ '@comment', '# DIFFRN_REFLN ##' ],
      [ '@comment', '#################' ],
      [ 'save_',
        'diffrn_refln',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_REFLN category record details about\n               the intensities in the diffraction data set\n               identified by _diffrn_refln.diffrn_id.\n\n               The DIFFRN_REFLN data items refer to individual intensity\n               measurements and must be included in looped lists.\n\n               The DIFFRN_REFLNS data items specify the parameters that apply\n               to all intensity  measurements in the particular diffraction\n               data set identified by _diffrn_reflns.diffrn_id.' ],
          [ 'item_', '_category.id', 'diffrn_refln' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_diffrn_refln.diffrn_id', '_diffrn_refln.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on CAD-4 diffractometer data obtained for\n                Yb(S-C5H4N)2(THF)4  for data set \'set1\' reflection 1102.',
              '\n    _diffrn_refln.diffrn_id            set1\n    _diffrn_refln.id                   1102\n    _diffrn_refln.wavelength_id        Cu1fixed\n    _diffrn_refln.angle_chi            32.21\n    _diffrn_refln.angle_kappa          20.12\n    _diffrn_refln.angle_omega          11.54\n    _diffrn_refln.angle_phi            176.02\n    _diffrn_refln.angle_psi            0.00\n    _diffrn_refln.angle_theta          23.08\n    _diffrn_refln.attenuator_code      \'Ni.005\'\n    _diffrn_refln.counts_bg_1          22\n    _diffrn_refln.counts_bg_2          25\n    _diffrn_refln.counts_net           3450\n    _diffrn_refln.counts_peak          321\n    _diffrn_refln.counts_total         3499\n    _diffrn_refln.detect_slit_horiz    0.04\n    _diffrn_refln.detect_slit_vert     0.02\n    _diffrn_refln.elapsed_time         1.00\n    _diffrn_refln.index_h              4\n    _diffrn_refln.index_k              0\n    _diffrn_refln.index_l              2\n    _diffrn_refln.intensity_net        202.56\n    _diffrn_refln.intensity_sigma      2.18\n    _diffrn_refln.scale_group_code     A24\n    _diffrn_refln.scan_mode            om\n    _diffrn_refln.scan_mode_backgd     mo\n    _diffrn_refln.scan_rate            1.2\n    _diffrn_refln.scan_time_backgd     900.00\n    _diffrn_refln.scan_width           1.0\n    _diffrn_refln.sint_over_lambda     0.25426\n    _diffrn_refln.standard_code        1\n    _diffrn_refln.wavelength           1.54184',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_refln.angle_chi',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer angle chi of a reflection in degrees. This\n               angle corresponds to the specified orientation matrix\n               and the original measured cell before any subsequent cell\n               transformations.' ],
          [ 'item_', '_item.name', '_diffrn_refln.angle_chi' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_angle_chi' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_refln.angle_kappa',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer angle kappa of a reflection in degrees. This\n               angle corresponds to the specified orientation matrix\n               and the original measured cell before any subsequent cell\n               transformations.' ],
          [ 'item_', '_item.name', '_diffrn_refln.angle_kappa' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_angle_kappa' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_refln.angle_omega',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer angle omega of a reflection in degrees. This\n               angle corresponds to the specified orientation matrix\n               and the original measured cell before any subsequent cell\n               transformations.' ],
          [ 'item_', '_item.name', '_diffrn_refln.angle_omega' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_angle_omega' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_refln.angle_phi',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer angle phi of a reflection in degrees. This\n               angle corresponds to the specified orientation matrix\n               and the original measured cell before any subsequent cell\n               transformations.' ],
          [ 'item_', '_item.name', '_diffrn_refln.angle_phi' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_angle_phi' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_refln.angle_psi',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer angle psi of a reflection in degrees. This\n               angle corresponds  to the specified orientation matrix\n               and the original measured cell before any subsequent cell\n               transformations.' ],
          [ 'item_', '_item.name', '_diffrn_refln.angle_psi' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_angle_psi' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_refln.angle_theta',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer angle theta of a reflection in degrees. This\n               angle corresponds  to the specified orientation matrix\n               and the original measured cell before any subsequent cell\n               transformations.' ],
          [ 'item_', '_item.name', '_diffrn_refln.angle_theta' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_angle_theta' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_refln.attenuator_code',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying the attenuator setting for this reflection.\n               This code must match one of the _diffrn_attenuator.code values.' ],
          [ 'item_', '_item.name', '_diffrn_refln.attenuator_code' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_attenuator_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_diffrn_refln.counts_bg_1',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer counts for the measurement of the background\n               before the peak.' ],
          [ 'item_', '_item.name', '_diffrn_refln.counts_bg_1' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_counts_bg_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_refln.counts_bg_2',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer counts for the measurement of the background\n               after the peak.' ],
          [ 'item_', '_item.name', '_diffrn_refln.counts_bg_2' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_counts_bg_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_refln.counts_net',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer counts for the measurement of net counts after\n               background removal.' ],
          [ 'item_', '_item.name', '_diffrn_refln.counts_net' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_counts_net' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_refln.counts_peak',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer counts for the measurement of counts for the\n               peak scan or position.' ],
          [ 'item_', '_item.name', '_diffrn_refln.counts_peak' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_counts_peak' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_refln.counts_total',
        [ [ 'item_',
            '_item_description.description',
            '              The diffractometer counts for the measurement of total counts\n               (background plus peak).' ],
          [ 'item_', '_item.name', '_diffrn_refln.counts_total' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_counts_total' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_refln.detect_slit_horiz',
        [ [ 'item_',
            '_item_description.description',
            '              Total slit aperture in degrees in the diffraction plane.' ],
          [ 'item_', '_item.name', '_diffrn_refln.detect_slit_horiz' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_detect_slit_horiz' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '90.0', '90.0', '90.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_refln.detect_slit_vert',
        [ [ 'item_',
            '_item_description.description',
            '              Total slit aperture in degrees perpendicular to the\n               diffraction plane.' ],
          [ 'item_', '_item.name', '_diffrn_refln.detect_slit_vert' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_detect_slit_vert' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '90.0', '90.0', '90.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_refln.diffrn_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn.id in the DIFFRN\n               category.' ],
          [ 'item_', '_item.name', '_diffrn_refln.diffrn_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_diffrn_refln.elapsed_time',
        [ [ 'item_',
            '_item_description.description',
            '              Elapsed time in minutes from the start of the diffraction\n               experiment to the measurement of this intensity.' ],
          [ 'item_', '_item.name', '_diffrn_refln.elapsed_time' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_elapsed_time' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'minutes' ] ] ],
      [ 'save_',
        '_diffrn_refln.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _diffrn_refln.id must uniquely identify the\n               reflection in the data set identified by the item\n              _diffrn_refln.diffrn_id.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_diffrn_refln.id' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_diffrn_refln.index_h',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index h of a reflection. The values of\n               the Miller indices in the DIFFRN_REFLN category need not match\n               the values of the Miller indices in the REFLN category if a\n               transformation of the original measured cell has taken place.\n               Details of the cell transformation are given in\n               _diffrn_reflns.reduction_process. See also\n               _diffrn_reflns.transf_matrix[][].' ],
          [ 'item_', '_item.name', '_diffrn_refln.index_h' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_refln_index_h' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_refln.index_h', '_diffrn_refln.index_k' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_refln.index_k',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index k of a reflection. The values of\n               the Miller indices in the DIFFRN_REFLN category need not match\n               the values of the Miller indices in the REFLN category if a\n               transformation of the original measured cell has taken place.\n               Details of the cell transformation are given in\n               _diffrn_reflns.reduction_process. See also\n               _diffrn_reflns.transf_matrix[][].' ],
          [ 'item_', '_item.name', '_diffrn_refln.index_k' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_refln_index_k' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_refln.index_h', '_diffrn_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_refln.index_l',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index l of a reflection. The values of\n               the Miller indices in the DIFFRN_REFLN category need not match\n               the values of the Miller indices in the REFLN category if a\n               transformation of the original measured cell has taken place.\n               Details of the cell transformation are given in\n               _diffrn_reflns.reduction_process. See also\n               _diffrn_reflns.transf_matrix[][].' ],
          [ 'item_', '_item.name', '_diffrn_refln.index_l' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_refln_index_l' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_refln.index_h', '_diffrn_refln.index_k' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_refln.intensity_net',
        [ [ 'item_',
            '_item_description.description',
            '              Net intensity calculated from the diffraction counts after the\n               attenuator and standard scales have been applied.' ],
          [ 'item_', '_item.name', '_diffrn_refln.intensity_net' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_intensity_net' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_refln.intensity_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              Standard uncertainty (estimated standard deviation) of the\n               intensity calculated from the diffraction counts after the\n               attenuator and standard scales have been applied.' ],
          [ 'item_', '_item.name', '_diffrn_refln.intensity_sigma' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_intensity_sigma' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_refln.scale_group_code',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying the scale applying to this reflection.\n\n               This data item is a pointer to _diffrn_scale_group.code in the\n               DIFFRN_SCALE_GROUP category.' ],
          [ 'item_', '_item.name', '_diffrn_refln.scale_group_code' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_scale_group_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_diffrn_refln.scan_mode',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying the mode of scanning for measurements\n               using a diffractometer.\n               See _diffrn_refln.scan_width and _diffrn_refln.scan_mode_backgd.' ],
          [ 'item_', '_item.name', '_diffrn_refln.scan_mode' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_scan_mode' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'om',
              'omega scan',
              'ot',
              'omega/2theta scan',
              'q',
              'Q scans (arbitrary reciprocal directions)' ] ] ] ],
      [ 'save_',
        '_diffrn_refln.scan_mode_backgd',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying the mode of scanning a reflection to\n               measure the background intensity.' ],
          [ 'item_', '_item.name', '_diffrn_refln.scan_mode_backgd' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_scan_mode_backgd' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'st',
              'stationary counter background',
              'mo',
              'moving counter background' ] ] ] ],
      [ 'save_',
        '_diffrn_refln.scan_rate',
        [ [ 'item_',
            '_item_description.description',
            '              The rate of scanning a reflection in degrees per minute\n               to measure the intensity.' ],
          [ 'item_', '_item.name', '_diffrn_refln.scan_rate' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_scan_rate' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees_per_minute' ] ] ],
      [ 'save_',
        '_diffrn_refln.scan_time_backgd',
        [ [ 'item_',
            '_item_description.description',
            '              The time spent measuring each background in seconds.' ],
          [ 'item_', '_item.name', '_diffrn_refln.scan_time_backgd' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_scan_time_backgd' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'seconds' ] ] ],
      [ 'save_',
        '_diffrn_refln.scan_width',
        [ [ 'item_',
            '_item_description.description',
            '              The scan width in degrees of the scan mode defined by the code\n               _diffrn_refln.scan_mode.' ],
          [ 'item_', '_item.name', '_diffrn_refln.scan_width' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_scan_width' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '90.0', '90.0', '90.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_refln.sint_over_lambda',
        [ [ 'item_',
            '_item_description.description',
            '              The (sin theta)/lambda value in reciprocal angstroms for this\n               reflection.' ],
          [ 'item_', '_item.name', '_diffrn_refln.sint_over_lambda' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_sint/lambda' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'reciprocal_angstroms' ] ] ],
      [ 'save_',
        '_diffrn_refln.standard_code',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying that this reflection was measured as a\n               standard intensity.\n\n               This data item is a pointer to _diffrn_standard_refln.code in the\n               DIFFRN_STANDARD_REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_refln.standard_code' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_standard_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_diffrn_refln.wavelength',
        [ [ 'item_',
            '_item_description.description',
            '              The mean wavelength in angstroms of the radiation used to measure\n               the intensity of this reflection. This is an important parameter\n               for data collected using energy-dispersive detectors or the\n               Laue method.' ],
          [ 'item_', '_item.name', '_diffrn_refln.wavelength' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_wavelength' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_diffrn_refln.wavelength_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn_radiation.wavelength_id in\n               the DIFFRN_RADIATION category.' ],
          [ 'item_', '_item.name', '_diffrn_refln.wavelength_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_refln_wavelength_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ '@comment', '##################' ],
      [ '@comment', '# DIFFRN_REFLNS ##' ],
      [ '@comment', '##################' ],
      [ 'save_',
        'diffrn_reflns',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_REFLNS category record details about\n               the set of intensities measured in the diffraction experiment.\n\n               The DIFFRN_REFLN data items refer to individual intensity\n               measurements and must be included in looped lists.\n\n               The DIFFRN_REFLNS data items specify the parameters that apply\n               to all intensity measurements in a diffraction data set.' ],
          [ 'item_', '_category.id', 'diffrn_reflns' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_diffrn_reflns.diffrn_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'diffrn_group',
              [ '@comment', '    loop_' ],
              [ '@comment', '    _category_examples.detail' ],
              [ '@comment', '    _category_examples.case' ],
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_reflns.av_R_equivalents',
        [ [ 'item_',
            '_item_description.description',
            '              The residual [sum|avdel(I)| / sum|av(I)|] for symmetry-equivalent\n               reflections used to calculate the average intensity av(I). The\n               avdel(I) term is the average absolute difference between av(I)\n               and the individual symmetry-equivalent intensities.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.av_R_equivalents' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_av_R_equivalents' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_reflns.av_sigmaI_over_netI',
        [ [ 'item_',
            '_item_description.description',
            '              Measure [sum|sigma(I)|/sum|net(I)|] for all measured reflections.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.av_sigmaI_over_netI' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_av_sigmaI/netI' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_reflns.diffrn_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn.id in the DIFFRN\n               category.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.diffrn_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_diffrn_reflns.limit_h_max',
        [ [ 'item_',
            '_item_description.description',
            '              The maximum value of the Miller index h for the\n               reflection data specified by _diffrn_refln.index_h.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.limit_h_max' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_limit_h_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_reflns.limit_h_min',
        [ [ 'item_',
            '_item_description.description',
            '              The minimum value of the Miller index h for the\n               reflection data specified by _diffrn_refln.index_h.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.limit_h_min' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_limit_h_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_reflns.limit_k_max',
        [ [ 'item_',
            '_item_description.description',
            '              The maximum value of the Miller index k for the\n               reflection data specified by _diffrn_refln.index_k.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.limit_k_max' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_limit_k_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_reflns.limit_k_min',
        [ [ 'item_',
            '_item_description.description',
            '              The minimum value of the Miller index k for the\n               reflection data specified by _diffrn_refln.index_k.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.limit_k_min' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_limit_k_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_reflns.limit_l_max',
        [ [ 'item_',
            '_item_description.description',
            '              The maximum value of the Miller index l for the\n               reflection data specified by _diffrn_refln.index_l.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.limit_l_max' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_limit_l_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_reflns.limit_l_min',
        [ [ 'item_',
            '_item_description.description',
            '              The minimum value of the Miller index l for the\n               reflection data specified by _diffrn_refln.index_l.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.limit_l_min' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_limit_l_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_reflns.number',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of measured intensities, excluding reflections\n               that are classified as systematically absent.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.number' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_reflns_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_reflns.reduction_process',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the process used to reduce the intensity data\n               into structure-factor magnitudes.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.reduction_process' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_reduction_process' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'data averaged using Fisher test' ] ] ],
      [ 'save_',
        '_diffrn_reflns.theta_max',
        [ [ 'item_',
            '_item_description.description',
            '              Maximum theta angle in degrees for the measured diffraction\n               intensities.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.theta_max' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_theta_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '90.0', '90.0', '90.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_reflns.theta_min',
        [ [ 'item_',
            '_item_description.description',
            '              Minimum theta angle in degrees for the measured diffraction\n               intensities.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.theta_min' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_theta_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '90.0', '90.0', '90.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_diffrn_reflns.transf_matrix[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the 3x3 matrix used to transform Miller\n               indices in the DIFFRN_REFLN category into the Miller indices in\n               the REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.transf_matrix[1][1]' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_transf_matrix_11' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_reflns.transf_matrix[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the 3x3 matrix used to transform Miller\n               indices in the DIFFRN_REFLN category into the Miller indices in\n               the REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.transf_matrix[1][2]' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_transf_matrix_12' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_reflns.transf_matrix[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the 3x3 matrix used to transform Miller\n               indices in the DIFFRN_REFLN category into the Miller indices in\n               the REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.transf_matrix[1][3]' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_transf_matrix_13' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_reflns.transf_matrix[2][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][1] element of the 3x3 matrix used to transform Miller\n               indices in the DIFFRN_REFLN category into the Miller indices in\n               the REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.transf_matrix[2][1]' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_transf_matrix_21' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_reflns.transf_matrix[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the 3x3 matrix used to transform Miller\n               indices in the DIFFRN_REFLN category into the Miller indices in\n               the REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.transf_matrix[2][2]' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_transf_matrix_22' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_reflns.transf_matrix[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the 3x3 matrix used to transform Miller\n               indices in the DIFFRN_REFLN category into the Miller indices in\n               the REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.transf_matrix[2][3]' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_transf_matrix_23' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_reflns.transf_matrix[3][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][1] element of the 3x3 matrix used to transform Miller\n               indices in the DIFFRN_REFLN category into the Miller indices in\n               the REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.transf_matrix[3][1]' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_transf_matrix_31' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_reflns.transf_matrix[3][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][2] element of the 3x3 matrix used to transform Miller\n               indices in the DIFFRN_REFLN category into the Miller indices in\n               the REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.transf_matrix[3][2]' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_transf_matrix_32' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_reflns.transf_matrix[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the 3x3 matrix used to transform Miller\n               indices in the DIFFRN_REFLN category into the Miller indices in\n               the REFLN category.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.transf_matrix[3][3]' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_reflns_transf_matrix_33' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# DIFFRN_SCALE_GROUP ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'diffrn_scale_group',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_SCALE_GROUP category record details\n               of the scaling factors applied to place all intensities in the\n               reflection lists on a common scale.\n               Scaling groups might, for example, correspond to each film in a\n               multi-film data set or each crystal in a multi-crystal data set.' ],
          [ 'item_', '_category.id', 'diffrn_scale_group' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_diffrn_scale_group.code' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on CAD-4 diffractometer data obtained for\n                Yb(S-C5H4N)2(THF)4.',
              '\n    _diffrn_scale_group.code        A24\n    _diffrn_scale_group.I_net       1.021',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_scale_group.code',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _diffrn_scale_group.code must uniquely identify a\n               record in the DIFFRN_SCALE_GROUP list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_diffrn_scale_group.code',
              'diffrn_scale_group',
              'yes',
              '_diffrn_refln.scale_group_code',
              'diffrn_refln',
              'yes' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_scale_group_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_diffrn_refln.scale_group_code', '_diffrn_scale_group.code' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', 'c1', 'c2' ] ] ] ],
      [ 'save_',
        '_diffrn_scale_group.I_net',
        [ [ 'item_',
            '_item_description.description',
            '              The scale for a specific measurement group which is to be\n               multiplied with the net intensity to place all intensities\n               in the DIFFRN_REFLN or REFLN list on a common scale.' ],
          [ 'item_', '_item.name', '_diffrn_scale_group.I_net' ],
          [ 'item_', '_item.category_id', 'diffrn_scale_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_scale_group_I_net' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '##################' ],
      [ '@comment', '# DIFFRN_SOURCE ##' ],
      [ '@comment', '##################' ],
      [ 'save_',
        'diffrn_source',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_SOURCE category record details of\n               the source of radiation used in the diffraction experiment.' ],
          [ 'item_', '_category.id', 'diffrn_source' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_diffrn_source.diffrn_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _diffrn_source.diffrn_id       \'s1\'\n    _diffrn_source.source          \'rotating anode\'\n    _diffrn_source.type            \'Rigaku RU-200\'\n    _diffrn_source.power           50\n    _diffrn_source.current        180\n    _diffrn_source.size            \'8mm x 0.4 mm broad-focus\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_source.current',
        [ [ 'item_',
            '_item_description.description',
            '              The current in milliamperes at which the radiation source\n               was operated.' ],
          [ 'item_', '_item.name', '_diffrn_source.current' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_source_current' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'milliamperes' ] ] ],
      [ 'save_',
        '_diffrn_source.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the radiation source used.' ],
          [ 'item_', '_item.name', '_diffrn_source.details' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_source_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_diffrn_source.diffrn_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn.id in the DIFFRN\n               category.' ],
          [ 'item_', '_item.name', '_diffrn_source.diffrn_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_diffrn_source.power',
        [ [ 'item_',
            '_item_description.description',
            '              The power in kilowatts at which the radiation source\n               was operated.' ],
          [ 'item_', '_item.name', '_diffrn_source.power' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_source_power' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kilowatts' ] ] ],
      [ 'save_',
        '_diffrn_source.size',
        [ [ 'item_',
            '_item_description.description',
            '              The dimensions of the source as viewed from the sample.' ],
          [ 'item_', '_item.name', '_diffrn_source.size' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_source_size' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '8mm x 0.4 mm fine-focus', 'broad focus' ] ] ] ],
      [ 'save_',
        '_diffrn_source.source',
        [ [ 'item_',
            '_item_description.description',
            '              The general class of the radiation source.' ],
          [ 'item_', '_item.name', '_diffrn_source.source' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_radiation_source',
              'cifdic.c91',
              '1.0',
              '_diffrn_source',
              'cif_core.dic',
              '2.0' ] ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'sealed X-ray tube',
              'nuclear reactor',
              'spallation source',
              'electron microscope',
              'rotating-anode X-ray tube',
              'synchrotron' ] ] ] ],
      [ 'save_',
        '_diffrn_source.target',
        [ [ 'item_',
            '_item_description.description',
            '              The chemical element symbol for the X-ray target\n               (usually the anode) used to generate X-rays.\n               This can also be used for spallation sources.' ],
          [ 'item_', '_item.name', '_diffrn_source.target' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_source_target' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'H',
              'He',
              'Li',
              'Be',
              'B',
              'C',
              'N',
              'O',
              'F',
              'Ne',
              'Na',
              'Mg',
              'Al',
              'Si',
              'P',
              'S',
              'Cl',
              'Ar',
              'K',
              'Ca',
              'Sc',
              'Ti',
              'V',
              'Cr',
              'Mn',
              'Fe',
              'Co',
              'Ni',
              'Cu',
              'Zn',
              'Ga',
              'Ge',
              'As',
              'Se',
              'Br',
              'Kr',
              'Rb',
              'Sr',
              'Y',
              'Zr',
              'Nb',
              'Mo',
              'Tc',
              'Ru',
              'Rh',
              'Pd',
              'Ag',
              'Cd',
              'In',
              'Sn',
              'Sb',
              'Te',
              'I',
              'Xe',
              'Cs',
              'Ba',
              'La',
              'Ce',
              'Pr',
              'Nd',
              'Pm',
              'Sm',
              'Eu',
              'Gd',
              'Tb',
              'Dy',
              'Ho',
              'Er',
              'Tm',
              'Yb',
              'Lu',
              'Hf',
              'Ta',
              'W',
              'Re',
              'Os',
              'Ir',
              'Pt',
              'Au',
              'Hg',
              'Tl',
              'Pb',
              'Bi',
              'Po',
              'At',
              'Rn',
              'Fr',
              'Ra',
              'Ac',
              'Th',
              'Pa',
              'U',
              'Np',
              'Pu',
              'Am',
              'Cm',
              'Bk',
              'Cf',
              'Es',
              'Fm',
              'Md',
              'No',
              'Lr' ] ] ] ],
      [ 'save_',
        '_diffrn_source.type',
        [ [ 'item_',
            '_item_description.description',
            '              The make, model or name of the source of radiation.' ],
          [ 'item_', '_item.name', '_diffrn_source.type' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_source_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'NSLS beamline X8C', 'Rigaku RU200' ] ] ] ],
      [ 'save_',
        '_diffrn_source.voltage',
        [ [ 'item_',
            '_item_description.description',
            '              The voltage in kilovolts at which the radiation source was\n               operated.' ],
          [ 'item_', '_item.name', '_diffrn_source.voltage' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_diffrn_source_voltage' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kilovolts' ] ] ],
      [ '@comment', '##########################' ],
      [ '@comment', '# DIFFRN_STANDARD_REFLN ##' ],
      [ '@comment', '##########################' ],
      [ 'save_',
        'diffrn_standard_refln',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_STANDARD_REFLN category record details\n               about the reflections treated as standards during the measurement\n               of a set of diffraction intensities.\n\n               Note that these are the individual standard reflections, not the\n               results of the analysis of the standard reflections.' ],
          [ 'item_', '_category.id', 'diffrn_standard_refln' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_diffrn_standard_refln.diffrn_id',
              '_diffrn_standard_refln.code' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    loop_\n    _diffrn_standard_refln.diffrn_id\n    _diffrn_standard_refln.code\n    _diffrn_standard_refln.index_h\n    _diffrn_standard_refln.index_k\n    _diffrn_standard_refln.index_l\n     s1 1  3  2  4\n     s1 1  1  9  1\n     s1 1  3  0 10',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_standard_refln.code',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying a reflection measured as a standard\n               reflection with the indices _diffrn_standard_refln.index_h,\n               _diffrn_standard_refln.index_k and\n               _diffrn_standard_refln.index_l. This is the same code as the\n               _diffrn_refln.standard_code in the DIFFRN_REFLN list.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_diffrn_standard_refln.code',
              'diffrn_standard_refln',
              'yes',
              '_diffrn_refln.standard_code',
              'diffrn_refln',
              'yes' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_standard_refln_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_diffrn_refln.standard_code', '_diffrn_standard_refln.code' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', 'c1', 'c2' ] ] ] ],
      [ 'save_',
        '_diffrn_standard_refln.diffrn_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn.id in the DIFFRN\n               category.' ],
          [ 'item_', '_item.name', '_diffrn_standard_refln.diffrn_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_diffrn_standard_refln.index_h',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index h of a standard reflection used in the diffraction\n               measurement process.' ],
          [ 'item_', '_item.name', '_diffrn_standard_refln.index_h' ],
          [ 'item_', '_item.category_id', 'diffrn_standard_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_standard_refln_index_h' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_standard_refln.index_k',
              '_diffrn_standard_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_standard_refln.index_k',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index k of a standard reflection used in the diffraction\n               measurement process.' ],
          [ 'item_', '_item.name', '_diffrn_standard_refln.index_k' ],
          [ 'item_', '_item.category_id', 'diffrn_standard_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_standard_refln_index_k' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_standard_refln.index_h',
              '_diffrn_standard_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_standard_refln.index_l',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index l of a standard reflection used in the diffraction\n               measurement process.' ],
          [ 'item_', '_item.name', '_diffrn_standard_refln.index_l' ],
          [ 'item_', '_item.category_id', 'diffrn_standard_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_standard_refln_index_l' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_diffrn_standard_refln.index_h',
              '_diffrn_standard_refln.index_k' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# DIFFRN_STANDARDS ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'diffrn_standards',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_STANDARDS category record details\n               about the set of standard reflections used to monitor intensity\n               stability during the measurement of diffraction intensities.\n\n               Note that these records describe properties common to the set of\n               standard reflections, not the standard reflections themselves.' ],
          [ 'item_', '_category.id', 'diffrn_standards' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_diffrn_standards.diffrn_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    _diffrn_standards.diffrn_id         \'s1\'\n    _diffrn_standards.number            3\n    _diffrn_standards.interval_time     120\n    _diffrn_standards.decay_%           0',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_standards.diffrn_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn.id in the DIFFRN\n               category.' ],
          [ 'item_', '_item.name', '_diffrn_standards.diffrn_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_diffrn_standards.decay_%',
        [ [ 'item_',
            '_item_description.description',
            '              The percentage decrease in the mean of the intensities\n               for the set of standard reflections from the start of the\n               measurement process to the end. This value usually\n               affords a measure of the overall decay in crystal quality\n               during the diffraction measurement process. Negative values\n               are used in exceptional instances where the final intensities\n               are greater than the initial ones.' ],
          [ 'item_', '_item.name', '_diffrn_standards.decay_%' ],
          [ 'item_', '_item.category_id', 'diffrn_standards' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_standards_decay_%' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '100.0', '100.0', '100.0', '.' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_diffrn_standards.interval_count',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflection intensities between the measurement of\n               standard reflection intensities.' ],
          [ 'item_', '_item.name', '_diffrn_standards.interval_count' ],
          [ 'item_', '_item.category_id', 'diffrn_standards' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_standards_interval_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_standards.interval_time',
        [ [ 'item_',
            '_item_description.description',
            '              The time in minutes between the measurement of standard\n               reflection intensities.' ],
          [ 'item_', '_item.name', '_diffrn_standards.interval_time' ],
          [ 'item_', '_item.category_id', 'diffrn_standards' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_standards_interval_time' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'minutes' ] ] ],
      [ 'save_',
        '_diffrn_standards.number',
        [ [ 'item_',
            '_item_description.description',
            '              The number of unique standard reflections used during the\n               measurement of  the diffraction intensities.' ],
          [ 'item_', '_item.name', '_diffrn_standards.number' ],
          [ 'item_', '_item.category_id', 'diffrn_standards' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_standards_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_diffrn_standards.scale_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation) of\n               the individual mean standard scales applied to the intensity\n               data.' ],
          [ 'item_', '_item.name', '_diffrn_standards.scale_sigma' ],
          [ 'item_', '_item.category_id', 'diffrn_standards' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_standards_scale_sigma' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '###########' ],
      [ '@comment', '# ENTITY ##' ],
      [ '@comment', '###########' ],
      [ 'save_',
        'entity',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ENTITY category record details (such as\n               chemical composition, name and source) about the molecular\n               entities that are present in the crystallographic structure.\n\n               Items in the various ENTITY subcategories provide a full\n               chemical description of these molecular entities.\n\n               Entities are of three types:  polymer, non-polymer and water.\n               Note that the water category includes only water;  ordered\n               solvent such as sulfate ion or acetone would be described as\n               individual non-polymer entities.\n\n               The ENTITY category is specific to macromolecular CIF\n               applications and replaces the function of the CHEMICAL category\n               in the CIF core.\n\n               It is important to remember that the ENTITY data are not the\n               result of the crystallographic experiment;  those results are\n               represented by the ATOM_SITE data items. ENTITY data items\n               describe the chemistry of the molecules under investigation\n               and can most usefully be thought of as the ideal groups to which\n               the structure is restrained or constrained during refinement.\n\n               It is also important to remember that entities do not correspond\n               directly to the enumeration of the contents of the asymmetric\n               unit. Entities are described only once, even in those structures\n               that contain multiple observations of an entity. The\n               STRUCT_ASYM data items, which reference the entity list,\n               describe and label the contents of the asymmetric unit.' ],
          [ 'item_', '_category.id', 'entity' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_entity.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _entity.id\n    _entity.type\n    _entity.formula_weight\n    _entity.details\n     1 polymer      10916\n    ;              The enzymatically competent form of HIV\n                   protease is a dimer. This entity\n                   corresponds to one monomer of an active dimer.\n    ;\n     2 non-polymer \'762\'  .\n     3 water        18    .\n',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_entity.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the entity.' ],
          [ 'item_', '_item.name', '_entity.details' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity.formula_weight',
        [ [ 'item_',
            '_item_description.description',
            '              Formula mass in daltons of the entity.' ],
          [ 'item_', '_item.name', '_entity.formula_weight' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1.0', '1.0', '1.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_entity.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _entity.id must uniquely identify a record in the\n               ENTITY list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_entity.id',
              'entity',
              'yes',
              '_atom_site.label_entity_id',
              'atom_site',
              'no',
              '_entity_keywords.entity_id',
              'entity_keywords',
              'yes',
              '_entity_link.entity_id_1',
              'entity_link',
              'yes',
              '_entity_link.entity_id_2',
              'entity_link',
              'yes',
              '_entity_name_com.entity_id',
              'entity_name_com',
              'yes',
              '_entity_name_sys.entity_id',
              'entity_name_sys',
              'yes',
              '_entity_poly.entity_id',
              'entity_poly',
              'yes',
              '_entity_poly_seq.entity_id',
              'entity_poly_seq',
              'yes',
              '_entity_src_gen.entity_id',
              'entity_src_gen',
              'yes',
              '_entity_src_nat.entity_id',
              'entity_src_nat',
              'yes',
              '_struct_asym.entity_id',
              'struct_asym',
              'yes',
              '_struct_ref.entity_id',
              'struct_ref',
              'yes' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_site.label_entity_id',
              '_entity.id',
              '_entity_keywords.entity_id',
              '_entity.id',
              '_entity_link.entity_id_1',
              '_entity.id',
              '_entity_link.entity_id_2',
              '_entity.id',
              '_entity_name_com.entity_id',
              '_entity.id',
              '_entity_name_sys.entity_id',
              '_entity.id',
              '_entity_poly.entity_id',
              '_entity.id',
              '_entity_poly_seq.entity_id',
              '_entity_poly.entity_id',
              '_entity_src_gen.entity_id',
              '_entity.id',
              '_entity_src_nat.entity_id',
              '_entity.id',
              '_struct_asym.entity_id',
              '_entity.id',
              '_struct_ref.entity_id',
              '_entity.id' ] ] ] ],
      [ 'save_',
        '_entity.src_method',
        [ [ 'item_',
            '_item_description.description',
            '              The method by which the sample for the entity was produced.\n               Entities isolated directly from natural sources (tissues, soil\n               samples etc.) are expected to have further information in the\n               ENTITY_SRC_NAT category. Entities isolated from genetically\n               manipulated sources are expected to have further information in\n               the ENTITY_SRC_GEN category.' ],
          [ 'item_', '_item.name', '_entity.src_method' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'nat',
              '                                 entity isolated from a natural source',
              'man',
              '                                 entity isolated from a genetically\n                                  manipulated source',
              'syn',
              '                                 entity obtained synthetically' ] ] ] ],
      [ 'save_',
        '_entity.type',
        [ [ 'item_',
            '_item_description.description',
            '              Defines the type of the entity.\n\n               Polymer entities are expected to have corresponding\n               ENTITY_POLY and associated entries.\n\n               Non-polymer entities are expected to have corresponding\n               CHEM_COMP and associated entries.\n\n               Water entities are not expected to have corresponding\n               entries in the ENTITY category.' ],
          [ 'item_', '_item.name', '_entity.type' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'polymer',
              'entity is a polymer',
              'non-polymer',
              'entity is not a polymer',
              'water',
              'water in the solvent model' ] ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# ENTITY_KEYWORDS ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'entity_keywords',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ENTITY_KEYWORDS category specify keywords\n               relevant to the molecular entities. Note that this list of\n               keywords is separate from the list that is used for the\n               STRUCT_BIOL data items and is intended to provide only the\n               information that one would know about the molecular entity *if\n               one did not know its structure*. Hence polypeptides are simply\n               polypeptides, not cytokines or beta-alpha-barrels, and\n               polyribonucleic acids are simply poly-RNA, not transfer-\n               RNA.' ],
          [ 'item_', '_category.id', 'entity_keywords' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_entity_keywords.entity_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _entity_keywords.entity_id\n    _entity_keywords.text\n      2  \'natural product, inhibitor, reduced peptide\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_entity_keywords.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_entity_keywords.entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_keywords.text',
        [ [ 'item_',
            '_item_description.description',
            '              Keywords describing this entity.' ],
          [ 'item_', '_item.name', '_entity_keywords.text' ],
          [ 'item_', '_item.category_id', 'entity_keywords' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'polypeptide', 'natural product', 'polysaccharide' ] ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# ENTITY_LINK ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'entity_link',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ENTITY_LINK category give details about\n               the links between entities.' ],
          [ 'item_', '_category.id', 'entity_link' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_entity_link.link_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_link_group' ] ] ] ],
      [ 'save_',
        '_entity_link.link_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_link.id in the\n               CHEM_LINK category.' ],
          [ 'item_', '_item.name', '_entity_link.link_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_link.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of a link between\n               chemical components in the structure.' ],
          [ 'item_', '_item.name', '_entity_link.details' ],
          [ 'item_', '_item.category_id', 'entity_link' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity_link.entity_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The entity ID of the first of the two entities joined by the\n               link.\n\n               This data item is a pointer to _entity.id in the ENTITY\n               category.' ],
          [ 'item_', '_item.name', '_entity_link.entity_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_link.entity_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The entity ID of the second of the two entities joined by the\n               link.\n\n               This data item is a pointer to _entity.id in the ENTITY\n               category.' ],
          [ 'item_', '_item.name', '_entity_link.entity_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_link.entity_seq_num_1',
        [ [ 'item_',
            '_item_description.description',
            '              For a polymer entity, the sequence number in the first of\n               the two entities containing the link.\n\n               This data item is a pointer to _entity_poly_seq.num in the\n               ENTITY_POLY_SEQ category.' ],
          [ 'item_', '_item.name', '_entity_link.entity_seq_num_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_entity_link.entity_seq_num_2',
        [ [ 'item_',
            '_item_description.description',
            '              For a polymer entity, the sequence number in the second of\n               the two entities containing the link.\n\n               This data item is a pointer to _entity_poly_seq.num in the\n               ENTITY_POLY_SEQ category.' ],
          [ 'item_', '_item.name', '_entity_link.entity_seq_num_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# ENTITY_NAME_COM ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'entity_name_com',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ENTITY_NAME_COM category record the common name\n               or names associated with the entity. In some cases, the entity\n               name may not be the same as the name of the biological structure.\n               For example, haemoglobin alpha chain would be the entity common\n               name, not haemoglobin.' ],
          [ 'item_', '_category.id', 'entity_name_com' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_entity_name_com.entity_id', '_entity_name_com.name' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _entity_name_com.entity_id\n    _entity_name_com.name\n     1  \'HIV-1 protease monomer\'\n     1  \'HIV-1 PR monomer\'\n     2  \'acetyl-pepstatin\'\n     2  \'acetyl-Ile-Val-Asp-Statine-Ala-Ile-Statine\'\n     3  \'water\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_entity_name_com.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_entity_name_com.entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_name_com.name',
        [ [ 'item_',
            '_item_description.description',
            '              A common name for the entity.' ],
          [ 'item_', '_item.name', '_entity_name_com.name' ],
          [ 'item_', '_item.category_id', 'entity_name_com' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'HIV protease monomer',
              'hemoglobin alpha chain',
              '2-fluoro-1,4-dichloro benzene',
              'arbutin' ] ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# ENTITY_NAME_SYS ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'entity_name_sys',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ENTITY_NAME_SYS category record the systematic\n               name or names associated with the entity and the system that\n               was used to construct the systematic name. In some cases, the\n               entity name may not be the same as the name of the biological\n               structure.' ],
          [ 'item_', '_category.id', 'entity_name_sys' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_entity_name_sys.entity_id', '_entity_name_sys.name' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _entity_name_sys.entity_id\n    _entity_name_sys.name\n      1  \'EC 3.4.23.16\'\n      2  \'acetyl-Ile-Val-Asp-Sta-Ala-Ile-Sta\'\n      3  water',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_entity_name_sys.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_entity_name_sys.entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_name_sys.name',
        [ [ 'item_',
            '_item_description.description',
            '              The systematic name for the entity.' ],
          [ 'item_', '_item.name', '_entity_name_sys.name' ],
          [ 'item_', '_item.category_id', 'entity_name_sys' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'hydroquinone-beta-D-pyranoside',
              'EC 2.1.1.1',
              '2-fluoro-1,4-dichlorobenzene' ] ] ] ],
      [ 'save_',
        '_entity_name_sys.system',
        [ [ 'item_',
            '_item_description.description',
            '              The system used to generate the systematic name of the entity.' ],
          [ 'item_', '_item.name', '_entity_name_sys.system' ],
          [ 'item_', '_item.category_id', 'entity_name_sys' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Chemical Abstracts conventions',
              'enzyme convention',
              'Sigma catalog' ] ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# ENTITY_POLY ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'entity_poly',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ENTITY_POLY category record details about the\n               polymer, such as the type of the polymer, the number of\n               monomers and whether it has nonstandard features.' ],
          [ 'item_', '_category.id', 'entity_poly' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_entity_poly.entity_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _entity_poly.entity_id\n    _entity_poly.type\n    _entity_poly.nstd_chirality\n    _entity_poly.nstd_linkage\n    _entity_poly.nstd_monomer\n    _entity_poly.type_details\n     1  polypeptide(L)  no  no  no  .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_entity_poly.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_entity_poly.entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_poly.nstd_chirality',
        [ [ 'item_',
            '_item_description.description',
            '              A flag to indicate whether the polymer contains at least\n               one monomer unit with chirality different from that specified in\n               _entity_poly.type.' ],
          [ 'item_', '_item.name', '_entity_poly.nstd_chirality' ],
          [ 'item_', '_item.category_id', 'entity_poly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              '                                 polymer contains no monomers with different\n                                  chirality',
              'n',
              '                                 abbreviation for "no"',
              'yes',
              '                                 polymer contains at least one monomer with\n                                  different chirality',
              'y',
              '                                 abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_entity_poly.nstd_linkage',
        [ [ 'item_',
            '_item_description.description',
            '              A flag to indicate whether the polymer contains at least\n               one monomer-to-monomer link different from that implied by\n               _entity_poly.type.' ],
          [ 'item_', '_item.name', '_entity_poly.nstd_linkage' ],
          [ 'item_', '_item.category_id', 'entity_poly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'polymer contains no different links',
              'n',
              'abbreviation for "no"',
              'yes',
              '                                 polymer contains at least one different\n                                  link',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_entity_poly.nstd_monomer',
        [ [ 'item_',
            '_item_description.description',
            '              A flag to indicate whether the polymer contains at least\n               one monomer that is not considered standard.' ],
          [ 'item_', '_item.name', '_entity_poly.nstd_monomer' ],
          [ 'item_', '_item.category_id', 'entity_poly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'polymer contains no nonstandard monomers',
              'n',
              'abbreviation for "no"',
              'yes',
              '                                 polymer contains at least one nonstandard\n                                  monomer',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_entity_poly.number_of_monomers',
        [ [ 'item_',
            '_item_description.description',
            '              The number of monomers in the polymer.' ],
          [ 'item_', '_item.name', '_entity_poly.number_of_monomers' ],
          [ 'item_', '_item.category_id', 'entity_poly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_entity_poly.type',
        [ [ 'item_',
            '_item_description.description',
            '              The type of the polymer.' ],
          [ 'item_', '_item.name', '_entity_poly.type' ],
          [ 'item_', '_item.category_id', 'entity_poly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'polypeptide(D)',
              'polypeptide(L)',
              'polydeoxyribonucleotide',
              'polyribonucleotide',
              'polysaccharide(D)',
              'polysaccharide(L)',
              'other' ] ] ] ],
      [ 'save_',
        '_entity_poly.type_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the polymer type.' ],
          [ 'item_', '_item.name', '_entity_poly.type_details' ],
          [ 'item_', '_item.category_id', 'entity_poly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'monomer Ala 16 is a D-amino acid',
              'the oligomer contains alternating RNA and DNA units' ] ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# ENTITY_POLY_SEQ ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'entity_poly_seq',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ENTITY_POLY_SEQ category specify the sequence\n               of monomers in a polymer. Allowance is made for the possibility\n               of microheterogeneity in a sample by allowing a given sequence\n               number to be correlated with more than one monomer ID. The\n               corresponding ATOM_SITE entries should reflect this\n               heterogeneity.' ],
          [ 'item_', '_category.id', 'entity_poly_seq' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_entity_poly_seq.entity_id',
              '_entity_poly_seq.num',
              '_entity_poly_seq.mon_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _entity_poly_seq.entity_id\n    _entity_poly_seq.num\n    _entity_poly_seq.mon_id\n      1  1  PRO   1  2  GLN   1  3  ILE   1  4  THR   1  5  LEU\n      1  6  TRP   1  7  GLN   1  8  ARG   1  9  PRO   1 10  LEU\n      1 11  VAL   1 12  THR   1 13  ILE   1 14  LYS   1 15  ILE\n      1 16  GLY   1 17  GLY   1 18  GLN   1 19  LEU   1 20  LYS\n      1 21  GLU   1 22  ALA   1 23  LEU   1 24  LEU   1 25  ASP\n    # - - - - data truncated for brevity - - - -',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_entity_poly_seq.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_entity_poly_seq.entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_poly_seq.hetero',
        [ [ 'item_',
            '_item_description.description',
            '              A flag to indicate whether this monomer in the polymer is\n               heterogeneous in sequence. This would be rare.' ],
          [ 'item_', '_item.name', '_entity_poly_seq.hetero' ],
          [ 'item_', '_item.category_id', 'entity_poly_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'sequence is not heterogeneous at this monomer',
              'n',
              'abbreviation for "no"',
              'yes',
              'sequence is heterogeneous at this monomer',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_entity_poly_seq.mon_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_entity_poly_seq.mon_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_poly_seq.num',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _entity_poly_seq.num must uniquely and sequentially\n               identify a record in the ENTITY_POLY_SEQ list.\n\n               Note that this item must be a number and that the sequence\n               numbers must progress in increasing numerical order.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_entity_poly_seq.num',
              'entity_poly_seq',
              'yes',
              '_atom_site.label_seq_id',
              'atom_site',
              'no',
              '_entity_link.entity_seq_num_1',
              'entity_link',
              'no',
              '_entity_link.entity_seq_num_2',
              'entity_link',
              'no',
              '_geom_angle.atom_site_label_seq_id_1',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_label_seq_id_2',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_label_seq_id_3',
              'geom_angle',
              'no',
              '_geom_bond.atom_site_label_seq_id_1',
              'geom_bond',
              'no',
              '_geom_bond.atom_site_label_seq_id_2',
              'geom_bond',
              'no',
              '_geom_contact.atom_site_label_seq_id_1',
              'geom_contact',
              'no',
              '_geom_contact.atom_site_label_seq_id_2',
              'geom_contact',
              'no',
              '_geom_hbond.atom_site_label_seq_id_A',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_label_seq_id_D',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_label_seq_id_H',
              'geom_hbond',
              'no',
              '_geom_torsion.atom_site_label_seq_id_1',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_seq_id_2',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_seq_id_3',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_seq_id_4',
              'geom_torsion',
              'no',
              '_struct_conf.beg_label_seq_id',
              'struct_conf',
              'yes',
              '_struct_conf.end_label_seq_id',
              'struct_conf',
              'yes',
              '_struct_conn.ptnr1_label_seq_id',
              'struct_conn',
              'yes',
              '_struct_conn.ptnr2_label_seq_id',
              'struct_conn',
              'yes',
              '_struct_mon_nucl.label_seq_id',
              'struct_mon_nucl',
              'yes',
              '_struct_mon_prot.label_seq_id',
              'struct_mon_prot',
              'yes',
              '_struct_mon_prot_cis.label_seq_id',
              'struct_mon_prot_cis',
              'yes',
              '_struct_ncs_dom_lim.beg_label_seq_id',
              'struct_ncs_dom_lim',
              'yes',
              '_struct_ncs_dom_lim.end_label_seq_id',
              'struct_ncs_dom_lim',
              'yes',
              '_struct_ref_seq.seq_align_beg',
              'struct_ref_seq',
              'yes',
              '_struct_ref_seq.seq_align_end',
              'struct_ref_seq',
              'yes',
              '_struct_ref_seq_dif.seq_num',
              'struct_ref_seq_dif',
              'yes',
              '_struct_sheet_hbond.range_1_beg_label_seq_id',
              'struct_sheet_hbond',
              'yes',
              '_struct_sheet_hbond.range_1_end_label_seq_id',
              'struct_sheet_hbond',
              'yes',
              '_struct_sheet_hbond.range_2_beg_label_seq_id',
              'struct_sheet_hbond',
              'yes',
              '_struct_sheet_hbond.range_2_end_label_seq_id',
              'struct_sheet_hbond',
              'yes',
              '_struct_sheet_range.beg_label_seq_id',
              'struct_sheet_range',
              'yes',
              '_struct_sheet_range.end_label_seq_id',
              'struct_sheet_range',
              'yes',
              '_struct_site_gen.label_seq_id',
              'struct_site_gen',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_site.label_seq_id',
              '_entity_poly_seq.num',
              '_entity_link.entity_seq_num_1',
              '_entity_poly_seq.num',
              '_entity_link.entity_seq_num_2',
              '_entity_poly_seq.num',
              '_geom_angle.atom_site_label_seq_id_1',
              '_atom_site.label_seq_id',
              '_geom_angle.atom_site_label_seq_id_2',
              '_atom_site.label_seq_id',
              '_geom_angle.atom_site_label_seq_id_3',
              '_atom_site.label_seq_id',
              '_geom_bond.atom_site_label_seq_id_1',
              '_atom_site.label_seq_id',
              '_geom_bond.atom_site_label_seq_id_2',
              '_atom_site.label_seq_id',
              '_geom_contact.atom_site_label_seq_id_1',
              '_atom_site.label_seq_id',
              '_geom_contact.atom_site_label_seq_id_2',
              '_atom_site.label_seq_id',
              '_geom_hbond.atom_site_label_seq_id_A',
              '_atom_site.label_seq_id',
              '_geom_hbond.atom_site_label_seq_id_D',
              '_atom_site.label_seq_id',
              '_geom_hbond.atom_site_label_seq_id_H',
              '_atom_site.label_seq_id',
              '_geom_torsion.atom_site_label_seq_id_1',
              '_atom_site.label_seq_id',
              '_geom_torsion.atom_site_label_seq_id_2',
              '_atom_site.label_seq_id',
              '_geom_torsion.atom_site_label_seq_id_3',
              '_atom_site.label_seq_id',
              '_geom_torsion.atom_site_label_seq_id_4',
              '_atom_site.label_seq_id',
              '_struct_conf.beg_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_conf.end_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_conn.ptnr1_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_conn.ptnr2_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_mon_nucl.label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_mon_prot.label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_mon_prot_cis.label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_ncs_dom_lim.beg_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_ncs_dom_lim.end_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_ref_seq.seq_align_beg',
              '_entity_poly_seq.num',
              '_struct_ref_seq.seq_align_end',
              '_entity_poly_seq.num',
              '_struct_ref_seq_dif.seq_num',
              '_entity_poly_seq.num',
              '_struct_sheet_hbond.range_1_beg_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_sheet_hbond.range_1_end_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_sheet_hbond.range_2_beg_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_sheet_hbond.range_2_end_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_sheet_range.beg_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_sheet_range.end_label_seq_id',
              '_atom_site.label_seq_id',
              '_struct_site_gen.label_seq_id',
              '_atom_site.label_seq_id' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# ENTITY_SRC_GEN ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'entity_src_gen',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ENTITY_SRC_GEN category record details of\n               the source from which the entity was obtained in cases\n               where the source was genetically manipulated.  The\n               following are treated separately:  items pertaining to the tissue\n               from which the gene was obtained, items pertaining to the host\n               organism for gene expression and items pertaining to the actual\n               producing organism (plasmid).' ],
          [ 'item_', '_category.id', 'entity_src_gen' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_entity_src_gen.entity_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _entity_src_gen.entity_id\n    _entity_src_gen.gene_src_common_name\n    _entity_src_gen.gene_src_genus\n    _entity_src_gen.gene_src_species\n    _entity_src_gen.gene_src_strain\n    _entity_src_gen.host_org_common_name\n    _entity_src_gen.host_org_genus\n    _entity_src_gen.host_org_species\n    _entity_src_gen.plasmid_name\n     1  \'HIV-1\'  \'?\'  \'?\'  \'NY-5\'\n        \'bacteria\'  \'Escherichia\'  \'coli\'  \'pB322\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_entity_src_gen.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_entity_src_gen.entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_src_gen.gene_src_common_name',
        [ [ 'item_',
            '_item_description.description',
            '              The common name of the natural organism from which the gene was\n               obtained.' ],
          [ 'item_', '_item.name', '_entity_src_gen.gene_src_common_name' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'man', 'yeast', 'bacteria' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.gene_src_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the natural organism from\n               which the gene was obtained.' ],
          [ 'item_', '_item.name', '_entity_src_gen.gene_src_details' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity_src_gen.gene_src_genus',
        [ [ 'item_',
            '_item_description.description',
            '              The genus of the natural organism from which the gene was\n               obtained.' ],
          [ 'item_', '_item.name', '_entity_src_gen.gene_src_genus' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Homo', 'Saccharomyces', 'Escherichia' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.gene_src_species',
        [ [ 'item_',
            '_item_description.description',
            '              The species of the natural organism from which the gene was\n               obtained.' ],
          [ 'item_', '_item.name', '_entity_src_gen.gene_src_species' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'sapiens', 'cerevisiae', 'coli' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.gene_src_strain',
        [ [ 'item_',
            '_item_description.description',
            '              The strain of the natural organism from which the gene was\n               obtained, if relevant.' ],
          [ 'item_', '_item.name', '_entity_src_gen.gene_src_strain' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'DH5a', 'BMH 71-18' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.gene_src_tissue',
        [ [ 'item_',
            '_item_description.description',
            '              The tissue of the natural organism from which the gene was\n               obtained.' ],
          [ 'item_', '_item.name', '_entity_src_gen.gene_src_tissue' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'heart', 'liver', 'eye lens' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.gene_src_tissue_fraction',
        [ [ 'item_',
            '_item_description.description',
            '              The subcellular fraction of the tissue of the natural organism\n               from which the gene was obtained.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.gene_src_tissue_fraction' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'mitochondria', 'nucleus', 'membrane' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.host_org_common_name',
        [ [ 'item_',
            '_item_description.description',
            '              The common name of the organism that served as host for the\n               production of the entity.' ],
          [ 'item_', '_item.name', '_entity_src_gen.host_org_common_name' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'yeast', 'bacteria' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.host_org_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the organism that served as\n               host for the production of the entity.' ],
          [ 'item_', '_item.name', '_entity_src_gen.host_org_details' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity_src_gen.host_org_genus',
        [ [ 'item_',
            '_item_description.description',
            '              The genus of the organism that served as host for the production\n               of the entity.' ],
          [ 'item_', '_item.name', '_entity_src_gen.host_org_genus' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Saccharomyces', 'Escherichia' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.host_org_species',
        [ [ 'item_',
            '_item_description.description',
            '              The species of the organism that served as host for the\n               production of the entity.' ],
          [ 'item_', '_item.name', '_entity_src_gen.host_org_species' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'cerevisiae', 'coli' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.host_org_strain',
        [ [ 'item_',
            '_item_description.description',
            '              The strain of the organism that served as host for the\n               production of the entity.' ],
          [ 'item_', '_item.name', '_entity_src_gen.host_org_strain' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'DH5a', 'BMH 71-18' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.plasmid_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the plasmid that produced the\n               entity in the host organism.' ],
          [ 'item_', '_item.name', '_entity_src_gen.plasmid_details' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity_src_gen.plasmid_name',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the plasmid that produced the entity in the host\n               organism.' ],
          [ 'item_', '_item.name', '_entity_src_gen.plasmid_name' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'pET3C', 'pT123sab' ] ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# ENTITY_SRC_NAT ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'entity_src_nat',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ENTITY_SRC_NAT category record details of\n               the source from which the entity was obtained in cases\n               where the entity was isolated directly from a natural tissue.' ],
          [ 'item_', '_category.id', 'entity_src_nat' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_entity_src_nat.entity_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _entity_src_nat.entity_id\n    _entity_src_nat.common_name\n    _entity_src_nat.genus\n    _entity_src_nat.species\n    _entity_src_nat.details\n     2  \'bacteria\'  \'Actinomycetes\' \'?\'\n    ;    Acetyl-pepstatin was isolated by Dr. K. Oda, Osaka\n         Prefecture University, and provided to us by Dr. Ben\n         Dunn, University of Florida, and Dr. J. Kay, University\n         of Wales.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_entity_src_nat.common_name',
        [ [ 'item_',
            '_item_description.description',
            '              The common name of the organism from which the entity\n               was isolated.' ],
          [ 'item_', '_item.name', '_entity_src_nat.common_name' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'man', 'yeast', 'bacteria' ] ] ] ],
      [ 'save_',
        '_entity_src_nat.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the organism from which the\n               entity was isolated.' ],
          [ 'item_', '_item.name', '_entity_src_nat.details' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity_src_nat.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_entity_src_nat.entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_src_nat.genus',
        [ [ 'item_',
            '_item_description.description',
            '              The genus of the organism from which the entity was isolated.' ],
          [ 'item_', '_item.name', '_entity_src_nat.genus' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Homo', 'Saccharomyces', 'Escherichia' ] ] ] ],
      [ 'save_',
        '_entity_src_nat.species',
        [ [ 'item_',
            '_item_description.description',
            '              The species of the organism from which the entity was isolated.' ],
          [ 'item_', '_item.name', '_entity_src_nat.species' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'sapiens', 'cerevisiae', 'coli' ] ] ] ],
      [ 'save_',
        '_entity_src_nat.strain',
        [ [ 'item_',
            '_item_description.description',
            '              The strain of the organism from which the entity was isolated.' ],
          [ 'item_', '_item.name', '_entity_src_nat.strain' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'DH5a', 'BMH 71-18' ] ] ] ],
      [ 'save_',
        '_entity_src_nat.tissue',
        [ [ 'item_',
            '_item_description.description',
            '              The tissue of the organism from which the entity was isolated.' ],
          [ 'item_', '_item.name', '_entity_src_nat.tissue' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'heart', 'liver', 'eye lens' ] ] ] ],
      [ 'save_',
        '_entity_src_nat.tissue_fraction',
        [ [ 'item_',
            '_item_description.description',
            '              The subcellular fraction of the tissue of the organism from\n               which the entity was isolated.' ],
          [ 'item_', '_item.name', '_entity_src_nat.tissue_fraction' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'mitochondria', 'nucleus', 'membrane' ] ] ] ],
      [ '@comment', '##########' ],
      [ '@comment', '# ENTRY ##' ],
      [ '@comment', '##########' ],
      [ 'save_',
        'entry',
        [ [ 'item_',
            '_category.description',
            '              There is only one item in the ENTRY category, _entry.id. This\n               data item gives a name to this entry and is indirectly a key to\n               the categories (such as CELL, GEOM, EXPTL) that describe\n               information pertinent to the entire data block.' ],
          [ 'item_', '_category.id', 'entry' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_entry.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entry_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _entry.id                              \'5HVP\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    _entry.id                              \'TOZ\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_entry.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _entry.id identifies the data block.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_entry.id',
              'entry',
              'yes',
              '_atom_sites.entry_id',
              'atom_sites',
              'yes',
              '_cell.entry_id',
              'cell',
              'yes',
              '_cell_measurement.entry_id',
              'cell_measurement',
              'yes',
              '_chemical.entry_id',
              'chemical',
              'yes',
              '_chemical_formula.entry_id',
              'chemical_formula',
              'yes',
              '_computing.entry_id',
              'computing',
              'yes',
              '_database.entry_id',
              'database',
              'yes',
              '_database_PDB_matrix.entry_id',
              'database_PDB_matrix',
              'yes',
              '_entry_link.entry_id',
              'entry_link',
              'yes',
              '_exptl.entry_id',
              'exptl',
              'yes',
              '_geom.entry_id',
              'geom',
              'yes',
              '_journal.entry_id',
              'journal',
              'yes',
              '_phasing_averaging.entry_id',
              'phasing_averaging',
              'yes',
              '_phasing_isomorphous.entry_id',
              'phasing_isomorphous',
              'yes',
              '_phasing_MAD.entry_id',
              'phasing_MAD',
              'yes',
              '_phasing_MIR.entry_id',
              'phasing_MIR',
              'yes',
              '_publ.entry_id',
              'publ',
              'yes',
              '_publ_manuscript_incl.entry_id',
              'publ_manuscript_incl',
              'yes',
              '_refine.entry_id',
              'refine',
              'yes',
              '_refine_analyze.entry_id',
              'refine_analyze',
              'yes',
              '_reflns.entry_id',
              'reflns',
              'yes',
              '_struct.entry_id',
              'struct',
              'yes',
              '_struct_keywords.entry_id',
              'struct_keywords',
              'yes',
              '_struct_mon_details.entry_id',
              'struct_mon_details',
              'yes',
              '_symmetry.entry_id',
              'symmetry',
              'yes' ] ],
          [ 'item_', '_item_aliases.alias_name', '_audit_block_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_sites.entry_id',
              '_entry.id',
              '_cell.entry_id',
              '_entry.id',
              '_cell_measurement.entry_id',
              '_entry.id',
              '_chemical.entry_id',
              '_entry.id',
              '_chemical_formula.entry_id',
              '_entry.id',
              '_computing.entry_id',
              '_entry.id',
              '_database.entry_id',
              '_entry.id',
              '_database_PDB_matrix.entry_id',
              '_entry.id',
              '_entry_link.entry_id',
              '_entry.id',
              '_exptl.entry_id',
              '_entry.id',
              '_geom.entry_id',
              '_entry.id',
              '_journal.entry_id',
              '_entry.id',
              '_phasing_averaging.entry_id',
              '_entry.id',
              '_phasing_isomorphous.entry_id',
              '_entry.id',
              '_phasing_MAD.entry_id',
              '_entry.id',
              '_phasing_MIR.entry_id',
              '_entry.id',
              '_publ.entry_id',
              '_entry.id',
              '_publ_manuscript_incl.entry_id',
              '_entry.id',
              '_refine.entry_id',
              '_entry.id',
              '_refine_analyze.entry_id',
              '_entry.id',
              '_reflns.entry_id',
              '_entry.id',
              '_struct.entry_id',
              '_entry.id',
              '_struct_keywords.entry_id',
              '_entry.id',
              '_struct_mon_details.entry_id',
              '_entry.id',
              '_symmetry.entry_id',
              '_entry.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        'entry_link',
        [ [ 'item_',
            '_category.description',
            '              Data items in the ENTRY_LINK category record the\n               relationships between the current data block\n               identified by _entry.id and other data blocks\n               within the current file which may be referenced\n               in the current data block.' ],
          [ 'item_', '_category.id', 'entry_link' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_entry_link.id', '_entry_link.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entry_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - example file for the one-dimensional incommensurately\n                modulated structure of K~2~SeO~4~.',
              '\n    loop_\n    _entry_link.id\n    _entry_link.entry_id\n    _entry_link.details\n     KSE_COM  KSE_TEXT\n              \'experimental data common to ref./mod. structures\'\n     KSE_REF  KSE_TEXT  \'reference structure\'\n     KSE_MOD  KSE_TEXT  \'modulated structure\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_entry_link.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_entry_link.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entry_link.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _entry_link.id identifies a data block\n               related to the current data block.' ],
          [ 'item_', '_item.name', '_entry_link.id' ],
          [ 'item_', '_item.category_id', 'entry_link' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_audit_link_block_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_entry_link.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the relationship between the data blocks\n               identified by _entry_link.id and _entry_link.entry_id.' ],
          [ 'item_', '_item.name', '_entry_link.details' ],
          [ 'item_', '_item.category_id', 'entry_link' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_audit_link_block_description' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '##########' ],
      [ '@comment', '# EXPTL ##' ],
      [ '@comment', '##########' ],
      [ 'save_',
        'exptl',
        [ [ 'item_',
            '_category.description',
            '              Data items in the EXPTL category record details about the\n               experimental work prior to the intensity measurements and\n               details about the absorption-correction technique employed.' ],
          [ 'item_', '_category.id', 'exptl' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_exptl.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'exptl_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for Yb(S-C5H4N)2(THF)4.',
              '\n    _exptl.entry_id                    datablock1\n    _exptl.absorpt_coefficient_mu      1.22\n    _exptl.absorpt_correction_T_max    0.896\n    _exptl.absorpt_correction_T_min    0.802\n    _exptl.absorpt_correction_type     integration\n    _exptl.absorpt_process_details\n    ; Gaussian grid method from SHELX76\n      Sheldrick, G. M., "SHELX-76: structure determination and\n      refinement program", Cambridge University, UK, 1976\n    ;\n    _exptl.crystals_number             1\n    _exptl.details\n    ; Enraf-Nonius LT2 liquid nitrogen variable-temperature\n      device used\n    ;\n    _exptl.method             \'single-crystal x-ray diffraction\'\n    _exptl.method_details\n    ; graphite monochromatized Cu K(alpha) fixed tube and\n      Enraf-Nonius CAD4 diffractometer used\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_exptl.absorpt_coefficient_mu',
        [ [ 'item_',
            '_item_description.description',
            '              The absorption coefficient mu in reciprocal millimetres\n               calculated from the atomic content of the cell, the density and\n               the radiation wavelength.' ],
          [ 'item_', '_item.name', '_exptl.absorpt_coefficient_mu' ],
          [ 'item_', '_item.category_id', 'exptl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_absorpt_coefficient_mu' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'reciprocal_millimetres' ] ] ],
      [ 'save_',
        '_exptl.absorpt_correction_T_max',
        [ [ 'item_',
            '_item_description.description',
            '              The maximum transmission factor for the crystal and radiation.\n               The maximum and minimum transmission factors are also referred\n               to as the absorption correction\n               A or 1/A*.' ],
          [ 'item_', '_item.name', '_exptl.absorpt_correction_T_max' ],
          [ 'item_', '_item.category_id', 'exptl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_absorpt_correction_T_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '1.0', '1.0', '1.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_exptl.absorpt_correction_T_min',
        [ [ 'item_',
            '_item_description.description',
            '              The minimum transmission factor for the crystal and radiation.\n               The maximum and minimum transmission factors are also referred\n               to as the absorption correction\n               A or 1/A*.' ],
          [ 'item_', '_item.name', '_exptl.absorpt_correction_T_min' ],
          [ 'item_', '_item.category_id', 'exptl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_absorpt_correction_T_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '1.0', '1.0', '1.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_exptl.absorpt_correction_type',
        [ [ 'item_',
            '_item_description.description',
            '              The absorption correction type and method. The value\n               \'empirical\' should NOT be used unless more detailed\n               information is not available.' ],
          [ 'item_', '_item.name', '_exptl.absorpt_correction_type' ],
          [ 'item_', '_item.category_id', 'exptl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_absorpt_correction_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'analytical',
              'analytical from crystal shape',
              'cylinder',
              'cylindrical',
              'empirical',
              'empirical from intensities',
              'gaussian',
              'Gaussian from crystal shape',
              'integration',
              'integration from crystal shape',
              'multi-scan',
              'symmetry-related measurements',
              'none',
              'no correction applied',
              'numerical',
              'numerical from crystal shape',
              'psi-scan',
              'psi-scan corrections',
              'refdelf',
              'refined from delta-F',
              'sphere',
              'spherical' ] ] ] ],
      [ 'save_',
        '_exptl.absorpt_process_details',
        [ [ 'item_',
            '_item_description.description',
            '              Description of the absorption process applied to the\n               intensities. A literature reference should be supplied for\n               psi-scan techniques.' ],
          [ 'item_', '_item.name', '_exptl.absorpt_process_details' ],
          [ 'item_', '_item.category_id', 'exptl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_absorpt_process_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'Tompa analytical' ] ] ],
      [ 'save_',
        '_exptl.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_exptl.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_exptl.crystals_number',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of crystals used in the  measurement of\n               intensities.' ],
          [ 'item_', '_item.name', '_exptl.crystals_number' ],
          [ 'item_', '_item.category_id', 'exptl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_exptl_crystals_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_exptl.details',
        [ [ 'item_',
            '_item_description.description',
            '              Any special information about the experimental work prior to the\n               intensity measurement. See also _exptl_crystal.preparation.' ],
          [ 'item_', '_item.name', '_exptl.details' ],
          [ 'item_', '_item.category_id', 'exptl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_exptl_special_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_exptl.method',
        [ [ 'item_',
            '_item_description.description',
            '              The method used in the experiment.' ],
          [ 'item_', '_item.name', '_exptl.method' ],
          [ 'item_', '_item.category_id', 'exptl' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'single-crystal x-ray diffraction',
              'single-crystal neutron diffraction',
              'single-crystal electron diffraction',
              'fiber x-ray diffraction',
              'fiber neutron diffraction',
              'fiber electron diffraction',
              'single-crystal joint x-ray and neutron diffraction',
              'single-crystal joint x-ray and electron diffraction',
              'solution nmr',
              'solid-state nmr',
              'theoretical model',
              'other' ] ] ] ],
      [ 'save_',
        '_exptl.method_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the experimental method.' ],
          [ 'item_', '_item.name', '_exptl.method_details' ],
          [ 'item_', '_item.category_id', 'exptl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '29 structures', 'minimized average structure' ] ] ] ],
      [ '@comment', '##################' ],
      [ '@comment', '# EXPTL_CRYSTAL ##' ],
      [ '@comment', '##################' ],
      [ 'save_',
        'exptl_crystal',
        [ [ 'item_',
            '_category.description',
            '              Data items in the EXPTL_CRYSTAL category record the results of\n               experimental measurements on the crystal or crystals used,\n               such as shape, size or density.' ],
          [ 'item_', '_category.id', 'exptl_crystal' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_exptl_crystal.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'exptl_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for Yb(S-C5H4N)2(THF)4.',
              '\n    _exptl_crystal.id                  xst2l\n    _exptl_crystal.colour              \'pale yellow\'\n    _exptl_crystal.density_diffrn      1.113\n    _exptl_crystal.density_Matthews    1.01\n    _exptl_crystal.density_meas        1.11\n    _exptl_crystal.density_meas_temp   294.5\n    _exptl_crystal.density_method      \'neutral buoyancy\'\n    _exptl_crystal.density_percent_sol 0.15\n                                   # P = 1 - (1.23*N*MMass) / V\n    _exptl_crystal.description         \'hexagonal rod, uncut\'\n    _exptl_crystal.F_000               202\n    _exptl_crystal.preparation\n    ; hanging drop, crystal soaked in 10% ethylene glycol for\n      10 h, then placed in nylon loop at data collection time\n    ;\n    _exptl_crystal.size_max            0.30\n    _exptl_crystal.size_mid            0.20\n    _exptl_crystal.size_min            0.05\n    _exptl_crystal.size_rad            0.025',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - using separate items to define upper and lower\n                limits for a value.',
              '   _exptl_crystal.density_meas_gt       2.5\n    _exptl_crystal.density_meas_lt       5.0',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 3 - here the density was measured at some\n                    unspecified temperature below room temperature.',
              '   _exptl_crystal.density_meas_temp_lt 300',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_exptl_crystal.colour',
        [ [ 'item_',
            '_item_description.description',
            '              The colour of the crystal.' ],
          [ 'item_', '_item.name', '_exptl_crystal.colour' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_exptl_crystal_colour' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'dark green' ] ] ],
      [ 'save_',
        '_exptl_crystal.density_diffrn',
        [ [ 'item_',
            '_item_description.description',
            '              Density values calculated from the crystal cell and contents. The\n               units are megagrams per cubic metre (grams per cubic centimetre).' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_diffrn' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_density_diffrn' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_exptl_crystal.density_Matthews',
        [ [ 'item_',
            '_item_description.description',
            '              The density of the crystal, expressed as the ratio of the\n               volume of the asymmetric unit to the molecular mass of a\n               monomer of the structure, in units of angstroms^3^ per dalton.\n\n               Ref: Matthews, B. W. (1968). J. Mol. Biol. 33, 491-497.' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_Matthews' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', 'save__exptl_crystal.density_meas' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              Density values measured using standard chemical and physical' ],
      [ '@comment',
        '               methods. The units are megagrams per cubic metre (grams per' ],
      [ '@comment', '               cubic centimetre).' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_exptl_crystal.density_meas\'' ],
      [ '@comment',
        '    _item.category_id             exptl_crystal' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '    _item_aliases.alias_name    \'_exptl_crystal_density_meas\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_range.maximum' ],
      [ '@comment', '    _item_range.minimum            .    0.0' ],
      [ '@comment', '                                  0.0   0.0' ],
      [ '@comment', '    _item_type.code               float' ],
      [ '@comment', '     save_' ],
      [ '@comment', 'save__exptl_crystal.density_meas_temp' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              The temperature in kelvins at which _exptl_crystal.density_meas' ],
      [ '@comment', '               was determined.' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_exptl_crystal.density_meas_temp\'' ],
      [ '@comment',
        '    _item.category_id             exptl_crystal' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '    _item_aliases.alias_name    \'_exptl_crystal_density_meas_temp\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_range.maximum' ],
      [ '@comment', '    _item_range.minimum            .    0.0' ],
      [ '@comment', '                                  0.0   0.0' ],
      [ '@comment', '    _item_type.code               float' ],
      [ '@comment', '    _item_units.code              kelvins' ],
      [ '@comment', '     save_' ],
      [ 'save_',
        '_exptl_crystal.density_method',
        [ [ 'item_',
            '_item_description.description',
            '              The method used to measure _exptl_crystal.density_meas.' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_method' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_density_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_exptl_crystal.density_percent_sol',
        [ [ 'item_',
            '_item_description.description',
            '              Density value P calculated from the crystal cell and contents,\n               expressed as per cent solvent.\n\n               P = 1 - (1.23 N MMass) / V\n\n               N     = the number of molecules in the unit cell\n               MMass = the molecular mass of each molecule (gm/mole)\n               V     = the volume of the unit cell (A^3^)\n               1.23  = a conversion factor evaluated as:\n\n                       (0.74 cm^3^/g) (10^24^ A^3^/cm^3^)\n                       --------------------------------------\n                            (6.02*10^23^) molecules/mole\n\n                       where 0.74 is an assumed value for the partial specific\n                       volume of the molecule' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_percent_sol' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_exptl_crystal.description',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the quality and habit of the crystal.\n               The crystal dimensions should not normally be reported here;\n               use instead the specific items in the EXPTL_CRYSTAL category\n               relating to size for the gross dimensions of the crystal and\n               data items in the EXPTL_CRYSTAL_FACE category to describe the\n               relationship between individual faces.' ],
          [ 'item_', '_item.name', '_exptl_crystal.description' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_description' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_exptl_crystal.F_000',
        [ [ 'item_',
            '_item_description.description',
            '              The effective number of electrons in the crystal unit cell\n               contributing to F(000). This may contain dispersion contributions\n               and is calculated as\n\n               F(000) = [ sum (f~r~^2^ + f~i~^2^) ]^1/2^\n\n               f~r~ = real part of the scattering factors at theta = 0 degree\n               f~i~ = imaginary part of the scattering factors at\n               theta = 0 degree\n\n                      the sum is taken over each atom in the unit cell' ],
          [ 'item_', '_item.name', '_exptl_crystal.F_000' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_exptl_crystal_F_000' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_exptl_crystal.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _exptl_crystal.id must uniquely identify a record in\n               the EXPTL_CRYSTAL list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_exptl_crystal.id',
              'exptl_crystal',
              'yes',
              '_exptl_crystal_face.crystal_id',
              'exptl_crystal_face',
              'yes',
              '_exptl_crystal_grow.crystal_id',
              'exptl_crystal_grow',
              'yes',
              '_exptl_crystal_grow_comp.crystal_id',
              'exptl_crystal_grow_comp',
              'yes',
              '_diffrn.crystal_id',
              'diffrn',
              'yes',
              '_refln.crystal_id',
              'refln',
              'yes' ] ],
          [ 'item_', '_item_aliases.alias_name', '_exptl_crystal_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_diffrn.crystal_id',
              '_exptl_crystal.id',
              '_exptl_crystal_grow.crystal_id',
              '_exptl_crystal.id',
              '_exptl_crystal_face.crystal_id',
              '_exptl_crystal.id',
              '_exptl_crystal_grow_comp.crystal_id',
              '_exptl_crystal.id',
              '_refln.crystal_id',
              '_exptl_crystal.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_exptl_crystal.preparation',
        [ [ 'item_',
            '_item_description.description',
            '              Details of crystal growth and preparation of the crystal (e.g.\n               mounting) prior to the intensity measurements.' ],
          [ 'item_', '_item.name', '_exptl_crystal.preparation' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_preparation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'mounted in an argon-filled quartz capillary' ] ] ],
      [ 'save_',
        '_exptl_crystal.size_max',
        [ [ 'item_',
            '_item_description.description',
            '              The maximum dimension of the crystal. This item may appear in a\n               list with _exptl_crystal.id if multiple crystals are used in the\n               experiment.' ],
          [ 'item_', '_item.name', '_exptl_crystal.size_max' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_size_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'millimetres' ] ] ],
      [ 'save_',
        '_exptl_crystal.size_mid',
        [ [ 'item_',
            '_item_description.description',
            '              The medial dimension of the crystal. This item may appear in a\n               list with _exptl_crystal.id if multiple crystals are used in the\n               experiment.' ],
          [ 'item_', '_item.name', '_exptl_crystal.size_mid' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_size_mid' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'millimetres' ] ] ],
      [ 'save_',
        '_exptl_crystal.size_min',
        [ [ 'item_',
            '_item_description.description',
            '              The minimum dimension of the crystal. This item may appear in a\n               list with _exptl_crystal.id if multiple crystals are used in the\n               experiment.' ],
          [ 'item_', '_item.name', '_exptl_crystal.size_min' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_size_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'millimetres' ] ] ],
      [ 'save_',
        '_exptl_crystal.size_rad',
        [ [ 'item_',
            '_item_description.description',
            '              The radius of the crystal, if the crystal is a sphere or a\n               cylinder. This item may appear in a list with _exptl_crystal.id\n               if multiple crystals are used in the experiment.' ],
          [ 'item_', '_item.name', '_exptl_crystal.size_rad' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_size_rad' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'millimetres' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# EXPTL_CRYSTAL_FACE ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'exptl_crystal_face',
        [ [ 'item_',
            '_category.description',
            '              Data items in the EXPTL_CRYSTAL_FACE category record details\n               of the crystal faces.' ],
          [ 'item_', '_category.id', 'exptl_crystal_face' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_exptl_crystal_face.crystal_id',
              '_exptl_crystal_face.index_h',
              '_exptl_crystal_face.index_k',
              '_exptl_crystal_face.index_l' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'exptl_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for Yb(S-C5H4N)2(THF)4\n                for the 100 face of crystal xstl1.',
              '\n    _exptl_crystal_face.crystal_id     xstl1\n    _exptl_crystal_face.index_h        1\n    _exptl_crystal_face.index_k        0\n    _exptl_crystal_face.index_l        0\n    _exptl_crystal_face.diffr_chi      42.56\n    _exptl_crystal_face.diffr_kappa    30.23\n    _exptl_crystal_face.diffr_phi      -125.56\n    _exptl_crystal_face.diffr_psi      -0.34\n    _exptl_crystal_face.perp_dist      0.025',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_exptl_crystal_face.crystal_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _exptl_crystal.id in the\n               EXPTL_CRYSTAL category.' ],
          [ 'item_', '_item.name', '_exptl_crystal_face.crystal_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_exptl_crystal_face.diffr_chi',
        [ [ 'item_',
            '_item_description.description',
            '              The chi diffractometer setting angle in degrees for a specific\n               crystal face associated with _exptl_crystal_face.perp_dist.' ],
          [ 'item_', '_item.name', '_exptl_crystal_face.diffr_chi' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_face' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_face_diffr_chi' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_exptl_crystal_face.diffr_kappa',
        [ [ 'item_',
            '_item_description.description',
            '              The kappa diffractometer setting angle in degrees for a specific\n               crystal face associated with _exptl_crystal_face.perp_dist.' ],
          [ 'item_', '_item.name', '_exptl_crystal_face.diffr_kappa' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_face' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_face_diffr_kappa' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_exptl_crystal_face.diffr_phi',
        [ [ 'item_',
            '_item_description.description',
            '              The phi diffractometer setting angle in degrees for a specific\n               crystal face associated with _exptl_crystal_face.perp_dist.' ],
          [ 'item_', '_item.name', '_exptl_crystal_face.diffr_phi' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_face' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_face_diffr_phi' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_exptl_crystal_face.diffr_psi',
        [ [ 'item_',
            '_item_description.description',
            '              The psi diffractometer setting angle in degrees for a specific\n               crystal face associated with _exptl_crystal_face.perp_dist.' ],
          [ 'item_', '_item.name', '_exptl_crystal_face.diffr_psi' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_face' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_face_diffr_psi' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_exptl_crystal_face.index_h',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index h of the crystal face associated with the value\n               _exptl_crystal_face.perp_dist.' ],
          [ 'item_', '_item.name', '_exptl_crystal_face.index_h' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_face' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_face_index_h' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_exptl_crystal_face.index_k', '_exptl_crystal_face.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_exptl_crystal_face.index_k',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index k of the crystal face associated with the value\n               _exptl_crystal_face.perp_dist.' ],
          [ 'item_', '_item.name', '_exptl_crystal_face.index_k' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_face' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_face_index_k' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_exptl_crystal_face.index_h', '_exptl_crystal_face.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_exptl_crystal_face.index_l',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index l of the crystal face associated with the value\n               _exptl_crystal_face.perp_dist.' ],
          [ 'item_', '_item.name', '_exptl_crystal_face.index_l' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_face' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_face_index_l' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_exptl_crystal_face.index_h', '_exptl_crystal_face.index_k' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_exptl_crystal_face.perp_dist',
        [ [ 'item_',
            '_item_description.description',
            '              The perpendicular distance in millimetres from the face to the\n               centre of rotation of the crystal.' ],
          [ 'item_', '_item.name', '_exptl_crystal_face.perp_dist' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_face' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_face_perp_dist' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'millimetres' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# EXPTL_CRYSTAL_GROW ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'exptl_crystal_grow',
        [ [ 'item_',
            '_category.description',
            '              Data items in the EXPTL_CRYSTAL_GROW category record details\n               about the conditions and methods used to grow the crystal.' ],
          [ 'item_', '_category.id', 'exptl_crystal_grow' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_exptl_crystal_grow.crystal_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'exptl_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _exptl_crystal_grow.crystal_id      1\n    _exptl_crystal_grow.method          \'hanging drop\'\n    _exptl_crystal_grow.apparatus       \'Linbro plates\'\n    _exptl_crystal_grow.atmosphere      \'room air\'\n    _exptl_crystal_grow.pH              4.7\n    _exptl_crystal_grow.temp            18(3)\n    _exptl_crystal_grow.time            \'approximately 2 days\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.apparatus',
        [ [ 'item_',
            '_item_description.description',
            '              The physical apparatus in which the crystal was grown.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.apparatus' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Linbro plate', 'sandwich box', 'ACA plates' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.atmosphere',
        [ [ 'item_',
            '_item_description.description',
            '              The nature of the gas or gas mixture in which the crystal was\n               grown.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.atmosphere' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'room air', 'nitrogen', 'argon' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.crystal_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _exptl_crystal.id in the\n               EXPTL_CRYSTAL category.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.crystal_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the crystal growth.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.details' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '                                 Solution 2 was prepared as a well solution and\n                                  mixed. A droplet containing 2 \\ml of solution\n                                  1 was delivered onto a cover slip; 2 \\ml of\n                                  solution 2 was added to the droplet without\n                                  mixing.',
              '                                 Crystal plates were originally stored at room\n                                  temperature for 1 week but no nucleation\n                                  occurred. They were then transferred to 4\n                                  degrees C, at which temperature well formed\n                                  single crystals grew in 2 days.',
              '                                 The dependence on pH for successful crystal\n                                  growth is very sharp. At pH 7.4 only showers\n                                  of tiny crystals grew, at pH 7.5 well formed\n                                  single crystals grew, at pH 7.6 no\n                                  crystallization occurred at all.' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.method',
        [ [ 'item_',
            '_item_description.description',
            '              The method used to grow the crystals.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.method' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'batch precipitation',
              'batch dialysis',
              'hanging drop vapor diffusion',
              'sitting drop vapor diffusion' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.method_ref',
        [ [ 'item_',
            '_item_description.description',
            '              A literature reference that describes the method used to grow\n               the crystals.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.method_ref' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'McPherson et al., 1988' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.pH',
        [ [ 'item_',
            '_item_description.description',
            '              The pH at which the crystal was grown. If more than one pH was\n               employed during the crystallization process, the final pH should\n               be noted here and the protocol involving multiple pH values\n               should be described in _exptl_crystal_grow.details.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.pH' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '7.4', '7.6', '4.3' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.pressure',
        [ [ 'item_',
            '_item_description.description',
            '              The ambient pressure in kilopascals at which the crystal was\n               grown.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.pressure' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal_grow.pressure_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'kilopascals' ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.pressure_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _exptl_crystal_grow.pressure.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.pressure_esd' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal_grow.pressure' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kilopascals' ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.seeding',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the protocol used for seeding the crystal\n               growth.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.seeding' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'macroseeding',
              '                                 Microcrystals were introduced from a previous\n                                  crystal growth experiment by transfer with a\n                                  human hair.' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.seeding_ref',
        [ [ 'item_',
            '_item_description.description',
            '              A literature reference that describes the protocol used to seed\n               the crystal.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.seeding_ref' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'Stura et al., 1989' ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.temp',
        [ [ 'item_',
            '_item_description.description',
            '              The temperature in kelvins at which the crystal was grown.\n               If more than one temperature was employed during the\n               crystallization process, the final temperature should be noted\n               here and the protocol  involving multiple temperatures should be\n               described in _exptl_crystal_grow.details.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.temp' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal_grow.temp_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.temp_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of temperature control during\n               crystal growth.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.temp_details' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.temp_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _exptl_crystal_grow.temp.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.temp_esd' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal_grow.temp' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.time',
        [ [ 'item_',
            '_item_description.description',
            '              The approximate time that the crystal took to grow to the size\n               used for data collection.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.time' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'overnight', '2-4 days', '6 months' ] ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', '# EXPTL_CRYSTAL_GROW_COMP ##' ],
      [ '@comment', '############################' ],
      [ 'save_',
        'exptl_crystal_grow_comp',
        [ [ 'item_',
            '_category.description',
            '              Data items in the EXPTL_CRYSTAL_GROW_COMP category record\n               details about the components of the solutions that were \'mixed\'\n               (by whatever means) to produce the crystal.\n\n               In general, solution 1 is the solution that contains the\n               molecule to be crystallized and solution 2 is the solution\n               that contains the precipitant. However, the number of solutions\n               required to describe the crystallization protocol is not limited\n               to 2.\n\n               Details of the crystallization protocol should be given in\n               _exptl_crystal_grow_comp.details using the solutions\n               described in EXPTL_CRYSTAL_GROW_COMP.' ],
          [ 'item_', '_category.id', 'exptl_crystal_grow_comp' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_exptl_crystal_grow_comp.id',
              '_exptl_crystal_grow_comp.crystal_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'exptl_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _exptl_crystal_grow_comp.crystal_id\n    _exptl_crystal_grow_comp.id\n    _exptl_crystal_grow_comp.sol_id\n    _exptl_crystal_grow_comp.name\n    _exptl_crystal_grow_comp.volume\n    _exptl_crystal_grow_comp.conc\n    _exptl_crystal_grow_comp.details\n     1 1  1  \'HIV-1 protease\'  \'0.002 ml\'  \'6 mg/ml\'\n    ; The protein solution was in a buffer containing 25 mM NaCl,\n      100 mM NaMES/ MES buffer, pH 7.5, 3 mM NaAzide\n    ;\n     1 2 2 \'NaCl\'         \'0.200 ml\'  \'4    M\' \'in 3 mM NaAzide\'\n     1 3 2 \'Acetic Acid\'  \'0.047 ml\'  \'100 mM\' \'in 3 mM NaAzide\'\n     1 4 2 \'Na Acetate\'   \'0.053 ml\'  \'100 mM\'\n    ; in 3 mM NaAzide. Buffer components were mixed to produce a\n      pH of 4.7 according to a ratio calculated from the pKa. The\n      actual pH of solution 2 was not measured.\n    ;\n     1 5 2 \'water\'        \'0.700 ml\'   \'neat\'  \'in 3 mM NaAzide\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow_comp.conc',
        [ [ 'item_',
            '_item_description.description',
            '              The concentration of the solution component.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow_comp.conc' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '200 \\ml', '0.1 ml' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow_comp.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of any special aspects of the solution component.\n               When the solution component is the one that contains the\n               macromolecule, this could be the specification of the buffer in\n               which the macromolecule was stored. When the solution component\n               is a buffer component, this could be the methods (or formula)\n               used to achieve a desired pH.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow_comp.details' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'in 3 mM NaAzide',
              '                                 The protein solution was in a buffer\n                                  containing 25 mM NaCl, 100 mM NaMES/MES\n                                  buffer, pH 7.5, 3 mM NaAzide',
              '                                 in 3 mM NaAzide. Buffer components were mixed\n                                  to produce a pH of 4.7 according to a ratio\n                                  calculated from the pKa. The actual pH of\n                                  solution 2 was not measured.' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow_comp.crystal_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _exptl_crystal.id in the\n               EXPTL_CRYSTAL category.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow_comp.crystal_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_exptl_crystal_grow_comp.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _exptl_crystal_grow_comp.id must uniquely identify\n               each item in the EXPTL_CRYSTAL_GROW_COMP list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow_comp.id' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '1', 'A', 'protein in buffer' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow_comp.name',
        [ [ 'item_',
            '_item_description.description',
            '              A common name for the component of the solution.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow_comp.name' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'protein in buffer', 'acetic acid' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow_comp.sol_id',
        [ [ 'item_',
            '_item_description.description',
            '              An identifier for the solution to which the given solution\n               component belongs.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow_comp.sol_id' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '1', 'well solution', 'solution A' ] ] ] ],
      [ 'save_',
        '_exptl_crystal_grow_comp.volume',
        [ [ 'item_',
            '_item_description.description',
            '              The volume of the solution component.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow_comp.volume' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '200 \\ml', '0.1 ml' ] ] ] ],
      [ '@comment', '#########' ],
      [ '@comment', '# GEOM ##' ],
      [ '@comment', '#########' ],
      [ 'save_',
        'geom',
        [ [ 'item_',
            '_category.description',
            '              Data items in the GEOM and related (GEOM_ANGLE,\n               GEOM_BOND, GEOM_CONTACT, GEOM_HBOND and GEOM_TORSION)\n               categories record details about the molecular\n               geometry as calculated from the contents of the ATOM, CELL\n               and SYMMETRY data.\n\n               Geometry data are therefore redundant, in that they can be\n               calculated from other more fundamental quantities in the data\n               block. However, they provide a check on the correctness of\n               both sets of data and enable the most important geometric data\n               to be identified for publication by setting the appropriate\n               publication flag.' ],
          [ 'item_', '_category.id', 'geom' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_geom.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'geom_group' ] ] ] ],
      [ 'save_',
        '_geom.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_geom.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_geom.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of geometry not covered by the\n               existing data names in the GEOM categories, such as\n               least-squares planes.' ],
          [ 'item_', '_item.name', '_geom.details' ],
          [ 'item_', '_item.category_id', 'geom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_geom_special_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '###############' ],
      [ '@comment', '# GEOM_ANGLE ##' ],
      [ '@comment', '###############' ],
      [ 'save_',
        'geom_angle',
        [ [ 'item_',
            '_category.description',
            '              Data items in the GEOM_ANGLE category record details about the\n               bond angles as calculated from the contents\n               of the ATOM, CELL and SYMMETRY data.' ],
          [ 'item_', '_category.id', 'geom_angle' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_geom_angle.atom_site_id_1',
              '_geom_angle.atom_site_id_2',
              '_geom_angle.atom_site_id_3',
              '_geom_angle.site_symmetry_1',
              '_geom_angle.site_symmetry_2',
              '_geom_angle.site_symmetry_3' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'geom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    loop_\n    _geom_angle.atom_site_id_1\n    _geom_angle.atom_site_id_2\n    _geom_angle.atom_site_id_3\n    _geom_angle.value\n    _geom_angle.value_esd\n    _geom_angle.site_symmetry_1\n    _geom_angle.site_symmetry_2\n    _geom_angle.site_symmetry_3\n    _geom_angle.publ_flag\n     C2  O1  C5   111.6  0.2  1_555  1_555  1_555  yes\n     O1  C2  C3   110.9  0.2  1_555  1_555  1_555  yes\n     O1  C2  O21  122.2  0.3  1_555  1_555  1_555  yes\n     C3  C2  O21  127.0  0.3  1_555  1_555  1_555  yes\n     C2  C3  N4   101.3  0.2  1_555  1_555  1_555  yes\n     C2  C3  C31  111.3  0.2  1_555  1_555  1_555  yes\n     C2  C3  H3   107    1    1_555  1_555  1_555  no\n     N4  C3  C31  116.7  0.2  1_555  1_555  1_555  yes\n    # - - - - data truncated for brevity - - - -',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the first of the three atom sites that define\n               the angle.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_angle_atom_site_label_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_geom_angle.atom_site_id_2', '_geom_angle.atom_site_id_3' ] ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_alt_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_label_alt_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.atom_site_label_atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_comp_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.atom_site_label_comp_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_label_seq_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.atom_site_label_asym_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the second of the three atom sites that define\n               the angle. The second atom is taken to be the apex of the angle.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_angle_atom_site_label_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_geom_angle.atom_site_id_1', '_geom_angle.atom_site_id_3' ] ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_alt_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the three atom sites\n               that define the angle.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_label_alt_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the three atom sites\n               that define the angle.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.atom_site_label_atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the three atom sites\n               that define the angle.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.atom_site_label_comp_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the three atom sites\n               that define the angle.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_label_seq_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the three atom sites\n               that define the angle.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.atom_site_label_asym_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the third of the three atom sites that define\n               the angle.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_id_3' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_angle_atom_site_label_3' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_geom_angle.atom_site_id_1', '_geom_angle.atom_site_id_2' ] ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_alt_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_label_alt_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_atom_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.atom_site_label_atom_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_comp_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.atom_site_label_comp_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_seq_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_label_seq_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_label_asym_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.atom_site_label_asym_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_asym_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_comp_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_comp_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_seq_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the three atom sites\n               that define the angle.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the three atom sites\n               that define the angle.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_asym_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the three atom sites\n               that define the angle.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_comp_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the three atom sites\n               that define the angle.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_seq_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_atom_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_atom_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_asym_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_asym_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_comp_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_comp_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.atom_site_auth_seq_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the three atom sites that\n               define the angle.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_angle.atom_site_auth_seq_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_angle.publ_flag',
        [ [ 'item_',
            '_item_description.description',
            '              This code signals whether the angle is referred to in a\n               publication or should be placed in a table of significant angles.' ],
          [ 'item_', '_item.name', '_geom_angle.publ_flag' ],
          [ 'item_', '_item.category_id', 'geom_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_geom_angle_publ_flag' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'do not include angle in special list',
              'n',
              'abbreviation for "no"',
              'yes',
              'do include angle in special list',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_geom_angle.site_symmetry_1',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the first of the three atom sites that\n               define the angle.' ],
          [ 'item_', '_item.name', '_geom_angle.site_symmetry_1' ],
          [ 'item_', '_item.category_id', 'geom_angle' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_angle_site_symmetry_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_angle.site_symmetry_2',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the second of the three atom sites that\n               define the angle.' ],
          [ 'item_', '_item.name', '_geom_angle.site_symmetry_2' ],
          [ 'item_', '_item.category_id', 'geom_angle' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_angle_site_symmetry_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_angle.site_symmetry_3',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the third of the three atom sites that\n               define the angle.' ],
          [ 'item_', '_item.name', '_geom_angle.site_symmetry_3' ],
          [ 'item_', '_item.category_id', 'geom_angle' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_angle_site_symmetry_3' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_angle.value',
        [ [ 'item_',
            '_item_description.description',
            '              Angle in degrees defined by the three sites\n               _geom_angle.atom_site_id_1, _geom_angle.atom_site_id_2 and\n               _geom_angle.atom_site_id_3.' ],
          [ 'item_', '_item.name', '_geom_angle.value' ],
          [ 'item_', '_item.category_id', 'geom_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_geom_angle' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_related.related_name',
            '_geom_angle.value_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_geom_angle.value_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _geom_angle.value.' ],
          [ 'item_', '_item.name', '_geom_angle.value_esd' ],
          [ 'item_', '_item.category_id', 'geom_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_related.related_name', '_geom_angle.value' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ '@comment', '##############' ],
      [ '@comment', '# GEOM_BOND ##' ],
      [ '@comment', '##############' ],
      [ 'save_',
        'geom_bond',
        [ [ 'item_',
            '_category.description',
            '              Data items in the GEOM_BOND category record details about\n               the bond lengths as calculated from the contents\n               of the ATOM, CELL and SYMMETRY data.' ],
          [ 'item_', '_category.id', 'geom_bond' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_geom_bond.atom_site_id_1',
              '_geom_bond.atom_site_id_2',
              '_geom_bond.site_symmetry_1',
              '_geom_bond.site_symmetry_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'geom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    loop_\n    _geom_bond.atom_site_id_1\n    _geom_bond.atom_site_id_2\n    _geom_bond.dist\n    _geom_bond.dist_esd\n    _geom_bond.site_symmetry_1\n    _geom_bond.site_symmetry_2\n    _geom_bond.publ_flag\n      O1  C2   1.342  0.004  1_555  1_555  yes\n      O1  C5   1.439  0.003  1_555  1_555  yes\n      C2  C3   1.512  0.004  1_555  1_555  yes\n      C2  O21  1.199  0.004  1_555  1_555  yes\n      C3  N4   1.465  0.003  1_555  1_555  yes\n      C3  C31  1.537  0.004  1_555  1_555  yes\n      C3  H3   1.00   0.03   1_555  1_555  no\n      N4  C5   1.472  0.003  1_555  1_555  yes\n    # - - - - data truncated for brevity - - - -',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the first of the two atom sites that define the\n               bond.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_bond_atom_site_label_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_geom_bond.atom_site_id_2' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_label_alt_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_label_alt_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_label_atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_label_atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_label_comp_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_label_comp_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_label_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_label_seq_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_label_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_label_asym_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the second of the two atom sites that define\n               the bond.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_bond_atom_site_label_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_geom_bond.atom_site_id_1' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_label_alt_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_label_alt_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_label_atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_label_atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_label_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_label_comp_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_label_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_label_seq_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_label_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_label_asym_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_auth_atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_auth_atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_auth_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_auth_asym_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_auth_comp_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_auth_comp_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_auth_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_auth_seq_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_auth_atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_auth_atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_auth_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_auth_asym_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_auth_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_auth_comp_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.atom_site_auth_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the bond.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_bond.atom_site_auth_seq_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_bond.dist',
        [ [ 'item_',
            '_item_description.description',
            '              The intramolecular bond distance in angstroms.' ],
          [ 'item_', '_item.name', '_geom_bond.dist' ],
          [ 'item_', '_item.category_id', 'geom_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_geom_bond_distance' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_related.related_name', '_geom_bond.dist_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_bond.dist_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _geom_bond.dist.' ],
          [ 'item_', '_item.name', '_geom_bond.dist_esd' ],
          [ 'item_', '_item.category_id', 'geom_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_related.related_name', '_geom_bond.dist' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_bond.publ_flag',
        [ [ 'item_',
            '_item_description.description',
            '              This code signals whether the bond distance is referred to in a\n               publication or should be placed in a list of significant bond\n               distances.' ],
          [ 'item_', '_item.name', '_geom_bond.publ_flag' ],
          [ 'item_', '_item.category_id', 'geom_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_geom_bond_publ_flag' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'do not include bond in special list',
              'n',
              'abbreviation for "no"',
              'yes',
              'do include bond in special list',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_geom_bond.site_symmetry_1',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the first of the two atom sites that\n               define the bond.' ],
          [ 'item_', '_item.name', '_geom_bond.site_symmetry_1' ],
          [ 'item_', '_item.category_id', 'geom_bond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_bond_site_symmetry_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_bond.site_symmetry_2',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the second of the two atom sites that\n               define the bond.' ],
          [ 'item_', '_item.name', '_geom_bond.site_symmetry_2' ],
          [ 'item_', '_item.category_id', 'geom_bond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_bond_site_symmetry_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ '@comment', '#################' ],
      [ '@comment', '# GEOM_CONTACT ##' ],
      [ '@comment', '#################' ],
      [ 'save_',
        'geom_contact',
        [ [ 'item_',
            '_category.description',
            '              Data items in the GEOM_CONTACT category record details about\n               interatomic contacts as calculated from the contents\n               of the ATOM, CELL and SYMMETRY data.' ],
          [ 'item_', '_category.id', 'geom_contact' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_geom_contact.atom_site_id_1',
              '_geom_contact.atom_site_id_2',
              '_geom_contact.site_symmetry_1',
              '_geom_contact.site_symmetry_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'geom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set CLPHO6 of Ferguson, Ruhl, McKervey & Browne\n                 [Acta Cryst. (1992), C48, 2262-2264].',
              '\n    loop_\n    _geom_contact.atom_site_id_1\n    _geom_contact.atom_site_id_2\n    _geom_contact.dist\n    _geom_contact.dist_esd\n    _geom_contact.site_symmetry_1\n    _geom_contact.site_symmetry_2\n    _geom_contact.publ_flag\n     O(1)  O(2)     2.735  0.003  .  .  yes\n     H(O1) O(2)     1.82      .   .  .  no',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the first of the two atom sites that define the\n               contact.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_contact.atom_site_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_contact_atom_site_label_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_geom_contact.atom_site_id_2' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_label_alt_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_label_alt_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_label_atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_label_atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_label_comp_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_label_comp_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_label_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_label_seq_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_label_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_label_asym_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the second of the two atom sites that define\n               the contact.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_contact.atom_site_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_contact_atom_site_label_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_dependent.dependent_name',
            '_geom_contact.atom_site_id_1' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_label_alt_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_label_alt_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_label_atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_label_atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_label_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_label_comp_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_label_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_label_seq_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_label_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_label_asym_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_auth_atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_auth_atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_auth_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_auth_asym_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_auth_comp_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_auth_comp_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_auth_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_auth_seq_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_auth_atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_auth_atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_auth_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_auth_asym_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_auth_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_auth_comp_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.atom_site_auth_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the two atom sites that\n               define the contact.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.atom_site_auth_seq_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_contact.dist',
        [ [ 'item_',
            '_item_description.description',
            '              The interatomic contact distance in angstroms.' ],
          [ 'item_', '_item.name', '_geom_contact.dist' ],
          [ 'item_', '_item.category_id', 'geom_contact' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_geom_contact_distance' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_geom_contact.dist_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_contact.dist_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _geom_contact.dist.' ],
          [ 'item_', '_item.name', '_geom_contact.dist_esd' ],
          [ 'item_', '_item.category_id', 'geom_contact' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_related.related_name', '_geom_contact.dist' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_contact.publ_flag',
        [ [ 'item_',
            '_item_description.description',
            '              This code signals whether the contact distance is referred to\n               in a publication or should be placed in a list of significant\n               contact distances.' ],
          [ 'item_', '_item.name', '_geom_contact.publ_flag' ],
          [ 'item_', '_item.category_id', 'geom_contact' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_contact_publ_flag' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'do not include distance in special list',
              'n',
              'abbreviation for "no"',
              'yes',
              'do include distance in special list',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_geom_contact.site_symmetry_1',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the first of the two atom sites that\n               define the contact.' ],
          [ 'item_', '_item.name', '_geom_contact.site_symmetry_1' ],
          [ 'item_', '_item.category_id', 'geom_contact' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_contact_site_symmetry_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_contact.site_symmetry_2',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the second of the two atom sites that\n               define the contact.' ],
          [ 'item_', '_item.name', '_geom_contact.site_symmetry_2' ],
          [ 'item_', '_item.category_id', 'geom_contact' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_contact_site_symmetry_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ '@comment', '###############' ],
      [ '@comment', '# GEOM_HBOND ##' ],
      [ '@comment', '###############' ],
      [ 'save_',
        'geom_hbond',
        [ [ 'item_',
            '_category.description',
            '              Data items in the GEOM_HBOND category record details about\n               hydrogen bonds as calculated from the contents of the ATOM,\n               CELL and SYMMETRY data.' ],
          [ 'item_', '_category.id', 'geom_hbond' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_geom_hbond.atom_site_id_A',
              '_geom_hbond.atom_site_id_D',
              '_geom_hbond.atom_site_id_H',
              '_geom_hbond.site_symmetry_A',
              '_geom_hbond.site_symmetry_D',
              '_geom_hbond.site_symmetry_H' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'geom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on C~14~H~13~ClN~2~O.H~2~O, reported by Palmer,\n                Puddle & Lisgarten [Acta Cryst. (1993), C49, 1777-1779].',
              '\n    loop_\n    _geom_hbond.atom_site_id_D\n    _geom_hbond.atom_site_id_H\n    _geom_hbond.atom_site_id_A\n    _geom_hbond.dist_DH\n    _geom_hbond.dist_HA\n    _geom_hbond.dist_DA\n    _geom_hbond.angle_DHA\n    _geom_hbond.publ_flag\n\n    N6   HN6  OW   0.888  1.921  2.801  169.6  yes\n    OW   HO2  O7   0.917  1.923  2.793  153.5  yes\n    OW   HO1  N10  0.894  1.886  2.842  179.7  yes',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_geom_hbond.angle_DHA',
        [ [ 'item_',
            '_item_description.description',
            '              The angle in degrees defined by the donor-, hydrogen- and\n               acceptor-atom sites in a hydrogen bond.' ],
          [ 'item_', '_item.name', '_geom_hbond.angle_DHA' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_geom_hbond_angle_DHA' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_geom_hbond.angle_DHA_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_hbond.angle_DHA_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _geom_hbond.angle_DHA.' ],
          [ 'item_', '_item.name', '_geom_hbond.angle_DHA_esd' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_geom_hbond.angle_DHA' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_id_A',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the acceptor-atom site that defines the\n               hydrogen bond.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_id_A' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_hbond_atom_site_label_A' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_geom_hbond.atom_site_id_D', '_geom_hbond.atom_site_id_H' ] ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_alt_id_A',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the acceptor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_label_alt_id_A' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_asym_id_A',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the acceptor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_hbond.atom_site_label_asym_id_A' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_atom_id_A',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the acceptor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_hbond.atom_site_label_atom_id_A' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_comp_id_A',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the acceptor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_hbond.atom_site_label_comp_id_A' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_seq_id_A',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the acceptor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_label_seq_id_A' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_id_D',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the donor-atom site that defines the hydrogen\n               bond.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_id_D' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_hbond_atom_site_label_D' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_geom_hbond.atom_site_id_A', '_geom_hbond.atom_site_id_H' ] ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_alt_id_D',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the donor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_label_alt_id_D' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_asym_id_D',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the donor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_hbond.atom_site_label_asym_id_D' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_atom_id_D',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the donor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_hbond.atom_site_label_atom_id_D' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_comp_id_D',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the donor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_hbond.atom_site_label_comp_id_D' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_seq_id_D',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the donor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_label_seq_id_D' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_id_H',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the hydrogen-atom site that defines the\n               hydrogen bond.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_id_H' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_hbond_atom_site_label_H' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_geom_hbond.atom_site_id_A', '_geom_hbond.atom_site_id_D' ] ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_alt_id_H',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the hydrogen-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_label_alt_id_H' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_asym_id_H',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the hydrogen-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_hbond.atom_site_label_asym_id_H' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_atom_id_H',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the hydrogen-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_hbond.atom_site_label_atom_id_H' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_comp_id_H',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the hydrogen-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_hbond.atom_site_label_comp_id_H' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_label_seq_id_H',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the hydrogen-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_label_seq_id_H' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_asym_id_A',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the acceptor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_asym_id_A' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_atom_id_A',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the acceptor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_atom_id_A' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_comp_id_A',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the acceptor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_comp_id_A' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_seq_id_A',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the acceptor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_seq_id_A' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_asym_id_D',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the donor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_asym_id_D' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_atom_id_D',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the donor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_atom_id_D' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_comp_id_D',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the donor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_comp_id_D' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_seq_id_D',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the donor-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_seq_id_D' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_asym_id_H',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the hydrogen-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_asym_id_H' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_atom_id_H',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the hydrogen-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_atom_id_H' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_comp_id_H',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the hydrogen-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_comp_id_H' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.atom_site_auth_seq_id_H',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the hydrogen-atom site that defines\n               the hydrogen bond.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_geom_hbond.atom_site_auth_seq_id_H' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_hbond.dist_DA',
        [ [ 'item_',
            '_item_description.description',
            '              The distance in angstroms between the donor- and acceptor-atom\n               sites in a hydrogen bond.' ],
          [ 'item_', '_item.name', '_geom_hbond.dist_DA' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_hbond_distance_DA' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_geom_hbond.dist_DA_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_hbond.dist_DA_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               in angstroms of _geom_hbond.dist_DA.' ],
          [ 'item_', '_item.name', '_geom_hbond.dist_DA_esd' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_related.related_name', '_geom_hbond.dist_DH' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_hbond.dist_DH',
        [ [ 'item_',
            '_item_description.description',
            '              The distance in angstroms between the donor- and hydrogen-atom\n               sites in a hydrogen bond.' ],
          [ 'item_', '_item.name', '_geom_hbond.dist_DH' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_hbond_distance_DH' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_geom_hbond.dist_DH_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_hbond.dist_DH_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               in angstroms of _geom_hbond.dist_DH.' ],
          [ 'item_', '_item.name', '_geom_hbond.dist_DH_esd' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_related.related_name', '_geom_hbond.dist_DH' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_hbond.dist_HA',
        [ [ 'item_',
            '_item_description.description',
            '              The distance in angstroms between the hydrogen- and acceptor-\n               atom sites in a hydrogen bond.' ],
          [ 'item_', '_item.name', '_geom_hbond.dist_HA' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_hbond_distance_HA' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_geom_hbond.dist_HA_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_hbond.dist_HA_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               in angstroms of _geom_hbond.dist_HA.' ],
          [ 'item_', '_item.name', '_geom_hbond.dist_HA_esd' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_related.related_name', '_geom_hbond.dist_HA' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_geom_hbond.publ_flag',
        [ [ 'item_',
            '_item_description.description',
            '              This code signals whether the hydrogen-bond information is\n               referred to in a publication or should be placed in a table of\n               significant hydrogen-bond geometry.' ],
          [ 'item_', '_item.name', '_geom_hbond.publ_flag' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_geom_hbond_publ_flag' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'do not include bond in special list',
              'n',
              'abbreviation for "no"',
              'yes',
              'do include bond in special list',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_geom_hbond.site_symmetry_A',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the acceptor-atom site that defines the\n               hydrogen bond.' ],
          [ 'item_', '_item.name', '_geom_hbond.site_symmetry_A' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_hbond_site_symmetry_A' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_hbond.site_symmetry_D',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the donor-atom site that defines the\n               hydrogen bond.' ],
          [ 'item_', '_item.name', '_geom_hbond.site_symmetry_D' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_hbond_site_symmetry_D' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_hbond.site_symmetry_H',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the hydrogen-atom site that defines the\n               hydrogen bond.' ],
          [ 'item_', '_item.name', '_geom_hbond.site_symmetry_H' ],
          [ 'item_', '_item.category_id', 'geom_hbond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_hbond_site_symmetry_H' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ '@comment', '#################' ],
      [ '@comment', '# GEOM_TORSION ##' ],
      [ '@comment', '#################' ],
      [ 'save_',
        'geom_torsion',
        [ [ 'item_',
            '_category.description',
            '              Data items in the GEOM_TORSION category record details about\n               torsion angles as calculated from the\n               contents of the ATOM, CELL and SYMMETRY data.\n\n               The vector direction _geom_torsion.atom_site_id_2 to\n               _geom_torsion.atom_site_id_3 is the viewing direction, and the\n               torsion angle is the angle of twist required to superimpose the\n               projection of the vector between site 2 and site 1 onto the\n               projection of the vector between site 3 and site 4. Clockwise\n               torsions are positive, anticlockwise torsions are negative.\n\n               Ref: Klyne, W. & Prelog, V. (1960). Experientia, 16, 521-523.' ],
          [ 'item_', '_category.id', 'geom_torsion' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_geom_torsion.atom_site_id_1',
              '_geom_torsion.atom_site_id_2',
              '_geom_torsion.atom_site_id_3',
              '_geom_torsion.atom_site_id_4',
              '_geom_torsion.site_symmetry_1',
              '_geom_torsion.site_symmetry_2',
              '_geom_torsion.site_symmetry_3',
              '_geom_torsion.site_symmetry_4' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'geom_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set CLPHO6 of Ferguson, Ruhl, McKervey & Browne\n                [Acta Cryst. (1992), C48, 2262-2264].',
              '\n    loop_\n    _geom_torsion.atom_site_id_1\n    _geom_torsion.atom_site_id_2\n    _geom_torsion.atom_site_id_3\n    _geom_torsion.atom_site_id_4\n    _geom_torsion.value\n    _geom_torsion.site_symmetry_1\n    _geom_torsion.site_symmetry_2\n    _geom_torsion.site_symmetry_3\n    _geom_torsion.site_symmetry_4\n    _geom_torsion.publ_flag\n     C(9)   O(2)    C(7)    C(2)     71.8  .  .  .  .      yes\n     C(7)   O(2)    C(9)    C(10)  -168.0  .  .  .  2_666  yes\n     C(10)  O(3)    C(8)    C(6)   -167.7  .  .  .  .      yes\n     C(8)   O(3)    C(10)   C(9)    -69.7  .  .  .  2_666  yes\n     O(1)   C(1)    C(2)    C(3)   -179.5  .  .  .  .      no\n     O(1)   C(1)    C(2)    C(7)     -0.6  .  .  .  .      no',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the first of the four atom sites that define\n               the torsion angle.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_torsion.atom_site_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion_atom_site_label_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_geom_torsion.atom_site_id_2',
              '_geom_torsion.atom_site_id_3',
              '_geom_torsion.atom_site_id_4' ] ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_alt_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_alt_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_comp_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_comp_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_seq_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_asym_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the second of the four atom sites that define\n               the torsion angle.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_torsion.atom_site_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion_atom_site_label_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_geom_torsion.atom_site_id_1',
              '_geom_torsion.atom_site_id_3',
              '_geom_torsion.atom_site_id_4' ] ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_alt_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_alt_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_comp_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_seq_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_asym_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the third of the four atom sites that define\n               the torsion angle.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_torsion.atom_site_id_3' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion_atom_site_label_3' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_geom_torsion.atom_site_id_1',
              '_geom_torsion.atom_site_id_2',
              '_geom_torsion.atom_site_id_4' ] ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_alt_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_alt_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_atom_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_atom_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_comp_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_comp_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_seq_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_seq_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_asym_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_asym_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier of the fourth of the four atom sites that define\n               the torsion angle.\n\n               This data item is a pointer to _atom_site.id in the ATOM_SITE\n               category.' ],
          [ 'item_', '_item.name', '_geom_torsion.atom_site_id_4' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion_atom_site_label_4' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_geom_torsion.atom_site_id_1',
              '_geom_torsion.atom_site_id_2',
              '_geom_torsion.atom_site_id_3' ] ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_alt_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the fourth of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_alt_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_alt_id_4' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_atom_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the fourth of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_atom_id_4' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_comp_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the fourth of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_comp_id_4' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_seq_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the fourth of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_seq_id_4' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_label_asym_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the fourth of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_label_asym_id_4' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_atom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_asym_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_comp_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_comp_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the first of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_seq_id_1' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_atom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_asym_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_comp_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the second of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_seq_id_2' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_atom_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_atom_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_asym_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_asym_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_comp_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_comp_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_seq_id_3',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the third of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_seq_id_3' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_atom_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the fourth of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_atom_id_4' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_asym_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the fourth of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_asym_id_4' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_comp_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the fourth of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_comp_id_4' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.atom_site_auth_seq_id_4',
        [ [ 'item_',
            '_item_description.description',
            '              An optional identifier of the fourth of the four atom sites that\n               define the torsion angle.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.atom_site_auth_seq_id_4' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_geom_torsion.publ_flag',
        [ [ 'item_',
            '_item_description.description',
            '              This code signals whether the torsion angle is referred to in a\n               publication or should be placed in a table of significant\n               torsion angles.' ],
          [ 'item_', '_item.name', '_geom_torsion.publ_flag' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion_publ_flag' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'do not include angle in special list',
              'n',
              'abbreviation for "no"',
              'yes',
              'do include angle in special list',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_geom_torsion.site_symmetry_1',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the first of the four atom sites that\n               define the torsion angle.' ],
          [ 'item_', '_item.name', '_geom_torsion.site_symmetry_1' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion_site_symmetry_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_torsion.site_symmetry_2',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the second of the four atom sites that\n               define the torsion angle.' ],
          [ 'item_', '_item.name', '_geom_torsion.site_symmetry_2' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion_site_symmetry_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_torsion.site_symmetry_3',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the third of the four atom sites that\n               define the torsion angle.' ],
          [ 'item_', '_item.name', '_geom_torsion.site_symmetry_3' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion_site_symmetry_3' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_torsion.site_symmetry_4',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry code of the fourth of the four atom sites that\n               define the torsion angle.' ],
          [ 'item_', '_item.name', '_geom_torsion.site_symmetry_4' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion_site_symmetry_4' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_geom_torsion.value',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the torsion angle in degrees.' ],
          [ 'item_', '_item.name', '_geom_torsion.value' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_geom_torsion' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_related.related_name',
            '_geom_torsion.value_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_geom_torsion.value_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _geom_torsion.value.' ],
          [ 'item_', '_item.name', '_geom_torsion.value_esd' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_related.related_name', '_geom_torsion.value' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ '@comment', '############' ],
      [ '@comment', '# JOURNAL ##' ],
      [ '@comment', '############' ],
      [ 'save_',
        'journal',
        [ [ 'item_',
            '_category.description',
            '              Data items in the JOURNAL category record details about the\n               book-keeping by the journal staff when processing\n               a data block submitted for publication.\n\n               The creator of a data block will not normally specify these data.\n               The data names are not defined in the dictionary because they are\n               for journal use only.' ],
          [ 'item_', '_category.id', 'journal' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_journal.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'iucr_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on Acta Cryst. file for entry HL0007 [Willis, Beckwith\n                & Tozer (1991). Acta Cryst. C47, 2276-2277].',
              '\n    _journal.entry_id                 \'TOZ\'\n    _journal.date_recd_electronic     1991-04-15\n    _journal.date_from_coeditor       1991-04-18\n    _journal.date_accepted            1991-04-18\n    _journal.date_printers_first      1991-08-07\n    _journal.date_proofs_out          1991-08-07\n    _journal.coeditor_code            HL0007\n    _journal.techeditor_code          C910963\n    _journal.coden_ASTM               ACSCEE\n    _journal.name_full      \'Acta Crystallographica Section C\'\n    _journal.year                     1991\n    _journal.volume                   47\n    _journal.issue                    NOV91\n    _journal.page_first               2276\n    _journal.page_last                2277',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_journal.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_journal.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_journal.coden_ASTM',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.coden_ASTM' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_coden_ASTM' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.coden_Cambridge',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.coden_Cambridge' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_coden_Cambridge' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.coeditor_address',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.coeditor_address' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_coeditor_address' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_journal.coeditor_code',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.coeditor_code' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_coeditor_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.coeditor_email',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.coeditor_email' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_coeditor_email' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.coeditor_fax',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.coeditor_fax' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_coeditor_fax' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.coeditor_name',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.coeditor_name' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_coeditor_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.coeditor_notes',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.coeditor_notes' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_coeditor_notes' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_journal.coeditor_phone',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.coeditor_phone' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_coeditor_phone' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.data_validation_number',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.data_validation_number' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_data_validation_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_journal.date_accepted',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.date_accepted' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_date_accepted' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_journal.date_from_coeditor',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.date_from_coeditor' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_date_from_coeditor' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_journal.date_to_coeditor',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.date_to_coeditor' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_date_to_coeditor' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_journal.date_printers_final',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.date_printers_final' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_date_printers_final' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_journal.date_printers_first',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.date_printers_first' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_date_printers_first' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_journal.date_proofs_in',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.date_proofs_in' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_date_proofs_in' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_journal.date_proofs_out',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.date_proofs_out' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_date_proofs_out' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_journal.date_recd_copyright',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.date_recd_copyright' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_date_recd_copyright' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_journal.date_recd_electronic',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.date_recd_electronic' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_date_recd_electronic' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_journal.date_recd_hard_copy',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.date_recd_hard_copy' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_date_recd_hard_copy' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_journal.issue',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.issue' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_issue' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.language',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.language' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_language' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.name_full',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.name_full' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_name_full' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.page_first',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.page_first' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_page_first' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.page_last',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.page_last' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_page_last' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.paper_category',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.paper_category' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_paper_category' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.suppl_publ_number',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.suppl_publ_number' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_suppl_publ_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.suppl_publ_pages',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.suppl_publ_pages' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_suppl_publ_pages' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.techeditor_address',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.techeditor_address' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_techeditor_address' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_journal.techeditor_code',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.techeditor_code' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_techeditor_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.techeditor_email',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.techeditor_email' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_techeditor_email' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.techeditor_fax',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.techeditor_fax' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_techeditor_fax' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.techeditor_name',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.techeditor_name' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_techeditor_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.techeditor_notes',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.techeditor_notes' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_techeditor_notes' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_journal.techeditor_phone',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.techeditor_phone' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_journal_techeditor_phone' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.volume',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.volume' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_volume' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal.year',
        [ [ 'item_',
            '_item_description.description',
            '              Journal data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal.year' ],
          [ 'item_', '_item.category_id', 'journal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_year' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '##################' ],
      [ '@comment', '# JOURNAL_INDEX ##' ],
      [ '@comment', '##################' ],
      [ 'save_',
        'journal_index',
        [ [ 'item_',
            '_category.description',
            '              Data items in the JOURNAL_INDEX category are used to list terms\n               used to generate the journal indexes.\n\n               The creator of a data block will not normally specify these data\n               items.' ],
          [ 'item_', '_category.id', 'journal_index' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_journal_index.type', '_journal_index.term' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'iucr_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Zhu, Reynolds, Klein & Trudell\n                [Acta Cryst. (1994), C50, 2067-2069].',
              '\n    loop_\n    _journal_index.type\n    _journal_index.term\n    _journal_index.subterm\n      O   C16H19NO4            .\n      S   alkaloids           (-)-norcocaine\n      S   (-)-norcocaine       .\n      S\n    ;  [2R,3S-(2\\b,3\\b)]-methyl\n       3-(benzoyloxy)-8-azabicyclo[3.2.1]octane-2-carboxylate\n    ;\n        .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_journal_index.subterm',
        [ [ 'item_',
            '_item_description.description',
            '              Journal index data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal_index.subterm' ],
          [ 'item_', '_item.category_id', 'journal_index' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_index_subterm' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal_index.term',
        [ [ 'item_',
            '_item_description.description',
            '              Journal index data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal_index.term' ],
          [ 'item_', '_item.category_id', 'journal_index' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_index_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_journal_index.type',
        [ [ 'item_',
            '_item_description.description',
            '              Journal index data items are defined by the journal staff.' ],
          [ 'item_', '_item.name', '_journal_index.type' ],
          [ 'item_', '_item.category_id', 'journal_index' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_journal_index_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '############' ],
      [ '@comment', '# PHASING ##' ],
      [ '@comment', '############' ],
      [ 'save_',
        'phasing',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING category record details about the\n               phasing of the structure, listing the various methods used in\n               the phasing process. Details about the application of each\n               method are listed in the appropriate subcategories.' ],
          [ 'item_', '_category.id', 'phasing' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_phasing.method' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - hypothetical example.',
              '\n    loop_\n    _phasing.method\n    \'mir\'\n    \'averaging\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing.method',
        [ [ 'item_',
            '_item_description.description',
            '              A listing of the method or methods used to phase this\n               structure.' ],
          [ 'item_', '_item.name', '_phasing.method' ],
          [ 'item_', '_item.category_id', 'phasing' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ 'abinitio',
              '                                 phasing by ab initio methods',
              'averaging',
              '                                 phase improvement by averaging over multiple\n                                  images of the structure',
              'dm',
              '                                 phasing by direct methods',
              'isas',
              '                                 phasing by iterative single-wavelength\n                                  anomalous scattering',
              'isir',
              '                                 phasing by iterative single-wavelength\n                                  isomorphous replacement',
              'isomorphous',
              '                                 phasing beginning with phases calculated from\n                                  an isomorphous structure',
              'mad',
              '                                 phasing by multiple-wavelength anomalous\n                                  dispersion',
              'mir',
              '                                 phasing by multiple isomorphous replacement',
              'miras',
              '                                 phasing by multiple isomorphous replacement\n                                  with anomalous scattering',
              'mr',
              '                                 phasing by molecular replacement',
              'sir',
              '                                 phasing by single isomorphous replacement',
              'siras',
              '                                 phasing by single isomorphous replacement\n                                  with anomalous scattering' ] ] ] ],
      [ '@comment', '######################' ],
      [ '@comment', '# PHASING_AVERAGING ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'phasing_averaging',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_AVERAGING category record details\n               about the phasing of the structure where methods involving\n               averaging of multiple observations of the molecule in the\n               asymmetric unit are involved.' ],
          [ 'item_', '_category.id', 'phasing_averaging' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_phasing_averaging.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - hypothetical example.',
              '\n    _phasing_averaging.entry_id     \'EXAMHYPO\'\n    _phasing_averaging.method\n    ; Iterative threefold averaging alternating with phase\n      extensions by 0.5 reciprocal lattice units per cycle.\n    ;\n    _phasing_averaging.details\n    ; The position of the threefold axis was redetermined every\n      five cycles.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_averaging.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the averaging process.' ],
          [ 'item_', '_item.name', '_phasing_averaging.details' ],
          [ 'item_', '_item.category_id', 'phasing_averaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_phasing_averaging.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_phasing_averaging.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_averaging.method',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the phase-averaging phasing method used to\n               phase this structure.\n\n               Note that this is not the computer program used, which is\n               described in the SOFTWARE category, but rather the method\n               itself.\n\n               This data item should be used to describe significant\n               methodological options used within the phase-averaging program.' ],
          [ 'item_', '_item.name', '_phasing_averaging.method' ],
          [ 'item_', '_item.category_id', 'phasing_averaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# PHASING_ISOMORPHOUS ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'phasing_isomorphous',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_ISOMORPHOUS category record details\n               about the phasing of the structure where a model isomorphous\n               to the structure being phased was used to generate the initial\n               phases.' ],
          [ 'item_', '_category.id', 'phasing_isomorphous' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_phasing_isomorphous.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 4PHV and laboratory records for the\n                structure corresponding to PDB entry 4PHV.',
              '\n    _phasing_isomorphous.parent            \'PDB entry 5HVP\'\n    _phasing_isomorphous.details\n    ; The inhibitor and all solvent atoms were removed from the\n      parent structure before beginning refinement. All static\n      disorder present in the parent structure was also removed.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_isomorphous.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the isomorphous phasing.' ],
          [ 'item_', '_item.name', '_phasing_isomorphous.details' ],
          [ 'item_', '_item.category_id', 'phasing_isomorphous' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Residues 13-18 were eliminated from the\n                                  starting model as it was anticipated that\n                                  binding of the inhibitor would cause a\n                                  structural rearrangement in this part of the\n                                  structure.' ] ] ],
      [ 'save_',
        '_phasing_isomorphous.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_phasing_isomorphous.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_isomorphous.method',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the isomorphous-phasing method used to\n               phase this structure.\n\n               Note that this is not the computer program used, which is\n               described in the SOFTWARE category, but rather the method\n               itself.\n\n               This data item should be used to describe significant\n               methodological options used within the isomorphous phasing\n               program.' ],
          [ 'item_', '_item.name', '_phasing_isomorphous.method' ],
          [ 'item_', '_item.category_id', 'phasing_isomorphous' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Iterative threefold averaging alternating with\n                                  phase extension by 0.5 reciprocal lattice\n                                  units per cycle.' ] ] ],
      [ 'save_',
        '_phasing_isomorphous.parent',
        [ [ 'item_',
            '_item_description.description',
            '              Reference to the structure used to generate starting phases\n               if the structure referenced in this data block was phased\n               by virtue of being isomorphous to a known structure (e.g.\n               a mutant that crystallizes in the same space group as the\n               wild-type protein.)' ],
          [ 'item_', '_item.name', '_phasing_isomorphous.parent' ],
          [ 'item_', '_item.category_id', 'phasing_isomorphous' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# PHASING_MAD ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'phasing_MAD',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MAD category record details about\n               the phasing of the structure where methods involving\n               multiple-wavelength anomalous-dispersion techniques are involved.' ],
          [ 'item_', '_category.id', 'phasing_MAD' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_phasing_MAD.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Shapiro et al. [Nature (London)\n                (1995), 374, 327-337].',
              '\n    _phasing_MAD.entry_id        \'NCAD\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MAD.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.details' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_phasing_MAD.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_phasing_MAD.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MAD.method',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the MAD phasing method used to phase\n               this structure.\n\n               Note that this is not the computer program used, which is\n               described in the SOFTWARE category, but rather the method\n               itself.\n\n               This data item should be used to describe significant\n               methodological options used within the MAD phasing program.' ],
          [ 'item_', '_item.name', '_phasing_MAD.method' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '######################' ],
      [ '@comment', '# PHASING_MAD_CLUST ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'phasing_MAD_clust',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MAD_CLUST category record details\n               about a cluster of experiments that contributed to the\n               generation of a set of phases.' ],
          [ 'item_', '_category.id', 'phasing_MAD_clust' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_phasing_MAD_clust.expt_id', '_phasing_MAD_clust.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Shapiro et al. [Nature (London)\n                (1995), 374, 327-337].',
              '\n    loop_\n    _phasing_MAD_clust.id\n    _phasing_MAD_clust.expt_id\n    _phasing_MAD_clust.number_set\n      \'4 wavelength\'  1  4\n      \'5 wavelength\'  1  5\n      \'5 wavelength\'  2  5',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MAD_clust.expt_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_MAD_expt.id in the\n               PHASING_MAD_EXPT category.' ],
          [ 'item_', '_item.name', '_phasing_MAD_clust.expt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MAD_clust.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _phasing_MAD_clust.id must, together with\n               _phasing_MAD_clust.expt_id, uniquely identify a record in the\n               PHASING_MAD_CLUST list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_phasing_MAD_clust.id',
              'phasing_MAD_clust',
              'yes',
              '_phasing_MAD_set.clust_id',
              'phasing_MAD_set',
              'yes',
              '_phasing_MAD_ratio.clust_id',
              'phasing_MAD_ratio',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_phasing_MAD_set.clust_id',
              '_phasing_MAD_clust.id',
              '_phasing_MAD_ratio.clust_id',
              '_phasing_MAD_clust.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_phasing_MAD_clust.number_set',
        [ [ 'item_',
            '_item_description.description',
            '              The number of data sets in this cluster of data sets.' ],
          [ 'item_', '_item.name', '_phasing_MAD_clust.number_set' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_clust' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# PHASING_MAD_EXPT ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'phasing_MAD_expt',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MAD_EXPT category record details about\n               a MAD phasing experiment, such as the number of experiments that\n               were clustered together to produce a set of phases or the\n               statistics for those phases.' ],
          [ 'item_', '_category.id', 'phasing_MAD_expt' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_phasing_MAD_expt.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Shapiro et al. [Nature (London)\n                (1995), 374, 327-337].',
              '\n    loop_\n    _phasing_MAD_expt.id\n    _phasing_MAD_expt.number_clust\n    _phasing_MAD_expt.R_normal_all\n    _phasing_MAD_expt.R_normal_anom_scat\n    _phasing_MAD_expt.delta_delta_phi\n    _phasing_MAD_expt.delta_phi_sigma\n    _phasing_MAD_expt.mean_fom\n      1  2  0.063  0.451  58.5  20.3  0.88\n      2  1  0.051  0.419  36.8  18.2  0.93',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MAD_expt.delta_delta_phi',
        [ [ 'item_',
            '_item_description.description',
            '              The difference between two independent determinations of\n               _phasing_MAD_expt.delta_phi.' ],
          [ 'item_', '_item.name', '_phasing_MAD_expt.delta_delta_phi' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_expt' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_expt.delta_phi',
        [ [ 'item_',
            '_item_description.description',
            '              The phase difference between F~t~(h), the structure factor due\n               to normal scattering from all atoms, and F~a~(h), the structure\n               factor due to normal scattering from only the anomalous\n               scatterers.' ],
          [ 'item_', '_item.name', '_phasing_MAD_expt.delta_phi' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_expt' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MAD_expt.delta_phi_sigma' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ] ] ],
      [ 'save_',
        '_phasing_MAD_expt.delta_phi_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_MAD_expt.delta_phi.' ],
          [ 'item_', '_item.name', '_phasing_MAD_expt.delta_phi_sigma' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_expt' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MAD_expt.delta_phi' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_expt.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _phasing_MAD_expt.id must uniquely identify each\n               record in the PHASING_MAD_EXPT list.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_phasing_MAD_expt.id',
              'phasing_MAD_expt',
              'yes',
              '_phasing_MAD_clust.expt_id',
              'phasing_MAD_clust',
              'yes',
              '_phasing_MAD_set.expt_id',
              'phasing_MAD_set',
              'yes',
              '_phasing_MAD_ratio.expt_id',
              'phasing_MAD_ratio',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_phasing_MAD_clust.expt_id',
              '_phasing_MAD_expt.id',
              '_phasing_MAD_set.expt_id',
              '_phasing_MAD_expt.id',
              '_phasing_MAD_ratio.expt_id',
              '_phasing_MAD_expt.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_phasing_MAD_expt.mean_fom',
        [ [ 'item_',
            '_item_description.description',
            '              The mean figure of merit.' ],
          [ 'item_', '_item.name', '_phasing_MAD_expt.mean_fom' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_expt' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_expt.number_clust',
        [ [ 'item_',
            '_item_description.description',
            '              The number of clusters of data sets in this phasing experiment.' ],
          [ 'item_', '_item.name', '_phasing_MAD_expt.number_clust' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_expt' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MAD_expt.R_normal_all',
        [ [ 'item_',
            '_item_description.description',
            '              Definition...' ],
          [ 'item_', '_item.name', '_phasing_MAD_expt.R_normal_all' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_expt' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_expt.R_normal_anom_scat',
        [ [ 'item_',
            '_item_description.description',
            '              Definition...' ],
          [ 'item_', '_item.name', '_phasing_MAD_expt.R_normal_anom_scat' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_expt' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '######################' ],
      [ '@comment', '# PHASING_MAD_RATIO ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'phasing_MAD_ratio',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MAD_RATIO category record\n               the ratios of phasing statistics between pairs of data sets\n               in a MAD phasing experiment, in given shells of resolution.' ],
          [ 'item_', '_category.id', 'phasing_MAD_ratio' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_phasing_MAD_ratio.clust_id',
              '_phasing_MAD_ratio.expt_id',
              '_phasing_MAD_ratio.wavelength_1',
              '_phasing_MAD_ratio.wavelength_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Shapiro et al. [Nature (London)\n                (1995), 374, 327-337].',
              '\n    loop_\n    _phasing_MAD_ratio.expt_id\n    _phasing_MAD_ratio.clust_id\n    _phasing_MAD_ratio.wavelength_1\n    _phasing_MAD_ratio.wavelength_2\n    _phasing_MAD_ratio.d_res_low\n    _phasing_MAD_ratio.d_res_high\n    _phasing_MAD_ratio.ratio_two_wl\n    _phasing_MAD_ratio.ratio_one_wl\n    _phasing_MAD_ratio.ratio_one_wl_centric\n     1 \'4 wavelength\' 1.4013 1.4013 20.00 4.00     . 0.084 0.076\n     1 \'4 wavelength\' 1.4013 1.3857 20.00 4.00 0.067     .     .\n     1 \'4 wavelength\' 1.4013 1.3852 20.00 4.00 0.051     .     .\n     1 \'4 wavelength\' 1.4013 1.3847 20.00 4.00 0.044     .     .\n     1 \'4 wavelength\' 1.3857 1.3857 20.00 4.00     . 0.110 0.049\n     1 \'4 wavelength\' 1.3857 1.3852 20.00 4.00 0.049     .     .\n     1 \'4 wavelength\' 1.3857 1.3847 20.00 4.00 0.067     .     .\n     1 \'4 wavelength\' 1.3852 1.3852 20.00 4.00     . 0.149 0.072\n     1 \'4 wavelength\' 1.3852 1.3847 20.00 4.00 0.039     .     .\n     1 \'4 wavelength\' 1.3847 1.3847 20.00 4.00     . 0.102 0.071\n\n     1 \'4 wavelength\' 1.4013 1.4013  4.00 3.00     . 0.114 0.111\n     1 \'4 wavelength\' 1.4013 1.3857  4.00 3.00 0.089     .     .\n     1 \'4 wavelength\' 1.4013 1.3852  4.00 3.00 0.086     .     .\n     1 \'4 wavelength\' 1.4013 1.3847  4.00 3.00 0.077     .     .\n     1 \'4 wavelength\' 1.3857 1.3857  4.00 3.00     . 0.140 0.127\n     1 \'4 wavelength\' 1.3857 1.3852  4.00 3.00 0.085     .     .\n     1 \'4 wavelength\' 1.3857 1.3847  4.00 3.00 0.089     .     .\n     1 \'4 wavelength\' 1.3852 1.3852  4.00 3.00     . 0.155 0.119\n     1 \'4 wavelength\' 1.3852 1.3847  4.00 3.00 0.082     .     .\n     1 \'4 wavelength\' 1.3847 1.3847  4.00 3.00     . 0.124 0.120\n\n     1 \'5 wavelength\' 1.3857 1.3857 20.00 4.00     . 0.075 0.027\n     1 \'5 wavelength\' 1.3857 1.3852 20.00 4.00 0.041     .     .\n     1 \'5 wavelength\' 1.3857 1.3847 20.00 4.00 0.060     .     .\n     1 \'5 wavelength\' 1.3857 1.3784 20.00 4.00 0.057     .     .\n     1 \'5 wavelength\' 1.3857 1.2862 20.00 4.00 0.072     .     .\n     1 \'5 wavelength\' 1.3852 1.3852 20.00 4.00     . 0.105 0.032\n     1 \'5 wavelength\' 1.3852 1.3847 20.00 4.00 0.036     .     .\n     1 \'5 wavelength\' 1.3852 1.3784 20.00 4.00 0.044     .     .\n     1 \'5 wavelength\' 1.3852 1.2862 20.00 4.00 0.065     .     .\n     1 \'5 wavelength\' 1.3847 1.3847 20.00 4.00     . 0.072 0.031\n     1 \'5 wavelength\' 1.3847 1.3784 20.00 4.00 0.040     .     .\n     1 \'5 wavelength\' 1.3847 1.2862 20.00 4.00 0.059     .     .\n     1 \'5 wavelength\' 1.3784 1.3784 20.00 4.00     . 0.059 0.032\n     1 \'5 wavelength\' 1.3784 1.2862 20.00 4.00 0.059     .     .\n     1 \'5 wavelength\' 1.2862 1.3847 20.00 4.00     . 0.058 0.028\n\n     1 \'5 wavelength\' 1.3857 1.3857  4.00 3.00     . 0.078 0.075\n     1 \'5 wavelength\' 1.3857 1.3852  4.00 3.00 0.059     .     .\n     1 \'5 wavelength\' 1.3857 1.3847  4.00 3.00 0.067     .     .\n     1 \'5 wavelength\' 1.3857 1.3784  4.00 3.00 0.084     .     .\n     1 \'5 wavelength\' 1.3857 1.2862  4.00 3.00 0.073     .     .\n     1 \'5 wavelength\' 1.3852 1.3852  4.00 3.00     . 0.101 0.088\n     1 \'5 wavelength\' 1.3852 1.3847  4.00 3.00 0.066     .     .\n     1 \'5 wavelength\' 1.3852 1.3784  4.00 3.00 0.082     .     .\n     1 \'5 wavelength\' 1.3852 1.2862  4.00 3.00 0.085     .     .\n     1 \'5 wavelength\' 1.3847 1.3847  4.00 3.00     . 0.097 0.074\n     1 \'5 wavelength\' 1.3847 1.3784  4.00 3.00 0.081     .     .\n     1 \'5 wavelength\' 1.3847 1.2862  4.00 3.00 0.085     .     .\n     1 \'5 wavelength\' 1.3784 1.3784  4.00 3.00     . 0.114 0.089\n     1 \'5 wavelength\' 1.3784 1.2862  4.00 3.00 0.103     .     .\n     1 \'5 wavelength\' 1.2862 1.2862  4.00 3.00     . 0.062 0.060\n\n     2 \'5 wavelength\' 0.7263 0.7263 15.00 3.00     . 0.035 0.026\n     2 \'5 wavelength\' 0.7263 0.7251 15.00 3.00 0.028     .     .\n     2 \'5 wavelength\' 0.7263 0.7284 15.00 3.00 0.023     .     .\n     2 \'5 wavelength\' 0.7263 0.7246 15.00 3.00 0.025     .     .\n     2 \'5 wavelength\' 0.7263 0.7217 15.00 3.00 0.026     .     .\n     2 \'5 wavelength\' 0.7251 0.7251 15.00 3.00     . 0.060 0.026\n     2 \'5 wavelength\' 0.7251 0.7284 15.00 3.00 0.029     .     .\n     2 \'5 wavelength\' 0.7251 0.7246 15.00 3.00 0.031     .     .\n     2 \'5 wavelength\' 0.7251 0.7217 15.00 3.00 0.035     .     .\n     2 \'5 wavelength\' 0.7284 0.7284 15.00 3.00     . 0.075 0.030\n     2 \'5 wavelength\' 0.7284 0.7246 15.00 3.00 0.023     .     .\n     2 \'5 wavelength\' 0.7284 0.7217 15.00 3.00 0.027     .     .\n     2 \'5 wavelength\' 0.7246 0.7246 15.00 3.00     . 0.069 0.026\n     2 \'5 wavelength\' 0.7246 0.7217 15.00 3.00 0.024     .     .\n     2 \'5 wavelength\' 0.7217 0.7284 15.00 3.00     . 0.060 0.028\n\n     2 \'5 wavelength\' 0.7263 0.7263  3.00 1.90     . 0.060 0.050\n     2 \'5 wavelength\' 0.7263 0.7251  3.00 1.90 0.056     .     .\n     2 \'5 wavelength\' 0.7263 0.7284  3.00 1.90 0.055     .     .\n     2 \'5 wavelength\' 0.7263 0.7246  3.00 1.90 0.053     .     .\n     2 \'5 wavelength\' 0.7263 0.7217  3.00 1.90 0.056     .     .\n     2 \'5 wavelength\' 0.7251 0.7251  3.00 1.90     . 0.089 0.050\n     2 \'5 wavelength\' 0.7251 0.7284  3.00 1.90 0.054     .     .\n     2 \'5 wavelength\' 0.7251 0.7246  3.00 1.90 0.058     .     .\n     2 \'5 wavelength\' 0.7251 0.7217  3.00 1.90 0.063     .     .\n     2 \'5 wavelength\' 0.7284 0.7284  3.00 1.90     . 0.104 0.057\n     2 \'5 wavelength\' 0.7284 0.7246  3.00 1.90 0.052     .     .\n     2 \'5 wavelength\' 0.7284 0.7217  3.00 1.90 0.057     .     .\n     2 \'5 wavelength\' 0.7246 0.7246  3.00 1.90     . 0.098 0.052\n     2 \'5 wavelength\' 0.7246 0.7217  3.00 1.90 0.054     .     .\n     2 \'5 wavelength\' 0.7217 0.7284  3.00 1.90     . 0.089 0.060',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MAD_ratio.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The lowest value for the interplanar spacings for the\n               reflection data used for the comparison of Bijvoet differences.\n               This is called the highest resolution.' ],
          [ 'item_', '_item.name', '_phasing_MAD_ratio.d_res_high' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_ratio' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_ratio.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The highest value for the interplanar spacings for the\n               reflection data used for the comparison of Bijvoet differences.\n               This is called the lowest resolution.' ],
          [ 'item_', '_item.name', '_phasing_MAD_ratio.d_res_low' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_ratio' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_ratio.expt_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_MAD_expt.id in the\n               PHASING_MAD_EXPT category.' ],
          [ 'item_', '_item.name', '_phasing_MAD_ratio.expt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MAD_ratio.clust_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_MAD_clust.id in\n               the PHASING_MAD_CLUST category.' ],
          [ 'item_', '_item.name', '_phasing_MAD_ratio.clust_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MAD_ratio.ratio_one_wl',
        [ [ 'item_',
            '_item_description.description',
            '              The root-mean-square Bijvoet difference at one wavelength for\n               all reflections.' ],
          [ 'item_', '_item.name', '_phasing_MAD_ratio.ratio_one_wl' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_ratio' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_ratio.ratio_one_wl_centric',
        [ [ 'item_',
            '_item_description.description',
            '              The root-mean-square Bijvoet difference at one wavelength for\n               centric reflections. This would be equal to zero for perfect\n               data and thus serves as an estimate of the noise in the\n               anomalous signals.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MAD_ratio.ratio_one_wl_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_ratio' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_ratio.ratio_two_wl',
        [ [ 'item_',
            '_item_description.description',
            '              The root-mean-square dispersive Bijvoet difference between\n               two wavelengths for all reflections.' ],
          [ 'item_', '_item.name', '_phasing_MAD_ratio.ratio_two_wl' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_ratio' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_ratio.wavelength_1',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_MAD_set.wavelength in\n               the PHASING_MAD_SET category.' ],
          [ 'item_', '_item.name', '_phasing_MAD_ratio.wavelength_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MAD_ratio.wavelength_2',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_MAD_set.wavelength in\n               the PHASING_MAD_SET category.' ],
          [ 'item_', '_item.name', '_phasing_MAD_ratio.wavelength_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# PHASING_MAD_SET ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'phasing_MAD_set',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MAD_SET category record\n               details about the individual data sets used in a MAD phasing\n               experiment.' ],
          [ 'item_', '_category.id', 'phasing_MAD_set' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_phasing_MAD_set.expt_id',
              '_phasing_MAD_set.clust_id',
              '_phasing_MAD_set.set_id',
              '_phasing_MAD_set.wavelength' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Shapiro et al. [Nature (London)\n                (1995), 374, 327-337].',
              '\n    loop_\n    _phasing_MAD_set.expt_id\n    _phasing_MAD_set.clust_id\n    _phasing_MAD_set.set_id\n    _phasing_MAD_set.wavelength\n    _phasing_MAD_set.wavelength_details\n    _phasing_MAD_set.d_res_low\n    _phasing_MAD_set.d_res_high\n    _phasing_MAD_set.f_prime\n    _phasing_MAD_set.f_double_prime\n      1 \'4 wavelength\' aa 1.4013 \'pre-edge\'        20.00   3.00\n                                                  -12.48   3.80\n      1 \'4 wavelength\' bb 1.3857 \'peak\'           20.00   3.00\n                                                  -31.22  17.20\n      1 \'4 wavelength\' cc 1.3852 \'edge\'            20.00   3.00\n                                                  -13.97  29.17\n      1 \'4 wavelength\' dd 1.3847 \'remote\'          20.00   3.00\n                                                   -6.67  17.34\n      1 \'5 wavelength\' ee 1.3857 \'ascending edge\'  20.00   3.00\n                                                  -28.33  14.84\n      1 \'5 wavelength\' ff 1.3852 \'peak\'            20.00   3.00\n                                                  -21.50  30.23\n      1 \'5 wavelength\' gg 1.3847 \'descending edge\' 20.00   3.00\n                                                  -10.71  20.35\n      1 \'5 wavelength\' hh 1.3784 \'remote 1\'        20.00   3.00\n                                                  -14.45  11.84\n      1 \'5 wavelength\' ii 1.2862 \'remote 2\'        20.00   3.00\n                                                   -9.03   9.01\n      2 \'5 wavelength\' jj 0.7263 \'pre-edge\'        15.00   1.90\n                                                  -21.10   4.08\n      2 \'5 wavelength\' kk 0.7251 \'edge\'            15.00   1.90\n                                                  -34.72   7.92\n      2 \'5 wavelength\' ll 0.7248 \'peak\'            15.00   1.90\n                                                  -24.87  10.30\n      2 \'5 wavelength\' mm 0.7246 \'descending edge\' 15.00   1.90\n                                                  -17.43   9.62\n      2 \'5 wavelength\' nn 0.7217 \'remote\'          15.00   1.90\n                                                  -13.26   8.40',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MAD_set.clust_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_MAD_clust.id in\n               the PHASING_MAD_CLUST category.' ],
          [ 'item_', '_item.name', '_phasing_MAD_set.clust_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MAD_set.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The lowest value for the interplanar spacings for the\n               reflection data used for this set of data. This is called\n               the highest resolution.' ],
          [ 'item_', '_item.name', '_phasing_MAD_set.d_res_high' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_set.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The highest value for the interplanar spacings for the\n               reflection data used for this set of data. This is called\n               the lowest resolution.' ],
          [ 'item_', '_item.name', '_phasing_MAD_set.d_res_low' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_set.expt_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_MAD_expt.id in the\n               PHASING_MAD_EXPT category.' ],
          [ 'item_', '_item.name', '_phasing_MAD_set.expt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MAD_set.f_double_prime',
        [ [ 'item_',
            '_item_description.description',
            '              The f\'\' component of the anomalous scattering factor for this\n               wavelength.' ],
          [ 'item_', '_item.name', '_phasing_MAD_set.f_double_prime' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_set.f_prime',
        [ [ 'item_',
            '_item_description.description',
            '              The f\' component of the anomalous scattering factor for this\n               wavelength.' ],
          [ 'item_', '_item.name', '_phasing_MAD_set.f_prime' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_set.set_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_set.id in the\n               PHASING_SET category.' ],
          [ 'item_', '_item.name', '_phasing_MAD_set.set_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MAD_set.wavelength',
        [ [ 'item_',
            '_item_description.description',
            '              The wavelength at which this data set was measured.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_phasing_MAD_set.wavelength',
              'phasing_MAD_set',
              'yes',
              '_phasing_MAD_ratio.wavelength_1',
              'phasing_MAD_ratio',
              'yes',
              '_phasing_MAD_ratio.wavelength_2',
              'phasing_MAD_ratio',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_phasing_MAD_ratio.wavelength_1',
              '_phasing_MAD_set.wavelength',
              '_phasing_MAD_ratio.wavelength_2',
              '_phasing_MAD_set.wavelength' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MAD_set.wavelength_details',
        [ [ 'item_',
            '_item_description.description',
            '              A descriptor for this wavelength in this cluster of data sets.' ],
          [ 'item_', '_item.name', '_phasing_MAD_set.wavelength_details' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'peak', 'remote', 'ascending edge' ] ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# PHASING_MIR ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'phasing_MIR',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MIR category record details about\n               the phasing of the structure where methods involving isomorphous\n               replacement are involved.\n\n               All isomorphous-replacement-based techniques are covered\n               by this category, including single isomorphous replacement (SIR),\n               multiple isomorphous replacement (MIR) and single or multiple\n               isomorphous replacement plus anomalous scattering (SIRAS, MIRAS).' ],
          [ 'item_', '_category.id', 'phasing_MIR' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_phasing_MIR.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Zanotti et al. [J. Biol. Chem.\n                (1993), 268, 10728-10738].',
              '\n    _phasing_MIR.method\n    ; Standard phase refinement (Blow & Crick, 1959)\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MIR.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the isomorphous-replacement\n               phasing.' ],
          [ 'item_', '_item.name', '_phasing_MIR.details' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_phasing_MIR.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The lowest value in angstroms for the interplanar spacings\n               for the reflection data used for the native data set. This is\n               called the highest resolution.' ],
          [ 'item_', '_item.name', '_phasing_MIR.d_res_high' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR.ebi_d_res_high' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The highest value in angstroms for the interplanar spacings\n               for the reflection data used for the native data set. This is\n               called the lowest resolution.' ],
          [ 'item_', '_item.name', '_phasing_MIR.d_res_low' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR.ebi_d_res_low' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_phasing_MIR.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MIR.FOM',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the figure of merit m for all reflections\n               phased in the native data set.\n\n                   int P~alpha~ exp(i*alpha) dalpha\n               m = --------------------------------\n                         int P~alpha~ dalpha\n\n               P~a~ = the probability that the phase angle a is correct\n\n               the integral is taken over the range alpha = 0 to 2 pi.' ],
          [ 'item_', '_item.name', '_phasing_MIR.FOM' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_phasing_MIR.ebi_fom' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR.FOM_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the figure of merit m for the acentric\n               reflections phased in the native data set.\n\n                   int P~alpha~ exp(i*alpha) dalpha\n               m = --------------------------------\n                         int P~alpha~ dalpha\n\n               P~a~ = the probability that the phase angle a is correct\n\n               the integral is taken over the range alpha = 0 to 2 pi.' ],
          [ 'item_', '_item.name', '_phasing_MIR.FOM_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR.ebi_fom_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR.FOM_centric',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the figure of merit m for the centric\n               reflections phased in the native data set.\n\n                   int P~alpha~ exp(i*alpha) dalpha\n               m = --------------------------------\n                         int P~alpha~ dalpha\n\n               P~a~ = the probability that the phase angle a is correct\n\n               the integral is taken over the range alpha = 0 to 2 pi.' ],
          [ 'item_', '_item.name', '_phasing_MIR.FOM_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR.ebi_fom_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR.method',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the MIR phasing method applied to phase this\n               structure.\n\n               Note that this is not the computer program used, which is\n               described in the SOFTWARE category, but rather the method\n               itself.\n\n               This data item should be used to describe significant\n               methodological options used within the MIR phasing program.' ],
          [ 'item_', '_item.name', '_phasing_MIR.method' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_phasing_MIR.reflns',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of reflections phased in the native data set.' ],
          [ 'item_', '_item.name', '_phasing_MIR.reflns' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR.ebi_reflns' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR.reflns_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              The number of acentric reflections phased in the native data\n               set.' ],
          [ 'item_', '_item.name', '_phasing_MIR.reflns_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR.ebi_reflns_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR.reflns_centric',
        [ [ 'item_',
            '_item_description.description',
            '              The number of centric reflections phased in the native data\n               set.' ],
          [ 'item_', '_item.name', '_phasing_MIR.reflns_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR.ebi_reflns_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR.reflns_criterion',
        [ [ 'item_',
            '_item_description.description',
            '              Criterion used to limit the reflections used in the phasing\n               calculations.' ],
          [ 'item_', '_item.name', '_phasing_MIR.reflns_criterion' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR.ebi_reflns_criteria' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', '> 4 \\s(I)' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# PHASING_MIR_DER ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'phasing_MIR_der',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MIR_DER category record details\n               about individual derivatives used in the phasing of the\n               structure when methods involving isomorphous replacement are\n               involved.\n\n               A derivative in this context does not necessarily equate with\n               a data set; for instance, the same data set could be used to\n               one resolution limit as an isomorphous scatterer and to a\n               different resolution (and with a different sigma cutoff) as an\n               anomalous scatterer. These would be treated as two distinct\n               derivatives, although both derivatives would point to the same\n               data sets via _phasing_MIR_der.der_set_id and\n               _phasing_MIR_der.native_set_id.' ],
          [ 'item_', '_category.id', 'phasing_MIR_der' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_phasing_MIR_der.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Zanotti et al. [J. Biol. Chem.\n                (1993), 268, 10728-10738].',
              '\n    loop_\n    _phasing_MIR_der.id\n    _phasing_MIR_der.number_of_sites\n    _phasing_MIR_der.details\n     KAu(CN)2  3\n           \'major site interpreted in difference Patterson\'\n     K2HgI4    6  \'sites found in cross-difference Fourier\'\n     K3IrCl6   2  \'sites found in cross-difference Fourier\'\n     All      11  \'data for all three derivatives combined\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MIR_der.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The lowest value for the interplanar spacings for the\n               reflection data used for this derivative. This is called the\n               highest resolution.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.d_res_high' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The highest value for  the interplanar spacings for the\n               reflection data used for this derivative. This is called the\n               lowest resolution.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.d_res_low' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.der_set_id',
        [ [ 'item_',
            '_item_description.description',
            '              The data set that was treated as the derivative in this\n               experiment.\n\n               This data item is a pointer to _phasing_set.id in the\n               PHASING_SET category.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.der_set_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of this derivative, its data,\n               its solution or its use in phasing.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.details' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _phasing_MIR_der.id must uniquely identify\n               a record in the PHASING_MIR_DER list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_phasing_MIR_der.id',
              'phasing_MIR_der',
              'yes',
              '_phasing_MIR_der_refln.der_id',
              'phasing_MIR_der_refln',
              'yes',
              '_phasing_MIR_der_shell.der_id',
              'phasing_MIR_der_shell',
              'yes',
              '_phasing_MIR_der_site.der_id',
              'phasing_MIR_der_site',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_phasing_MIR_der_refln.der_id',
              '_phasing_MIR_der.id',
              '_phasing_MIR_der_shell.der_id',
              '_phasing_MIR_der.id',
              '_phasing_MIR_der_site.der_id',
              '_phasing_MIR_der.id' ] ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'KAu(CN)2', 'K2HgI4_anom', 'K2HgI4_iso' ] ] ] ],
      [ 'save_',
        '_phasing_MIR_der.native_set_id',
        [ [ 'item_',
            '_item_description.description',
            '              The data set that was treated as the native in this\n               experiment.\n\n               This data item is a pointer to _phasing_set.id in the\n               PHASING_SET category.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.native_set_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.number_of_sites',
        [ [ 'item_',
            '_item_description.description',
            '              The number of heavy-atom sites in this derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.number_of_sites' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.power_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              The mean phasing power P for acentric reflections for this\n               derivative.\n\n                          sum|Fh~calc~^2^|\n               P = (----------------------------)^1/2^\n                    sum|Fph~obs~ - Fph~calc~|^2^\n\n               Fph~obs~  = the observed structure-factor amplitude of this\n                           derivative\n               Fph~calc~ = the calculated structure-factor amplitude of this\n                           derivative\n               Fh~calc~  = the calculated structure-factor amplitude from the\n                           heavy-atom model\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.power_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.ebi_power_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.power_centric',
        [ [ 'item_',
            '_item_description.description',
            '              The mean phasing power P for centric reflections for this\n               derivative.\n\n                          sum|Fh~calc~^2^|\n               P = (----------------------------)^1/2^\n                    sum|Fph~obs~ - Fph~calc~|^2^\n\n               Fph~obs~  = the observed structure-factor amplitude of the\n                           derivative\n               Fph~calc~ = the calculated structure-factor amplitude of the\n                           derivative\n               Fh~calc~  = the calculated structure-factor amplitude from the\n                           heavy-atom model\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.power_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.ebi_power_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.R_cullis_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R~cullis,acen~ for acentric reflections for this\n               derivative.\n\n               The Cullis R factor was originally defined only for centric\n               reflections.  It is, however, also a useful statistical\n               measure for acentric reflections, which is how it is used in\n               this data item.\n\n                           sum| |Fph~obs~ +/- Fp~obs~| - Fh~calc~ |\n          R~cullis,acen~ = ----------------------------------------\n                                   sum|Fph~obs~ - Fp~obs~|\n\n               Fp~obs~  = the observed structure-factor amplitude of the native\n               Fph~obs~ = the observed structure-factor amplitude of the\n                          derivative\n               Fh~calc~ = the calculated structure-factor amplitude from the\n                          heavy-atom model\n\n               sum is taken over the specified reflections\n\n               Ref: Cullis, A. F., Muirhead, H., Perutz, M. F., Rossmann, M. G.\n                    & North, A. C. T. (1961). Proc. R. Soc. London Ser. A,\n                    265, 15-38.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.R_cullis_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.ebi_Rcullis_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.R_cullis_anomalous',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R~cullis,ano~ for anomalous reflections for this\n               derivative.\n\n               The Cullis R factor was originally defined only for centric\n               reflections.  It is, however, also a useful statistical\n               measure for anomalous  reflections, which is how it is used in\n               this data item.\n\n               This is tabulated for acentric terms.  A value less than 1.0\n               means there is some contribution to the phasing from the\n               anomalous data.\n\n                            sum |Fph+~obs~Fph-~obs~ - Fh+~calc~ - Fh-~calc~|\n            R~cullis,ano~ = ------------------------------------------------\n                                   sum|Fph+~obs~ - Fph-~obs~|\n\n               Fph+~obs~  = the observed positive Friedel structure-factor\n                            amplitude for the derivative\n               Fph-~obs~  = the observed negative Friedel structure-factor\n                            amplitude for the derivative\n\n               Fh+~calc~  = the calculated positive Friedel structure-factor\n                            amplitude from the heavy-atom model\n               Fh-~calc~  = the calculated negative Friedel structure-factor\n                            amplitude from the heavy-atom model\n\n               sum is taken over the specified reflections\n\n               Ref: Cullis, A. F., Muirhead, H., Perutz, M. F., Rossmann, M. G.\n                    & North, A. C. T. (1961). Proc. R. Soc.  London Ser. A,\n                    265, 15-38.\n' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.R_cullis_anomalous' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.ebi_Rcullis_anomalous' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.R_cullis_centric',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R~cullis~ for centric reflections for this\n               derivative.\n\n                           sum| |Fph~obs~ +/- Fp~obs~| - Fh~calc~ |\n               R~cullis~ = ----------------------------------------\n                                   sum|Fph~obs~ - Fp~obs~|\n\n               Fp~obs~  = the observed structure-factor amplitude of the native\n               Fph~obs~ = the observed structure-factor amplitude of the\n                          derivative\n               Fh~calc~ = the calculated structure-factor amplitude from the\n                          heavy-atom model\n\n               sum is taken over the specified reflections\n\n               Ref: Cullis, A. F., Muirhead, H., Perutz, M. F., Rossmann, M. G.\n                    & North, A. C. T. (1961). Proc. R. Soc. London Ser. A,\n                    265, 15-38.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.R_cullis_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.ebi_Rcullis_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.reflns_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              The number of acentric reflections used in phasing for this\n               derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.reflns_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.ebi_reflns_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.reflns_anomalous',
        [ [ 'item_',
            '_item_description.description',
            '              The number of anomalous reflections used in phasing for this\n               derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.reflns_anomalous' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.ebi_reflns_anomalous' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.reflns_centric',
        [ [ 'item_',
            '_item_description.description',
            '              The number of centric reflections used in phasing for this\n               derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.reflns_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.ebi_reflns_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.reflns_criteria',
        [ [ 'item_',
            '_item_description.description',
            '              Criteria used to limit the reflections used in the phasing\n               calculations.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.reflns_criteria' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', '> 4 \\s(I)' ] ] ],
      [ '@comment', '##########################' ],
      [ '@comment', '# PHASING_MIR_DER_REFLN ##' ],
      [ '@comment', '##########################' ],
      [ 'save_',
        'phasing_MIR_der_refln',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MIR_DER_REFLN category record details\n               about the calculated structure factors obtained in an MIR\n               phasing experiment.\n\n               This list may contain information from a number of different\n               derivatives; _phasing_MIR_der_refln.der_id indicates to which\n               derivative a given record corresponds. (A derivative in this\n               context does not necessarily equate with a data set; see the\n               definition of the PHASING_MIR_DER category for a\n               discussion of the meaning of derivative.)\n\n               It is not necessary for the data items describing the measured\n               value of F to appear in this list, as they will be\n               given in the PHASING_SET_REFLN category. However, these\n               items can also be listed here for completeness.' ],
          [ 'item_', '_category.id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_phasing_MIR_der_refln.index_h',
              '_phasing_MIR_der_refln.index_k',
              '_phasing_MIR_der_refln.index_l',
              '_phasing_MIR_der_refln.der_id',
              '_phasing_MIR_der_refln.set_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for the 6,1,25 reflection\n                of an Hg/Pt derivative of protein NS1.',
              '\n    _phasing_MIR_der_refln.index_h         6\n    _phasing_MIR_der_refln.index_k         1\n    _phasing_MIR_der_refln.index_l        25\n    _phasing_MIR_der_refln.der_id         HGPT1\n    _phasing_MIR_der_refln.set_id       \'NS1-96\'\n    _phasing_MIR_der_refln.F_calc_au     106.66\n    _phasing_MIR_der_refln.F_meas_au     204.67\n    _phasing_MIR_der_refln.F_meas_sigma    6.21\n    _phasing_MIR_der_refln.HL_A_iso       -3.15\n    _phasing_MIR_der_refln.HL_B_iso       -0.76\n    _phasing_MIR_der_refln.HL_C_iso        0.65\n    _phasing_MIR_der_refln.HL_D_iso        0.23\n    _phasing_MIR_der_refln.phase_calc    194.48',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.der_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_MIR_der.id in the\n               PHASING_MIR_DER category.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.der_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.F_calc',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of the structure factor for this derivative,\n               in electrons.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.F_calc' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_refln.F_calc_au' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.F_calc_au',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of the structure factor for this derivative,\n               in arbitrary units.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.F_calc_au' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_refln.F_calc' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.F_meas',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of the structure factor for this derivative,\n               in electrons.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.F_meas' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_phasing_MIR_der_refln.F_meas_sigma',
              'associated_esd',
              '_phasing_MIR_der_refln.F_meas_au',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.F_meas_au',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of the structure factor for this derivative,\n               in arbitrary units.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.F_meas_au' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_phasing_MIR_der_refln.F_meas_sigma_au',
              'associated_esd',
              '_phasing_MIR_der_refln.F_meas',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.F_meas_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_MIR_der_refln.F_meas, in electrons.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.F_meas_sigma' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_phasing_MIR_der_refln.F_meas',
              'associated_value',
              '_phasing_MIR_der_refln.F_meas_sigma_au',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.F_meas_sigma_au',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_MIR_der_refln.F_meas_au, in arbitrary units.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_refln.F_meas_sigma_au' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_phasing_MIR_der_refln.F_meas_au',
              'associated_value',
              '_phasing_MIR_der_refln.F_meas_sigma',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.HL_A_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The isomorphous Hendrickson-Lattman coefficient A~iso~ for this\n               reflection for this derivative.\n\n                         -2.0 * (Fp~obs~^2^ + Fh~calc~^2^ - Fph~obs~^2^)\n                              *  Fp~obs~ * cos(alphah~calc~)\n               A~iso~ =  -----------------------------------------------\n                                              E^2^\n\n               E            = (Fph~obs~ - Fp~obs~ - Fh~calc~)^2^\n                              for centric reflections\n                            = [(Fph~obs~ - Fp~obs~) * 2^1/2^ - Fh~calc~]^2^\n                              for acentric reflections\n\n               Fp~obs~      = the observed structure-factor amplitude of the\n                              native\n               Fph~obs~     = the observed structure-factor amplitude of the\n                              derivative\n               Fh~calc~     = the calculated structure-factor amplitude\n                              from the heavy-atom model\n               alphah~calc~ = the calculated phase from the heavy-atom model\n\n               This coefficient appears in the expression for the phase\n               probability of each isomorphous derivative:\n\n               P~i~(alpha) = exp[k + A * cos(alpha) + B * sin(alpha)\n                                   + C * cos(2 * alpha) + D * sin(2 * alpha)]\n\n               Ref: Hendrickson, W. A. & Lattman, E. E. (1970). Acta\n                    Cryst. B26, 136-143.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.HL_A_iso' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.HL_B_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The isomorphous Hendrickson-Lattman coefficient B~iso~ for this\n               reflection for this derivative.\n\n                         -2.0 * (Fp~obs~^2^ + Fh~calc~^2^ - Fph~obs~^2^)\n                              *  Fp~obs~ * sin(alphah~calc~)\n               B~iso~ =  -----------------------------------------------\n                                              E^2^\n\n               E            = (Fph~obs~ - Fp~obs~ - Fh~calc~)^2^\n                              for centric reflections\n                            = [(Fph~obs~ - Fp~obs~) * 2^1/2^ - Fh~calc~]^2^\n                              for acentric reflections\n\n               Fp~obs~      = the observed structure-factor amplitude of the\n                              native\n               Fph~obs~     = the observed structure-factor amplitude of the\n                              derivative\n               Fh~calc~     = the calculated structure-factor amplitude\n                              from the heavy-atom model\n               alphah~calc~ = the phase calculated from the heavy-atom model\n\n               This coefficient appears in the expression for the phase\n               probability of each isomorphous derivative:\n\n               P~i~(alpha) = exp[k + A * cos(alpha) + B * sin(alpha)\n                                   + C * cos(2 * alpha) + D * sin(2 * alpha)]\n\n               Ref: Hendrickson, W. A. & Lattman, E. E. (1970). Acta\n                    Cryst. B26, 136-143.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.HL_B_iso' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.HL_C_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The isomorphous Hendrickson-Lattman coefficient C~iso~ for this\n               reflection for this derivative.\n\n                         -Fp~obs~^2^ * [sin(alphah~calc~)^2^\n                                     -  cos(alphah~calc~)^2^]\n               C~iso~ =  ------------------------------------\n                                        E^2^\n\n               E            = (Fph~obs~ - Fp~obs~ - Fh~calc~)^2^\n                              for centric reflections\n                            = [(Fph~obs~ - Fp~obs~) * 2^1/2^ - Fh~calc~]^2^\n                              for acentric reflections\n\n               Fp~obs~      = the observed structure-factor amplitude of the\n                              native\n               Fph~obs~     = the observed structure-factor amplitude of the\n                              derivative\n               Fh~calc~     = the calculated structure-factor amplitude\n                              from the heavy-atom model\n               alphah~calc~ = the phase calculated from the heavy-atom model\n\n               This coefficient appears in the expression for the phase\n               probability of each isomorphous derivative:\n\n               P~i~(alpha) = exp[k + A * cos(alpha) + B * sin(alpha)\n                                   + C * cos(2 * alpha) + D * sin(2 * alpha)]\n\n               Ref: Hendrickson, W. A. & Lattman, E. E. (1970). Acta\n                    Cryst. B26, 136-143.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.HL_C_iso' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.HL_D_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The isomorphous Hendrickson-Lattman coefficient D~iso~ for this\n               reflection for this derivative.\n\n                         -2.0 * Fp~obs~^2^ * sin(alphah~calc~)^2^\n                                           * cos(alphah~calc~)^2^\n               D~iso~ =  ----------------------------------------\n                                          E^2^\n\n               E            = (Fph~obs~ - Fp~obs~ - Fh~calc~)^2^\n                              for centric reflections\n                            = [(Fph~obs~ - Fp~obs~) * 2^1/2^ - Fh~calc~]^2^\n                              for acentric reflections\n\n               Fp~obs~      = the observed structure-factor amplitude of the\n                              native\n               Fph~obs~     = the observed structure-factor amplitude of the\n                              derivative\n               Fh~calc~     = the calculated structure-factor amplitude\n                              from the heavy-atom model\n               alphah~calc~ = the phase calculated from the heavy-atom model\n\n               This coefficient appears in the expression for the phase\n               probability of each isomorphous derivative:\n\n               P~i~(alpha) = exp[k + A * cos(alpha) + B * sin(alpha)\n                                   + C * cos(2 * alpha) + D * sin(2 * alpha)]\n\n               Ref: Hendrickson, W. A. & Lattman, E. E. (1970). Acta\n                    Cryst. B26, 136-143.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.HL_D_iso' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.index_h',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index h for this reflection for this derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.index_h' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_refln.index_k',
              '_phasing_MIR_der_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.index_k',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index k for this reflection for this derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.index_k' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_refln.index_h',
              '_phasing_MIR_der_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.index_l',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index l for this reflection for this derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.index_l' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_refln.index_h',
              '_phasing_MIR_der_refln.index_k' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.phase_calc',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of the structure-factor phase based on the\n               heavy-atom model for this derivative in degrees.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.phase_calc' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_refln.set_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_set.id in the\n               PHASING_SET category.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_refln.set_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '##########################' ],
      [ '@comment', '# PHASING_MIR_DER_SHELL ##' ],
      [ '@comment', '##########################' ],
      [ 'save_',
        'phasing_MIR_der_shell',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MIR_DER_SHELL category record\n               statistics, broken down into shells of resolution, for an MIR\n               phasing experiment.\n\n               This list may contain information from a number of different\n               derivatives; _phasing_MIR_der_shell.der_id indicates to which\n               derivative a given record corresponds. (A derivative in this\n               context does not necessarily equate with a data set; see the\n               definition of the PHASING_MIR_DER category for a\n               discussion of the meaning of derivative.)' ],
          [ 'item_', '_category.id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_phasing_MIR_der_shell.der_id',
              '_phasing_MIR_der_shell.d_res_low',
              '_phasing_MIR_der_shell.d_res_high' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Zanotti et al. [J. Biol. Chem.\n                (1993), 268, 10728-10738]\n                with addition of an arbitrary low-resolution limit.',
              '\n    loop_\n    _phasing_MIR_der_shell.der_id\n    _phasing_MIR_der_shell.d_res_low\n    _phasing_MIR_der_shell.d_res_high\n    _phasing_MIR_der_shell.ha_ampl\n    _phasing_MIR_der_shell.loc\n     KAu(CN)2  15.0  8.3   54   26\n     KAu(CN)2   8.3  6.4   54   20\n     KAu(CN)2   6.4  5.2   50   20\n     KAu(CN)2   5.2  4.4   44   23\n     KAu(CN)2   4.4  3.8   39   23\n     KAu(CN)2   3.8  3.4   33   21\n     KAu(CN)2   3.4  3.0   28   17\n     KAu(CN)2  15.0  3.0   38   21\n     K2HgI4    15.0  8.3  149   87\n     K2HgI4     8.3  6.4  121   73\n     K2HgI4     6.4  5.2   95   61\n     K2HgI4     5.2  4.4   80   60\n     K2HgI4     4.4  3.8   73   63\n     K2HgI4     3.8  3.4   68   57\n     K2HgI4     3.4  3.0   63   46\n     K2HgI4    15.0  3.0   79   58\n     K3IrCl6   15.0  8.3   33   27\n     K3IrCl6    8.3  6.4   40   23\n     K3IrCl6    6.4  5.2   31   22\n     K3IrCl6    5.2  4.4   27   23\n     K3IrCl6    4.4  3.8   22   23\n     K3IrCl6    3.8  3.4   19   20\n     K3IrCl6    3.4  3.0   16   20\n     K3IrCl6   15.0  3.0   23   21',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The lowest value  for the interplanar spacings for the\n               reflection data for this derivative in this shell. This is called\n               the highest resolution.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.d_res_high' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The highest value for the interplanar spacings for the\n               reflection data for this derivative in this shell. This is called\n               the lowest resolution.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.d_res_low' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.der_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_MIR_der.id in the\n               PHASING_MIR_DER category.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.der_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.fom',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the figure of merit m for reflections for this\n               derivative in this shell.\n\n                   int P~alpha~ exp(i*alpha) dalpha\n               m = --------------------------------\n                         int P~alpha~ dalpha\n\n               P~alpha~ = the probability that the phase angle alpha is correct\n\n               int is taken over the range alpha = 0 to 2 pi.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.fom' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.ha_ampl',
        [ [ 'item_',
            '_item_description.description',
            '              The mean heavy-atom amplitude for reflections for this\n               derivative in this shell.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.ha_ampl' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.loc',
        [ [ 'item_',
            '_item_description.description',
            '              The mean lack-of-closure error loc for reflections for this\n               derivative in this shell.\n\n               loc = sum|Fph~obs~ - Fph~calc~|\n\n               Fph~obs~  = the observed structure-factor amplitude of the\n                           derivative\n               Fph~calc~ = the calculated structure-factor amplitude of the\n                           derivative\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.loc' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.phase',
        [ [ 'item_',
            '_item_description.description',
            '              The mean of the phase values for reflections for this\n               derivative in this shell.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.phase' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.power',
        [ [ 'item_',
            '_item_description.description',
            '              The mean phasing power P for reflections for this derivative\n               in this shell.\n\n                          sum|Fh~calc~^2^|\n               P = (----------------------------)^1/2^\n                    sum|Fph~obs~ - Fph~calc~|^2^\n\n               Fph~obs~  = the observed structure-factor amplitude of the\n                           derivative\n               Fph~calc~ = the calculated structure-factor amplitude of the\n                           derivative\n               Fh~calc~  = the calculated structure-factor amplitude from the\n                           heavy-atom model\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.power' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.R_cullis',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R~cullis~ for centric reflections for this\n               derivative in this shell.\n\n                           sum| |Fph~obs~ +/- Fp~obs~| - Fh~calc~ |\n               R~cullis~ = ----------------------------------------\n                                   sum|Fph~obs~ - Fp~obs~|\n\n               Fp~obs~  = the observed structure-factor amplitude of the native\n               Fph~obs~ = the observed structure-factor amplitude of the\n                          derivative\n               Fh~calc~ = the calculated structure-factor amplitude from the\n                          heavy-atom model\n\n               sum is taken over the specified reflections\n\n               Ref: Cullis, A. F., Muirhead, H., Perutz, M. F., Rossmann, M. G.\n                    & North, A. C. T. (1961). Proc. R. Soc. London Ser. A,\n                    265, 15-38.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.R_cullis' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.R_kraut',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R~kraut~ for general reflections for this\n               derivative in this shell.\n\n                          sum|Fph~obs~ - Fph~calc~|\n               R~kraut~ = -------------------------\n                                sum|Fph~obs~|\n\n               Fph~obs~  = the observed structure-factor amplitude of the\n                           derivative\n               Fph~calc~ = the calculated structure-factor amplitude of the\n                           derivative\n\n               sum is taken over the specified reflections\n\n               Ref: Kraut, J., Sieker, L. C., High, D. F. & Freer, S. T.\n                    (1962). Proc. Natl Acad. Sci. USA, 48, 1417-1424.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.R_kraut' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.reflns',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections in this shell.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_shell.reflns' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# PHASING_MIR_DER_SITE ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'phasing_MIR_der_site',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MIR_DER_SITE category record details\n               about the heavy-atom sites in an MIR phasing experiment.\n\n               This list may contain information from a number of different\n               derivatives; _phasing_MIR_der_site.der_id indicates to which\n               derivative a given record corresponds. (A derivative in this\n               context does not necessarily equate with a data set; see the\n               definition of the PHASING_MIR_DER category for a\n               discussion of the meaning of derivative.)' ],
          [ 'item_', '_category.id', 'phasing_MIR_der_site' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_phasing_MIR_der_site.der_id', '_phasing_MIR_der_site.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Zanotti et al. [J. Biol. Chem.\n                (1993), 268, 10728-10738]\n                with occupancies converted from electrons to fractional.',
              '\n    loop_\n    _phasing_MIR_der_site.der_id\n    _phasing_MIR_der_site.id\n    _phasing_MIR_der_site.atom_type_symbol\n    _phasing_MIR_der_site.occupancy\n    _phasing_MIR_der_site.fract_x\n    _phasing_MIR_der_site.fract_y\n    _phasing_MIR_der_site.fract_z\n    _phasing_MIR_der_site.B_iso\n     KAu(CN)2  1  Au  0.40  0.082  0.266  0.615  33.0\n     KAu(CN)2  2  Au  0.03  0.607  0.217  0.816  25.9\n     KAu(CN)2  3  Au  0.02  0.263  0.782  0.906  15.7\n     K2HgI4    1  Hg  0.63  0.048  0.286  0.636  33.7\n     K2HgI4    2  Hg  0.34  0.913  0.768  0.889  36.7\n     K2HgI4    3  Hg  0.23  0.974  0.455  0.974  24.2\n     K2HgI4    4  Hg  0.28  0.903  0.836  0.859  14.7\n     K2HgI4    5  Hg  0.07  0.489  0.200  0.885   6.4\n     K2HgI4    6  Hg  0.07  0.162  0.799  0.889  32.9\n     K3IrCl6   1  Ir  0.26  0.209  0.739  0.758  40.8\n     K3IrCl6   2  Ir  0.05  0.279  0.613  0.752  24.9',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.atom_type_symbol',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _atom_type.symbol in the\n               ATOM_TYPE category.\n\n               The scattering factors referenced via this data item should be\n               those used in the refinement of the heavy-atom data; in some\n               cases this is the scattering factor for the single heavy\n               atom, in other cases these are the scattering factors for an\n               atomic cluster.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_site.atom_type_symbol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.B_iso',
        [ [ 'item_',
            '_item_description.description',
            '              Isotropic displacement parameter for this heavy-atom site in this\n               derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.B_iso' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.B_iso_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.B_iso_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_MIR_der_site.B_iso.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.B_iso_esd' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.B_iso' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.Cartn_x',
        [ [ 'item_',
            '_item_description.description',
            '              The x coordinate of this heavy-atom position in this derivative\n               specified as orthogonal angstroms. The orthogonal Cartesian axes\n               are related to the cell axes as specified by the description\n               given in _atom_sites.Cartn_transform_axes.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.Cartn_x' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.Cartn_y',
              '_phasing_MIR_der_site.Cartn_z' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.Cartn_x_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.Cartn_x_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_MIR_der_site.Cartn_x.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.Cartn_x_esd' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.Cartn_y_esd',
              '_phasing_MIR_der_site.Cartn_z_esd' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.Cartn_x' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.Cartn_y',
        [ [ 'item_',
            '_item_description.description',
            '              The y coordinate of this heavy-atom position in this derivative\n               specified as orthogonal angstroms. The orthogonal Cartesian axes\n               are related to the cell axes as specified by the description\n               given in _atom_sites.Cartn_transform_axes.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.Cartn_y' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.Cartn_x',
              '_phasing_MIR_der_site.Cartn_z' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.Cartn_y_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.Cartn_y_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_MIR_der_site.Cartn_y.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.Cartn_y_esd' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.Cartn_x_esd',
              '_phasing_MIR_der_site.Cartn_z_esd' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.Cartn_y' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.Cartn_z',
        [ [ 'item_',
            '_item_description.description',
            '              The z coordinate of this heavy-atom position in this derivative\n               specified as orthogonal angstroms. The orthogonal Cartesian axes\n               are related to the cell axes as specified by the description\n               given in _atom_sites.Cartn_transform_axes.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.Cartn_z' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.Cartn_x',
              '_phasing_MIR_der_site.Cartn_y' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.Cartn_z_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.Cartn_z_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_MIR_der_site.Cartn_z.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.Cartn_z_esd' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.Cartn_x_esd',
              '_phasing_MIR_der_site.Cartn_y_esd' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.Cartn_z' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.der_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_MIR_der.id in the\n               PHASING_MIR_DER category.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.der_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the derivative site.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.details' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'binds to His 117',
              'minor site obtained from difference Fourier',
              'same as site 2 in the K2HgI4 derivative' ] ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.fract_x',
        [ [ 'item_',
            '_item_description.description',
            '              The x coordinate of this heavy-atom position in this derivative\n               specified as a fraction of _cell.length_a.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.fract_x' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.fract_y',
              '_phasing_MIR_der_site.fract_z' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.fract_x_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.fract_x_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_MIR_der_site.fract_x.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.fract_x_esd' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.fract_y_esd',
              '_phasing_MIR_der_site.fract_z_esd' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.fract_x' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.fract_y',
        [ [ 'item_',
            '_item_description.description',
            '              The y coordinate of this heavy-atom position in this derivative\n               specified as a fraction of _cell.length_b.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.fract_y' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.fract_x',
              '_phasing_MIR_der_site.fract_z' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.fract_y_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.fract_y_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_MIR_der_site.fract_y.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.fract_y_esd' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.fract_x_esd',
              '_phasing_MIR_der_site.fract_z_esd' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.fract_y' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.fract_z',
        [ [ 'item_',
            '_item_description.description',
            '              The z coordinate of this heavy-atom position in this derivative\n               specified as a fraction of _cell.length_c.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.fract_z' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.fract_x',
              '_phasing_MIR_der_site.fract_y' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.fract_z_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.fract_z_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_MIR_der_site.fract_z.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.fract_z_esd' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_MIR_der_site.fract_x_esd',
              '_phasing_MIR_der_site.fract_y_esd' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.fract_z' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_sub_category.id', 'fractional_coordinate_esd' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _phasing_MIR_der_site.id must uniquely identify each\n               site in each derivative in the PHASING_MIR_DER_SITE list.\n\n               The atom identifiers need not be unique over all sites in all\n               derivatives; they need only be unique for each site in each\n               derivative.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.id' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.occupancy',
        [ [ 'item_',
            '_item_description.description',
            '              The fraction of the atom type present at this heavy-atom site\n               in a given derivative. The sum of the occupancies of all the\n               atom types at this site may not significantly exceed 1.0 unless\n               it is a dummy site.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.occupancy' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.occupancy_anom',
        [ [ 'item_',
            '_item_description.description',
            '              The relative anomalous occupancy of the atom type\n               present at this heavy-atom site in a given derivative.\n               This atom occupancy will probably be on an arbitrary scale.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.occupancy_anom' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_site.ebi_occupancy_anom' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.occupancy_anom_su' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.occupancy_anom_su',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation) of\n               _phasing_MIR_der_site.occupancy_anom.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_site.occupancy_anom_su' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_site.ebi_occupancy_anom_esd' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.occupancy_anom' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.occupancy_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The relative real isotropic occupancy of the atom type\n               present at this heavy-atom site in a given derivative.\n               This atom occupancy will probably be on an arbitrary scale.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der_site.occupancy_iso' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_site.ebi_occupancy_iso' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.occupancy_iso_su' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_site.occupancy_iso_su',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation) of\n               _phasing_MIR_der_site.occupancy_iso.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_site.occupancy_iso_su' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_site.ebi_occupancy_iso_esd' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_phasing_MIR_der_site.occupancy_iso' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '######################' ],
      [ '@comment', '# PHASING_MIR_SHELL ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'phasing_MIR_shell',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_MIR_SHELL category record statistics\n               for an isomorphous replacement phasing experiment.broken\n               down into shells of resolution.' ],
          [ 'item_', '_category.id', 'phasing_MIR_shell' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_phasing_MIR_shell.d_res_low',
              '_phasing_MIR_shell.d_res_high' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by Zanotti et al. [J. Biol. Chem.\n                (1993), 268, 10728-10738]\n                with addition of an arbitrary low-resolution limit.',
              '\n    loop_\n    _phasing_MIR_shell.d_res_low\n    _phasing_MIR_shell.d_res_high\n    _phasing_MIR_shell.reflns\n    _phasing_MIR_shell.FOM\n     15.0  8.3   80  0.69\n      8.3  6.4  184  0.73\n      6.4  5.2  288  0.72\n      5.2  4.4  406  0.65\n      4.4  3.8  554  0.54\n      3.8  3.4  730  0.53\n      3.4  3.0  939  0.50',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The lowest value for the interplanar spacings for the\n               reflection data in this shell. This is called the highest\n               resolution. Note that the resolution limits of shells in\n               the items _phasing_MIR_shell.d_res_high and\n               _phasing_MIR_shell.d_res_low are independent of the resolution\n               limits of shells in the items _reflns_shell.d_res_high and\n               _reflns_shell.d_res_low.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.d_res_high' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The highest value for the interplanar spacings for the\n               reflection data in this shell. This is called the lowest\n               resolution. Note that the resolution limits of shells in the\n               items _phasing_MIR_shell.d_res_high and\n               _phasing_MIR_shell.d_res_low are independent of the resolution\n               limits of shells in the items _reflns_shell.d_res_high and\n               _reflns_shell.d_res_low.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.d_res_low' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.FOM',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the figure of merit m for reflections in this\n               shell.\n\n                   int P~alpha~ exp(i*alpha) dalpha\n               m = --------------------------------\n                         int P~alpha~ dalpha\n\n               P~alpha~ = the probability that the phase angle alpha is correct\n\n               the integral is taken over the range alpha = 0 to 2 pi.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.FOM' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.FOM_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the figure of merit m for acentric reflections\n               in this shell.\n\n                   int P~alpha~ exp(i*alpha) dalpha\n               m = --------------------------------\n                         int P~alpha~ dalpha\n\n               P~a~ = the probability that the phase angle a is correct\n\n               the integral is taken over the range alpha = 0 to 2 pi.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.FOM_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.ebi_fom_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.FOM_centric',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the figure of merit m for centric reflections\n               in this shell.\n\n                   int P~alpha~ exp(i*alpha) dalpha\n               m = --------------------------------\n                         int P~alpha~ dalpha\n\n               P~a~ = the probability that the phase angle a is correct\n\n               the integral is taken over the range alpha = 0 to 2 pi.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.FOM_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.ebi_fom_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.loc',
        [ [ 'item_',
            '_item_description.description',
            '              The mean lack-of-closure error loc for reflections in this shell.\n\n               loc = sum|Fph~obs~ - Fph~calc~|\n\n               Fph~obs~  = the observed structure-factor amplitude of the\n                           derivative\n               Fph~calc~ = the calculated structure-factor amplitude of the\n                           derivative\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.loc' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.mean_phase',
        [ [ 'item_',
            '_item_description.description',
            '              The mean of the phase values for all reflections in this shell.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.mean_phase' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.power',
        [ [ 'item_',
            '_item_description.description',
            '              The mean phasing power P for reflections in this shell.\n\n                          sum|Fh~calc~^2^|\n               P = (----------------------------)^1/2^\n                    sum|Fph~obs~ - Fph~calc~|^2^\n\n               Fph~obs~  = the observed structure-factor amplitude of the\n                           derivative\n               Fph~calc~ = the calculated structure-factor amplitude of the\n                           derivative\n               Fh~calc~  = the calculated structure-factor amplitude from the\n                           heavy-atom model\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.power' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.R_cullis',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R~cullis~ for centric reflections in this shell.\n\n                           sum| |Fph~obs~ +/- Fp~obs~| - Fh~calc~ |\n               R~cullis~ = ----------------------------------------\n                                   sum|Fph~obs~ - Fp~obs~|\n\n               Fp~obs~  = the observed structure-factor amplitude of the native\n               Fph~obs~ = the observed structure-factor amplitude of the\n                          derivative\n               Fh~calc~ = the calculated structure-factor amplitude from the\n                          heavy-atom model\n\n               sum is taken over the specified reflections\n\n               Ref: Cullis, A. F., Muirhead, H., Perutz, M. F., Rossmann, M. G.\n                    & North, A. C. T. (1961). Proc. R. Soc. London Ser. A,\n                    265, 15-38.\n' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.R_cullis' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.R_kraut',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R~kraut~ for general reflections in this shell.\n\n                          sum|Fph~obs~ - Fph~calc~|\n               R~kraut~ = -------------------------\n                                sum|Fph~obs~|\n\n               Fph~obs~  = the observed structure-factor amplitude of the\n                           derivative\n               Fph~calc~ = the calculated structure-factor amplitude of the\n                           derivative\n\n               sum is taken over the specified reflections\n\n               Ref: Kraut, J., Sieker, L. C., High, D. F. & Freer, S. T.\n                    (1962). Proc. Natl Acad. Sci. USA, 48, 1417-1424.\n' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.R_kraut' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.reflns',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections in this shell.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.reflns' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.reflns_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              The number of acentric reflections in this shell.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.reflns_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.ebi_reflns_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.reflns_anomalous',
        [ [ 'item_',
            '_item_description.description',
            '              The number of anomalous reflections in this shell.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.reflns_anomalous' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment',
            '   _item_aliases.alias_name    \'_phasing_MIR_shell.ebi_reflns_anomalous\'' ],
          [ '@comment',
            '   _item_aliases.dictionary      ebi_extensions' ],
          [ '@comment', '   _item_aliases.version         1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.reflns_centric',
        [ [ 'item_',
            '_item_description.description',
            '              The number of centric reflections in this shell.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.reflns_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.ebi_reflns_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# PHASING_SET ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'phasing_set',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_SET category record details about\n               the data sets used in a phasing experiment. A given data set\n               may be used in a number of different ways; for instance, a\n               single data set could be used both as an isomorphous derivative\n               and as a component of a multiple-wavelength calculation. This\n               category establishes identifiers for each data set and permits\n               the archiving of a subset of experimental information for each\n               data set (cell constants, wavelength, temperature etc.).\n\n               This and related categories of data items are provided so that\n               derivative intensity and phase information can be stored in\n               the same data block as the information for the refined\n               structure.\n\n               If all the possible experimental information for each data\n               set (raw data sets, crystal growth conditions etc.) is to be\n               archived, these data items should be recorded in a separate\n               data block.' ],
          [ 'item_', '_category.id', 'phasing_set' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_phasing_set.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for an Hg/Pt derivative of\n                protein NS1.',
              '\n    _phasing_set.id                    \'NS1-96\'\n    _phasing_set.cell_angle_alpha      90.0\n    _phasing_set.cell_angle_beta       90.0\n    _phasing_set.cell_angle_gamma      90.0\n    _phasing_set.cell_length_a         38.63\n    _phasing_set.cell_length_b         38.63\n    _phasing_set.cell_length_c         82.88\n    _phasing_set.radiation_wavelength  1.5145\n    _phasing_set.detector_type         \'image plate\'\n    _phasing_set.detector_specific     \'RXII\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_set.cell_angle_alpha',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell angle alpha for this data set in degrees.' ],
          [ 'item_', '_item.name', '_phasing_set.cell_angle_alpha' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '90.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_set.cell_angle_beta',
              '_phasing_set.cell_angle_gamma' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_sub_category.id', 'cell_angle' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_phasing_set.cell_angle_beta',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell angle beta for this data set in degrees.' ],
          [ 'item_', '_item.name', '_phasing_set.cell_angle_beta' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '90.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_set.cell_angle_alpha',
              '_phasing_set.cell_angle_gamma' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_sub_category.id', 'cell_angle' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_phasing_set.cell_angle_gamma',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell angle gamma for this data set in degrees.' ],
          [ 'item_', '_item.name', '_phasing_set.cell_angle_gamma' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '90.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_set.cell_angle_alpha',
              '_phasing_set.cell_angle_beta' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '180.0', '180.0', '180.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_sub_category.id', 'cell_angle' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_phasing_set.cell_length_a',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell length a for this data set in angstroms.' ],
          [ 'item_', '_item.name', '_phasing_set.cell_length_a' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_set.cell_length_b', '_phasing_set.cell_length_c' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_sub_category.id', 'cell_length' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_set.cell_length_b',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell length b for this data set in angstroms.' ],
          [ 'item_', '_item.name', '_phasing_set.cell_length_b' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_set.cell_length_a', '_phasing_set.cell_length_c' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_sub_category.id', 'cell_length' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_set.cell_length_c',
        [ [ 'item_',
            '_item_description.description',
            '              Unit-cell length c for this data set in angstroms.' ],
          [ 'item_', '_item.name', '_phasing_set.cell_length_c' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_set.cell_length_a', '_phasing_set.cell_length_b' ] ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_sub_category.id', 'cell_length' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_set.detector_specific',
        [ [ 'item_',
            '_item_description.description',
            '              The particular radiation detector. In general, this will be a\n               manufacturer, description, model number or some combination of\n               these.' ],
          [ 'item_', '_item.name', '_phasing_set.detector_specific' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Siemens model x', 'Kodak XG', 'MAR Research model y' ] ] ] ],
      [ 'save_',
        '_phasing_set.detector_type',
        [ [ 'item_',
            '_item_description.description',
            '              The general class of the radiation detector.' ],
          [ 'item_', '_item.name', '_phasing_set.detector_type' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'multiwire', 'imaging plate', 'CCD', 'film' ] ] ] ],
      [ 'save_',
        '_phasing_set.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _phasing_set.id must uniquely identify\n               a record in the PHASING_SET list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_phasing_set.id',
              'phasing_set',
              'yes',
              '_phasing_set_refln.set_id',
              'phasing_set_refln',
              'yes',
              '_phasing_MAD_set.set_id',
              'phasing_MAD_set',
              'yes',
              '_phasing_MIR_der.der_set_id',
              'phasing_MIR_der',
              'yes',
              '_phasing_MIR_der.native_set_id',
              'phasing_MIR_der',
              'yes',
              '_phasing_MIR_der_refln.set_id',
              'phasing_MIR_der_refln',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_phasing_set_refln.set_id',
              '_phasing_set.id',
              '_phasing_MAD_set.set_id',
              '_phasing_set.id',
              '_phasing_MIR_der.der_set_id',
              '_phasing_set.id',
              '_phasing_MIR_der.native_set_id',
              '_phasing_set.id',
              '_phasing_MIR_der_refln.set_id',
              '_phasing_set.id' ] ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'KAu(CN)2', 'K2HgI4' ] ] ] ],
      [ 'save_',
        '_phasing_set.radiation_source_specific',
        [ [ 'item_',
            '_item_description.description',
            '              The particular source of radiation. In general, this will be a\n               manufacturer, description, or model number (or some combination\n               of these) for laboratory sources and an institution name and\n               beamline name for synchrotron sources.' ],
          [ 'item_',
            '_item.name',
            '_phasing_set.radiation_source_specific' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Rigaku RU200', 'Philips fine focus Mo', 'NSLS beamline X8C' ] ] ] ],
      [ 'save_',
        '_phasing_set.radiation_wavelength',
        [ [ 'item_',
            '_item_description.description',
            '              The mean wavelength of the radiation used to measure this\n               data set.' ],
          [ 'item_', '_item.name', '_phasing_set.radiation_wavelength' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_set.temp',
        [ [ 'item_',
            '_item_description.description',
            '              The temperature in kelvins at which the data set was\n               measured.' ],
          [ 'item_', '_item.name', '_phasing_set.temp' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ '@comment', '######################' ],
      [ '@comment', '# PHASING_SET_REFLN ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'phasing_set_refln',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PHASING_SET_REFLN category record the values\n               of the measured structure factors used in a phasing experiment.\n               This list may contain information from a number of different\n               data sets; _phasing_set_refln.set_id indicates the data set\n               to which a given record corresponds.' ],
          [ 'item_', '_category.id', 'phasing_set_refln' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_phasing_set_refln.index_h',
              '_phasing_set_refln.index_k',
              '_phasing_set_refln.index_l',
              '_phasing_set_refln.set_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for the 15,15,32\n                reflection of an Hg/Pt derivative of protein NS1.',
              '\n    _phasing_set_refln.set_id           \'NS1-96\'\n    _phasing_set_refln.index_h           15\n    _phasing_set_refln.index_k           15\n    _phasing_set_refln.index_l           32\n    _phasing_set_refln.F_meas_au        181.79\n    _phasing_set_refln.F_meas_sigma_au    3.72',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_phasing_set_refln.set_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _phasing_set.id in the\n               PHASING_SET category.' ],
          [ 'item_', '_item.name', '_phasing_set_refln.set_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_phasing_set_refln.F_meas',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of the structure factor for this reflection\n               in this data set in electrons.' ],
          [ 'item_', '_item.name', '_phasing_set_refln.F_meas' ],
          [ 'item_', '_item.category_id', 'phasing_set_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_phasing_set_refln.F_meas_sigma',
              'associated_esd',
              '_phasing_set_refln.F_meas_au',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_phasing_set_refln.F_meas_au',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of the structure factor for this reflection\n               in this data set in arbitrary units.' ],
          [ 'item_', '_item.name', '_phasing_set_refln.F_meas_au' ],
          [ 'item_', '_item.category_id', 'phasing_set_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_phasing_set_refln.F_meas_sigma_au',
              'associated_esd',
              '_phasing_set_refln.F_meas',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_phasing_set_refln.F_meas_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_set_refln.F_meas in electrons.' ],
          [ 'item_', '_item.name', '_phasing_set_refln.F_meas_sigma' ],
          [ 'item_', '_item.category_id', 'phasing_set_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_phasing_set_refln.F_meas',
              'associated_value',
              '_phasing_set_refln.F_meas_sigma_au',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_phasing_set_refln.F_meas_sigma_au',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _phasing_set_refln.F_meas_au in arbitrary units.' ],
          [ 'item_', '_item.name', '_phasing_set_refln.F_meas_sigma_au' ],
          [ 'item_', '_item.category_id', 'phasing_set_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_phasing_set_refln.F_meas_au',
              'associated_value',
              '_phasing_set_refln.F_meas_sigma',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_phasing_set_refln.index_h',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index h of this reflection in this data set.' ],
          [ 'item_', '_item.name', '_phasing_set_refln.index_h' ],
          [ 'item_', '_item.category_id', 'phasing_set_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_set_refln.index_k', '_phasing_set_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_set_refln.index_k',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index k of this reflection in this data set.' ],
          [ 'item_', '_item.name', '_phasing_set_refln.index_k' ],
          [ 'item_', '_item.category_id', 'phasing_set_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_set_refln.index_h', '_phasing_set_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_set_refln.index_l',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index l of this reflection in this data set.' ],
          [ 'item_', '_item.name', '_phasing_set_refln.index_l' ],
          [ 'item_', '_item.category_id', 'phasing_set_refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_phasing_set_refln.index_h', '_phasing_set_refln.index_k' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '#########' ],
      [ '@comment', '# PUBL ##' ],
      [ '@comment', '#########' ],
      [ 'save_',
        'publ',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PUBL category are used when submitting a\n               manuscript for publication.' ],
          [ 'item_', '_category.id', 'publ' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_publ.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'iucr_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '    _publ.section_title\n    ;  trans-3-Benzoyl-2-(tert-butyl)-4-(iso-butyl)-\n       1,3-oxazolidin-5-one\n    ;\n\n    _publ.section_abstract\n    ;  The oxazolidinone ring is a shallow envelope\n       conformation with the tert-butyl and iso-butyl groups\n       occupying trans-positions with respect to the ring.  The\n       angles at the N atom sum to 356.2\\%, indicating a very\n       small degree of pyramidalization at this atom.  This is\n       consistent with electron delocalization between the N\n       atom and the carbonyl centre [N-C=O = 1.374(3)\\%A].\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on C~31~H~48~N~4~O~4~, reported by Coleman, Patrick,\n                Andersen & Rettig [Acta Cryst. (1996), C52, 1525-1527].',
              '    _publ.section_title\n    ;  Hemiasterlin methyl ester\n    ;\n\n    _publ.section_title_footnote\n    ;  IUPAC name: methyl 2,5-dimethyl-4-{2-[3-methyl-\n       2-methylamino-3-(N-methylbenzo[b]pyrrol-\n       3-yl)butanamido]-3,3-dimethyl-N-methyl-\n       butanamido}-2-hexenoate.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_publ.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_publ.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_publ.contact_author',
        [ [ 'item_',
            '_item_description.description',
            '              The name and address of the author submitting the manuscript and\n               data block. This is the person contacted by the journal\n               editorial staff. It is preferable to use the separate data items\n               _publ.contact_author_name and _publ.contact_author_address.' ],
          [ 'item_', '_item.name', '_publ.contact_author' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_contact_author' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Professor George Ferguson\n                                  Department of Chemistry and Biochemistry\n                                  University of Guelph\n                                  Ontario\n                                  Canada\n                                  N1G 2W1' ] ] ],
      [ 'save_',
        '_publ.contact_author_address',
        [ [ 'item_',
            '_item_description.description',
            '              The address of the author submitting the manuscript and data\n               block. This is the person contacted by the journal editorial\n               staff.' ],
          [ 'item_', '_item.name', '_publ.contact_author_address' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_contact_author_address' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Department of Chemistry and Biochemistry\n                                  University of Guelph\n                                  Ontario\n                                  Canada\n                                  N1G 2W1' ] ] ],
      [ 'save_',
        '_publ.contact_author_email',
        [ [ 'item_',
            '_item_description.description',
            '              E-mail address in a form recognizable to international networks.\n               The format of e-mail addresses is given in Section 3.4, Address\n               Specification, of Internet Message Format, RFC 2822, P. Resnick\n               (Editor), Network Standards Group, April 2001.' ],
          [ 'item_', '_item.name', '_publ.contact_author_email' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_contact_author_email' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'name@host.domain.country', 'uur5@banjo.bitnet' ] ] ] ],
      [ 'save_',
        '_publ.contact_author_fax',
        [ [ 'item_',
            '_item_description.description',
            '              Facsimile telephone number of the author submitting the\n               manuscript and data block.\n\n               The recommended style starts with the international dialing\n               prefix, followed by the area code in parentheses, followed by the\n               local number with no spaces. The earlier convention of including\n               the international dialing prefix in parentheses is no longer\n               recommended.' ],
          [ 'item_', '_item.name', '_publ.contact_author_fax' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_contact_author_fax' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '12(34)9477330', '12()349477330' ] ] ] ],
      [ 'save_',
        '_publ.contact_author_name',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the author submitting the manuscript and data\n               block. This is the person contacted by the journal editorial\n               staff.' ],
          [ 'item_', '_item.name', '_publ.contact_author_name' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_contact_author_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Professor George Ferguson' ] ] ],
      [ 'save_',
        '_publ.contact_author_phone',
        [ [ 'item_',
            '_item_description.description',
            '              Telephone number of the author submitting the manuscript and\n               data block.\n\n               The recommended style starts with the international dialing\n               prefix, followed by the area code in parentheses, followed by the\n               local number and any extension number prefixed by \'x\',\n               with no spaces. The earlier convention of including\n               the international dialing prefix in parentheses is no longer\n               recommended.' ],
          [ 'item_', '_item.name', '_publ.contact_author_phone' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_contact_author_phone' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '12(34)9477330', '12()349477330', '12(34)9477330x5543' ] ] ] ],
      [ 'save_',
        '_publ.contact_letter',
        [ [ 'item_',
            '_item_description.description',
            '              A letter submitted to the journal editor by the contact author.' ],
          [ 'item_', '_item.name', '_publ.contact_letter' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_contact_letter' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.manuscript_creation',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the word-processor package and computer used to\n               create the word-processed manuscript stored as\n               _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.manuscript_creation' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_manuscript_creation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'Tex file created by FrameMaker on a Sun 3/280' ] ] ],
      [ 'save_',
        '_publ.manuscript_processed',
        [ [ 'item_',
            '_item_description.description',
            '              The full manuscript of a paper (excluding possibly the figures\n               and the tables) output in ASCII characters from a word processor.\n               Information about the generation of this data item must be\n               specified in the data item _publ.manuscript_creation.' ],
          [ 'item_', '_item.name', '_publ.manuscript_processed' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_manuscript_processed' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.manuscript_text',
        [ [ 'item_',
            '_item_description.description',
            '              The full manuscript of a paper (excluding figures and possibly\n               the tables) output as standard ASCII text.' ],
          [ 'item_', '_item.name', '_publ.manuscript_text' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_manuscript_text' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.requested_category',
        [ [ 'item_',
            '_item_description.description',
            '              The category of paper submitted. For submission to\n               Acta Crystallographica Section C or\n               Acta Crystallographica Section E, ONLY the codes indicated\n               for use with these journals should be used.' ],
          [ 'item_', '_item.name', '_publ.requested_category' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_requested_category' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', 'FA' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'FA',
              'Full article',
              'FI',
              'Full submission - inorganic (Acta C)',
              'FO',
              'Full submission - organic (Acta C)',
              'FM',
              'Full submission - metal-organic (Acta C)',
              'CI',
              'CIF-access paper - inorganic (Acta C) (no longer in use)',
              'CO',
              'CIF-access paper - organic (Acta C) (no longer in use)',
              'CM',
              'CIF-access paper - metal-organic (Acta C) (no longer in use)',
              'EI',
              'Electronic submission - inorganic (Acta E)',
              'EO',
              'Electronic submission - organic (Acta E)',
              'EM',
              'Electronic submission - metal-organic (Acta E)',
              'AD',
              'Addenda and Errata (Acta C, Acta E)',
              'SC',
              'Short communication' ] ] ] ],
      [ 'save_',
        '_publ.requested_coeditor_name',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the co-editor whom the authors would like to\n               handle the submitted manuscript.' ],
          [ 'item_', '_item.name', '_publ.requested_coeditor_name' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_requested_coeditor_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_publ.requested_journal',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the journal to which the manuscript is being\n               submitted.' ],
          [ 'item_', '_item.name', '_publ.requested_journal' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_requested_journal' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_publ.section_abstract',
        [ [ 'item_',
            '_item_description.description',
            '              The abstract section of a manuscript if the manuscript is\n               submitted in parts. As an alternative see _publ.manuscript_text\n               and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_abstract' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_section_abstract' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_acknowledgements',
        [ [ 'item_',
            '_item_description.description',
            '              The acknowledgements section of a manuscript if the manuscript is\n               submitted in parts. As an alternative see _publ.manuscript_text\n               and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_acknowledgements' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_acknowledgements' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_comment',
        [ [ 'item_',
            '_item_description.description',
            '              The comment section of a manuscript if the manuscript is\n               submitted in parts. As an alternative see _publ.manuscript_text\n               and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_comment' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_section_comment' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_discussion',
        [ [ 'item_',
            '_item_description.description',
            '              The discussion section of a manuscript if the manuscript is\n               submitted in parts. As an alternative see _publ.manuscript_text\n               and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_discussion' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_discussion' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_experimental',
        [ [ 'item_',
            '_item_description.description',
            '              The experimental section of a manuscript if the manuscript is\n               submitted in parts. As an alternative see _publ.manuscript_text\n               and _publ.manuscript_processed.\n               The _publ.section_exptl_prep, _publ.section_exptl_solution and\n               _publ.section_exptl_refinement items are preferred for\n               separating the chemical preparation, structure solution and\n               refinement aspects of the description of the experiment.' ],
          [ 'item_', '_item.name', '_publ.section_experimental' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_experimental' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_exptl_prep',
        [ [ 'item_',
            '_item_description.description',
            '              The experimental preparation section of a manuscript if the\n               manuscript is submitted in parts. As an alternative see\n               _publ.manuscript_text and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_exptl_prep' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_exptl_prep' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_exptl_refinement',
        [ [ 'item_',
            '_item_description.description',
            '              The experimental refinement section of a manuscript if the\n               manuscript is submitted in parts. As an alternative see\n               _publ.manuscript_text and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_exptl_refinement' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_exptl_refinement' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_exptl_solution',
        [ [ 'item_',
            '_item_description.description',
            '              The experimental solution section of a manuscript if the\n               manuscript is submitted in parts. As an alternative see\n               _publ.manuscript_text and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_exptl_solution' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_exptl_solution' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_figure_captions',
        [ [ 'item_',
            '_item_description.description',
            '              The figure captions section of a manuscript if the manuscript is\n               submitted in parts. As an alternative see _publ.manuscript_text\n               and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_figure_captions' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_figure_captions' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_introduction',
        [ [ 'item_',
            '_item_description.description',
            '              The introduction section of a manuscript if the manuscript is\n               submitted in parts. As an alternative see _publ.manuscript_text\n               and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_introduction' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_introduction' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_references',
        [ [ 'item_',
            '_item_description.description',
            '              The references section of a manuscript if the manuscript is\n               submitted in parts. As an alternative see _publ.manuscript_text\n               and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_references' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_references' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_synopsis',
        [ [ 'item_',
            '_item_description.description',
            '              The synopsis section of a manuscript if the manuscript is\n               submitted in parts. As an alternative see _publ.manuscript_text\n               and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_synopsis' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_section_synopsis' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_table_legends',
        [ [ 'item_',
            '_item_description.description',
            '              The table legends section of a manuscript if the manuscript\n               is submitted in parts. As an alternative see\n               _publ.manuscript_text and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_table_legends' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_table_legends' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_title',
        [ [ 'item_',
            '_item_description.description',
            '              The title of a manuscript if the manuscript is submitted in\n               parts. As an alternative see _publ.manuscript_text and\n               _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_title' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_section_title' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ.section_title_footnote',
        [ [ 'item_',
            '_item_description.description',
            '              The footnote to the title of a manuscript if the manuscript\n               is submitted in parts. As an alternative see\n               _publ.manuscript_text and _publ.manuscript_processed.' ],
          [ 'item_', '_item.name', '_publ.section_title_footnote' ],
          [ 'item_', '_item.category_id', 'publ' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_section_title_footnote' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# PUBL_AUTHOR ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'publ_author',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PUBL_AUTHOR category record details of\n               the authors of a manuscript submitted for publication.' ],
          [ 'item_', '_category.id', 'publ_author' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_publ_author.name' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'iucr_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set TOZ of Willis, Beckwith & Tozer\n                [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    loop_\n    _publ_author.name\n    _publ_author.address\n         \'Willis, Anthony C.\'\n    ;     Research School of Chemistry\n          Australian National University\n          GPO Box 4\n          Canberra, A.C.T.\n          Australia    2601\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_publ_author.address',
        [ [ 'item_',
            '_item_description.description',
            '              The address of a publication author. If there is more than one\n               author this is looped with _publ_author.name.' ],
          [ 'item_', '_item.name', '_publ_author.address' ],
          [ 'item_', '_item.category_id', 'publ_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_author_address' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Department\n                                  Institute\n                                  Street\n                                  City and postcode\n                                  COUNTRY' ] ] ],
      [ 'save_',
        '_publ_author.email',
        [ [ 'item_',
            '_item_description.description',
            '              The e-mail address of a publication author. If there is more\n               than one author, this will be looped with _publ_author.name.\n               The format of e-mail addresses is given in Section 3.4, Address\n               Specification, of  Internet Message Format, RFC 2822, P. Resnick\n               (Editor), Network Standards Group, April 2001.' ],
          [ 'item_', '_item.name', '_publ_author.email' ],
          [ 'item_', '_item.category_id', 'publ_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_author_email' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.3.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'name@host.domain.country', 'bm@iucr.org' ] ] ] ],
      [ 'save_',
        '_publ_author.footnote',
        [ [ 'item_',
            '_item_description.description',
            '              A footnote accompanying an author\'s name in the list of authors\n               of a paper. Typically indicates sabbatical address, additional\n               affiliations or date of decease.' ],
          [ 'item_', '_item.name', '_publ_author.footnote' ],
          [ 'item_', '_item.category_id', 'publ_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_author_footnote' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'On leave from U. Western Australia',
              'Also at Department of Biophysics' ] ] ] ],
      [ 'save_',
        '_publ_author.name',
        [ [ 'item_',
            '_item_description.description',
            '              The name of a publication author. If there are multiple authors\n               this will be looped with _publ_author.address. The family\n               name(s), followed by a comma and including any dynastic\n               components, precedes the first names or initials.' ],
          [ 'item_', '_item.name', '_publ_author.name' ],
          [ 'item_', '_item.category_id', 'publ_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_author_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Bleary, Percival R.',
              'O\'Neil, F.K.',
              'Van den Bossche, G.',
              'Yang, D.-L.',
              'Simonov, Yu.A' ] ] ] ],
      [ '@comment', '##############' ],
      [ '@comment', '# PUBL_BODY ##' ],
      [ '@comment', '##############' ],
      [ 'save_',
        'publ_body',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PUBL_BODY category permit the labelling of\n               different text sections within the body of a paper.\n               Note that these should not be used in a paper which has\n               a standard format with sections tagged by specific data names\n               (such as in Acta Crystallographica Section C). Typically,\n               each journal will supply a list of the specific items it\n               requires in its Notes for Authors.' ],
          [ 'item_', '_category.id', 'publ_body' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_publ_body.element', '_publ_body.label' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'iucr_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on a paper by R. Restori & D. Schwarzenbach\n                [Acta Cryst. (1996), A52, 369-378].',
              '\n    loop_\n    _publ_body.element\n    _publ_body.label\n    _publ_body.title\n    _publ_body.format\n    _publ_body.contents\n\n         section   1         Introduction                    cif\n    ; X-ray diffraction from a crystalline material provides\n      information on the thermally and spatially averaged\n      electron density in the crystal...\n    ;\n         section   2         Theory                          tex\n    ; In the rigid-atom approximation, the dynamic electron\n      density of an atom is described by the convolution\n      product of the static atomic density and a probability\n      density function,\n      $\\rho_{dyn}(\\bf r) = \\rho_{stat}(\\bf r) * P(\\bf r). \\eqno(1)$\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on a paper by R. J. Papoular, Y. Vekhter & P. Coppens\n                [Acta Cryst. (1996), A52, 397-407].',
              '\n    loop_\n    _publ_body.element\n    _publ_body.label\n    _publ_body.title\n    _publ_body.contents\n\n         section     3\n    ; The two-channel method for retrieval of the deformation\n      electron density\n    ;\n         .\n         subsection  3.1  \'The two-channel entropy S[\\D\\r(r)]\'\n    ; As the wide dynamic range involved in the total electron\n      density...\n    ;\n         subsection  3.2\n    \'Uniform vs informative prior model densities\'        .\n         subsubsection  3.2.1  \'Use of uniform models\'\n    ; Straightforward algebra leads to expressions analogous\n      to...\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_publ_body.contents',
        [ [ 'item_',
            '_item_description.description',
            '              A text section of a paper.' ],
          [ 'item_', '_item.name', '_publ_body.contents' ],
          [ 'item_', '_item.category_id', 'publ_body' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_body_contents' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_publ_body.element',
        [ [ 'item_',
            '_item_description.description',
            '              The functional role of the associated text section.' ],
          [ 'item_', '_item.name', '_publ_body.element' ],
          [ 'item_', '_item.category_id', 'publ_body' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_body_element' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'section',
              'subsection',
              'subsubsection',
              'appendix',
              'footnote' ] ] ] ],
      [ 'save_',
        '_publ_body.format',
        [ [ 'item_',
            '_item_description.description',
            '              Code indicating the appropriate typesetting conventions\n               for accented characters and special symbols in the text\n               section.' ],
          [ 'item_', '_item.name', '_publ_body.format' ],
          [ 'item_', '_item.category_id', 'publ_body' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_body_format' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'ascii',
              'no coding for special symbols',
              'cif',
              'CIF convention',
              'latex',
              'LaTeX',
              'sgml',
              'SGML (ISO 8879)',
              'tex',
              'TeX',
              'troff',
              'troff or nroff' ] ] ] ],
      [ 'save_',
        '_publ_body.label',
        [ [ 'item_',
            '_item_description.description',
            '              Code identifying the section of text.' ],
          [ 'item_', '_item.name', '_publ_body.label' ],
          [ 'item_', '_item.category_id', 'publ_body' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_body_label' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '1.1', '2.1.3' ] ] ] ],
      [ 'save_',
        '_publ_body.title',
        [ [ 'item_',
            '_item_description.description',
            '              Title of the associated section of text.' ],
          [ 'item_', '_item.name', '_publ_body.title' ],
          [ 'item_', '_item.category_id', 'publ_body' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_publ_body_title' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# PUBL_MANUSCRIPT_INCL ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'publ_manuscript_incl',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PUBL_MANUSCRIPT_INCL category allow\n               the authors of a manuscript submitted for publication to list\n               data names that should be added to the standard request list\n               used by the journal printing software.' ],
          [ 'item_', '_category.id', 'publ_manuscript_incl' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_publ_manuscript_incl.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'iucr_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - hypothetical example.',
              '\n\n    loop_\n    _publ_manuscript_incl.entry_id    \n    _publ_manuscript_incl.extra_item\n    _publ_manuscript_incl.extra_info\n    _publ_manuscript_incl.extra_defn\n   \'EXAMHYPO\' \'_atom_site.symmetry_multiplicity\'\n                         \'to emphasise special sites\'      yes\n   \'EXAMHYPO\' \'_chemical.compound_source\'\n                         \'rare material, unusual source\'   yes\n   \'EXAMHYPO\' \'_reflns.d_resolution_high\'\n                         \'limited data is a problem here\'  yes\n   \'EXAMHYPO\' \'_crystal.magnetic_permeability\'\n                         \'unusual value for this material\'  no',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_publ_manuscript_incl.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_publ_manuscript_incl.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_publ_manuscript_incl.extra_defn',
        [ [ 'item_',
            '_item_description.description',
            '              Flags whether the corresponding data item marked for inclusion\n               in a journal request list is a standard CIF definition or not.' ],
          [ 'item_', '_item.name', '_publ_manuscript_incl.extra_defn' ],
          [ 'item_', '_item.category_id', 'publ_manuscript_incl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_manuscript_incl_extra_defn' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'not a standard CIF data name',
              'n',
              'abbreviation for "no"',
              'yes',
              'a standard CIF data name',
              'y',
              'abbreviation for "yes"' ] ] ] ],
      [ 'save_',
        '_publ_manuscript_incl.extra_info',
        [ [ 'item_',
            '_item_description.description',
            '              A short note indicating the reason why the author wishes the\n               corresponding data item marked for inclusion in the journal\n               request list to be published.' ],
          [ 'item_', '_item.name', '_publ_manuscript_incl.extra_info' ],
          [ 'item_', '_item.category_id', 'publ_manuscript_incl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_manuscript_incl_extra_info' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'to emphasise very special sites',
              'rare material from unusual source',
              'the limited data is a problem here',
              'a new data quantity needed here' ] ] ] ],
      [ 'save_',
        '_publ_manuscript_incl.extra_item',
        [ [ 'item_',
            '_item_description.description',
            '              Specifies the inclusion of specific data into a manuscript\n               which are not normally requested by the journal. The values\n               of this item are the extra data names (which MUST be enclosed\n               in single quotes) that will be added to the journal request list.' ],
          [ 'item_', '_item.name', '_publ_manuscript_incl.extra_item' ],
          [ 'item_', '_item.category_id', 'publ_manuscript_incl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_publ_manuscript_incl_extra_item' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '_atom_site.symmetry_multiplicity',
              '_chemical.compound_source',
              '_reflns.d_resolution_high',
              '_crystal.magnetic_permeability' ] ] ] ],
      [ '@comment', '###########' ],
      [ '@comment', '# REFINE ##' ],
      [ '@comment', '###########' ],
      [ 'save_',
        'refine',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE category record details about the\n               structure-refinement parameters.' ],
          [ 'item_', '_category.id', 'refine' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_refine.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _refine.entry_id                       \'5HVP\'\n    _refine.ls_number_reflns_obs           12901\n    _refine.ls_number_restraints           6609\n    _refine.ls_number_parameters           7032\n    _refine.ls_R_Factor_obs                0.176\n    _refine.ls_weighting_scheme            calc\n    _refine.ls_weighting_details\n    ; Sigdel model of Konnert-Hendrickson:\n      Sigdel: Afsig +  Bfsig*(sin(theta)/lambda-1/6)\n      Afsig = 22.0, Bfsig = -150.0 at beginning of refinement\n      Afsig = 15.5, Bfsig =  -50.0 at end of refinement\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                 [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    _refine.details              \'sfls:_F_calc_weight_full_matrix\'\n    _refine.ls_structure_factor_coef   F\n    _refine.ls_matrix_type             full\n    _refine.ls_weighting_scheme   calc\n    _refine.ls_weighting_details  \'w=1/(\\s^2^(F)+0.0004F^2^)\'\n    _refine.ls_hydrogen_treatment     \'refxyz except H332B noref\'\n    _refine.ls_extinction_method       Zachariasen\n    _refine.ls_extinction_coef         3514\n    _refine.ls_extinction_expression\n    ; Larson, A. C. (1970). "Crystallographic Computing", edited\n      by F. R. Ahmed. Eq. (22) p. 292. Copenhagen: Munksgaard.\n    ;\n    _refine.ls_abs_structure_details\n    ; The absolute configuration was assigned to agree with the\n      known chirality at C3 arising from its precursor l-leucine.\n    ;\n    _refine.ls_abs_structure_Flack     0\n    _refine.ls_number_reflns_obs       1408\n    _refine.ls_number_parameters       272\n    _refine.ls_number_restraints       0\n    _refine.ls_number_constraints      0\n    _refine.ls_R_factor_all               .038\n    _refine.ls_R_factor_obs               .034\n    _refine.ls_wR_factor_all              .044\n    _refine.ls_wR_factor_obs              .042\n    _refine.ls_goodness_of_fit_all       1.462\n    _refine.ls_goodness_of_fit_obs       1.515\n    _refine.ls_shift_over_esd_max         .535\n    _refine.ls_shift_over_esd_mean        .044\n    _refine.diff_density_min              -.108\n    _refine.diff_density_max              .131',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine.aniso_B[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the matrix that defines the overall\n               anisotropic displacement model if one was refined for this\n               structure.' ],
          [ 'item_', '_item.name', '_refine.aniso_B[1][1]' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_refine.aniso_B[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the matrix that defines the overall\n               anisotropic displacement model if one was refined for this\n               structure.' ],
          [ 'item_', '_item.name', '_refine.aniso_B[1][2]' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_refine.aniso_B[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the matrix that defines the overall\n               anisotropic displacement model if one was refined for this\n               structure.' ],
          [ 'item_', '_item.name', '_refine.aniso_B[1][3]' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_refine.aniso_B[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the matrix that defines the overall\n               anisotropic displacement model if one was refined for this\n               structure.' ],
          [ 'item_', '_item.name', '_refine.aniso_B[2][2]' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_refine.aniso_B[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the matrix that defines the overall\n               anisotropic displacement model if one was refined for this\n               structure.' ],
          [ 'item_', '_item.name', '_refine.aniso_B[2][3]' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_refine.aniso_B[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the matrix that defines the overall\n               anisotropic displacement model if one was refined for this\n               structure.' ],
          [ 'item_', '_item.name', '_refine.aniso_B[3][3]' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_refine.B_iso_max',
        [ [ 'item_',
            '_item_description.description',
            '              The maximum isotropic displacement parameter (B value)\n               found in the coordinate set.' ],
          [ 'item_', '_item.name', '_refine.B_iso_max' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_refine.B_iso_mean',
        [ [ 'item_',
            '_item_description.description',
            '              The mean isotropic displacement parameter (B value)\n               for the coordinate set.' ],
          [ 'item_', '_item.name', '_refine.B_iso_mean' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_refine.B_iso_min',
        [ [ 'item_',
            '_item_description.description',
            '              The minimum isotropic displacement parameter (B value)\n               found in the coordinate set.' ],
          [ 'item_', '_item.name', '_refine.B_iso_min' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_refine.correlation_coeff_Fo_to_Fc',
        [ [ 'item_',
            '_item_description.description',
            '              The correlation coefficient between the observed and\n               calculated structure factors for reflections included in\n               the refinement.\n\n               The correlation coefficient is scale-independent and gives\n               an idea of the quality of the refined model.\n\n                            sum~i~(Fo~i~ Fc~i~ - <Fo><Fc>)\n  R~corr~ = ------------------------------------------------------------\n            SQRT{sum~i~(Fo~i~)^2^-<Fo>^2^} SQRT{sum~i~(Fc~i~)^2^-<Fc>^2^}\n\n               Fo = observed structure factors\n               Fc = calculated structure factors\n               <>   denotes average value\n\n               summation is over reflections included in the refinement' ],
          [ 'item_', '_item.name', '_refine.correlation_coeff_Fo_to_Fc' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_Correlation_coeff_Fo_to_Fc' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.correlation_coeff_Fo_to_Fc_free',
        [ [ 'item_',
            '_item_description.description',
            '              The correlation coefficient between the observed and\n               calculated structure factors for reflections not included\n               in the refinement (free reflections).\n\n                The correlation coefficient is scale-independent and gives\n                an idea of the quality of the refined model.\n\n                            sum~i~(Fo~i~ Fc~i~ - <Fo><Fc>)\n  R~corr~ = ------------------------------------------------------------\n            SQRT{sum~i~(Fo~i~)^2^-<Fo>^2^} SQRT{sum~i~(Fc~i~)^2^-<Fc>^2^}\n\n                Fo  = observed structure factors\n                Fc  = calculated structure factors\n                <>    denotes average value\n\n                summation is over reflections not included\n                in the refinement (free reflections)' ],
          [ 'item_',
            '_item.name',
            '_refine.correlation_coeff_Fo_to_Fc_free' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_Correlation_coeff_Fo_to_Fc_free' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.details',
        [ [ 'item_',
            '_item_description.description',
            '              Description of special aspects of the refinement process.' ],
          [ 'item_', '_item.name', '_refine.details' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_special_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_refine.diff_density_max',
        [ [ 'item_',
            '_item_description.description',
            '              The maximum value of the electron density in the final difference\n               Fourier map.' ],
          [ 'item_', '_item.name', '_refine.diff_density_max' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_diff_density_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.diff_density_max_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'electrons_per_angstroms_cubed' ] ] ],
      [ 'save_',
        '_refine.diff_density_max_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _refine.diff_density_max.' ],
          [ 'item_', '_item.name', '_refine.diff_density_max_esd' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.diff_density_max' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons_per_angstroms_cubed' ] ] ],
      [ 'save_',
        '_refine.diff_density_min',
        [ [ 'item_',
            '_item_description.description',
            '              The minimum value of the electron density in the final difference\n               Fourier map.' ],
          [ 'item_', '_item.name', '_refine.diff_density_min' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_diff_density_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.diff_density_min_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'electrons_per_angstroms_cubed' ] ] ],
      [ 'save_',
        '_refine.diff_density_min_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _refine.diff_density_min.' ],
          [ 'item_', '_item.name', '_refine.diff_density_min_esd' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.diff_density_min' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons_per_angstroms_cubed' ] ] ],
      [ 'save_',
        '_refine.diff_density_rms',
        [ [ 'item_',
            '_item_description.description',
            '              The root-mean-square-deviation of the electron density in the\n               final difference Fourier map. This value is measured with respect\n               to the arithmetic mean density and is derived from summations\n               over each grid point in the asymmetric unit of the cell. This\n               quantity is useful for assessing the significance of the values\n               of _refine.diff_density_min and _refine.diff_density_max, and\n               also for defining suitable contour levels.' ],
          [ 'item_', '_item.name', '_refine.diff_density_rms' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_diff_density_rms' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.diff_density_rms_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'electrons_per_angstroms_cubed' ] ] ],
      [ 'save_',
        '_refine.diff_density_rms_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _refine.diff_density_rms.' ],
          [ 'item_', '_item.name', '_refine.diff_density_rms_esd' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.diff_density_rms' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons_per_angstroms_cubed' ] ] ],
      [ 'save_',
        '_refine.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_refine.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_refine.ls_abs_structure_details',
        [ [ 'item_',
            '_item_description.description',
            '              The nature of the absolute structure and how it was determined.\n               For example, this may describe the Friedel pairs used.' ],
          [ 'item_', '_item.name', '_refine.ls_abs_structure_details' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_abs_structure_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_refine.ls_abs_structure_Flack',
        [ [ 'item_',
            '_item_description.description',
            '              The measure of absolute structure (enantiomorph or polarity) as\n               defined by Flack (1983).\n\n               For centrosymmetric structures, the only permitted value, if the\n               data name is present, is \'inapplicable\', represented by \'.\' .\n\n               For noncentrosymmetric structures the value must lie in the\n               99.97% Gaussian confidence interval  -3u =< x =< 1 + 3u and a\n               standard uncertainty (estimated standard deviation) u must\n               be supplied. The item range of [0.0:1.0] is correctly\n               interpreted as meaning (0.0 - 3u) =< x =< (1.0 + 3u).\n\n                  Ref: Flack, H. D. (1983). Acta Cryst. A39, 876-881.' ],
          [ 'item_', '_item.name', '_refine.ls_abs_structure_Flack' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_abs_structure_Flack' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '1.0', '1.0', '1.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_abs_structure_Flack_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_refine.ls_abs_structure_Flack_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _refine.ls_abs_structure_Flack.' ],
          [ 'item_', '_item.name', '_refine.ls_abs_structure_Flack_esd' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_abs_structure_Flack' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_abs_structure_Rogers',
        [ [ 'item_',
            '_item_description.description',
            '              The measure of absolute structure (enantiomorph or polarity) as\n               defined by Rogers.\n\n\n               The value must lie in the 99.97% Gaussian confidence interval\n               -1 -3u =< \\h =< 1 + 3u and a standard uncertainty (estimated\n               standard deviation) u must be supplied. The item range of\n               [-1.0, 1.0] is correctly interpreted as meaning\n               (-1.0 - 3u) =< \\h =< (1.0 + 3u).\n\n               Ref: Rogers, D. (1981). Acta Cryst. A37, 734-741.' ],
          [ 'item_', '_item.name', '_refine.ls_abs_structure_Rogers' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_abs_structure_Rogers' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '1.0', '1.0', '1.0', '-1.0', '-1.0', '-1.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_abs_structure_Rogers_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_refine.ls_abs_structure_Rogers_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _refine.ls_abs_structure_Rogers.' ],
          [ 'item_', '_item.name', '_refine.ls_abs_structure_Rogers_esd' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_abs_structure_Rogers' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The smallest value for the interplanar spacings for the\n               reflection data used in the refinement in angstroms. This is\n               called the highest resolution.' ],
          [ 'item_', '_item.name', '_refine.ls_d_res_high' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_refine_ls_d_res_high' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine.ls_d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The largest value for the interplanar spacings for\n               the reflection data used in the refinement in angstroms.\n               This is called the lowest resolution.' ],
          [ 'item_', '_item.name', '_refine.ls_d_res_low' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_refine_ls_d_res_low' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine.ls_extinction_coef',
        [ [ 'item_',
            '_item_description.description',
            '              The extinction coefficient used to calculate the correction\n               factor applied to the structure-factor data. The nature of the\n               extinction coefficient is given in the definitions of\n               _refine.ls_extinction_expression and\n               _refine.ls_extinction_method.\n\n               For the \'Zachariasen\' method it is the r* value; for the\n               \'Becker-Coppens type 1 isotropic\' method it is the \'g\' value,\n               and for \'Becker-Coppens type 2 isotropic\' corrections it is\n               the \'rho\' value. Note that the magnitude of these values is\n               usually of the order of 10000.\n\n               Ref:  Becker, P. J. & Coppens, P. (1974). Acta Cryst. A30,\n                     129-47, 148-153.\n                     Zachariasen, W. H. (1967). Acta Cryst. 23, 558-564.\n                     Larson, A. C. (1967). Acta Cryst. 23, 664-665.' ],
          [ 'item_', '_item.name', '_refine.ls_extinction_coef' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_extinction_coef' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_extinction_coef_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_examples.case', '3472' ],
          [ 'item_',
            '_item_examples.detail',
            'Zachariasen coefficient r* = 0.347 E04' ] ] ],
      [ 'save_',
        '_refine.ls_extinction_coef_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _refine.ls_extinction_coef.' ],
          [ 'item_', '_item.name', '_refine.ls_extinction_coef_esd' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_extinction_coef' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_extinction_expression',
        [ [ 'item_',
            '_item_description.description',
            '              A description of or reference to the extinction-correction\n               equation used to apply the data item\n               _refine.ls_extinction_coef. This information must be sufficient\n               to reproduce the extinction-correction factors applied to the\n               structure factors.' ],
          [ 'item_', '_item.name', '_refine.ls_extinction_expression' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_extinction_expression' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '      Larson, A. C. (1970). "Crystallographic Computing", edited by\n       F. R. Ahmed. Eq. (22), p.292. Copenhagen: Munksgaard.' ] ] ],
      [ 'save_',
        '_refine.ls_extinction_method',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the extinction-correction method applied.\n               This description should\n               include information about the correction method, either\n               \'Becker-Coppens\' or \'Zachariasen\'.  The latter is sometimes\n               referred to as the \'Larson\' method even though it employs\n               Zachariasen\'s formula.\n\n               The Becker-Coppens procedure is referred to as \'type 1\' when\n               correcting secondary extinction dominated by the mosaic spread;\n               as \'type 2\' when secondary extinction is dominated by particle\n               size and includes a primary extinction component; and as \'mixed\'\n               when there is a mixture of types 1 and 2.\n\n               For the Becker-Coppens method, it is also necessary to set the\n               mosaic distribution as either \'Gaussian\' or \'Lorentzian\' and the\n               nature of the extinction as \'isotropic\' or \'anisotropic\'. Note\n               that if either the \'mixed\' or \'anisotropic\' corrections are\n               applied, the multiple coefficients cannot be contained in\n               *_extinction_coef and must be listed in _refine.details.\n\n               Ref: Becker, P. J. & Coppens, P. (1974). Acta Cryst. A30,\n                    129-147, 148-153.\n                    Zachariasen, W. H. (1967). Acta Cryst. 23, 558- 564.\n                    Larson, A. C. (1967). Acta Cryst. 23, 664-665.' ],
          [ 'item_', '_item.name', '_refine.ls_extinction_method' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_extinction_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           Zachariasen' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'B-C type 2 Gaussian isotropic' ] ] ],
      [ 'save_',
        '_refine.ls_goodness_of_fit_all',
        [ [ 'item_',
            '_item_description.description',
            '              The least-squares goodness-of-fit parameter S for all data\n               after the final cycle of refinement. Ideally, account should be\n               taken of parameters restrained in the least-squares refinement.\n               See also the definition of _refine.ls_restrained_S_all.\n\n                   ( sum|w |Y~obs~ - Y~calc~|^2^| )^1/2^\n               S = ( ---------------------------- )\n                   (       N~ref~ - N~param~      )\n\n               Y~obs~   = the observed coefficients\n                          (see _refine.ls_structure_factor_coef)\n               Y~calc~  = the calculated coefficients\n                          (see _refine.ls_structure_factor_coef)\n               w        = the least-squares reflection weight\n                          [1/(e.s.d. squared)]\n\n               N~ref~   = the number of reflections used in the refinement\n               N~param~ = the number of refined parameters\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_goodness_of_fit_all' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_goodness_of_fit_all' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_goodness_of_fit_all_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_refine.ls_goodness_of_fit_all_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _refine.ls_goodness_of_fit_all.' ],
          [ 'item_', '_item.name', '_refine.ls_goodness_of_fit_all_esd' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_goodness_of_fit_all' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_goodness_of_fit_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The least-squares goodness-of-fit parameter S for reflection data\n               classified as \'observed\' (see _reflns.observed_criterion) after\n               the final cycle of refinement. Ideally, account should be taken\n               of parameters restrained in the least-squares refinement.\n               See also the definition of _refine.ls_restrained_S_obs.\n\n                   ( sum|w |Y~obs~ - Y~calc~|^2^| )^1/2^\n               S = ( ---------------------------- )\n                   (       N~ref~ - N~param~      )\n\n               Y~obs~   = the observed coefficients\n                          (see _refine.ls_structure_factor_coef)\n               Y~calc~  = the calculated coefficients\n                          (see _refine.ls_structure_factor_coef)\n               w        = the least-squares reflection weight\n                          [1/(e.s.d. squared)]\n\n               N~ref~   = the number of reflections used in the refinement\n               N~param~ = the number of refined parameters\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_goodness_of_fit_obs' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_goodness_of_fit_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_goodness_of_fit_obs_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ] ] ],
      [ 'save_',
        '_refine.ls_goodness_of_fit_obs_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation)\n               of _refine.ls_goodness_of_fit_obs.' ],
          [ 'item_', '_item.name', '_refine.ls_goodness_of_fit_obs_esd' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_goodness_of_fit_obs' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_hydrogen_treatment',
        [ [ 'item_',
            '_item_description.description',
            '              Treatment of hydrogen atoms in the least-squares refinement.' ],
          [ 'item_', '_item.name', '_refine.ls_hydrogen_treatment' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_hydrogen_treatment' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           undef' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'refall',
              'refined all H-atom parameters',
              'refxyz',
              'refined H-atom coordinates only',
              'refU',
              'refined H-atom U\'s only',
              'noref',
              'no refinement of H-atom parameters',
              'constr',
              'H-atom parameters constrained',
              'mixed',
              'some constrained, some independent',
              'undef',
              'H-atom parameters not defined' ] ] ] ],
      [ 'save_',
        '_refine.ls_matrix_type',
        [ [ 'item_',
            '_item_description.description',
            '              Type of matrix used to accumulate the least-squares derivatives.' ],
          [ 'item_', '_item.name', '_refine.ls_matrix_type' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refine_ls_matrix_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           full' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'full',
              'full',
              'fullcycle',
              'full with fixed elements per cycle',
              'atomblock',
              'block diagonal per atom',
              'userblock',
              'user-defined blocks',
              'diagonal',
              'diagonal elements only',
              'sparse',
              'selected elements only' ] ] ] ],
      [ 'save_',
        '_refine.ls_number_constraints',
        [ [ 'item_',
            '_item_description.description',
            '              The number of constrained (non-refined or dependent) parameters\n               in the least-squares process. These may be due to symmetry or any\n               other constraint process (e.g. rigid-body refinement). See also\n               _atom_site.constraints and _atom_site.refinement_flags. A general\n               description of constraints may appear in _refine.details.' ],
          [ 'item_', '_item.name', '_refine.ls_number_constraints' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_number_constraints' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine.ls_number_parameters',
        [ [ 'item_',
            '_item_description.description',
            '              The number of parameters refined in the least-squares process.\n               If possible, this number should include some contribution from\n               the restrained parameters. The restrained parameters are\n               distinct from the constrained parameters (where one or more\n               parameters are linearly dependent on the refined value of\n               another). Least-squares restraints often depend on geometry or\n               energy considerations and this makes their direct contribution\n               to this number, and to the goodness-of-fit calculation,\n               difficult to assess.' ],
          [ 'item_', '_item.name', '_refine.ls_number_parameters' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_number_parameters' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine.ls_number_reflns_all',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine.ls_d_res_high and _refine.ls_d_res_low.' ],
          [ 'item_', '_item.name', '_refine.ls_number_reflns_all' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine.ls_number_reflns_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine.ls_d_res_high and _refine.ls_d_res_low\n               and the observation limit established by\n               _reflns.observed_criterion.' ],
          [ 'item_', '_item.name', '_refine.ls_number_reflns_obs' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_number_reflns' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine.ls_number_reflns_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine.ls_d_res_high and _refine.ls_d_res_low\n               and the observation limit established by\n               _reflns.observed_criterion, and that were used as the test\n               reflections (i.e. were excluded from the refinement) when the\n               refinement included the calculation of a \'free\' R factor.\n               Details of how reflections were assigned to the working and\n               test sets are given in _reflns.R_free_details.' ],
          [ 'item_', '_item.name', '_refine.ls_number_reflns_R_free' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine.ls_number_reflns_R_work',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine.ls_d_res_high and _refine.ls_d_res_low\n               and the observation limit established by\n               _reflns.observed_criterion, and that were used as the working\n               reflections (i.e. were included in the refinement) when the\n               refinement included the calculation of a \'free\' R factor.\n               Details of how reflections were assigned to the working and\n               test sets are given in _reflns.R_free_details.' ],
          [ 'item_', '_item.name', '_refine.ls_number_reflns_R_work' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine.ls_number_restraints',
        [ [ 'item_',
            '_item_description.description',
            '              The number of restrained parameters. These are parameters which\n               are not directly dependent on another refined parameter.\n               Restrained parameters often involve geometry or energy\n               dependencies.\n               See also _atom_site.constraints and _atom_site.refinement_flags.\n               A general description of refinement constraints may appear in\n               _refine.details.' ],
          [ 'item_', '_item.name', '_refine.ls_number_restraints' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_number_restraints' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine.ls_percent_reflns_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine.ls_d_res_high and _refine.ls_d_res_low\n               and the observation limit established by\n               _reflns.observed_criterion, expressed as a percentage of the\n               number of geometrically observable reflections that satisfy\n               the resolution limits.' ],
          [ 'item_', '_item.name', '_refine.ls_percent_reflns_obs' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_percent_reflns_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine.ls_d_res_high and _refine.ls_d_res_low\n               and the observation limit established by\n               _reflns.observed_criterion, and that were used as the test\n               reflections (i.e. were excluded from the refinement) when the\n               refinement included the calculation of a \'free\' R factor,\n               expressed as a percentage of the number of geometrically\n               observable reflections that satisfy the resolution limits.' ],
          [ 'item_', '_item.name', '_refine.ls_percent_reflns_R_free' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_R_factor_all',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for all reflections that satisfy the resolution\n               limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_R_factor_all' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_R_factor_all' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_wR_factor_all' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_R_factor_obs',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low and the observation limit established by\n               _reflns.observed_criterion.\n\n               _refine.ls_R_factor_obs should not be confused with\n               _refine.ls_R_factor_R_work; the former reports the results of a\n               refinement in which all observed reflections were used, the\n               latter a refinement in which a subset of the observed\n               reflections were excluded from refinement for the calculation\n               of a \'free\' R factor. However, it would be meaningful to quote\n               both values if a \'free\' R factor were calculated for most of\n               the refinement, but all of the observed reflections were used\n               in the final rounds of refinement; such a protocol should be\n               explained in _refine.details.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_R_factor_obs' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_R_factor_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_wR_factor_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_R_factor_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low and the observation limit established by\n               _reflns.observed_criterion, and that were used as the test\n               reflections (i.e. were excluded from the refinement) when the\n               refinement included the calculation of a \'free\' R factor.\n               Details of how reflections were assigned to the working and\n               test sets are given in _reflns.R_free_details.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_R_factor_R_free' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_refine.ls_wR_factor_R_free',
              'alternate',
              '_refine.ls_R_factor_R_free_error',
              'associated_error' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_R_factor_R_free_error',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated error in _refine.ls_R_factor_R_free.\n               The method used to estimate the error is described in the\n               item _refine.ls_R_factor_R_free_error_details.' ],
          [ 'item_', '_item.name', '_refine.ls_R_factor_R_free_error' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_R_factor_R_free' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_R_factor_R_free_error_details',
        [ [ 'item_',
            '_item_description.description',
            '              Special aspects of the method used to estimated the error in\n               _refine.ls_R_factor_R_free.' ],
          [ 'item_',
            '_item.name',
            '_refine.ls_R_factor_R_free_error_details' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_refine.ls_R_factor_R_work',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low and the observation limit established by\n               _reflns.observed_criterion, and that were used as the working\n               reflections (i.e. were included in the refinement)  when the\n               refinement included the calculation of a \'free\' R factor.\n               Details of how reflections were assigned to the working and\n               test sets are given in _reflns.R_free_details.\n\n               _refine.ls_R_factor_obs should not be confused with\n               _refine.ls_R_factor_R_work; the former reports the results of a\n               refinement in which all observed reflections were used, the\n               latter a refinement in which a subset of the observed\n               reflections were excluded from refinement for the calculation\n               of a \'free\' R factor. However, it would be meaningful to quote\n               both values if a \'free\' R factor were calculated for most of\n               the refinement, but all of the observed reflections were used\n               in the final rounds of refinement; such a protocol should be\n               explained in _refine.details.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_R_factor_R_work' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_wR_factor_R_work' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_R_Fsqd_factor_obs',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R(Fsqd) for reflections that satisfy the\n               resolution limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low and the observation limit established by\n               _reflns.observed_criterion, calculated on the squares of the\n               observed and calculated structure-factor amplitudes.\n\n                         sum|F~obs~^2^ - F~calc~^2^|\n               R(Fsqd) = ---------------------------\n                                sum|F~obs~^2^|\n\n               F~obs~^2^  = squares of the observed structure-factor amplitudes\n               F~calc~^2^ = squares of the calculated structure-factor\n                            amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_R_Fsqd_factor_obs' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_R_Fsqd_factor' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_R_I_factor_obs',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R(I) for reflections that satisfy the\n               resolution limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low and the observation limit established by\n               _reflns.observed_criterion, calculated on the estimated\n               reflection intensities.\n\n               This is most often calculated in Rietveld refinements against\n               powder data, where it is referred to as R~B~ or R~Bragg~.\n\n                      sum|I~obs~ - I~calc~|\n               R(I) = ---------------------\n                           sum|I~obs~|\n\n               I~obs~  = the net observed intensities\n               I~calc~ = the net calculated intensities\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_R_I_factor_obs' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refine_ls_R_I_factor' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_redundancy_reflns_all',
        [ [ 'item_',
            '_item_description.description',
            '              The ratio of the total number of observations of the\n               reflections that satisfy the resolution limits established by\n               _refine.ls_d_res_high and _refine.ls_d_res_low to the number\n               of crystallographically unique reflections that satisfy the\n               same limits.' ],
          [ 'item_', '_item.name', '_refine.ls_redundancy_reflns_all' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_redundancy_reflns_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The ratio of the total number of observations of the\n               reflections that satisfy the resolution limits established by\n               _refine.ls_d_res_high and _refine.ls_d_res_low and the\n               observation limit established by _reflns.observed_criterion to\n               the number of crystallographically unique reflections that\n               satisfy the same limits.' ],
          [ 'item_', '_item.name', '_refine.ls_redundancy_reflns_obs' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_restrained_S_all',
        [ [ 'item_',
            '_item_description.description',
            '              The least-squares goodness-of-fit parameter S\' for all\n               reflections after the final cycle of least-squares refinement.\n               This parameter explicitly includes the restraints applied\n               in the least-squares process. See also the definition of\n               _refine.ls_goodness_of_fit_all.\n\n                    (   sum   |w    |Y~obs~  - Y~calc~|^2^| )^1/2^\n                    ( + sum~r~|w~r~ |P~calc~ - P~targ~|^2^| )\n               S\' = ( ------------------------------------- )\n                    (      N~ref~ + N~restr~ - N~param~     )\n\n               Y~obs~   = the observed coefficients\n                          (see _refine.ls_structure_factor_coef)\n               Y~calc~  = the calculated coefficients\n                          (see _refine.ls_structure_factor_coef)\n               w        = the least-squares reflection weight\n                          [1/(e.s.d. squared)]\n\n               P~calc~  = the calculated restraint values\n               P~targ~  = the target restraint values\n               w~r~     = the restraint weight\n\n               N~ref~   = the number of reflections used in the refinement\n                          (see _refine.ls_number_reflns_obs)\n               N~restr~ = the number of restraints\n                          (see _refine.ls_number_restraints)\n               N~param~ = the number of refined parameters\n                          (see _refine.ls_number_parameters)\n\n               sum  is taken over the specified reflections\n               sumr is taken over the restraints' ],
          [ 'item_', '_item.name', '_refine.ls_restrained_S_all' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_restrained_S_all' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_restrained_S_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The least-squares goodness-of-fit parameter S\' for reflection\n               data classified as observed (see _reflns.observed_criterion)\n               after the final cycle of least-squares refinement. This\n               parameter explicitly includes the restraints applied in\n               the least-squares process. See also the definition of\n               _refine.ls_goodness_of_fit_obs.\n\n                    (   sum   |w    |Y~obs~  - Y~calc~|^2^| )^1/2^\n                    ( + sum~r~|w~r~ |P~calc~ - P~targ~|^2^| )\n               S\' = ( ------------------------------------- )\n                    (      N~ref~ + N~restr~ - N~param~     )\n\n               Y~obs~   = the observed coefficients\n                          (see _refine.ls_structure_factor_coef)\n               Y~calc~  = the calculated coefficients\n                          (see _refine.ls_structure_factor_coef)\n               w        = the least-squares reflection weight\n                          [1/(e.s.d. squared)]\n\n               P~calc~  = the calculated restraint values\n               P~targ~  = the target restraint values\n               w~r~     = the restraint weight\n\n               N~ref~   = the number of reflections used in the refinement\n                          (see _refine.ls_number_reflns_obs)\n               N~restr~ = the number of restraints\n                          (see _refine.ls_number_restraints)\n               N~param~ = the number of refined parameters\n                          (see _refine.ls_number_parameters)\n\n               sum  is taken over the specified reflections\n               sumr is taken over the restraints' ],
          [ 'item_', '_item.name', '_refine.ls_restrained_S_obs' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_restrained_S_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_shift_over_esd_max',
        [ [ 'item_',
            '_item_description.description',
            '              The largest ratio of the final least-squares parameter shift\n               to the final standard uncertainty (estimated standard\n               deviation).' ],
          [ 'item_', '_item.name', '_refine.ls_shift_over_esd_max' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_shift/esd_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_shift_over_esd_mean',
        [ [ 'item_',
            '_item_description.description',
            '              The average ratio of the final least-squares parameter shift\n               to the final standard uncertainty (estimated standard\n               deviation).' ],
          [ 'item_', '_item.name', '_refine.ls_shift_over_esd_mean' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_shift/esd_mean' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_structure_factor_coef',
        [ [ 'item_',
            '_item_description.description',
            '              Structure-factor coefficient |F|, F^2^ or I used in the least-\n               squares refinement process.' ],
          [ 'item_', '_item.name', '_refine.ls_structure_factor_coef' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_structure_factor_coef' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           F' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'F',
              'structure-factor magnitude',
              'Fsqd',
              'structure factor squared',
              'Inet',
              'net intensity' ] ] ] ],
      [ 'save_',
        '_refine.ls_weighting_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the weighting scheme used\n               in least-squares refinement. Used to describe the weighting\n               when the value of _refine.ls_weighting_scheme is specified\n               as \'calc\'.' ],
          [ 'item_', '_item.name', '_refine.ls_weighting_details' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_weighting_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Sigdel model of Konnert-Hendrickson:\n                                  Sigdel =\n                                  Afsig +  Bfsig*(sin(theta)/lambda-1/6)\n                                  Afsig = 22.0, Bfsig = 150.0\n                                    at the beginning of refinement.\n                                  Afsig = 16.0, Bfsig =  60.0\n                                    at the end of refinement.' ] ] ],
      [ 'save_',
        '_refine.ls_weighting_scheme',
        [ [ 'item_',
            '_item_description.description',
            '              The weighting scheme applied in the least-squares process. The\n               standard code may be followed by a description of the weight\n               (but see _refine.ls_weighting_details for a preferred approach).' ],
          [ 'item_', '_item.name', '_refine.ls_weighting_scheme' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_weighting_scheme' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           sigma' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'sigma',
              'based on measured e.s.d.\'s',
              'unit',
              'unit or no weights applied',
              'calc',
              'calculated weights applied' ] ] ] ],
      [ 'save_',
        '_refine.ls_wR_factor_all',
        [ [ 'item_',
            '_item_description.description',
            '              Weighted residual factor wR for all reflections that satisfy the\n               resolution limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low.\n\n                    ( sum|w |Y~obs~ - Y~calc~|^2^| )^1/2^\n               wR = ( ---------------------------- )\n                    (        sum|w Y~obs~^2^|      )\n\n               Y~obs~  = the observed amplitude specified by\n                         _refine.ls_structure_factor_coef\n               Y~calc~ = the calculated amplitude specified by\n                         _refine.ls_structure_factor_coef\n               w       = the least-squares weight\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_wR_factor_all' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_wR_factor_all' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_R_factor_all' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_wR_factor_obs',
        [ [ 'item_',
            '_item_description.description',
            '              Weighted residual factor wR for reflections that satisfy the\n               resolution limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low and the observation limit established by\n               _reflns.observed_criterion.\n\n                    ( sum|w |Y~obs~ - Y~calc~|^2^| )^1/2^\n               wR = ( ---------------------------- )\n                    (        sum|w Y~obs~^2^|      )\n\n               Y~obs~  = the observed amplitude specified by\n                         _refine.ls_structure_factor_coef\n               Y~calc~ = the calculated amplitude specified by\n                         _refine.ls_structure_factor_coef\n               w       = the least-squares weight\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_wR_factor_obs' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_wR_factor_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_R_factor_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_wR_factor_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              Weighted residual factor wR for reflections that satisfy the\n               resolution limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low and the observation limit established by\n               _reflns.observed_criterion, and that were used as the test\n               reflections (i.e. were excluded from the refinement) when the\n               refinement included the calculation of a \'free\' R factor.\n               Details of how reflections were assigned to the working and\n               test sets are given in _reflns.R_free_details.\n\n                    ( sum|w |Y~obs~ - Y~calc~|^2^| )^1/2^\n               wR = ( ---------------------------- )\n                    (        sum|w Y~obs~^2^|      )\n\n               Y~obs~  = the observed amplitude specified by\n                         _refine.ls_structure_factor_coef\n               Y~calc~ = the calculated amplitude specified by\n                         _refine.ls_structure_factor_coef\n               w       = the least-squares weight\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_wR_factor_R_free' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_R_factor_R_free' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.ls_wR_factor_R_work',
        [ [ 'item_',
            '_item_description.description',
            '              Weighted residual factor wR for reflections that satisfy the\n               resolution limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low and the observation limit established by\n               _reflns.observed_criterion, and that were used as the working\n               reflections (i.e. were included in the refinement) when the\n               refinement included the calculation of a \'free\' R factor.\n               Details of how reflections were assigned to the working and\n               test sets are given in _reflns.R_free_details.\n\n                    ( sum|w |Y~obs~ - Y~calc~|^2^| )^1/2^\n               wR = ( ---------------------------- )\n                    (        sum|w Y~obs~^2^|      )\n\n               Y~obs~  = the observed amplitude specified by\n                         _refine.ls_structure_factor_coef\n               Y~calc~ = the calculated amplitude specified by\n                         _refine.ls_structure_factor_coef\n               w       = the least-squares weight\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_wR_factor_R_work' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_wR_factor_ref' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_R_factor_R_work' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.occupancy_max',
        [ [ 'item_',
            '_item_description.description',
            '              The maximum value for occupancy found in the coordinate set.' ],
          [ 'item_', '_item.name', '_refine.occupancy_max' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.occupancy_min',
        [ [ 'item_',
            '_item_description.description',
            '              The minimum value for occupancy found in the coordinate set.' ],
          [ 'item_', '_item.name', '_refine.occupancy_min' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.overall_SU_B',
        [ [ 'item_',
            '_item_description.description',
            '              The overall standard uncertainty (estimated standard deviation)\n               of the displacement parameters based on a maximum-likelihood\n               residual.\n\n               The overall standard uncertainty (sigma~B~)^2 gives an idea\n               of the uncertainty in the B values of averagely defined\n               atoms (atoms with B values equal to the average B value).\n\n                                             N_a\n    (sigma~B~)^2  = 8 ----------------------------------------------\n                      sum~i~ {[1/Sigma - (E_o)^2 (1-m^2)](SUM_AS)s^4}\n\n               SUM_AS         = (sigma_A)^2/Sigma^2\n               N_a            = number of atoms\n               Sigma          = (sigma_{E;exp})^2 + epsilon [1-(sigma_A)^2]\n               E_o            = normalized structure factors\n               sigma_{E;exp}  = experimental uncertainties of normalized\n                                structure factors\n               sigma_A        = <cos 2 pi s delta_x> SQRT(Sigma_P/Sigma_N)\n                                estimated using maximum likelihood\n               Sigma_P        = sum_{atoms in model} f^2\n               Sigma_N        = sum_{atoms in crystal} f^2\n               f              = form factor of atoms\n               delta_x        = expected error\n               m              = figure of merit of phases of reflections\n                                included in the summation\n               s              = reciprocal-space vector\n               epsilon        = multiplicity of diffracting plane\n\n               summation is over all reflections included in refinement\n\n               Ref: (sigma_A estimation) "Refinement of macromolecular\n                    structures by the maximum-likelihood method",\n                    Murshudov, G. N., Vagin, A. A. & Dodson, E. J. (1997).\n                    Acta Cryst. D53, 240-255.\n\n                    (SU ML estimation) Murshudov, G. N. & Dodson,\n                    E. J. (1997). Simplified error estimation a la\n                    Cruickshank in macromolecular crystallography.\n                    CCP4 Newsletter on Protein Crystallography, No. 33,\n                    January 1997, pp. 31-39.\n                   http://www.ccp4.ac.uk/newsletters/newsletter33/murshudov.html' ],
          [ 'item_', '_item.name', '_refine.overall_SU_B' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_Overall_ESU_B' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.overall_SU_ML',
        [ [ 'item_',
            '_item_description.description',
            '              The overall standard uncertainty (estimated standard deviation)\n               of the positional parameters based on a maximum likelihood\n               residual.\n\n               The overall standard uncertainty (sigma~X~)^2 gives an\n               idea of the uncertainty in the position of averagely\n               defined atoms (atoms with B values equal to average B value)\n\n                    3                         N_a\n    (sigma~X~)^2  = -----------------------------------------------------\n                    8 pi^2 sum~i~ {[1/Sigma - (E_o)^2 (1-m^2)](SUM_AS)s^2}\n\n               SUM_AS         = (sigma_A)^2/Sigma^2)\n               N_a            = number of atoms\n               Sigma          = (sigma_{E;exp})^2 + epsilon [1-{sigma_A)^2]\n               E_o            = normalized structure factors\n\n               sigma_{E;exp}  = experimental uncertainties of normalized\n                                structure factors\n               sigma_A        = <cos 2 pi s delta_x> SQRT(Sigma_P/Sigma_N)\n                                estimated using maximum likelihood\n               Sigma_P        = sum_{atoms in model} f^2\n               Sigma_N        = sum_{atoms in crystal} f^2\n               f              = form factor of atoms\n               delta_x        = expected error\n               m              = figure of merit of phases of reflections\n                                included in the summation\n               s              = reciprocal-space vector\n               epsilon        = multiplicity of the diffracting plane\n\n               summation is over all reflections included in refinement\n\n               Ref: (sigma_A estimation) "Refinement of macromolecular\n                    structures by the maximum-likelihood method",\n                    Murshudov, G. N., Vagin, A. A. & Dodson, E. J. (1997).\n                    Acta Cryst. D53, 240-255.\n\n                    (SU ML estimation) Murshudov, G. N. & Dodson,\n                    E. J. (1997). Simplified error estimation a la\n                    Cruickshank in macromolecular crystallography.\n                    CCP4 Newsletter on Protein Crystallography, No. 33,\n                    January 1997, pp. 31-39.\n                   http://www.ccp4.ac.uk/newsletters/newsletter33/murshudov.html' ],
          [ 'item_', '_item.name', '_refine.overall_SU_ML' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_Overall_ESU_ML' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.overall_SU_R_Cruickshank_DPI',
        [ [ 'item_',
            '_item_description.description',
            '              The overall standard uncertainty (estimated standard deviation)\n               of the displacement parameters based on the crystallographic\n               R value, expressed in a formalism known as the dispersion\n               precision indicator (DPI).\n\n               The overall standard uncertainty (sigma~B~) gives an idea\n               of the uncertainty in the B values of averagely defined\n               atoms (atoms with B values equal to the average B value).\n\n                                     N_a\n               (sigma_B)^2  = 0.65 --------- (R_value)^2 (D_min)^2 C^(-2/3)\n                                   (N_o-N_p)\n\n\n               N_a      = number of atoms\n               N_o      = number of reflections included in refinement\n               N_p      = number of refined parameters\n               R_value  = conventional crystallographic R value\n               D_min    = maximum resolution\n               C        = completeness of data\n\n               Ref: Cruickshank, D. W. J. (1999). Acta Cryst. D55, 583-601.\n\n                    Murshudov, G. N. & Dodson,\n                    E. J. (1997). Simplified error estimation a la\n                    Cruickshank in macromolecular crystallography.\n                    CCP4 Newsletter on Protein Crystallography, No. 33,\n                    January 1997, pp. 31-39.\n                   http://www.ccp4.ac.uk/newsletters/newsletter33/murshudov.html\n' ],
          [ 'item_', '_item.name', '_refine.overall_SU_R_Cruickshank_DPI' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_Overall_ESU_R_Cruickshanks_DPI' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.overall_SU_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              The overall standard uncertainty (estimated standard deviation)\n               of the displacement parameters based on the free R value.\n\n               The overall standard uncertainty gives an idea of the\n               uncertainty in the B values of averagely defined atoms\n               (atoms with B values equal to the average B value).\n\n                                    N_a\n               (sigma_B)^2  = 0.65 ----- (R_free)^2 (D_min)^2 C^(-2/3)\n                                    N_o\n\n               N_a    = number of atoms\n               N_o    = number of reflections included in refinement\n               R_free = conventional free crystallographic R value calculated\n                        using reflections not included in refinement\n               D_min  = maximum resolution\n               C      = completeness of data\n\n               Ref: Cruickshank, D. W. J. (1999). Acta Cryst. D55, 583-601.\n\n                    Murshudov, G. N. & Dodson,\n                    E. J. (1997). Simplified error estimation a la\n                    Cruickshank in macromolecular crystallography.\n                    CCP4 Newsletter on Protein Crystallography, No. 33,\n                    January 1997, pp. 31-39.\n                   http://www.ccp4.ac.uk/newsletters/newsletter33/murshudov.html' ],
          [ 'item_', '_item.name', '_refine.overall_SU_R_free' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_Overall_ESU_Rfree' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.overall_FOM_free_R_set',
        [ [ 'item_',
            '_item_description.description',
            '              Average figure of merit of phases of reflections not included\n               in the refinement.\n\n               This value is derived from the likelihood function.\n\n               FOM           = I_1(X)/I_0(X)\n\n               I_0, I_1      = zero- and first-order modified Bessel functions\n                               of the first kind\n               X             = sigma_A |E_o| |E_c|/SIGMA\n               E_o, E_c      = normalized observed and calculated structure\n                               factors\n               sigma_A       = <cos 2 pi s delta_x> SQRT(Sigma_P/Sigma_N)\n                               estimated using maximum likelihood\n               Sigma_P       = sum_{atoms in model} f^2\n               Sigma_N       = sum_{atoms in crystal} f^2\n               f             = form factor of atoms\n               delta_x       = expected error\n               SIGMA         = (sigma_{E;exp})^2 + epsilon [1-(sigma_A)^2]\n               sigma_{E;exp} = uncertainties of normalized observed\n                               structure factors\n               epsilon       = multiplicity of the diffracting plane\n\n               Ref: Murshudov, G. N., Vagin, A. A. & Dodson, E. J. (1997).\n                    Acta Cryst. D53, 240-255.' ],
          [ 'item_', '_item.name', '_refine.overall_FOM_free_R_set' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_overall_FOM_free_Rset' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.overall_FOM_work_R_set',
        [ [ 'item_',
            '_item_description.description',
            '              Average figure of merit of phases of reflections included in\n               the refinement.\n\n               This value is derived from the likelihood function\n\n               FOM           = I_1(X)/I_0(X)\n\n               I_0, I_1      = zero- and first-order modified Bessel functions\n                               of the first kind\n               X             = sigma_A |E_o| |E_c|/SIGMA\n               E_o, E_c      = normalized observed and calculated structure\n                               factors\n               sigma_A       = <cos 2 pi s delta_x> SQRT(Sigma_P/Sigma_N)\n                               estimated using maximum likelihood\n               Sigma_P       = sum_{atoms in model} f^2\n               Sigma_N       = sum_{atoms in crystal} f^2\n               f             = form factor of atoms\n               delta_x       = expected error\n               SIGMA         = (sigma_{E;exp})^2 + epsilon [1-(sigma_A)^2]\n               sigma_{E;exp} = uncertainties of normalized observed\n                               structure factors\n               epsilon       = multiplicity of the diffracting plane\n\n               Ref: Murshudov, G. N., Vagin, A. A. & Dodson, E. J. (1997).\n                    Acta Cryst. D53, 240-255.' ],
          [ 'item_', '_item.name', '_refine.overall_FOM_work_R_set' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_overall_FOM_work_Rset' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.solvent_model_details',
        [ [ 'item_',
            '_item_description.description',
            '              Special aspects of the solvent model used during refinement.' ],
          [ 'item_', '_item.name', '_refine.solvent_model_details' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_refine.solvent_model_param_bsol',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the BSOL solvent-model parameter describing\n               the average isotropic displacement parameter of disordered\n               solvent atoms.\n\n               This is one of the two parameters (the other is\n               _refine.solvent_model_param_ksol) in Tronrud\'s method of\n               modelling the contribution of bulk solvent to the\n               scattering. The standard scale factor is modified according\n               to the expression\n\n                   k0 exp(-B0 * s^2^)[1-KSOL * exp(-BSOL * s^2^)]\n\n               where k0 and B0 are the scale factors for the protein.\n\n               Ref: Tronrud, D. E. (1997). Methods Enzymol. 277, 243-268.' ],
          [ 'item_', '_item.name', '_refine.solvent_model_param_bsol' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.solvent_model_param_ksol',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the KSOL solvent-model parameter describing\n               the ratio of the electron density in the bulk solvent to the\n               electron density in the molecular solute.\n\n               This is one of the two parameters (the other is\n               _refine.solvent_model_param_bsol) in Tronrud\'s method of\n               modelling the contribution of bulk solvent to the\n               scattering. The standard scale factor is modified according\n               to the expression\n\n                   k0 exp(-B0 * s^2^)[1-KSOL * exp(-BSOL * s^2^)]\n\n               where k0 and B0 are the scale factors for the protein.\n\n               Ref: Tronrud, D. E. (1997). Methods Enzymol. 277, 243-268.' ],
          [ 'item_', '_item.name', '_refine.solvent_model_param_ksol' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# REFINE_ANALYZE ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'refine_analyze',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_ANALYZE category record details\n               about the refined structure that are often used to analyze the\n               refinement and assess its quality. A given computer program\n               may or may not produce values corresponding to these data\n               names.' ],
          [ 'item_', '_category.id', 'refine_analyze' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_refine_analyze.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _refine_analyze.entry_id\n    _refine_analyze.Luzzati_coordinate_error_obs\n    _refine_analyze.Luzzati_d_res_low_obs\n        5HVP 0.056  2.51',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine_analyze.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_refine_analyze.entry_id' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_refine_analyze.Luzzati_coordinate_error_free',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated coordinate error obtained from the plot of\n               the R value versus sin(theta)/lambda for the reflections\n               treated as a test set during refinement.\n\n               Ref:  Luzzati, V. (1952). Traitement statistique des erreurs\n               dans la determination des structures cristallines. Acta\n               Cryst. 5, 802-810.' ],
          [ 'item_',
            '_item.name',
            '_refine_analyze.Luzzati_coordinate_error_free' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_analyze.Luzzati_coordinate_error_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated coordinate error obtained from the plot of\n               the R value versus sin(theta)/lambda for reflections classified\n               as observed.\n\n               Ref:  Luzzati, V. (1952). Traitement statistique des erreurs\n               dans la determination des structures cristallines. Acta\n               Cryst. 5, 802-810.' ],
          [ 'item_',
            '_item.name',
            '_refine_analyze.Luzzati_coordinate_error_obs' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_analyze.Luzzati_d_res_low_free',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the low-resolution cutoff used in constructing the\n               Luzzati plot for reflections treated as a test set during\n               refinement.\n\n               Ref:  Luzzati, V. (1952). Traitement statistique des erreurs\n               dans la determination des structures cristallines. Acta\n               Cryst. 5, 802-810.' ],
          [ 'item_',
            '_item.name',
            '_refine_analyze.Luzzati_d_res_low_free' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_analyze.Luzzati_d_res_low_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the low-resolution cutoff used in\n               constructing the Luzzati plot for reflections classified as\n               observed.\n\n               Ref:  Luzzati, V. (1952). Traitement statistique des erreurs\n               dans la determination des structures cristallines. Acta\n               Cryst. 5, 802-810.' ],
          [ 'item_',
            '_item.name',
            '_refine_analyze.Luzzati_d_res_low_obs' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_analyze.Luzzati_sigma_a_free',
        [ [ 'item_',
            '_item_description.description',
            '              The value of sigma~a~ used in constructing the Luzzati plot for\n               the reflections treated as a test set during refinement.\n               Details of the estimation of sigma~a~ can be specified\n               in _refine_analyze.Luzzati_sigma_a_free_details.\n\n               Ref:  Luzzati, V. (1952). Traitement statistique des erreurs\n               dans la determination des structures cristallines. Acta\n               Cryst. 5, 802-810.' ],
          [ 'item_', '_item.name', '_refine_analyze.Luzzati_sigma_a_free' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_analyze.Luzzati_sigma_a_free_details',
        [ [ 'item_',
            '_item_description.description',
            '              Details of the estimation of sigma~a~ for the reflections\n               treated as a test set during refinement.\n\n               Ref:  Luzzati, V. (1952). Traitement statistique des erreurs\n               dans la determination des structures cristallines. Acta\n               Cryst. 5, 802-810.' ],
          [ 'item_',
            '_item.name',
            '_refine_analyze.Luzzati_sigma_a_free_details' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_refine_analyze.Luzzati_sigma_a_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The value of sigma~a~ used in constructing the Luzzati plot for\n               reflections classified as observed. Details of the\n               estimation of sigma~a~ can be specified in\n               _refine_analyze.Luzzati_sigma_a_obs_details.\n\n               Ref:  Luzzati, V. (1952). Traitement statistique des erreurs\n               dans la determination des structures cristallines. Acta\n               Cryst. 5, 802-810.' ],
          [ 'item_', '_item.name', '_refine_analyze.Luzzati_sigma_a_obs' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_analyze.Luzzati_sigma_a_obs_details',
        [ [ 'item_',
            '_item_description.description',
            '              Special aspects of the estimation of sigma~a~ for the\n               reflections classified as observed.\n\n               Ref:  Luzzati, V. (1952). Traitement statistique des erreurs\n               dans la determination des structures cristallines. Acta\n               Cryst. 5, 802-810.' ],
          [ 'item_',
            '_item.name',
            '_refine_analyze.Luzzati_sigma_a_obs_details' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_refine_analyze.number_disordered_residues',
        [ [ 'item_',
            '_item_description.description',
            '              The number of discretely disordered residues in the refined\n               model.' ],
          [ 'item_',
            '_item.name',
            '_refine_analyze.number_disordered_residues' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_analyze.occupancy_sum_hydrogen',
        [ [ 'item_',
            '_item_description.description',
            '              The sum of the occupancies of the hydrogen atoms in the refined\n               model.' ],
          [ 'item_',
            '_item.name',
            '_refine_analyze.occupancy_sum_hydrogen' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_analyze.occupancy_sum_non_hydrogen',
        [ [ 'item_',
            '_item_description.description',
            '              The sum of the occupancies of the non-hydrogen atoms in the\n                refined model.' ],
          [ 'item_',
            '_item.name',
            '_refine_analyze.occupancy_sum_non_hydrogen' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_analyze.RG_d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the high-resolution cutoff in angstroms\n               used in the calculation of the Hamilton generalized\n               R factor (RG) stored in _refine_analyze.RG_work and\n               _refine_analyze.RG_free.\n\n               Ref: Hamilton, W. C. (1965). Acta Cryst. 18, 502-510.' ],
          [ 'item_', '_item.name', '_refine_analyze.RG_d_res_high' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_analyze.ebi_RG_d_res_high' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_analyze.RG_d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the low-resolution cutoff in angstroms\n               used in the calculation of the Hamilton generalized\n               R factor (RG) stored in _refine_analyze.RG_work and\n               _refine_analyze.RG_free.\n\n               Ref: Hamilton, W. C. (1965). Acta Cryst. 18, 502-510.' ],
          [ 'item_', '_item.name', '_refine_analyze.RG_d_res_low' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_analyze.ebi_RG_d_res_low' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_analyze.RG_free',
        [ [ 'item_',
            '_item_description.description',
            '              The Hamilton generalized R factor for all reflections that\n               satisfy the resolution limits established by\n               _refine_analyze.RG_d_res_high and\n               _refine_analyze.RG_d_res_low for the free R set of\n               reflections that were excluded from the refinement.\n\n             sum_i sum_j w_{i,j}(|Fobs|_i - G|Fcalc|_i)(|Fobs|_j - G|Fcalc|_j)\n RG = Sqrt( ----------------------------------------------------------------- )\n                          sum_i sum_j w_{i,j} |Fobs|_i |Fobs|_j\n\n               where\n\n               |Fobs|  = the observed structure-factor amplitudes\n               |Fcalc| = the calculated structure-factor amplitudes\n               G       = the scale factor which puts |Fcalc| on the\n                         same scale as |Fobs|\n               w_{i,j} = the weight for the combination of the reflections\n                         i and j.\n\n               sum_i and sum_j are taken over the specified reflections\n\n               When the covariance of the amplitudes of reflection i and\n               reflection j is zero (i.e. the reflections are independent)\n               w{i,i} can be redefined as  w_i and the nested sums collapsed\n               into one sum.\n\n                           sum_i w_i(|Fobs|_i - G|Fcalc|_i)^2\n               RG = Sqrt( ----------------------------------- )\n                                  sum_i w_i |Fobs|_i^2\n\n               Ref: Hamilton, W. C. (1965). Acta Cryst. 18, 502-510.' ],
          [ 'item_', '_item.name', '_refine_analyze.RG_free' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_analyze.ebi_RG_free' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_analyze.RG_work',
        [ [ 'item_',
            '_item_description.description',
            '              The Hamilton generalized R factor for all reflections\n               that satisfy the resolution limits established by\n               _refine_analyze.RG_d_res_high and\n               _refine_analyze.RG_d_res_low  and for those\n               reflections included in the working set when a free R set\n               of reflections is omitted from the refinement.\n\n             sum_i sum_j w_{i,j}(|Fobs|_i - G|Fcalc|_i)(|Fobs|_j - G|Fcalc|_j)\n RG = Sqrt( ----------------------------------------------------------------- )\n                          sum_i sum_j w_{i,j} |Fobs|_i |Fobs|_j\n\n               where\n\n               |Fobs|  = the observed structure-factor amplitudes\n               |Fcalc| = the calculated structure-factor amplitudes\n               G       = the scale factor which puts |Fcalc| on the\n                         same scale as |Fobs|\n               w_{i,j} = the weight for the combination of the reflections\n                         i and j.\n\n               sum_i and sum_j are taken over the specified reflections\n\n               When the covariance of the amplitudes of reflection i and\n               reflection j is zero (i.e. the reflections are independent)\n               w{i,i} can be redefined as w_i and the nested sums collapsed\n               into one sum.\n\n                           sum_i w_i(|Fobs|_i - G|Fcalc|_i)^2\n               RG = Sqrt( ----------------------------------- )\n                                  sum_i w_i |Fobs|_i^2\n\n               Ref: Hamilton, W. C. (1965). Acta Cryst. 18, 502-510.' ],
          [ 'item_', '_item.name', '_refine_analyze.RG_work' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_analyze.ebi_RG_work' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_analyze.RG_free_work_ratio',
        [ [ 'item_',
            '_item_description.description',
            '              The observed ratio of RGfree to RGwork. The expected RG ratio\n               is the value that should be achievable at the end of a structure\n               refinement when only random uncorrelated errors exist in the data\n               and the model provided that the observations are properly\n               weighted. When compared with the observed RG ratio it may\n               indicate that a structure has not reached convergence or a\n               model has been over-refined with no corresponding improvement\n               in the model.\n\n               In an unrestrained refinement, the ratio of RGfree to RGwork with\n               only random uncorrelated errors at convergence depends only\n               on the number of reflections and the number of parameters\n               according to\n\n                                 sqrt[(f + m) / (f - m) ]\n\n               where f = the number of included structure amplitudes and\n                         target distances, and\n                     m = the number of parameters being refined.\n\n               In the restrained case, RGfree is calculated from a random\n               selection of residuals including both structure amplitudes\n               and restraints.  When restraints are included in the refinement,\n               the RG ratio requires a term for the contribution to the\n               minimized residual at convergence, D~restr~, due to those\n               restraints:\n\n                        D~restr~ = r - sum [w_i . (a_i)^t . (H)^-1 a_i]\n\n               where\n\n               r is the number of geometrical, displacement-parameter and\n                 other restraints\n               H is the (m,m) normal matrix given by A^t.W.A\n               W is the (n,n) symmetric weight matrix of the included\n                 observations\n               A is the least-squares design matrix of derivatives of\n                 order (n,m)\n               a_i is the ith row of A\n\n              Then the expected RGratio becomes\n\n                  sqrt [ (f + (m - r + D~restr~))/ (f - (m - r + D~restr~)) ]\n\n              There is no data name for the expected value of RGfree/RGwork yet.\n\n               Ref: Tickle, I. J., Laskowski, R. A. & Moss, D. S. (1998).\n                    Acta Cryst. D54, 547-557.\n' ],
          [ 'item_', '_item.name', '_refine_analyze.RG_free_work_ratio' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_analyze.ebi_RG_work_free_ratio' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#################' ],
      [ '@comment', '# REFINE_B_ISO ##' ],
      [ '@comment', '#################' ],
      [ 'save_',
        'refine_B_iso',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_B_ISO category record details about\n               the treatment of isotropic B factors (displacement parameters)\n               during refinement.' ],
          [ 'item_', '_category.id', 'refine_B_iso' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_refine_B_iso.class' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _refine_B_iso.class\n    _refine_B_iso.treatment\n     \'protein\'    isotropic\n     \'solvent\'    isotropic\n     \'inhibitor\'  isotropic',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine_B_iso.class',
        [ [ 'item_',
            '_item_description.description',
            '              A class of atoms treated similarly for isotropic B-factor\n               (displacement-parameter) refinement.' ],
          [ 'item_', '_item.name', '_refine_B_iso.class' ],
          [ 'item_', '_item.category_id', 'refine_B_iso' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'all', 'protein', 'solvent', 'sugar-phosphate backbone' ] ] ] ],
      [ 'save_',
        '_refine_B_iso.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the isotropic B-factor\n               (displacement-parameter) refinement for the class of atoms\n               described in _refine_B_iso.class.' ],
          [ 'item_', '_item.name', '_refine_B_iso.details' ],
          [ 'item_', '_item.category_id', 'refine_B_iso' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The temperature factors of atoms in the side\n                                  chain of Arg 92 were held fixed due to\n                                  unstable behavior in refinement.' ] ] ],
      [ 'save_',
        '_refine_B_iso.treatment',
        [ [ 'item_',
            '_item_description.description',
            '              The treatment of isotropic B-factor (displacement-parameter)\n               refinement for a class of atoms defined in _refine_B_iso.class.' ],
          [ 'item_', '_item.name', '_refine_B_iso.treatment' ],
          [ 'item_', '_item.category_id', 'refine_B_iso' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'fixed', 'isotropic', 'anisotropic' ] ] ] ],
      [ 'save_',
        '_refine_B_iso.value',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the isotropic B factor (displacement parameter)\n               assigned to a class of atoms defined in _refine_B_iso.class.\n               Meaningful only for atoms with fixed isotropic B factors.' ],
          [ 'item_', '_item.name', '_refine_B_iso.value' ],
          [ 'item_', '_item.category_id', 'refine_B_iso' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ '@comment', '##############################' ],
      [ '@comment', '# REFINE_FUNCTION_MINIMIZED ##' ],
      [ '@comment', '##############################' ],
      [ 'save_',
        'refine_funct_minimized',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_FUNCT_MINIMIZED category record\n               details about the individual terms of the function minimized\n               during refinement.' ],
          [ 'item_', '_category.id', 'refine_funct_minimized' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_refine_funct_minimized.type' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on RESTRAIN refinement for the CCP4 test data set\n                toxd.',
              '\n    loop_\n    _refine_funct_minimized.type\n    _refine_funct_minimized.number_terms\n    _refine_funct_minimized.residual\n    \'sum(W*Delta(Amplitude)^2\'               3009      1621.3\n    \'sum(W*Delta(Plane+Rigid)^2\'               85       56.68\n    \'sum(W*Delta(Distance)^2\'                1219      163.59\n    \'sum(W*Delta(U-tempfactors)^2\'           1192      69.338',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine_funct_minimized.number_terms',
        [ [ 'item_',
            '_item_description.description',
            '              The number of observations in this term.  For example, if the\n               term is a residual of the X-ray intensities, this item would\n               contain the number of reflections used in the refinement.' ],
          [ 'item_', '_item.name', '_refine_funct_minimized.number_terms' ],
          [ 'item_', '_item.category_id', 'refine_funct_minimized' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ebi_refine_funct_minimized.NumTerms' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_funct_minimized.residual',
        [ [ 'item_',
            '_item_description.description',
            '              The residual for this term of the function that was minimized\n              during the refinement.' ],
          [ 'item_', '_item.name', '_refine_funct_minimized.residual' ],
          [ 'item_', '_item.category_id', 'refine_funct_minimized' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ebi_refine_funct_minimized.Residual' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_funct_minimized.type',
        [ [ 'item_',
            '_item_description.description',
            '              The type of the function being minimized.' ],
          [ 'item_', '_item.name', '_refine_funct_minimized.type' ],
          [ 'item_', '_item.category_id', 'refine_funct_minimized' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ebi_refine_funct_minimized.type' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_refine_funct_minimized.weight',
        [ [ 'item_',
            '_item_description.description',
            '              The weight applied to this term of the function that was\n               minimized during the refinement.' ],
          [ 'item_', '_item.name', '_refine_funct_minimized.weight' ],
          [ 'item_', '_item.category_id', 'refine_funct_minimized' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ebi_refine_funct_minimized.weight' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# REFINE_HIST ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'refine_hist',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_HIST category record details about the\n               steps during the refinement of the structure.\n               These data items are not meant to be as thorough a description\n               of the refinement as is provided for the final model in other\n               categories; rather, these data items provide a mechanism for\n               sketching out the progress of the refinement, supported by a\n               small set of representative statistics.' ],
          [ 'item_', '_category.id', 'refine_hist' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_refine_hist.cycle_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for the collagen-like\n                peptide [(POG)4 EKG (POG)5]3.',
              '\n    _refine_hist.cycle_id                C134\n    _refine_hist.d_res_high              1.85\n    _refine_hist.d_res_low               20.0\n    _refine_hist.number_atoms_solvent     217\n    _refine_hist.number_atoms_total       808\n    _refine_hist.number_reflns_all       6174\n    _refine_hist.number_reflns_obs       4886\n    _refine_hist.number_reflns_R_free     476\n    _refine_hist.number_reflns_R_work    4410\n    _refine_hist.R_factor_all            .265\n    _refine_hist.R_factor_obs            .195\n    _refine_hist.R_factor_R_free         .274\n    _refine_hist.R_factor_R_work         .160\n    _refine_hist.details\n    ; Add majority of solvent molecules. B factors refined by\n      group. Continued to remove misplaced water molecules.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine_hist.cycle_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _refine_hist.cycle_id must uniquely identify a\n               record in the REFINE_HIST list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_refine_hist.cycle_id' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_refine_hist.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of this cycle of the refinement\n               process.' ],
          [ 'item_', '_item.name', '_refine_hist.details' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Residues 13-17 fit and added to model;\n                                  substantial rebuilding of loop containing\n                                  residues 43-48; addition of first atoms to\n                                  solvent model; ten cycles of Prolsq\n                                  refinement.' ] ] ],
      [ 'save_',
        '_refine_hist.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The lowest value for the interplanar spacings for the\n               reflection data for this cycle of refinement. This is called\n               the highest resolution.' ],
          [ 'item_', '_item.name', '_refine_hist.d_res_high' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_hist.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The highest value for the interplanar spacings for the\n               reflection data for this cycle of refinement. This is\n               called the lowest resolution.' ],
          [ 'item_', '_item.name', '_refine_hist.d_res_low' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_hist.number_atoms_solvent',
        [ [ 'item_',
            '_item_description.description',
            '              The number of solvent atoms that were included in the model at\n               this cycle of the refinement.' ],
          [ 'item_', '_item.name', '_refine_hist.number_atoms_solvent' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_hist.number_atoms_total',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of atoms that were included in the model at\n               this cycle of the refinement.' ],
          [ 'item_', '_item.name', '_refine_hist.number_atoms_total' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_hist.number_reflns_all',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine_hist.d_res_high and\n               _refine_hist.d_res_low.' ],
          [ 'item_', '_item.name', '_refine_hist.number_reflns_all' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_hist.number_reflns_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine_hist.d_res_high and\n               _refine_hist.d_res_low and the observation criterion\n               established by _reflns.observed_criterion.' ],
          [ 'item_', '_item.name', '_refine_hist.number_reflns_obs' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_hist.number_reflns_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine_hist.d_res_high and\n               _refine_hist.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were used\n               as the test reflections (i.e. were excluded from the refinement)\n               when the refinement included the calculation of a \'free\'\n               R factor. Details of how reflections were assigned to the\n               working and test sets are given in _reflns.R_free_details.' ],
          [ 'item_', '_item.name', '_refine_hist.number_reflns_R_free' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_hist.number_reflns_R_work',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine_hist.d_res_high and\n               _refine_hist.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were used\n               as the working reflections (i.e. were included in the\n               refinement) when the refinement included the calculation\n               of a \'free\' R factor. Details of how reflections were\n               assigned to the working and test sets are given in\n               _reflns.R_free_details.' ],
          [ 'item_', '_item.name', '_refine_hist.number_reflns_R_work' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_hist.R_factor_all',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine_hist.d_res_high and\n               _refine_hist.d_res_low.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_hist.R_factor_all' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_hist.R_factor_obs',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine_hist.d_res_high and\n               _refine_hist.d_res_low and the observation criterion\n               established by _reflns.observed_criterion.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_hist.R_factor_obs' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_hist.R_factor_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine_hist.d_res_high and\n               _refine_hist.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were used\n               as the test reflections (i.e. were excluded from the refinement)\n               when the refinement included the calculation of a \'free\'\n               R factor. Details of how reflections were assigned to the\n               working and test sets are given in _reflns.R_free_details.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_hist.R_factor_R_free' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_hist.R_factor_R_work',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine_hist.d_res_high and\n               _refine_hist.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were used\n               as the working reflections (i.e. were included in the refinement)\n               when the refinement included the calculation of a \'free\'\n               R factor. Details of how reflections were assigned to the\n               working and test sets are given in _reflns.R_free_details.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_hist.R_factor_R_work' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# REFINE_LS_RESTR ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'refine_ls_restr',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_LS_RESTR category record details about\n               the restraints applied to various classes of parameters during\n               the least-squares refinement.' ],
          [ 'item_', '_category.id', 'refine_ls_restr' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_refine_ls_restr.type' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _refine_ls_restr.type\n    _refine_ls_restr.dev_ideal_target\n    _refine_ls_restr.dev_ideal\n    _refine_ls_restr.number\n    _refine_ls_restr.criterion\n    _refine_ls_restr.rejects\n     \'bond_d\'           0.020  0.018  1654  \'> 2\\s\'  22\n     \'angle_d\'          0.030  0.038  2246  \'> 2\\s\'  139\n     \'planar_d\'         0.040  0.043  498   \'> 2\\s\'  21\n     \'planar\'           0.020  0.015  270   \'> 2\\s\'  1\n     \'chiral\'           0.150  0.177  278   \'> 2\\s\'  2\n     \'singtor_nbd\'      0.500  0.216  582   \'> 2\\s\'  0\n     \'multtor_nbd\'      0.500  0.207  419   \'> 2\\s\'  0\n     \'xyhbond_nbd\'      0.500  0.245  149   \'> 2\\s\'  0\n     \'planar_tor\'       3.0    2.6    203   \'> 2\\s\'  9\n     \'staggered_tor\'    15.0   17.4   298   \'> 2\\s\'  31\n     \'orthonormal_tor\'  20.0   18.1   12    \'> 2\\s\'  1',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine_ls_restr.criterion',
        [ [ 'item_',
            '_item_description.description',
            '              A criterion used to define a parameter value that deviates\n               significantly from its ideal value in the model obtained by\n               restrained least-squares refinement.' ],
          [ 'item_', '_item.name', '_refine_ls_restr.criterion' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', '> 3\\s' ] ] ],
      [ 'save_',
        '_refine_ls_restr.dev_ideal',
        [ [ 'item_',
            '_item_description.description',
            '              For the given parameter type, the root-mean-square deviation\n               between the ideal values used as restraints in the least-squares\n               refinement and the values obtained by refinement. For instance,\n               bond distances may deviate by 0.018 \\%A (r.m.s.) from ideal\n               values in the current model.' ],
          [ 'item_', '_item.name', '_refine_ls_restr.dev_ideal' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_restr.dev_ideal_target',
        [ [ 'item_',
            '_item_description.description',
            '              For the given parameter type, the target root-mean-square\n               deviation between the ideal values used as restraints in the\n               least-squares refinement and the values obtained by refinement.' ],
          [ 'item_', '_item.name', '_refine_ls_restr.dev_ideal_target' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_restr.number',
        [ [ 'item_',
            '_item_description.description',
            '              The number of parameters of this type subjected to restraint in\n               least-squares refinement.' ],
          [ 'item_', '_item.name', '_refine_ls_restr.number' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_ls_restr.rejects',
        [ [ 'item_',
            '_item_description.description',
            '              The number of parameters of this type that deviate from ideal\n               values by more than the amount defined in\n               _refine_ls_restr.criterion in the model obtained by restrained\n               least-squares refinement.' ],
          [ 'item_', '_item.name', '_refine_ls_restr.rejects' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_ls_restr.type',
        [ [ 'item_',
            '_item_description.description',
            '              The type of the parameter being restrained.\n               Explicit sets of data values are provided for the programs\n               PROTIN/PROLSQ (beginning with p_) and RESTRAIN (beginning with\n               RESTRAIN_). As computer programs change, these data values\n               are given as examples, not as an enumeration list. Computer\n               programs that convert a data block to a refinement table will\n               expect the exact form of the data values given here to be used.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_refine_ls_restr.type',
              'refine_ls_restr',
              'yes',
              '_refine_ls_restr_type.type',
              'refine_ls_restr_type',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_refine_ls_restr_type.type', '_refine_ls_restr.type' ] ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ 'p_bond_d',
              'bond distance',
              'p_angle_d',
              'bond angle expressed as a distance',
              'p_planar_d',
              'planar 1,4 distance',
              'p_xhbond_d',
              'X-H bond distance',
              'p_xhangle_d',
              'X-H bond angle expressed as a distance',
              'p_hydrog_d',
              'hydrogen distance',
              'p_special_d',
              'special distance',
              'p_planar',
              'planes',
              'p_chiral',
              'chiral centres',
              'p_singtor_nbd',
              'single-torsion non-bonded contact',
              'p_multtor_nbd',
              'multiple-torsion non-bonded contact',
              'p_xyhbond_nbd',
              'possible (X...Y) hydrogen bond',
              'p_xhyhbond_nbd',
              'possible (X-H...Y) hydrogen bond',
              'p_special_tor',
              'special torsion angle',
              'p_planar_tor',
              'planar torsion angle',
              'p_staggered_tor',
              'staggered torsion angle',
              'p_orthonormal_tor',
              'orthonormal torsion angle',
              'p_mcbond_it',
              'main-chain bond isotropic displacement parameter',
              'p_mcangle_it',
              'main-chain angle isotropic displacement parameter',
              'p_scbond_it',
              'side-chain bond isotropic displacement parameter',
              'p_scangle_it',
              'side-chain angle isotropic displacement parameter',
              'p_xhbond_it',
              'X-H bond isotropic displacement parameter',
              'p_xhangle_it',
              'X-H angle isotropic displacement parameter',
              'p_special_it',
              'special isotropic displacement parameter',
              'RESTRAIN_Distances < 2.12',
              '  The root-mean-square deviation\n   of the difference between the values calculated from the structures\n   used to compile the restraints dictionary parameters and the dictionary\n   values themselves in the distance range less than 2.12 Angstroms.',
              'RESTRAIN_Distances 2.12 < D < 2.625',
              '  The root-mean-square deviation\n   of the difference between the values calculated from the structures\n   used to compile the restraints dictionary parameters and the dictionary\n   values themselves in the distance range 2.12 - 2.625 Angstroms.',
              'RESTRAIN_Distances > 2.625',
              '  The root-mean-square deviation\n   of the difference between the values calculated from the structures\n   used to compile the restraints dictionary parameters and the dictionary\n   values themselves in the distance range greater than 2.625 Angstroms.',
              'RESTRAIN_Peptide Planes',
              '  The root-mean-square deviation\n   of the difference between the values calculated from the structures\n   used to compile the restraints dictionary parameters and the dictionary\n   values themselves for peptide planes.',
              'RESTRAIN_Ring and other planes',
              '  The root-mean-square deviation\n   of the difference between the values calculated from the structures\n   used to compile the restraints dictionary parameters and the dictionary\n   values themselves for rings and planes other than peptide planes.',
              'RESTRAIN_rms diffs for Uiso atoms at dist 1.2-1.4',
              '.',
              'RESTRAIN_rms diffs for Uiso atoms at dist 1.4-1.6',
              '.',
              'RESTRAIN_rms diffs for Uiso atoms at dist 1.8-2.0',
              '.',
              'RESTRAIN_rms diffs for Uiso atoms at dist 2.0-2.2',
              '.',
              'RESTRAIN_rms diffs for Uiso atoms at dist 2.2-2.4',
              '.',
              'RESTRAIN_rms diffs for Uiso atoms at dist >2.4',
              '.' ] ] ] ],
      [ 'save_',
        '_refine_ls_restr.weight',
        [ [ 'item_',
            '_item_description.description',
            '              The weighting value applied to this type of restraint in\n               the least-squares refinement.' ],
          [ 'item_', '_item.name', '_refine_ls_restr.weight' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# REFINE_LS_RESTR_NCS ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'refine_ls_restr_ncs',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_LS_RESTR_NCS category record details\n               about the restraints applied to atom positions in domains\n               related by noncrystallographic symmetry during least-squares\n               refinement, and also about the deviation of the restrained\n               atomic parameters at the end of the refinement.  It is\n               expected that these values will only be reported once for each\n               set of restrained domains.' ],
          [ 'item_', '_category.id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_refine_ls_restr_ncs.dom_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for the collagen-like\n                peptide, HYP-.',
              '\n    _refine_ls_restr_ncs.dom_id              d2\n    _refine_ls_restr_ncs.weight_position     300.0\n    _refine_ls_restr_ncs.weight_B_iso        2.0\n    _refine_ls_restr_ncs.rms_dev_position    0.09\n    _refine_ls_restr_ncs.rms_dev_B_iso       0.16\n    _refine_ls_restr_ncs.ncs_model_details\n    ;\n     NCS restraint for pseudo-twofold symmetry between domains\n     d1 and d2.  Position weight coefficient given in\n     Kcal/(mol \\%A^2^) and isotropic B weight coefficient given\n     in \\%A^2^.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine_ls_restr_ncs.dom_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_ncs_dom.id in the\n               STRUCT_NCS_DOM category.' ],
          [ 'item_', '_item.name', '_refine_ls_restr_ncs.dom_id' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_refine_ls_restr_ncs.ncs_model_details',
        [ [ 'item_',
            '_item_description.description',
            '              Special aspects of the manner in which noncrystallographic\n               restraints were applied to atomic parameters in the domain\n               specified by _refine_ls_restr_ncs.dom_id and equivalent\n               atomic parameters in the domains against which it was restrained.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_restr_ncs.ncs_model_details' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_refine_ls_restr_ncs.rms_dev_B_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The root-mean-square deviation in equivalent isotropic\n               displacement parameters in the domain specified by\n               _refine_ls_restr_ncs.dom_id\n               and in the domains against which it was restrained.' ],
          [ 'item_', '_item.name', '_refine_ls_restr_ncs.rms_dev_B_iso' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_refine_ls_restr_ncs.rms_dev_position',
        [ [ 'item_',
            '_item_description.description',
            '              The root-mean-square deviation in equivalent atom positions in\n               the domain specified by _refine_ls_restr_ncs.dom_id and in the\n               domains against which it was restrained.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_restr_ncs.rms_dev_position' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_ls_restr_ncs.weight_B_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the weighting coefficient used in\n               noncrystallographic symmetry restraint of isotropic displacement\n               parameters in  the domain specified by\n               _refine_ls_restr_ncs.dom_id to equivalent isotropic\n               displacement parameters in the domains against\n               which it was restrained.' ],
          [ 'item_', '_item.name', '_refine_ls_restr_ncs.weight_B_iso' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_restr_ncs.weight_position',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the weighting coefficient used in\n               noncrystallographic symmetry restraint of atom positions in the\n               domain specified by _refine_ls_restr_ncs.dom_id to equivalent\n               atom positions in the domains against which it was restrained.' ],
          [ 'item_', '_item.name', '_refine_ls_restr_ncs.weight_position' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# REFINE_LS_RESTR_TYPE ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'refine_ls_restr_type',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_LS_RESTR_TYPE category record details\n               about the restraint types used in the least-squares refinement.' ],
          [ 'item_', '_category.id', 'refine_ls_restr_type' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_refine_ls_restr_type.type' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on RESTRAIN refinement for the CCP4 test data set\n                toxd.',
              '\nloop_\n_refine_ls_restr.type\n_refine_ls_restr.number\n_refine_ls_restr.dev_ideal\n_refine_ls_restr.dev_ideal_target\n \'RESTRAIN_Distances < 2.12\'              509     0.005 0.022\n \'RESTRAIN_Distances 2.12 < D < 2.625\'    671     0.016 0.037\n \'RESTRAIN_Distances > 2.625\'             39      0.034 0.043\n \'RESTRAIN_Peptide Planes\'                59      0.002 0.010\n \'RESTRAIN_Ring and other planes\'         26      0.014 0.010\n \'RESTRAIN_rms diffs for Uiso atoms at dist 1.2-1.4\'\n                                             212     0.106     .\n \'RESTRAIN_rms diffs for Uiso atoms at dist 1.4-1.6\'\n                                             288     0.101     .\n \'RESTRAIN_rms diffs for Uiso atoms at dist 1.8-2.0\'\n                                             6       0.077     .\n \'RESTRAIN_rms diffs for Uiso atoms at dist 2.0-2.2\'\n                                             10      0.114     .\n \'RESTRAIN_rms diffs for Uiso atoms at dist 2.2-2.4\'\n                                             215     0.119     .\n \'RESTRAIN_rms diffs for Uiso atoms at dist >2.4\'\n                                             461     0.106     .\n\nloop_\n_refine_ls_restr_type.type\n_refine_ls_restr_type.distance_cutoff_low\n_refine_ls_restr_type.distance_cutoff_high\n \'RESTRAIN_Distances < 2.12\'                    .     2.12\n \'RESTRAIN_Distances 2.12 < D < 2.625\'          2.12  2.625\n \'RESTRAIN_Distances > 2.625\'                   2.625 .\n \'RESTRAIN_Peptide Planes\'                      .     .\n \'RESTRAIN_Ring and other planes\'               .     .\n \'RESTRAIN_rms diffs for Uiso atoms at dist 1.2-1.4\'\n                                                   1.2   1.4\n \'RESTRAIN_rms diffs for Uiso atoms at dist 1.4-1.6\'\n                                                   1.4   1.6\n \'RESTRAIN_rms diffs for Uiso atoms at dist 1.8-2.0\'\n                                                   1.8   2.0\n \'RESTRAIN_rms diffs for Uiso atoms at dist 2.0-2.2\'\n                                                   2.0   2.2\n \'RESTRAIN_rms diffs for Uiso atoms at dist 2.2-2.4\'\n                                                   2.2   2.4\n \'RESTRAIN_rms diffs for Uiso atoms at dist >2.4\'\n                                                   2.4   .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine_ls_restr_type.distance_cutoff_high',
        [ [ 'item_',
            '_item_description.description',
            '              The upper limit in angstroms of the distance range applied to\n               the current restraint type.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_restr_type.distance_cutoff_high' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_ls_restr_type.distance_cutoff_low',
        [ [ 'item_',
            '_item_description.description',
            '              The lower limit in angstroms of the distance range applied to\n               the current restraint type.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_restr_type.distance_cutoff_low' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_ls_restr_type.type',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _refine_ls_restr.type in the\n               REFINE_LS_RESTR category.' ],
          [ 'item_', '_item.name', '_refine_ls_restr_type.type' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_type' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# REFINE_LS_SHELL ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'refine_ls_shell',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_LS_SHELL category record details about\n               the results of the least-squares refinement broken down into\n               shells of resolution.' ],
          [ 'item_', '_category.id', 'refine_ls_shell' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_refine_ls_shell.d_res_low', '_refine_ls_shell.d_res_high' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _refine_ls_shell.d_res_low\n    _refine_ls_shell.d_res_high\n    _refine_ls_shell.number_reflns_obs\n    _refine_ls_shell.R_factor_obs\n      8.00   4.51   1226   0.196\n      4.51   3.48   1679   0.146\n      3.48   2.94   2014   0.160\n      2.94   2.59   2147   0.182\n      2.59   2.34   2127   0.193\n      2.34   2.15   2061   0.203\n      2.15   2.00   1647   0.188',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine_ls_shell.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The lowest value for the interplanar spacings for the\n               reflection data in this shell. This is called\n               the highest resolution.' ],
          [ 'item_', '_item.name', '_refine_ls_shell.d_res_high' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_ls_shell.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The highest value for the interplanar spacings for the\n               reflection data in this shell. This is called the lowest\n               resolution.' ],
          [ 'item_', '_item.name', '_refine_ls_shell.d_res_low' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refine_ls_shell.number_reflns_all',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine_ls_shell.d_res_high and\n               _refine_ls_shell.d_res_low.' ],
          [ 'item_', '_item.name', '_refine_ls_shell.number_reflns_all' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_ls_shell.number_reflns_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine_ls_shell.d_res_high and\n               _refine_ls_shell.d_res_low and the observation criterion\n               established by _reflns.observed_criterion.' ],
          [ 'item_', '_item.name', '_refine_ls_shell.number_reflns_obs' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_ls_shell.number_reflns_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine_ls_shell.d_res_high and\n               _refine_ls_shell.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were used\n               as the test reflections (i.e. were excluded from the refinement)\n               when the refinement included the calculation of a \'free\'\n               R factor. Details of how reflections were assigned to the\n               working and test sets are given in _reflns.R_free_details.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_shell.number_reflns_R_free' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_ls_shell.number_reflns_R_work',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine_ls_shell.d_res_high and\n               _refine_ls_shell.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were used\n               as the working reflections (i.e. were included in the\n               refinement) when the refinement included the calculation of\n               a \'free\' R factor. Details of how reflections were assigned\n               to the working and test sets are given in _reflns.R_free_details.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_shell.number_reflns_R_work' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine_ls_shell.percent_reflns_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine_ls_shell.d_res_high and\n               _refine_ls_shell.d_res_low and the observation criterion\n               established by _reflns.observed_criterion, expressed as a\n               percentage of the number of geometrically observable\n               reflections that satisfy the resolution limits.' ],
          [ 'item_', '_item.name', '_refine_ls_shell.percent_reflns_obs' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.percent_reflns_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections that satisfy the resolution limits\n               established by _refine_ls_shell.d_res_high and\n               _refine_ls_shell.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were used\n               as the test reflections (i.e. were excluded from the refinement)\n               when the refinement included the calculation of a \'free\'\n               R factor, expressed as a percentage of the number of\n               geometrically observable reflections that satisfy the\n               reflection limits.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_shell.percent_reflns_R_free' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.R_factor_all',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine_ls_shell.d_res_high and\n               _refine_ls_shell.d_res_low.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_ls_shell.R_factor_all' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine_ls_shell.wR_factor_all' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.R_factor_obs',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine_ls_shell.d_res_high and\n               _refine_ls_shell.d_res_low and the observation criterion\n               established by _reflns.observed_criterion.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_ls_shell.R_factor_obs' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine_ls_shell.wR_factor_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.R_factor_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine_ls_shell.d_res_high and\n               _refine_ls_shell.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were\n               used as the test reflections (i.e. were excluded from the\n               refinement) when the refinement included the calculation\n               of a \'free\' R factor. Details of how reflections were assigned\n               to the working and test sets are given in _reflns.R_free_details.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_ls_shell.R_factor_R_free' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_refine_ls_shell.wR_factor_R_free',
              'alternate',
              '_refine_ls_shell.R_factor_R_free_error',
              'associated_error' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.R_factor_R_free_error',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated error in _refine_ls_shell.R_factor_R_free.\n               The method used to estimate the error is described in the\n               item _refine.ls_R_factor_R_free_error_details.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_shell.R_factor_R_free_error' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine_ls_shell.R_factor_R_free' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.R_factor_R_work',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor R for reflections that satisfy the resolution\n               limits established by _refine_ls_shell.d_res_high and\n               _refine_ls_shell.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were used\n               as the working reflections (i.e. were included in the\n               refinement) when the refinement included the calculation of\n               a \'free\' R factor. Details of how reflections were assigned\n               to the working and test sets are given in _reflns.R_free_details.\n\n                   sum|F~obs~ - F~calc~|\n               R = ---------------------\n                        sum|F~obs~|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_ls_shell.R_factor_R_work' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine_ls_shell.wR_factor_R_work' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.redundancy_reflns_all',
        [ [ 'item_',
            '_item_description.description',
            '              The ratio of the total number of observations of the reflections\n               that satisfy the resolution limits established by\n               _refine_ls_shell.d_res_high and _refine_ls_shell.d_res_low\n               to the number of crystallographically unique reflections that\n               satisfy the same limits.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_shell.redundancy_reflns_all' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.redundancy_reflns_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The ratio of the total number of observations of the\n               reflections that satisfy the resolution limits established by\n               _refine_ls_shell.d_res_high and _refine_ls_shell.d_res_low and\n               the observation criterion established by\n               _reflns.observed_criterion to the number of crystallographically\n               unique reflections that satisfy the same limits.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_shell.redundancy_reflns_obs' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.wR_factor_all',
        [ [ 'item_',
            '_item_description.description',
            '              Weighted residual factor wR for reflections that satisfy the\n               resolution limits established by _refine_ls_shell.d_res_high\n               and _refine_ls_shell.d_res_low.\n\n                    ( sum|w |Y~obs~ - Y~calc~|^2^| )^1/2^\n               wR = ( ---------------------------- )\n                    (        sum|w Y~obs~^2^|      )\n\n               Y~obs~  = the observed amplitude specified by\n                         _refine.ls_structure_factor_coef\n               Y~calc~ = the calculated amplitude specified by\n                         _refine.ls_structure_factor_coef\n               w       = the least-squares weight\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_ls_shell.wR_factor_all' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine_ls_shell.R_factor_all' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.wR_factor_obs',
        [ [ 'item_',
            '_item_description.description',
            '              Weighted residual factor wR for reflections that satisfy the\n               resolution limits established by _refine_ls_shell.d_res_high\n               and _refine_ls_shell.d_res_low and the observation criterion\n               established by _reflns.observed_criterion.\n\n                    ( sum|w |Y~obs~ - Y~calc~|^2^| )^1/2^\n               wR = ( ---------------------------- )\n                    (        sum|w Y~obs~^2^|      )\n\n               Y~obs~  = the observed amplitude specified by\n                         _refine.ls_structure_factor_coef\n               Y~calc~ = the calculated amplitude specified by\n                         _refine.ls_structure_factor_coef\n               w       = the least-squares weight\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_ls_shell.wR_factor_obs' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine_ls_shell.R_factor_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.wR_factor_R_free',
        [ [ 'item_',
            '_item_description.description',
            '              Weighted residual factor wR for reflections that satisfy the\n               resolution limits established by _refine_ls_shell.d_res_high\n               and _refine_ls_shell.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were used\n               as the test reflections (i.e. were excluded from the refinement)\n               when the refinement included the calculation of a \'free\'\n               R factor. Details of how reflections were assigned to the\n               working and test sets are given in _reflns.R_free_details.\n\n                    ( sum|w |Y~obs~ - Y~calc~|^2^| )^1/2^\n               wR = ( ---------------------------- )\n                    (        sum|w Y~obs~^2^|      )\n\n               Y~obs~  = the observed amplitude specified by\n                         _refine.ls_structure_factor_coef\n               Y~calc~ = the calculated amplitude specified by\n                         _refine.ls_structure_factor_coef\n               w       = the least-squares weight\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_ls_shell.wR_factor_R_free' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine_ls_shell.R_factor_R_free' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine_ls_shell.wR_factor_R_work',
        [ [ 'item_',
            '_item_description.description',
            '              Weighted residual factor wR for reflections that satisfy the\n               resolution limits established by _refine_ls_shell.d_res_high\n               and _refine_ls_shell.d_res_low and the observation limit\n               established by _reflns.observed_criterion, and that were used\n               as the working reflections (i.e. were included in the\n               refinement) when the refinement included the calculation of\n               a \'free\' R factor. Details of how reflections were assigned\n               to the working and test sets are given in _reflns.R_free_details.\n\n                    ( sum|w |Y~obs~ - Y~calc~|^2^| )^1/2^\n               wR = ( ---------------------------- )\n                    (        sum|w Y~obs~^2^|      )\n\n               Y~obs~  = the observed amplitude specified by\n                         _refine.ls_structure_factor_coef\n               Y~calc~ = the calculated amplitude specified by\n                         _refine.ls_structure_factor_coef\n               w       = the least-squares weight\n\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine_ls_shell.wR_factor_R_work' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_refine_ls_shell.R_factor_R_work' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# REFINE_OCCUPANCY ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'refine_occupancy',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_OCCUPANCY category record details\n               about the treatment of atom occupancies during refinement.' ],
          [ 'item_', '_category.id', 'refine_occupancy' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_refine_occupancy.class' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _refine_occupancy.class\n    _refine_occupancy.treatment\n    _refine_occupancy.value\n    _refine_occupancy.details\n     \'protein\'                  fix  1.00  .\n     \'solvent\'                  fix  1.00  .\n     \'inhibitor orientation 1\'  fix  0.65  .\n     \'inhibitor orientation 2\'  fix  0.35\n    ; The inhibitor binds to the enzyme in two alternative\n      conformations. The occupancy of each conformation was\n      adjusted so as to result in approximately equal mean\n      thermal factors for the atoms in each conformation.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine_occupancy.class',
        [ [ 'item_',
            '_item_description.description',
            '              The class of atoms treated similarly for occupancy refinement.' ],
          [ 'item_', '_item.name', '_refine_occupancy.class' ],
          [ 'item_', '_item.category_id', 'refine_occupancy' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'all', 'protein', 'solvent', 'sugar-phosphate backbone' ] ] ] ],
      [ 'save_',
        '_refine_occupancy.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the occupancy refinement for\n               a class of atoms described in _refine_occupancy.class.' ],
          [ 'item_', '_item.name', '_refine_occupancy.details' ],
          [ 'item_', '_item.category_id', 'refine_occupancy' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The inhibitor binds to the enzyme in two\n                                  alternative conformations. The occupancy of\n                                  each conformation was adjusted so as to result\n                                  in approximately equal mean thermal factors\n                                  for the atoms in each conformation.' ] ] ],
      [ 'save_',
        '_refine_occupancy.treatment',
        [ [ 'item_',
            '_item_description.description',
            '              The treatment of occupancies for a class of atoms\n               described in _refine_occupancy.class.' ],
          [ 'item_', '_item.name', '_refine_occupancy.treatment' ],
          [ 'item_', '_item.category_id', 'refine_occupancy' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'fix', 'fixed', 'ref', 'refined' ] ] ] ],
      [ 'save_',
        '_refine_occupancy.value',
        [ [ 'item_',
            '_item_description.description',
            '              The value of occupancy assigned to a class of atoms defined in\n               _refine_occupancy.class. Meaningful only for atoms with fixed\n               occupancy.' ],
          [ 'item_', '_item.name', '_refine_occupancy.value' ],
          [ 'item_', '_item.category_id', 'refine_occupancy' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '1.0', '1.0', '1.0', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1.0', '0.41' ] ] ] ],
      [ '@comment', '##########' ],
      [ '@comment', '# REFLN ##' ],
      [ '@comment', '##########' ],
      [ 'save_',
        'refln',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFLN category record details about the\n               reflection data used to determine the ATOM_SITE data items.\n\n               The REFLN data items refer to individual reflections and must\n               be included in looped lists.\n\n               The REFLNS data items specify the parameters that apply to all\n               reflections. The REFLNS data items are not looped.' ],
          [ 'item_', '_category.id', 'refln' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_refln.index_h', '_refln.index_k', '_refln.index_l' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refln_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on data set fetod of Todres, Yanovsky, Ermekov & Struchkov\n                [Acta Cryst. (1993), C49, 1352-1354].',
              '\n    loop_\n    _refln.index_h\n    _refln.index_k\n    _refln.index_l\n    _refln.F_squared_calc\n    _refln.F_squared_meas\n    _refln.F_squared_sigma\n    _refln.status\n       2   0   0       85.57       58.90      1.45 o\n       3   0   0    15718.18    15631.06     30.40 o\n       4   0   0    55613.11    49840.09     61.86 o\n       5   0   0      246.85      241.86     10.02 o\n       6   0   0       82.16       69.97      1.93 o\n       7   0   0     1133.62      947.79     11.78 o\n       8   0   0     2558.04     2453.33     20.44 o\n       9   0   0      283.88      393.66      7.79 o\n      10   0   0      283.70      171.98      4.26 o',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refln.A_calc',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of structure-factor component A in\n               electrons.\n\n               A = |F|cos(phase)' ],
          [ 'item_', '_item.name', '_refln.A_calc' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_A_calc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_related.related_name', '_refln.A_calc_au' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_refln.A_calc_au',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of structure-factor component A in\n               arbitrary units.\n\n               A = |F|cos(phase)' ],
          [ 'item_', '_item.name', '_refln.A_calc_au' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_related.related_name', '_refln.A_calc' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_refln.A_meas',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of structure-factor component A in electrons.\n\n               A = |F|cos(phase)' ],
          [ 'item_', '_item.name', '_refln.A_meas' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_A_meas' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_related.related_name', '_refln.A_meas_au' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_refln.A_meas_au',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of structure-factor component A in\n               arbitrary units.\n\n               A = |F|cos(phase)' ],
          [ 'item_', '_item.name', '_refln.A_meas_au' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_related.related_name', '_refln.A_meas' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_refln.B_calc',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of structure-factor component B in\n               electrons.\n\n               B = |F|sin(phase)' ],
          [ 'item_', '_item.name', '_refln.B_calc' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_B_calc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_related.related_name', '_refln.B_calc_au' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_refln.B_calc_au',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of structure-factor component B in\n               arbitrary units.\n\n               B = |F|sin(phase)' ],
          [ 'item_', '_item.name', '_refln.B_calc_au' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_related.related_name', '_refln.B_calc' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_refln.B_meas',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of structure-factor component B in electrons.\n\n               B = |F|sin(phase)' ],
          [ 'item_', '_item.name', '_refln.B_meas' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_B_meas' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_related.related_name', '_refln.B_meas_au' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_refln.B_meas_au',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of structure-factor component B in\n               arbitrary units.\n\n               B = |F|sin(phase)' ],
          [ 'item_', '_item.name', '_refln.B_meas_au' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_related.related_name', '_refln.B_meas' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_refln.crystal_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _exptl_crystal.id in the\n               EXPTL_CRYSTAL category.' ],
          [ 'item_', '_item.name', '_refln.crystal_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_crystal_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_refln.F_calc',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of the structure factor in electrons.' ],
          [ 'item_', '_item.name', '_refln.F_calc' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_F_calc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_related.related_name', '_refln.F_calc_au' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_refln.F_calc_au',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of the structure factor in arbitrary\n               units.' ],
          [ 'item_', '_item.name', '_refln.F_calc_au' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_related.related_name', '_refln.F_calc' ],
          [ 'item_',
            '_item_related.function_code',
            'conversion_arbitrary' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_refln.F_meas',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of the structure factor in electrons.' ],
          [ 'item_', '_item.name', '_refln.F_meas' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_F_meas' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_refln.F_meas_sigma',
              'associated_esd',
              '_refln.F_meas_au',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_refln.F_meas_au',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of the structure factor in arbitrary units.' ],
          [ 'item_', '_item.name', '_refln.F_meas_au' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_refln.F_meas_sigma_au',
              'associated_esd',
              '_refln.F_meas',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_refln.F_meas_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation) of\n               _refln.F_meas in electrons.' ],
          [ 'item_', '_item.name', '_refln.F_meas_sigma' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_F_sigma' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_refln.F_meas',
              'associated_value',
              '_refln.F_meas_sigma_au',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons' ] ] ],
      [ 'save_',
        '_refln.F_meas_sigma_au',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation) of\n               _refln.F_meas_au in arbitrary units.' ],
          [ 'item_', '_item.name', '_refln.F_meas_sigma_au' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_refln.F_meas_au',
              'associated_value',
              '_refln.F_meas_sigma',
              'conversion_arbitrary' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_refln.F_squared_calc',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of the squared structure factor in\n               electrons squared.' ],
          [ 'item_', '_item.name', '_refln.F_squared_calc' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_F_squared_calc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons_squared' ] ] ],
      [ 'save_',
        '_refln.F_squared_meas',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of the squared structure factor in electrons\n               squared.' ],
          [ 'item_', '_item.name', '_refln.F_squared_meas' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_F_squared_meas' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons_squared' ] ] ],
      [ 'save_',
        '_refln.F_squared_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (derived from measurement) of the\n               squared structure factor in electrons squared.' ],
          [ 'item_', '_item.name', '_refln.F_squared_sigma' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_F_squared_sigma' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons_squared' ] ] ],
      [ 'save_',
        '_refln.fom',
        [ [ 'item_',
            '_item_description.description',
            '              The figure of merit m for this reflection.\n\n                   int P~alpha~ exp(i*alpha) dalpha\n               m = --------------------------------\n                         int P~alpha~ dalpha\n\n               P~a~ = the probability that the phase angle a is correct\n\n               int is taken over the range alpha = 0 to 2 pi.' ],
          [ 'item_', '_item.name', '_refln.fom' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.index_h',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index h of the reflection. The values of the Miller\n               indices in the REFLN category must correspond to the cell\n               defined by cell lengths and cell angles in the CELL category.' ],
          [ 'item_', '_item.name', '_refln.index_h' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_index_h' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_refln.index_k', '_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refln.index_k',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index k of the reflection. The values of the Miller\n               indices in the REFLN category must correspond to the cell\n               defined by cell lengths and cell angles in the CELL category.' ],
          [ 'item_', '_item.name', '_refln.index_k' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_index_k' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_refln.index_h', '_refln.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refln.index_l',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index l of the reflection. The values of the Miller\n               indices in the REFLN category must correspond to the cell\n               defined by cell lengths and cell angles in the CELL category.' ],
          [ 'item_', '_item.name', '_refln.index_l' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_index_l' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_refln.index_h', '_refln.index_k' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refln.intensity_calc',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of the intensity in the same units as\n               _refln.intensity_meas.' ],
          [ 'item_', '_item.name', '_refln.intensity_calc' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_intensity_calc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.intensity_meas',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of the intensity.' ],
          [ 'item_', '_item.name', '_refln.intensity_meas' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_intensity_meas' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.intensity_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (derived from measurement) of the\n               intensity in the same units as _refln.intensity_meas.' ],
          [ 'item_', '_item.name', '_refln.intensity_sigma' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_intensity_sigma' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', 'save__refln.mean_path_length_tbar' ],
      [ '@comment', '    _item_description.description' ],
      [ '@comment',
        ';              Mean path length in millimetres through the crystal for this' ],
      [ '@comment', '               reflection.' ],
      [ '@comment', ';' ],
      [ '@comment',
        '    _item.name                  \'_refln.mean_path_length_tbar\'' ],
      [ '@comment', '    _item.category_id             refln' ],
      [ '@comment', '    _item.mandatory_code          no' ],
      [ '@comment',
        '    _item_aliases.alias_name    \'_refln_mean_path_length_tbar\'' ],
      [ '@comment', '    _item_aliases.dictionary      cif_core.dic' ],
      [ '@comment', '    _item_aliases.version         2.0.1' ],
      [ '@comment', '    _item_type.code               float' ],
      [ '@comment', '    _item_units.code              millimetres' ],
      [ '@comment', '     save_' ],
      [ 'save_',
        '_refln.status',
        [ [ 'item_',
            '_item_description.description',
            '              Classification of a reflection so as to indicate its status with\n               respect to inclusion in the refinement and the calculation of\n               R factors.' ],
          [ 'item_', '_item.name', '_refln.status' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_observed_status' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ '@comment', '    _item_default.value           o' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'o',
              '                                     satisfies _refine.ls_d_res_high,\n                                      satisfies _refine.ls_d_res_low,\n                                      observed by _reflns.observed_criterion,\n                                      not flagged as systematically absent,\n                                      not flagged as unreliable',
              '<',
              '                                     satisfies _refine.ls_d_res_high,\n                                      satisfies _refine.ls_d_res_low,\n                                      unobserved by _reflns.observed_criterion,\n                                      not flagged as systematically absent,\n                                      not flagged as unreliable',
              '-',
              'systematically absent reflection',
              'x',
              'unreliable measurement -- not used',
              'h',
              'does not satisfy _refine.ls_d_res_high',
              'l',
              'does not satisfy _refine.ls_d_res_low',
              'f',
              '                                     satisfies _refine.ls_d_res_high,\n                                      satisfies _refine.ls_d_res_low,\n                                      observed by _reflns.observed_criterion,\n                                      not flagged as systematically absent,\n                                      not flagged as unreliable,\n                                      excluded from refinement so as to be\n                                      included in the calculation of a \'free\' R\n                                      factor' ] ] ] ],
      [ 'save_',
        '_refln.phase_calc',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated structure-factor phase in degrees.' ],
          [ 'item_', '_item.name', '_refln.phase_calc' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_phase_calc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_refln.phase_meas',
        [ [ 'item_',
            '_item_description.description',
            '              The measured structure-factor phase in degrees.' ],
          [ 'item_', '_item.name', '_refln.phase_meas' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_phase_meas' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_refln.refinement_status',
        [ [ 'item_',
            '_item_description.description',
            '              Status of a reflection in the structure-refinement process.' ],
          [ 'item_', '_item.name', '_refln.refinement_status' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refln_refinement_status' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_default.value', 'incl' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'incl',
              'included in ls process',
              'excl',
              'excluded from ls process',
              'extn',
              'excluded due to extinction' ] ] ] ],
      [ 'save_',
        '_refln.scale_group_code',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _reflns_scale.group_code in the\n               REFLNS_SCALE category.' ],
          [ 'item_', '_item.name', '_refln.scale_group_code' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refln_scale_group_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ 'save_',
        '_refln.sint_over_lambda',
        [ [ 'item_',
            '_item_description.description',
            '              The (sin theta)/lambda value in reciprocal angstroms for this\n               reflection.' ],
          [ 'item_', '_item.name', '_refln.sint_over_lambda' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_sint/lambda' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'reciprocal_angstroms' ] ] ],
      [ 'save_',
        '_refln.symmetry_epsilon',
        [ [ 'item_',
            '_item_description.description',
            '              The symmetry reinforcement factor corresponding to the number of\n               times the reflection indices are generated identically from the\n               space-group symmetry operations.' ],
          [ 'item_', '_item.name', '_refln.symmetry_epsilon' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refln_symmetry_epsilon' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '48', '48', '48', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refln.symmetry_multiplicity',
        [ [ 'item_',
            '_item_description.description',
            '              The number of symmetry-equivalent reflections. The equivalent\n               reflections have the same structure-factor magnitudes because\n               of the space-group symmetry and the Friedel relationship.' ],
          [ 'item_', '_item.name', '_refln.symmetry_multiplicity' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refln_symmetry_multiplicity' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '48', '48', '48', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refln.wavelength',
        [ [ 'item_',
            '_item_description.description',
            '              The mean wavelength in angstroms of radiation used to measure\n               this reflection. This is an important parameter for data\n               collected using energy-dispersive detectors or the Laue\n               method.' ],
          [ 'item_', '_item.name', '_refln.wavelength' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_wavelength' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refln.wavelength_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _diffrn_radiation.wavelength_id in\n               the DIFFRN_RADIATION category.' ],
          [ 'item_', '_item.name', '_refln.wavelength_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_refln_wavelength_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ] ] ],
      [ '@comment', '##################' ],
      [ '@comment', '# REFLN_SYS_ABS ##' ],
      [ '@comment', '##################' ],
      [ 'save_',
        'refln_sys_abs',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFLN_SYS_ABS category record details about\n               the reflection data that should be systematically absent,\n               given the designated space group.' ],
          [ 'item_', '_category.id', 'refln_sys_abs' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_refln_sys_abs.index_h',
              '_refln_sys_abs.index_k',
              '_refln_sys_abs.index_l' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refln_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - hypothetical example.',
              '\n    loop_\n    _refln_sys_abs.index_h\n    _refln_sys_abs.index_k\n    _refln_sys_abs.index_l\n    _refln_sys_abs.I\n    _refln_sys_abs.sigmaI\n    _refln_sys_abs.I_over_sigmaI\n     0  3  0   28.32  22.95  1.23\n     0  5  0   14.11  16.38  0.86\n     0  7  0  114.81  20.22  5.67\n     0  9  0   32.99  24.51  1.35',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refln_sys_abs.I',
        [ [ 'item_',
            '_item_description.description',
            '              The measured value of the intensity in arbitrary units.' ],
          [ 'item_', '_item.name', '_refln_sys_abs.I' ],
          [ 'item_', '_item.category_id', 'refln_sys_abs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_ebi_refln_sys_abs.I' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_refln_sys_abs.sigmaI', 'associated_esd' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_refln_sys_abs.I_over_sigmaI',
        [ [ 'item_',
            '_item_description.description',
            '              The ratio of _refln_sys_abs.I to _refln_sys_abs.sigmaI.  Used\n               to evaluate whether a reflection that should be systematically\n               absent according to the designated space group is in fact\n               absent.' ],
          [ 'item_', '_item.name', '_refln_sys_abs.I_over_sigmaI' ],
          [ 'item_', '_item.category_id', 'refln_sys_abs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ebi_refln_sys_abs.I_over_sigma' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln_sys_abs.index_h',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index h of the reflection. The values of the Miller\n               indices in the REFLN_SYS_ABS category must correspond to\n               the cell defined by cell lengths and cell angles in the CELL\n               category.' ],
          [ 'item_', '_item.name', '_refln_sys_abs.index_h' ],
          [ 'item_', '_item.category_id', 'refln_sys_abs' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_ebi_refln_sys_abs.h' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_refln_sys_abs.index_k', '_refln_sys_abs.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refln_sys_abs.index_k',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index k of the reflection. The values of the Miller\n               indices in the REFLN_SYS_ABS category must correspond to the\n               cell defined by cell lengths and cell angles in the CELL\n               category.' ],
          [ 'item_', '_item.name', '_refln_sys_abs.index_k' ],
          [ 'item_', '_item.category_id', 'refln_sys_abs' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_ebi_refln_sys_abs.k' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_refln_sys_abs.index_h', '_refln_sys_abs.index_l' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refln_sys_abs.index_l',
        [ [ 'item_',
            '_item_description.description',
            '              Miller index l of the reflection. The values of the Miller\n               indices in the REFLN_SYS_ABS category must correspond to the\n               cell defined by cell lengths and cell angles in the CELL\n               category.' ],
          [ 'item_', '_item.name', '_refln_sys_abs.index_l' ],
          [ 'item_', '_item.category_id', 'refln_sys_abs' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_aliases.alias_name', '_ebi_refln_sys_abs.l' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_refln_sys_abs.index_h', '_refln_sys_abs.index_k' ] ],
          [ 'item_', '_item_sub_category.id', 'miller_index' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refln_sys_abs.sigmaI',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (estimated standard deviation) of\n               _refln_sys_abs.I in arbitrary units.' ],
          [ 'item_', '_item.name', '_refln_sys_abs.sigmaI' ],
          [ 'item_', '_item.category_id', 'refln_sys_abs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ebi_refln_sys_abs.sigmaI' ],
          [ 'item_', '_item_aliases.dictionary', 'ebi_extensions' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_refln_sys_abs.I', 'associated_value' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ '@comment', '###########' ],
      [ '@comment', '# REFLNS ##' ],
      [ '@comment', '###########' ],
      [ 'save_',
        'reflns',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFLNS category record details about the\n               reflection data used to determine the ATOM_SITE data items.\n\n               The REFLN data items refer to individual reflections and must\n               be included in looped lists.\n\n               The REFLNS data items specify the parameters that apply to all\n               reflections. The REFLNS data items are not looped.' ],
          [ 'item_', '_category.id', 'reflns' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_reflns.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refln_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _reflns.entry_id                       \'5HVP\'\n    _reflns.data_reduction_method\n    ; Xengen program scalei. Anomalous pairs were merged. Scaling\n      proceeded in several passes, beginning with 1-parameter\n      fit and ending with 3-parameter fit.\n    ;\n    _reflns.data_reduction_details\n    ; Merging and scaling based on only those reflections\n      with I > \\s(I).\n    ;\n\n    _reflns.d_resolution_high              2.00\n    _reflns.d_resolution_low               8.00\n\n    _reflns.limit_h_max                    22\n    _reflns.limit_h_min                    0\n    _reflns.limit_k_max                    46\n    _reflns.limit_k_min                    0\n    _reflns.limit_l_max                    57\n    _reflns.limit_l_min                    0\n\n    _reflns.number_obs                     7228\n    _reflns.observed_criterion             \'> 1 \\s(I)\'\n    _reflns.details                        none',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - based on data set TOZ of Willis, Beckwith & Tozer\n                 [Acta Cryst. (1991), C47, 2276-2277].',
              '\n    _reflns.limit_h_min                0\n    _reflns.limit_h_max                6\n    _reflns.limit_k_min                0\n    _reflns.limit_k_max                17\n    _reflns.limit_l_min                0\n    _reflns.limit_l_max                22\n    _reflns.number_all                 1592\n    _reflns.number_obs                 1408\n    _reflns.observed_criterion         F_>_6.0_\\s(F)\n    _reflns.d_resolution_high          0.8733\n    _reflns.d_resolution_low           11.9202',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_reflns.B_iso_Wilson_estimate',
        [ [ 'item_',
            '_item_description.description',
            '              The value of the overall isotropic displacement parameter\n               estimated from the slope of the Wilson plot.' ],
          [ 'item_', '_item.name', '_reflns.B_iso_Wilson_estimate' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ] ] ],
      [ 'save_',
        '_reflns.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_reflns.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_reflns.data_reduction_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the data-reduction\n               procedures.' ],
          [ 'item_', '_item.name', '_reflns.data_reduction_details' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Merging and scaling based on only those\n                                  reflections with I > sig(I).' ] ] ],
      [ 'save_',
        '_reflns.data_reduction_method',
        [ [ 'item_',
            '_item_description.description',
            '              The method used for data reduction.\n\n               Note that this is not the computer program used, which is\n               described in the SOFTWARE category, but the method\n               itself.\n\n               This data item should be used to describe significant\n               methodological options used within the data-reduction programs.' ],
          [ 'item_', '_item.name', '_reflns.data_reduction_method' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Profile fitting by method of Kabsch (1987).\n                                  Scaling used spherical harmonic coefficients.' ] ] ],
      [ 'save_',
        '_reflns.d_resolution_high',
        [ [ 'item_',
            '_item_description.description',
            '              The smallest value for the interplanar spacings for\n               the reflection data. This is called the highest resolution.' ],
          [ 'item_', '_item.name', '_reflns.d_resolution_high' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_d_resolution_high' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_reflns.d_resolution_low',
        [ [ 'item_',
            '_item_description.description',
            '              The largest value for the interplanar spacings for the\n               reflection data. This is called the lowest resolution.' ],
          [ 'item_', '_item.name', '_reflns.d_resolution_low' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_d_resolution_low' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_reflns.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of reflection data not covered by other data\n               names. This should include details of the Friedel pairs.' ],
          [ 'item_', '_item.name', '_reflns.details' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_special_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_reflns.limit_h_max',
        [ [ 'item_',
            '_item_description.description',
            '              Maximum value of the Miller index h for the reflection data. This\n               need not have the same value as _diffrn_reflns.limit_h_max.' ],
          [ 'item_', '_item.name', '_reflns.limit_h_max' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_reflns_limit_h_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns.limit_h_min',
        [ [ 'item_',
            '_item_description.description',
            '              Minimum value of the Miller index h for the reflection data. This\n               need not have the same value as _diffrn_reflns.limit_h_min.' ],
          [ 'item_', '_item.name', '_reflns.limit_h_min' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_reflns_limit_h_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns.limit_k_max',
        [ [ 'item_',
            '_item_description.description',
            '              Maximum value of the Miller index k for the reflection data. This\n               need not have the same value as _diffrn_reflns.limit_k_max.' ],
          [ 'item_', '_item.name', '_reflns.limit_k_max' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_reflns_limit_k_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns.limit_k_min',
        [ [ 'item_',
            '_item_description.description',
            '              Minimum value of the Miller index k for the reflection data. This\n               need not have the same value as _diffrn_reflns.limit_k_min.' ],
          [ 'item_', '_item.name', '_reflns.limit_k_min' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_reflns_limit_k_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns.limit_l_max',
        [ [ 'item_',
            '_item_description.description',
            '              Maximum value of the Miller index l for the reflection data. This\n               need not have the same value as _diffrn_reflns.limit_l_max.' ],
          [ 'item_', '_item.name', '_reflns.limit_l_max' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_reflns_limit_l_max' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns.limit_l_min',
        [ [ 'item_',
            '_item_description.description',
            '              Minimum value of the Miller index l for the reflection data. This\n               need not have the same value as _diffrn_reflns.limit_l_min.' ],
          [ 'item_', '_item.name', '_reflns.limit_l_min' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_reflns_limit_l_min' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns.number_all',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of reflections in the REFLN list (not the\n               DIFFRN_REFLN list). This number may contain Friedel-equivalent\n               reflections according to the nature of the structure and the\n               procedures used. The item _reflns.details describes the\n               reflection data.' ],
          [ 'item_', '_item.name', '_reflns.number_all' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_reflns_number_total' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns.number_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections in the REFLN list (not the DIFFRN_REFLN\n               list) classified as observed (see _reflns.observed_criterion).\n               This number may contain Friedel-equivalent reflections according\n               to the nature of the structure and the procedures used.' ],
          [ 'item_', '_item.name', '_reflns.number_obs' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_number_observed' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns.observed_criterion',
        [ [ 'item_',
            '_item_description.description',
            '              The criterion used to classify a reflection as \'observed\'. This\n               criterion is usually expressed in terms of a sigma(I) or\n               sigma(F) threshold.' ],
          [ 'item_', '_item.name', '_reflns.observed_criterion' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_observed_criterion' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_reflns.observed_criterion_sigma_F',
              'alternate',
              '_reflns.observed_criterion_sigma_I',
              'alternate',
              '_reflns.observed_criterion_I_min',
              'alternate',
              '_reflns.observed_criterion_I_max',
              'alternate',
              '_reflns.observed_criterion_F_min',
              'alternate',
              '_reflns.observed_criterion_F_max',
              'alternate' ] ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', '>2sigma(I)' ] ] ],
      [ 'save_',
        '_reflns.observed_criterion_F_max',
        [ [ 'item_',
            '_item_description.description',
            '              The criterion used to classify a reflection as \'observed\'\n               expressed as an upper limit for the value of F.' ],
          [ 'item_', '_item.name', '_reflns.observed_criterion_F_max' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_reflns.observed_criterion',
              'alternate',
              '_reflns.observed_criterion_I_max',
              'convention' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns.observed_criterion_F_min',
        [ [ 'item_',
            '_item_description.description',
            '              The criterion used to classify a reflection as \'observed\'\n               expressed as a lower limit for the value of F.' ],
          [ 'item_', '_item.name', '_reflns.observed_criterion_F_min' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_reflns.observed_criterion',
              'alternate',
              '_reflns.observed_criterion_I_min',
              'convention' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns.observed_criterion_I_max',
        [ [ 'item_',
            '_item_description.description',
            '              The criterion used to classify a reflection as \'observed\'\n               expressed as an upper limit for the value of I.' ],
          [ 'item_', '_item.name', '_reflns.observed_criterion_I_max' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_reflns.observed_criterion',
              'alternate',
              '_reflns.observed_criterion_F_max',
              'convention' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns.observed_criterion_I_min',
        [ [ 'item_',
            '_item_description.description',
            '              The criterion used to classify a reflection as \'observed\'\n               expressed as a lower limit for the value of I.' ],
          [ 'item_', '_item.name', '_reflns.observed_criterion_I_min' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_reflns.observed_criterion',
              'alternate',
              '_reflns.observed_criterion_F_min',
              'convention' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns.observed_criterion_sigma_F',
        [ [ 'item_',
            '_item_description.description',
            '              The criterion used to classify a reflection as \'observed\'\n               expressed as a multiple of the value of sigma(F).' ],
          [ 'item_', '_item.name', '_reflns.observed_criterion_sigma_F' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_reflns.observed_criterion',
              'alternate',
              '_reflns.observed_criterion_sigma_I',
              'convention' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns.observed_criterion_sigma_I',
        [ [ 'item_',
            '_item_description.description',
            '              The criterion used to classify a reflection as \'observed\'\n               expressed as a multiple of the value of sigma(I).' ],
          [ 'item_', '_item.name', '_reflns.observed_criterion_sigma_I' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_reflns.observed_criterion',
              'alternate',
              '_reflns.observed_criterion_sigma_F',
              'convention' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns.percent_possible_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The percentage of geometrically possible reflections represented\n               by reflections that satisfy the resolution limits established\n               by _reflns.d_resolution_high and _reflns.d_resolution_low and\n               the observation limit established by\n               _reflns.observed_criterion.' ],
          [ 'item_', '_item.name', '_reflns.percent_possible_obs' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns.R_free_details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the method by which a subset of reflections was\n               selected for exclusion from refinement so as to be used in the\n               calculation of a \'free\' R factor.' ],
          [ 'item_', '_item.name', '_reflns.R_free_details' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The data set was sorted with l varying most\n                                  rapidly and h varying least rapidly. Every\n                                  10th reflection in this sorted list was\n                                  excluded from refinement and included in the\n                                  calculation of a \'free\' R factor.' ] ] ],
      [ 'save_',
        '_reflns.Rmerge_F_all',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor Rmerge for all reflections that satisfy the\n               resolution limits established by _reflns.d_resolution_high\n               and _reflns.d_resolution_low.\n\n                           sum~i~(sum~j~|F~j~ - <F>|)\n               Rmerge(F) = --------------------------\n                                sum~i~(sum~j~<F>)\n\n               F~j~ = the amplitude of the jth observation of reflection i\n               <F>  = the mean of the amplitudes of all observations of\n                      reflection i\n\n               sum~i~ is taken over all reflections\n               sum~j~ is taken over all observations of each reflection' ],
          [ 'item_', '_item.name', '_reflns.Rmerge_F_all' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns.Rmerge_F_obs',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor Rmerge for reflections that satisfy the\n               resolution limits established by _reflns.d_resolution_high\n               and _reflns.d_resolution_low and the observation limit\n               established by _reflns.observed_criterion.\n\n                           sum~i~(sum~j~|F~j~ - <F>|)\n               Rmerge(F) = --------------------------\n                                sum~i~(sum~j~<F>)\n\n               F~j~ = the amplitude of the jth observation of reflection i\n               <F>  = the mean of the amplitudes of all observations of\n                      reflection i\n\n               sum~i~ is taken over all reflections\n               sum~j~ is taken over all observations of each reflection' ],
          [ 'item_', '_item.name', '_reflns.Rmerge_F_obs' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#################' ],
      [ '@comment', '# REFLNS_SCALE ##' ],
      [ '@comment', '#################' ],
      [ 'save_',
        'reflns_scale',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFLNS_SCALE category record details about\n               the structure-factor scales. They are referenced from within\n               the REFLN list through _refln.scale_group_code.' ],
          [ 'item_', '_category.id', 'reflns_scale' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_reflns_scale.group_code' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refln_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for the collagen-like\n                peptide [(POG)4 EKG (POG)5]3.',
              '\n    _reflns_scale.group_code     SG1\n    _reflns_scale.meas_F         4.0',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_reflns_scale.group_code',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying a scale _reflns_scale.meas_F,\n               _reflns_scale.meas_F_squared or _reflns_scale.meas_intensity.\n               These are linked to the REFLN list by the\n               _refln.scale_group_code. These codes\n               need not correspond to those in the DIFFRN_SCALE list.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_reflns_scale.group_code',
              'reflns_scale',
              'yes',
              '_refln.scale_group_code',
              'refln',
              'yes' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_scale_group_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_refln.scale_group_code', '_reflns_scale.group_code' ] ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', 'c1', 'c2' ] ] ] ],
      [ 'save_',
        '_reflns_scale.meas_F',
        [ [ 'item_',
            '_item_description.description',
            '              A scale associated with _reflns_scale.group_code.' ],
          [ 'item_', '_item.name', '_reflns_scale.meas_F' ],
          [ 'item_', '_item.category_id', 'reflns_scale' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_reflns_scale_meas_F' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns_scale.meas_F_squared',
        [ [ 'item_',
            '_item_description.description',
            '              A scale associated with _reflns_scale.group_code.' ],
          [ 'item_', '_item.name', '_reflns_scale.meas_F_squared' ],
          [ 'item_', '_item.category_id', 'reflns_scale' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_scale_meas_F_squared' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns_scale.meas_intensity',
        [ [ 'item_',
            '_item_description.description',
            '              A scale associated with _reflns_scale.group_code.' ],
          [ 'item_', '_item.name', '_reflns_scale.meas_intensity' ],
          [ 'item_', '_item.category_id', 'reflns_scale' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_scale_meas_intensity' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#################' ],
      [ '@comment', '# REFLNS_SHELL ##' ],
      [ '@comment', '#################' ],
      [ 'save_',
        'reflns_shell',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFLNS_SHELL category record details about\n               the reflection data used to determine the ATOM_SITE data items\n               broken down into shells of resolution.' ],
          [ 'item_', '_category.id', 'reflns_shell' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_reflns_shell.d_res_high', '_reflns_shell.d_res_low' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refln_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _reflns_shell.d_res_high\n    _reflns_shell.d_res_low\n    _reflns_shell.meanI_over_sigI_obs\n    _reflns_shell.number_measured_obs\n    _reflns_shell.number_unique_obs\n    _reflns_shell.percent_possible_obs\n    _reflns_shell.Rmerge_F_obs\n      31.38  3.82  69.8  9024  2540  96.8   1.98\n       3.82  3.03  26.1  7413  2364  95.1   3.85\n       3.03  2.65  10.5  5640  2123  86.2   6.37\n       2.65  2.41   6.4  4322  1882  76.8   8.01\n       2.41  2.23   4.3  3247  1714  70.4   9.86\n       2.23  2.10   3.1  1140   812  33.3  13.99',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_reflns_shell.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The smallest value in angstroms for the interplanar spacings\n               for the reflections in this shell. This is called the highest\n               resolution.' ],
          [ 'item_', '_item.name', '_reflns_shell.d_res_high' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_d_res_high' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_reflns_shell.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The highest value in angstroms for the interplanar spacings\n               for the reflections in this shell. This is called the lowest\n               resolution.' ],
          [ 'item_', '_item.name', '_reflns_shell.d_res_low' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_d_res_low' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_reflns_shell.meanI_over_sigI_all',
        [ [ 'item_',
            '_item_description.description',
            '              The ratio of the mean of the intensities of all reflections\n               in this shell to the mean of the standard uncertainties of the\n               intensities of all reflections in this shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.meanI_over_sigI_all' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_meanI_over_sigI_all' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns_shell.meanI_over_sigI_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The ratio of the mean of the intensities of the reflections\n               classified as \'observed\' (see _reflns.observed_criterion) in\n               this shell to the mean of the standard uncertainties of the\n               intensities of the \'observed\' reflections in this\n               shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.meanI_over_sigI_obs' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_meanI_over_sigI_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns_shell.number_measured_all',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of reflections measured for this\n               shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.number_measured_all' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_number_measured_all' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns_shell.number_measured_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections classified as \'observed\'\n               (see _reflns.observed_criterion) for this\n               shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.number_measured_obs' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_number_measured_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns_shell.number_possible',
        [ [ 'item_',
            '_item_description.description',
            '              The number of unique reflections it is possible to measure in\n               this shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.number_possible' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_number_possible' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns_shell.number_unique_all',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of measured reflections which are symmetry-\n               unique after merging for this shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.number_unique_all' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_number_unique_all' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns_shell.number_unique_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of measured reflections classified as \'observed\'\n               (see _reflns.observed_criterion) which are symmetry-unique\n               after merging for this shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.number_unique_obs' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_number_unique_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_reflns_shell.percent_possible_all',
        [ [ 'item_',
            '_item_description.description',
            '              The percentage of geometrically possible reflections represented\n               by all reflections measured for this shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.percent_possible_all' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_percent_possible_all' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns_shell.percent_possible_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The percentage of geometrically possible reflections represented\n               by reflections classified as \'observed\' (see\n               _reflns.observed_criterion) for this shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.percent_possible_obs' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_percent_possible_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns_shell.Rmerge_F_all',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor Rmerge for all reflections that satisfy the\n               resolution limits established by _reflns_shell.d_res_high and\n               _reflns_shell.d_res_low.\n\n                           sum~i~(sum~j~|F~j~ - <F>|)\n               Rmerge(F) = --------------------------\n                                sum~i~(sum~j~<F>)\n\n               F~j~ = the amplitude of the jth observation of reflection i\n               <F>  = the mean of the amplitudes of all observations of\n                      reflection i\n\n               sum~i~ is taken over all reflections\n               sum~j~ is taken over all observations of each reflection' ],
          [ 'item_', '_item.name', '_reflns_shell.Rmerge_F_all' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_Rmerge_F_all' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns_shell.Rmerge_F_obs',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor Rmerge for reflections that satisfy the\n               resolution limits established by _reflns_shell.d_res_high and\n               _reflns_shell.d_res_low and the observation criterion\n               established by _reflns.observed_criterion.\n\n                           sum~i~(sum~j~|F~j~ - <F>|)\n               Rmerge(F) = --------------------------\n                                sum~i~(sum~j~<F>)\n\n               F~j~ = the amplitude of the jth observation of reflection i\n               <F>  = the mean of the amplitudes of all observations of\n                      reflection i\n\n               sum~i~ is taken over all reflections\n               sum~j~ is taken over all observations of each reflection' ],
          [ 'item_', '_item.name', '_reflns_shell.Rmerge_F_obs' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_Rmerge_F_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns_shell.Rmerge_I_all',
        [ [ 'item_',
            '_item_description.description',
            '              The value of Rmerge(I) for all reflections in a given shell.\n\n                           sum~i~(sum~j~|I~j~ - <I>|)\n               Rmerge(I) = --------------------------\n                               sum~i~(sum~j~<I>)\n\n               I~j~ = the intensity of the jth observation of reflection i\n               <I>  = the mean of the intensities of all observations of\n                      reflection i\n\n               sum~i~ is taken over all reflections\n               sum~j~ is taken over all observations of each reflection\n' ],
          [ 'item_', '_item.name', '_reflns_shell.Rmerge_I_all' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_Rmerge_I_all' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns_shell.Rmerge_I_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The value of Rmerge(I) for reflections classified as \'observed\'\n               (see _reflns.observed_criterion) in a given shell.\n\n                           sum~i~(sum~j~|I~j~ - <I>|)\n               Rmerge(I) = --------------------------\n                               sum~i~(sum~j~<I>)\n\n               I~j~ = the intensity of the jth observation of reflection i\n               <I>  = the mean of the intensities of all observations of\n                      reflection i\n\n               sum~i~ is taken over all reflections\n               sum~j~ is taken over all observations of each reflection' ],
          [ 'item_', '_item.name', '_reflns_shell.Rmerge_I_obs' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell_Rmerge_I_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#############' ],
      [ '@comment', '# SOFTWARE ##' ],
      [ '@comment', '#############' ],
      [ 'save_',
        'software',
        [ [ 'item_',
            '_category.description',
            '              Data items in the SOFTWARE category record details about\n               the software used in the structure analysis, which implies\n               any software used in the generation of any data items\n               associated with the structure determination and\n               structure representation.\n\n               These data items allow computer programs to be referenced\n               in more detail than data items in the COMPUTING category do.' ],
          [ 'item_', '_category.id', 'software' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_software.name', '_software.version' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'computing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _software.name\n    _software.version\n    _software.date\n    _software.type\n    _software.contact_author\n    _software.contact_author_email\n    _software.location\n    _software.classification\n    _software.citation_id\n    _software.language\n    _software.compiler_name\n    _software.compiler_version\n    _software.hardware\n    _software.os\n    _software.os_version\n    _software.dependencies\n    _software.mods\n    _software.description\n       Prolsq   unknown  .   program   \'Wayne A. Hendrickson\'  ?\n       \'ftp://rosebud.sdsc.edu/pub/sdsc/xtal/CCP4/ccp4/\'\n       refinement ref5  Fortran\n       \'Convex Fortran\' v8.0 \'Convex C220\' ConvexOS  v10.1\n       \'Requires that Protin be run first\'    optimized\n       \'restrained least-squares refinement\'' ] ] ] ],
      [ '@comment',
        ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
      [ 'save_',
        '_software.citation_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _citation.id in the CITATION\n               category.' ],
          [ 'item_', '_item.name', '_software.citation_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_software.classification',
        [ [ 'item_',
            '_item_description.description',
            '              The classification of the program according to its\n               major function.' ],
          [ 'item_', '_item.name', '_software.classification' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uline' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'data collection',
              'data reduction',
              'phasing',
              'model building',
              'refinement',
              'validation',
              'other' ] ] ] ],
      [ 'save_',
        '_software.compiler_name',
        [ [ 'item_',
            '_item_description.description',
            '              The compiler used to compile the software.' ],
          [ 'item_', '_item.name', '_software.compiler_name' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Convex Fortran', 'gcc', 'DEC C' ] ] ] ],
      [ 'save_',
        '_software.compiler_version',
        [ [ 'item_',
            '_item_description.description',
            '              The version of the compiler used to compile the software.' ],
          [ 'item_', '_item.name', '_software.compiler_version' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '3.1', '2.1 alpha' ] ] ] ],
      [ 'save_',
        '_software.contact_author',
        [ [ 'item_',
            '_item_description.description',
            '              The recognized contact author of the software. This could be\n               the original author, someone who has modified the code or\n               someone who maintains the code.  It should be the person\n               most commonly associated with the code.' ],
          [ 'item_', '_item.name', '_software.contact_author' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'T. Alwyn Jones', 'Axel Brunger' ] ] ] ],
      [ 'save_',
        '_software.contact_author_email',
        [ [ 'item_',
            '_item_description.description',
            '              The e-mail address of the person specified in\n               _software.contact_author.' ],
          [ 'item_', '_item.name', '_software.contact_author_email' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'bourne@sdsc.edu' ] ] ],
      [ 'save_',
        '_software.date',
        [ [ 'item_',
            '_item_description.description',
            '              The date the software was released.' ],
          [ 'item_', '_item.name', '_software.date' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '1991-10-01', '1990-04-30' ] ] ] ],
      [ 'save_',
        '_software.description',
        [ [ 'item_',
            '_item_description.description',
            '              Description of the software.' ],
          [ 'item_', '_item.name', '_software.description' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_examples.case',
            'Uses method of restrained least squares' ] ] ],
      [ 'save_',
        '_software.dependencies',
        [ [ 'item_',
            '_item_description.description',
            '              Any prerequisite software required to run _software.name.' ],
          [ 'item_', '_item.name', '_software.dependencies' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'PDBlib class library' ] ] ],
      [ 'save_',
        '_software.hardware',
        [ [ 'item_',
            '_item_description.description',
            '              The hardware upon which the software was run.' ],
          [ 'item_', '_item.name', '_software.hardware' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Sun Sparc 10 model 41',
              'Dec Alpha 3000 model 500S',
              'Silicon Graphics Elan',
              'Compaq PC 486/66' ] ] ] ],
      [ 'save_',
        '_software.language',
        [ [ 'item_',
            '_item_description.description',
            '              The major computing language in which the software is\n               coded.' ],
          [ 'item_', '_item.name', '_software.language' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uline' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'Ada',
              'assembler',
              'Awk',
              'Basic',
              'C++',
              'C/C++',
              'C',
              'csh',
              'Fortran',
              'Fortran_77',
              'Fortran 77',
              'Fortran 90',
              'Java',
              'ksh',
              'Pascal',
              'Perl',
              'Python',
              'sh',
              'Tcl',
              'Other' ] ] ] ],
      [ 'save_',
        '_software.location',
        [ [ 'item_',
            '_item_description.description',
            '              The URL for an Internet address at which\n               details of the software can be found.' ],
          [ 'item_', '_item.name', '_software.location' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'http://rosebud.sdsc.edu/projects/pb/IUCr/software.html',
              'ftp://ftp.sdsc.edu/pub/sdsc/biology/' ] ] ] ],
      [ 'save_',
        '_software.mods',
        [ [ 'item_',
            '_item_description.description',
            '              Any noteworthy modifications to the base software, if applicable.' ],
          [ 'item_', '_item.name', '_software.mods' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_examples.case',
            'Added support for space group F432' ] ] ],
      [ 'save_',
        '_software.name',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the software.' ],
          [ 'item_', '_item.name', '_software.name' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Merlot', 'O', 'Xengen', 'X-plor' ] ] ] ],
      [ 'save_',
        '_software.os',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the operating system under which the software\n               runs.' ],
          [ 'item_', '_item.name', '_software.os' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Ultrix',
              'OpenVMS',
              'DOS',
              'Windows 95',
              'Windows NT',
              'Irix',
              'HPUX',
              'DEC Unix' ] ] ] ],
      [ 'save_',
        '_software.os_version',
        [ [ 'item_',
            '_item_description.description',
            '              The version of the operating system under which the software\n               runs.' ],
          [ 'item_', '_item.name', '_software.os_version' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ '3.1', '4.2.1' ] ] ] ],
      [ 'save_',
        '_software.type',
        [ [ 'item_',
            '_item_description.description',
            '              The classification of the software according to the most\n               common types.' ],
          [ 'item_', '_item.name', '_software.type' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uline' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'program',
              '                                 individual program with limited\n                                  functionality',
              'library',
              '                                 used by a program at load time',
              'package',
              '                                 collections of programs with multiple\n                                  functionality',
              'filter',
              '                                 filters input and output streams',
              'jiffy',
              '                                 short, simple program',
              'other',
              '                                 all other kinds of software' ] ] ] ],
      [ 'save_',
        '_software.version',
        [ [ 'item_',
            '_item_description.description',
            '              The version of the software.' ],
          [ 'item_', '_item.name', '_software.version' ],
          [ 'item_', '_item.category_id', 'software' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'v1.0', 'beta', '3.1-2', 'unknown' ] ] ] ],
      [ '@comment', '###########' ],
      [ '@comment', '# STRUCT ##' ],
      [ '@comment', '###########' ],
      [ 'save_',
        'struct',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT category record details about the\n               description of the crystallographic structure.' ],
          [ 'item_', '_category.id', 'struct' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _struct.entry_id                       \'5HVP\'\n    _struct.title\n    ; HIV-1 protease complex with acetyl-pepstatin\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_struct.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct.title',
        [ [ 'item_',
            '_item_description.description',
            '              A title for the data block. The author should attempt to convey\n               the essence of the structure archived in the CIF in the title,\n               and to distinguish this structural result from others.' ],
          [ 'item_', '_item.name', '_struct.title' ],
          [ 'item_', '_item.category_id', 'struct' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '5\'-D(*(I)CP*CP*GP*G)-3',
              'T4 lysozyme mutant - S32A',
              'hen egg white lysozyme at -30 degrees C',
              'quail egg white lysozyme at 2 atmospheres' ] ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# STRUCT_ASYM ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'struct_asym',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_ASYM category record details about the\n               structural elements in the asymmetric unit.' ],
          [ 'item_', '_category.id', 'struct_asym' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_asym.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_asym.id\n    _struct_asym.entity_id\n    _struct_asym.details\n      A  1  \'one monomer of the dimeric enzyme\'\n      B  1  \'one monomer of the dimeric enzyme\'\n      C  2  \'one partially occupied position for the inhibitor\'\n      D  2  \'one partially occupied position for the inhibitor\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_asym.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of this portion of the contents\n               of the asymmetric unit.' ],
          [ 'item_', '_item.name', '_struct_asym.details' ],
          [ 'item_', '_item.category_id', 'struct_asym' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The drug binds to this enzyme in two roughly\n                                  twofold symmetric modes. Hence this\n                                  biological unit (3) is roughly twofold\n                                  symmetric to biological unit (2). Disorder in\n                                  the protein chain indicated with alternative\n                                  ID 2 should be used with this biological unit.' ] ] ],
      [ 'save_',
        '_struct_asym.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_struct_asym.entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_asym.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_asym.id must uniquely identify a record in\n               the STRUCT_ASYM list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_asym.id',
              'struct_asym',
              'yes',
              '_atom_site.label_asym_id',
              'atom_site',
              'no',
              '_geom_angle.atom_site_label_asym_id_1',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_label_asym_id_2',
              'geom_angle',
              'no',
              '_geom_angle.atom_site_label_asym_id_3',
              'geom_angle',
              'no',
              '_geom_bond.atom_site_label_asym_id_1',
              'geom_bond',
              'no',
              '_geom_bond.atom_site_label_asym_id_2',
              'geom_bond',
              'no',
              '_geom_contact.atom_site_label_asym_id_1',
              'geom_contact',
              'no',
              '_geom_contact.atom_site_label_asym_id_2',
              'geom_contact',
              'no',
              '_geom_hbond.atom_site_label_asym_id_A',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_label_asym_id_D',
              'geom_hbond',
              'no',
              '_geom_hbond.atom_site_label_asym_id_H',
              'geom_hbond',
              'no',
              '_geom_torsion.atom_site_label_asym_id_1',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_asym_id_2',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_asym_id_3',
              'geom_torsion',
              'no',
              '_geom_torsion.atom_site_label_asym_id_4',
              'geom_torsion',
              'no',
              '_struct_biol_gen.asym_id',
              'struct_biol_gen',
              'yes',
              '_struct_conf.beg_label_asym_id',
              'struct_conf',
              'yes',
              '_struct_conf.end_label_asym_id',
              'struct_conf',
              'yes',
              '_struct_conn.ptnr1_label_asym_id',
              'struct_conn',
              'yes',
              '_struct_conn.ptnr2_label_asym_id',
              'struct_conn',
              'yes',
              '_struct_mon_nucl.label_asym_id',
              'struct_mon_nucl',
              'yes',
              '_struct_mon_prot.label_asym_id',
              'struct_mon_prot',
              'yes',
              '_struct_mon_prot_cis.label_asym_id',
              'struct_mon_prot_cis',
              'yes',
              '_struct_ncs_dom_lim.beg_label_asym_id',
              'struct_ncs_dom_lim',
              'yes',
              '_struct_ncs_dom_lim.end_label_asym_id',
              'struct_ncs_dom_lim',
              'yes',
              '_struct_sheet_range.beg_label_asym_id',
              'struct_sheet_range',
              'yes',
              '_struct_sheet_range.end_label_asym_id',
              'struct_sheet_range',
              'yes',
              '_struct_site_gen.label_asym_id',
              'struct_site_gen',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_atom_site.label_asym_id',
              '_struct_asym.id',
              '_struct_biol_gen.asym_id',
              '_struct_asym.id',
              '_geom_angle.atom_site_label_asym_id_1',
              '_atom_site.label_asym_id',
              '_geom_angle.atom_site_label_asym_id_2',
              '_atom_site.label_asym_id',
              '_geom_angle.atom_site_label_asym_id_3',
              '_atom_site.label_asym_id',
              '_geom_bond.atom_site_label_asym_id_1',
              '_atom_site.label_asym_id',
              '_geom_bond.atom_site_label_asym_id_2',
              '_atom_site.label_asym_id',
              '_geom_contact.atom_site_label_asym_id_1',
              '_atom_site.label_asym_id',
              '_geom_contact.atom_site_label_asym_id_2',
              '_atom_site.label_asym_id',
              '_geom_hbond.atom_site_label_asym_id_A',
              '_atom_site.label_asym_id',
              '_geom_hbond.atom_site_label_asym_id_D',
              '_atom_site.label_asym_id',
              '_geom_hbond.atom_site_label_asym_id_H',
              '_atom_site.label_asym_id',
              '_geom_torsion.atom_site_label_asym_id_1',
              '_atom_site.label_asym_id',
              '_geom_torsion.atom_site_label_asym_id_2',
              '_atom_site.label_asym_id',
              '_geom_torsion.atom_site_label_asym_id_3',
              '_atom_site.label_asym_id',
              '_geom_torsion.atom_site_label_asym_id_4',
              '_atom_site.label_asym_id',
              '_struct_conf.beg_label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_conf.end_label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_conn.ptnr1_label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_conn.ptnr2_label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_mon_nucl.label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_mon_prot.label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_mon_prot_cis.label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_ncs_dom_lim.beg_label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_ncs_dom_lim.end_label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_sheet_range.beg_label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_sheet_range.end_label_asym_id',
              '_atom_site.label_asym_id',
              '_struct_site_gen.label_asym_id',
              '_atom_site.label_asym_id' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', 'A', '2B3' ] ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# STRUCT_BIOL ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'struct_biol',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_BIOL category record details about\n               the structural elements that form each structure of biological\n               significance.\n\n               A given crystal structure may contain many different biological\n               structures. A given structural component in the asymmetric\n               unit may be part of more than one biological unit. A given\n               biological structure may involve crystallographic symmetry.\n\n               For instance, in a structure of a lysozyme-FAB structure, the\n               light- and heavy-chain components of the FAB could be one\n               biological unit, while the two chains of the FAB and the lysozyme\n               could constitute a second biological unit.' ],
          [ 'item_', '_category.id', 'struct_biol' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_biol.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_biol.id\n    _struct_biol.details\n      1\n    ; significant deviations from twofold symmetry exist in this\n      dimeric enzyme\n    ;\n      2\n    ; The drug binds to this enzyme in two roughly twofold\n      symmetric modes. Hence this biological unit (2) is roughly\n      twofold symmetric to biological unit (3). Disorder in the\n      protein chain indicated with alternative ID 1 should be\n      used with this biological unit.\n    ;\n      3\n    ; The drug binds to this enzyme in two roughly twofold\n      symmetric modes. Hence this biological unit (3) is roughly\n      twofold symmetric to biological unit (2). Disorder in the\n      protein chain indicated with alternative ID 2 should be\n      used with this biological unit.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_biol.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the biological unit.' ],
          [ 'item_', '_item.name', '_struct_biol.details' ],
          [ 'item_', '_item.category_id', 'struct_biol' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The drug binds to this enzyme in two roughly\n                                  twofold symmetric modes. Hence this\n                                  biological unit (3) is roughly twofold\n                                  symmetric to biological unit (2). Disorder in\n                                  the protein chain indicated with alternative\n                                  ID 2 should be used with this biological unit.' ] ] ],
      [ 'save_',
        '_struct_biol.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_biol.id must uniquely identify a record in\n               the STRUCT_BIOL list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_biol.id',
              'struct_biol',
              'yes',
              '_struct_biol_gen.biol_id',
              'struct_biol_gen',
              'yes',
              '_struct_biol_keywords.biol_id',
              'struct_biol_keywords',
              'yes',
              '_struct_biol_view.biol_id',
              'struct_biol_view',
              'yes',
              '_struct_ref.biol_id',
              'struct_ref',
              'no' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_biol_gen.biol_id',
              '_struct_biol.id',
              '_struct_biol_keywords.biol_id',
              '_struct_biol.id',
              '_struct_biol_view.biol_id',
              '_struct_biol.id',
              '_struct_ref.biol_id',
              '_struct_biol.id' ] ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# STRUCT_BIOL_GEN ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'struct_biol_gen',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_BIOL_GEN category record details about\n               the generation of each biological unit. The STRUCT_BIOL_GEN\n               data items provide the specifications of the components that\n               constitute that biological unit, which may include symmetry\n               elements.' ],
          [ 'item_', '_category.id', 'struct_biol_gen' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_biol_gen.biol_id',
              '_struct_biol_gen.asym_id',
              '_struct_biol_gen.symmetry' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_biol_gen.biol_id\n    _struct_biol_gen.asym_id\n    _struct_biol_gen.symmetry\n      1  A  1_555\n      1  B  1_555\n      2  A  1_555\n      2  B  1_555\n      2  C  1_555\n      3  A  1_555\n      3  B  1_555\n      3  D  1_555',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_biol_gen.asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_asym.id in the STRUCT_ASYM\n               category.' ],
          [ 'item_', '_item.name', '_struct_biol_gen.asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_biol_gen.biol_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_biol.id in the STRUCT_BIOL\n               category.' ],
          [ 'item_', '_item.name', '_struct_biol_gen.biol_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_biol_gen.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the symmetry generation of\n               this portion of the biological structure.' ],
          [ 'item_', '_item.name', '_struct_biol_gen.details' ],
          [ 'item_', '_item.category_id', 'struct_biol_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The zinc atom lies on a special position;\n                                  application of symmetry elements to generate\n                                  the insulin hexamer will generate excess zinc\n                                  atoms, which must be removed by hand.' ] ] ],
      [ 'save_',
        '_struct_biol_gen.symmetry',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               atom set specified by _struct_biol_gen.asym_id to generate a\n               portion of the biological structure.' ],
          [ 'item_', '_item.name', '_struct_biol_gen.symmetry' ],
          [ 'item_', '_item.category_id', 'struct_biol_gen' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ '@comment', '    _item_default.value           1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# STRUCT_BIOL_KEYWORDS ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'struct_biol_keywords',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_BIOL_KEYWORDS category record\n               keywords that describe each biological unit.' ],
          [ 'item_', '_category.id', 'struct_biol_keywords' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_biol_keywords.biol_id',
              '_struct_biol_keywords.text' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_biol_keywords.biol_id\n    _struct_biol_keywords.text\n      1  \'aspartyl-protease\'\n      1  \'aspartic-protease\'\n      1  \'acid-protease\'\n      1  \'aspartyl-proteinase\'\n      1  \'aspartic-proteinase\'\n      1  \'acid-proteinase\'\n      1  \'enzyme\'\n      1  \'protease\'\n      1  \'proteinase\'\n      1  \'dimer\'\n      2  \'drug-enzyme complex\'\n      2  \'inhibitor-enzyme complex\'\n      2  \'drug-protease complex\'\n      2  \'inhibitor-protease complex\'\n      3  \'drug-enzyme complex\'\n      3  \'inhibitor-enzyme complex\'\n      3  \'drug-protease complex\'\n      3  \'inhibitor-protease complex\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_biol_keywords.biol_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_biol.id in the STRUCT_BIOL\n               category.' ],
          [ 'item_', '_item.name', '_struct_biol_keywords.biol_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_biol_keywords.text',
        [ [ 'item_',
            '_item_description.description',
            '              Keywords describing this biological entity.' ],
          [ 'item_', '_item.name', '_struct_biol_keywords.text' ],
          [ 'item_', '_item.category_id', 'struct_biol_keywords' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'antibody', 'antigen', 'enzyme', 'cytokine', 'tRNA' ] ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# STRUCT_BIOL_VIEW ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'struct_biol_view',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_BIOL_VIEW category record details\n               about how to draw and annotate an informative view of the\n               biological structure.' ],
          [ 'item_', '_category.id', 'struct_biol_view' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_biol_view.biol_id', '_struct_biol_view.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on NDB structure GDL001 by Coll, Aymami,\n                Van Der Marel, Van Boom, Rich & Wang\n                [Biochemistry, (1989), 28, 310-320].',
              '\n    _struct_biol_view.biol_id               c1\n    _struct_biol_view.id                    1\n    _struct_biol_view.rot_matrix[1][1]      0.132\n    _struct_biol_view.rot_matrix[1][2]      0.922\n    _struct_biol_view.rot_matrix[1][3]     -0.363\n    _struct_biol_view.rot_matrix[2][1]      0.131\n    _struct_biol_view.rot_matrix[2][2]     -0.380\n    _struct_biol_view.rot_matrix[2][3]     -0.916\n    _struct_biol_view.rot_matrix[3][1]     -0.982\n    _struct_biol_view.rot_matrix[3][2]      0.073\n    _struct_biol_view.rot_matrix[3][3]     -0.172\n    _struct_biol_view.details\n    ; This view highlights the ATAT-Netropsin interaction in the\n      DNA-drug complex.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_biol_view.biol_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_biol.id in the STRUCT_BIOL\n               category.' ],
          [ 'item_', '_item.name', '_struct_biol_view.biol_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_biol_view.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of this view of the biological\n               structure.\n\n               This data item can be used as a figure legend.' ],
          [ 'item_', '_item.name', '_struct_biol_view.details' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The enzyme has been oriented with the\n                                  molecular twofold axis aligned with the\n                                  horizontal axis of the figure.' ] ] ],
      [ 'save_',
        '_struct_biol_view.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_biol_view.id must uniquely identify a\n               record in the STRUCT_BIOL_VIEW list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_struct_biol_view.id' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Figure 1',
              'unliganded enzyme',
              'view down enzyme active site' ] ] ] ],
      [ 'save_',
        '_struct_biol_view.rot_matrix[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_BIOL_GEN category to give a view useful for describing the\n               structure. The conventions used in the rotation are described in\n               _struct_biol_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_biol_view.rot_matrix[1][1]' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_biol_view.rot_matrix[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_BIOL_GEN category to give a view useful for describing the\n               structure. The conventions used in the rotation are described in\n               _struct_biol_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_biol_view.rot_matrix[1][2]' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_biol_view.rot_matrix[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_BIOL_GEN category to give a view useful for describing the\n               structure. The conventions used in the rotation are described in\n               _struct_biol_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_biol_view.rot_matrix[1][3]' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_biol_view.rot_matrix[2][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][1] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_BIOL_GEN category to give a view useful for describing the\n               structure. The conventions used in the rotation are described in\n               _struct_biol_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_biol_view.rot_matrix[2][1]' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_biol_view.rot_matrix[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_BIOL_GEN category to give a view useful for describing the\n               structure. The conventions used in the rotation are described in\n               _struct_biol_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_biol_view.rot_matrix[2][2]' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_biol_view.rot_matrix[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_BIOL_GEN category to give a view useful for describing the\n               structure. The conventions used in the rotation are described in\n               _struct_biol_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_biol_view.rot_matrix[2][3]' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_biol_view.rot_matrix[3][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][1] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_BIOL_GEN category to give a view useful for describing the\n               structure. The conventions used in the rotation are described in\n               _struct_biol_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_biol_view.rot_matrix[3][1]' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_biol_view.rot_matrix[3][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][2] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_BIOL_GEN category to give a view useful for describing the\n               structure. The conventions used in the rotation are described in\n               _struct_biol_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_biol_view.rot_matrix[3][2]' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_biol_view.rot_matrix[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_BIOL_GEN category to give a view useful for describing the\n               structure. The conventions used in the rotation are described in\n               _struct_biol_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_biol_view.rot_matrix[3][3]' ],
          [ 'item_', '_item.category_id', 'struct_biol_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# STRUCT_CONF ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'struct_conf',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_CONF category record details about\n               the backbone conformation of a segment of polymer.\n\n               Data items in the STRUCT_CONF_TYPE category define the\n               criteria used to identify the backbone conformations.' ],
          [ 'item_', '_category.id', 'struct_conf' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_conf.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_conf.id\n    _struct_conf.conf_type_id\n    _struct_conf.beg_label_comp_id\n    _struct_conf.beg_label_asym_id\n    _struct_conf.beg_label_seq_id\n    _struct_conf.end_label_comp_id\n    _struct_conf.end_label_asym_id\n    _struct_conf.end_label_seq_id\n    _struct_conf.details\n      HELX1  HELX_RH_AL_P  ARG  A   87  GLN  A   92  .\n      HELX2  HELX_RH_AL_P  ARG  B  287  GLN  B  292  .\n      STRN1  STRN_P        PRO  A    1  LEU  A    5  .\n      STRN2  STRN_P        CYS  B  295  PHE  B  299  .\n      STRN3  STRN_P        CYS  A   95  PHE  A  299  .\n      STRN4  STRN_P        PRO  B  201  LEU  B  205  .\n    # - - - - data truncated for brevity - - - -\n      TURN1  TURN_TY1P_P   ILE  A   15  GLN  A   18  .\n      TURN2  TURN_TY2_P    GLY  A   49  GLY  A   52  .\n      TURN3  TURN_TY1P_P   ILE  A   55  HIS  A   69  .\n      TURN4  TURN_TY1_P    THR  A   91  GLY  A   94  .\n    # - - - - data truncated for brevity - - - -',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_conf.beg_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment begins.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.beg_label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conf.beg_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment begins.\n\n               This data item is a pointer to _atom_site.label_comp_id in\n               the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.beg_label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conf.beg_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment begins.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.beg_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conf.beg_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment begins.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.beg_auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conf.beg_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment begins.\n\n               This data item is a pointer to _atom_site.auth_comp_id in\n               the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.beg_auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conf.beg_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment begins.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.beg_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conf.conf_type_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_conf_type.id in the\n               STRUCT_CONF_TYPE category.' ],
          [ 'item_', '_item.name', '_struct_conf.conf_type_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conf.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the conformation assignment.' ],
          [ 'item_', '_item.name', '_struct_conf.details' ],
          [ 'item_', '_item.category_id', 'struct_conf' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_struct_conf.end_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment ends.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.end_label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conf.end_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment ends.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.end_label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conf.end_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment ends.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.end_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conf.end_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment ends.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.end_auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conf.end_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment ends.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.end_auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conf.end_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment ends.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conf.end_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conf.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_conf.id must uniquely identify a record in\n               the STRUCT_CONF list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_struct_conf.id' ],
          [ 'item_', '_item.category_id', 'struct_conf' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# STRUCT_CONF_TYPE ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'struct_conf_type',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_CONF_TYPE category record details\n               about the criteria used to identify backbone conformations of a\n               segment of polymer.' ],
          [ 'item_', '_category.id', 'struct_conf_type' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_conf_type.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_conf_type.id\n    _struct_conf_type.criteria\n    _struct_conf_type.reference\n      HELX_RH_AL_P  \'author judgement\'  .\n      STRN_P        \'author judgement\'  .\n      TURN_TY1_P    \'author judgement\'  .\n      TURN_TY1P_P   \'author judgement\'  .\n      TURN_TY2_P    \'author judgement\'  .\n      TURN_TY2P_P   \'author judgement\'  .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_conf_type.criteria',
        [ [ 'item_',
            '_item_description.description',
            '              The criteria used to assign this conformation type.' ],
          [ 'item_', '_item.name', '_struct_conf_type.criteria' ],
          [ 'item_', '_item.category_id', 'struct_conf_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'author judgement', 'phi=54-74, psi=30-50' ] ] ] ],
      [ 'save_',
        '_struct_conf_type.id',
        [ [ 'item_',
            '_item_description.description',
            '              The descriptor that categorizes the type of the conformation\n               of the backbone of the polymer (whether protein or nucleic acid).\n               Explicit values for the torsion angles that define each\n               conformation are not given here, but it is expected that the\n               author would provide such information in either the\n               _struct_conf_type.criteria or _struct_conf_type.reference data\n               items, or both.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_conf_type.id',
              'struct_conf_type',
              'yes',
              '_struct_conf.conf_type_id',
              'struct_conf',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_conf.conf_type_id', '_struct_conf_type.id' ] ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'HELX_P',
              '                                 helix with handedness and type not specified\n                                  (protein)',
              'HELX_OT_P',
              '                                 helix with handedness and type that do not\n                                  conform to an accepted category (protein)',
              'HELX_RH_P',
              '                                 right-handed helix with type not specified\n                                  (protein)',
              'HELX_RH_OT_P',
              '                                 right-handed helix with type that does not\n                                  conform to an accepted category (protein)',
              'HELX_RH_AL_P',
              'right-handed alpha helix (protein)',
              'HELX_RH_GA_P',
              'right-handed gamma helix (protein)',
              'HELX_RH_OM_P',
              'right-handed omega helix (protein)',
              'HELX_RH_PI_P',
              'right-handed pi helix (protein)',
              'HELX_RH_27_P',
              'right-handed 2-7 helix (protein)',
              'HELX_RH_3T_P',
              'right-handed 3-10 helix (protein)',
              'HELX_RH_PP_P',
              'right-handed polyproline helix (protein)',
              'HELX_LH_P',
              '                                 left-handed helix with type not specified\n                                  (protein)',
              'HELX_LH_OT_P',
              '                                 left-handed helix with type that does not\n                                  conform to an accepted category (protein)',
              'HELX_LH_AL_P',
              'left-handed alpha helix (protein)',
              'HELX_LH_GA_P',
              'left-handed gamma helix (protein)',
              'HELX_LH_OM_P',
              'left-handed omega helix (protein)',
              'HELX_LH_PI_P',
              'left-handed pi helix (protein)',
              'HELX_LH_27_P',
              'left-handed 2-7 helix (protein)',
              'HELX_LH_3T_P',
              'left-handed 3-10 helix (protein)',
              'HELX_LH_PP_P',
              'left-handed polyproline helix (protein)',
              'HELX_N',
              '                                 helix with handedness and type not specified\n                                  (nucleic acid)',
              'HELX_OT_N',
              '                                 helix with handedness and type that do not\n                                  conform to an accepted category (nucleic\n                                  acid)',
              'HELX_RH_N',
              '                                 right-handed helix with type not specified\n                                  (nucleic acid)',
              'HELX_RH_OT_N',
              '                                 right-handed helix with type that does not\n                                  conform to an accepted category (nucleic\n                                  acid)',
              'HELX_RH_A_N',
              'right-handed A helix (nucleic acid)',
              'HELX_RH_B_N',
              'right-handed B helix (nucleic acid)',
              'HELX_RH_Z_N',
              'right-handed Z helix (nucleic acid)',
              'HELX_LH_N',
              '                                 left-handed helix with type not specified\n                                  (nucleic acid)',
              'HELX_LH_OT_N',
              '                                 left-handed helix with type that does not\n                                  conform to an accepted category (nucleic\n                                  acid)',
              'HELX_LH_A_N',
              'left-handed A helix (nucleic acid)',
              'HELX_LH_B_N',
              'left-handed B helix (nucleic acid)',
              'HELX_LH_Z_N',
              'left-handed Z helix (nucleic acid)',
              'TURN_P',
              'turn with type not specified (protein)',
              'TURN_OT_P',
              '                                 turn with type that does not conform to an\n                                  accepted category (protein)',
              'TURN_TY1_P',
              'type I turn (protein)',
              'TURN_TY1P_P',
              'type I prime turn (protein)',
              'TURN_TY2_P',
              'type II turn (protein)',
              'TURN_TY2P_P',
              'type II prime turn (protein)',
              'TURN_TY3_P',
              'type III turn (protein)',
              'TURN_TY3P_P',
              'type III prime turn (protein)',
              'STRN',
              'beta strand (protein)' ] ] ] ],
      [ 'save_',
        '_struct_conf_type.reference',
        [ [ 'item_',
            '_item_description.description',
            '              A literature reference that defines the criteria used to assign\n               this conformation type and subtype.' ],
          [ 'item_', '_item.name', '_struct_conf_type.reference' ],
          [ 'item_', '_item.category_id', 'struct_conf_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# STRUCT_CONN ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'struct_conn',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_CONN category record details about\n               the connections between portions of the structure. These can be\n               hydrogen bonds, salt bridges, disulfide bridges and so on.\n\n               The STRUCT_CONN_TYPE records define the criteria used to\n               identify these connections.' ],
          [ 'item_', '_category.id', 'struct_conn' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_conn.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_conn.id\n    _struct_conn.conn_type_id\n    _struct_conn.ptnr1_label_comp_id\n    _struct_conn.ptnr1_label_asym_id\n    _struct_conn.ptnr1_label_seq_id\n    _struct_conn.ptnr1_label_atom_id\n    _struct_conn.ptnr1_role\n    _struct_conn.ptnr1_symmetry\n    _struct_conn.ptnr2_label_comp_id\n    _struct_conn.ptnr2_label_asym_id\n    _struct_conn.ptnr2_label_seq_id\n    _struct_conn.ptnr2_label_atom_id\n    _struct_conn.ptnr2_role\n    _struct_conn.ptnr2_symmetry\n    _struct_conn.details\n      C1  saltbr  ARG  A  87 NZ1 positive 1_555 GLU  A  92  OE1\n          negative 1_555  .\n      C2  hydrog  ARG  B 287 N   donor    1_555 GLY  B 292  O\n          acceptor 1_555  .\n    # - - - - data truncated for brevity - - - -',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_conn.conn_type_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_conn_type.id in the\n               STRUCT_CONN_TYPE category.' ],
          [ 'item_', '_item.name', '_struct_conn.conn_type_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conn.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the connection.' ],
          [ 'item_', '_item.name', '_struct_conn.details' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'disulfide bridge C-S-S-C is highly distorted' ] ] ],
      [ 'save_',
        '_struct_conn.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_conn.id must uniquely identify a record in\n               the STRUCT_CONN list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_struct_conn.id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 1 of the structure\n               connection.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_label_alt_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 1 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 1 of the structure\n               connection.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_label_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 1 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 1 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 1 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 1 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_auth_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 1 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 1 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_role',
        [ [ 'item_',
            '_item_description.description',
            '              The chemical or structural role of the first partner in\n               the structure connection.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_role' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uline' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'donor',
              'acceptor',
              'negative',
              'positive',
              'metal',
              'metal coordination' ] ] ] ],
      [ 'save_',
        '_struct_conn.ptnr1_symmetry',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               atom set specified by _struct_conn.ptnr1_label* to generate the\n               first partner in the structure connection.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr1_symmetry' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 2 of the structure\n               connection.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_label_alt_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 2 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 2 of the structure\n               connection.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_label_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 2 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 2 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 2 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 2 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_auth_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 2 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 2 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_role',
        [ [ 'item_',
            '_item_description.description',
            '              The chemical or structural role of the second partner in\n               the structure connection.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_role' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uline' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'donor',
              'acceptor',
              'negative',
              'positive',
              'metal',
              'metal coordination' ] ] ] ],
      [ 'save_',
        '_struct_conn.ptnr2_symmetry',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               atom set specified by _struct_conn.ptnr2_label* to generate the\n               second partner in the structure connection.' ],
          [ 'item_', '_item.name', '_struct_conn.ptnr2_symmetry' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# STRUCT_CONN_TYPE ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'struct_conn_type',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_CONN_TYPE category record details\n               about the criteria used to identify interactions between\n               portions of the structure.' ],
          [ 'item_', '_category.id', 'struct_conn_type' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_conn_type.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_conn_type.id\n    _struct_conn_type.criteria\n    _struct_conn_type.reference\n      saltbr\n     \'negative to positive distance > 2.5 \\%A, < 3.2 \\%A\'      .\n      hydrog\n     \'NO distance > 2.5\\%A, < 3.5\\%A, NOC angle < 120 degrees\' .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_conn_type.criteria',
        [ [ 'item_',
            '_item_description.description',
            '              The criteria used to define the interaction.' ],
          [ 'item_', '_item.name', '_struct_conn_type.criteria' ],
          [ 'item_', '_item.category_id', 'struct_conn_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'O to N distance > 2.5 \\%A, < 3.2 \\%A',
              'authors judgement' ] ] ] ],
      [ 'save_',
        '_struct_conn_type.id',
        [ [ 'item_',
            '_item_description.description',
            '              The chemical or structural type of the interaction.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_conn_type.id',
              'struct_conn_type',
              'yes',
              '_struct_conn.conn_type_id',
              'struct_conn',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_conn.conn_type_id', '_struct_conn_type.id' ] ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'covale',
              'covalent bond',
              'disulf',
              'disulfide bridge',
              'hydrog',
              'hydrogen bond',
              'metalc',
              'metal coordination',
              'mismat',
              'mismatched base pairs',
              'saltbr',
              'ionic interaction',
              'modres',
              'covalent residue modification',
              'covale_base',
              'covalent modification of a nucleotide base',
              'covale_sugar',
              'covalent modification of a nucleotide sugar',
              'covale_phosphate',
              'covalent modification of a nucleotide phosphate' ] ] ] ],
      [ 'save_',
        '_struct_conn_type.reference',
        [ [ 'item_',
            '_item_description.description',
            '              A reference that specifies the criteria used to define the\n               interaction.' ],
          [ 'item_', '_item.name', '_struct_conn_type.reference' ],
          [ 'item_', '_item.category_id', 'struct_conn_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# STRUCT_KEYWORDS ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'struct_keywords',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_KEYWORDS category specify keywords\n               that describe the chemical structure in this entry.' ],
          [ 'item_', '_category.id', 'struct_keywords' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_keywords.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_keywords.entry_id\n    _struct_keywords.text\n     \'5HVP\' \'enzyme-inhibitor complex, aspartyl protease, static disorder\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_keywords.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_struct_keywords.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_keywords.text',
        [ [ 'item_',
            '_item_description.description',
            '              Keywords describing this structure.' ],
          [ 'item_', '_item.name', '_struct_keywords.text' ],
          [ 'item_', '_item.category_id', 'struct_keywords' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'serine protease',
              'inhibited complex',
              'high-resolution refinement' ] ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# STRUCT_MON_DETAILS ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'struct_mon_details',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_MON_DETAILS category record details\n               about specifics of calculations summarized in data items in the\n               STRUCT_MON_PROT and STRUCT_MON_NUCL categories. These can\n               include the coefficients used in map calculations,\n               the radii used for including points in a calculation and so on.' ],
          [ 'item_', '_category.id', 'struct_mon_details' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_mon_details.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ] ] ],
      [ 'save_',
        '_struct_mon_details.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_struct_mon_details.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_details.prot_cis',
        [ [ 'item_',
            '_item_description.description',
            '              An ideal cis peptide bond would have an omega torsion angle of\n               zero. This data item gives the value in degrees by which the\n               observed torsion angle can differ from 0.0 and still be\n               considered cis.' ],
          [ 'item_', '_item.name', '_struct_mon_details.prot_cis' ],
          [ 'item_', '_item.category_id', 'struct_mon_details' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_', '_item_examples.case', '30.0' ] ] ],
      [ 'save_',
        '_struct_mon_details.RSCC',
        [ [ 'item_',
            '_item_description.description',
            '              This data item describes the specifics of the calculations that\n               generated the values given in _struct_mon_prot.RSCC_all,\n               _struct_mon_prot.RSCC_main and _struct_mon_prot.RSCC_side. The\n               coefficients used to calculate the p(o) and p(c) maps should be\n               given as well as the criterion for the inclusion of map grid\n               points in the calculation.' ],
          [ 'item_', '_item.name', '_struct_mon_details.RSCC' ],
          [ 'item_', '_item.category_id', 'struct_mon_details' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '                                 The map p(o) was calculated with coefficients\n                                  2F(o) - F(c) and with phase alpha(c). F(o)\n                                  are the observed structure-factor amplitudes,\n                                  F(c) are the amplitudes calculated from the\n                                  current model and alpha(c) are the phases\n                                  calculated from the current model.\n                                  The map p(c) was calculated in program O using\n                                  a Gaussian distribution function around the\n                                  atoms in the current model.\n                                  Map grid points within 1.5 A of the\n                                  designated atoms were included in the\n                                  calculation.',
              '                                 The map p(o) was calculated with coefficients\n                                  F(o) and with phase alpha(c). F(o) are the\n                                  observed structure-factor amplitudes, and\n                                  alpha(c) are the phases calculated from the\n                                  current model.\n                                  The map p(c) was calculated with coefficients\n                                  F(c) and with phases alpha(c). F(c) and\n                                  alpha(c) are the structure-factor amplitudes\n                                  and phases, respectively, calculated from the\n                                  current model.\n                                  Map grid points within a van der Waals radius\n                                  of the designated atoms were included in the\n                                  calculation.' ] ] ] ],
      [ 'save_',
        '_struct_mon_details.RSR',
        [ [ 'item_',
            '_item_description.description',
            '              This data item describes the specifics of the calculations that\n               generated the values given in _struct_mon_prot.RSR_all,\n               _struct_mon_prot.RSR_main and _struct_mon_prot.RSR_side. The\n               coefficients used to calculate the p(o) and p(c) maps should be\n               given as well as the criterion for the inclusion of map grid\n               points in the calculation.' ],
          [ 'item_', '_item.name', '_struct_mon_details.RSR' ],
          [ 'item_', '_item.category_id', 'struct_mon_details' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '                                 The map p(o) was calculated with coefficients\n                                  2F(o) - F(c) and with phase alpha(c). F(o)\n                                  are the observed structure-factor amplitudes,\n                                  F(c) are the amplitudes calculated from the\n                                  current model and alpha(c) are the phases\n                                  calculated from the current model.\n                                  The map p(c) was calculated in program O using\n                                  a Gaussian distribution function around the\n                                  atoms in the current model.\n                                  Map grid points within 1.5 A of the\n                                  designated atoms were included in the\n                                  calculation.',
              '                                 The map p(o) was calculated with coefficients\n                                  F(o) and with phase alpha(c). F(o) are the\n                                  observed structure-factor amplitudes, and\n                                  alpha(c) are the phases calculated from the\n                                  current model.\n                                  The map p(c) was calculated with coefficients\n                                  F(c) and with phases alpha(c). F(c) and\n                                  alpha(c) are the structure-factor amplitudes\n                                  and phases, respectively, calculated from the\n                                  current model.\n                                  Map grid points within a van der Waals radius\n                                  of the designated atoms were included in the\n                                  calculation.' ] ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# STRUCT_MON_NUCL ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'struct_mon_nucl',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_MON_NUCL category record details about\n               structural properties of a nucleic acid when analyzed at the\n               monomer level. Analogous data items for proteins are given in\n               the STRUCT_MON_PROT category. For items where the value of the\n               property depends on the method employed to calculate it,\n               details of the method of calculation are given using data items\n               in the STRUCT_MON_DETAILS category.' ],
          [ 'item_', '_category.id', 'struct_mon_nucl' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_mon_nucl.label_alt_id',
              '_struct_mon_nucl.label_asym_id',
              '_struct_mon_nucl.label_comp_id',
              '_struct_mon_nucl.label_seq_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on NDB structure BDL028.',
              '\n    loop_\n    _struct_mon_nucl.label_comp_id\n    _struct_mon_nucl.label_seq_id\n    _struct_mon_nucl.label_asym_id\n    _struct_mon_nucl.label_alt_id\n    _struct_mon_nucl.alpha\n    _struct_mon_nucl.beta\n    _struct_mon_nucl.gamma\n    _struct_mon_nucl.delta\n    _struct_mon_nucl.epsilon\n    _struct_mon_nucl.zeta\n    C  1 A .      .      .   29.9  131.9  222.1  174.2\n    G  2 A .  334.0  130.6   33.1  125.6  167.6  270.9\n    T  3 A .  258.2  178.7   101.0 114.6  216.6  259.3\n    #         ---- abbreviated list -----',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_mon_nucl.alpha',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the backbone torsion angle alpha\n               (O3\'-P-O5\'-C5\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.alpha' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.beta',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the backbone torsion angle beta\n               (P-O5\'-C5\'-C4\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.beta' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.chi1',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar-base torsion angle chi1\n               (O4\'-C1\'-N1-C2).' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.chi1' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.chi2',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar-base torsion angle chi2\n               (O4\'-C1\'-N9-C4).' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.chi2' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.delta',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the backbone torsion angle delta\n               (C5\'-C4\'-C3\'-O3\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.delta' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the residue, its\n               conformation, behaviour in refinement, or any other aspect\n               that requires annotation.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.details' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Part of the phosphodiester backbone not in\n                                  density.' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.epsilon',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the backbone torsion angle epsilon\n               (C4\'-C3\'-O3\'-P).' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.epsilon' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.gamma',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the backbone torsion angle gamma\n               (O5\'-C5\'-C4\'-C3\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.gamma' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.label_alt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.mean_B_all',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the isotropic displacement parameter\n               for all atoms in the monomer.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.mean_B_all' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.mean_B_base',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the isotropic displacement parameter\n               for atoms in the base moiety of the nucleic acid monomer.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.mean_B_base' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.mean_B_phos',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the isotropic displacement parameter\n               for atoms in the phosphate moiety of the nucleic acid monomer.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.mean_B_phos' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.mean_B_sugar',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the isotropic displacement parameter\n               for atoms in the sugar moiety of the nucleic acid monomer.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.mean_B_sugar' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.nu0',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar torsion angle nu0\n               (C4\'-O4\'-C1\'-C2\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.nu0' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.nu1',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar torsion angle nu1\n               (O4\'-C1\'-C2\'-C3\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.nu1' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.nu2',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar torsion angle nu2\n               (C1\'-C2\'-C3\'-C4\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.nu2' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.nu3',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar torsion angle nu3\n               (C2\'-C3\'-C4\'-O4\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.nu3' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.nu4',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar torsion angle nu4\n               (C3\'-C4\'-O4\'-C1\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.nu4' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.P',
        [ [ 'item_',
            '_item_description.description',
            '              P is the phase angle of pseudorotation for five-membered rings.\n               For ribose and deoxyribose sugars in nucleic\n               acids\n                          (tau4 +tau1)-(tau3+tau0)\n               P = ATAN (-------------------------)\n                           2tau2 (sin 36+sin 72)\n\n               If tau2 is <0, then P=P+180 degree (Altona & Sundaralingam,\n               1972).\n\n               Ref: Altona, C. & Sundaralingam, M. (1972).\n               J. Am. Chem. Soc. 94, 8205-8212.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.P' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.RSCC_all',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space (linear) correlation coefficient RSCC, as\n               described by Jones et al. (1991), evaluated over all atoms in the\n               nucleic acid monomer.\n\n                      sum|p~obs~ - <p~obs~>| * sum|p~calc~ - <p~calc~>|\n               RSCC = -------------------------------------------------\n                            [ sum|p~obs~  - <p~obs~> |^2^\n                            * sum|p~calc~ - <p~calc~>|^2^ ]^1/2^\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSCC. < > indicates an average and the\n               sums are taken over all map grid  points near the relevant atoms.\n               The radius for including grid points in the calculation should\n               also be given in _struct_mon_details.RSCC.\n\n               Ref: Jones, T. A., Zou, J.-Y., Cowan, S. W. & Kjeldgaard, M.\n                    (1991). Acta Cryst. A47, 110-119.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.RSCC_all' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.RSCC_base',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space (linear) correlation coefficient RSCC, as\n               described by Jones et al. (1991), evaluated over all atoms in the\n               base moiety of the nucleic acid monomer.\n\n                      sum|p~obs~ - <p~obs~>| * sum|p~calc~ - <p~calc~>|\n               RSCC = -------------------------------------------------\n                            [ sum|p~obs~  - <p~obs~> |^2^\n                            * sum|p~calc~ - <p~calc~>|^2^ ]^1/2^\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSCC. < > indicates an average and the\n               sums are taken over all map grid  points near the relevant atoms.\n               The radius for including grid points in the calculation should\n               also be given in _struct_mon_details.RSCC.\n\n               Ref: Jones, T. A., Zou, J.-Y., Cowan, S. W. & Kjeldgaard, M.\n                    (1991). Acta Cryst. A47, 110-119.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.RSCC_base' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.RSCC_phos',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space (linear) correlation coefficient RSCC, as\n               described by Jones et al. (1991), evaluated over all atoms in the\n               phosphate moiety of the nucleic acid monomer.\n\n                      sum|p~obs~ - <p~obs~>| * sum|p~calc~ - <p~calc~>|\n               RSCC = -------------------------------------------------\n                            [ sum|p~obs~  - <p~obs~> |^2^\n                            * sum|p~calc~ - <p~calc~>|^2^ ]^1/2^\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSCC. < > indicates an average and the\n               sums are taken over all map grid  points near the relevant atoms.\n               The radius for including grid points in the calculation should\n               also be given in _struct_mon_details.RSCC.\n\n               Ref: Jones, T. A., Zou, J.-Y., Cowan, S. W. & Kjeldgaard, M.\n                    (1991). Acta Cryst. A47, 110-119.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.RSCC_phos' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.RSCC_sugar',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space (linear) correlation coefficient RSCC, as\n               described by Jones et al. (1991), evaluated over all atoms in the\n               sugar moiety of the nucleic acid monomer.\n\n                      sum|p~obs~ - <p~obs~>| * sum|p~calc~ - <p~calc~>|\n               RSCC = -------------------------------------------------\n                            [ sum|p~obs~  - <p~obs~> |^2^\n                            * sum|p~calc~ - <p~calc~>|^2^ ]^1/2^\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSCC. < > indicates an average and the\n               sums are taken over all map grid  points near the relevant atoms.\n               The radius for including grid points in the calculation should\n               also be given in _struct_mon_details.RSCC.\n\n               Ref: Jones, T. A., Zou, J.-Y., Cowan, S. W. & Kjeldgaard, M.\n                    (1991). Acta Cryst. A47, 110-119.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.RSCC_sugar' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.RSR_all',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space residual RSR, as described by Branden & Jones\n               (1990), evaluated over all atoms in the nucleic acid monomer.\n\n                     sum|p~obs~ - p~calc~|\n               RSR = ---------------------\n                     sum|p~obs~ + p~calc~|\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSR. The sums are taken over all map grid\n               points near the relevant atoms. The radius for including grid\n               points in the calculation should also be given in\n               _struct_mon_details.RSR.\n\n               Ref: Branden, C.-I. & Jones, T. A. (1990). Nature (London), 343,\n                    687-689.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.RSR_all' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.RSR_base',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space residual RSR, as described by Branden & Jones\n               (1990), evaluated over all atoms in the base moiety of the\n               nucleic acid monomer.\n\n                     sum|p~obs~ - p~calc~|\n               RSR = ---------------------\n                     sum|p~obs~ + p~calc~|\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSR. The sums are taken over all map grid\n               points near the relevant atoms. The radius for including grid\n               points in the calculation should also be given in\n               _struct_mon_details.RSR.\n\n               Ref: Branden, C.-I. & Jones, T. A. (1990). Nature (London), 343,\n                    687-689.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.RSR_base' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.RSR_phos',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space residual RSR, as described by Branden & Jones\n               (1990), evaluated over all atoms in the phosphate moiety of the\n               nucleic acid monomer.\n\n                     sum|p~obs~ - p~calc~|\n               RSR = ---------------------\n                     sum|p~obs~ + p~calc~|\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSR. The sums are taken over all map grid\n               points near the relevant atoms. The radius for including grid\n               points in the calculation should also be given in\n               _struct_mon_details.RSR.\n\n               Ref: Branden, C.-I. & Jones, T. A. (1990). Nature (London), 343,\n                    687-689.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.RSR_phos' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.RSR_sugar',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space residual RSR, as described by Branden & Jones\n               (1990), evaluated over all atoms in the sugar moiety of the\n               nucleic acid monomer.\n\n                     sum|p~obs~ - p~calc~|\n               RSR = ---------------------\n                     sum|p~obs~ + p~calc~|\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSR. The sums are taken over all map grid\n               points near the relevant atoms. The radius for including grid\n               points in the calculation should also be given in\n               _struct_mon_details.RSR.\n\n               Ref: Branden, C.-I. & Jones, T. A. (1990). Nature (London), 343,\n                    687-689.' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.RSR_sugar' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.tau0',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar torsion angle tau0\n               (C4\'-O4\'-C1\'-C2\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.tau0' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.tau1',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar torsion angle tau1\n               (O4\'-C1\'-C2\'-C3\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.tau1' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.tau2',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar torsion angle tau2\n               (C1\'-C2\'-C3\'-C4\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.tau2' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.tau3',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar torsion angle tau3\n               (C2\'-C3\'-C4\'-O4\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.tau3' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.tau4',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the sugar torsion angle tau4\n               (C3\'-C4\'-O4\'-C1\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.tau4' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.taum',
        [ [ 'item_',
            '_item_description.description',
            '              The maximum amplitude of puckering. This is derived from the\n               pseudorotation value P and the torsion angles in the ribose\n               ring.\n\n               Tau2= Taum cosP\n               Tau3= Taum cos(P+144)\n               Tau4= Taum cos(P+288)\n               Tau0= Taum cos(P+ 72)\n               Tau1= Taum cos(P+216)' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.taum' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_nucl.zeta',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the backbone torsion angle zeta\n               (C3\'-O3\'-P-O5\').' ],
          [ 'item_', '_item.name', '_struct_mon_nucl.zeta' ],
          [ 'item_', '_item.category_id', 'struct_mon_nucl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# STRUCT_MON_PROT ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'struct_mon_prot',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_MON_PROT category record details about\n               structural properties of a protein when analyzed at the monomer\n               level. Analogous data items for nucleic acids are given in the\n               STRUCT_MON_NUCL category. For items where the value of the\n               property depends on the method employed to calculate it,\n               details of the method of calculation are given using data items\n               in the STRUCT_MON_DETAILS category.' ],
          [ 'item_', '_category.id', 'struct_mon_prot' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_mon_prot.label_alt_id',
              '_struct_mon_prot.label_asym_id',
              '_struct_mon_prot.label_comp_id',
              '_struct_mon_prot.label_seq_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for protein NS1.\n                This example provides details for residue ARG 35.',
              '\n    _struct_mon_prot.label_comp_id    ARG\n    _struct_mon_prot.label_seq_id      35\n    _struct_mon_prot.label_asym_id      A\n    _struct_mon_prot.label_alt_id       .\n    _struct_mon_prot.chi1            -67.9\n    _struct_mon_prot.chi2           -174.7\n    _struct_mon_prot.chi3            -67.7\n    _struct_mon_prot.chi4            -86.3\n    _struct_mon_prot.chi5              4.2\n    _struct_mon_prot.RSCC_all          0.90\n    _struct_mon_prot.RSR_all           0.18\n    _struct_mon_prot.mean_B_all        30.0\n    _struct_mon_prot.mean_B_main       25.0\n    _struct_mon_prot.mean_B_side       35.1\n    _struct_mon_prot.omega            180.1\n    _struct_mon_prot.phi              -60.3\n    _struct_mon_prot.psi              -46.0',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_mon_prot.chi1',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the side-chain torsion angle chi1, for\n               those residues containing such an angle.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.chi1' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_prot.chi2',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the side-chain torsion angle chi2, for\n               those residues containing such an angle.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.chi2' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_prot.chi3',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the side-chain torsion angle chi3, for\n               those residues containing such an angle.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.chi3' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_prot.chi4',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the side-chain torsion angle chi4, for\n               those residues containing such an angle.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.chi4' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_prot.chi5',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the side-chain torsion angle chi5, for\n               those residues containing such an angle.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.chi5' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_prot.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the residue, its\n               conformation, behaviour in refinement, or any other aspect that\n               requires annotation.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.details' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'very poor density',
              '                                 The side chain of this density may occupy\n                                  alternative conformations, but alternative\n                                  conformations were not fit in this model.',
              '                                 This residue has a close contact with the\n                                  bound inhibitor, which may account for\n                                  the nonstandard conformation of the side\n                                  chain.' ] ] ] ],
      [ 'save_',
        '_struct_mon_prot.label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.label_alt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_prot.label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_prot.label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_prot.label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_prot.auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_mon_prot.auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_mon_prot.auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_mon_prot.RSCC_all',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space (linear) correlation coefficient RSCC, as\n               described by Jones et al. (1991),  evaluated over all atoms\n               in the monomer.\n\n                      sum|p~obs~ - <p~obs~>| * sum|p~calc~ - <p~calc~>|\n               RSCC = -------------------------------------------------\n                            [ sum|p~obs~  - <p~obs~> |^2^\n                            * sum|p~calc~ - <p~calc~>|^2^ ]^1/2^\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSCC. < > indicates an average and the\n               sums are taken over all map grid  points near the relevant atoms.\n               The radius for including grid points in the calculation should\n               also be given in _struct_mon_details.RSCC.\n\n               Ref: Jones, T. A., Zou, J.-Y., Cowan, S. W. & Kjeldgaard, M.\n                    (1991). Acta Cryst. A47, 110-119.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.RSCC_all' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_prot.RSCC_main',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space (linear) correlation coefficient RSCC, as\n               described by Jones et al. (1991), evaluated over all atoms\n               in the main chain of the monomer.\n\n                      sum|p~obs~ - <p~obs~>| * sum|p~calc~ - <p~calc~>|\n               RSCC = -------------------------------------------------\n                            [ sum|p~obs~  - <p~obs~> |^2^\n                            * sum|p~calc~ - <p~calc~>|^2^ ]^1/2^\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSCC. < > indicates an average and the\n               sums are taken over all map grid  points near the relevant atoms.\n               The radius for including grid points in the calculation should\n               also be given in _struct_mon_details.RSCC.\n\n               Ref: Jones, T. A., Zou, J.-Y., Cowan, S. W. & Kjeldgaard, M.\n                    (1991). Acta Cryst. A47, 110-119.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.RSCC_main' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_prot.RSCC_side',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space (linear) correlation coefficient RSCC, as\n               described by Jones et al. (1991), evaluated over all atoms\n               in the side chain of the monomer.\n\n                      sum|p~obs~ - <p~obs~>| * sum|p~calc~ - <p~calc~>|\n               RSCC = -------------------------------------------------\n                            [ sum|p~obs~  - <p~obs~> |^2^\n                            * sum|p~calc~ - <p~calc~>|^2^ ]^1/2^\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSCC. < > indicates an average and the\n               sums are taken over all map grid  points near the relevant atoms.\n               The radius for including grid points in the calculation should\n               also be given in _struct_mon_details.RSCC.\n\n               Ref: Jones, T. A., Zou, J.-Y., Cowan, S. W. & Kjeldgaard, M.\n                    (1991). Acta Cryst. A47, 110-119.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.RSCC_side' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_prot.RSR_all',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space residual RSR, as described by Branden & Jones\n               (1990), evaluated over all atoms in the monomer.\n\n                     sum|p~obs~ - p~calc~|\n               RSR = ---------------------\n                     sum|p~obs~ + p~calc~|\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSR. The sums are taken over all map grid\n               points near the relevant atoms. The radius for including grid\n               points in the calculation should also be given in\n               _struct_mon_details.RSR.\n\n               Ref: Branden, C.-I. & Jones, T. A. (1990). Nature (London), 343,\n                    687-689.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.RSR_all' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_prot.RSR_main',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space residual RSR, as described by Branden & Jones\n               (1990), evaluated over all atoms in the main chain of the\n               monomer.\n\n                     sum|p~obs~ - p~calc~|\n               RSR = ---------------------\n                     sum|p~obs~ + p~calc~|\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSR. The sums are taken over all map grid\n               points near the relevant atoms. The radius for including grid\n               points in the calculation should also be given in\n               _struct_mon_details.RSR.\n\n               Ref: Branden, C.-I. & Jones, T. A. (1990). Nature (London), 343,\n                    687-689.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.RSR_main' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_prot.RSR_side',
        [ [ 'item_',
            '_item_description.description',
            '              The real-space residual RSR, as described by Branden & Jones\n               (1990), evaluated over all atoms in the side chain of the\n               monomer.\n\n                     sum|p~obs~ - p~calc~|\n               RSR = ---------------------\n                     sum|p~obs~ + p~calc~|\n\n               p~obs~  = the density in an \'experimental\' map\n               p~calc~ = the density in a \'calculated\' map\n\n               sum is taken over the specified grid points\n\n               Details of how these maps were calculated should be given\n               in _struct_mon_details.RSR. The sums are taken over all map grid\n               points near the relevant atoms. The radius for including grid\n               points in the calculation should also be given in\n               _struct_mon_details.RSR.\n\n               Ref: Branden, C.-I. & Jones, T. A. (1990). Nature (London), 343,\n                    687-689.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.RSR_side' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_prot.mean_B_all',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the isotropic displacement parameter for all\n               atoms in the monomer.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.mean_B_all' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_prot.mean_B_main',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the isotropic displacement parameter for atoms\n               in the main chain of the monomer.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.mean_B_main' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_prot.mean_B_side',
        [ [ 'item_',
            '_item_description.description',
            '              The mean value of the isotropic displacement parameter for atoms\n               in the side chain of the monomer.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.mean_B_side' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_mon_prot.omega',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the main-chain torsion angle omega.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.omega' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_prot.phi',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the main-chain torsion angle phi.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.phi' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_struct_mon_prot.psi',
        [ [ 'item_',
            '_item_description.description',
            '              The value in degrees of the main-chain torsion angle psi.' ],
          [ 'item_', '_item.name', '_struct_mon_prot.psi' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# STRUCT_MON_PROT_CIS ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'struct_mon_prot_cis',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_MON_PROT_CIS category identify\n               monomers that have been found to have the peptide bond in the cis\n               conformation. The criterion used to select residues to be\n               designated as containing cis peptide bonds is given in\n               _struct_mon_details.prot_cis.' ],
          [ 'item_', '_category.id', 'struct_mon_prot_cis' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_mon_prot_cis.label_alt_id',
              '_struct_mon_prot_cis.label_asym_id',
              '_struct_mon_prot_cis.label_comp_id',
              '_struct_mon_prot_cis.label_seq_id',
              '_struct_mon_prot_cis.pdbx_PDB_model_num' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB structure 1ACY of Ghiara, Stura, Stanfield,\n                Profy & Wilson [Science (1994),  264,  82-85].',
              '\n    loop_\n    _struct_mon_prot_cis.label_comp_id\n    _struct_mon_prot_cis.label_seq_id\n    _struct_mon_prot_cis.label_asym_id\n    _struct_mon_prot_cis.label_alt_id\n    _struct_mon_prot_cis.pdbx_PDB_model_num\n     PRO   8  L . 1 \n     PRO  77  L . 1\n     PRO  95  L . 1\n     PRO 141  L . 1\n     #  -----   abbreviated  -----',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot_cis.label_alt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot_cis.label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot_cis.label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot_cis.label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot_cis.auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot_cis.auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_mon_prot_cis.auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# STRUCT_NCS_DOM ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'struct_ncs_dom',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_NCS_DOM category record information\n               about the domains in an ensemble of domains related by one or\n               more noncrystallographic symmetry operators.\n\n               A domain need not correspond to a complete polypeptide chain;\n               it can be composed of one or more segments in a single chain,\n               or by segments from more than one chain.' ],
          [ 'item_', '_category.id', 'struct_ncs_dom' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_ncs_dom.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for the collagen-like\n                peptide, HYP-.',
              '\n    loop_\n    _struct_ncs_dom.id\n    _struct_ncs_dom.details\n     d1  \'Chains A, B, and C\'\n     d2  \'Chains D, E, and F\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_ncs_dom.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the structural elements that\n               comprise a domain in an ensemble of domains related by\n               noncrystallographic symmetry.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom.details' ],
          [ 'item_', '_item.category_id', 'struct_ncs_dom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The loop between residues 18 and 23 in this\n                                  domain interacts with a symmetry-related\n                                  molecule, and thus deviates significantly from\n                                  the noncrystallographic threefold.' ] ] ],
      [ 'save_',
        '_struct_ncs_dom.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_ncs_dom.id must uniquely identify a\n               record in the STRUCT_NCS_DOM list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_ncs_dom.id',
              'struct_ncs_dom',
              'yes',
              '_struct_ncs_dom_lim.dom_id',
              'struct_ncs_dom_lim',
              'yes',
              '_struct_ncs_ens_gen.dom_id_1',
              'struct_ncs_ens_gen',
              'yes',
              '_struct_ncs_ens_gen.dom_id_2',
              'struct_ncs_ens_gen',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_ncs_dom_lim.dom_id',
              '_struct_ncs_dom.id',
              '_struct_ncs_ens_gen.dom_id_1',
              '_struct_ncs_dom.id',
              '_struct_ncs_ens_gen.dom_id_2',
              '_struct_ncs_dom.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# STRUCT_NCS_DOM_LIM ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'struct_ncs_dom_lim',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_NCS_DOM_LIM category identify the\n               start and end points of polypeptide chain segments\n               that form all or part of a domain in an ensemble of domains\n               related by noncrystallographic symmetry.' ],
          [ 'item_', '_category.id', 'struct_ncs_dom_lim' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_ncs_dom_lim.dom_id',
              '_struct_ncs_dom_lim.beg_label_alt_id',
              '_struct_ncs_dom_lim.beg_label_asym_id',
              '_struct_ncs_dom_lim.beg_label_comp_id',
              '_struct_ncs_dom_lim.beg_label_seq_id',
              '_struct_ncs_dom_lim.end_label_alt_id',
              '_struct_ncs_dom_lim.end_label_asym_id',
              '_struct_ncs_dom_lim.end_label_comp_id',
              '_struct_ncs_dom_lim.end_label_seq_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for the collagen-like\n                peptide, HYP-.',
              '\n    loop_\n    _struct_ncs_dom_lim.dom_id\n    _struct_ncs_dom_lim.beg_label_alt_id\n    _struct_ncs_dom_lim.beg_label_asym_id\n    _struct_ncs_dom_lim.beg_label_comp_id\n    _struct_ncs_dom_lim.beg_label_seq_id\n    _struct_ncs_dom_lim.end_label_alt_id\n    _struct_ncs_dom_lim.end_label_asym_id\n    _struct_ncs_dom_lim.end_label_comp_id\n    _struct_ncs_dom_lim.end_label_seq_id\n     d1   .  A PRO  1  . A GLY  29\n     d1   .  B PRO  31 . B GLY  59\n     d1   .  C PRO  61 . B GLY  89\n     d2   .  D PRO  91 . D GLY 119\n     d2   .  E PRO 121 . E GLY 149\n     d2   .  F PRO 151 . F GLY 179',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.beg_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.beg_label_alt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.beg_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_ncs_dom_lim.beg_label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.beg_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_ncs_dom_lim.beg_label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.beg_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.beg_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.beg_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.beg_auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.beg_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.beg_auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.beg_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.beg_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.dom_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_ncs_dom.id in the\n               STRUCT_NCS_DOM category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.dom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.end_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.end_label_alt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.end_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_ncs_dom_lim.end_label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.end_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_ncs_dom_lim.end_label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.end_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.end_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.end_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.end_auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.end_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.end_auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.end_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.end_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# STRUCT_NCS_ENS ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'struct_ncs_ens',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_NCS_ENS category record information\n               about ensembles of domains related by noncrystallographic\n               symmetry. The point group of the ensemble when taken as a\n               whole may be specified, as well as any special aspects of the\n               ensemble that require description.' ],
          [ 'item_', '_category.id', 'struct_ncs_ens' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_ncs_ens.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for the collagen-like\n                peptide, HYP-.',
              '\n    _struct_ncs_ens.id          en1\n    _struct_ncs_ens.details\n    ; The ensemble represents the pseudo-twofold symmetry\n      between domains d1 and d2.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_ncs_ens.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the ensemble.' ],
          [ 'item_', '_item.name', '_struct_ncs_ens.details' ],
          [ 'item_', '_item.category_id', 'struct_ncs_ens' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The ensemble has a slight translation between\n                                  domains 1 and 4, but overall it can accurately\n                                  be described as point group 222' ] ] ],
      [ 'save_',
        '_struct_ncs_ens.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_ncs_ens.id must uniquely identify a\n               record in the STRUCT_NCS_ENS list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_ncs_ens.id',
              'struct_ncs_ens',
              'yes',
              '_struct_ncs_ens_gen.ens_id',
              'struct_ncs_ens_gen',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_ncs_ens_gen.ens_id', '_struct_ncs_ens.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_struct_ncs_ens.point_group',
        [ [ 'item_',
            '_item_description.description',
            '              The point group of the ensemble of structural elements related by\n               one or more noncrystallographic symmetry operations. The\n               relationships need not be precise; this data item is intended\n               to give a rough description of the noncrystallographic symmetry\n               relationships.' ],
          [ 'item_', '_item.name', '_struct_ncs_ens.point_group' ],
          [ 'item_', '_item.category_id', 'struct_ncs_ens' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '3', '422', 'non-proper' ] ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# STRUCT_NCS_ENS_GEN ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'struct_ncs_ens_gen',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_NCS_ENS_GEN category list domains\n               related by a noncrystallographic symmetry operation and\n               identify the operator.' ],
          [ 'item_', '_category.id', 'struct_ncs_ens_gen' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_ncs_ens_gen.ens_id',
              '_struct_ncs_ens_gen.dom_id_1',
              '_struct_ncs_ens_gen.dom_id_2',
              '_struct_ncs_ens_gen.oper_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for the collagen-like\n                peptide, HYP-.',
              '\n    _struct_ncs_ens_gen.dom_id_1   d1\n    _struct_ncs_ens_gen.dom_id_2   d2\n    _struct_ncs_ens_gen.ens_id     en1\n    _struct_ncs_ens_gen.oper_id    ncsop1',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_ncs_ens_gen.dom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier for the domain that will remain unchanged by the\n               transformation operator.\n\n               This data item is a pointer to _struct_ncs_dom.id in the\n               STRUCT_NCS_DOM category.' ],
          [ 'item_', '_item.name', '_struct_ncs_ens_gen.dom_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_ens_gen.dom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier for the domain that will be transformed by\n               application of the transformation operator.\n\n               This data item is a pointer to _struct_ncs_dom.id in the\n               STRUCT_NCS_DOM category.' ],
          [ 'item_', '_item.name', '_struct_ncs_ens_gen.dom_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_ens_gen.ens_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_ncs_ens.id in the\n               STRUCT_NCS_ENS category.' ],
          [ 'item_', '_item.name', '_struct_ncs_ens_gen.ens_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ncs_ens_gen.oper_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_ncs_oper.id in the\n               STRUCT_NCS_OPER category.' ],
          [ 'item_', '_item.name', '_struct_ncs_ens_gen.oper_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# STRUCT_NCS_OPER ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'struct_ncs_oper',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_NCS_OPER category describe the\n               noncrystallographic symmetry operations.\n\n               Each operator is specified as a matrix and a subsequent\n               translation vector. Operators need not represent proper\n               rotations.' ],
          [ 'item_', '_category.id', 'struct_ncs_oper' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_ncs_oper.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for the protein NS1.',
              '\n    _struct_ncs_oper.id              ncsop1\n    _struct_ncs_oper.code            given\n    _struct_ncs_oper.matrix[1][1]    0.247\n    _struct_ncs_oper.matrix[1][2]    0.935\n    _struct_ncs_oper.matrix[1][3]    0.256\n    _struct_ncs_oper.matrix[2][1]    0.929\n    _struct_ncs_oper.matrix[2][2]    0.153\n    _struct_ncs_oper.matrix[2][3]    0.337\n    _struct_ncs_oper.matrix[3][1]    0.276\n    _struct_ncs_oper.matrix[3][2]    0.321\n    _struct_ncs_oper.matrix[3][3]   -0.906\n    _struct_ncs_oper.vector[1]      -8.253\n    _struct_ncs_oper.vector[2]     -11.743\n    _struct_ncs_oper.vector[3]      -1.782\n    _struct_ncs_oper.details\n    ; Matrix and translation vector for pseudo-twofold operation.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_ncs_oper.code',
        [ [ 'item_',
            '_item_description.description',
            '              A code to indicate whether this operator describes a\n               relationship between coordinates all of which are given in the\n               data block (in which case the value of code is \'given\'), or\n               whether the operator is used to generate new coordinates from\n               those that are given in the data block (in which case the value\n               of code is \'generate\').' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.code' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'given',
              '                                 operator relates coordinates given in the\n                                  data block',
              'generate',
              '                                 operator generates new coordinates from\n                                  those given in the data block' ] ] ] ],
      [ 'save_',
        '_struct_ncs_oper.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the noncrystallographic\n               symmetry operator.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.details' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The operation is given as a precise threefold\n                                  rotation, despite the fact the best rms\n                                  fit between domain 1 and domain 2 yields a\n                                  rotation of 119.7 degrees and a translation\n                                  of 0.13 angstroms.' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_ncs_oper.id must uniquely identify a\n               record in the STRUCT_NCS_OPER list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_ncs_oper.id',
              'struct_ncs_oper',
              'yes',
              '_struct_ncs_ens_gen.oper_id',
              'struct_ncs_ens_gen',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_ncs_ens_gen.oper_id', '_struct_ncs_oper.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.matrix[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the 3x3 matrix component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.matrix[1][1]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.matrix[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the 3x3 matrix component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.matrix[1][2]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.matrix[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the 3x3 matrix component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.matrix[1][3]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.matrix[2][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][1] element of the 3x3 matrix component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.matrix[2][1]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.matrix[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the 3x3 matrix component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.matrix[2][2]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.matrix[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the 3x3 matrix component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.matrix[2][3]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.matrix[3][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][1] element of the 3x3 matrix component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.matrix[3][1]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.matrix[3][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][2] element of the 3x3 matrix component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.matrix[3][2]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.matrix[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the 3x3 matrix component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.matrix[3][3]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.vector[1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1] element of the three-element vector component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.vector[1]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.vector[2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2] element of the three-element vector component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.vector[2]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_ncs_oper.vector[3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3] element of the three-element vector component of a\n               noncrystallographic symmetry operation.' ],
          [ 'item_', '_item.name', '_struct_ncs_oper.vector[3]' ],
          [ 'item_', '_item.category_id', 'struct_ncs_oper' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           0.0' ],
          [ 'item_', '_item_sub_category.id', 'vector' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '###############' ],
      [ '@comment', '# STRUCT_REF ##' ],
      [ '@comment', '###############' ],
      [ 'save_',
        'struct_ref',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_REF category allow the author of a\n               data block to relate the entities or biological units\n               described in the data block to information archived in external\n               databases.\n\n               For references to the sequence of a polymer, the value of\n               the data item _struct_ref.seq_align is used to indicate\n               whether the correspondence between the sequence of the entity\n               or biological unit in the data block and the sequence in the\n               referenced database entry is \'complete\' or \'partial\'. If\n               this value is \'partial\', the region (or regions) of the\n               alignment may be delimited using data items in the\n               STRUCT_REF_SEQ category.\n\n               Similarly, the value of _struct_ref.seq_dif is used to indicate\n               whether the two sequences contain point differences. If the\n               value is \'yes\', the differences may be identified and annotated\n               using data items in the STRUCT_REF_SEQ_DIF category.' ],
          [ 'item_', '_category.id', 'struct_ref' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_ref.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_ref.id\n    _struct_ref.entity_id\n    _struct_ref.biol_id\n    _struct_ref.db_name\n    _struct_ref.db_code\n    _struct_ref.seq_align\n    _struct_ref.seq_dif\n    _struct_ref.details\n      1   1  .  \'Genbank\'  \'12345\' \'entire\' \'yes\' .\n      2   .  2  \'PDB\'      \'1ABC\'  .        .\n    ; The structure of the closely related compound,\n      isobutyryl-pepstatin (pepstatin A) in complex with\n      rhizopuspepsin\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_ref.biol_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_biol.id in the\n               STRUCT_BIOL category.' ],
          [ 'item_', '_item.name', '_struct_ref.biol_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_ref.db_code',
        [ [ 'item_',
            '_item_description.description',
            '              The code for this entity or biological unit or for a closely\n               related entity or biological unit in the named database.' ],
          [ 'item_', '_item.name', '_struct_ref.db_code' ],
          [ 'item_', '_item.category_id', 'struct_ref' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1ABC', 'ABCDEF' ] ] ] ],
      [ 'save_',
        '_struct_ref.db_name',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the database containing reference information about\n               this entity or biological unit.' ],
          [ 'item_', '_item.name', '_struct_ref.db_name' ],
          [ 'item_', '_item.category_id', 'struct_ref' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'PDB', 'CSD', 'Genbank' ] ] ] ],
      [ 'save_',
        '_struct_ref.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the relationship between\n               the entity or biological unit described in the data block and\n               that in the referenced database entry.' ],
          [ 'item_', '_item.name', '_struct_ref.details' ],
          [ 'item_', '_item.category_id', 'struct_ref' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_struct_ref.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_struct_ref.entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ref.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_ref.id must uniquely identify a record\n               in the STRUCT_REF list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_ref.id',
              'struct_ref',
              'yes',
              '_struct_ref_seq.ref_id',
              'struct_ref_seq',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_ref_seq.ref_id', '_struct_ref.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_struct_ref.seq_align',
        [ [ 'item_',
            '_item_description.description',
            '              A flag to indicate the scope of the alignment between the\n               sequence of the entity or biological unit described in the data\n               block and that in the referenced database entry. \'entire\'\n               indicates that alignment spans the entire length of both\n               sequences (although point differences may occur and can be\n               annotated using the data items in the STRUCT_REF_SEQ_DIF\n               category). \'partial\' indicates a partial alignment. The region\n               (or regions) of the alignment may be delimited using data items\n               in the STRUCT_REF_SEQ category. This data item may also take\n               the value \'.\', indicating that the reference is not to a\n               sequence.' ],
          [ 'item_', '_item.name', '_struct_ref.seq_align' ],
          [ 'item_', '_item.category_id', 'struct_ref' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'complete',
              'alignment is complete',
              'partial',
              'alignment is partial',
              '.',
              'reference is not to a sequence' ] ] ] ],
      [ 'save_',
        '_struct_ref.seq_dif',
        [ [ 'item_',
            '_item_description.description',
            '              A flag to indicate the presence (\'yes\') or absence (\'no\') of\n               point differences between the sequence of the entity or\n               biological unit described in the data block and that in\n               the referenced database entry. This data item may also\n               take the value \'.\', indicating that the reference is not to a\n               sequence.' ],
          [ 'item_', '_item.name', '_struct_ref.seq_dif' ],
          [ 'item_', '_item.category_id', 'struct_ref' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'no',
              'there are no point differences',
              'n',
              'abbreviation for "no"',
              'yes',
              'there are point difference',
              'y',
              'abbreviation for "yes"',
              '.',
              'reference is not to a sequence' ] ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# STRUCT_REF_SEQ ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'struct_ref_seq',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_REF_SEQ category provide a mechanism\n               for indicating and annotating a region (or regions) of alignment\n               between the sequence of an entity or biological unit described\n               in the data block and the sequence in the referenced database\n               entry.' ],
          [ 'item_', '_category.id', 'struct_ref_seq' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_ref_seq.align_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on the sequence alignment of CHER from M. xantus\n                (36 to 288) and CHER from S. typhimurium (18 to 276).',
              '\n    _struct_ref_seq.align_id          alg1\n    _struct_ref_seq.ref_id            seqdb1\n    _struct_ref_seq.seq_align_beg     36\n    _struct_ref_seq.seq_align_end     288\n    _struct_ref_seq.db_align_beg      18\n    _struct_ref_seq.db_align_end      276\n    _struct_ref_seq.details\n    ; The alignment contains 3 gaps larger than 2 residues\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_ref_seq.align_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_ref_seq.align_id must uniquely identify a\n               record in the STRUCT_REF_SEQ list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_ref_seq.align_id',
              'struct_ref_seq',
              'yes',
              '_struct_ref_seq_dif.align_id',
              'struct_ref_seq_dif',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_ref_seq_dif.align_id', '_struct_ref_seq.align_id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_struct_ref_seq.db_align_beg',
        [ [ 'item_',
            '_item_description.description',
            '              The sequence position in the referenced database entry\n               at which the alignment begins.' ],
          [ 'item_', '_item.name', '_struct_ref_seq.db_align_beg' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_range.maximum' ],
          [ '@comment', '    _item_range.minimum           .   1' ],
          [ '@comment', '                                  1   1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_struct_ref_seq.db_align_end',
        [ [ 'item_',
            '_item_description.description',
            '              The sequence position in the referenced database entry\n               at which the alignment ends.' ],
          [ 'item_', '_item.name', '_struct_ref_seq.db_align_end' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_range.maximum' ],
          [ '@comment', '    _item_range.minimum           .   1' ],
          [ '@comment', '                                  1   1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_struct_ref_seq.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the sequence alignment.' ],
          [ 'item_', '_item.name', '_struct_ref_seq.details' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_struct_ref_seq.ref_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_ref.id in the\n               STRUCT_REF category.' ],
          [ 'item_', '_item.name', '_struct_ref_seq.ref_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ref_seq.seq_align_beg',
        [ [ 'item_',
            '_item_description.description',
            '              The sequence position in the entity or biological unit described\n                in the data block at which the alignment begins.\n\n               This data item is a pointer to _entity_poly_seq.num in the\n               ENTITY_POLY_SEQ category.' ],
          [ 'item_', '_item.name', '_struct_ref_seq.seq_align_beg' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ref_seq.seq_align_end',
        [ [ 'item_',
            '_item_description.description',
            '              The sequence position in the entity or biological unit described\n               in the data block at which the alignment ends.\n\n               This data item is a pointer to _entity_poly_seq.num in the\n               ENTITY_POLY_SEQ category.' ],
          [ 'item_', '_item.name', '_struct_ref_seq.seq_align_end' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# STRUCT_REF_SEQ_DIF ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'struct_ref_seq_dif',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_REF_SEQ_DIF category provide a\n               mechanism for indicating and annotating point differences\n               between the sequence of the entity or biological unit described\n               in the data block and the sequence of the referenced database\n               entry.' ],
          [ 'item_', '_category.id', 'struct_ref_seq_dif' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_struct_ref_seq_dif.pdbx_ordinal' ],
          [ '@comment', '    loop_' ],
          [ '@comment',
            '    _category_key.name          \'_struct_ref_seq_dif.align_id\'' ],
          [ '@comment',
            '                                \'_struct_ref_seq_dif.seq_num\'' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on laboratory records for CAP-DNA complex.',
              '\n    _struct_ref_seq_dif.pdbx_ordinal 1\n    _struct_ref_seq_dif.align_id    algn2\n    _struct_ref_seq_dif.seq_num     181\n    _struct_ref_seq_dif.db_mon_id   GLU\n    _struct_ref_seq_dif.mon_id      PHE\n    _struct_ref_seq_dif.details\n    ; A point mutation was introduced in the CAP at position 181\n      substituting PHE for GLU.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.align_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_ref_seq.align_id in\n               the STRUCT_REF_SEQ  category.' ],
          [ 'item_', '_item.name', '_struct_ref_seq_dif.align_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.db_mon_id',
        [ [ 'item_',
            '_item_description.description',
            '              The monomer type found at this position in the referenced\n               database entry.\n\n               This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_struct_ref_seq_dif.db_mon_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the point differences\n               between the sequence of the entity or biological unit described\n               in the data block and that in the referenced database entry.' ],
          [ 'item_', '_item.name', '_struct_ref_seq_dif.details' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq_dif' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.mon_id',
        [ [ 'item_',
            '_item_description.description',
            '              The monomer type found at this position in the sequence of\n               the entity or biological unit described in this data block.\n\n               This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_struct_ref_seq_dif.mon_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.seq_num',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity_poly_seq.num in the\n               ENTITY_POLY_SEQ category.' ],
          [ 'item_', '_item.name', '_struct_ref_seq_dif.seq_num' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '#################' ],
      [ '@comment', '# STRUCT_SHEET ##' ],
      [ '@comment', '#################' ],
      [ 'save_',
        'struct_sheet',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_SHEET category record details about\n               the beta-sheets.' ],
          [ 'item_', '_category.id', 'struct_sheet' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_sheet.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - simple beta-barrel.\n\nN  O    N  O    N  O    N  O    N  O    N  O\n 10--11--12--13--14--15--16--17--18--19--20  strand_a\n    N  O    N  O    N  O    N  O    N  O\n    /  \\    /  \\    /  \\    /  \\    /  \\\nN  O    N  O    N  O    N  O    N  O    N  O\n 30--31--32--33--34--35--36--37--38--39--40  strand_b\n    N  O    N  O    N  O    N  O    N  O\n    /  \\    /  \\    /  \\    /  \\    /  \\\nN  O    N  O    N  O    N  O    N  O    N  O\n 50--51--52--53--54--55--56--57--58--59--60  strand_c\n    N  O    N  O    N  O    N  O    N  O\n    /  \\    /  \\    /  \\    /  \\    /  \\\nN  O    N  O    N  O    N  O    N  O    N  O\n 70--71--72--73--74--75--76--77--78--79--80  strand_d\n    N  O    N  O    N  O    N  O    N  O\n    /  \\    /  \\    /  \\    /  \\    /  \\\nN  O    N  O    N  O    N  O    N  O    N  O\n 90--91--92--93--94--95--96--97--98--99-100  strand_e\n    N  O    N  O    N  O    N  O    N  O\n    /  \\    /  \\    /  \\    /  \\    /  \\\nN  O    N  O    N  O    N  O    N  O    N  O\n110-111-112-113-114-115-116-117-118-119-120  strand_f\n    N  O    N  O    N  O    N  O    N  O\n    /  \\    /  \\    /  \\    /  \\    /  \\\nN  O    N  O    N  O    N  O    N  O    N  O\n130-131-132-133-134-135-136-137-138-139-140  strand_g\n     N  O    N  O    N  O    N  O    N  O\n   /  \\    /  \\    /  \\    /  \\    /  \\\nN  O    N  O    N  O    N  O    N  O    N  O\n150-151-152-153-154-155-156-157-158-159-160  strand_h\n     N  O    N  O    N  O    N  O    N  O\n   /  \\    /  \\    /  \\    /  \\    /  \\',
              '\n    _struct_sheet.id                sheet_1\n    _struct_sheet.type             \'beta-barrel\'\n    _struct_sheet.number_strands    8\n    _struct_sheet.details           .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - five stranded mixed-sense sheet with one two-piece strand.\n\n      N  O    N  O    N  O    N  O\n  -10--11--12--13--14--15--16--17--18->  strand_a\n  N  O    N  O    N  O    N  O    N  O\n  |  |    |  |    |  |    |  |    |  |\n  O  N    O  N    O  N    O  N    O  N\n<-119-118-117-116-115-114-113-112-111-110-  strand_b\n      O  N    O  N    O  N    O  N    O  N\n          \\  /   \\   /   \\   /   \\   /   \\\n          O  N    O  N    O  N    O  N    O  N    O  N\n         <-41--40--39--38--37--36--35--34--33--32--31--30-  strand_c\n              O  N    O  N    O  N    O  N    O  N    O  N\n              |  |    |  |    |  |    |  |    |  |    |  |\n              N  O    N  O    N  O    N  O    N  O    N  O\n   strand_d1  -50--51--52->   -90--91--92--93--95--95--96--97->  strand_d2\n                  N  O            N  O    N  O    N  O    N  O\n                  |  |    |  |    |  |    |  |    |  |    |  |\n                  O  N    O  N    O  N    O  N    O  N    O  N\n                 <-80--79--78--77--76--75--74--73--72--71--70-   strand_e\n                      O  N    O  N    O  N    O  N    O  N',
              '\n    _struct_sheet.id                sheet_2\n    _struct_sheet.type             \'five stranded, mixed-sense\'\n    _struct_sheet.number_strands    5\n    _struct_sheet.details          \'strand_d is in two pieces\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_sheet.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the beta-sheet.' ],
          [ 'item_', '_item.name', '_struct_sheet.details' ],
          [ 'item_', '_item.category_id', 'struct_sheet' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_struct_sheet.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_sheet.id must uniquely identify a record in\n               the STRUCT_SHEET list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_sheet.id',
              'struct_sheet',
              'yes',
              '_struct_sheet_hbond.sheet_id',
              'struct_sheet_hbond',
              'yes',
              '_struct_sheet_order.sheet_id',
              'struct_sheet_order',
              'yes',
              '_struct_sheet_range.sheet_id',
              'struct_sheet_range',
              'yes',
              '_struct_sheet_topology.sheet_id',
              'struct_sheet_topology',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_sheet_hbond.sheet_id',
              '_struct_sheet.id',
              '_struct_sheet_order.sheet_id',
              '_struct_sheet.id',
              '_struct_sheet_range.sheet_id',
              '_struct_sheet.id',
              '_struct_sheet_topology.sheet_id',
              '_struct_sheet.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_struct_sheet.number_strands',
        [ [ 'item_',
            '_item_description.description',
            '            The number of strands in the sheet. If a given range of residues\n             bulges out from the strands, it is still counted as one strand.\n             If a strand is composed of two different regions of polypeptide,\n             it is still counted as one strand, as long as the proper hydrogen-\n             bonding connections are made to adjacent strands.' ],
          [ 'item_', '_item.name', '_struct_sheet.number_strands' ],
          [ 'item_', '_item.category_id', 'struct_sheet' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_struct_sheet.type',
        [ [ 'item_',
            '_item_description.description',
            '              A simple descriptor for the type of the sheet.' ],
          [ 'item_', '_item.name', '_struct_sheet.type' ],
          [ 'item_', '_item.category_id', 'struct_sheet' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'jelly-roll', 'Rossmann fold', 'beta barrel' ] ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# STRUCT_SHEET_HBOND ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'struct_sheet_hbond',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_SHEET_HBOND category record details\n               about the hydrogen bonding between residue ranges in a beta-\n               sheet. It is necessary to treat hydrogen bonding independently\n               of the designation of ranges, because the hydrogen bonding may\n               begin in different places for the interactions of a given strand\n               with the one preceding it and the one following it in the sheet.' ],
          [ 'item_', '_category.id', 'struct_sheet_hbond' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_sheet_hbond.sheet_id',
              '_struct_sheet_hbond.range_id_1',
              '_struct_sheet_hbond.range_id_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - simple beta-barrel.',
              '\n    loop_\n    _struct_sheet_hbond.sheet_id\n    _struct_sheet_hbond.range_id_1\n    _struct_sheet_hbond.range_id_2\n    _struct_sheet_hbond.range_1_beg_label_seq_id\n    _struct_sheet_hbond.range_1_beg_label_atom_id\n    _struct_sheet_hbond.range_2_beg_label_seq_id\n    _struct_sheet_hbond.range_2_beg_label_atom_id\n    _struct_sheet_hbond.range_1_end_label_seq_id\n    _struct_sheet_hbond.range_1_end_label_atom_id\n    _struct_sheet_hbond.range_2_end_label_seq_id\n    _struct_sheet_hbond.range_2_end_label_atom_id\n     sheet_1  strand_a  strand_b   11  N   30  O   19  O   40  N\n     sheet_1  strand_b  strand_c   31  N   50  O   39  O   60  N\n     sheet_1  strand_c  strand_d   51  N   70  O   59  O   80  N\n     sheet_1  strand_d  strand_e   71  N   90  O   89  O  100  N\n     sheet_1  strand_e  strand_f   91  N  110  O   99  O  120  N\n     sheet_1  strand_f  strand_g  111  N  130  O  119  O  140  N\n     sheet_1  strand_g  strand_h  131  N  150  O  139  O  160  N\n     sheet_1  strand_h  strand_a  151  N   10  O  159  O  180  N',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - five stranded mixed-sense sheet with one two-piece strand.',
              '\n    loop_\n    _struct_sheet_hbond.sheet_id\n    _struct_sheet_hbond.range_id_1\n    _struct_sheet_hbond.range_id_2\n    _struct_sheet_hbond.range_1_beg_label_seq_id\n    _struct_sheet_hbond.range_1_beg_label_atom_id\n    _struct_sheet_hbond.range_2_beg_label_seq_id\n    _struct_sheet_hbond.range_2_beg_label_atom_id\n    _struct_sheet_hbond.range_1_end_label_seq_id\n    _struct_sheet_hbond.range_1_end_label_atom_id\n    _struct_sheet_hbond.range_2_end_label_seq_id\n    _struct_sheet_hbond.range_2_end_label_atom_id\n     sheet_2  strand_a   strand_b    20  N  119  O  18  O  111  N\n     sheet_2  strand_b   strand_c   110  N   33  O 118  N   41  O\n     sheet_2  strand_c   strand_d1   38  N   52  O  40  O   50  N\n     sheet_2  strand_c   strand_d2   30  N   96  O  36  O   90  N\n     sheet_2  strand_d1  strand_e    51  N   80  O  51  O   80  N\n     sheet_2  strand_d2  strand_e    91  N   76  O  97  O   70  N',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_1_beg_label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the first\n               partner of the first hydrogen bond between two residue ranges\n               in a sheet.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_1_beg_label_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_1_beg_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the first\n               partner of the first hydrogen bond between two residue ranges\n               in a sheet.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_1_beg_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_1_end_label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the first\n               partner of the last hydrogen bond between two residue ranges in\n               a sheet.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_1_end_label_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_1_end_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the first\n               partner of the last hydrogen bond between two residue ranges in\n               a sheet.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_1_end_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_2_beg_label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the second\n               partner of the first hydrogen bond between two residue ranges\n               in a sheet.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_2_beg_label_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_2_beg_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the second\n               partner of the first hydrogen bond between two residue ranges\n               in a sheet.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_2_beg_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_2_end_label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the second\n               partner of the last hydrogen bond between two residue ranges in\n               a sheet.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_2_end_label_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_2_end_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the second\n               partner of the last hydrogen bond between two residue ranges in\n               a sheet.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_2_end_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_1_beg_auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the first\n               partner of the first hydrogen bond between two residue ranges\n               in a sheet.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_1_beg_auth_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_1_beg_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the first\n               partner of the first hydrogen bond between two residue ranges\n               in a sheet.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_1_beg_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_1_end_auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the first\n               partner of the last hydrogen bond between two residue ranges in\n               a sheet.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_1_end_auth_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_1_end_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the first\n               partner of the last hydrogen bond between two residue ranges in\n               a sheet.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_1_end_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_2_beg_auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the second\n               partner of the first hydrogen bond between two residue ranges\n               in a sheet.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_2_beg_auth_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_2_beg_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the second\n               partner of the first hydrogen bond between two residue ranges\n               in a sheet.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_2_beg_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_2_end_auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the second\n               partner of the last hydrogen bond between two residue ranges in\n               a sheet.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_2_end_auth_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_2_end_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue for the second\n               partner of the last hydrogen bond between two residue ranges in\n               a sheet.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.range_2_end_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet_range.id in\n               the STRUCT_SHEET_RANGE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_hbond.range_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.range_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet_range.id in\n               the STRUCT_SHEET_RANGE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_hbond.range_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.sheet_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet.id in the\n               STRUCT_SHEET category.' ],
          [ 'item_', '_item.name', '_struct_sheet_hbond.sheet_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# STRUCT_SHEET_ORDER ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'struct_sheet_order',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_SHEET_ORDER category record details\n               about the order of the residue ranges that form a beta-sheet.\n               All order links are pairwise and the specified pairs are\n               assumed to be adjacent to one another in the sheet. These data\n               items are an alternative to the STRUCT_SHEET_TOPOLOGY data\n               items and they allow all manner of sheets to be described.' ],
          [ 'item_', '_category.id', 'struct_sheet_order' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_sheet_order.sheet_id',
              '_struct_sheet_order.range_id_1',
              '_struct_sheet_order.range_id_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - simple beta-barrel.',
              '\n    loop_\n    _struct_sheet_order.sheet_id\n    _struct_sheet_order.range_id_1\n    _struct_sheet_order.range_id_2\n    _struct_sheet_order.offset\n    _struct_sheet_order.sense\n     sheet_1  strand_a  strand_b  +1  parallel\n     sheet_1  strand_b  strand_c  +1  parallel\n     sheet_1  strand_c  strand_d  +1  parallel\n     sheet_1  strand_d  strand_e  +1  parallel\n     sheet_1  strand_e  strand_f  +1  parallel\n     sheet_1  strand_f  strand_g  +1  parallel\n     sheet_1  strand_g  strand_h  +1  parallel\n     sheet_1  strand_h  strand_a  +1  parallel',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - five stranded mixed-sense sheet with one two-piece strand.',
              '\n    loop_\n    _struct_sheet_order.sheet_id\n    _struct_sheet_order.range_id_1\n    _struct_sheet_order.range_id_2\n    _struct_sheet_order.offset\n    _struct_sheet_order.sense\n     sheet_2  strand_a   strand_b    +1 anti-parallel\n     sheet_2  strand_b   strand_c    +1 parallel\n     sheet_2  strand_c   strand_d1   +1 anti-parallel\n     sheet_2  strand_c   strand_d2   +1 anti-parallel\n     sheet_2  strand_d1  strand_e    +1 anti-parallel\n     sheet_2  strand_d2  strand_e    +1 anti-parallel',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_sheet_order.offset',
        [ [ 'item_',
            '_item_description.description',
            '              Designates the relative position in the sheet, plus or minus, of\n               the second residue range to the first.' ],
          [ 'item_', '_item.name', '_struct_sheet_order.offset' ],
          [ 'item_', '_item.category_id', 'struct_sheet_order' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_struct_sheet_order.range_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet_range.id in\n               the STRUCT_SHEET_RANGE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_order.range_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_order.range_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet_range.id in\n               the STRUCT_SHEET_RANGE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_order.range_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_order.sense',
        [ [ 'item_',
            '_item_description.description',
            '              A flag to indicate whether the two designated residue ranges are\n               parallel or antiparallel to one another.' ],
          [ 'item_', '_item.name', '_struct_sheet_order.sense' ],
          [ 'item_', '_item.category_id', 'struct_sheet_order' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'parallel', 'anti-parallel' ] ] ] ],
      [ 'save_',
        '_struct_sheet_order.sheet_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet.id in the\n               STRUCT_SHEET category.' ],
          [ 'item_', '_item.name', '_struct_sheet_order.sheet_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# STRUCT_SHEET_RANGE ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'struct_sheet_range',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_SHEET_RANGE category record details\n               about the residue ranges that form a beta-sheet. Residues are\n               included in a range if they made beta-sheet-type hydrogen-bonding\n               interactions with at least one adjacent strand and if there are\n               at least two residues in the range.' ],
          [ 'item_', '_category.id', 'struct_sheet_range' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_sheet_range.sheet_id', '_struct_sheet_range.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - simple beta-barrel.',
              '\n    loop_\n    _struct_sheet_range.sheet_id\n    _struct_sheet_range.id\n    _struct_sheet_range.beg_label_comp_id\n    _struct_sheet_range.beg_label_asym_id\n    _struct_sheet_range.beg_label_seq_id\n    _struct_sheet_range.end_label_comp_id\n    _struct_sheet_range.end_label_asym_id\n    _struct_sheet_range.end_label_seq_id\n    _struct_sheet_range.symmetry\n     sheet_1  strand_a  ala  A   20  ala  A   30  1_555\n     sheet_1  strand_b  ala  A   40  ala  A   50  1_555\n     sheet_1  strand_c  ala  A   60  ala  A   70  1_555\n     sheet_1  strand_d  ala  A   80  ala  A   90  1_555\n     sheet_1  strand_e  ala  A  100  ala  A  110  1_555\n     sheet_1  strand_f  ala  A  120  ala  A  130  1_555\n     sheet_1  strand_g  ala  A  140  ala  A  150  1_555\n     sheet_1  strand_h  ala  A  160  ala  A  170  1_555',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - five stranded mixed-sense sheet with one two-piece strand.',
              '\n    loop_\n    _struct_sheet_range.sheet_id\n    _struct_sheet_range.id\n    _struct_sheet_range.beg_label_comp_id\n    _struct_sheet_range.beg_label_asym_id\n    _struct_sheet_range.beg_label_seq_id\n    _struct_sheet_range.end_label_comp_id\n    _struct_sheet_range.end_label_asym_id\n    _struct_sheet_range.end_label_seq_id\n    _struct_sheet_range.symmetry\n     sheet_2  strand_a   ala  A  10  ala  A  18  1_555\n     sheet_2  strand_b   ala  A 110  ala  A 119  1_555\n     sheet_2  strand_c   ala  A  30  ala  A  41  1_555\n     sheet_2  strand_d1  ala  A  50  ala  A  52  1_555\n     sheet_2  strand_d2  ala  A  90  ala  A  97  1_555\n     sheet_2  strand_e   ala  A  70  ala  A  80  1_555',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_sheet_range.beg_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range begins.\n\n               This data item is a pointer to _struct_asym.id in the\n               STRUCT_ASYM category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_range.beg_label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_range.beg_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range begins.\n\n               This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_range.beg_label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_range.beg_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range begins.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_range.beg_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_range.end_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range ends.\n\n               This data item is a pointer to _struct_asym.id in the\n               STRUCT_ASYM category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_range.end_label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_range.end_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range ends.\n\n               This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_range.end_label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_range.end_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range ends.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_range.end_label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_range.beg_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range begins.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_range.beg_auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_range.beg_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range begins.\n\n               This data item is a pointer to _atom_site.auth_comp_id in\n               the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_range.beg_auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_range.beg_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range begins.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_range.beg_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_range.end_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range ends.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.\n' ],
          [ 'item_', '_item.name', '_struct_sheet_range.end_auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_range.end_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range ends.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_range.end_auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_range.end_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta-sheet range ends.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_range.end_auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_sheet_range.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_sheet_range.id must uniquely identify a\n               range in a given sheet in the STRUCT_SHEET_RANGE list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_sheet_range.id',
              'struct_sheet_range',
              'yes',
              '_struct_sheet_hbond.range_id_1',
              'struct_sheet_hbond',
              'yes',
              '_struct_sheet_hbond.range_id_2',
              'struct_sheet_hbond',
              'yes',
              '_struct_sheet_order.range_id_1',
              'struct_sheet_order',
              'yes',
              '_struct_sheet_order.range_id_2',
              'struct_sheet_order',
              'yes',
              '_struct_sheet_topology.range_id_1',
              'struct_sheet_topology',
              'yes',
              '_struct_sheet_topology.range_id_2',
              'struct_sheet_topology',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_sheet_hbond.range_id_1',
              '_struct_sheet_range.id',
              '_struct_sheet_hbond.range_id_2',
              '_struct_sheet_range.id',
              '_struct_sheet_order.range_id_1',
              '_struct_sheet_range.id',
              '_struct_sheet_order.range_id_2',
              '_struct_sheet_range.id',
              '_struct_sheet_topology.range_id_1',
              '_struct_sheet_range.id',
              '_struct_sheet_topology.range_id_2',
              '_struct_sheet_range.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_struct_sheet_range.sheet_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet.id in the\n               STRUCT_SHEET category.' ],
          [ 'item_', '_item.name', '_struct_sheet_range.sheet_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_range.symmetry',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               residues delimited by the start and end designators in\n               order to generate the appropriate strand in this sheet.' ],
          [ 'item_', '_item.name', '_struct_sheet_range.symmetry' ],
          [ 'item_', '_item.category_id', 'struct_sheet_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           1_555' ],
          [ 'item_', '_item_type.code', 'symop' ] ] ],
      [ '@comment', '##########################' ],
      [ '@comment', '# STRUCT_SHEET_TOPOLOGY ##' ],
      [ '@comment', '##########################' ],
      [ 'save_',
        'struct_sheet_topology',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_SHEET_TOPOLOGY category record details\n               about the topology of the residue ranges that form a beta-sheet.\n               All topology links are pairwise and the specified pairs are\n               assumed to be successive in the amino-acid sequence. These\n               data items are useful in describing various simple and complex\n               folds, but they become inadequate when the strands in the sheet\n               come from more than one chain. The\n               STRUCT_SHEET_ORDER data items can be used to describe\n               single- and multiple-chain-containing sheets.' ],
          [ 'item_', '_category.id', 'struct_sheet_topology' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_sheet_topology.sheet_id',
              '_struct_sheet_topology.range_id_1',
              '_struct_sheet_topology.range_id_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - simple beta-barrel.',
              '\n    loop_\n    _struct_sheet_topology.sheet_id\n    _struct_sheet_topology.range_id_1\n    _struct_sheet_topology.range_id_2\n    _struct_sheet_topology.offset\n    _struct_sheet_topology.sense\n     sheet_1  strand_a  strand_b  +1  parallel\n     sheet_1  strand_b  strand_c  +1  parallel\n     sheet_1  strand_c  strand_d  +1  parallel\n     sheet_1  strand_d  strand_e  +1  parallel\n     sheet_1  strand_e  strand_f  +1  parallel\n     sheet_1  strand_f  strand_g  +1  parallel\n     sheet_1  strand_g  strand_h  +1  parallel\n     sheet_1  strand_h  strand_a  +1  parallel',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    Example 2 - five stranded mixed-sense sheet with one two-piece strand.',
              '\n    loop_\n    _struct_sheet_topology.sheet_id\n    _struct_sheet_topology.range_id_1\n    _struct_sheet_topology.range_id_2\n    _struct_sheet_topology.offset\n    _struct_sheet_topology.sense\n     sheet_2  strand_a   strand_c    +2 anti-parallel\n     sheet_2  strand_c   strand_d1   +1 anti-parallel\n     sheet_2  strand_d1  strand_e    +1 anti-parallel\n     sheet_2  strand_e   strand_d2   -1 anti-parallel\n     sheet_2  strand_d2  strand_b    -2 anti-parallel',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_sheet_topology.offset',
        [ [ 'item_',
            '_item_description.description',
            '              Designates the relative position in the sheet, plus or minus, of\n               the second residue range to the first.' ],
          [ 'item_', '_item.name', '_struct_sheet_topology.offset' ],
          [ 'item_', '_item.category_id', 'struct_sheet_topology' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_struct_sheet_topology.range_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet_range.id in\n               the STRUCT_SHEET_RANGE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_topology.range_id_1' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_topology.range_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet_range.id in\n               the STRUCT_SHEET_RANGE category.' ],
          [ 'item_', '_item.name', '_struct_sheet_topology.range_id_2' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_sheet_topology.sense',
        [ [ 'item_',
            '_item_description.description',
            '              A flag to indicate whether the two designated residue ranges are\n               parallel or antiparallel to one another.' ],
          [ 'item_', '_item.name', '_struct_sheet_topology.sense' ],
          [ 'item_', '_item.category_id', 'struct_sheet_topology' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'parallel', 'anti-parallel' ] ] ] ],
      [ 'save_',
        '_struct_sheet_topology.sheet_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet.id in the\n               STRUCT_SHEET category.' ],
          [ 'item_', '_item.name', '_struct_sheet_topology.sheet_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# STRUCT_SITE ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'struct_site',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_SITE category record details about\n               portions of the structure that contribute to structurally\n               relevant sites (e.g. active sites, substrate-binding subsites,\n               metal-coordination sites).' ],
          [ 'item_', '_category.id', 'struct_site' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_site.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_site.id\n    _struct_site.details\n     \'P2 site C\'\n    ; residues with a contact < 3.7 \\%A to an atom in the P2\n      moiety of the inhibitor in the conformation with\n      _struct_asym.id = C\n    ;\n     \'P2 site D\'\n    ; residues with a contact < 3.7 \\%A to an atom in the P1\n      moiety of the inhibitor in the conformation with\n      _struct_asym.id = D)\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_site.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the site.' ],
          [ 'item_', '_item.name', '_struct_site.details' ],
          [ 'item_', '_item.category_id', 'struct_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_struct_site.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_site.id must uniquely identify a record in\n               the STRUCT_SITE list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_struct_site.id',
              'struct_site',
              'yes',
              '_struct_site_gen.site_id',
              'struct_site_gen',
              'yes',
              '_struct_site_keywords.site_id',
              'struct_site_keywords',
              'yes',
              '_struct_site_view.site_id',
              'struct_site_view',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_struct_site_gen.site_id',
              '_struct_site.id',
              '_struct_site_keywords.site_id',
              '_struct_site.id',
              '_struct_site_view.site_id',
              '_struct_site.id' ] ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# STRUCT_SITE_GEN ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'struct_site_gen',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_SITE_GEN category record details about\n               the generation of portions of the structure that contribute to\n               structurally relevant sites.' ],
          [ 'item_', '_category.id', 'struct_site_gen' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_site_gen.id', '_struct_site_gen.site_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_site_gen.id\n    _struct_site_gen.site_id\n    _struct_site_gen.label_comp_id\n    _struct_site_gen.label_asym_id\n    _struct_site_gen.label_seq_id\n    _struct_site_gen.symmetry\n    _struct_site_gen.details\n      1  1  VAL  A   32  1_555  .\n      2  1  ILE  A   47  1_555  .\n      3  1  VAL  A   82  1_555  .\n      4  1  ILE  A   84  1_555  .\n      5  2  VAL  B  232  1_555  .\n      6  2  ILE  B  247  1_555  .\n      7  2  VAL  B  282  1_555  .\n      8  2  ILE  B  284  1_555  .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_site_gen.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the symmetry generation of\n               this portion of the structural site.' ],
          [ 'item_', '_item.name', '_struct_site_gen.details' ],
          [ 'item_', '_item.category_id', 'struct_site_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The zinc atom lies on a special position;\n                                  application of symmetry elements to generate\n                                  the insulin hexamer will generate excess zinc\n                                  atoms, which must be removed by hand.' ] ] ],
      [ 'save_',
        '_struct_site_gen.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_site_gen.id must uniquely identify a record\n               in the STRUCT_SITE_GEN list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_struct_site_gen.id' ],
          [ 'item_', '_item.category_id', 'struct_site_gen' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_struct_site_gen.label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_struct_site_gen.label_alt_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_site_gen.label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_site_gen.label_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_site_gen.label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _chem_comp_atom.atom_id in the\n               CHEM_COMP_ATOM category.' ],
          [ 'item_', '_item.name', '_struct_site_gen.label_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_site_gen.label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_site_gen.label_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_site_gen.label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_site_gen.label_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_site_gen.auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_site_gen.auth_asym_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_site_gen.auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.auth_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_site_gen.auth_atom_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_site_gen.auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_site_gen.auth_comp_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_site_gen.auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for participants in the site.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_site_gen.auth_seq_id' ],
          [ 'item_', '_item.mandatory_code', 'no' ] ] ],
      [ 'save_',
        '_struct_site_gen.site_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_site.id in the STRUCT_SITE\n               category.' ],
          [ 'item_', '_item.name', '_struct_site_gen.site_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_site_gen.symmetry',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               atom set specified by _struct_site_gen.label* to generate a\n               portion of the site.' ],
          [ 'item_', '_item.name', '_struct_site_gen.symmetry' ],
          [ 'item_', '_item.category_id', 'struct_site_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_default.value           1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# STRUCT_SITE_KEYWORDS ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'struct_site_keywords',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_SITE_KEYWORDS category record\n               keywords describing the site.' ],
          [ 'item_', '_category.id', 'struct_site_keywords' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_struct_site_keywords.site_id',
              '_struct_site_keywords.text' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _struct_site_keywords.site_id\n    _struct_site_keywords.text\n     \'P2 site C\'  \'binding site\'\n     \'P2 site C\'  \'binding pocket\'\n     \'P2 site C\'  \'P2 site\'\n     \'P2 site C\'  \'P2 pocket\'\n     \'P2 site D\'  \'binding site\'\n     \'P2 site D\'  \'binding pocket\'\n     \'P2 site D\'  \'P2 site\'\n     \'P2 site D\'  \'P2 pocket\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_site_keywords.site_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_site.id in the STRUCT_SITE\n               category.' ],
          [ 'item_', '_item.name', '_struct_site_keywords.site_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_struct_site_keywords.text',
        [ [ 'item_',
            '_item_description.description',
            '              Keywords describing this site.' ],
          [ 'item_', '_item.name', '_struct_site_keywords.text' ],
          [ 'item_', '_item.category_id', 'struct_site_keywords' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'active site', 'binding pocket', 'Ca coordination' ] ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# STRUCT_SITE_VIEW ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'struct_site_view',
        [ [ 'item_',
            '_category.description',
            '              Data items in the STRUCT_SITE_VIEW category record details\n               about how to draw and annotate an informative view of the\n               site.' ],
          [ 'item_', '_category.id', 'struct_site_view' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_struct_site_view.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on NDB structure GDL001 by Coll, Aymami,\n                Van Der Marel, Van Boom, Rich & Wang\n                [Biochemistry (1989), 28, 310-320].',
              '\n    _struct_site_view.id                    1\n    _struct_site_view.rot_matrix[1][1]      0.132\n    _struct_site_view.rot_matrix[1][2]      0.922\n    _struct_site_view.rot_matrix[1][3]     -0.363\n    _struct_site_view.rot_matrix[2][1]      0.131\n    _struct_site_view.rot_matrix[2][2]     -0.380\n    _struct_site_view.rot_matrix[2][3]     -0.916\n    _struct_site_view.rot_matrix[3][1]     -0.982\n    _struct_site_view.rot_matrix[3][2]      0.073\n    _struct_site_view.rot_matrix[3][3]     -0.172\n    _struct_site_view.details\n    ; This view highlights the site of ATAT-Netropsin\n      interaction.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_struct_site_view.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of this view of the\n               site. This data item can be used as a figure legend.' ],
          [ 'item_', '_item.name', '_struct_site_view.details' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The active site has been oriented with the\n                                  specificity pocket on the right and the active\n                                  site machinery on the left.' ] ] ],
      [ 'save_',
        '_struct_site_view.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _struct_site_view.id must uniquely identify a\n               record in the STRUCT_SITE_VIEW list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_struct_site_view.id' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Figure 1',
              'unliganded enzyme',
              'view down enzyme active site' ] ] ] ],
      [ 'save_',
        '_struct_site_view.rot_matrix[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][1] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_SITE_GEN category to an orientation useful for\n               visualizing the site. The conventions used in the rotation are\n               described in _struct_site_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_site_view.rot_matrix[1][1]' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_site_view.rot_matrix[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][2] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_SITE_GEN category to an orientation useful for\n               visualizing the site. The conventions used in the rotation are\n               described in _struct_site_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_site_view.rot_matrix[1][2]' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_site_view.rot_matrix[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [1][3] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_SITE_GEN category to an orientation useful for\n               visualizing the site. The conventions used in the rotation are\n               described in _struct_site_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_site_view.rot_matrix[1][3]' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_site_view.rot_matrix[2][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][1] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_SITE_GEN category to an orientation useful for\n               visualizing the site. The conventions used in the rotation are\n               described in _struct_site_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_site_view.rot_matrix[2][1]' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_site_view.rot_matrix[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][2] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_SITE_GEN category to an orientation useful for\n               visualizing the site. The conventions used in the rotation are\n               described in _struct_site_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_site_view.rot_matrix[2][2]' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_site_view.rot_matrix[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [2][3] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_SITE_GEN category to an orientation useful for\n               visualizing the site. The conventions used in the rotation are\n               described in _struct_site_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_site_view.rot_matrix[2][3]' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_site_view.rot_matrix[3][1]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][1] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_SITE_GEN category to an orientation useful for\n               visualizing the site. The conventions used in the rotation are\n               described in _struct_site_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_site_view.rot_matrix[3][1]' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_site_view.rot_matrix[3][2]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][2] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_SITE_GEN category to an orientation useful for\n               visualizing the site. The conventions used in the rotation are\n               described in _struct_site_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_site_view.rot_matrix[3][2]' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_site_view.rot_matrix[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '              The [3][3] element of the matrix used to rotate the subset of the\n               Cartesian coordinates in the ATOM_SITE category identified in the\n               STRUCT_SITE_GEN category an orientation useful for visualizing\n               the site. The conventions used in the rotation are\n               described in _struct_site_view.details.\n\n               |x\'|                         |11 12 13| |x|\n               |y\'|~reoriented Cartesian~ = |21 22 23| |y|~Cartesian~\n               |z\'|                         |31 32 33| |z|' ],
          [ 'item_', '_item.name', '_struct_site_view.rot_matrix[3][3]' ],
          [ 'item_', '_item.category_id', 'struct_site_view' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_sub_category.id', 'matrix' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_site_view.site_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_site.id in the STRUCT_SITE\n               category.' ],
          [ 'item_', '_item.name', '_struct_site_view.site_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ '@comment', '#############' ],
      [ '@comment', '# SYMMETRY ##' ],
      [ '@comment', '#############' ],
      [ 'save_',
        'symmetry',
        [ [ 'item_',
            '_category.description',
            '              Data items in the SYMMETRY category record details about the\n               space-group symmetry.' ],
          [ 'item_', '_category.id', 'symmetry' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_symmetry.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'symmetry_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    _symmetry.entry_id                     \'5HVP\'\n    _symmetry.cell_setting                 orthorhombic\n    _symmetry.Int_Tables_number            18\n    _symmetry.space_group_name_H-M         \'P 21 21 2\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_symmetry.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_symmetry.entry_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_symmetry.cell_setting',
        [ [ 'item_',
            '_item_description.description',
            '              The cell settings for this space-group symmetry.' ],
          [ 'item_', '_item.name', '_symmetry.cell_setting' ],
          [ 'item_', '_item.category_id', 'symmetry' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_aliases.alias_name', '_symmetry_cell_setting' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'triclinic',
              'monoclinic',
              'orthorhombic',
              'tetragonal',
              'rhombohedral',
              'trigonal',
              'hexagonal',
              'cubic' ] ] ] ],
      [ 'save_',
        '_symmetry.Int_Tables_number',
        [ [ 'item_',
            '_item_description.description',
            '              Space-group number from International Tables for Crystallography\n               Vol. A (2002).' ],
          [ 'item_', '_item.name', '_symmetry.Int_Tables_number' ],
          [ 'item_', '_item.category_id', 'symmetry' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_symmetry_Int_Tables_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_symmetry.space_group_name_Hall',
        [ [ 'item_',
            '_item_description.description',
            '              Space-group symbol as described by Hall (1981). This symbol\n               gives the space-group setting explicitly. Leave spaces between\n               the separate components of the symbol.\n\n               Ref: Hall, S. R. (1981). Acta Cryst. A37, 517-525; erratum\n               (1981) A37, 921.' ],
          [ 'item_', '_item.name', '_symmetry.space_group_name_Hall' ],
          [ 'item_', '_item.category_id', 'symmetry' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_symmetry_space_group_name_Hall' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '-P 2ac 2n', '-R 3 2"', 'P 61 2 2 (0 0 -1)' ] ] ] ],
      [ 'save_',
        '_symmetry.space_group_name_H-M',
        [ [ 'item_',
            '_item_description.description',
            '              Hermann-Mauguin space-group symbol. Note that the\n               Hermann-Mauguin symbol does not necessarily contain complete\n               information about the symmetry and the space-group origin. If\n               used, always supply the FULL symbol from International Tables\n               for Crystallography Vol. A (2002) and indicate the origin and\n               the setting if it is not implicit. If there is any doubt that\n               the equivalent positions can be uniquely deduced from this\n               symbol, specify the  _symmetry_equiv.pos_as_xyz or\n               _symmetry.space_group_name_Hall  data items as well. Leave\n               spaces between symbols referring to\n               different axes.' ],
          [ 'item_', '_item.name', '_symmetry.space_group_name_H-M' ],
          [ 'item_', '_item.category_id', 'symmetry' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_symmetry_space_group_name_H-M' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'P 1 21/m 1', 'P 2/n 2/n 2/n (origin at -1)', 'R -3 2/m' ] ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# SYMMETRY_EQUIV ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'symmetry_equiv',
        [ [ 'item_',
            '_category.description',
            '              Data items in the SYMMETRY_EQUIV category list the\n               symmetry-equivalent positions for the space group.' ],
          [ 'item_', '_category.id', 'symmetry_equiv' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_symmetry_equiv.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'symmetry_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP.',
              '\n    loop_\n    _symmetry_equiv.id\n    _symmetry_equiv.pos_as_xyz\n      1  \'+x,+y,+z\'\n      2  \'-x,-y,z\'\n      3  \'1/2+x,1/2-y,-z\'\n      4  \'1/2-x,1/2+y,-z\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_symmetry_equiv.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _symmetry_equiv.id must uniquely identify\n               a record in the SYMMETRY_EQUIV category.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_symmetry_equiv.id' ],
          [ 'item_', '_item.category_id', 'symmetry_equiv' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_symmetry_equiv_pos_site_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_symmetry_equiv.pos_as_xyz',
        [ [ 'item_',
            '_item_description.description',
            '              Symmetry-equivalent position in the \'xyz\' representation. Except\n               for the space group P1, these data will be repeated in a loop.\n               The format of the data item is as per International Tables for\n               Crystallography Vol. A (2002). All equivalent positions should\n               be entered, including those for lattice centring and a centre of\n               symmetry, if present.' ],
          [ 'item_', '_item.name', '_symmetry_equiv.pos_as_xyz' ],
          [ 'item_', '_item.category_id', 'symmetry_equiv' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_symmetry_equiv_pos_as_xyz' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.0.1' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', '-y+x,-y,1/3+z' ] ] ],
      [ '@comment', '## EOF mmcif_std-def-1.dic' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment', ' File:  mmcif_std-def-2.dic' ],
      [ '@comment', '                      mmCIF Data Dictionary' ],
      [ '@comment',
        ' This data dictionary contains the standard mmCIF data definitions.' ],
      [ '@comment', '                           Defintion Section 2' ],
      [ '@comment',
        ' Section 2 contains definitions added to maintain correspondence' ],
      [ '@comment', ' with the Core CIF dictionary  V2.3' ],
      [ '@comment',
        '##########################################################################' ],
      [ 'save_',
        '_atom_site.adp_type',
        [ [ 'item_',
            '_item_description.description',
            '              A standard code used to describe the type of atomic displacement\n               parameters used for the site.' ],
          [ 'item_', '_item.name', '_atom_site.adp_type' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.thermal_displace_type' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Uani',
              'anisotropic Uij',
              'Uiso',
              'isotropic U',
              'Uovl',
              'overall U',
              'Umpe',
              'multipole expansion U',
              'Bani',
              'anisotropic Bij',
              'Biso',
              'isotropic B',
              'Bovl',
              'overall B' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_atom_site_adp_type', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_atom_site.refinement_flags',
        [ [ 'item_',
            '_item_description.description',
            '           A concatenated series of single-letter codes which indicate the\n            refinement restraints or constraints applied to this site.  This\n            item should not be used.  It has been replaced by\n            _atom_site.refinement_flags_posn, *_adp and *_occupancy. It is\n            retained in this dictionary only to provide compatibility with\n            old CIFs.' ],
          [ 'item_', '_item.name', '_atom_site.refinement_flags' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_atom_site.refinement_flags_posn',
              'replaces',
              '_atom_site.refinement_flags_adp',
              'replaces',
              '_atom_site.refinement_flags_occupancy',
              'replaces' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '.',
              'no refinement constraints',
              'S',
              'special-position constraint on site',
              'G',
              'rigid-group refinement of site',
              'R',
              'riding-atom site attached to non-riding atom',
              'D',
              'distance or angle restraint on site',
              'T',
              'thermal displacement constraints',
              'U',
              'Uiso or Uij restraint (rigid bond)',
              'P',
              'partial occupancy constraint' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_atom_site_refinement_flags', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_atom_site.refinement_flags_adp',
        [ [ 'item_',
            '_item_description.description',
            '              A code which indicates the refinement restraints or constraints\n               applied to the atomic displacement parameters of this site.' ],
          [ 'item_', '_item.name', '_atom_site.refinement_flags_adp' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.refinement_flags_posn' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '.',
              'no constraints on atomic displacement parameters',
              'T',
              'special-position constraints on atomic displacement parameters',
              'U',
              'Uiso or Uij restraint (rigid bond)',
              'TU',
              'both constraints applied' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_atom_site_refinement_flags_adp', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_atom_site.refinement_flags_occupancy',
        [ [ 'item_',
            '_item_description.description',
            '              A code which indicates that refinement restraints or\n               constraints were applied to the occupancy of this site.' ],
          [ 'item_',
            '_item.name',
            '_atom_site.refinement_flags_occupancy' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.refinement_flags_posn' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '.',
              'no constraints on site-occupancy parameters',
              'P',
              'site-occupancy constraint' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_atom_site_refinement_flags_occupancy',
              'cif_core.dic',
              '2.3' ] ] ] ],
      [ 'save_',
        '_atom_site.refinement_flags_posn',
        [ [ 'item_',
            '_item_description.description',
            '              A code which indicates the refinement restraints or constraints\n               applied to the positional coordinates of this site.' ],
          [ 'item_', '_item.name', '_atom_site.refinement_flags_posn' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_related.related_name',
            '_atom_site.refinement_flags_posn' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '.',
              'no constraints on  positional coordinates',
              'D',
              'distance or angle restraint on positional coordinates',
              'G',
              'rigid-group refinement of positional coordinates',
              'R',
              'riding-atom site attached to non-riding atom',
              'S',
              'special-position constraint on positional coordinates',
              'DG',
              'combination of the above constraints',
              'DR',
              'combination of the above constraints',
              'DS',
              'combination of the above constraints',
              'GR',
              'combination of the above constraints',
              'GS',
              'combination of the above constraints',
              'RS',
              'combination of the above constraints',
              'DGR',
              'combination of the above constraints',
              'DGS',
              'combination of the above constraints',
              'DRS',
              'combination of the above constraints',
              'GRS',
              'combination of the above constraints',
              'DGRS',
              'combination of the above constraints' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_atom_site_refinement_flags_posn', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_atom_sites.special_details',
        [ [ 'item_',
            '_item_description.description',
            '              Additional information about the atomic coordinates not coded\n               elsewhere in the CIF.' ],
          [ 'item_', '_item.name', '_atom_sites.special_details' ],
          [ 'item_', '_item.category_id', 'atom_sites' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_atom_sites_special_details', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_atom_type.scat_dispersion_source',
        [ [ 'item_',
            '_item_description.description',
            '              Reference to the source of the real and imaginary dispersion\n               corrections for scattering factors used for this atom type.' ],
          [ 'item_', '_item.name', '_atom_type.scat_dispersion_source' ],
          [ 'item_', '_item.category_id', 'atom_type' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'International Tables Vol. IV Table 2.3.1' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_atom_type_scat_dispersion_source', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', '###############' ],
      [ '@comment', '# AUDIT_LINK ##' ],
      [ '@comment', '###############' ],
      [ 'save_',
        'audit_link',
        [ [ 'item_',
            '_category.description',
            '              Data items in the AUDIT_LINK category record details about the\n               relationships between data blocks in the current CIF.' ],
          [ 'item_', '_category.id', 'audit_link' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_audit_link.block_code', '_audit_link.block_description' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'audit_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              '_category_examples.detail',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    loop_\n    _audit_link.block_code\n    _audit_link.block_description\n       .             \'discursive text of paper with two structures\'\n       morA_(1)      \'structure 1 of 2\'\n       morA_(2)      \'structure 2 of 2\'',
              '\n    Example 1 - multiple structure paper, as illustrated\n                in A Guide to CIF for Authors (1995). IUCr: Chester.',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n    loop_\n    _audit_link.block_code\n    _audit_link.block_description\n       .        \'publication details\'\n       KSE_COM  \'experimental data common to ref./mod. structures\'\n       KSE_REF  \'reference structure\'\n       KSE_MOD  \'modulated structure\'',
              '\n    Example 2 - example file for the one-dimensional incommensurately\n                modulated structure of K~2~SeO~4~.',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_audit_link.block_code',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _audit_block.code associated with a data block\n               in the current file related to the current data block. The\n               special value \'.\' may be used to refer to the current data\n               block for completeness.' ],
          [ 'item_', '_item.name', '_audit_link.block_code' ],
          [ 'item_', '_item.category_id', 'audit_link' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_audit_link_block_code', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_audit_link.block_description',
        [ [ 'item_',
            '_item_description.description',
            '              A textual description of the relationship of the referenced\n               data block to the current one.' ],
          [ 'item_', '_item.name', '_audit_link.block_description' ],
          [ 'item_', '_item.category_id', 'audit_link' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_audit_link_block_description', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_cell.reciprocal_angle_alpha',
        [ [ 'item_',
            '_item_description.description',
            '              The angle (recip-alpha) defining the reciprocal cell in degrees.\n               (recip-alpha), (recip-alpha) and (recip-alpha) related to the\n               angles in the real cell by:\n\n               cos(recip-alpha)\n                   = [cos(beta)*cos(gamma) - cos(alpha)]/[sin(beta)*sin(gamma)]\n\n               cos(recip-beta)\n                   = [cos(gamma)*cos(alpha) - cos(beta)]/[sin(gamma)*sin(alpha)]\n\n               cos(recip-gamma)\n                   = [cos(alpha)*cos(beta) - cos(gamma)]/[sin(alpha)*sin(beta)]\n\n               Ref: Buerger, M. J. (1942). X-ray Crystallography, p. 360.\n                       New York: John Wiley & Sons Inc.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_angle_alpha' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '180.0', '180.0', '180.0' ] ],
          [ 'item_', '_item_default.value', '90.0' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_angle_alpha_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_cell_reciprocal_angle_alpha', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_cell.reciprocal_angle_beta',
        [ [ 'item_',
            '_item_description.description',
            '              The angle (recip-beta) defining the reciprocal cell in degrees.\n               (recip-alpha), (recip-alpha) and (recip-alpha) related to the\n               angles in the real cell by:\n\n               cos(recip-alpha)\n                   = [cos(beta)*cos(gamma) - cos(alpha)]/[sin(beta)*sin(gamma)]\n\n               cos(recip-beta)\n                   = [cos(gamma)*cos(alpha) - cos(beta)]/[sin(gamma)*sin(alpha)]\n\n               cos(recip-gamma)\n                   = [cos(alpha)*cos(beta) - cos(gamma)]/[sin(alpha)*sin(beta)]\n\n               Ref: Buerger, M. J. (1942). X-ray Crystallography, p. 360.\n                       New York: John Wiley & Sons Inc.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_angle_beta' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '180.0', '180.0', '180.0' ] ],
          [ 'item_', '_item_default.value', '90.0' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_angle_beta_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_cell_reciprocal_angle_beta', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_cell.reciprocal_angle_gamma',
        [ [ 'item_',
            '_item_description.description',
            '              The angle (recip-gamma) defining the reciprocal cell in degrees.\n               (recip-alpha), (recip-alpha) and (recip-alpha) related to the\n               angles in the real cell by:\n\n               cos(recip-alpha)\n                   = [cos(beta)*cos(gamma) - cos(alpha)]/[sin(beta)*sin(gamma)]\n\n               cos(recip-beta)\n                   = [cos(gamma)*cos(alpha) - cos(beta)]/[sin(gamma)*sin(alpha)]\n\n               cos(recip-gamma)\n                   = [cos(alpha)*cos(beta) - cos(gamma)]/[sin(alpha)*sin(beta)]\n\n               Ref: Buerger, M. J. (1942). X-ray Crystallography, p. 360.\n                       New York: John Wiley & Sons Inc.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_angle_gamma' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '180.0', '180.0', '180.0' ] ],
          [ 'item_', '_item_default.value', '90.0' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_angle_gamma_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_cell_reciprocal_angle_gamma', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_cell.reciprocal_angle_alpha_esd',
        [ [ 'item_',
            '_item_description.description',
            '             The estimated standard deviation of _cell.reciprocal_angle_alpha.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_angle_alpha_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_angle_alpha' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '    \'_cell_reciprocal_angle_alpha_esd\'          cif_core.dic 2.3' ] ] ],
      [ 'save_',
        '_cell.reciprocal_angle_beta_esd',
        [ [ 'item_',
            '_item_description.description',
            '             The estimated standard deviation of _cell.reciprocal_angle_beta.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_angle_beta_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_angle_beta' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '     \'_cell_reciprocal_angle_beta_esd\'    cif_core.dic 2.3' ] ] ],
      [ 'save_',
        '_cell.reciprocal_angle_gamma_esd',
        [ [ 'item_',
            '_item_description.description',
            '             The estimated standard deviation of _cell.reciprocal_angle_gamma.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_angle_gamma_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_angle_gamma' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '     \'_cell_reciprocal_angle_gamma_esd\'         cif_core.dic 2.3' ] ] ],
      [ 'save_',
        '_cell.reciprocal_length_a',
        [ [ 'item_',
            '_item_description.description',
            '              The reciprocal cell length (recip-a) in inverse Angstroms.\n              (recip-a), (recip-b) and (recip-c) are related to the real cell\n              by the following equation:\n\n               recip-a = b*c*sin(alpha)/V\n\n               recip-b = c*a*sin(beta)/V\n\n               recip-c = a*b*sin(gamma)/V\n\n               where V is the cell volume.\n\n               Ref: Buerger, M. J. (1942). X-ray Crystallography, p. 360.\n                       New York: John Wiley & Sons Inc.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_length_a' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'reciprocal_angstroms' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_length_a_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_cell_reciprocal_length_a', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_cell.reciprocal_length_b',
        [ [ 'item_',
            '_item_description.description',
            '              The reciprocal cell length (recip-b) in inverse Angstroms.\n              (recip-a), (recip-b) and (recip-c) are related to the real cell\n              by the following equation:\n\n               recip-a = b*c*sin(alpha)/V\n\n               recip-b = c*a*sin(beta)/V\n\n               recip-c = a*b*sin(gamma)/V\n\n               where V is the cell volume.\n\n               Ref: Buerger, M. J. (1942). X-ray Crystallography, p. 360.\n                       New York: John Wiley & Sons Inc.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_length_b' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'reciprocal_angstroms' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_length_b_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_cell_reciprocal_length_b', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_cell.reciprocal_length_c',
        [ [ 'item_',
            '_item_description.description',
            '              The reciprocal cell length (recip-c) in inverse Angstroms.\n              (recip-a), (recip-b) and (recip-c) are related to the real cell\n              by the following equation:\n\n               recip-a = b*c*sin(alpha)/V\n\n               recip-b = c*a*sin(beta)/V\n\n               recip-c = a*b*sin(gamma)/V\n\n               where V is the cell volume.\n\n               Ref: Buerger, M. J. (1942). X-ray Crystallography, p. 360.\n                       New York: John Wiley & Sons Inc.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_length_c' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'reciprocal_angstroms' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_length_c_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_cell_reciprocal_length_c', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_cell.reciprocal_length_a_esd',
        [ [ 'item_',
            '_item_description.description',
            '             The estimated standard deviation of _cell.reciprocal_length_a.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_length_a_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'reciprocal_angstroms' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_length_a' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '     \'_cell_reciprocal_length_a_esd\'         cif_core.dic 2.3' ] ] ],
      [ 'save_',
        '_cell.reciprocal_length_b_esd',
        [ [ 'item_',
            '_item_description.description',
            '             The estimated standard deviation of _cell.reciprocal_length_b.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_length_b_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'reciprocal_angstroms' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_length_b' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '    \'_cell_reciprocal_length_b_esd\'          cif_core.dic 2.3' ] ] ],
      [ 'save_',
        '_cell.reciprocal_length_c_esd',
        [ [ 'item_',
            '_item_description.description',
            '             The estimated standard deviation of _cell.reciprocal_length_c.' ],
          [ 'item_', '_item.name', '_cell.reciprocal_length_c_esd' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'reciprocal_angstroms' ],
          [ 'item_',
            '_item_related.related_name',
            '_cell.reciprocal_length_c' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '     \'_cell_reciprocal_length_c_esd\'         cif_core.dic 2.3' ] ] ],
      [ '@comment', ' save__cell.special_details' ],
      [ '@comment', '      _item_description.description' ],
      [ '@comment',
        ' ;              A description of special aspects of the cell choice, noting' ],
      [ '@comment', '                possible alternative settings.' ],
      [ '@comment', ' ;' ],
      [ '@comment',
        '      _item.name                \'_cell.special_details\'' ],
      [ '@comment', '      _item.category_id          cell' ],
      [ '@comment', '      _item.mandatory_code       no' ],
      [ '@comment', '      _item_type.code            text' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_examples.case' ],
      [ '@comment',
        '                              \'pseudo-orthorhombic\'' ],
      [ '@comment',
        '                              \'standard setting from 45 deg rotation around c\'' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_aliases.alias_name' ],
      [ '@comment', '      _item_aliases.dictionary' ],
      [ '@comment', '      _item_aliases.version' ],
      [ '@comment',
        '      \'_cell_special_details\'          cif_core.dic 2.3' ],
      [ '@comment', '       save_' ],
      [ '@comment', ' #' ],
      [ 'save_',
        '_chemical.absolute_configuration',
        [ [ 'item_',
            '_item_description.description',
            '              Necessary conditions for the assignment of\n               _chemical.absolute_configuration are given by H. D. Flack and\n               G. Bernardinelli (1999, 2000).\n\n               Ref: Flack, H. D. & Bernardinelli, G. (1999). Acta Cryst. A55,\n                       908-915. (http://www.iucr.org/paper?sh0129)\n                    Flack, H. D. & Bernardinelli, G. (2000). J. Appl. Cryst.\n                       33, 1143-1148. (http://www.iucr.org/paper?ks0021)' ],
          [ 'item_', '_item.name', '_chemical.absolute_configuration' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'rm',
              '                                        absolute configuration established by\n                                         the structure determination of a\n                                         compound containing a chiral reference\n                                         molecule of known absolute\n                                         configuration',
              'ad',
              '                                        absolute configuration established by\n                                         anomalous-dispersion effects in\n                                         diffraction measurements on the\n                                         crystal',
              'rmad',
              '                                        absolute configuration established by\n                                         the structure determination of a\n                                         compound containing a chiral reference\n                                         molecule of known absolute\n                                         configuration and confirmed by\n                                         anomalous-dispersion effects in\n                                         diffraction measurements on the\n                                         crystal',
              'syn',
              '                                        absolute configuration has not been\n                                         established by anomalous-dispersion\n                                         effects in diffraction measurements on\n                                         the crystal. The enantiomer has been\n                                         assigned by reference to an unchanging\n                                         chiral centre in the synthetic\n                                         procedure',
              'unk',
              '                                        absolute configuration is unknown,\n                                         there being no firm chemical evidence\n                                         for its assignment to hand and it\n                                         having not been established by\n                                         anomalous-dispersion effects in\n                                         diffraction measurements on the\n                                         crystal. An arbitrary choice of\n                                         enantiomer has been made',
              '.',
              'inapplicable' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_absolute_configuration', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.melting_point_gt',
        [ [ 'item_',
            '_item_description.description',
            '              A temperature in kelvins above\n               which the melting point (the temperature at which the\n               crystalline solid changes to a liquid) lies.\n               _chemical.melting_point_gt and _chemical.melting_point_lt\n               allow a range of temperatures to be given.\n\n               _chemical.melting_point should always be used in preference\n               to these two items whenever possible.' ],
          [ 'item_', '_item.name', '_chemical.melting_point_gt' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_chemical.melting_point' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_melting_point_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.melting_point_lt',
        [ [ 'item_',
            '_item_description.description',
            '              A temperature in kelvins below which the melting point (the\n               temperature at which the crystalline solid changes to a liquid)\n               lies. _chemical.melting_point_gt and _chemical.melting_point_lt\n               allow a range of temperatures to be given.\n\n               _chemical.melting_point should always be used in preference\n               to these two items whenever possible.' ],
          [ 'item_', '_item.name', '_chemical.melting_point_lt' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_chemical.melting_point' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_melting_point_lt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.optical_rotation',
        [ [ 'item_',
            '_item_description.description',
            '              The optical rotation in solution of the compound is\n               specified in the following format:\n\n                    \'[\\a]^TEMP^~WAVE~ = SORT (c = CONC, SOLV)\'\n\n               where:\n\n                 TEMP is the temperature of the measurement in degrees\n                      Celsius,\n                 WAVE is an indication of the wavelength of the light\n                      used for the measurement,\n                 CONC is the concentration of the solution given as the\n                      mass of the substance in g in 100 ml of solution,\n                 SORT is the signed value (preceded by a + or a - sign)\n                      of 100.\\a/(l.c), where \\a is the signed optical\n                      rotation in degrees measured in a cell of length l in\n                      dm and c is the value of CONC as defined above, and\n                 SOLV is the chemical formula of the solvent.' ],
          [ 'item_', '_item.name', '_chemical.optical_rotation' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_examples.case',
            '[\\a]^25^~D~ = +108 (c = 3.42, CHCl~3~)' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_optical_rotation', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.properties_biological',
        [ [ 'item_',
            '_item_description.description',
            '              A free-text description of the biological properties of the\n               material.' ],
          [ 'item_', '_item.name', '_chemical.properties_biological' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '              diverse biological activities including use as a\n               laxative and strong antibacterial activity against\n               S. aureus and weak activity against\n               cyclooxygenase-1 (COX-1)',
              '              antibiotic activity against Bacillus subtilis\n               (ATCC 6051) but no significant activity against\n               Candida albicans (ATCC 14053), Aspergillus flavus\n               (NRRL 6541) and Fusarium verticillioides (NRRL\n               25457)',
              '              weakly potent lipoxygenase nonredox inhibitor',
              '              no influenza A virus sialidase inhibitory and\n               plaque reduction activities',
              '              low toxicity against Drosophila melanogaster' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_properties_biological', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.properties_physical',
        [ [ 'item_',
            '_item_description.description',
            '    A free-text description of the physical properties of the material.' ],
          [ 'item_', '_item.name', '_chemical.properties_physical' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'air-sensitive',
              'moisture-sensitive',
              'hygroscopic',
              'deliquescent',
              'oxygen-sensitive',
              'photo-sensitive',
              'pyrophoric',
              'semiconductor',
              'ferromagnetic at low temperature',
              'paramagnetic and thermochromic' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_properties_physical', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.temperature_decomposition',
        [ [ 'item_',
            '_item_description.description',
            '              The temperature in kelvins at which the solid decomposes.' ],
          [ 'item_', '_item.name', '_chemical.temperature_decomposition' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_', '_item_examples.case', '350' ],
          [ 'item_',
            '_item_related.related_name',
            '_chemical.temperature_decomposition_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_temperature_decomposition', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.temperature_decomposition_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of\n               _chemical.temperature_decomposition.' ],
          [ 'item_',
            '_item.name',
            '_chemical.temperature_decomposition_esd' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_',
            '_item_related.related_name',
            '_chemical.temperature_decomposition' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '     \'_chemical_temperature_decomposition_esd\'         cif_core.dic 2.3' ] ] ],
      [ 'save_',
        '_chemical.temperature_decomposition_gt',
        [ [ 'item_',
            '_item_description.description',
            '              A temperature in kelvins above which the solid is known to\n               decompose. _chemical.temperature_decomposition_gt and\n               _chemical.temperature_decomposition_lt allow\n               a range of temperatures to be given.\n\n               _chemical.temperature_decomposition should always be used in\n               preference to these two items whenever possible.' ],
          [ 'item_',
            '_item.name',
            '_chemical.temperature_decomposition_gt' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_',
            '_item_related.related_name',
            '_chemical.temperature_decomposition' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_examples.case', '350' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_temperature_decomposition_gt',
              'cif_core.dic',
              '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.temperature_decomposition_lt',
        [ [ 'item_',
            '_item_description.description',
            '              A temperature in kelvins below which the solid is known to\n              decompose. _chemical.temperature_decomposition_gt and\n               _chemical.temperature_decomposition_lt allow\n               a range of temperatures to be given.\n\n               _chemical.temperature_decomposition should always be used in\n               preference to these two items whenever possible.' ],
          [ 'item_',
            '_item.name',
            '_chemical.temperature_decomposition_lt' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_',
            '_item_related.related_name',
            '_chemical.temperature_decomposition' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_examples.case', '350' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_temperature_decomposition_lt',
              'cif_core.dic',
              '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.temperature_sublimation',
        [ [ 'item_',
            '_item_description.description',
            '              The temperature in kelvins at which the solid sublimes.' ],
          [ 'item_', '_item.name', '_chemical.temperature_sublimation' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_', '_item_examples.case', '350' ],
          [ 'item_',
            '_item_related.related_name',
            '_chemical.temperature_sublimation_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_temperature_sublimation', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.temperature_sublimation_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of\n               _chemical.temperature_sublimation.' ],
          [ 'item_',
            '_item.name',
            '_chemical.temperature_sublimation_esd' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_',
            '_item_related.related_name',
            '_chemical.temperature_sublimation' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '     \'_chemical_temperature_sublimation_esd\'         cif_core.dic 2.3' ] ] ],
      [ 'save_',
        '_chemical.temperature_sublimation_gt',
        [ [ 'item_',
            '_item_description.description',
            '             A temperature in kelvins above which the solid is known to\n              sublime. _chemical.temperature_sublimation_gt and\n              _chemical.temperature_sublimation_lt allow a\n              range of temperatures to be given.\n\n              _chemical.temperature_sublimation should always be used in\n              preference to these two items whenever possible.' ],
          [ 'item_', '_item.name', '_chemical.temperature_sublimation_gt' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_',
            '_item_related.related_name',
            '_chemical.temperature_sublimation' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_examples.case', '350' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_temperature_sublimation_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_chemical.temperature_sublimation_lt',
        [ [ 'item_',
            '_item_description.description',
            '             A temperature in kelvins below which the solid is known to\n              sublime. _chemical.temperature_sublimation_gt and\n              _chemical.temperature_sublimation_lt allow a\n              range of temperatures to be given.\n\n              _chemical.temperature_sublimation should always be used in\n              preference to these two items whenever possible.' ],
          [ 'item_', '_item.name', '_chemical.temperature_sublimation_lt' ],
          [ 'item_', '_item.category_id', 'chemical' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_',
            '_item_related.related_name',
            '_chemical.temperature_sublimation' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'item_', '_item_examples.case', '350' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_chemical_temperature_sublimation_lt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_citation.database_id_CSD',
        [ [ 'item_',
            '_item_description.description',
            '           Identifier (\'refcode\') of the database record in the Cambridge\n            Structural Database that contains details of the cited structure.' ],
          [ 'item_', '_item.name', '_citation.database_id_CSD' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', 'LEKKUH' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_citation_database_id_CSD', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.CSD_history',
        [ [ 'item_',
            '_item_description.description',
            '              A history of changes made by the Cambridge Crystallographic Data\n               Centre and incorporated into the Cambridge Structural Database\n               (CSD).' ],
          [ 'item_', '_item.name', '_database.CSD_history' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_CSD_history', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.code_CAS',
        [ [ 'item_',
            '_item_description.description',
            '              The code assigned by Chemical Abstracts.' ],
          [ 'item_', '_item.name', '_database.code_CAS' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_code_CAS', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.code_CSD',
        [ [ 'item_',
            '_item_description.description',
            '              The code assigned by the Cambridge Structural Database.' ],
          [ 'item_', '_item.name', '_database.code_CSD' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_code_CSD', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.code_ICSD',
        [ [ 'item_',
            '_item_description.description',
            '              The code assigned by the Inorganic Crystal Structure\n               Database.' ],
          [ 'item_', '_item.name', '_database.code_ICSD' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_code_ICSD', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.code_MDF',
        [ [ 'item_',
            '_item_description.description',
            '              The code assigned by the Metals Data File.' ],
          [ 'item_', '_item.name', '_database.code_MDF' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_code_MDF', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.code_NBS',
        [ [ 'item_',
            '_item_description.description',
            '              The code assigned by the NBS (NIST) Crystal Data Database.' ],
          [ 'item_', '_item.name', '_database.code_NBS' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_code_NBS', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.code_PDB',
        [ [ 'item_',
            '_item_description.description',
            '              The code assigned by the Protein Data Bank.' ],
          [ 'item_', '_item.name', '_database.code_PDB' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_code_PDB', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.code_PDF',
        [ [ 'item_',
            '_item_description.description',
            '              The code assigned by the Powder Diffraction File (JCPDS/ICDD).' ],
          [ 'item_', '_item.name', '_database.code_PDF' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_code_PDF', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.code_depnum_ccdc_fiz',
        [ [ 'item_',
            '_item_description.description',
            '              Deposition numbers assigned by the Fachinformationszentrum\n               Karlsruhe (FIZ) to files containing structural information\n               archived by the Cambridge Crystallographic Data Centre (CCDC).' ],
          [ 'item_', '_item.name', '_database.code_depnum_ccdc_fiz' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_code_depnum_ccdc_fiz', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.code_depnum_ccdc_journal',
        [ [ 'item_',
            '_item_description.description',
            '              Deposition numbers assigned by various journals to files\n               containing structural information archived by the Cambridge\n               Crystallographic Data Centre (CCDC).' ],
          [ 'item_', '_item.name', '_database.code_depnum_ccdc_journal' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_code_depnum_ccdc_journal', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_database.code_depnum_ccdc_archive',
        [ [ 'item_',
            '_item_description.description',
            '              Deposition numbers assigned by the Cambridge Crystallographic\n               Data Centre (CCDC) to files containing structural information\n               archived by the CCDC.' ],
          [ 'item_', '_item.name', '_database.code_depnum_ccdc_archive' ],
          [ 'item_', '_item.category_id', 'database' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_database_code_depnum_ccdc_archive', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn.ambient_pressure',
        [ [ 'item_',
            '_item_description.description',
            '              The mean hydrostatic pressure in kilopascals at which the\n               intensities were measured.' ],
          [ 'item_', '_item.name', '_diffrn.ambient_pressure' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kilopascals' ],
          [ 'item_',
            '_item_related.related_name',
            '_diffrn.ambient_pressure_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_ambient_pressure', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn.ambient_pressure_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _diffrn.ambient_pressure.' ],
          [ 'item_', '_item.name', '_diffrn.ambient_pressure_esd' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kilopascals' ],
          [ 'item_',
            '_item_related.related_name',
            '_diffrn.ambient_pressure' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '     \'_diffrn_ambient_pressure\'         cif_core.dic 2.3' ] ] ],
      [ 'save_',
        '_diffrn.ambient_pressure_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The mean hydrostatic pressure in kilopascals above which\n               the intensities were measured. _diffrn.ambient_pressure_gt and\n               _diffrn.ambient_pressure_lt allow a pressure range to be given.\n\n               _diffrn.ambient_pressure should always be used in\n               preference to these two items whenever possible.' ],
          [ 'item_', '_item.name', '_diffrn.ambient_pressure_gt' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_diffrn.ambient_pressure' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kilopascals' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_ambient_pressure_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn.ambient_pressure_lt',
        [ [ 'item_',
            '_item_description.description',
            '              The mean hydrostatic pressure in kilopascals below which\n               the intensities were measured. _diffrn.ambient_pressure_gt and\n               _diffrn.ambient_pressure_lt allow a pressure range to be given.\n\n               _diffrn.ambient_pressure should always be used in\n               preference to these two items whenever possible.' ],
          [ 'item_', '_item.name', '_diffrn.ambient_pressure_lt' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_diffrn.ambient_pressure' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kilopascals' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_ambient_pressure_lt', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', ' save__diffrn.ambient_temperature' ],
      [ '@comment', '      _item_description.description' ],
      [ '@comment',
        ' ;             The mean temperature in kelvins at which the intensities' ],
      [ '@comment', '               were measured.' ],
      [ '@comment', ' ;' ],
      [ '@comment',
        '      _item.name                 \'_diffrn.ambient_temperature\'' ],
      [ '@comment', '      _item.category_id           diffrn' ],
      [ '@comment', '      _item.mandatory_code        no' ],
      [ '@comment', '      _item_type.code             float' ],
      [ '@comment', '      _item_type_conditions.code  esd' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_range.minimum' ],
      [ '@comment', '      _item_range.maximum' ],
      [ '@comment', '                            0.0   0.0' ],
      [ '@comment', '                            0.0    .' ],
      [ '@comment', '      _item_units.code            kelvins' ],
      [ '@comment',
        '      _item_related.related_name  \'_diffrn.ambient_temperature_esd\'' ],
      [ '@comment',
        '      _item_related.function_code  associated_esd' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_aliases.alias_name' ],
      [ '@comment', '      _item_aliases.dictionary' ],
      [ '@comment', '      _item_aliases.version' ],
      [ '@comment',
        '       \'_diffrn_ambient_temperature\'         cif_core.dic 2.3' ],
      [ '@comment', '       save_' ],
      [ '@comment', 'save__diffrn.ambient_temperature_esd' ],
      [ '@comment', '     _item_description.description' ],
      [ '@comment',
        ';             The estimated standard deviation of _diffrn.ambient_temp.' ],
      [ '@comment', ';' ],
      [ '@comment',
        '     _item.name                 \'_diffrn.ambient_temperature_esd\'' ],
      [ '@comment', '     _item.category_id           diffrn' ],
      [ '@comment', '     _item.mandatory_code        no' ],
      [ '@comment', '     _item_type.code             float' ],
      [ '@comment', '     _item_units.code            kelvins' ],
      [ '@comment',
        '     _item_related.related_name  \'_diffrn.ambient_temp\'' ],
      [ '@comment',
        '     _item_related.function_code  associated_value' ],
      [ '@comment', '    loop_' ],
      [ '@comment', '    _item_aliases.alias_name' ],
      [ '@comment', '    _item_aliases.dictionary' ],
      [ '@comment', '    _item_aliases.version' ],
      [ '@comment',
        '    \'_diffrn_ambient_temperature_esd\'          cif_core.dic 2.3' ],
      [ '@comment', '      save_' ],
      [ 'save_',
        '_diffrn.ambient_temp_gt',
        [ [ 'item_',
            '_item_description.description',
            '             The mean temperature in kelvins above which the intensities were\n              measured.  _diffrn.ambient_temp_gt and _diffrn.ambient_temp_lt\n              allow a range of temperatures to be given.\n\n              _diffrn.ambient_temp should always be used in preference\n              to these two items whenever possible.' ],
          [ 'item_', '_item.name', '_diffrn.ambient_temp_gt' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_related.related_name', '_diffrn.ambient_temp' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_ambient_temperature_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn.ambient_temp_lt',
        [ [ 'item_',
            '_item_description.description',
            '             The mean temperature in kelvins below which the intensities were\n              measured. _diffrn.ambient_temp_gt and _diffrn.ambient_temp_lt\n              allow a range of temperatures to be given.\n\n              _diffrn.ambient_temp should always be used in preference\n              to these two items whenever possible.' ],
          [ 'item_', '_item.name', '_diffrn.ambient_temp_lt' ],
          [ 'item_', '_item.category_id', 'diffrn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_related.related_name', '_diffrn.ambient_temp' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_ambient_temperature_lt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_attenuator.material',
        [ [ 'item_',
            '_item_description.description',
            '              Material from which the attenuator is made.' ],
          [ 'item_', '_item.name', '_diffrn_attenuator.material' ],
          [ 'item_', '_item.category_id', 'diffrn_attenuator' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_attenuator_material', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_detector.area_resol_mean',
        [ [ 'item_',
            '_item_description.description',
            '              The resolution of an area detector, in pixels/mm.' ],
          [ 'item_', '_item.name', '_diffrn_detector.area_resol_mean' ],
          [ 'item_', '_item.category_id', 'diffrn_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'pixels_per_millimetre' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_detector_area_resol_mean', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_detector.dtime',
        [ [ 'item_',
            '_item_description.description',
            '              The deadtime in microseconds of the detector used to measure\n               the diffraction intensities.' ],
          [ 'item_', '_item.name', '_diffrn_detector.dtime' ],
          [ 'item_', '_item.category_id', 'diffrn_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0',
              '0.0',
              '0.0',
              '.',
              [ '@comment',
                '     _item_related.related_name \'_diffrn_radiation_detector.dtime\'' ],
              [ '@comment', '     _item_related.function_code alternate' ] ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_detector_dtime', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_refln.class_code',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying the class to which this reflection has\n               been assigned. This code must match a value of\n               _diffrn_reflns_class.code. Reflections may be grouped into\n               classes for a variety of purposes. For example, for modulated\n               structures each reflection class may be defined by the\n               number m=sum|m~i~|, where the m~i~ are the integer coefficients\n               that, in addition to h,k,l, index the corresponding diffraction\n               vector in the basis defined for the reciprocal lattice.' ],
          [ 'item_', '_item.name', '_diffrn_refln.class_code' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_refln_class_code', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_refln.intensity_u',
        [ [ 'item_',
            '_item_description.description',
            '              Standard uncertainty of the net intensity calculated from\n               the diffraction counts after the attenuator and standard\n               scales have been applied.' ],
          [ 'item_', '_item.name', '_diffrn_refln.intensity_u' ],
          [ 'item_', '_item.category_id', 'diffrn_refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_diffrn_refln.intensity_sigma' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_refln_intensity_u', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_reflns.av_unetI/netI',
        [ [ 'item_',
            '_item_description.description',
            '              Measure [sum u(net I)|/sum|net I|] for all measured reflections.' ],
          [ 'item_', '_item.name', '_diffrn_reflns.av_unetI/netI' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ '@comment',
            '     _item_related.related_name  \'_diffrn_reflns.av_sigmaI/netI\'' ],
          [ '@comment', '     _item_related.function_code  alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_reflns_av_unetI/netI', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# DIFFRN_REFLNS_CLASS ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'diffrn_reflns_class',
        [ [ 'item_',
            '_category.description',
            '              Data items in the DIFFRN_REFLNS_CLASS category record details\n               about the classes of reflections measured in the diffraction\n               experiment.' ],
          [ 'item_', '_category.id', 'diffrn_reflns_class' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_diffrn_reflns_class.code' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'diffrn_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              '_category_examples.detail',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    loop_\n        _diffrn_reflns_class.number\n        _diffrn_reflns_class.d_res_high\n        _diffrn_reflns_class.d_res_low\n        _diffrn_reflns_class.av_R_eq\n        _diffrn_reflns_class.code\n        _diffrn_reflns_class.description\n               1580 0.551 6.136 0.015 \'Main\' \'m=0; main reflections\'\n               1045 0.551 6.136 0.010 \'Sat1\' \'m=1; first-order satellites\'',
              '\n    Example 1 - example corresponding to the one-dimensional incommensurately\n                modulated structure of K~2~SeO~4~. Each reflection class is\n                defined by the number m=sum|m~i~|, where the m~i~ are the\n                integer coefficients that, in addition to h,k,l, index the\n                corresponding diffraction vector in the basis defined for\n                the reciprocal lattice.',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_diffrn_reflns_class.av_R_eq',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the residual\n               [sum av|del(I)|/sum|av(I)|] for symmetry-equivalent reflections\n               used to calculate the average intensity av(I). The av|del(I)|\n               term is the average absolute difference between av(I) and the\n               individual intensities.' ],
          [ 'item_', '_item.name', '_diffrn_reflns_class.av_R_eq' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_reflns_class_av_R_eq', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_reflns_class.av_sgI/I',
        [ [ 'item_',
            '_item_description.description',
            '         Measure [sum|sigma(net I)|/sum|net I|] for all measured intensities\n          in a reflection class.' ],
          [ 'item_', '_item.name', '_diffrn_reflns_class.av_sgI/I' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_diffrn_reflns_class.av_uI/I' ],
          [ 'item_', '_item_related.function_code', 'replaces' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_reflns_class_av_sgI/I', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_reflns_class.av_uI/I',
        [ [ 'item_',
            '_item_description.description',
            '              Measure [sum|u(net I)|/sum|net I|] for all measured intensities\n               in a reflection class.' ],
          [ 'item_', '_item.name', '_diffrn_reflns_class.av_uI/I' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_diffrn_reflns_class.av_sgI/I' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_reflns_class_av_uI/I', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_reflns_class.code',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying a certain reflection class.' ],
          [ 'item_', '_item.name', '_diffrn_reflns_class.code' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', 'm1', 's2' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_reflns_class_code', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_reflns_class.description',
        [ [ 'item_',
            '_item_description.description',
            '              Description of each reflection class.' ],
          [ 'item_', '_item.name', '_diffrn_reflns_class.description' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'm=1 first order satellites',
              'H0L0 common projection reflections' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_reflns_class_description', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_reflns_class.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The smallest value in angstroms for the interplanar\n               spacings for the reflections in each measured reflection class.\n               This is called the highest resolution for this reflection class.' ],
          [ 'item_', '_item.name', '_diffrn_reflns_class.d_res_high' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_reflns_class_d_res_high', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_reflns_class.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The largest value in angstroms of the interplanar\n               spacings for the reflections for each measured reflection class.\n               This is called the lowest resolution for this reflection class.' ],
          [ 'item_', '_item.name', '_diffrn_reflns_class.d_res_low' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_reflns_class_d_res_low', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_reflns_class.number',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of measured intensities for each reflection\n               class, excluding the systematic absences arising from\n               centring translations.' ],
          [ 'item_', '_item.name', '_diffrn_reflns_class.number' ],
          [ 'item_', '_item.category_id', 'diffrn_reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0', '0', '0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_reflns_class_number', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_source.take-off_angle',
        [ [ 'item_',
            '_item_description.description',
            '              The complement of the angle in degrees between the normal\n               to the surface of the X-ray tube target and the primary\n               X-ray beam for beams generated by traditional X-ray tubes.' ],
          [ 'item_', '_item.name', '_diffrn_source.take-off_angle' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.00', '0.00', '0.00', '90.0', '90.0', '90.0' ] ],
          [ 'item_', '_item_examples.case', '1.5' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_source_take-off_angle', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_diffrn_standards.scale_u',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty of the individual mean\n               standard scales applied to the intensity data.' ],
          [ 'item_', '_item.name', '_diffrn_standards.scale_u' ],
          [ 'item_', '_item.category_id', 'diffrn_standards' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_diffrn_standards.scale_sigma' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_diffrn_standards_scale_u', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_exptl_crystal.colour_lustre',
        [ [ 'item_',
            '_item_description.description',
            '             The enumeration list of standardized names developed for the\n              International Centre for Diffraction Data.\n              The colour of a crystal is given by the combination of\n              _exptl_crystal.colour_modifier with\n              _exptl_crystal.colour_primary, as in \'dark-green\' or\n              \'bluish-violet\', if necessary combined with\n              _exptl_crystal.colour_lustre, as in \'metallic-green\'.' ],
          [ 'item_', '_item.name', '_exptl_crystal.colour_lustre' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'metallic', 'dull', 'clear' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal.colour' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_exptl_crystal_colour_lustre', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_exptl_crystal.colour_modifier',
        [ [ 'item_',
            '_item_description.description',
            '             The enumeration list of standardized names developed for the\n              International Centre for Diffraction Data.\n              The colour of a crystal is given by the combination of\n              _exptl_crystal.colour_modifier with\n              _exptl_crystal.colour_primary, as in \'dark-green\' or\n              \'bluish-violet\', if necessary combined with\n              _exptl_crystal.colour_lustre, as in \'metallic-green\'.' ],
          [ 'item_', '_item.name', '_exptl_crystal.colour_modifier' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'light',
              'dark',
              'whitish',
              'blackish',
              'grayish',
              'brownish',
              'reddish',
              'pinkish',
              'orangish',
              'yellowish',
              'greenish',
              'bluish' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal.colour' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_exptl_crystal_colour_modifier', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_exptl_crystal.colour_primary',
        [ [ 'item_',
            '_item_description.description',
            '             The enumeration list of standardized names developed for the\n              International Centre for Diffraction Data.\n              The colour of a crystal is given by the combination of\n              _exptl_crystal.colour_modifier with\n              _exptl_crystal.colour_primary, as in \'dark-green\' or\n              \'bluish-violet\', if necessary combined with\n              _exptl_crystal.colour_lustre, as in \'metallic-green\'.' ],
          [ 'item_', '_item.name', '_exptl_crystal.colour_primary' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'colourless',
              'white',
              'black',
              'gray',
              'brown',
              'red',
              'pink',
              'orange',
              'yellow',
              'green',
              'blue',
              'violet' ] ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal.colour' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_exptl_crystal_colour_primary', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_exptl_crystal.density_meas',
        [ [ 'item_',
            '_item_description.description',
            '              Density values measured using standard chemical and physical\n               methods. The units are megagrams per cubic metre (grams per\n               cubic centimetre).' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_meas' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'megagrams_per_cubic_metre' ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal.density_meas_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_exptl_crystal_density_meas', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_exptl_crystal.density_meas_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _exptl_crystal.density_meas.' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_meas_esd' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'megagrams_per_cubic_metre' ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal.density_meas' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '    \'_exptl_crystal_density_meas_esd\'          cif_core.dic 2.3' ] ] ],
      [ 'save_',
        '_exptl_crystal.density_meas_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The value above which the density measured using standard\n               chemical and physical methods lies. The units are megagrams\n               per cubic metre (grams per cubic centimetre).\n               _exptl_crystal.density_meas_gt and _exptl_crystal.density_meas_lt\n               should not be used to report new experimental work, for which\n               _exptl_crystal.density_meas should be used.  These items are\n               intended for use in reporting information in existing databases\n               and archives which would be misleading if reported under\n               _exptl_crystal.density_meas.' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_meas_gt' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'megagrams_per_cubic_metre' ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal.density_meas' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '2.5',
              '                              lower limit for the density (only the range\n                               within which the density lies was given in the\n                               original paper)' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_exptl_crystal_density_meas_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_exptl_crystal.density_meas_lt',
        [ [ 'item_',
            '_item_description.description',
            '              The value below which the density measured using standard\n               chemical and physical methods lies. The units are megagrams\n               per cubic metre (grams per cubic centimetre).\n               _exptl_crystal.density_meas_gt and _exptl_crystal.density_meas_lt\n               should not be used to report new experimental work, for which\n               _exptl_crystal.density_meas should be used.  These items are\n               intended for use in reporting information in existing databases\n               and archives which would be misleading if reported under\n               _exptl_crystal.density_meas.' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_meas_lt' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'megagrams_per_cubic_metre' ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal.density_meas' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '1.0',
              'specimen floats in water',
              '5.0',
              '                              upper limit for the density (only the range\n                               within which the density lies was given in the\n                               original paper)' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_exptl_crystal_density_meas_lt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_exptl_crystal.density_meas_temp',
        [ [ 'item_',
            '_item_description.description',
            '              Temperature in kelvins at which _exptl_crystal.density_meas\n               was determined.' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_meas_temp' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_exptl_crystal_density_meas_temp', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_exptl_crystal.density_meas_temp_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of\n               _exptl_crystal.density_meas_temp.' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_meas_temp_esd' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ '@comment', '    loop_' ],
          [ '@comment', '    _item_aliases.alias_name' ],
          [ '@comment', '    _item_aliases.dictionary' ],
          [ '@comment', '    _item_aliases.version' ],
          [ '@comment',
            '     \'_exptl_crystal_density_meas_temp_esd\'         cif_core.dic 2.3' ] ] ],
      [ 'save_',
        '_exptl_crystal.density_meas_temp_gt',
        [ [ 'item_',
            '_item_description.description',
            '              Temperature in kelvins above which _exptl_crystal.density_meas\n               was determined. _exptl_crystal.density_meas_temp_gt and\n               _exptl_crystal.density_meas_temp_lt should not be used for\n               reporting new work,  for which the correct temperature of\n               measurement should be given. These items are intended for\n               use in reporting information stored in databases or archives\n               which would be misleading if reported under\n               _exptl_crystal.density_meas_temp.' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_meas_temp_gt' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal.density_meas_temp' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_exptl_crystal_density_meas_temp_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_exptl_crystal.density_meas_temp_lt',
        [ [ 'item_',
            '_item_description.description',
            '              Temperature in kelvins below which _exptl_crystal.density_meas\n               was determined. _exptl_crystal.density_meas_temp_gt and\n               _exptl_crystal.density_meas_temp_lt should not be used for\n               reporting new work,  for which the correct temperature of\n               measurement should be given. These items are intended for\n               use in reporting information stored in databases or archives\n               which would be misleading if reported under\n               _exptl_crystal.density_meas_temp.' ],
          [ 'item_', '_item.name', '_exptl_crystal.density_meas_temp_lt' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_',
            '_item_related.related_name',
            '_exptl_crystal.density_meas_temp' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '300',
              '                               The density was measured at some unspecified\n                                temperature below room temperature.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_exptl_crystal_density_meas_temp_lt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_geom_bond.valence',
        [ [ 'item_',
            '_item_description.description',
            '              The bond valence calculated from _geom_bond.dist.' ],
          [ 'item_', '_item.name', '_geom_bond.valence' ],
          [ 'item_', '_item.category_id', 'geom_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_geom_bond_valence', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_publ_author.id_iucr',
        [ [ 'item_',
            '_item_description.description',
            '              Identifier in the IUCr contact database of a publication\n               author.  This identifier may be available from the World\n               Directory of Crystallographers (http://wdc.iucr.org).' ],
          [ 'item_', '_item.name', '_publ_author.id_iucr' ],
          [ 'item_', '_item.category_id', 'publ_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '2985' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_publ_author_id_iucr', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine.ls_R_factor_gt',
        [ [ 'item_',
            '_item_description.description',
            '              Residual factor for the reflections (with number given by\n               _reflns.number_gt) judged significantly intense (i.e. satisfying\n               the threshold specified by _reflns.threshold_expression)\n               and included in the refinement. The reflections also satisfy\n               the resolution limits established by _refine.ls_d_res_high and\n               _refine.ls_d_res_low. This is the conventional R\n               factor. See also _refine.ls_wR_factor_ definitions.\n\n                   sum | F(obs) - F(calc) |\n               R = ------------------------\n                         sum | F(obs) |\n\n               F(obs)  = the observed structure-factor amplitudes\n               F(calc) = the calculated structure-factor amplitudes\n\n               and the sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_R_factor_gt' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_R_factor_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_R_factor_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine.ls_goodness_of_fit_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The least-squares goodness-of-fit parameter S for\n               significantly intense reflections (see\n               _reflns.threshold_expression) after the final cycle of\n               refinement. Ideally, account should be taken of parameters\n               restrained in the least-squares refinement. See also\n               _refine.ls_restrained_S_ definitions.\n\n                   {  sum { w [ Y(obs) - Y(calc) ]^2^ }  }^1/2^\n               S = { ----------------------------------- }\n                   {            Nref - Nparam            }\n\n               Y(obs)  = the observed coefficients\n                         (see _refine_ls_structure_factor_coef)\n               Y(calc) = the calculated coefficients\n                         (see _refine_ls_structure_factor_coef)\n               w       = the least-squares reflection weight\n                         [1/(u^2^)]\n               u       = standard uncertainty\n\n               Nref   = the number of reflections used in the refinement\n               Nparam = the number of refined parameters\n\n               and the sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_goodness_of_fit_gt' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_goodness_of_fit_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_goodness_of_fit_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine.ls_goodness_of_fit_ref',
        [ [ 'item_',
            '_item_description.description',
            '              The least-squares goodness-of-fit parameter S for all\n               reflections included in the refinement after the final cycle\n               of refinement. Ideally, account should be taken of parameters\n               restrained in the least-squares refinement. See also\n               _refine_ls_restrained_S_ definitions.\n\n                   {  sum | w | Y(obs) - Y(calc) |^2^ |  }^1/2^\n               S = { ----------------------------------- }\n                   {            Nref - Nparam            }\n\n               Y(obs)  = the observed coefficients\n                         (see _refine_ls_structure_factor_coef)\n               Y(calc) = the calculated coefficients\n                         (see _refine_ls_structure_factor_coef)\n               w       = the least-squares reflection weight\n                         [1/(u^2^)]\n               u       = standard uncertainty\n\n               Nref   = the number of reflections used in the refinement\n               Nparam = the number of refined parameters\n\n               and the sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.ls_goodness_of_fit_ref' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_goodness_of_fit_ref', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', ' save__refine.ls_shift/esd_max' ],
      [ '@comment', '      _item_description.description' ],
      [ '@comment',
        ' ;              The largest ratio of the final least-squares parameter' ],
      [ '@comment',
        '                shift to the final standard uncertainty (s.u.,' ],
      [ '@comment',
        '                formerly described as estimated standard deviation, e.s.d.).' ],
      [ '@comment', ' ;' ],
      [ '@comment',
        '      _item.name                 \'_refine.ls_shift/esd_max\'' ],
      [ '@comment', '      _item.category_id           refine' ],
      [ '@comment', '      _item.mandatory_code        no' ],
      [ '@comment', '      _item_type.code             float' ],
      [ '@comment',
        '      _item_related.related_name \'_refine.ls_shift/su_max\'' ],
      [ '@comment', '      _item_related.function_code  replaces' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_range.minimum' ],
      [ '@comment', '      _item_range.maximum' ],
      [ '@comment', '                            0.0   0.0' ],
      [ '@comment', '                            0.0    .' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_aliases.alias_name' ],
      [ '@comment', '      _item_aliases.dictionary' ],
      [ '@comment', '      _item_aliases.version' ],
      [ '@comment',
        '       \'_refine_ls_shift/esd_max\'         cif_core.dic 2.3' ],
      [ '@comment', '       save_' ],
      [ '@comment', ' save__refine.ls_shift/esd_mean' ],
      [ '@comment', '      _item_description.description' ],
      [ '@comment',
        ' ;              The average ratio of the final least-squares parameter' ],
      [ '@comment',
        '                shift to the final standard uncertainty (s.u.,' ],
      [ '@comment',
        '                formerly described as estimated standard deviation, e.s.d.).' ],
      [ '@comment', ' ;' ],
      [ '@comment',
        '      _item.name                 \'_refine.ls_shift/esd_mean\'' ],
      [ '@comment', '      _item.category_id           refine' ],
      [ '@comment', '      _item.mandatory_code        no' ],
      [ '@comment', '      _item_type.code             float' ],
      [ '@comment',
        '      _item_related.related_name  \'_refine.ls_shift/su_mean\'' ],
      [ '@comment', '      _item_related.function_code  replaces' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_range.minimum' ],
      [ '@comment', '      _item_range.maximum' ],
      [ '@comment', '                            0.0   0.0' ],
      [ '@comment', '                            0.0    .' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_aliases.alias_name' ],
      [ '@comment', '      _item_aliases.dictionary' ],
      [ '@comment', '      _item_aliases.version' ],
      [ '@comment',
        '        \'_refine_ls_shift/esd_mean\'         cif_core.dic 2.3' ],
      [ '@comment', '       save_' ],
      [ 'save_',
        '_refine.ls_shift_over_su_max',
        [ [ 'item_',
            '_item_description.description',
            '              The largest ratio of the final least-squares parameter\n               shift to the final standard uncertainty.' ],
          [ 'item_', '_item.name', '_refine.ls_shift_over_su_max' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_shift_over_esd_max' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_shift/su_max', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine.ls_shift_over_su_max_lt',
        [ [ 'item_',
            '_item_description.description',
            '              An upper limit for the largest ratio of the final\n               least-squares parameter shift to the final\n               standard uncertainty.  This item is used when the largest\n               value of the shift divided by the final standard uncertainty\n               is too small to measure.' ],
          [ 'item_', '_item.name', '_refine.ls_shift_over_su_max_lt' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_shift_over_su_max' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_shift/su_max_lt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine.ls_shift_over_su_mean',
        [ [ 'item_',
            '_item_description.description',
            '              The average ratio of the final least-squares parameter\n               shift to the final standard uncertainty.' ],
          [ 'item_', '_item.name', '_refine.ls_shift_over_su_mean' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_shift_over_esd_mean' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_shift/su_mean', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine.ls_shift_over_su_mean_lt',
        [ [ 'item_',
            '_item_description.description',
            '              An upper limit for the average ratio of the final\n               least-squares parameter shift to the\n               final standard uncertainty.  This\n               item is used when the average value of the shift divided by\n               the final standard uncertainty is too small to measure.' ],
          [ 'item_', '_item.name', '_refine.ls_shift_over_su_mean_lt' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_refine.ls_shift_over_su_mean' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_shift/su_mean_lt', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# REFINE_LS_CLASS ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'refine_ls_class',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_LS_CLASS category record details\n               about the reflections used for the structure refinement\n               for each reflection class separately.' ],
          [ 'item_', '_category.id', 'refine_ls_class' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_refine_ls_class.code' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              '_category_examples.detail',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    loop_\n        _refine_ls_class.R_factor_gt\n        _refine_ls_class.code\n                0.057    \'Main\'\n                0.074    \'Com\'\n                0.064    \'NbRefls\'\n                0.046    \'LaRefls\'\n                0.112    \'Sat1\'\n                0.177    \'Sat2\'',
              '\n    Example 1 - data for a modulated structure from van Smaalen\n    [J. Phys. Condens. Matter (1991), 3, 1247-1263].',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_refine_ls_class.code',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying a certain reflection class. This code must\n               match a _reflns_class.code.' ],
          [ 'item_', '_item.name', '_refine_ls_class.code' ],
          [ 'item_', '_item.category_id', 'refine_ls_class' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', 'm1', 's2' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_class_code', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine_ls_class.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the lowest value in angstroms\n               for the interplanar spacings for the reflections used in the\n               refinement. This is called the highest resolution.' ],
          [ 'item_', '_item.name', '_refine_ls_class.d_res_high' ],
          [ 'item_', '_item.category_id', 'refine_ls_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_class_d_res_high', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine_ls_class.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the highest value in angstroms\n               for the interplanar spacings for the reflections used in the\n               refinement. This is called the lowest resolution.' ],
          [ 'item_', '_item.name', '_refine_ls_class.d_res_low' ],
          [ 'item_', '_item.category_id', 'refine_ls_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_class_d_res_low', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine_ls_class.R_factor_gt',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the residual factor for significantly\n               intense reflections (see _reflns.threshold_expression) included\n               in the refinement.\n               The reflections also satisfy the resolution limits established by\n               _refine_ls_class.d_res_high and _refine_ls_class.d_res_low.\n               This is the conventional R factor. See also the definition of\n               _refine_ls_class.wR_factor_all.\n\n                   sum | F(obs) - F(calc) |\n               R = ------------------------\n                         sum | F(obs) |\n\n               F(obs)  = the observed structure-factor amplitudes\n               F(calc) = the calculated structure-factor amplitudes\n\n               and the sum is taken over the reflections of this class.' ],
          [ 'item_', '_item.name', '_refine_ls_class.R_factor_gt' ],
          [ 'item_', '_item.category_id', 'refine_ls_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_class_R_factor_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine_ls_class.R_factor_all',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the residual factor for all\n               reflections satisfying the resolution limits established by\n               _refine_ls_class.d_res_high and _refine_ls_class.d_res_low.\n               This is the conventional R factor. See also the\n               definition of _refine_ls_class.wR_factor_all.\n\n                   sum | F(obs) - F(calc) |\n               R = ------------------------\n                         sum | F(obs) |\n\n               F(obs)  = the observed structure-factor amplitudes\n               F(calc) = the calculated structure-factor amplitudes\n\n               and the sum is taken over the reflections of this class.' ],
          [ 'item_', '_item.name', '_refine_ls_class.R_factor_all' ],
          [ 'item_', '_item.category_id', 'refine_ls_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_class_R_factor_all', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine_ls_class.R_Fsqd_factor',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the residual factor R(F^2^) calculated\n               on the squared amplitudes of the observed and calculated\n               structure factors, for the reflections judged significantly\n               intense (i.e. satisfying the threshold specified by\n               _reflns.threshold_expression) and included in the refinement.\n\n               The reflections also satisfy the resolution limits established\n               by _refine_ls_class.d_res_high and _refine_ls_class.d_res_low.\n\n                          sum | F(obs)^2^ - F(calc)^2^ |\n               R(Fsqd) = -------------------------------\n                                  sum F(obs)^2^\n\n               F(obs)^2^  = squares of the observed structure-factor amplitudes\n               F(calc)^2^ = squares of the calculated structure-factor\n                            amplitudes\n\n               and the sum is taken over the reflections of this class.' ],
          [ 'item_', '_item.name', '_refine_ls_class.R_Fsqd_factor' ],
          [ 'item_', '_item.category_id', 'refine_ls_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_class_R_Fsqd_factor', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine_ls_class.R_I_factor',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the residual factor R(I) for the\n               reflections judged significantly intense (i.e. satisfying the\n               threshold specified by _reflns.threshold_expression) and\n               included in the refinement.\n\n               This is most often calculated in Rietveld refinements\n               against powder data, where it is referred to as R~B~ or R~Bragg~\n\n                       sum | I(obs) - I(calc) |\n               R(I) =  ------------------------\n                              sum | I(obs) |\n\n               I(obs)  = the net observed intensities\n               I(calc) = the net calculated intensities\n\n               and the sum is taken over the reflections of this class.' ],
          [ 'item_', '_item.name', '_refine_ls_class.R_I_factor' ],
          [ 'item_', '_item.category_id', 'refine_ls_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_class_R_I_factor', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refine_ls_class.wR_factor_all',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the weighted residual factor for all\n               reflections included in the refinement. The reflections also\n               satisfy the resolution limits established by\n               _refine_ls_class.d_res_high and _refine_ls_class.d_res_low.\n               See also the _refine_ls_class.R_factor_ definitions.\n\n                    ( sum w [ Y(obs) - Y(calc) ]^2^  )^1/2^\n               wR = ( ------------------------------ )\n                    (         sum w Y(obs)^2^       )\n\n               Y(obs)  = the observed amplitude specified by\n                         _refine.ls_structure_factor_coef\n               Y(calc) = the calculated amplitude specified by\n                         _refine.ls_structure_factor_coef\n               w       = the least-squares weight\n\n               and the sum is taken over the reflections of this class.' ],
          [ 'item_', '_item.name', '_refine_ls_class.wR_factor_all' ],
          [ 'item_', '_item.category_id', 'refine_ls_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine_ls_class_wR_factor_all', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refln.class_code',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying the class to which this reflection has been\n               assigned. This code must match a value of _reflns_class.code.\n               Reflections may be grouped into classes for a variety of\n               purposes. For example, for modulated structures each reflection\n               class may be defined by the number m=sum|m~i~|, where the m~i~\n               are the integer coefficients that, in addition to h,k,l, index\n               the corresponding diffraction vector in the basis defined\n               for the reciprocal lattice.' ],
          [ 'item_', '_item.name', '_refln.class_code' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refln_class_code', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refln.d_spacing',
        [ [ 'item_',
            '_item_description.description',
            '              The d spacing in angstroms for this reflection. This is related\n               to the (sin theta)/lambda value by the expression\n                    _refln.d_spacing = 2/(_refln.sint/lambda).' ],
          [ 'item_', '_item.name', '_refln.d_spacing' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refln_d_spacing', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refln.include_status',
        [ [ 'item_',
            '_item_description.description',
            '              Classification of a reflection so as to indicate its status with\n               respect to inclusion in the refinement and the calculation of\n               R factors.' ],
          [ 'item_', '_item.name', '_refln.include_status' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_related.related_name', '_refln.status' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'o',
              '                                     (lower-case letter o for \'observed\')\n                                      satisfies _refine.ls_d_res_high\n                                      satisfies _refine.ls_d_res_low\n                                      exceeds _reflns.threshold_expression',
              '<',
              '                                     satisfies _refine.ls_d_res_high\n                                      satisfies _refine.ls_d_res_low\n                                      does not exceed\n                                        _reflns.threshold_expression',
              '-',
              'systematically absent reflection',
              'x',
              'unreliable measurement -- not used',
              'h',
              'does not satisfy _refine.ls_d_res_high',
              'l',
              'does not satisfy _refine.ls_d_res_low',
              [ '@comment', '    _item_default.value         o' ] ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refln_include_status', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_refln.mean_path_length_tbar',
        [ [ 'item_',
            '_item_description.description',
            '              Mean path length in millimetres through the crystal for this\n               reflection.' ],
          [ 'item_', '_item.name', '_refln.mean_path_length_tbar' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'millimetres' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refln_mean_path_length_tbar', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', ' save__refln.observed_status' ],
      [ '@comment', '      _item_description.description' ],
      [ '@comment',
        ' ;              Classification of a reflection so as to indicate its status' ],
      [ '@comment',
        '                with respect to inclusion in refinement and calculation of' ],
      [ '@comment', '                R factors.' ],
      [ '@comment', ' ;' ],
      [ '@comment',
        '      _item.name                 \'_refln.observed_status\'' ],
      [ '@comment', '      _item.category_id           refln' ],
      [ '@comment', '      _item.mandatory_code        no' ],
      [ '@comment', '      _item_type.code             code' ],
      [ '@comment',
        '      _item_related.related_name \'_refln.include_status\'' ],
      [ '@comment', '      _item_related.function_code replaces' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_enumeration.value' ],
      [ '@comment', '      _item_enumeration.detail' ],
      [ '@comment', '                                       o' ],
      [ '@comment',
        ' ;                                     satisfies _refine.ls_d_res_high' ],
      [ '@comment',
        '                                       satisfies _refine.ls_d_res_low' ],
      [ '@comment',
        '                                       observed by _reflns_observed_criterion' ],
      [ '@comment', ' ;' ],
      [ '@comment', '                                  <' ],
      [ '@comment',
        ' ;                                     satisfies _refine.ls_d_res_high' ],
      [ '@comment',
        '                                       satisfies _refine.ls_d_res_low' ],
      [ '@comment',
        '                                       unobserved by _reflns.observed_criterion' ],
      [ '@comment', ' ;' ],
      [ '@comment',
        '                                  -  \'systematically absent reflection\'' ],
      [ '@comment',
        '                                  x  \'unreliable measurement -- not used\'' ],
      [ '@comment',
        '                                  h  \'does not satisfy _refine.ls_d_res_high\'' ],
      [ '@comment',
        '                                  l  \'does not satisfy _refine.ls_d_res_low\'' ],
      [ '@comment', ' #    _item_default.value         o' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_aliases.alias_name' ],
      [ '@comment', '      _item_aliases.dictionary' ],
      [ '@comment', '      _item_aliases.version' ],
      [ '@comment',
        '       \'_refln_observed_status\'         cif_core.dic 2.3' ],
      [ '@comment', '       save_' ],
      [ '@comment', ' save__refln.sint/lambda' ],
      [ '@comment', '      _item_description.description' ],
      [ '@comment',
        ' ;              The (sin theta)/lambda value in reciprocal angstroms for this' ],
      [ '@comment', '                reflection.' ],
      [ '@comment', ' ;' ],
      [ '@comment',
        '      _item.name                 \'_refln.sint/lambda\'' ],
      [ '@comment', '      _item.category_id           refln' ],
      [ '@comment', '      _item.mandatory_code        no' ],
      [ '@comment', '      _item_type.code             float' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_range.minimum' ],
      [ '@comment', '      _item_range.maximum' ],
      [ '@comment', '                            0.0   0.0' ],
      [ '@comment', '                            0.0    .' ],
      [ '@comment',
        '      _item_units.code           \'reciprocal_angstroms\'' ],
      [ '@comment', '     loop_' ],
      [ '@comment', '      _item_aliases.alias_name' ],
      [ '@comment', '      _item_aliases.dictionary' ],
      [ '@comment', '      _item_aliases.version' ],
      [ '@comment',
        '       \'_refln_sint/lambda\'         cif_core.dic 2.3' ],
      [ '@comment', '       save_' ],
      [ 'save_',
        '_reflns.Friedel_coverage',
        [ [ 'item_',
            '_item_description.description',
            '              The proportion of Friedel-related reflections present in\n               the number of  \'independent\' reflections specified by\n               the item _reflns.number_all.\n\n               This proportion is calculated as the ratio:\n\n                 [N(crystal class) - N(Laue symmetry)] / N(Laue symmetry)\n\n               where, working from the DIFFRN_REFLN list,\n\n               N(crystal class) is the number of reflections obtained on\n                  averaging under the symmetry of the crystal class\n               N(Laue symmetry) is the number of reflections obtained on\n                  averaging under the Laue symmetry.\n\n               Examples:\n\n                 (a) For centrosymmetric structures, the value of\n                     _reflns.Friedel_coverage is\n                     necessarily equal to 0.0, as the crystal class\n                     is identical to the Laue symmetry.\n\n                 (b) For whole-sphere data for a crystal in the space\n                     group P1, _reflns.Friedel_coverage is equal to 1.0,\n                     as no reflection h k l is equivalent to -h -k -l\n                     in the crystal class and all Friedel pairs\n                     {h k l; -h -k -l} have been measured.\n\n                 (c) For whole-sphere data in space group Pmm2,\n                    _reflns.Friedel_coverage\n                     will be < 1.0 because although reflections h k l and\n                     -h -k -l are not equivalent when h k l indices are\n                     nonzero, they are when l=0.\n\n                 (d) For a crystal in space group Pmm2, measurements of the\n                     two inequivalent octants h >= 0, k >=0, l lead to the\n                     same value as in (c), whereas measurements of the\n                     two equivalent octants h >= 0, k, l >= 0 will lead to\n                     a zero value for _reflns.Friedel_coverage.' ],
          [ 'item_', '_item.name', '_reflns.Friedel_coverage' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '1.0', '1.0', '1.0' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_Friedel_coverage', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns.number_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The number of reflections in the REFLN list (not the\n               DIFFRN_REFLN list) that are significantly intense, satisfying\n               the criterion specified by _reflns.threshold_expression. This may\n               include Friedel-equivalent reflections (i.e. those which are\n               symmetry-equivalent under the Laue symmetry but inequivalent\n               under the crystal class) according to the nature of the\n               structure and the procedures used. Any special characteristics\n               of the reflections included in the REFLN list should be\n               described using the item _reflns.details.' ],
          [ 'item_', '_item.name', '_reflns.number_gt' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ '@comment',
            '     _item_related.related_name \'_reflns.number_observed\'' ],
          [ '@comment', '     _item_related.function_code  alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0', '0', '0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_number_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns.threshold_expression',
        [ [ 'item_',
            '_item_description.description',
            '              The threshold, usually based on multiples of u(I), u(F^2^)\n               or u(F), that serves to identify significantly intense\n               reflections, the number of which is given by _reflns.number_gt.\n               These reflections are used in the calculation of\n               _refine.ls_R_factor_gt.' ],
          [ 'item_', '_item.name', '_reflns.threshold_expression' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_threshold_expression' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_core.dic' ],
          [ 'item_', '_item_aliases.version', '2.3' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_reflns.observed_criterion', 'alternate' ] ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'I>2u(I)' ] ] ],
      [ '@comment', '#################' ],
      [ '@comment', '# REFLNS_CLASS ##' ],
      [ '@comment', '#################' ],
      [ 'save_',
        'reflns_class',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFLNS_CLASS category record details\n               of the reflections used to determine the structural\n               parameters for each reflection class.' ],
          [ 'item_', '_category.id', 'reflns_class' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_reflns_class.code' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refln_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              '_category_examples.detail',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    loop_\n        _reflns_class.number_gt\n        _reflns_class.code\n               584     \'Main\'\n               226     \'Sat1\'\n               50      \'Sat2\'',
              '\n    Example 1 - example corresponding to the one-dimensional incommensurately\n                modulated structure of K~2~SeO~4~.',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_reflns_class.code',
        [ [ 'item_',
            '_item_description.description',
            '              The code identifying a certain reflection class.' ],
          [ 'item_', '_item.name', '_reflns_class.code' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', 'm1', 's2' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_code', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_class.description',
        [ [ 'item_',
            '_item_description.description',
            '              Description of each reflection class.' ],
          [ 'item_', '_item.name', '_reflns_class.description' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'm=1 first order satellites',
              'H0L0 common projection reflections' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_description', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_class.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the smallest value in angstroms\n               for the interplanar spacings for the reflections used in the\n               refinement. This is called the highest resolution.' ],
          [ 'item_', '_item.name', '_reflns_class.d_res_high' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_d_res_high', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_class.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the largest value in angstroms\n               for the interplanar spacings for the reflections used in the\n               refinement. This is called the lowest resolution.' ],
          [ 'item_', '_item.name', '_reflns_class.d_res_low' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_d_res_low', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_class.number_gt',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the number of significantly intense\n               reflections (see _reflns.threshold_expression) in the REFLN\n               list (not the DIFFRN_REFLN list). This may include Friedel-\n               equivalent reflections (i.e. those which are symmetry-equivalent\n               under the Laue symmetry but inequivalent under the crystal\n               class)  according to the nature of the structure and the\n               procedures used. Any special characteristics of the reflections\n               included in the REFLN list should be described using the item\n               _reflns.details.' ],
          [ 'item_', '_item.name', '_reflns_class.number_gt' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0', '0', '0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_number_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_class.number_total',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the total number of reflections\n               in the REFLN list (not the DIFFRN_REFLN list). This may\n               include Friedel-equivalent reflections (i.e. those which are\n               symmetry-equivalent under the Laue symmetry but inequivalent\n               under the crystal class) according to the nature of the\n               structure and the procedures used.  Any special characteristics\n               of the reflections included in the REFLN list should be\n               described using the item _reflns.details.' ],
          [ 'item_', '_item.name', '_reflns_class.number_total' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0', '0', '0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_number_total', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_class.R_factor_all',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the residual factor for all\n               reflections included in the refinement.\n               The reflections also satisfy the resolution limits established by\n               _reflns_class.d_res_high and _reflns_class.d_res_low.\n               This is the conventional R factor. See also the\n               definition of _reflns_class.wR_factor_all.\n\n                   sum | F(obs) - F(calc) |\n               R = ------------------------\n                         sum | F(obs) |\n\n               F(obs)  = the observed structure-factor amplitudes\n               F(calc) = the calculated structure-factor amplitudes\n\n               and the sum is taken over the reflections of this class.' ],
          [ 'item_', '_item.name', '_reflns_class.R_factor_all' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_R_factor_all', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_class.R_factor_gt',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the residual factor for significantly\n               intense reflections (see _reflns.threshold_expression) included\n               in the refinement.\n               The reflections also satisfy the resolution limits established by\n               _reflns_class.d_res_high and _reflns_class.d_res_low.\n               This is the conventional R factor. See also the\n               definition of _reflns_class.wR_factor_all.\n\n                   sum | F(obs) - F(calc) |\n               R = ------------------------\n                         sum | F(obs) |\n\n               F(obs)  = the observed structure-factor amplitudes\n               F(calc) = the calculated structure-factor amplitudes\n\n               and the sum is taken over the reflections of this class.' ],
          [ 'item_', '_item.name', '_reflns_class.R_factor_gt' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_R_factor_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_class.R_Fsqd_factor',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the residual factor R(F^2^) calculated\n               on the squared amplitudes of the observed and calculated\n               structure factors for the reflections judged significantly\n               intense (i.e. satisfying the threshold specified by\n               _reflns.threshold_expression) and included in the refinement.\n\n               The reflections also satisfy the resolution limits established\n               by _reflns_class.d_res_high and _reflns_class.d_res_low.\n\n                          sum | F(obs)^2^ - F(calc)^2^ |\n               R(Fsqd) = -------------------------------\n                                  sum F(obs)^2^\n\n               F(obs)^2^  = squares of the observed structure-factor amplitudes\n               F(calc)^2^ = squares of the calculated structure-factor\n                            amplitudes\n\n               and the sum is taken over the reflections of this class.' ],
          [ 'item_', '_item.name', '_reflns_class.R_Fsqd_factor' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_R_Fsqd_factor', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_class.R_I_factor',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the residual factor R(I) for the\n               reflections judged significantly intense (i.e. satisfying the\n               threshold specified by _reflns.threshold_expression) and\n               included in the refinement.\n\n               This is most often calculated in Rietveld refinements\n               against powder data, where it is referred to as R~B~ or R~Bragg~.\n\n                       sum | I(obs) - I(calc) |\n               R(I) =  ------------------------\n                              sum | I(obs) |\n\n               I(obs)  = the net observed intensities\n               I(calc) = the net calculated intensities\n\n               and the sum is taken over the reflections of this class.' ],
          [ 'item_', '_item.name', '_reflns_class.R_I_factor' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_R_I_factor', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_class.wR_factor_all',
        [ [ 'item_',
            '_item_description.description',
            '              For each reflection class, the weighted residual factors for all\n               reflections included in the refinement. The reflections also\n               satisfy the resolution limits established by\n               _reflns_class.d_res_high and _reflns_class.d_res_low.\n               See also _reflns_class.R_factor_ definitions.\n\n                    ( sum w [ Y(obs) - Y(calc) ]^2^  )^1/2^\n               wR = ( ------------------------------ )\n                    (         sum w Y(obs)^2^       )\n\n               Y(obs)  = the observed amplitude specified by\n                         _refine.ls_structure_factor_coef\n               Y(calc) = the calculated amplitude specified by\n                         _refine.ls_structure_factor_coef\n               w       = the least-squares weight\n\n               and the sum is taken over the reflections of this class.' ],
          [ 'item_', '_item.name', '_reflns_class.wR_factor_all' ],
          [ 'item_', '_item.category_id', 'reflns_class' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_class_wR_factor_all', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_shell.meanI_over_sigI_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The ratio of the mean of the intensities of the significantly\n               intense reflections (see _reflns.threshold_expression) in\n               this shell to the mean of the standard uncertainties of the\n               intensities of the significantly intense reflections in this\n               shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.meanI_over_sigI_gt' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_reflns_shell.meanI_over_uI_gt' ],
          [ 'item_', '_item_related.function_code', 'replaces' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_shell_meanI_over_sigI_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_shell.meanI_over_uI_all',
        [ [ 'item_',
            '_item_description.description',
            '              The ratio of the mean of the intensities of all reflections\n               in this shell to the mean of the standard uncertainties of the\n               intensities of all reflections in this shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.meanI_over_uI_all' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_reflns_shell.meanI_over_sigI_all' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_shell_meanI_over_uI_all', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_shell.meanI_over_uI_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The ratio of the mean of the intensities of the significantly\n               intense reflections (see _reflns.threshold_expression) in\n               this shell to the mean of the standard uncertainties of the\n               intensities of the significantly intense reflections in this\n               shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.meanI_over_uI_gt' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_reflns_shell.meanI_over_sigI_gt',
              'alternate',
              '_reflns_shell.meanI_over_sigI_obs',
              'alternate' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_shell_meanI_over_uI_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_shell.number_measured_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The number of significantly intense reflections\n               (see _reflns.threshold_expression) measured for this\n               shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.number_measured_gt' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_related.related_name',
            '_reflns_shell.number_measured_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0', '0', '0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_shell_number_measured_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_shell.number_unique_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The total number of significantly intense reflections\n               (see _reflns.threshold_expression) resulting from merging\n               measured symmetry-equivalent reflections for this resolution\n               shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.number_unique_gt' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_related.related_name',
            '_reflns_shell.number_unique_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0', '0', '0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_shell_number_unique_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_shell.percent_possible_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The percentage of geometrically possible reflections\n               represented by significantly intense reflections\n               (see _reflns.threshold_expression) measured for this\n               shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.percent_possible_gt' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_reflns_shell.percent_possible_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '100.0', '100.0', '100.0' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_shell_percent_possible_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_shell.Rmerge_F_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The value of Rmerge(F) for significantly intense reflections\n               (see _reflns.threshold_expression) in a given shell.\n\n                           sum~i~ ( sum~j~ | F~j~ - <F> | )\n               Rmerge(F) = --------------------------------\n                               sum~i~ ( sum~j~ <F> )\n\n               F~j~  = the amplitude of the jth observation of reflection i\n               <F> = the mean of the amplitudes of all observations of\n                      reflection i\n\n               sum~i~ is taken over all reflections\n               sum~j~ is taken over all observations of each reflection.' ],
          [ 'item_', '_item.name', '_reflns_shell.Rmerge_F_gt' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_reflns_shell.Rmerge_F_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_shell_Rmerge_F_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_reflns_shell.Rmerge_I_gt',
        [ [ 'item_',
            '_item_description.description',
            '              The value of Rmerge(I) for significantly intense reflections\n               (see _reflns.threshold_expression) in a given shell.\n\n                           sum~i~ ( sum~j~ | I~j~ - <I> | )\n               Rmerge(I) = --------------------------------\n                               sum~i~ ( sum~j~ <I> )\n\n               I~j~  = the intensity of the jth observation of reflection i\n               <I> = the mean of the intensities of all observations of\n                      reflection i\n\n               sum~i~ is taken over all reflections\n               sum~j~ is taken over all observations of each reflection.' ],
          [ 'item_', '_item.name', '_reflns_shell.Rmerge_I_gt' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_reflns_shell.Rmerge_I_obs' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '0.0', '0.0', '0.0', '.' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_reflns_shell_Rmerge_I_gt', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# SPACE_GROUP ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'space_group',
        [ [ 'item_',
            '_category.description',
            '              Contains all the data items that refer to the space group as a\n               whole, such as its name or crystal system. They may be looped,\n               for example, in a list of space groups and their properties.\n\n               Only a subset of the SPACE_GROUP category items appear in\n               this dictionary.  The remainder are found in the symmetry CIF\n               dictionary.\n\n               Space-group types are identified by their number as given in\n               International Tables for Crystallography Vol. A. Specific\n               settings of the space groups can be identified either by their\n               Hall symbol or by specifying their symmetry operations.\n\n               The commonly used Hermann-Mauguin symbol determines the\n               space-group type uniquely but several different Hermann-Mauguin\n               symbols may refer to the same space-group type. A Hermann-Mauguin\n               symbol contains information on the choice of the basis, but not\n               on the choice of origin.  Different formats for the\n               Hermann-Mauguin symbol are found in the symmetry CIF dictionary.' ],
          [ 'item_', '_category.id', 'space_group' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_space_group.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'symmetry_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              '_category_examples.detail',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    _space_group.id                 1\n    _space_group.name_H-M_alt       \'C 2/c\'\n    _space_group.IT_number          15\n    _space_group.name_Hall          \'-C 2yc\'\n    _space_group.crystal_system     monoclinic',
              '\n    Example 1 - the monoclinic space group No. 15 with unique axis b.',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_space_group.crystal_system',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the system of geometric crystal classes of space\n               groups (crystal system) to which the space group belongs.\n               Note that rhombohedral space groups belong to the\n               trigonal system.' ],
          [ 'item_', '_item.name', '_space_group.crystal_system' ],
          [ 'item_', '_item.category_id', 'space_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_related.related_name',
            '_symmetry.cell_setting' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'triclinic',
              'monoclinic',
              'orthorhombic',
              'tetragonal',
              'trigonal',
              'hexagonal',
              'cubic' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_space_group_crystal_system', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_space_group.id',
        [ [ 'item_',
            '_item_description.description',
            '              This is the unique identifier for the SPACE_GROUP category.' ],
          [ 'item_', '_item.name', '_space_group.id' ],
          [ 'item_', '_item.category_id', 'space_group' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_space_group_id', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_space_group.IT_number',
        [ [ 'item_',
            '_item_description.description',
            '              The number as assigned in International Tables for\n               Crystallography Vol. A, specifying the proper affine class (i.e.\n               the orientation-preserving affine class) of space groups\n               (crystallographic space-group type) to which the space group\n               belongs.  This number defines the space-group type but not\n               the coordinate system in which it is expressed.' ],
          [ 'item_', '_item.name', '_space_group.IT_number' ],
          [ 'item_', '_item.category_id', 'space_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_related.related_name',
            '_symmetry.Int_Tables_number' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_range.minimum', '_item_range.maximum' ],
            [ '1', '1', '1', '230', '230', '230' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_space_group_IT_number', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_space_group.name_Hall',
        [ [ 'item_',
            '_item_description.description',
            '              Space-group symbol defined by Hall.\n\n               Each component of the space-group name is separated by a\n               space or an underscore.  The use of a space is strongly\n               recommended.  The underscore is only retained because it\n               was used in old CIFs.  It should not be\n               used in new CIFs.\n\n               _space_group.name_Hall uniquely defines the space group and\n               its reference to a particular coordinate system.\n\n               Ref: Hall, S. R. (1981). Acta Cryst. A37, 517-525; erratum\n                    (1981), A37, 921.\n                    [See also International Tables for Crystallography\n                    Vol. B (2001), Chapter 1.4, Appendix 1.4.2.]' ],
          [ 'item_', '_item.name', '_space_group.name_Hall' ],
          [ 'item_', '_item.category_id', 'space_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_related.related_name',
            '_symmetry.space_group_name_Hall' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ 'P 2c -2ac',
              'equivalent to Pca21',
              '-I 4bd 2ab 3',
              'equivalent to Ia3d' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_space_group_name_Hall', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_space_group.name_H-M_alt',
        [ [ 'item_',
            '_item_description.description',
            '              _space_group.name_H-M_alt allows any Hermann-Mauguin symbol\n               to be given. The way in which this item is used is determined\n               by the user and in general is not intended to be interpreted by\n               computer. It may, for example, be used to give one of the\n               extended Hermann-Mauguin symbols given in Table 4.3.2.1 of\n               International Tables for Crystallography Vol. A (2002) or\n               a Hermann-Mauguin symbol for a conventional or unconventional\n               setting.\n\n               Each component of the space-group name is separated by a\n               space or an underscore. The use of a space is strongly\n               recommended.  The underscore is only retained because it\n               was used in old CIFs. It should not be\n               used in new CIFs. Subscripts should appear without special\n               symbols. Bars should be given as negative signs before the\n               numbers to which they apply.\n\n               The commonly used Hermann-Mauguin symbol determines the space-\n               group type uniquely but a given space-group type may be\n               described by more than one Hermann-Mauguin symbol. The space-\n               group type is best described using _space_group.IT_number.\n\n               The Hermann-Mauguin symbol may contain information on the\n               choice of basis, but not on the choice of origin. To\n               define the setting uniquely, use _space_group.name_Hall or\n               list the symmetry operations.' ],
          [ 'item_', '_item.name', '_space_group.name_H-M_alt' ],
          [ 'item_', '_item.category_id', 'space_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_related.related_name',
            '_symmetry.space_group_name_H-M' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '                               loop_\n                                   _space_group.name_H-M_alt\n                                    \'C m c m\'\n                                    \'C 2/c 2/m 21/m\'\n                                    \'A m a m\'',
              'three examples for space group No. 63' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_space_group_name_H-M_alt', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', '######################' ],
      [ '@comment', '# SPACE_GROUP_SYMOP ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'space_group_symop',
        [ [ 'item_',
            '_category.description',
            '              Contains information about the symmetry operations of the\n               space group.' ],
          [ 'item_', '_category.id', 'space_group_symop' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_space_group_symop.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'symmetry_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              '_category_examples.detail',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '   loop_\n    _space_group_symop.id\n    _space_group_symop.operation_xyz\n      1    x,y,z\n      2   -x,-y,-z\n      3   -x,1/2+y,1/2-z\n      4    x,1/2-y,1/2+z',
              '\n    Example 1 - The symmetry operations for the space group P21/c.',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_space_group_symop.id',
        [ [ 'item_',
            '_item_description.description',
            '              An arbitrary identifier that uniquely labels each symmetry\n               operation in the list.' ],
          [ 'item_', '_item.name', '_space_group_symop.id' ],
          [ 'item_', '_item.category_id', 'space_group_symop' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_related.related_name', '_symmetry_equiv.id' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_space_group_symop_id', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_space_group_symop.operation_xyz',
        [ [ 'item_',
            '_item_description.description',
            '               A parsable string giving one of the symmetry operations of the\n                space group in algebraic form.  If W is a matrix representation\n                of the rotational part of the symmetry operation defined by the\n                positions and signs of x, y and z, and w is a column of\n                translations defined by the fractions, an equivalent position\n                X\' is generated from a given position X by the equation\n\n                          X\' = WX + w\n\n                (Note: X is used to represent bold_italics_x in International\n                Tables for Crystallography Vol. A, Part 5)\n\n                When a list of symmetry operations is given, it must contain\n                a complete set of coordinate representatives which generates\n                all the operations of the space group by the addition of\n                all primitive translations of the space group. Such\n                representatives are to be found as the coordinates of\n                the general-equivalent position in International Tables for\n                Crystallography Vol. A (2002), to which it is necessary to\n                add any centring translations shown above the\n                general-equivalent position.\n\n                That is to say, it is necessary to list explicity all the\n                symmetry operations required to generate all the atoms in\n                the unit cell defined by the setting used.' ],
          [ 'item_', '_item.name', '_space_group_symop.operation_xyz' ],
          [ 'item_', '_item.category_id', 'space_group_symop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ '@comment', '     _item_default.value        \'x,y,z\'' ],
          [ 'item_',
            '_item_related.related_name',
            '_symmetry_equiv.pos_as_xyz' ],
          [ 'item_', '_item_related.function_code', 'alternate' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ 'x,1/2-y,1/2+z',
              '                                glide reflection through the plane (x,1/4,z),\n                                 with glide vector 1/2 c' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_space_group_symop_operation_xyz', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_space_group_symop.sg_id',
        [ [ 'item_',
            '_item_description.description',
            '              This must match a particular value of _space_group.id, allowing\n               the symmetry operation to be identified with a particular space\n               group.' ],
          [ 'item_', '_item.name', '_space_group_symop.sg_id' ],
          [ 'item_', '_item.category_id', 'space_group_symop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_space_group_symop_sg_id', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', '##################' ],
      [ '@comment', '# VALENCE_PARAM ##' ],
      [ '@comment', '##################' ],
      [ 'save_',
        'valence_param',
        [ [ 'item_',
            '_category.description',
            '              Data items in the VALENCE_PARAM category define the\n               parameters used for calculating bond valences from bond\n               lengths.  In addition to the parameters, a pointer\n               is given to the reference (in VALENCE_REF) from which\n               the bond-valence parameters were taken.' ],
          [ 'item_', '_category.id', 'valence_param' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'valence_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_valence_param.atom_1',
              '_valence_param.atom_1_valence',
              '_valence_param.atom_2',
              '_valence_param.atom_2_valence' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              '_category_examples.detail',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    loop_\n    _valence_param.atom_1\n    _valence_param.atom_1_valence\n    _valence_param.atom_2\n    _valence_param.atom_2_valence\n    _valence_param.Ro\n    _valence_param.B\n    _valence_param.ref_id\n    _valence_param.details\n      Cu 2 O -2 1.679 0.37 a .\n      Cu 2 O -2 1.649 0.37 j .\n      Cu 2 N -3 1.64  0.37 m \'2-coordinate N\'\n      Cu 2 N -3 1.76  0.37 m \'3-coordinate N\'\n    loop_\n    _valence_ref.id\n    _valence_ref.reference\n      a  \'Brown & Altermatt (1985), Acta Cryst. B41, 244-247\'\n      j  \'Liu & Thorp (1993), Inorg. Chem. 32, 4102-4205\'\n      m  \'See, Krause & Strub (1998), Inorg. Chem. 37, 5369-5375\'',
              '\n    Example 1 - a bond-valence parameter list with accompanying references.',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_valence_param.atom_1',
        [ [ 'item_',
            '_item_description.description',
            '              The element symbol of the first atom forming the bond whose\n               bond-valence parameters are given in this category.' ],
          [ 'item_', '_item.name', '_valence_param.atom_1' ],
          [ 'item_', '_item.category_id', 'valence_param' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_param_atom_1', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_valence_param.atom_1_valence',
        [ [ 'item_',
            '_item_description.description',
            '              The valence (formal charge) of the first atom whose\n               bond-valence parameters are given in this category.' ],
          [ 'item_', '_item.name', '_valence_param.atom_1_valence' ],
          [ 'item_', '_item.category_id', 'valence_param' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_param_atom_1_valence', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_valence_param.atom_2',
        [ [ 'item_',
            '_item_description.description',
            '              The element symbol of the second atom forming the bond whose\n               bond-valence parameters are given in this category.' ],
          [ 'item_', '_item.name', '_valence_param.atom_2' ],
          [ 'item_', '_item.category_id', 'valence_param' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_param_atom_2', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_valence_param.atom_2_valence',
        [ [ 'item_',
            '_item_description.description',
            '              The valence (formal charge) of the second atom whose\n               bond-valence parameters are given in this category.' ],
          [ 'item_', '_item.name', '_valence_param.atom_2_valence' ],
          [ 'item_', '_item.category_id', 'valence_param' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_param_atom_2_valence', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_valence_param.B',
        [ [ 'item_',
            '_item_description.description',
            '              The bond-valence parameter B used in the expression\n\n                     s = exp[(Ro - R)/B]\n\n               where s is the valence of a bond of length R.' ],
          [ 'item_', '_item.name', '_valence_param.B' ],
          [ 'item_', '_item.category_id', 'valence_param' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_param_B', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_valence_param.details',
        [ [ 'item_',
            '_item_description.description',
            '              Details of or comments on the bond-valence parameters.' ],
          [ 'item_', '_item.name', '_valence_param.details' ],
          [ 'item_', '_item.category_id', 'valence_param' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_param_details', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_valence_param.id',
        [ [ 'item_',
            '_item_description.description',
            '              An identifier for the valence parameters of a bond between\n               the given atoms.' ],
          [ 'item_', '_item.name', '_valence_param.id' ],
          [ 'item_', '_item.category_id', 'valence_param' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_param_id', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_valence_param.ref_id',
        [ [ 'item_',
            '_item_description.description',
            '              An identifier which links to the reference to the source\n               from which the bond-valence parameters are taken. A child\n               of _valence_ref.id which it must match.' ],
          [ 'item_', '_item.name', '_valence_param.ref_id' ],
          [ 'item_', '_item.category_id', 'valence_param' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_param_ref_id', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_valence_param.Ro',
        [ [ 'item_',
            '_item_description.description',
            '              The bond-valence parameter Ro used in the expression\n\n                     s = exp[(Ro - R)/B]\n\n               where s is the valence of a bond of length R.' ],
          [ 'item_', '_item.name', '_valence_param.Ro' ],
          [ 'item_', '_item.category_id', 'valence_param' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_param_Ro', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# VALENCE_REF ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'valence_ref',
        [ [ 'item_',
            '_category.description',
            '              Data items in the VALENCE_REF category list the references\n               from which the bond-valence parameters have been taken.' ],
          [ 'item_', '_category.id', 'valence_ref' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_valence_ref.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'valence_group' ] ] ] ],
      [ 'save_',
        '_valence_ref.id',
        [ [ 'item_',
            '_item_description.description',
            '              An identifier for items in this category. Parent of\n               _valence_param.ref_id, which must have the same value.' ],
          [ 'item_', '_item.name', '_valence_ref.id' ],
          [ 'item_', '_item.category_id', 'valence_ref' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_ref_id', 'cif_core.dic', '2.3' ] ] ] ],
      [ 'save_',
        '_valence_ref.reference',
        [ [ 'item_',
            '_item_description.description',
            '              Literature reference from which the valence parameters\n               identified by _valence_param.id were taken.' ],
          [ 'item_', '_item.name', '_valence_ref.reference' ],
          [ 'item_', '_item.category_id', 'valence_ref' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_valence_ref_reference', 'cif_core.dic', '2.3' ] ] ] ],
      [ '@comment',
        '-eof-eof-eof-eof-eof-eof-eof-eof-eof-eof-eof-eof-eof-eof-eof-eof-eof-eof-eof' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment',
        ' File:  mmcif_pdbx-def-2.dic                                             ' ],
      [ '@comment', ' ' ],
      [ '@comment',
        '                      PDB Exchange Data Dictionary' ],
      [ '@comment',
        ' This data dictionary contains definitions used by wwPDB for data exchange' ],
      [ '@comment', ' and data processing.  ' ],
      [ '@comment',
        '                           Definition Section 2' ],
      [ '@comment', '                            ' ],
      [ '@comment',
        '                                                                        ' ],
      [ '@comment',
        '##########################################################################' ],
      [ 'save_',
        'pdbx_audit',
        [ [ 'item_',
            '_category.description',
            '              The PDBX_AUDIT holds current version information.' ],
          [ 'item_', '_category.id', 'pdbx_audit' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_', [ '_category_key.name' ], [ '_pdbx_audit.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'audit_group' ] ] ] ],
      [ 'save_',
        '_pdbx_audit.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_audit.entry_id identifies the data block.' ],
          [ 'item_', '_item.name', '_pdbx_audit.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_audit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.child_name', '_pdbx_audit.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_', '_item_examples.case', 'BDL001' ] ] ],
      [ 'save_',
        '_pdbx_audit.current_version',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_audit.entry_id identifies the data block.' ],
          [ 'item_', '_item.name', '_pdbx_audit.current_version' ],
          [ 'item_', '_item.category_id', 'pdbx_audit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_audit.current_version' ],
          [ 'item_', '_item_linked.parent_name', '_audit.revision_id' ],
          [ 'item_', '_item_examples.case', '1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ '@comment', '######################' ],
      [ '@comment', '# PDBX_AUDIT_AUTHOR ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'pdbx_audit_author',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_AUDIT_AUTHOR category record details about\n               the author(s) of the data block.' ],
          [ 'item_', '_category.id', 'pdbx_audit_author' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_audit_author.ordinal' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'audit_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 5HVP and laboratory records for the\n                structure corresponding to PDB entry 5HVP',
              '\n     loop_\n    _pdbx_audit_author.name\n    _pdbx_audit_author.address\n    _pdbx_audit_author.ordinal\n     \'Fitzgerald, Paula M.D.\'\n    ; Department of Biophysical Chemistry\n      Merck Research Laboratories\n      P. O. Box 2000, Ry80M203\n      Rahway, New Jersey  07065\n      USA\n    ;\n      1\n     \'McKeever, Brian M.\'\n    ; Department of Biophysical Chemistry\n      Merck Research Laboratories\n      P. O. Box 2000, Ry80M203\n      Rahway, New Jersey  07065\n      USA\n    ;\n     2\n     \'Van Middlesworth, J.F.\'\n    ; Department of Biophysical Chemistry\n      Merck Research Laboratories\n      P. O. Box 2000, Ry80M203\n      Rahway, New Jersey  07065\n      USA\n    ;\n     3\n     \'Springer, James P.\'\n    ; Department of Biophysical Chemistry\n      Merck Research Laboratories\n      P. O. Box 2000, Ry80M203\n      Rahway, New Jersey  07065\n      USA\n    ;\n     4',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_audit_author.address',
        [ [ 'item_',
            '_item_description.description',
            '              The address of an author of this data block. If there are\n               multiple authors, _pdbx_audit_author.address is looped with\n               _pdbx_audit_author.name.' ],
          [ 'item_', '_item.name', '_pdbx_audit_author.address' ],
          [ 'item_', '_item.category_id', 'pdbx_audit_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 Department\n                                  Institute\n                                  Street\n                                  City and postcode\n                                  COUNTRY' ] ] ],
      [ 'save_',
        '_pdbx_audit_author.name',
        [ [ 'item_',
            '_item_description.description',
            '              The name of an author of this data block. If there are multiple\n               authors, _pdbx_audit_author.name is looped with _pdbx_audit_author.address.\n               The family name(s), followed by a comma and including any\n               dynastic compoents, precedes the first name(s) or initial(s).' ],
          [ 'item_', '_item.name', '_pdbx_audit_author.name' ],
          [ 'item_', '_item.category_id', 'pdbx_audit_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Bleary, Percival R.',
              'O\'Neil, F.K.',
              'Van den Bossche, G.',
              'Yang, D.-L.',
              'Simonov, Yu.A' ] ] ] ],
      [ 'save_',
        '_pdbx_audit_author.ordinal',
        [ [ 'item_',
            '_item_description.description',
            '              A unique sequential integer identifier for each author. ' ],
          [ 'item_', '_item.name', '_pdbx_audit_author.ordinal' ],
          [ 'item_', '_item.category_id', 'pdbx_audit_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_database_message',
        [ [ 'item_',
            '_category.description',
            '              The PDBX_DATABASE_MESSAGE category provides information about\n               correspondance related to a structure deposition.' ],
          [ 'item_', '_category.id', 'pdbx_database_message' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_database_message.message_id',
              '_pdbx_database_message.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdbx_group' ] ] ] ],
      [ 'save_',
        '_pdbx_database_message.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_database_message.entry_id identifies the data block.' ],
          [ 'item_', '_item.name', '_pdbx_database_message.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_database_message.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_', '_item_examples.case', 'BDL001' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.message_id',
        [ [ 'item_',
            '_item_description.description',
            '              This is an unique and sequential identifier for a message.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_message.message_id' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'message 1' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.message_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.date',
        [ [ 'item_',
            '_item_description.description',
            '              This is the date when a message was sent or received.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_message.date' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.date' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.content_type',
        [ [ 'item_',
            '_item_description.description',
            '              This code defines the content of the message.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_message.content_type' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'DEPOSIT',
              'A component of a deposition or revision',
              'REMINDER',
              'A message reminding the depositor to send materials',
              'QUERY',
              'A query to a depositor for specific information',
              'OTHER',
              'A miscellaneous message' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.content_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.message_type',
        [ [ 'item_',
            '_item_description.description',
            '              Defines how the message was sent or received.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_message.message_type' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'FAX',
              'A facsimile message',
              'EMAIL',
              'An electronic mail message',
              'MAIL',
              'A conventional mail message',
              'PHONE',
              'A phone message' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.message_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.sender',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the sender.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_message.sender' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.sender' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.sender_address_fax',
        [ [ 'item_',
            '_item_description.description',
            '              The FAX phone number of the sender.\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_message.sender_address_fax' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'fax' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.sender_address_fax' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.sender_address_phone',
        [ [ 'item_',
            '_item_description.description',
            '              The phone number of the sender.\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_message.sender_address_phone' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'phone' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.sender_address_phone' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.sender_address_email',
        [ [ 'item_',
            '_item_description.description',
            '              The email address of the sender.\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_message.sender_address_email' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'email' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.sender_address_email' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.sender_address_mail',
        [ [ 'item_',
            '_item_description.description',
            '              The postal address of the sender.\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_message.sender_address_mail' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.sender_address_mail' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.receiver',
        [ [ 'item_',
            '_item_description.description',
            '              The name of the receiver.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_message.receiver' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.receiver' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.receiver_address_fax',
        [ [ 'item_',
            '_item_description.description',
            '              The FAX phone number of the receiver.\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_message.receiver_address_fax' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'fax' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.receiver_address_fax' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.receiver_address_phone',
        [ [ 'item_',
            '_item_description.description',
            '              The phone number of the receiver.\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_message.receiver_address_phone' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'phone' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.receiver_address_phone' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.receiver_address_email',
        [ [ 'item_',
            '_item_description.description',
            '              The email address of the receiver.\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_message.receiver_address_email' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'email' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.receiver_address_email' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.receiver_address_mail',
        [ [ 'item_',
            '_item_description.description',
            '              The postal address of the receiver.\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_message.receiver_address_mail' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.receiver_address_mail' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_message.message',
        [ [ 'item_',
            '_item_description.description',
            '              The text of the message.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_message.message' ],
          [ 'item_', '_item.category_id', 'pdbx_database_message' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_message.message' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_database_PDB_obs_spr',
        [ [ 'item_',
            '_category.description',
            '              The PDBX_DATABASE_PDB_OBS_SPR category provides placeholders\n               for information on obsolete/superseded PDB entries' ],
          [ 'item_', '_category.id', 'pdbx_database_PDB_obs_spr' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_database_PDB_obs_spr.pdb_id',
              '_pdbx_database_PDB_obs_spr.replace_pdb_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdbx_group' ] ] ] ],
      [ 'save_',
        '_pdbx_database_PDB_obs_spr.id',
        [ [ 'item_',
            '_item_description.description',
            '     Identifier for the type of obsolete entry to be added to this entry.' ],
          [ 'item_', '_item.name', '_pdbx_database_PDB_obs_spr.id' ],
          [ 'item_', '_item.category_id', 'pdbx_database_PDB_obs_spr' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', 'OBSLTE' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'OBSLTE', 'SPRSDE' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_PDB_obs_spr.id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_PDB_obs_spr.date',
        [ [ 'item_',
            '_item_description.description',
            '              The date of replacement.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_PDB_obs_spr.date' ],
          [ 'item_', '_item.category_id', 'pdbx_database_PDB_obs_spr' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1997-03-30' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_PDB_obs_spr.date' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_PDB_obs_spr.pdb_id',
        [ [ 'item_',
            '_item_description.description',
            '              The new PDB identifier for the replaced entry.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_PDB_obs_spr.pdb_id' ],
          [ 'item_', '_item.category_id', 'pdbx_database_PDB_obs_spr' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '2ABC' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_PDB_obs_spr.pdb_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_PDB_obs_spr.replace_pdb_id',
        [ [ 'item_',
            '_item_description.description',
            '              The PDB identifier for the replaced (OLD) entry.\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_PDB_obs_spr.replace_pdb_id' ],
          [ 'item_', '_item.category_id', 'pdbx_database_PDB_obs_spr' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '3ABC' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_PDB_obs_spr.replace_pdb_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_database_proc',
        [ [ 'item_',
            '_category.description',
            '              Internal records to track the data processing cycle.' ],
          [ 'item_', '_category.id', 'pdbx_database_proc' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_database_proc.cycle_id',
              '_pdbx_database_proc.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdbx_group' ] ],
          [ 'item_',
            '_category_examples.case',
            '\n     _pdbx_database_proc.entry_id           \'BDL001\'\n     _pdbx_database_proc.cycle_id           1\n     _pdbx_database_proc.date_begin_cycle   1998-02-27\n     _pdbx_database_proc.date_end_cycle     1998-02-27' ] ] ],
      [ 'save_',
        '_pdbx_database_proc.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '     The value of _pdbx_database_proc.entry_id identifies the data block.' ],
          [ 'item_', '_item.name', '_pdbx_database_proc.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_database_proc' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_database_proc.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_', '_item_examples.case', 'BDL001' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_proc.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_proc.cycle_id',
        [ [ 'item_',
            '_item_description.description',
            '              This is a number of the processing cycle.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_proc.cycle_id' ],
          [ 'item_', '_item.category_id', 'pdbx_database_proc' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '1 for the initial cycle' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_proc.cycle_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_proc.date_begin_cycle',
        [ [ 'item_',
            '_item_description.description',
            '              This is the date of the start of the processing cycle.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_proc.date_begin_cycle' ],
          [ 'item_', '_item.category_id', 'pdbx_database_proc' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-27' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_proc.date_begin_cycle' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_proc.date_end_cycle',
        [ [ 'item_',
            '_item_description.description',
            '              This is the date of the end of the processing cycle.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_proc.date_end_cycle' ],
          [ 'item_', '_item.category_id', 'pdbx_database_proc' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-27' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_proc.date_end_cycle' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_proc.details',
        [ [ 'item_',
            '_item_description.description',
            '              Special details about the current processing cycle.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_proc.details' ],
          [ 'item_', '_item.category_id', 'pdbx_database_proc' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_proc.details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_database_remark',
        [ [ 'item_',
            '_category.description',
            '\n       Data items in the PDBX_DATABASE_REMARK category record keep additional \n       information about the entry.  They are mostly used to create \n       \'non-standard\' PDB REMARK annotations (6-99).' ],
          [ 'item_', '_category.id', 'pdbx_database_remark' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_database_remark.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on PDB entry 1ABC',
              '\n     loop_\n    _pdbx_database_remark.id\n    _pdbx_database_remark.text\n    1\n    ; THE NON-CRYSTALLOGRAPHIC RELATIONSHIP BETWEEN THE THREE     \n      DOUBLE HELICES IN THE ASYMMETRIC UNIT IS DESCRIBED           \n      IN THE MTRIX1-3 RECORDS.\n    ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_database_remark.id',
        [ [ 'item_',
            '_item_description.description',
            '              A unique identifier for the PDB remark record.' ],
          [ 'item_', '_item.name', '_pdbx_database_remark.id' ],
          [ 'item_', '_item.category_id', 'pdbx_database_remark' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_database_PDB_remark.id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_remark.text',
        [ [ 'item_',
            '_item_description.description',
            '              The full text of the PDB remark record.' ],
          [ 'item_', '_item.name', '_pdbx_database_remark.text' ],
          [ 'item_', '_item.category_id', 'pdbx_database_remark' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_database_PDB_remark.text' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_database_status',
        [ [ 'item_',
            '_category.description',
            '              These are internal RCSB records to keep track of data processing\n               and status of the entry.' ],
          [ 'item_', '_category.id', 'pdbx_database_status' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_database_status.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'database_group', 'pdbx_group' ] ],
          [ 'item_',
            '_category_examples.case',
            '\n     _pdbx_database_status.entry_id                      1ABC\n     _pdbx_database_status.status_code                   REL  \n     _pdbx_database_status.author_release_status_code    REL  \n\n     _pdbx_database_status.deposit_site                  RCSB\n     _pdbx_database_status.process_site                  RCSB\n     _pdbx_database_status.recvd_initial_deposition_date 1996-02-13\n\n     _pdbx_database_status.date_author_approval          1996-02-13 \n     _pdbx_database_status.recvd_author_approval         Y\n     _pdbx_database_status.author_approval_type          EXPLICIT\n\n     _pdbx_database_status.hold_for_publication          N\n     _pdbx_database_status.date_hold_coordinates         1996-02-13\n     _pdbx_database_status.date_hold_struct_fact         1996-02-13\n\n     _pdbx_database_status.date_of_PDB_release           1996-02-14 \n\n     _pdbx_database_status.date_deposition_form          1996-02-13\n     _pdbx_database_status.date_coordinates              1996-02-13\n     _pdbx_database_status.date_struct_fact              1996-02-13\n     _pdbx_database_status.date_manuscript               1996-02-13\n     _pdbx_database_status.date_nmr_constraints          ?\n\n     _pdbx_database_status.recvd_deposit_form            Y\n     _pdbx_database_status.recvd_coordinates             Y\n     _pdbx_database_status.recvd_struct_fact             Y\n     _pdbx_database_status.recvd_manuscript              Y\n     _pdbx_database_status.recvd_nmr_constraints         N' ] ] ],
      [ 'save_',
        '_pdbx_database_status.status_code',
        [ [ 'item_',
            '_item_description.description',
            '              Code for status of file.' ],
          [ 'item_', '_item.name', '_pdbx_database_status.status_code' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'PROC',
              'Processing in progress',
              'WAIT',
              'Awaiting author approval',
              'REL',
              'Released',
              'HOLD',
              'On hold until yyyy-mm-dd',
              'HPUB',
              'On hold until publication',
              'OBS',
              'Entry has been obsoleted and replaced by another entry',
              'DEL',
              'Entry has been deleted',
              'WDRN',
              'Entry has been withdrawn',
              'BIB',
              'Only a bibliographic entry exists for this entry; there are no coordinates',
              'REV',
              'Entry currently under revision',
              'UPD',
              'Deprecated code' ] ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'REL', 'HOLD', 'REV', 'BIB' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.status_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.author_release_status_code',
        [ [ 'item_',
            '_item_description.description',
            '              The release status authorized by the depositor.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.author_release_status_code' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'REL',
              'Release',
              'HOLD',
              'On hold until yyyy-mm-dd',
              'HPUB',
              'On hold until publication',
              'OBS',
              'Entry has been obsoleted and replaced by another entry',
              'WDRN',
              'Entry has been withdrawn' ] ],
          [ 'item_',
            '_item_examples.case',
            '\n     REL   = Release \n     HOLD  = On hold until yyyy-mm-dd\n     HPUB  = On hold until publication\n     OBS   = Entry has been obsoleted and replaced by another entry\n     WDRN  = Entry has been withdrawn by depositor\t' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.author_release_status_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.status_code_sf',
        [ [ 'item_',
            '_item_description.description',
            '              Code for status of structure factor file.' ],
          [ 'item_', '_item.name', '_pdbx_database_status.status_code_sf' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'PROC',
              'Processing in progress',
              'REL',
              'Release',
              'HOLD',
              'On hold until yyyy-mm-dd',
              'HPUB',
              'On hold until publication',
              'OBS',
              'Entry has been obsoleted and replaced by another entry',
              'WDRN',
              'Entry has been withdrawn',
              'WAIT',
              'Awaiting author approval' ] ],
          [ 'item_',
            '_item_examples.case',
            '\n     PROC  = Processing in progress\n     REL   = Release\n     HOLD  = On hold until yyyy-mm-dd\n     HPUB  = On hold until publication\n     OBS   = Entry has been obsoleted and replaced by another entry\n     WDRN  = Entry has been withdrawn by depositor\n     WAIT  = Awaiting author approval' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.status_code_sf' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.status_code_mr',
        [ [ 'item_',
            '_item_description.description',
            '              Code for status of NMR constraints file.' ],
          [ 'item_', '_item.name', '_pdbx_database_status.status_code_mr' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'PROC',
              'Processing in progress',
              'REL',
              'Release',
              'HOLD',
              'On hold until yyyy-mm-dd',
              'HPUB',
              'On hold until publication',
              'OBS',
              'Entry has been obsoleted and replaced by another entry',
              'WDRN',
              'Entry has been withdrawn',
              'WAIT',
              'Awaiting author approval' ] ],
          [ 'item_',
            '_item_examples.case',
            '\n     PROC  = Processing in progress\n     REL   = Release\n     HOLD  = On hold until yyyy-mm-dd\n     HPUB  = On hold until publication\n     OBS   = Entry has been obsoleted and replaced by another entry\n     WDRN  = Entry has been withdrawn by depositor\n     WAIT  = Awaiting author approval' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.status_code_mr' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.dep_release_code_coordinates',
        [ [ 'item_',
            '_item_description.description',
            '\nThe deposited coordinates for this deposition will be released according\nthe value of this item.  ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.dep_release_code_coordinates' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_default.value', 'RELEASE NOW' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'RELEASE NOW',
              'Release immediately',
              'HOLD FOR PUBLICATION',
              'Hold until primary citation is published',
              'HOLD FOR 6 WEEKS',
              'Hold for 6 weeks',
              'HOLD FOR 6 MONTHS',
              'Hold for 6 months',
              'HOLD FOR 1 YEAR',
              'Hold for 1 year' ] ],
          [ 'item_',
            '_item_examples.case',
            '\n     RELEASE NOW          =    Release immediately\n     HOLD FOR PUBLICATION =    Hold until the primary citation is published\n     HOLD FOR 6 WEEKS     =    Hold for 6 weeks\n     HOLD FOR 6 MONTHS    =    Hold for 6 months\n     HOLD FOR 1 YEAR      =    Hold for 1 year' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.dep_release_code_coordinates' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.dep_release_code_sequence',
        [ [ 'item_',
            '_item_description.description',
            '\nThe sequence information for this deposition will be released according\nthe value of this item.  Setting this status code to "RELEASE NOW" indicates \nthat the macromolecular sequence(s) for this entry may be displayed in PDB\nstatus reports prior to the release of the entry.   Setting this status\ncode to "HOLD FOR RELEASE" conceals the sequence information in PDB status\nreports until the coordinate data for this entry are released.  ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.dep_release_code_sequence' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ '@comment',
            '    _item_default.value           \'RELEASE NOW\'' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'RELEASE NOW',
              'Release sequence information in status reports immediately',
              'HOLD FOR RELEASE',
              'Conceal sequence information in status reports until coordinate data \nis released' ] ],
          [ 'item_',
            '_item_examples.case',
            '\n RELEASE NOW =      Release sequence information in status reports immediately\n HOLD FOR RELEASE = Conceal sequence information in status reports until \n                    coordinate data is release' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.dep_release_code_sequence' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.dep_release_code_struct_fact',
        [ [ 'item_',
            '_item_description.description',
            '\nThe deposited structure factors for this deposition will be released according\nthe value of this item.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.dep_release_code_struct_fact' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_default.value', 'RELEASE NOW' ],
          [ '@comment', '     loop_' ],
          [ '@comment', '    _item_enumeration.value' ],
          [ '@comment', '    _item_enumeration.detail      ' ],
          [ '@comment',
            '     \'RELEASE NOW\'            \'Release immediately\'' ],
          [ '@comment',
            '     \'HOLD FOR PUBLICATION\'   \'Hold until primary citation is published\'' ],
          [ '@comment',
            '     \'HOLD FOR 6 MONTHS\'      \'Hold for 6 months\'' ],
          [ '@comment',
            '     \'HOLD FOR 1 YEAR\'        \'Hold for 1 year\'' ],
          [ '@comment',
            '     \'HOLD FOR 4 YEARS\'       \'Hold for 4 years\'' ],
          [ 'item_',
            '_item_examples.case',
            '\n     RELEASE NOW          =    Release immediately\n     HOLD FOR PUBLICATION =    Hold until the primary citation is published\n     HOLD FOR 6 WEEKS     =    Hold for 6 weeks\n     HOLD FOR 6 MONTHS    =    Hold for 6 months\n     HOLD FOR 1 YEAR      =    Hold for 1 year' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.dep_release_code_struct_fact' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.dep_release_code_nmr_constraints',
        [ [ 'item_',
            '_item_description.description',
            '\nThe deposited NMR constrait data for this deposition will be released according\nthe value of this item.  ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.dep_release_code_nmr_constraints' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_default.value', 'RELEASE NOW' ],
          [ '@comment', '     loop_' ],
          [ '@comment', '    _item_enumeration.value' ],
          [ '@comment', '    _item_enumeration.detail      ' ],
          [ '@comment',
            '     \'RELEASE NOW\'            \'Release immediately\'' ],
          [ '@comment',
            '     \'HOLD FOR PUBLICATION\'   \'Hold until primary citation is published\'' ],
          [ '@comment',
            '     \'HOLD FOR 6 MONTHS\'      \'Hold for 6 months\'' ],
          [ '@comment',
            '     \'HOLD FOR 1 YEAR\'        \'Hold for 1 year\'' ],
          [ 'item_',
            '_item_examples.case',
            '\n     RELEASE NOW          =    Release immediately\n     HOLD FOR PUBLICATION =    Hold until the primary citation is published\n     HOLD FOR 6 WEEKS     =    Hold for 6 weeks\n     HOLD FOR 6 MONTHS    =    Hold for 6 months\n     HOLD FOR 1 YEAR      =    Hold for 1 year' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.dep_release_code_nmr_constraints' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '       The value of _pdbx_database_status.entry_id identifies the data block.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_status.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_database_status.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_', '_item_examples.case', 'BDL001' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.recvd_deposit_form',
        [ [ 'item_',
            '_item_description.description',
            '              This code indicates whether the deposition form for an entry\n               has been received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.recvd_deposit_form' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar1' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes', 'N', 'No' ] ],
          [ 'item_', '_item_examples.case', 'Y' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.recvd_deposit_form' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_deposition_form',
        [ [ 'item_',
            '_item_description.description',
            '              The date the deposition form is received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_deposition_form' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1982-02-21' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_deposition_form' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.recvd_coordinates',
        [ [ 'item_',
            '_item_description.description',
            '              This code indicates whether the coordinates for an entry\n               have been received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.recvd_coordinates' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar1' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes', 'N', 'No' ] ],
          [ 'item_', '_item_examples.case', 'Y' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.recvd_coordinates' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_coordinates',
        [ [ 'item_',
            '_item_description.description',
            '              The date the coordinates are received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_coordinates' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-21' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_coordinates' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.recvd_struct_fact',
        [ [ 'item_',
            '_item_description.description',
            '              This code indicates whether the structure factors for an entry\n               have been received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.recvd_struct_fact' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar1' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes', 'N', 'No' ] ],
          [ 'item_', '_item_examples.case', 'Y' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.recvd_struct_fact' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_struct_fact',
        [ [ 'item_',
            '_item_description.description',
            '              The date the structure factors are received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_struct_fact' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-28' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_struct_fact' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.recvd_nmr_constraints',
        [ [ 'item_',
            '_item_description.description',
            '              This code indicates whether the NMR contraint data for an entry\n               have been received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.recvd_nmr_constraints' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar1' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes', 'N', 'No' ] ],
          [ 'item_', '_item_examples.case', 'Y' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.recvd_nmr_constraints' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_nmr_constraints',
        [ [ 'item_',
            '_item_description.description',
            '              The date the structure factors are received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_nmr_constraints' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-28' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_nmr_constraints' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.recvd_internal_approval',
        [ [ 'item_',
            '_item_description.description',
            '              This code indicates whether the internal approval for an entry  \n               have been received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.recvd_internal_approval' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar1' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes', 'N', 'No' ] ],
          [ 'item_', '_item_examples.case', 'Y' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.recvd_internal_approval' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.recvd_manuscript',
        [ [ 'item_',
            '_item_description.description',
            '              This code indicates whether the manuscript for an entry\n               has been received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.recvd_manuscript' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar1' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes', 'N', 'No' ] ],
          [ 'item_', '_item_examples.case', 'Y' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.recvd_manuscript' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_manuscript',
        [ [ 'item_',
            '_item_description.description',
            '              The date the manuscript is received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_manuscript' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-28' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_manuscript' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.name_depositor',
        [ [ 'item_',
            '_item_description.description',
            '              The last name of the depositor to be used in correspondance.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_status.name_depositor' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'Smith' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.name_depositor' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.recvd_author_approval',
        [ [ 'item_',
            '_item_description.description',
            '              This code indicates whether the author\'s approval for \n               an entry has been received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.recvd_author_approval' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar1' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes', 'N', 'No' ] ],
          [ 'item_', '_item_examples.case', 'Y' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.recvd_author_approval' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.author_approval_type',
        [ [ 'item_',
            '_item_description.description',
            '              This code indicates whether the author\'s approval for \n               an entry was received explicitly or implicitly.  The\n               latter is automatically implied by failure to respond\n               to the validation summary within the prescribed period.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.author_approval_type' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'implicit', 'explicit' ] ],
          [ 'item_',
            '_item_examples.case',
            '\nimplicit = automatic approval by failure to acknowledge\nexplicit = approval via depositor acknowledgement' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.author_approval_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_author_approval',
        [ [ 'item_',
            '_item_description.description',
            '              The date the author\'s approval is received.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_author_approval' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-20' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_author_approval' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.recvd_initial_deposition_date',
        [ [ 'item_',
            '_item_description.description',
            '              The date of initial deposition.  (The first message for\n               deposition has been received.)' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.recvd_initial_deposition_date' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-21' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.recvd_initial_deposition_date' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_submitted',
        [ [ 'item_',
            '_item_description.description',
            '              The date of complete deposition.  This corresponds to the date\n               at which the PDB identifier is assigned.' ],
          [ 'item_', '_item.name', '_pdbx_database_status.date_submitted' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-20' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_submitted' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.rcsb_annotator',
        [ [ 'item_',
            '_item_description.description',
            '              The initials of the annotator processing this entry.' ],
          [ 'item_', '_item.name', '_pdbx_database_status.rcsb_annotator' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'BS', 'SJ', 'KB' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.rcsb_annotator' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_of_sf_release',
        [ [ 'item_',
            '_item_description.description',
            '              The date of PDB/RCSB release.  This corresponds to the date\n               at which the entry is placed into the public archive.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_of_sf_release' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ],
          [ 'item_', '_item_examples.case', '1999-02-28' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_of_sf_release' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_of_mr_release',
        [ [ 'item_',
            '_item_description.description',
            '              The date of PDB/RCSB release.  This corresponds to the date\n               at which the entry is placed into the public archive.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_of_mr_release' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ],
          [ 'item_', '_item_examples.case', '1999-02-28' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_of_mr_release' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_of_PDB_release',
        [ [ 'item_',
            '_item_description.description',
            '              PDB release date.  This is the date that appears in the PDB\n               REVDAT record.\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_of_PDB_release' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-28' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_of_PDB_release' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_hold_coordinates',
        [ [ 'item_',
            '_item_description.description',
            '              At an author\'s request, a coordinate entry may be held after\n               processing for some period of time.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_hold_coordinates' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-28' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_hold_coordinates' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_hold_struct_fact',
        [ [ 'item_',
            '_item_description.description',
            '              At an author\'s request, the structure factors may be held after\n               processing for some period of time.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_hold_struct_fact' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-28' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_hold_struct_fact' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.date_hold_nmr_constraints',
        [ [ 'item_',
            '_item_description.description',
            '              At an author\'s request, the NMR constraint data may be held after\n               processing for some period of time.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.date_hold_nmr_constraints' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-28' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.date_hold_nmr_constraints' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.hold_for_publication',
        [ [ 'item_',
            '_item_description.description',
            '              At an author\'s request, an entry is to be held until\n               publication.  ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.hold_for_publication' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar1' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes', 'N', 'No' ] ],
          [ 'item_', '_item_examples.case', 'Y' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.hold_for_publication' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.SG_entry',
        [ [ 'item_',
            '_item_description.description',
            '              This code indicates whether the entry belongs to\n               Structural Genomics Project.' ],
          [ 'item_', '_item.name', '_pdbx_database_status.SG_entry' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uchar1' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes', 'N', 'No' ] ],
          [ 'item_', '_item_examples.case', 'Y' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.SG_entry' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.pdb_date_of_author_approval',
        [ [ 'item_',
            '_item_description.description',
            '              This is the date when PDB received the author\'s approval for an\n               entry which has been processed by NDB.  (This is a place holder \n               for entries processed before Jan. 1, 1996.)\n' ],
          [ 'item_',
            '_item.name',
            '_pdbx_database_status.pdb_date_of_author_approval' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1983-02-27' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.pdb_date_of_author_approval' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.deposit_site',
        [ [ 'item_',
            '_item_description.description',
            '              The site where the file was deposited.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_status.deposit_site' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'NDB', 'RCSB', 'PDB', 'EBI', 'OSAKA', 'BNL' ] ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'NDB', 'RCSB', 'PDB', 'EBI', 'OSAKA', 'BNL' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.deposit_site' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_status.process_site',
        [ [ 'item_',
            '_item_description.description',
            '              The site where the file was deposited.\n' ],
          [ 'item_', '_item.name', '_pdbx_database_status.process_site' ],
          [ 'item_', '_item.category_id', 'pdbx_database_status' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'NDB', 'RCSB', 'EBI', 'OSAKA', 'PDB', 'PRAGUE' ] ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'NDB', 'RCSB', 'EBI', 'OSAKA', 'PDB', 'PRAGUE' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_database_status.process_site' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_entity_name',
        [ [ 'item_',
            '_category.description',
            '        The PDBX_ENTITY_NAME records additional name information for \n         each entity.' ],
          [ 'item_', '_category.id', 'pdbx_entity_name' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_name.entity_id',
              '_pdbx_entity_name.name',
              '_pdbx_entity_name.name_type' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n     loop_\n    _pdbx_entity_name.entity_id\n    _pdbx_entity_name.name\n    _pdbx_entity_name.name_type\n    1   "PLASTOCYANIN"        \'SWS-NAME\'\n    1   "Electron transport" \'SWS-KEYWORD\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_entity_name.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '    Pointer to _entity.id.' ],
          [ 'item_', '_item.name', '_pdbx_entity_name.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_name' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_name.entity_id' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_name.entity_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_entity_name.name',
        [ [ 'item_', '_item_description.description', '    Entity name.' ],
          [ 'item_', '_item.name', '_pdbx_entity_name.name' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_name' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_aliases.alias_name', '_rcsb_entity_name.name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_entity_name.name_type',
        [ [ 'item_',
            '_item_description.description',
            '    Entity name type.' ],
          [ 'item_', '_item.name', '_pdbx_entity_name.name_type' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_name' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_name.name_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'RCSB_NAME',
              'RCSB_SYNONYM',
              'SWS-NAME',
              'SWS-SYNONYM',
              'SWS-KEYWORD',
              'GB-NAME',
              'GB-SYNONYM',
              'GB-KEYWORD' ] ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_prerelease_seq',
        [ [ 'item_',
            '_category.description',
            '              This category provides a placeholder for pre-release \n               sequence information.  After release this category \n               should be discarded.' ],
          [ 'item_', '_category.id', 'pdbx_prerelease_seq' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_prerelease_seq.entity_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n     loop_\n     _pdbx_prerelease_seq.entity_id\n     _pdbx_prerelease_seq.seq_one_letter_code\n     1 \'GKHNGPEHWHKDFPIAKGERQSPVDIDTHTAKYDPSLKPLSVSYDQATSLRILNNGAAFNVEFD\'\n     2 \'HKDFPIAKGERQSPVDIDTHTAKYDPSLKPLSVSYDQATSLRILNN\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_prerelease_seq.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_pdbx_prerelease_seq.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_prerelease_seq' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_prerelease_seq.entity_id' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_prerelease_seq.entity_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_prerelease_seq.seq_one_letter_code',
        [ [ 'item_',
            '_item_description.description',
            '              Chemical sequence expressed as string of one-letter \n               amino acid codes. \n\nA  for alanine or adenine\nB  for ambiguous asparagine/aspartic-acid\nR  for arginine\nN  for asparagine\nD  for aspartic-acid\nC  for cysteine or cystine or cytosine\nQ  for glutamine\nE  for glutamic-acid\nZ  for ambiguous glutamine/glutamic acid\nG  for glycine or guanine\nH  for histidine\nI  for isoleucine\nL  for leucine\nK  for lysine\nM  for methionine\nF  for phenylalanine\nP  for proline\nS  for serine\nT  for threonine or thymine\nW  for tryptophan\nY  for tyrosine\nV  for valine\nU  for uracil' ],
          [ 'item_',
            '_item.name',
            '_pdbx_prerelease_seq.seq_one_letter_code' ],
          [ 'item_', '_item.category_id', 'pdbx_prerelease_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '\nMSHHWGYGKHNGPEHWHKDFPIAKGERQSPVDIDTHTAKYDPSLKPLSVSYDQATSLRILNNGAAFNVEFD' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_prerelease_seq.seq_one_letter_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_poly_seq_scheme',
        [ [ 'item_',
            '_category.description',
            '        The PDBX_POLY_SEQ_SCHEME category provides residue level nomenclature \n         mapping for polymer entities.' ],
          [ 'item_', '_category.id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_poly_seq_scheme.asym_id',
              '_pdbx_poly_seq_scheme.entity_id',
              '_pdbx_poly_seq_scheme.seq_id',
              '_pdbx_poly_seq_scheme.mon_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - based on NDB entry DDFB25',
              '\n     loop_\n    _pdbx_poly_seq_scheme.asym_id \n    _pdbx_poly_seq_scheme.entity_id\n    _pdbx_poly_seq_scheme.seq_id\n    _pdbx_poly_seq_scheme.mon_id\n    _pdbx_poly_seq_scheme.ndb_num\n    _pdbx_poly_seq_scheme.pdb_num\n    _pdbx_poly_seq_scheme.auth_num\n    _pdbx_poly_seq_scheme.pdb_mon_id\n    _pdbx_poly_seq_scheme.auth_mon_id\n    _pdbx_poly_seq_scheme.pdb_strand_id\n    _pdbx_poly_seq_scheme.pdb_ins_code\nA  1  1  C   1  1   1  C   C    A  .\nA  1  2  G   2  2   2  G   G    A  .\nA  1  3  T   3  3   3  T   T    A  .\nA  1  4  +A  4  4   4  +A  +A   A  .\nA  1  5  C   5  5   5  C   C    A  . \nA  1  6  G   6  6   6  G   G    A  .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.asym_id',
        [ [ 'item_',
            '_item_description.description',
            '    Pointer to _atom_site.label_asym_id.' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_poly_seq_scheme.asym_id' ],
          [ 'item_', '_item_linked.parent_name', '_struct_asym.id' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', 'A', '2B3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '    Pointer to _entity.id.' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_poly_seq_scheme.entity_id' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.entity_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.seq_id',
        [ [ 'item_',
            '_item_description.description',
            '    Pointer to _entity_poly_seq.num' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_poly_seq_scheme.seq_id' ],
          [ 'item_', '_item_linked.child_name', '_entity_poly_seq.num' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.seq_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.mon_id',
        [ [ 'item_',
            '_item_description.description',
            '     Pointer to _entity_poly_seq.mon_id.' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.mon_id' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_poly_seq_scheme.mon_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_entity_poly_seq.mon_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.mon_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.pdb_strand_id',
        [ [ 'item_',
            '_item_description.description',
            '         PDB strand/chain id.  ' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.pdb_strand_id' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.pdb_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.ndb_seq_num',
        [ [ 'item_',
            '_item_description.description',
            '         NDB residue number.' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.ndb_seq_num' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.ndb_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.pdb_seq_num',
        [ [ 'item_',
            '_item_description.description',
            '         PDB residue number.' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.pdb_seq_num' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.pdb_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.auth_seq_num',
        [ [ 'item_',
            '_item_description.description',
            '      _atom_site.auth_seq_id' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.auth_seq_num' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.auth_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.pdb_mon_id',
        [ [ 'item_',
            '_item_description.description',
            '      PDB residue name.' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.pdb_mon_id' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.pdb_mon_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.auth_mon_id',
        [ [ 'item_',
            '_item_description.description',
            '           _atom_site.auth_comp_id' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.auth_mon_id' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.auth_mon_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_poly_seq_scheme.pdb_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '      PDB insertion code.' ],
          [ 'item_', '_item.name', '_pdbx_poly_seq_scheme.pdb_ins_code' ],
          [ 'item_', '_item.category_id', 'pdbx_poly_seq_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_poly_seq_scheme.pdb_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        'pdbx_nonpoly_scheme',
        [ [ 'item_',
            '_category.description',
            '        The PDBX_NONPOLY_SCHEME category provides residue level nomenclature \n         mapping for non-polymer entities.' ],
          [ 'item_', '_category.id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_nonpoly_scheme.asym_id',
              '_pdbx_nonpoly_scheme.ndb_seq_num' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n     loop_\n    _pdbx_nonpoly_scheme.asym_id \n    _pdbx_nonpoly_scheme.entity_id\n    _pdbx_nonpoly_scheme.mon_id\n    _pdbx_nonpoly_scheme.ndb_seq_num\n    _pdbx_nonpoly_scheme.pdb_seq_num\n    _pdbx_nonpoly_scheme.auth_seq_num\n    _pdbx_nonpoly_scheme.pdb_mon_id\n    _pdbx_nonpoly_scheme.auth_mon_id\n    _pdbx_nonpoly_scheme.pdb_strand_id\n    _pdbx_nonpoly_scheme.pdb_ins_code\n C  3    HOH 100 100 100 HOH HOH C .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nonpoly_scheme.asym_id',
        [ [ 'item_',
            '_item_description.description',
            '    Pointer to _atom_site.label_asym_id.' ],
          [ 'item_', '_item.name', '_pdbx_nonpoly_scheme.asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_nonpoly_scheme.asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', 'A', '2B3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_nonpoly_scheme.asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nonpoly_scheme.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '    Pointer to _atom_site.label_entity_id.' ],
          [ 'item_', '_item.name', '_pdbx_nonpoly_scheme.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_nonpoly_scheme.entity_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_entity_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_nonpoly_scheme.entity_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nonpoly_scheme.mon_id',
        [ [ 'item_',
            '_item_description.description',
            '     Pointer to _atom_site.label_comp_id.' ],
          [ 'item_', '_item.name', '_pdbx_nonpoly_scheme.mon_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_nonpoly_scheme.mon_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_nonpoly_scheme.mon_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nonpoly_scheme.pdb_strand_id',
        [ [ 'item_',
            '_item_description.description',
            '         PDB strand/chain id.  ' ],
          [ 'item_', '_item.name', '_pdbx_nonpoly_scheme.pdb_strand_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_nonpoly_scheme.pdb_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nonpoly_scheme.ndb_seq_num',
        [ [ 'item_',
            '_item_description.description',
            '         NDB/RCSB residue number.' ],
          [ 'item_', '_item.name', '_pdbx_nonpoly_scheme.ndb_seq_num' ],
          [ 'item_', '_item.category_id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_nonpoly_scheme.ndb_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nonpoly_scheme.pdb_seq_num',
        [ [ 'item_',
            '_item_description.description',
            '         PDB residue number.' ],
          [ 'item_', '_item.name', '_pdbx_nonpoly_scheme.pdb_seq_num' ],
          [ 'item_', '_item.category_id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_nonpoly_scheme.pdb_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nonpoly_scheme.auth_seq_num',
        [ [ 'item_',
            '_item_description.description',
            '      _atom_site.auth_seq_id' ],
          [ 'item_', '_item.name', '_pdbx_nonpoly_scheme.auth_seq_num' ],
          [ 'item_', '_item.category_id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_nonpoly_scheme.auth_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nonpoly_scheme.pdb_mon_id',
        [ [ 'item_',
            '_item_description.description',
            '      PDB residue name.' ],
          [ 'item_', '_item.name', '_pdbx_nonpoly_scheme.pdb_mon_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_nonpoly_scheme.pdb_mon_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nonpoly_scheme.auth_mon_id',
        [ [ 'item_',
            '_item_description.description',
            '           _atom_site.auth_comp_id' ],
          [ 'item_', '_item.name', '_pdbx_nonpoly_scheme.auth_mon_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_nonpoly_scheme.auth_mon_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nonpoly_scheme.pdb_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '      PDB insertion code.' ],
          [ 'item_', '_item.name', '_pdbx_nonpoly_scheme.pdb_ins_code' ],
          [ 'item_', '_item.category_id', 'pdbx_nonpoly_scheme' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_nonpoly_scheme.pdb_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_refine',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_REFINE category record details about \n               additional structure refinement parameters which are needed \n               to complete legacy REMARK 3 refinement templates in PDB \n               format files.' ],
          [ 'item_', '_category.id', 'pdbx_refine' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_refine.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - PDB placeholders for refinement program SHELX',
              '\n    _pdbx_refine.entry_id                                 ABC001\n    _pdbx_refine.R_factor_all_4sig_cutoff                  0.174\n    _pdbx_refine.R_factor_obs_4sig_cutoff                  0.169\n    _pdbx_refine.number_reflns_obs_4sig_cutoff              1263\n    _pdbx_refine.free_R_factor_4sig_cutoff                 0.216\n    _pdbx_refine.free_R_val_test_set_ct_4sig_cutoff          164\n    _pdbx_refine.free_R_val_test_set_size_perc_4sig_cutoff  1.29\n',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_refine.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_pdbx_refine.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.child_name', '_pdbx_refine.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_', '_item_aliases.alias_name', '_ndb_refine.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.R_factor_all_no_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '              R-value (all reflections, no cutoff)  \n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_', '_item.name', '_pdbx_refine.R_factor_all_no_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.R_factor_all_no_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.R_factor_obs_no_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '              R-value (working set reflections, no cutoff)\n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_', '_item.name', '_pdbx_refine.R_factor_obs_no_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.R_factor_obs_no_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.free_R_factor_4sig_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '               R free value (4 sigma cutoff). \n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine.free_R_factor_4sig_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '0.216' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.free_R_factor_4sig_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.free_R_factor_no_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '                  Free R-value (no cutoff)\n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_', '_item.name', '_pdbx_refine.free_R_factor_no_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.free_R_factor_no_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.free_R_val_test_set_size_perc_no_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '              Free R-value test set size (in percent, no cutoff)\n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine.free_R_val_test_set_size_perc_no_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.free_R_val_test_set_size_perc_no_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.free_R_val_test_set_ct_no_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '              Free R-value test set count (no cutoff)\n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine.free_R_val_test_set_ct_no_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.free_R_val_test_set_ct_no_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.number_reflns_obs_no_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '              Total number of reflections (no cutoff).\n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine.number_reflns_obs_no_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.number_reflns_obs_no_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.R_factor_all_4sig_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '              R-value (all reflections, 4 sigma cutoff)\n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine.R_factor_all_4sig_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '0.174' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.R_factor_all_4sig_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.R_factor_obs_4sig_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '              R-value (working set, 4 sigma cutoff)\n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine.R_factor_obs_4sig_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '0.169' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.R_factor_obs_4sig_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.free_R_val_4sig_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '              Free R-value (4 sigma cutoff)\n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_', '_item.name', '_pdbx_refine.free_R_val_4sig_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.free_R_val_4sig_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.free_R_val_test_set_size_perc_4sig_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '       Free R-value test set size (in percent, 4 sigma cutoff)\n       Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine.free_R_val_test_set_size_perc_4sig_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '1.29' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.free_R_val_test_set_size_perc_4sig_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.free_R_val_test_set_ct_4sig_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '       Free R-value test set count (4 sigma cutoff)\n    Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine.free_R_val_test_set_ct_4sig_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '164' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.free_R_val_test_set_ct_4sig_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine.number_reflns_obs_4sig_cutoff',
        [ [ 'item_',
            '_item_description.description',
            '          Total number of reflections (4 sigma cutoff).\n    Placeholder for PDB mapping of SHELXL refinement data.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine.number_reflns_obs_4sig_cutoff' ],
          [ 'item_', '_item.category_id', 'pdbx_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '1263' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_refine.number_reflns_obs_4sig_cutoff' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '############################' ],
      [ '@comment', '# PDBX_STRUCT_SHEET_HBOND ##' ],
      [ '@comment', '############################' ],
      [ 'save_',
        'pdbx_struct_sheet_hbond',
        [ [ 'item_',
            '_category.description',
            '          Data items in the PDBX_STRUCT_SHEET_HBOND category record details\n           about the hydrogen bonding between residue ranges in a beta sheet.\n           This category is provided for cases where only a single hydrogen\n           bond is used to register the two residue ranges.   Category\n           STRUCT_SHEET_HBOND should be used when the initial and terminal\n           hydrogen bonds for strand pair are known. \n' ],
          [ 'item_', '_category.id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_struct_sheet_hbond.sheet_id',
              '_pdbx_struct_sheet_hbond.range_id_1',
              '_pdbx_struct_sheet_hbond.range_id_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group', 'pdbx_group' ] ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet_range.id in\n               the STRUCT_SHEET_RANGE category.' ],
          [ 'item_', '_item.name', '_pdbx_struct_sheet_hbond.range_id_1' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_id_1' ],
          [ 'item_', '_item_linked.parent_name', '_struct_sheet_range.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_id_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet_range.id in\n               the STRUCT_SHEET_RANGE category.' ],
          [ 'item_', '_item.name', '_pdbx_struct_sheet_hbond.range_id_2' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_id_2' ],
          [ 'item_', '_item_linked.parent_name', '_struct_sheet_range.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_id_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.sheet_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _struct_sheet.id in the\n               STRUCT_SHEET category.' ],
          [ 'item_', '_item.name', '_pdbx_struct_sheet_hbond.sheet_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.sheet_id' ],
          [ 'item_', '_item_linked.parent_name', '_struct_sheet.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.sheet_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_1_label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the residue identifier for the first partner of the \n            registration hydrogen bond between two residue ranges in a sheet.\n\n               This data item is a pointer to _atom_site.label_atom_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_1_label_atom_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'atcode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_1_label_atom_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_atom_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_1_label_atom_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_1_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '          A component of the residue identifier for the first partner of the \n           registration hydrogen bond between two residue ranges in a sheet.\n\n           This data item is a pointer to _atom_site.label_seq_id in the\n           ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_1_label_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_1_label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_1_label_seq_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_1_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the residue identifier for the first partner of the \n            registration hydrogen bond between two residue ranges in a sheet.\n\n            This data item is a pointer to _atom_site.label_comp_id in the\n            ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_1_label_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_1_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_1_label_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_1_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the residue identifier for the first partner of the \n            registration hydrogen bond between two residue ranges in a sheet.\n\n            This data item is a pointer to _atom_site.label_asym_id in the\n            ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_1_label_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_1_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_1_label_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_1_auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '          A component of the residue identifier for the first partner of the \n           registration hydrogen bond between two residue ranges in a sheet.\n\n           This data item is a pointer to _atom_site.auth_atom_id in the\n           ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_1_auth_atom_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'atcode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_1_auth_atom_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_atom_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_1_auth_atom_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_1_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '          A component of the residue identifier for the first partner of the \n           registration hydrogen bond between two residue ranges in a sheet.\n\n           This data item is a pointer to _atom_site.auth_seq_id in the\n           ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_1_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_1_auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_1_auth_seq_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_1_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the residue identifier for the first partner of the \n            registration hydrogen bond between two residue ranges in a sheet.\n\n            This data item is a pointer to _atom_site.auth_comp_id in the\n            ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_1_auth_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_1_auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_1_auth_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_1_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '          A component of the residue identifier for the first partner of the \n           registration hydrogen bond between two residue ranges in a sheet.\n\n           This data item is a pointer to _atom_site.auth_asym_id in the\n           ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_1_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_1_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_1_auth_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_1_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the residue identifier for the first partner of the \n            registration hydrogen bond between two residue ranges in a sheet.\n\n            This data item is a pointer to _atom_site.label_asym_id in the\n            ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_1_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_1_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_1_PDB_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_2_label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '         A component of the residue identifier for the second partner of the \n          registration hydrogen bond between two residue ranges in a sheet.\n\n          This data item is a pointer to _atom_site.label_atom_id in the\n          ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_2_label_atom_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'atcode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_2_label_atom_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_atom_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_2_label_atom_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_2_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the residue identifier for the second partner of the \n            registration hydrogen bond between two residue ranges in a sheet.\n\n            This data item is a pointer to _atom_site.label_seq_id in the\n            ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_2_label_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_2_label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_2_label_seq_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_2_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the residue identifier for the second partner of the \n            registration hydrogen bond between two residue ranges in a sheet.\n\n            This data item is a pointer to _atom_site.label_comp_id in the\n            ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_2_label_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_2_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_2_label_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_2_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the residue identifier for the second partner of the \n            registration hydrogen bond between two residue ranges in a sheet.\n\n            This data item is a pointer to _atom_site.label_asym_id in the\n            ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_2_label_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_2_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_2_label_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_2_auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the residue identifier for the second partner of the \n            registration hydrogen bond between two residue ranges in a sheet.\n\n            This data item is a pointer to _atom_site.auth_atom_id in the\n            ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_2_auth_atom_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'atcode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_2_auth_atom_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_atom_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_2_auth_atom_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_2_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '          A component of the residue identifier for the second partner of the \n           registration hydrogen bond between two residue ranges in a sheet.\n\n           This data item is a pointer to _atom_site.auth_seq_id in the\n           ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_2_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_2_auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_2_auth_seq_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_2_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '          A component of the residue identifier for the second partner of the \n           registration hydrogen bond between two residue ranges in a sheet.\n\n           This data item is a pointer to _atom_site.auth_comp_id in the\n           ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_2_auth_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_2_auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_2_auth_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_2_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '          A component of the residue identifier for the second partner of the \n           registration hydrogen bond between two residue ranges in a sheet.\n\n           This data item is a pointer to _atom_site.auth_asym_id in the\n           ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_2_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_2_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_2_auth_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_struct_sheet_hbond.range_2_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the residue identifier for the second partner of the \n            registration hydrogen bond between two residue ranges in a sheet.\n\n            This data item is a pointer to _atom_site.label_asym_id in the\n            ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_struct_sheet_hbond.range_2_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'pdbx_struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_struct_sheet_hbond.range_2_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_struct_sheet_hbond.range_2_PDB_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_xplor_file',
        [ [ 'item_',
            '_category.description',
            '         \n           Parameter and topology files used in X-PLOR/CNS refinement.' ],
          [ 'item_', '_category.id', 'pdbx_xplor_file' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_xplor_file.serial_no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n     _pdbx_xplor_file.serial_no                         1\n     _pdbx_xplor_file.param_file                        parm_hol.dat\n     _pdbx_xplor_file.topol_file                        topol_hol.dat',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_xplor_file.serial_no',
        [ [ 'item_',
            '_item_description.description',
            '                 Serial number.' ],
          [ 'item_', '_item.name', '_pdbx_xplor_file.serial_no' ],
          [ 'item_', '_item.category_id', 'pdbx_xplor_file' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_xplor_file.serial_no' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_xplor_file.param_file',
        [ [ 'item_',
            '_item_description.description',
            '                 Parameter file name in X-PLOR/CNS refinement.' ],
          [ 'item_', '_item.name', '_pdbx_xplor_file.param_file' ],
          [ 'item_', '_item.category_id', 'pdbx_xplor_file' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'PARAM_NDBX_HIGH.DNA' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_xplor_file.param_file' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_xplor_file.topol_file',
        [ [ 'item_',
            '_item_description.description',
            '                 Topology file name in X-PLOR/CNS refinement.' ],
          [ 'item_', '_item.name', '_pdbx_xplor_file.topol_file' ],
          [ 'item_', '_item.category_id', 'pdbx_xplor_file' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'TOP_NDBX.DNA' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_ndb_xplor_file.topol_file' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        'pdbx_refine_aux_file',
        [ [ 'item_',
            '_category.description',
            '         \n        Auxilary parameter and topology files used in refinement.' ],
          [ 'item_', '_category.id', 'pdbx_refine_aux_file' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_refine_aux_file.serial_no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n     _pdbx_refine_aux_file.serial_no                         1\n     _pdbx_refine_aux_file.file_name                        parm_hol.dat\n     _pdbx_refine_aux_file.file_type                        PARAMETER',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_refine_aux_file.serial_no',
        [ [ 'item_',
            '_item_description.description',
            '                 Serial number.' ],
          [ 'item_', '_item.name', '_pdbx_refine_aux_file.serial_no' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_aux_file' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_refine_aux_file.serial_no' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine_aux_file.file_name',
        [ [ 'item_',
            '_item_description.description',
            '                 Auxilary file name.' ],
          [ 'item_', '_item.name', '_pdbx_refine_aux_file.file_name' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_aux_file' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'PARAM_NDBX_HIGH.DNA' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_refine_aux_file.file_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_refine_aux_file.file_type',
        [ [ 'item_',
            '_item_description.description',
            '                 Auxilary file type.' ],
          [ 'item_', '_item.name', '_pdbx_refine_aux_file.file_type' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_aux_file' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'PARAMETER' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'PARAMETER', 'TOPOLOGY' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_refine_aux_file.file_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        'pdbx_database_related',
        [ [ 'item_',
            '_category.description',
            '    Data items in PDBX_DATABASE_RELATED contain references to entries\n     that are related to the this entry.' ],
          [ 'item_', '_category.id', 'pdbx_database_related' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_database_related.db_name',
              '_pdbx_database_related.db_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'item_',
            '_category_examples.case',
            '\n   _pdbx_database_related.db_name  PDB\n   _pdbx_database_related.db_id    1ABC' ] ] ],
      [ 'save_',
        '_pdbx_database_related.db_name',
        [ [ 'item_',
            '_item_description.description',
            '    The name of the database containing the related entry.' ],
          [ 'item_', '_item.name', '_pdbx_database_related.db_name' ],
          [ 'item_', '_item.category_id', 'pdbx_database_related' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ '@comment', '     loop_' ],
          [ '@comment',
            '    _item_enumeration.value  \'PDB\' \'NDB\' \'BMRB\' \'BMCD\' ' ],
          [ 'item_',
            '_item_examples.case',
            '\nPDB  - Protein Databank \nNDB  - Nucleic Acid Database \nBMRB - BioMagResBank\nBMCD - Biological Macromolecule Crystallization Database' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_database_related.db_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_related.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the related entry.' ],
          [ 'item_', '_item.name', '_pdbx_database_related.details' ],
          [ 'item_', '_item.category_id', 'pdbx_database_related' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '\n1ABC contains the same protein complexed with Netropsin.  ' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_database_related.details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_related.db_id',
        [ [ 'item_',
            '_item_description.description',
            '              The identifying code in the related database.' ],
          [ 'item_', '_item.name', '_pdbx_database_related.db_id' ],
          [ 'item_', '_item.category_id', 'pdbx_database_related' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1ABC', 'BDL001' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_database_related.db_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_database_related.content_type',
        [ [ 'item_',
            '_item_description.description',
            '              The identifying content type of the related entry.' ],
          [ 'item_', '_item.name', '_pdbx_database_related.content_type' ],
          [ 'item_', '_item.category_id', 'pdbx_database_related' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'minimized average structure',
              'representative structure',
              'ensemble',
              'derivative structure',
              'native structure',
              'other' ] ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'minimized average structure',
              'representative structure',
              'ensemble',
              'derivative structure',
              'native structure',
              'other' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_database_related.content_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# PDBX_ENTITY_ASSEMBLY ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'pdbx_entity_assembly',
        [ [ 'item_',
            '_category.description',
            '              The PDBX_ENTITY_ASSEMBLY category provides a chemical description \n               of the biological assembly studied in terms of its constituent\n               entities.' ],
          [ 'item_', '_category.id', 'pdbx_entity_assembly' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_assembly.id',
              '_pdbx_entity_assembly.entity_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '  A complex composed of one copy of entities 1 and 2.',
              '\n    loop_\n    _pdbx_entity_assembly.id\n    _pdbx_entity_assembly.biol_id\n    _pdbx_entity_assembly.entity_id\n    _pdbx_entity_assembly.num_copies\n     1  1  1   1\n     1  1  2   1',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_entity_assembly.id',
        [ [ 'item_',
            '_item_description.description',
            '              An  identifier for the assembly.  ' ],
          [ 'item_', '_item.name', '_pdbx_entity_assembly.id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_assembly.id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_entity_assembly.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              An enity identifier.  A reference to _entity.id.' ],
          [ 'item_', '_item.name', '_pdbx_entity_assembly.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_assembly.entity_id' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_assembly.entity_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_entity_assembly.biol_id',
        [ [ 'item_',
            '_item_description.description',
            '              An  identifier for the assembly.  ' ],
          [ 'item_', '_item.name', '_pdbx_entity_assembly.biol_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_assembly.biol_id' ],
          [ 'item_', '_item_linked.parent_name', '_struct_biol.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_assembly.biol_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_entity_assembly.num_copies',
        [ [ 'item_',
            '_item_description.description',
            '              The number of copies of this entity in the assembly.' ],
          [ 'item_', '_item.name', '_pdbx_entity_assembly.num_copies' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_assembly.num_copies' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# PDBX_ENTITY_SRC_SYN ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'pdbx_entity_src_syn',
        [ [ 'item_',
            '_category.description',
            '            PDBX_ENTITY_SRC_SYN records the details about each chemically\n             synthesized molecule (entity) in the asymmetric unit.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_syn' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_entity_src_syn.entity_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group', 'pdbx_group' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_syn.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the source for the \n               synthetic entity.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_syn.details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_syn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'This sequence occurs naturally in humans.' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_src_syn.details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_syn.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_syn.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_syn' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_syn.entity_id' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3', '4' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_src_syn.entity_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment',
        '## ---------------------------------------------------------------------------' ],
      [ '@comment',
        '##               Supplemental items in existing mmCIF categories ...' ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        '_atom_site.pdbx_auth_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '    Author\'s alternate location identifier.' ],
          [ 'item_', '_item.name', '_atom_site.pdbx_auth_alt_id' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site.ndb_auth_alt_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site.pdbx_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '    PDB insertion code.' ],
          [ 'item_', '_item.name', '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site.ndb_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site.pdbx_PDB_model_num',
        [ [ 'item_',
            '_item_description.description',
            '   PDB model number.' ],
          [ 'item_', '_item.name', '_atom_site.pdbx_PDB_model_num' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_aliases.alias_name', '_atom_site.ndb_model' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site.pdbx_PDB_residue_no',
        [ [ 'item_',
            '_item_description.description',
            '              PDB residue number.' ],
          [ 'item_', '_item.name', '_atom_site.pdbx_PDB_residue_no' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site.ndb_PDB_residue_no' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site.pdbx_PDB_residue_name',
        [ [ 'item_',
            '_item_description.description',
            '              PDB residue name.' ],
          [ 'item_', '_item.name', '_atom_site.pdbx_PDB_residue_name' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site.ndb_PDB_residue_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site.pdbx_PDB_strand_id',
        [ [ 'item_',
            '_item_description.description',
            '              PDB strand id.' ],
          [ 'item_', '_item.name', '_atom_site.pdbx_PDB_strand_id' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site.ndb_PDB_strand_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site.pdbx_PDB_atom_name',
        [ [ 'item_',
            '_item_description.description',
            '              PDB atom name.' ],
          [ 'item_', '_item.name', '_atom_site.pdbx_PDB_atom_name' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site.ndb_PDB_atom_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site.pdbx_auth_atom_name',
        [ [ 'item_',
            '_item_description.description',
            '              Author\'s atom name.' ],
          [ 'item_', '_item.name', '_atom_site.pdbx_auth_atom_name' ],
          [ 'item_', '_item.category_id', 'atom_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'atcode' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site.ndb_auth_atom_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              Pointer to _atom_site.auth_seq_id' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_anisotrop.ndb_PDB_residue_no' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_auth_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '      Pointer to _atom_site.pdbx_auth_alt_id.' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_auth_alt_id' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_auth_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_auth_alt_id' ],
          [ '@comment',
            '    _item_aliases.alias_name     \'_atom_site_anisotrop.ndb_auth_alt_id\'' ],
          [ '@comment',
            '    _item_aliases.dictionary     \'cif_rcsb.dic\'' ],
          [ '@comment', '    _item_aliases.version        1.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '      Pointer to _atom_site.auth_asym_id' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_anisotrop.ndb_PDB_strand_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '       Pointer to _atom_site.auth_atom_id' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_auth_atom_id' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'atcode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_auth_atom_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_atom_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_anisotrop.ndb_PDB_atom_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '     Pointer to _atom_site.auth_comp_id' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_auth_comp_id' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_anisotrop.ndb_PDB_residue_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              Pointer to _atom_site.label_seq_id' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_label_seq_id' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_anisotrop.ndb_label_seq_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '      Pointer to _atom_site.label_alt_id.' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_label_alt_id' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_label_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_alt_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_anisotrop.ndb_label_alt_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '      Pointer to _atom_site.label_asym_id' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_label_asym_id' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_anisotrop.ndb_label_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '       Pointer to _atom_site.label_atom_id' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_label_atom_id' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'atcode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_label_atom_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_atom_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_anisotrop.ndb_label_atom_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '     Pointer to _atom_site.label_comp_id' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_label_comp_id' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_anisotrop.ndb_label_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_atom_site_anisotrop.pdbx_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '            Pointer to _atom_site.pdbx_PDB_ins_code         ' ],
          [ 'item_',
            '_item.name',
            '_atom_site_anisotrop.pdbx_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'atom_site_anisotrop' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_atom_site_anisotrop.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_atom_site_anisotrop.ndb_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_cell.pdbx_unique_axis',
        [ [ 'item_',
            '_item_description.description',
            '              To further identify unique axis if necessary.  E.g., P 21 with\n               an unique C axis will have \'C\' in this field.' ],
          [ 'item_', '_item.name', '_cell.pdbx_unique_axis' ],
          [ 'item_', '_item.category_id', 'cell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_aliases.alias_name', '_cell.ndb_unique_axis' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_chem_comp.pdbx_synonyms',
        [ [ 'item_',
            '_item_description.description',
            '              Synonym list for the component.' ],
          [ 'item_', '_item.name', '_chem_comp.pdbx_synonyms' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'ATP' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chem_comp.ndb_synonyms' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_chem_comp.pdbx_modification_details',
        [ [ 'item_',
            '_item_description.description',
            '             For nonstandard components a text description \n              of modification of the parent component.' ],
          [ 'item_', '_item.name', '_chem_comp.pdbx_modification_details' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'ATP' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chem_comp.rcsb_modification_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_chem_comp.pdbx_component_no',
        [ [ 'item_',
            '_item_description.description',
            '              A serial number used by PDB in the FORMUL record.' ],
          [ 'item_', '_item.name', '_chem_comp.pdbx_component_no' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '3' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_chem_comp.ndb_component_no' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_chem_comp.pdbx_type',
        [ [ 'item_',
            '_item_description.description',
            '              A preliminary classification used by PDB.' ],
          [ 'item_', '_item.name', '_chem_comp.pdbx_type' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uline' ] ] ],
      [ 'save_',
        '_chem_comp.pdbx_formal_charge',
        [ [ 'item_',
            '_item_description.description',
            '              The net integer charge assigned to this component. This is the\n               formal charge assignment normally found in chemical diagrams.' ],
          [ 'item_', '_item.name', '_chem_comp.pdbx_formal_charge' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '0' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_chem_comp_atom.pdbx_align',
        [ [ 'item_',
            '_item_description.description',
            '              Atom name alignment offset in PDB atom field.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.pdbx_align' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        '_citation.pdbx_database_id_DOI',
        [ [ 'item_',
            '_item_description.description',
            '              Document Object Identifier used by doi.org to uniquely\n               specify bibliographic entry.' ],
          [ 'item_', '_item.name', '_citation.pdbx_database_id_DOI' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_examples.case',
            'DOI:10.2345/S1384107697000225' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation.rcsb_database_id_DOI' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_citation.pdbx_database_id_PubMed',
        [ [ 'item_',
            '_item_description.description',
            '              Ascession number used by PubMed to categorize a specific\n               bibliographic entry.' ],
          [ 'item_', '_item.name', '_citation.pdbx_database_id_PubMed' ],
          [ 'item_', '_item.category_id', 'citation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '12627512' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_citation.rcsb_database_id_PubMed' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_computing.pdbx_structure_refinement_method',
        [ [ 'item_',
            '_item_description.description',
            '              Program/package name for structure refinement method.\n' ],
          [ 'item_',
            '_item.name',
            '_computing.pdbx_structure_refinement_method' ],
          [ 'item_', '_item.category_id', 'computing' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_computing.ndb_structure_refinement_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_computing.pdbx_data_reduction_ii',
        [ [ 'item_',
            '_item_description.description',
            '       Program/package name for data reduction/intensity integration software' ],
          [ 'item_', '_item.name', '_computing.pdbx_data_reduction_ii' ],
          [ 'item_', '_item.category_id', 'computing' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_computing.rcsb_data_reduction_ii' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_computing.pdbx_data_reduction_ds',
        [ [ 'item_',
            '_item_description.description',
            '              Program/package name for data reduction/data scaling' ],
          [ 'item_', '_item.name', '_computing.pdbx_data_reduction_ds' ],
          [ 'item_', '_item.category_id', 'computing' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_computing.rcsb_data_reduction_ds' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_diffrn_detector.pdbx_frames_total',
        [ [ 'item_',
            '_item_description.description',
            '           The total number of data frames collected for this \n            data set.' ],
          [ 'item_', '_item.name', '_diffrn_detector.pdbx_frames_total' ],
          [ 'item_', '_item.category_id', 'diffrn_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '20', '100' ] ] ] ],
      [ 'save_',
        '_diffrn_detector.pdbx_collection_time_total',
        [ [ 'item_',
            '_item_description.description',
            '           The total number of seconds required to measure this\n            data set.' ],
          [ 'item_',
            '_item.name',
            '_diffrn_detector.pdbx_collection_time_total' ],
          [ 'item_', '_item.category_id', 'diffrn_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'seconds' ],
          [ 'loop_', [ '_item_examples.case' ], [ '120.0' ] ] ] ],
      [ 'save_',
        '_diffrn_detector.pdbx_collection_date',
        [ [ 'item_',
            '_item_description.description',
            '              The date of data collection.' ],
          [ 'item_',
            '_item.name',
            '_diffrn_detector.pdbx_collection_date' ],
          [ 'item_', '_item.category_id', 'diffrn_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '1996-12-25' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_detector.ndb_collection_date' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_diffrn_radiation.pdbx_monochromatic_or_laue_m_l',
        [ [ 'item_',
            '_item_description.description',
            '           Monochromatic or Laue.' ],
          [ 'item_',
            '_item.name',
            '_diffrn_radiation.pdbx_monochromatic_or_laue_m_l' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_default.value', 'M' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'M', '.', 'L', '.' ] ],
          [ 'loop_', [ '_item_examples.case' ], [ 'M', 'L' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation.ndb_monochromatic_or_laue_m_l' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_diffrn_radiation.pdbx_wavelength_list',
        [ [ 'item_',
            '_item_description.description',
            '           Comma separated list of wavelengths or wavelength range.' ],
          [ 'item_',
            '_item.name',
            '_diffrn_radiation.pdbx_wavelength_list' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_source.rcsb_wavelength_list' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_diffrn_radiation.pdbx_wavelength',
        [ [ 'item_',
            '_item_description.description',
            '           Wavelength of radiation.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.pdbx_wavelength' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_source.rcsb_wavelength' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_diffrn_radiation.pdbx_diffrn_protocol',
        [ [ 'item_',
            '_item_description.description',
            '           SINGLE WAVELENGTH, LAUE, or MAD.' ],
          [ 'item_',
            '_item.name',
            '_diffrn_radiation.pdbx_diffrn_protocol' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_default.value', 'SINGLE WAVELENGTH' ],
          [ '@comment', '     loop_' ],
          [ '@comment', '    _item_enumeration.value' ],
          [ '@comment', '\t\'SINGLE WAVELENGTH\'' ],
          [ '@comment', '        \'MONOCHROMATIC\'' ],
          [ '@comment', '\t\'LAUE\'' ],
          [ '@comment', '        \'MAD\'' ],
          [ '@comment', '        \'OTHER\'' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'SINGLE WAVELENGTH', 'MONOCHROMATIC', 'LAUE', 'MAD', 'OTHER' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_radiation.rcsb_diffrn_protocol' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_diffrn_source.pdbx_synchrotron_beamline',
        [ [ 'item_',
            '_item_description.description',
            '             Synchrotron beamline.' ],
          [ 'item_',
            '_item.name',
            '_diffrn_source.pdbx_synchrotron_beamline' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_source.ndb_synchrotron_beamline' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_diffrn_source.pdbx_synchrotron_site',
        [ [ 'item_',
            '_item_description.description',
            '     Synchrotron site.' ],
          [ 'item_', '_item.name', '_diffrn_source.pdbx_synchrotron_site' ],
          [ 'item_', '_item.category_id', 'diffrn_source' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_diffrn_source.ndb_synchrotron_site' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_entity.pdbx_description',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the entity, with the name of the entity\n               in parenthesis.\n\n               Maps to PDB compound name.' ],
          [ 'item_', '_item.name', '_entity.pdbx_description' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'DNA (5\'-D(*GP*(CH3)CP*GP*(CH3)CP*GP*C)-3\')',
              'PROFLAVINE',
              'PROTEIN (DEOXYRIBONUCLEASE I (E.C.3.1.21.1))' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity.ndb_description' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity.pdbx_number_of_molecules',
        [ [ 'item_',
            '_item_description.description',
            '              A place holder for the number of molecules of the entity in\n               the entry.' ],
          [ 'item_', '_item.name', '_entity.pdbx_number_of_molecules' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1.0', '2.0', '3.0' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity.ndb_number_of_molecules' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity.pdbx_parent_entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              An identifier for the parent entity if this entity\n               is part of a complex entity.  For instance a chimeric\n               entity may be decomposed into several independent \n               chemical entities where each component entity was\n               obtained from a different source.' ],
          [ 'item_', '_item.name', '_entity.pdbx_parent_entity_id' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_entity.pdbx_parent_entity_id' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity.rcsb_parent_entity_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity.pdbx_mutation',
        [ [ 'item_',
            '_item_description.description',
            '              Details about any entity mutation(s).' ],
          [ 'item_', '_item.name', '_entity.pdbx_mutation' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Y31H', 'DEL(298-323)' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_keywords.ndb_mutation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity.pdbx_fragment',
        [ [ 'item_',
            '_item_description.description',
            '              Entity fragment description(s).' ],
          [ 'item_', '_item.name', '_entity.pdbx_fragment' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'KLENOW FRAGMENT',
              'REPLICASE OPERATOR HAIRPIN',
              'C-TERMINAL DOMAIN' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_keywords.ndb_fragment' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity.pdbx_ec',
        [ [ 'item_',
            '_item_description.description',
            '             Enzyme Commission (EC) number(s)' ],
          [ 'item_', '_item.name', '_entity.pdbx_ec' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', '2.7.7.7' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_keywords.ndb_ec' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity.pdbx_modification',
        [ [ 'item_',
            '_item_description.description',
            '             Description(s) of any chemical or post-translational modifications' ],
          [ 'item_', '_item.name', '_entity.pdbx_modification' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_keywords.rcsb_modification' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity.pdbx_formula_weight_exptl',
        [ [ 'item_',
            '_item_description.description',
            '              Experimentally determined formula mass in daltons of the entity ' ],
          [ 'item_', '_item.name', '_entity.pdbx_formula_weight_exptl' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1.0', '1.0', '1.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity.rcsb_formula_weight_exptl' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity.pdbx_formula_weight_exptl_method',
        [ [ 'item_',
            '_item_description.description',
            '              Method used to determine _entity.pdbx_formula_weight_exptl.' ],
          [ 'item_',
            '_item.name',
            '_entity.pdbx_formula_weight_exptl_method' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'MASS SPEC' ],
          [ 'loop_', [ '_item_enumeration.value' ], [ 'MASS SPEC' ] ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '# ####' ],
      [ 'save_',
        '_entity_keywords.pdbx_mutation',
        [ [ 'item_',
            '_item_description.description',
            '              Entity mutation description(s).' ],
          [ 'item_', '_item.name', '_entity_keywords.pdbx_mutation' ],
          [ 'item_', '_item.category_id', 'entity_keywords' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Y31H',
              'DEL(298-323)',
              [ '@comment',
                '    _item_aliases.alias_name     \'_entity_keywords.ndb_mutation\'' ],
              [ '@comment',
                '    _item_aliases.dictionary     \'cif_rcsb.dic\'' ],
              [ '@comment', '    _item_aliases.version        1.1' ] ] ],
          [ 'item_',
            '_item_related.related_name',
            '_entity.pdbx_mutation' ],
          [ 'item_', '_item_related.function_code', 'replacedby' ] ] ],
      [ 'save_',
        '_entity_keywords.pdbx_fragment',
        [ [ 'item_',
            '_item_description.description',
            '              Entity fragment description(s).' ],
          [ 'item_', '_item.name', '_entity_keywords.pdbx_fragment' ],
          [ 'item_', '_item.category_id', 'entity_keywords' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'KLENOW FRAGMENT',
              'REPLICASE OPERATOR HAIRPIN',
              'C-TERMINAL DOMAIN',
              [ '@comment',
                '    _item_aliases.alias_name     \'_entity_keywords.ndb_fragment\'' ],
              [ '@comment',
                '    _item_aliases.dictionary     \'cif_rcsb.dic\'' ],
              [ '@comment', '    _item_aliases.version        1.1' ] ] ],
          [ 'item_',
            '_item_related.related_name',
            '_entity.pdbx_fragment' ],
          [ 'item_', '_item_related.function_code', 'replacedby' ] ] ],
      [ 'save_',
        '_entity_keywords.pdbx_ec',
        [ [ 'item_',
            '_item_description.description',
            '             Enzyme Commission (EC) number(s)' ],
          [ 'item_', '_item.name', '_entity_keywords.pdbx_ec' ],
          [ 'item_', '_item.category_id', 'entity_keywords' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', '2.7.7.7' ],
          [ '@comment',
            '    _item_aliases.alias_name     \'_entity_keywords.ndb_ec\'' ],
          [ '@comment',
            '    _item_aliases.dictionary     \'cif_rcsb.dic\'' ],
          [ '@comment', '    _item_aliases.version        1.1' ],
          [ 'item_',
            '_item_related.related_name',
            '_entity.pdbx_mutation' ],
          [ 'item_', '_item_related.function_code', 'replacedby' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_entity_poly.pdbx_strand_id',
        [ [ 'item_',
            '_item_description.description',
            '              The PDB strand/chain id(s) corresponding to this polymer entity.' ],
          [ 'item_', '_item.name', '_entity_poly.pdbx_strand_id' ],
          [ 'item_', '_item.category_id', 'entity_poly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B', 'A,B,C' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_poly.ndb_chain_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_poly.pdbx_seq_one_letter_code',
        [ [ 'item_',
            '_item_description.description',
            '              Chemical sequence expressed as string of one-letter \n               amino acid codes. Modifications and non-standard \n               amino acids are coded as X.' ],
          [ 'item_',
            '_item.name',
            '_entity_poly.pdbx_seq_one_letter_code' ],
          [ 'item_', '_item.category_id', 'entity_poly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '\nA  for alanine or adenine\nB  for ambiguous asparagine/aspartic-acid\nR  for arginine\nN  for asparagine\nD  for aspartic-acid\nC  for cysteine or cystine or cytosine\nQ  for glutamine\nE  for glutamic-acid\nZ  for ambiguous glutamine/glutamic acid\nG  for glycine or guanine\nH  for histidine\nI  for isoleucine\nL  for leucine\nK  for lysine\nM  for methionine\nF  for phenylalanine\nP  for proline\nS  for serine\nT  for threonine or thymine\nW  for tryptophan\nY  for tyrosine\nV  for valine\nU  for uracil\nO  for water\nX  for other' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_poly.ndb_seq_one_letter_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_poly.pdbx_seq_one_letter_code_can',
        [ [ 'item_',
            '_item_description.description',
            '              Cannonical chemical sequence expressed as string of \n               one-letter amino acid codes. Modifications are coded\n               as the parent amino acid where possible. \n\nA  for alanine or adenine\nB  for ambiguous asparagine/aspartic-acid\nR  for arginine\nN  for asparagine\nD  for aspartic-acid\nC  for cysteine or cystine or cytosine\nQ  for glutamine\nE  for glutamic-acid\nZ  for ambiguous glutamine/glutamic acid\nG  for glycine or guanine\nH  for histidine\nI  for isoleucine\nL  for leucine\nK  for lysine\nM  for methionine\nF  for phenylalanine\nP  for proline\nS  for serine\nT  for threonine or thymine\nW  for tryptophan\nY  for tyrosine\nV  for valine\nU  for uracil' ],
          [ 'item_',
            '_item.name',
            '_entity_poly.pdbx_seq_one_letter_code_can' ],
          [ 'item_', '_item.category_id', 'entity_poly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '\nMSHHWGYGKHNGPEHWHKDFPIAKGERQSPVDIDTHTAKYDPSLKPLSVSYDQATSLRILNNGAAFNVEFD' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_poly.ndb_seq_one_letter_code_can' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_poly.pdbx_target_identifier',
        [ [ 'item_',
            '_item_description.description',
            '              The sequence\'s target identifier registered at target database. ' ],
          [ 'item_', '_item.name', '_entity_poly.pdbx_target_identifier' ],
          [ 'item_', '_item.category_id', 'entity_poly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', '356560' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_fragment',
        [ [ 'item_',
            '_item_description.description',
            '              A domain or fragment of the molecule.              ' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_gene_src_fragment' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'CYTOPLASM', 'NUCLEUS' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_fragment' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_gene',
        [ [ 'item_',
            '_item_description.description',
            '              Identifies the gene.              ' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_gene_src_gene' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_gene' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_scientific_name',
        [ [ 'item_',
            '_item_description.description',
            '              Scientific name of the organism.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_gene_src_scientific_name' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '\nESCHERICHIA COLI\nHOMO SAPIENS\nSACCHAROMYCES CEREVISIAE' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_scientific_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_variant',
        [ [ 'item_',
            '_item_description.description',
            '              Identifies the variant.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_gene_src_variant' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'DELTAH1DELTATRP' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_variant' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_cell_line',
        [ [ 'item_',
            '_item_description.description',
            '              The specific line of cells.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_gene_src_cell_line' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'HELA CELLS' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_cell_line' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_atcc',
        [ [ 'item_',
            '_item_description.description',
            '              American Type Culture Collection tissue culture number.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_gene_src_atcc' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', '6051' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_atcc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_organ',
        [ [ 'item_',
            '_item_description.description',
            '           Organized group of tissues that carries on a specialized function.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_gene_src_organ' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'KIDNEY', 'LIVER', 'PANCREAS' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_organ' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_organelle',
        [ [ 'item_',
            '_item_description.description',
            '              Organized structure within cell.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_gene_src_organelle' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'MITOCHONDRIA' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_organelle' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_plasmid',
        [ [ 'item_',
            '_item_description.description',
            '              The source plasmid.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_gene_src_plasmid' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_plasmid' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_plasmid_name',
        [ [ 'item_',
            '_item_description.description',
            '              The source plasmid.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_gene_src_plasmid_name' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_plasmid_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_cell',
        [ [ 'item_',
            '_item_description.description',
            '              Cell type.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_gene_src_cell' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'ENDOTHELIAL' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_cell' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_gene_src_cellular_location',
        [ [ 'item_',
            '_item_description.description',
            '              Identifies the location inside (or outside) the cell.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_gene_src_cellular_location' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'CYTOPLASM', 'NUCLEUS' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_gene_src_cellular_location' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_gene',
        [ [ 'item_',
            '_item_description.description',
            '              Specific gene which expressed the molecule.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_gene' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'HIV-1 POL', 'GLNS7', 'U1A (2-98, Y31H, Q36R)' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_gene' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_organ',
        [ [ 'item_',
            '_item_description.description',
            '              Specific organ which expressed the molecule.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_organ' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'KIDNEY' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_organ' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_variant',
        [ [ 'item_',
            '_item_description.description',
            '              Variant of the organism used as the expression system.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_variant' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'TRP-LAC', 'LAMBDA DE3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_variant' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_cell_line',
        [ [ 'item_',
            '_item_description.description',
            '              A specific line of cells used as the expression system.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_cell_line' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'HELA' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_cell_line' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_atcc',
        [ [ 'item_',
            '_item_description.description',
            '            Americal Tissue Culture Collection of the expression system.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_atcc' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_atcc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_culture_collection',
        [ [ 'item_',
            '_item_description.description',
            '               Culture collection of the expression system.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_culture_collection' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_culture_collection' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_cell',
        [ [ 'item_',
            '_item_description.description',
            '               Specific cell type which expressed the molecule.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_cell' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'ENDOTHELIAL' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_cell' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_organelle',
        [ [ 'item_',
            '_item_description.description',
            '               Specific organelle which expressed the molecule.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_organelle' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'MITOCHONDRIA' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_organelle' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_cellular_location',
        [ [ 'item_',
            '_item_description.description',
            '              Identifies the location inside (or outside) the cell which \n               expressed the molecule.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_cellular_location' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'CYTOPLASM', 'NUCLEUS' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_cellular_location' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_scientific_name',
        [ [ 'item_',
            '_item_description.description',
            '\n     The scientific name of the organism that served as host for the\n     production of the entity. ' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_scientific_name' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '\nESCHERICHIA COLI\nHOMO SAPIENS\nSACCHAROMYCES CEREVISIAE' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_scientific_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_strain',
        [ [ 'item_',
            '_item_description.description',
            '\n               The strain of the organism in which the entity was\n               expressed.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_strain' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'AR120' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_strain' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_tissue',
        [ [ 'item_',
            '_item_description.description',
            '\n              The specific tissue which expressed the molecule.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_tissue' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'heart', 'liver', 'eye lens' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_tissue' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_tissue_fraction',
        [ [ 'item_',
            '_item_description.description',
            '\n               The fraction of the tissue which expressed the\n               molecule.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_tissue_fraction' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'mitochondria', 'nucleus', 'membrane' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_tissue_fraction' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_vector',
        [ [ 'item_',
            '_item_description.description',
            '              \n               Identifies the vector used.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_vector' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'PBIT36', 'PET15B', 'PUC18' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_vector' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_vector_type',
        [ [ 'item_',
            '_item_description.description',
            '\n              Identifies the type of vector used (plasmid, virus, or\n              cosmid).' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_vector_type' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'COSMID', 'PLASMID' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_host_org_vector_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_description',
        [ [ 'item_',
            '_item_description.description',
            '              Information on the source which is not given elsewhere.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_description' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_gen.ndb_description' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_entity_src_nat.pdbx_organism_scientific',
        [ [ 'item_',
            '_item_description.description',
            '              Scientific name of the organism of the natural source.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_nat.pdbx_organism_scientific' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'BOS TAURUS', 'SUS SCROFA', 'ASPERGILLUS ORYZAE' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_organism_scientific' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_secretion',
        [ [ 'item_',
            '_item_description.description',
            '              Identifies the secretion from which the molecule was isolated. ' ],
          [ 'item_', '_item.name', '_entity_src_nat.pdbx_secretion' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'saliva', 'urine', 'venom' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_secretion' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_fragment',
        [ [ 'item_',
            '_item_description.description',
            '              A domain or fragment of the molecule.' ],
          [ 'item_', '_item.name', '_entity_src_nat.pdbx_fragment' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_fragment' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_variant',
        [ [ 'item_',
            '_item_description.description',
            '              Identifies the variant.' ],
          [ 'item_', '_item.name', '_entity_src_nat.pdbx_variant' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_variant' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_cell_line',
        [ [ 'item_',
            '_item_description.description',
            '              The specific line of cells.' ],
          [ 'item_', '_item.name', '_entity_src_nat.pdbx_cell_line' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'HELA' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_cell_line' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_atcc',
        [ [ 'item_',
            '_item_description.description',
            '              Americal Tissue Culture Collection number.' ],
          [ 'item_', '_item.name', '_entity_src_nat.pdbx_atcc' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', '6051' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_atcc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_cellular_location',
        [ [ 'item_',
            '_item_description.description',
            '              Identifies the location inside (or outside) the cell.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_nat.pdbx_cellular_location' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_cellular_location' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_organ',
        [ [ 'item_',
            '_item_description.description',
            '          Organized group of tissues that carries on a specialized function.' ],
          [ 'item_', '_item.name', '_entity_src_nat.pdbx_organ' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'KIDNEY' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_organ' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_organelle',
        [ [ 'item_',
            '_item_description.description',
            '              Organized structure within cell.' ],
          [ 'item_', '_item.name', '_entity_src_nat.pdbx_organelle' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'MITOCHONDRIA' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_organelle' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_cell',
        [ [ 'item_',
            '_item_description.description',
            '              A particular cell type.' ],
          [ 'item_', '_item.name', '_entity_src_nat.pdbx_cell' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'BHK-21' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_cell' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_plasmid_name',
        [ [ 'item_',
            '_item_description.description',
            '              The plasmid containing the gene.' ],
          [ 'item_', '_item.name', '_entity_src_nat.pdbx_plasmid_name' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'pB322' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_plasmid_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_entity_src_nat.pdbx_plasmid_details',
        [ [ 'item_',
            '_item_description.description',
            '              Details about the plasmid.' ],
          [ 'item_', '_item.name', '_entity_src_nat.pdbx_plasmid_details' ],
          [ 'item_', '_item.category_id', 'entity_src_nat' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'PLC28 DERIVATIVE' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_entity_src_nat.ndb_plasmid_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_exptl_crystal_grow.pdbx_details',
        [ [ 'item_',
            '_item_description.description',
            ' Text description of crystal grow procedure.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.pdbx_details' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'PEG 4000, potassium phosphate, magnesium chloride, cacodylate' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_grow.rcsb_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_exptl_crystal_grow.pdbx_pH_range',
        [ [ 'item_',
            '_item_description.description',
            ' The range of pH values at which the crystal was grown.   Used when \n  a point estimate of pH is not appropriate.' ],
          [ 'item_', '_item.name', '_exptl_crystal_grow.pdbx_pH_range' ],
          [ 'item_', '_item.category_id', 'exptl_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '5.6 - 6.4' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal_grow.rcsb_pH_range' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_exptl_crystal.pdbx_crystal_image_url',
        [ [ 'item_',
            '_item_description.description',
            ' The URL for an a file containing the image of crystal.' ],
          [ 'item_',
            '_item.name',
            '_exptl_crystal.pdbx_crystal_image_url' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal.rcsb_crystal_image_url' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_exptl_crystal.pdbx_crystal_image_format',
        [ [ 'item_',
            '_item_description.description',
            ' The image format for the file containing the image of crystal specified\n  as an RFC2045/RFC2046 mime type.  ' ],
          [ 'item_',
            '_item.name',
            '_exptl_crystal.pdbx_crystal_image_format' ],
          [ 'item_', '_item.category_id', 'exptl_crystal' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'jpeg', 'gif', 'tiff' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_exptl_crystal.rcsb_crystal_image_format' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ '@comment', '#################################' ],
      [ '@comment', '# PDBX_EXPTL_CRYSTAL_GROW_COMP ##' ],
      [ '@comment', '#################################' ],
      [ 'save_',
        'pdbx_exptl_crystal_grow_comp',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_EXPTL_CRYSTAL_GROW_COMP category record\n               details about the components of the solutions that were \'mixed\'\n               to produce the crystal.' ],
          [ 'item_', '_category.id', 'pdbx_exptl_crystal_grow_comp' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_exptl_crystal_grow_comp.comp_id',
              '_pdbx_exptl_crystal_grow_comp.crystal_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'exptl_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              'loop_\n_pdbx_exptl_crystal_grow_comp.crystal_id\n_pdbx_exptl_crystal_grow_comp.sol_id\n_pdbx_exptl_crystal_grow_comp.comp_id\n_pdbx_exptl_crystal_grow_comp.comp_name\n_pdbx_exptl_crystal_grow_comp.conc\n_pdbx_exptl_crystal_grow_comp.conc_range\n_pdbx_exptl_crystal_grow_comp.conc_units\n4    \'protein\'     1  \'protein\'   25.    .  \'mg/ml\'\n4    \'protein\'     2  \'Tris HCl\'  20.    .  \'millimolar\'\n4    \'protein\'     3  \'NaCl\'       0.2   .  \'molar\'\n4    \'precipitant\' 1  \'PEG 4000\'  12.5   .  \'percent_weight_by_volume\'\n4    \'precipitant\' 2  \'MES\'        0.1   .  \'molar\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_comp.crystal_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _exptl_crystal.id in the\n               EXPTL_CRYSTAL category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_grow_comp.crystal_id' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_exptl_crystal_grow_comp.crystal_id' ],
          [ 'item_', '_item_linked.parent_name', '_exptl_crystal.id' ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_comp.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _exptl_crystal_grow_comp.comp_id must uniquely identify\n               each item in the PDBX_EXPTL_CRYSTAL_GROW_COMP list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_grow_comp.comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_comp.comp_name',
        [ [ 'item_',
            '_item_description.description',
            '              A common name for the component of the solution.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_grow_comp.comp_name' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'protein in buffer', 'acetic acid' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_comp.sol_id',
        [ [ 'item_',
            '_item_description.description',
            '              An identifier for the solution to which the given solution\n               component belongs.' ],
          [ 'item_', '_item.name', '_pdbx_exptl_crystal_grow_comp.sol_id' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_exptl_crystal_grow_comp.sol_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_exptl_crystal_grow_sol.sol_id' ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_comp.conc',
        [ [ 'item_',
            '_item_description.description',
            '              The concentration value of the solution component.' ],
          [ 'item_', '_item.name', '_pdbx_exptl_crystal_grow_comp.conc' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '200.', '0.1' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_comp.conc_range',
        [ [ 'item_',
            '_item_description.description',
            '              The concentration range of the solution component.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_grow_comp.conc_range' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '200. - 230.', '0.1 - 0.2' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_comp.conc_units',
        [ [ 'item_',
            '_item_description.description',
            '              The concentration units for the solution component.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_grow_comp.conc_units' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'millimolar',
              'percent_weight_by_volume',
              'milligrams_per_milliliter' ] ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment', '###########################' ],
      [ '@comment', '# PDBX_EXPTL_CRYSTAL_SOL ##' ],
      [ '@comment', '###########################' ],
      [ 'save_',
        'pdbx_exptl_crystal_grow_sol',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_EXPTL_CRYSTAL_GROW_SOL category record\n               details about the solutions that were \'mixed\'\n               to produce the crystal.\n' ],
          [ 'item_', '_category.id', 'pdbx_exptl_crystal_grow_sol' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_exptl_crystal_grow_sol.sol_id',
              '_pdbx_exptl_crystal_grow_sol.crystal_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'exptl_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ ' Example 1',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\nloop_\n_pdbx_exptl_crystal_grow_sol.crystal_id  \n_pdbx_exptl_crystal_grow_sol.sol_id      \n_pdbx_exptl_crystal_grow_sol.volume\n_pdbx_exptl_crystal_grow_sol.volume_units\n_pdbx_exptl_crystal_grow_sol.pH\n1  \'protein\'       0.5   \'microliter\'   7.5\n1  \'precipitant\'   0.5   \'microliter\'   7.3\n1  \'reservoir\'     0.5   \'milliliter\'   7.3',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_sol.crystal_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _exptl_crystal.id in the\n               EXPTL_CRYSTAL category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_grow_sol.crystal_id' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_sol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_exptl_crystal_grow_sol.crystal_id' ],
          [ 'item_', '_item_linked.parent_name', '_exptl_crystal.id' ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_sol.sol_id',
        [ [ 'item_',
            '_item_description.description',
            ' An identifier for this solution (e.g. precipitant, reservoir, macromolecule)' ],
          [ 'item_', '_item.name', '_pdbx_exptl_crystal_grow_sol.sol_id' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_sol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'precipitant', 'reservoir', 'macromolecule' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_sol.volume',
        [ [ 'item_',
            '_item_description.description',
            '              The volume of the solution.' ],
          [ 'item_', '_item.name', '_pdbx_exptl_crystal_grow_sol.volume' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_sol' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '200.', '0.1' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_sol.volume_units',
        [ [ 'item_',
            '_item_description.description',
            '              The volume units of the solution.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_grow_sol.volume_units' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_sol' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'milliliter', 'micoliter' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_grow_sol.pH',
        [ [ 'item_',
            '_item_description.description',
            '              The pH of the solution.' ],
          [ 'item_', '_item.name', '_pdbx_exptl_crystal_grow_sol.pH' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_crystal_grow_sol' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '7.2' ] ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment', '######################################' ],
      [ '@comment', '# PDBX_EXPTL_CRYSTAL_CRYO_TREATMENT ##' ],
      [ '@comment', '######################################' ],
      [ 'save_',
        'pdbx_exptl_crystal_cryo_treatment',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_EXPTL_CRYSTAL_CRYO_TREATMENT category \n               record details cryogenic treatments applied to this crystal.\n' ],
          [ 'item_', '_category.id', 'pdbx_exptl_crystal_cryo_treatment' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_exptl_crystal_cryo_treatment.crystal_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'exptl_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ ' Example 1',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ],
              '\n _pdbx_exptl_crystal_cryo_treatment.crystal_id  4\n _pdbx_exptl_crystal_cryo_treatment.final_solution_details\n ;  25% (v/v) glycerol in precipitant solution\n ;\n _pdbx_exptl_crystal_cryo_treatment.soaking_details\n ;  A series of 1 min soaks beginning at 5% (v/v) glycerol in precipitant, \n    transiting to 20% (v/v) glycerol, and finally to 25% glycerol\n ;\n _pdbx_exptl_crystal_cryo_treatment.cooling_details\n ;  Direct immersion in liquid nitrogen\n ;\n _pdbx_exptl_crystal_cryo_treatment.annealing_details   ?',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_cryo_treatment.crystal_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _exptl_crystal.id in the\n               EXPTL_CRYSTAL category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_cryo_treatment.crystal_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_exptl_crystal_cryo_treatment' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_exptl_crystal_cryo_treatment.crystal_id' ],
          [ 'item_', '_item_linked.parent_name', '_exptl_crystal.id' ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_cryo_treatment.final_solution_details',
        [ [ 'item_',
            '_item_description.description',
            '         Details of the final solution used in the treatment of this crystal' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_cryo_treatment.final_solution_details' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_exptl_crystal_cryo_treatment' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ ' 25% (v/v) glycerol in precipitant solution' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_cryo_treatment.soaking_details',
        [ [ 'item_',
            '_item_description.description',
            '             Details of the soaking treatment applied to this crystal.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_cryo_treatment.soaking_details' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_exptl_crystal_cryo_treatment' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '  A series of 1 min soaks beginning at 5% (v/v) glycerol in precipitant, \n   transiting to 20% (v/v) glycerol, and finally to 25% glycerol' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_cryo_treatment.cooling_details',
        [ [ 'item_',
            '_item_description.description',
            '             Details of the cooling treatment applied to this crystal.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_cryo_treatment.cooling_details' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_exptl_crystal_cryo_treatment' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '  Direct immersion in liquid nitrogen' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_crystal_cryo_treatment.annealing_details',
        [ [ 'item_',
            '_item_description.description',
            '             Details of the annealing treatment applied to this crystal.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_crystal_cryo_treatment.annealing_details' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_exptl_crystal_cryo_treatment' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ ' 10 sec interruption of cold stream with plastic ruler.\n  Performed twice.' ] ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_geom_angle.pdbx_atom_site_PDB_ins_code_1',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.pdbx_atom_site_PDB_ins_code_1' ],
          [ 'item_', '_item.category_id', 'geom_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_angle.pdbx_atom_site_PDB_ins_code_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_angle.ndb_atom_site_PDB_ins_code_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_angle.pdbx_atom_site_PDB_ins_code_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.pdbx_atom_site_PDB_ins_code_2' ],
          [ 'item_', '_item.category_id', 'geom_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_angle.pdbx_atom_site_PDB_ins_code_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_angle.ndb_atom_site_PDB_ins_code_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_angle.pdbx_atom_site_PDB_ins_code_3',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_angle.pdbx_atom_site_PDB_ins_code_3' ],
          [ 'item_', '_item.category_id', 'geom_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_angle.pdbx_atom_site_PDB_ins_code_3' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_angle.ndb_atom_site_PDB_ins_code_3' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_angle.pdbx_PDB_model_num',
        [ [ 'item_',
            '_item_description.description',
            '      Pointer to _atom_site.pdbx_PDB_model_num' ],
          [ 'item_', '_item.name', '_geom_angle.pdbx_PDB_model_num' ],
          [ 'item_', '_item.category_id', 'geom_angle' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_angle.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_angle.ndb_model_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_bond.pdbx_atom_site_PDB_ins_code_1',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_bond.pdbx_atom_site_PDB_ins_code_1' ],
          [ 'item_', '_item.category_id', 'geom_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_bond.pdbx_atom_site_PDB_ins_code_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_bond.ndb_atom_site_PDB_ins_code_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_bond.pdbx_atom_site_PDB_ins_code_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_bond.pdbx_atom_site_PDB_ins_code_2' ],
          [ 'item_', '_item.category_id', 'geom_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_bond.pdbx_atom_site_PDB_ins_code_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_bond.ndb_atom_site_PDB_ins_code_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_bond.pdbx_PDB_model_num',
        [ [ 'item_',
            '_item_description.description',
            '      Pointer to _atom_site.pdbx_PDB_model_num' ],
          [ 'item_', '_item.name', '_geom_bond.pdbx_PDB_model_num' ],
          [ 'item_', '_item.category_id', 'geom_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_bond.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_bond.ndb_model_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_contact.pdbx_atom_site_PDB_ins_code_1',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.pdbx_atom_site_PDB_ins_code_1' ],
          [ 'item_', '_item.category_id', 'geom_contact' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_contact.pdbx_atom_site_PDB_ins_code_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_contact.ndb_atom_site_PDB_ins_code_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_contact.pdbx_atom_site_PDB_ins_code_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_contact.pdbx_atom_site_PDB_ins_code_2' ],
          [ 'item_', '_item.category_id', 'geom_contact' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_contact.pdbx_atom_site_PDB_ins_code_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_contact.ndb_atom_site_PDB_ins_code_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_contact.pdbx_PDB_model_num',
        [ [ 'item_',
            '_item_description.description',
            '      Pointer to _atom_site.pdbx_PDB_model_num' ],
          [ 'item_', '_item.name', '_geom_contact.pdbx_PDB_model_num' ],
          [ 'item_', '_item.category_id', 'geom_contact' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_contact.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_contact.ndb_model_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_torsion.pdbx_atom_site_PDB_ins_code_1',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.pdbx_atom_site_PDB_ins_code_1' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_torsion.pdbx_atom_site_PDB_ins_code_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion.ndb_atom_site_PDB_ins_code_1' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_torsion.pdbx_atom_site_PDB_ins_code_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.pdbx_atom_site_PDB_ins_code_2' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_torsion.pdbx_atom_site_PDB_ins_code_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion.ndb_atom_site_PDB_ins_code_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_torsion.pdbx_atom_site_PDB_ins_code_3',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.pdbx_atom_site_PDB_ins_code_3' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_torsion.pdbx_atom_site_PDB_ins_code_3' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion.ndb_atom_site_PDB_ins_code_3' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_torsion.pdbx_atom_site_PDB_ins_code_4',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_geom_torsion.pdbx_atom_site_PDB_ins_code_4' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_torsion.pdbx_atom_site_PDB_ins_code_4' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion.ndb_atom_site_PDB_ins_code_4' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_geom_torsion.pdbx_PDB_model_num',
        [ [ 'item_',
            '_item_description.description',
            '      Pointer to _atom_site.pdbx_PDB_model_num' ],
          [ 'item_', '_item.name', '_geom_torsion.pdbx_PDB_model_num' ],
          [ 'item_', '_item.category_id', 'geom_torsion' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_geom_torsion.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_geom_torsion.ndb_model_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_refine.pdbx_ls_sigma_I',
        [ [ 'item_',
            '_item_description.description',
            '              Data cutoff (SIGMA(I))' ],
          [ 'item_', '_item.name', '_refine.pdbx_ls_sigma_I' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_aliases.alias_name', '_refine.ndb_ls_sigma_I' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_ls_sigma_F',
        [ [ 'item_',
            '_item_description.description',
            '              Data cutoff (SIGMA(F))' ],
          [ 'item_', '_item.name', '_refine.pdbx_ls_sigma_F' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_aliases.alias_name', '_refine.ndb_ls_sigma_F' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_ls_sigma_Fsqd',
        [ [ 'item_',
            '_item_description.description',
            '              Data cutoff (SIGMA(F^2))' ],
          [ 'item_', '_item.name', '_refine.pdbx_ls_sigma_Fsqd' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.pdbx_data_cutoff_high_absF',
        [ [ 'item_',
            '_item_description.description',
            '              Value of F at "high end" of data cutoff.' ],
          [ 'item_', '_item.name', '_refine.pdbx_data_cutoff_high_absF' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '17600' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ndb_data_cutoff_high_absF' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_data_cutoff_high_rms_absF',
        [ [ 'item_',
            '_item_description.description',
            '              Value of RMS |F| used as high data cutoff.' ],
          [ 'item_',
            '_item.name',
            '_refine.pdbx_data_cutoff_high_rms_absF' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '205.1' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.rcsb_data_cutoff_high_rms_absF' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_data_cutoff_low_absF',
        [ [ 'item_',
            '_item_description.description',
            '              Value of F at "low end" of data cutoff.' ],
          [ 'item_', '_item.name', '_refine.pdbx_data_cutoff_low_absF' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '0.30' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ndb_data_cutoff_low_absF' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_isotropic_thermal_model',
        [ [ 'item_',
            '_item_description.description',
            '\n               Whether the structure was refined with indvidual\n               isotropic, anisotropic or overall temperature factor.' ],
          [ 'item_', '_item.name', '_refine.pdbx_isotropic_thermal_model' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Isotropic', 'Overall' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ndb_isotropic_thermal_model' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_ls_cross_valid_method',
        [ [ 'item_',
            '_item_description.description',
            '\n               Whether the cross validataion method was used through\n               out or only at the end.' ],
          [ 'item_', '_item.name', '_refine.pdbx_ls_cross_valid_method' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'FREE R-VALUE' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ndb_ls_cross_valid_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_method_to_determine_struct',
        [ [ 'item_',
            '_item_description.description',
            '                 Method(s) used to determine the structure.' ],
          [ 'item_',
            '_item.name',
            '_refine.pdbx_method_to_determine_struct' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ 'AB INITIO PHASING',
              '?',
              'DM',
              'Direct Methods',
              'ISAS ',
              'Iterative Single wavelength Anomalous Scattering',
              'ISIR',
              'Iterative Single Isomorphous Replacement',
              'ISIRAS',
              'Iterative Single Isomorphous Replacement with Anomalous Scattering',
              'MAD',
              'Multi wavelength Anomalous Diffraction',
              'MIR',
              'Multiple Isomorphous Replacement',
              'MIRAS',
              'Multiple Isomorphous Replacement with Anomalous Scattering',
              'MR',
              'Molecular Replacement',
              'SIR',
              'Single Isomorphous Replacement',
              'SIRAS',
              'Single Isomorphous Replacement with Anomalous Scattering' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ndb_method_to_determine_struct' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_starting_model',
        [ [ 'item_',
            '_item_description.description',
            '                 Starting model for refinement.  Starting model for \n                  molecular replacement should refer to a previous\n                  structure or experiment.' ],
          [ 'item_', '_item.name', '_refine.pdbx_starting_model' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'BDL001' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ndb_starting_model' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_stereochemistry_target_values',
        [ [ 'item_',
            '_item_description.description',
            '                  Stereochemistry target values used in refinement. ' ],
          [ 'item_',
            '_item.name',
            '_refine.pdbx_stereochemistry_target_values' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ndb_stereochemistry_target_values' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_R_Free_selection_details',
        [ [ 'item_',
            '_item_description.description',
            '            Details of the manner in which the cross validation \n             reflections were selected.' ],
          [ 'item_',
            '_item.name',
            '_refine.pdbx_R_Free_selection_details' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'Random selection' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ndb_R_Free_selection_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine.pdbx_stereochem_target_val_spec_case',
        [ [ 'item_',
            '_item_description.description',
            '\n                  Special case of stereochemistry target values used\n                  in SHELXL refinement.  \n' ],
          [ 'item_',
            '_item.name',
            '_refine.pdbx_stereochem_target_val_spec_case' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ndb_stereochem_target_val_spec_case' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine_hist.pdbx_number_atoms_protein',
        [ [ 'item_',
            '_item_description.description',
            '              Number of protein atoms included in refinement' ],
          [ 'item_',
            '_item.name',
            '_refine_hist.pdbx_number_atoms_protein' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_hist.rcsb_number_atoms_protein' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine_hist.pdbx_number_atoms_nucleic_acid',
        [ [ 'item_',
            '_item_description.description',
            '              Number of nucleic atoms included in refinement' ],
          [ 'item_',
            '_item.name',
            '_refine_hist.pdbx_number_atoms_nucleic_acid' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_hist.rcsb_number_atoms_nucleic_acid' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine_hist.pdbx_number_atoms_ligand',
        [ [ 'item_',
            '_item_description.description',
            '              Number of ligand atoms included in refinement' ],
          [ 'item_',
            '_item.name',
            '_refine_hist.pdbx_number_atoms_ligand' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_hist.rcsb_number_atoms_ligand' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine_hist.pdbx_number_atoms_lipid',
        [ [ 'item_',
            '_item_description.description',
            '              Number of lipid atoms included in refinement' ],
          [ 'item_', '_item.name', '_refine_hist.pdbx_number_atoms_lipid' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_hist.rcsb_number_atoms_lipid' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine_hist.pdbx_number_atoms_carb',
        [ [ 'item_',
            '_item_description.description',
            '              Number of carbohydrate atoms included in refinement' ],
          [ 'item_', '_item.name', '_refine_hist.pdbx_number_atoms_carb' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_hist.rcsb_number_atoms_carb' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine_hist.pdbx_pseudo_atom_details',
        [ [ 'item_',
            '_item_description.description',
            '              Details of pseduo atoms used to model unexplained density' ],
          [ 'item_',
            '_item.name',
            '_refine_hist.pdbx_pseudo_atom_details' ],
          [ 'item_', '_item.category_id', 'refine_hist' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_hist.rcsb_pseudo_atom_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        '_refine_ls_shell.pdbx_total_number_of_bins_used',
        [ [ 'item_',
            '_item_description.description',
            '              Total number of bins used.' ],
          [ 'item_',
            '_item.name',
            '_refine_ls_shell.pdbx_total_number_of_bins_used' ],
          [ 'item_', '_item.category_id', 'refine_ls_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_shell.ndb_total_number_of_bins_used' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_refln.pdbx_F_calc_part_solvent',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of the structure factor in arbitrary\n               units reflecting only the contribution of the solvent model.' ],
          [ 'item_', '_item.name', '_refln.pdbx_F_calc_part_solvent' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refln.rcsb_F_calc_part_solvent' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refln.pdbx_phase_calc_part_solvent',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated structure-factor phase in degrees reflecting\n               only the contribution of the solvent model.' ],
          [ 'item_', '_item.name', '_refln.pdbx_phase_calc_part_solvent' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refln.rcsb_phase_calc_part_solvent' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ 'save_',
        '_refln.pdbx_F_calc_with_solvent',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated value of the structure factor in arbitrary\n               units including the contribution of the solvent model.' ],
          [ 'item_', '_item.name', '_refln.pdbx_F_calc_with_solvent' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refln.rcsb_F_calc_with_solvent' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refln.pdbx_phase_calc_with_solvent',
        [ [ 'item_',
            '_item_description.description',
            '              The calculated structure-factor phase in degrees including\n               the contribution of the solvent model.' ],
          [ 'item_', '_item.name', '_refln.pdbx_phase_calc_with_solvent' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refln.rcsb_phase_calc_with_solvent' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_reflns.pdbx_redundancy',
        [ [ 'item_',
            '_item_description.description',
            '            Overall redundancy for this data set (%).' ],
          [ 'item_', '_item.name', '_reflns.pdbx_redundancy' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_aliases.alias_name', '_reflns.ndb_redundancy' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_reflns.pdbx_Rmerge_I_obs',
        [ [ 'item_',
            '_item_description.description',
            '     The R value for merging intensities satisfying the observed\n      criteria in this data set.' ],
          [ 'item_', '_item.name', '_reflns.pdbx_Rmerge_I_obs' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns.ndb_Rmerge_I_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_reflns.pdbx_Rmerge_I_all',
        [ [ 'item_',
            '_item_description.description',
            '     The R value for merging all intensities in this data set.' ],
          [ 'item_', '_item.name', '_reflns.pdbx_Rmerge_I_all' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns.pdbx_Rsym_value',
        [ [ 'item_',
            '_item_description.description',
            '              R Sym value in percent.' ],
          [ 'item_', '_item.name', '_reflns.pdbx_Rsym_value' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_examples.case', '2.0' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_aliases.alias_name', '_reflns.ndb_Rsym_value' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_reflns.pdbx_netI_over_av_sigmaI',
        [ [ 'item_',
            '_item_description.description',
            '           Overall  <I>/<sigma(I)>' ],
          [ 'item_', '_item.name', '_reflns.pdbx_netI_over_av_sigmaI' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns.ndb_netI_over_av_sigmaI' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_reflns.pdbx_netI_over_sigmaI',
        [ [ 'item_',
            '_item_description.description',
            '           Overall  <I/sigma(I)>' ],
          [ 'item_', '_item.name', '_reflns.pdbx_netI_over_sigmaI' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns.pdbx_res_netI_over_av_sigmaI_2',
        [ [ 'item_',
            '_item_description.description',
            '         Resolution (angstrom) for reflections with  <I>/<sigma(I)> = 2.' ],
          [ 'item_',
            '_item.name',
            '_reflns.pdbx_res_netI_over_av_sigmaI_2' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_reflns.pdbx_res_netI_over_sigmaI_2',
        [ [ 'item_',
            '_item_description.description',
            '         Resolution (angstroms) for reflections with  <I/sigma(I)> = 2.' ],
          [ 'item_', '_item.name', '_reflns.pdbx_res_netI_over_sigmaI_2' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_reflns.pdbx_chi_squared',
        [ [ 'item_',
            '_item_description.description',
            '           Overall  Chi-squared statistic.' ],
          [ 'item_', '_item.name', '_reflns.pdbx_chi_squared' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns.rcsb_chi_squared' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_reflns.pdbx_scaling_rejects',
        [ [ 'item_',
            '_item_description.description',
            '              Number of reflections rejected in scaling operations.' ],
          [ 'item_', '_item.name', '_reflns.pdbx_scaling_rejects' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns.rcsb_scaling_rejects' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_reflns.pdbx_d_res_high_opt',
        [ [ 'item_',
            '_item_description.description',
            '              The highest optical resolution for this reflection data set\n               as determined by computational method _reflns.pdbx_d_res_opt_method.' ],
          [ 'item_', '_item.name', '_reflns.pdbx_d_res_high_opt' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns.rcsb_d_res_high_opt' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ 'item_', '_item_examples.case', '1.2' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_reflns.pdbx_d_res_low_opt',
        [ [ 'item_',
            '_item_description.description',
            '              The lowest optical resolution for this reflection data set \n               as determined by computational method _reflns.pdbx_d_res_opt_method.' ],
          [ 'item_', '_item.name', '_reflns.pdbx_d_res_low_opt' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns.rcsb_d_res_low_opt' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ 'item_', '_item_examples.case', '20.5' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_reflns.pdbx_d_res_opt_method',
        [ [ 'item_',
            '_item_description.description',
            '              The computational method used to determine the optical \n               resolution limits _reflns.pdbx_d_res_high_opt and \n               _reflns.pdbx_d_res_low_opt.' ],
          [ 'item_', '_item.name', '_reflns.pdbx_d_res_opt_method' ],
          [ 'item_', '_item.category_id', 'reflns' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns.rcsb_d_res_opt_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ 'item_', '_item_examples.case', 'SFCHECK' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_reflns_shell.pdbx_redundancy',
        [ [ 'item_',
            '_item_description.description',
            '        Redundancy for the current shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.pdbx_redundancy' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell.ndb_redundancy' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_reflns_shell.pdbx_Rsym_value',
        [ [ 'item_',
            '_item_description.description',
            '              R sym value in percent.' ],
          [ 'item_', '_item.name', '_reflns_shell.pdbx_Rsym_value' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell.ndb_Rsym_value' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_reflns_shell.pdbx_chi_squared',
        [ [ 'item_',
            '_item_description.description',
            '          Chi-squared statistic for this resolution shell.' ],
          [ 'item_', '_item.name', '_reflns_shell.pdbx_chi_squared' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_reflns_shell.rcsb_chi_squared' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_reflns_shell.pdbx_netI_over_sigmaI_all',
        [ [ 'item_',
            '_item_description.description',
            '              The mean of the ratio of the intensities to their\n               standard uncertainties of all reflections in the \n               resolution shell.\n\n               _reflns_shell.pdbx_meanI_over_sigmaI_all =  <I/sigma(I)>' ],
          [ 'item_',
            '_item.name',
            '_reflns_shell.pdbx_netI_over_sigmaI_all' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_reflns_shell.pdbx_netI_over_sigmaI_obs',
        [ [ 'item_',
            '_item_description.description',
            '              The mean of the ratio of the intensities to their\n               standard uncertainties of observed reflections \n               (see _reflns.observed_criterion) in the resolution shell.\n\n               _reflns_shell.pdbx_meanI_over_sigmaI_obs =  <I/sigma(I)>' ],
          [ 'item_',
            '_item.name',
            '_reflns_shell.pdbx_netI_over_sigmaI_obs' ],
          [ 'item_', '_item.category_id', 'reflns_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct.pdbx_descriptor',
        [ [ 'item_',
            '_item_description.description',
            '              An automatically generated descriptor for an NDB structure or\n               the unstructured content of the PDB COMPND record.' ],
          [ 'item_', '_item.name', '_struct.pdbx_descriptor' ],
          [ 'item_', '_item.category_id', 'struct' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '\n     5\'-D(*CP*GP*CP*(HYD)AP*AP*AP*TP*TP*TP*GP*CP*G)-3\'         ' ] ],
          [ 'item_', '_item_aliases.alias_name', '_struct.ndb_descriptor' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct.pdbx_model_details',
        [ [ 'item_',
            '_item_description.description',
            '              Text description of the methodology which produced this\n               model structure.' ],
          [ 'item_', '_item.name', '_struct.pdbx_model_details' ],
          [ 'item_', '_item.category_id', 'struct' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '\nThis model was produced from a 10 nanosecond Amber/MD simulation \nstarting from PDB structure ID 1ABC.' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct.rcsb_model_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct.pdbx_formula_weight',
        [ [ 'item_',
            '_item_description.description',
            '              Estimated formula mass in daltons of the \n               deposited structure assembly.' ],
          [ 'item_', '_item.name', '_struct.pdbx_formula_weight' ],
          [ 'item_', '_item.category_id', 'struct' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1.0', '1.0', '1.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct.pdbx_formula_weight_method',
        [ [ 'item_',
            '_item_description.description',
            '              Method used to determine _struct.pdbx_formula_weight.' ],
          [ 'item_', '_item.name', '_struct.pdbx_formula_weight_method' ],
          [ 'item_', '_item.category_id', 'struct' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'MASS SPEC', 'CALCULATION' ] ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_asym.pdbx_modified',
        [ [ 'item_',
            '_item_description.description',
            '        This data item indicates whether the structural elements are modified.' ],
          [ 'item_', '_item.name', '_struct_asym.pdbx_modified' ],
          [ 'item_', '_item.category_id', 'struct_asym' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'y' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_asym.ndb_modified' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_asym.pdbx_blank_PDB_chainid_flag',
        [ [ 'item_',
            '_item_description.description',
            '              A flag indicating that this entity was originally labeled\n               with a blank PDB chain id.' ],
          [ 'item_',
            '_item.name',
            '_struct_asym.pdbx_blank_PDB_chainid_flag' ],
          [ 'item_', '_item.category_id', 'struct_asym' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes', 'N', 'No' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_asym.rcsb_blank_PDB_chainid_flag' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_biol.pdbx_parent_biol_id',
        [ [ 'item_',
            '_item_description.description',
            '              An identifier for the parent biological assembly\n               if this biological unit is part of a complex assembly.' ],
          [ 'item_', '_item.name', '_struct_biol.pdbx_parent_biol_id' ],
          [ 'item_', '_item.category_id', 'struct_biol' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_biol.pdbx_parent_biol_id' ],
          [ 'item_', '_item_linked.parent_name', '_struct_biol.id' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_biol.rcsb_parent_biol_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_biol.pdbx_formula_weight',
        [ [ 'item_',
            '_item_description.description',
            '              Estimated formula mass in daltons of the \n               biological assembly.' ],
          [ 'item_', '_item.name', '_struct_biol.pdbx_formula_weight' ],
          [ 'item_', '_item.category_id', 'struct_biol' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1.0', '1.0', '1.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_struct_biol.pdbx_formula_weight_method',
        [ [ 'item_',
            '_item_description.description',
            '              Method used to determine _struct_biol.pdbx_formula_weight.' ],
          [ 'item_',
            '_item.name',
            '_struct_biol.pdbx_formula_weight_method' ],
          [ 'item_', '_item.category_id', 'struct_biol' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'MASS SPEC', 'CALCULATION' ] ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_biol_gen.pdbx_full_symmetry_operation',
        [ [ 'item_',
            '_item_description.description',
            '\n               This item expresses category _struct_biol_gen.symmetry\n               on an X, Y and Z basis.         ' ],
          [ 'item_',
            '_item.name',
            '_struct_biol_gen.pdbx_full_symmetry_operation' ],
          [ 'item_', '_item.category_id', 'struct_biol_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '-X, Y+1/2, -Z' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_biol_gen.ndb_full_symmetry_operation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_biol_gen.pdbx_PDB_order',
        [ [ 'item_',
            '_item_description.description',
            '              An ordering index used to reproduce the presentation of \n               chain order in the original PDB format data files.' ],
          [ 'item_', '_item.name', '_struct_biol_gen.pdbx_PDB_order' ],
          [ 'item_', '_item.category_id', 'struct_biol_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_biol_gen.rcsb_pdb_order' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_conf.pdbx_beg_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment starts.\n' ],
          [ 'item_', '_item.name', '_struct_conf.pdbx_beg_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_conf' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conf.pdbx_beg_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conf.ndb_beg_label_ins_code_pdb' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conf.pdbx_end_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               conformation segment ends.\n' ],
          [ 'item_', '_item.name', '_struct_conf.pdbx_end_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_conf' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conf.pdbx_end_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conf.ndb_end_label_ins_code_pdb' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conf.pdbx_PDB_helix_class',
        [ [ 'item_',
            '_item_description.description',
            '              This item is a place holder for the helix class used in the PDB \n               HELIX record.' ],
          [ 'item_', '_item.name', '_struct_conf.pdbx_PDB_helix_class' ],
          [ 'item_', '_item.category_id', 'struct_conf' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conf.ndb_helix_class_pdb' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conf.pdbx_PDB_helix_length',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A placeholder for the lengths of the helix of the PDB\n               HELIX record.' ],
          [ 'item_', '_item.name', '_struct_conf.pdbx_PDB_helix_length' ],
          [ 'item_', '_item.category_id', 'struct_conf' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conf.ndb_length' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conf.pdbx_PDB_helix_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n                A placeholder for the helix identifier of the PDB\n                HELIX record.' ],
          [ 'item_', '_item.name', '_struct_conf.pdbx_PDB_helix_id' ],
          [ 'item_', '_item.category_id', 'struct_conf' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_aliases.alias_name', '_struct_conf.pdb_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr1_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for partner 1 of the structure\n               connection.\n\n               This data item is a pointer to _atom_site.ins_code in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr1_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr1_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr1_label_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr1_auth_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 1 of the\n               structure connection. This data item is a pointer to\n               _atom_site.pdbx_auth_alt_id in the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr1_auth_alt_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr1_auth_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_auth_alt_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr1_auth_alt_loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr1_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 1 of the\n               structure connection. This data item is a pointer to\n               _atom_site.label_alt_id in the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr1_label_alt_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr1_label_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_alt_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr1_label_alt_loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr1_standard_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A placeholder for the standard residue name found in\n               the MODRES record of a PDB file.' ],
          [ 'item_',
            '_item.name',
            '_struct_conn.pdbx_ptnr1_standard_comp_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'A', 'T', 'C', 'G', 'GLY', 'ALA', 'MET' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr1_standard_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr2_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '           A component of the identifier for partner 1 of the structure\n            connection.\n\n            This data item is a pointer to _atom_site.pdbx_PDB_ins_code in the\n            ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr2_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr2_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr2_label_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr2_auth_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 2 of the\n               structure connection. This data item is a pointer to\n               _atom_site.pdbx_auth_alt_id in the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr2_auth_alt_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr2_auth_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_auth_alt_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr2_auth_alt_loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr2_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 2 of the\n               structure connection. This data item is a pointer to\n               _atom_site.label_alt_id in the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr2_label_alt_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr2_label_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_alt_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr2_label_alt_loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_auth_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 3 of the\n               structure connection. This data item is a pointer to\n               _atom_site.auth_alt_id in the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr3_auth_alt_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_auth_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_auth_alt_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_auth_alt_loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 3 of the\n               structure connection. This data item is a pointer to\n               _atom_site.auth_asym_id in the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr3_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B', 'C' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_auth_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_auth_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 3 of the\n               structure connection. This data item is a pointer to\n               _atom_site.auth_atom_id in the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr3_auth_atom_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'atcode' ],
          [ 'item_', '_item_examples.case', 'O5*' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_auth_atom_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_atom_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_auth_atom_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '\n               A component of the identifier for partner 3 of the\n               structure connection. This data item is a pointer to\n               _atom_site.auth_comp_id in the ATOM_SITE category.        ' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr3_auth_comp_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'A', 'T', 'C', 'G', 'GLY', 'ALA', 'MET' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_auth_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '\n               A component of the identifier for partner 3 of the\n               structure connection. This data item is a pointer to\n               _atom_site.pdbx_PDB_ins_code in the ATOM_SITE category.  ' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr3_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_label_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 1 of the\n               structure connection. This data item is a pointer to\n               _atom_site.auth_seq_id in the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr3_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '12' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_auth_seq_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 3 of the\n               structure connection. This data item is a pointer to\n               _atom_site.label_alt_id in the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr3_label_alt_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_label_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_alt_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_label_alt_loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 3 of the\n               structure connection. This data item is a pointer to\n               _atom_site.label_asym_id in the ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_conn.pdbx_ptnr3_label_asym_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B', 'C' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_label_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_label_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 3 of the\n               structure connection. This data item is a pointer to\n               _atom_site.label_atom_id in the ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_struct_conn.pdbx_ptnr3_label_atom_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'atcode' ],
          [ 'item_', '_item_examples.case', 'O5*' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_label_atom_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_atom_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_label_atom_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '\n               A component of the identifier for partner 3 of the\n               structure connection. This data item is a pointer to\n               _atom_site.label_comp_id in the ATOM_SITE category.        ' ],
          [ 'item_',
            '_item.name',
            '_struct_conn.pdbx_ptnr3_label_comp_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'A', 'T', 'C', 'G', 'GLY', 'ALA', 'MET' ] ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_label_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_conn.pdbx_ptnr3_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A component of the identifier for partner 1 of the\n               structure connection. This data item is a pointer to\n               _atom_site.label_seq_id in the ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_ptnr3_label_seq_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '12' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_conn.pdbx_ptnr3_label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_ptnr3_label_seq_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_conn.pdbx_PDB_id',
        [ [ 'item_',
            '_item_description.description',
            '         \n               A placeholder for the PDB id in the case the category\n               is used to hold the information of the MODRES record of\n               a PDB file.' ],
          [ 'item_', '_item.name', '_struct_conn.pdbx_PDB_id' ],
          [ 'item_', '_item.category_id', 'struct_conn' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '1ABC' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_conn.ndb_PDB_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_keywords.pdbx_keywords',
        [ [ 'item_', '_item_description.description', '         ' ],
          [ 'item_', '_item.name', '_struct_keywords.pdbx_keywords' ],
          [ 'item_', '_item.category_id', 'struct_keywords' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'DNA',
              'RNA',
              'T-RNA',
              'DNA/RNA',
              'RIBOZYME',
              'PROTEIN/DNA',
              'PROTEIN/RNA',
              'PEPTIDE NUCLEIC ACID',
              'PEPTIDE NUCLEIC ACID/DNA',
              'DNA-BINDING PROTEIN',
              'RNA-BINDING PROTEIN' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_keywords.ndb_keywords' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_auth_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.auth_asym_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_mon_prot_cis.pdbx_auth_asym_id_2' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_mon_prot_cis.pdbx_auth_asym_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.ndb_auth_asym_id_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_auth_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.auth_comp_id.         ' ],
          [ 'item_',
            '_item.name',
            '_struct_mon_prot_cis.pdbx_auth_comp_id_2' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_mon_prot_cis.pdbx_auth_comp_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.ndb_auth_comp_id_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_auth_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.auth_seq_id         ' ],
          [ 'item_',
            '_item.name',
            '_struct_mon_prot_cis.pdbx_auth_seq_id_2' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_mon_prot_cis.pdbx_auth_seq_id_2' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.ndb_auth_seq_id_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#jdw' ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_label_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.label_asym_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_mon_prot_cis.pdbx_label_asym_id_2' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_mon_prot_cis.pdbx_label_asym_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.ndb_label_asym_id_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_label_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.label_comp_id.         ' ],
          [ 'item_',
            '_item.name',
            '_struct_mon_prot_cis.pdbx_label_comp_id_2' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_mon_prot_cis.pdbx_label_comp_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.ndb_label_comp_id_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_label_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.label_seq_id         ' ],
          [ 'item_',
            '_item.name',
            '_struct_mon_prot_cis.pdbx_label_seq_id_2' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_mon_prot_cis.pdbx_label_seq_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.ndb_label_seq_id_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code         ' ],
          [ 'item_',
            '_item.name',
            '_struct_mon_prot_cis.pdbx_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_mon_prot_cis.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.ndb_label_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_PDB_ins_code_2',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code         ' ],
          [ 'item_',
            '_item.name',
            '_struct_mon_prot_cis.pdbx_PDB_ins_code_2' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_mon_prot_cis.pdbx_PDB_ins_code_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.ndb_label_ins_code_2' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_PDB_model_num',
        [ [ 'item_',
            '_item_description.description',
            '      Pointer to _atom_site.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item.name',
            '_struct_mon_prot_cis.pdbx_PDB_model_num' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_mon_prot_cis.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_model_num' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.ndb_model_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_omega_angle',
        [ [ 'item_',
            '_item_description.description',
            '         omega torsion angle ' ],
          [ 'item_',
            '_item.name',
            '_struct_mon_prot_cis.pdbx_omega_angle' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.ndb_omega_angle' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_mon_prot_cis.pdbx_id',
        [ [ 'item_',
            '_item_description.description',
            '    ordinal index         ' ],
          [ 'item_', '_item.name', '_struct_mon_prot_cis.pdbx_id' ],
          [ 'item_', '_item.category_id', 'struct_mon_prot_cis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_mon_prot_cis.id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_ref.pdbx_db_accession',
        [ [ 'item_',
            '_item_description.description',
            '              Accession code of the reference database.' ],
          [ 'item_', '_item.name', '_struct_ref.pdbx_db_accession' ],
          [ 'item_', '_item.category_id', 'struct_ref' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'P07617' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref.rcsb_db_accession' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref.pdbx_seq_one_letter_code',
        [ [ 'item_',
            '_item_description.description',
            '              Database chemical sequence expressed as string of one-letter \n               amino acid codes. ' ],
          [ 'item_', '_item.name', '_struct_ref.pdbx_seq_one_letter_code' ],
          [ 'item_', '_item.category_id', 'struct_ref' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '\nA  for alanine or adenine\nB  for ambiguous asparagine/aspartic-acid\nR  for arginine\nN  for asparagine\nD  for aspartic-acid\nC  for cysteine or cystine or cytosine\nQ  for glutamine\nE  for glutamic-acid\nZ  for ambiguous glutamine/glutamic acid\nG  for glycine or guanine\nH  for histidine\nI  for isoleucine\nL  for leucine\nK  for lysine\nM  for methionine\nF  for phenylalanine\nP  for proline\nS  for serine\nT  for threonine or thymine\nW  for tryptophan\nY  for tyrosine\nV  for valine\nU  for uracil\nO  for water\nX  for other' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref.rcsb_seq_one_letter_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref.pdbx_align_begin',
        [ [ 'item_',
            '_item_description.description',
            '         \n               Beginning index in the chemical sequence from the \n               reference database.' ],
          [ 'item_', '_item.name', '_struct_ref.pdbx_align_begin' ],
          [ 'item_', '_item.category_id', 'struct_ref' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref.rcsb_align_begin' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_ref_seq.pdbx_strand_id',
        [ [ 'item_',
            '_item_description.description',
            '         The PDB strand/chain ID .' ],
          [ 'item_', '_item.name', '_struct_ref_seq.pdbx_strand_id' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq.ndb_chain_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq.pdbx_db_accession',
        [ [ 'item_',
            '_item_description.description',
            '               Accession code of the reference database.' ],
          [ 'item_', '_item.name', '_struct_ref_seq.pdbx_db_accession' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'P07617' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq.ndb_db_accession' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq.pdbx_db_align_beg_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '         \n               Initial insertion code of the sequence segment of the\n               reference database.' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq.pdbx_db_align_beg_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq.ndb_db_align_beg_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq.pdbx_db_align_end_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '         \n               Ending insertion code of the sequence segment of the\n               reference database.' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq.pdbx_db_align_end_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq.ndb_db_align_end_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq.pdbx_PDB_id_code',
        [ [ 'item_',
            '_item_description.description',
            '              The PDB code of the structure.\n  ' ],
          [ 'item_', '_item.name', '_struct_ref_seq.pdbx_PDB_id_code' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '1BBP' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq.ndb_pdb_id_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq.pdbx_auth_seq_align_beg',
        [ [ 'item_',
            '_item_description.description',
            '\n               Initial position in the PDB sequence segment.     ' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq.pdbx_auth_seq_align_beg' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq.ndb_auth_seq_align_beg' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq.pdbx_auth_seq_align_end',
        [ [ 'item_',
            '_item_description.description',
            '\n               Ending position in the PDB sequence segment' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq.pdbx_auth_seq_align_end' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq.ndb_auth_seq_align_end' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_ref_seq.pdbx_seq_align_beg_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '\n               Initial insertion code of the PDB sequence segment.     ' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq.pdbx_seq_align_beg_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq.ndb_seq_align_beg_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq.pdbx_seq_align_end_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '\n               Ending insertion code of the sequence segment' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq.pdbx_seq_align_end_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq.ndb_seq_align_end_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_ref_seq_dif.pdbx_pdb_id_code',
        [ [ 'item_',
            '_item_description.description',
            '\n               The PDB ID code.' ],
          [ 'item_', '_item.name', '_struct_ref_seq_dif.pdbx_pdb_id_code' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq_dif' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '1BBP' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq_dif.ndb_pdb_id_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.pdbx_pdb_strand_id',
        [ [ 'item_',
            '_item_description.description',
            '              PDB strand/chain id.       ' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq_dif.pdbx_pdb_strand_id' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq_dif' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'A', 'B' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq_dif.ndb_pdb_chain_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.pdbx_pdb_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '              Insertion code in PDB sequence\n' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq_dif.pdbx_pdb_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq_dif' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq_dif.ndb_pdb_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.pdbx_auth_seq_num',
        [ [ 'item_',
            '_item_description.description',
            '              Insertion code in PDB sequence\n' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq_dif.pdbx_auth_seq_num' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq_dif' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq_dif.ndb_auth_seq_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.pdbx_seq_db_name',
        [ [ 'item_',
            '_item_description.description',
            '              Sequence database name.\n' ],
          [ 'item_', '_item.name', '_struct_ref_seq_dif.pdbx_seq_db_name' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq_dif' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', 'SWS' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq_dif.ndb_seq_db_name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.pdbx_seq_db_accession_code',
        [ [ 'item_',
            '_item_description.description',
            '              Sequence database accession number.' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq_dif.pdbx_seq_db_accession_code' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq_dif' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', 'P07617' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq_dif.ndb_seq_db_accession_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.pdbx_seq_db_seq_num',
        [ [ 'item_',
            '_item_description.description',
            '              Sequence database sequence number.\n' ],
          [ 'item_',
            '_item.name',
            '_struct_ref_seq_dif.pdbx_seq_db_seq_num' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq_dif' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '142' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq_dif.ndb_seq_db_seq_num' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_sheet_range.pdbx_beg_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta sheet range begins.  Insertion code.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_range.pdbx_beg_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_sheet_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_range.pdbx_beg_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_range.ndb_beg_label_ins_code_pdb' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_range.pdbx_end_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the residue at which the\n               beta sheet range ends. Insertion code.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_range.pdbx_end_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_sheet_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_range.pdbx_end_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_range.ndb_end_label_ins_code_pdb' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_1_beg_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              Pointer to _atom_site.auth_comp_id' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_1_beg_auth_comp_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_1_beg_auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_1_beg_auth_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_1_beg_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '           Pointer to _atom_site.auth_asym_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_1_beg_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_1_beg_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_1_beg_auth_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_1_end_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '           Pointer to _atom_site.auth_comp_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_1_end_auth_comp_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_1_end_auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_1_end_auth_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_1_end_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '           Pointer to _atom_site.auth_comp_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_1_end_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_1_end_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_1_end_auth_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_1_beg_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              Pointer to _atom_site.label_comp_id' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_1_beg_label_comp_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_1_beg_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_1_beg_label_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_1_beg_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              Pointer to _atom_site.label_asym_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_1_beg_label_asym_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_1_beg_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_1_beg_label_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_1_beg_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '  Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_1_beg_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_1_beg_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_1_beg_label_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_1_end_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            ' Pointer to _atom_site.label_comp_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_1_end_label_comp_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_1_end_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_1_end_label_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_1_end_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            ' Pointer to _atom_site.label_asym_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_1_end_label_asym_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_1_end_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_1_end_label_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_1_end_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_1_end_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_1_end_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_1_end_label_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_2_beg_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.label_comp_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_2_beg_label_comp_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_2_beg_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_2_beg_label_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_2_beg_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.label_asym_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_2_beg_label_asym_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_2_beg_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_2_beg_label_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_2_beg_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.pdbx_PDB_ins_code.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_2_beg_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_2_beg_PDB_ins_code' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_ins_code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_2_beg_label_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_2_end_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.label_comp_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_2_end_label_comp_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_2_end_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_2_end_label_comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_2_end_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '   Pointer to _atom_site.label_asym_id.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_2_end_label_asym_id' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_struct_sheet_hbond.pdbx_range_2_end_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_2_end_label_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_sheet_hbond.pdbx_range_2_end_label_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '              Place holder for PDB insertion code.' ],
          [ 'item_',
            '_item.name',
            '_struct_sheet_hbond.pdbx_range_2_end_label_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_sheet_hbond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_sheet_hbond.ndb_range_2_end_label_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_struct_site.pdbx_num_residues',
        [ [ 'item_',
            '_item_description.description',
            '         Number of residues in the site.' ],
          [ 'item_', '_item.name', '_struct_site.pdbx_num_residues' ],
          [ 'item_', '_item.category_id', 'struct_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_site.rcsb_num_residues' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_site_gen.pdbx_auth_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '         PDB insertion code.' ],
          [ 'item_', '_item.name', '_struct_site_gen.pdbx_auth_ins_code' ],
          [ 'item_', '_item.category_id', 'struct_site_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_site_gen.ndb_ins_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_site_gen.pdbx_num_res',
        [ [ 'item_',
            '_item_description.description',
            '         Number of residues in the site.' ],
          [ 'item_', '_item.name', '_struct_site_gen.pdbx_num_res' ],
          [ 'item_', '_item.category_id', 'struct_site_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_site_gen.ndb_num_res' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_symmetry.pdbx_full_space_group_name_H-M',
        [ [ 'item_',
            '_item_description.description',
            '              Used for PDB space group:\n\n               Example: \'C 1 2 1\'  (instead of C 2)\n                        \'P 1 2 1\'  (instead of P 2)\n                        \'P 1 21 1\' (instead of P 21) \n                        \'P 1 1 21\' (instead of P 21 -unique C axis)\n                        \'H 3\'      (instead of R 3   -hexagonal)\n                        \'H 3 2\'    (instead of R 3 2 -hexagonal)\n ' ],
          [ 'item_',
            '_item.name',
            '_symmetry.pdbx_full_space_group_name_H-M' ],
          [ 'item_', '_item.category_id', 'symmetry' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_examples.case',
            '\n  Example: \'C 1 2 1\'  (instead of C 2)\n           \'P 1 2 1\'  (instead of P 2)\n           \'P 1 21 1\' (instead of P 21) \n           \'P 1 1 21\' (instead of P 21 -unique C axis)\n           \'H 3\'      (instead of R 3   -hexagonal)\n           \'H 3 2\'    (instead of R 3 2 -hexagonal)' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_symmetry.ndb_full_space_group_name_H-M' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_refine.pdbx_overall_ESU_R',
        [ [ 'item_',
            '_item_description.description',
            '              Overall estimated standard uncertainties of positional\n               parameters based on R value.' ],
          [ 'item_', '_item.name', '_refine.pdbx_overall_ESU_R' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_overall_ESU_R' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.pdbx_overall_ESU_R_Free',
        [ [ 'item_',
            '_item_description.description',
            '              Overall estimated standard uncertainties of positional\n               parameters based on R value.' ],
          [ 'item_', '_item.name', '_refine.pdbx_overall_ESU_R_Free' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.rcsb_overall_ESU_R_Free' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_refine_tls',
        [ [ 'item_',
            '_category.description',
            '              Data items in the REFINE_TLS category record details about \n               TLS parameters used in structure refinement. Note that the\n               intention is primarily to describe directly refined TLS\n               parameters, although other methods of obtaining TLS parameters\n               may be covered, see item _pdbx_refine_tls.method' ],
          [ 'item_', '_category.id', 'pdbx_refine_tls' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_', [ '_category_key.name' ], [ '_pdbx_refine_tls.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group', 'ccp4_group' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_refine_tls.id must uniquely identify a record in\n               the PDBX_REFINE_TLS list.\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_pdbx_refine_tls_group.refine_tls_id',
              '_pdbx_refine_tls.id' ] ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', 'A' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the TLS group, such as a domain name or a \n               chemical group name.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.details' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Chain A catalytic domain', 'Chain A Tyr 56 side chain' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.details',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.details',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.method',
        [ [ 'item_',
            '_item_description.description',
            '              The method by which the TLS parameters were obtained.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.method' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'refined',
              '                                 TLS parameters refined directly\n                                  against crystallographic residual',
              'fitted',
              '                                 TLS parameters fitted to previously\n                                  refined anisotropic displacement\n                                  parameters' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.method',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.method',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.origin_x',
        [ [ 'item_',
            '_item_description.description',
            '              The x coordinate in angstroms of the origin to which the\n               TLS parameters are referred, specified according to\n               a set of orthogonal Cartesian axes related to the cell axes as\n               given in _atom_sites.Cartn_transform_axes.\n\n               If the origin is omitted, it is assumed to be the centre of\n               reaction of the group, in which case S must be symmetric' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.origin_x' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_pdbx_refine_tls.origin_y', '_pdbx_refine_tls.origin_z' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.origin_x',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.origin_x',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.origin_y',
        [ [ 'item_',
            '_item_description.description',
            '              The y coordinate in angstroms of the origin to which the\n               TLS parameters are referred, specified according to\n               a set of orthogonal Cartesian axes related to the cell axes as\n               given in _atom_sites.Cartn_transform_axes.\n\n               If the origin is omitted, it is assumed to be the centre of\n               reaction of the group, in which case S must be symmetric' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.origin_y' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_pdbx_refine_tls.origin_x', '_pdbx_refine_tls.origin_z' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.origin_y',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.origin_y',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.origin_z',
        [ [ 'item_',
            '_item_description.description',
            '              The z coordinate in angstroms of the origin to which the\n               TLS parameters are referred, specified according to\n               a set of orthogonal Cartesian axes related to the cell axes as\n               given in _atom_sites.Cartn_transform_axes.\n\n               If the origin is omitted, it is assumed to be the centre of\n               reaction of the group, in which case S must be symmetric' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.origin_z' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_pdbx_refine_tls.origin_x', '_pdbx_refine_tls.origin_y' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.origin_z',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.origin_z',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '             The [1][1] element of the translation tensor T. This should\n              be given in the same coordinate frame and units as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[1][1]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[1][1]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[1][1]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[1][1]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[1][1]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.T[1][1].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[1][1]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[1][1]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[1][1]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[1][1]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '             The [1][2] element of the translation tensor T. This should\n              be given in the same coordinate frame and units as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[1][2]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[1][2]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[1][2]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[1][2]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[1][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.T[1][2].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[1][2]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[1][2]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[1][2]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[1][2]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '             The [1][3] element of the translation tensor T. This should\n              be given in the same coordinate frame and units as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[1][3]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[1][3]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[1][3]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[1][3]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[1][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.T[1][3].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[1][3]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[1][3]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[1][3]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[1][3]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '             The [2][2] element of the translation tensor T. This should\n              be given in the same coordinate frame and units as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[2][2]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[2][2]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[2][2]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[2][2]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[2][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.T[2][2].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[2][2]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[2][2]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[2][2]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[2][2]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '             The [2][3] element of the translation tensor T. This should\n              be given in the same coordinate frame and units as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[2][3]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[2][3]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[2][3]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[2][3]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[2][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.T[2][3].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[2][3]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[2][3]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[2][3]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[2][3]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '             The [3][3] element of the translation tensor T. This should\n              be given in the same coordinate frame and units as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[3][3]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[3][3]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[3][3]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[3][3]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.T[3][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.T[3][3].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.T[3][3]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.T[3][3]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.T[3][3]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.T[3][3]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '             The [1][1] element of the libration tensor L. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[1][1]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[1][1]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[1][1]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[1][1]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[1][1]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.L[1][1].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[1][1]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[1][1]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[1][1]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[1][1]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '             The [1][2] element of the libration tensor L. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[1][2]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[1][2]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[1][2]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[1][2]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[1][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.L[1][2].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[1][2]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[1][2]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[1][2]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[1][2]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '             The [1][3] element of the libration tensor L. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[1][3]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[1][3]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[1][3]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[1][3]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[1][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.L[1][3].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[1][3]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[1][3]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[1][3]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[1][3]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '             The [2][2] element of the libration tensor L. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[2][2]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[2][2]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[2][2]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[2][2]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[2][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.L[2][2].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[2][2]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[2][2]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[2][2]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[2][2]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '             The [2][3] element of the libration tensor L. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[2][3]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[2][3]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[2][3]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[2][3]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[2][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.L[2][3].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[2][3]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[2][3]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[2][3]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[2][3]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '             The [3][3] element of the libration tensor L. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[3][3]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[3][3]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[3][3]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[3][3]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.L[3][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.L[3][3].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.L[3][3]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.L[3][3]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'degrees_squared' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.L[3][3]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.L[3][3]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[1][1]',
        [ [ 'item_',
            '_item_description.description',
            '             The [1][1] element of the screw-rotation tensor S. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.\n\n              The trace of S is indeterminate by crystallography, and should\n              be set to zero.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[1][1]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[1][1]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[1][1]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[1][1]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[1][1]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.S[1][1].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[1][1]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[1][1]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[1][1]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[1][1]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[1][2]',
        [ [ 'item_',
            '_item_description.description',
            '             The [1][2] element of the screw-rotation tensor S. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.\n\n               If the origin is omitted, it is assumed to be the centre of\n               reaction of the group, in which case S must be symmetric' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[1][2]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[1][2]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[1][2]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[1][2]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[1][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.S[1][2].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[1][2]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[1][2]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[1][2]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[1][2]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[1][3]',
        [ [ 'item_',
            '_item_description.description',
            '             The [1][3] element of the screw-rotation tensor S. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.\n\n               If the origin is omitted, it is assumed to be the centre of\n               reaction of the group, in which case S must be symmetric' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[1][3]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[1][3]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[1][3]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[1][3]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[1][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.S[1][3].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[1][3]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[1][3]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[1][3]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[1][3]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[2][1]',
        [ [ 'item_',
            '_item_description.description',
            '             The [2][1] element of the screw-rotation tensor S. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.\n\n               If the origin is omitted, it is assumed to be the centre of\n               reaction of the group, in which case S must be symmetric' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[2][1]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[2][1]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[2][1]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[2][1]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[2][1]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.S[2][1].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[2][1]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[2][1]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[2][1]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[2][1]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[2][2]',
        [ [ 'item_',
            '_item_description.description',
            '             The [2][2] element of the screw-rotation tensor S. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.\n\n              The trace of S is indeterminate by crystallography, and should\n              be set to zero.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[2][2]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[2][2]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[2][2]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[2][2]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[2][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.S[2][2].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[2][2]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[2][2]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[2][2]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[2][2]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[2][3]',
        [ [ 'item_',
            '_item_description.description',
            '             The [2][3] element of the screw-rotation tensor S. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.\n\n               If the origin is omitted, it is assumed to be the centre of\n               reaction of the group, in which case S must be symmetric' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[2][3]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[2][3]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[2][3]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[2][3]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[2][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.S[2][3].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[2][3]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[2][3]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[2][3]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[2][3]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[3][1]',
        [ [ 'item_',
            '_item_description.description',
            '             The [3][1] element of the screw-rotation tensor S. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.\n\n               If the origin is omitted, it is assumed to be the centre of\n               reaction of the group, in which case S must be symmetric' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[3][1]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[3][1]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[3][1]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[3][1]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[3][1]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.S[3][1].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[3][1]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[3][1]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[3][1]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[3][1]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[3][2]',
        [ [ 'item_',
            '_item_description.description',
            '             The [3][2] element of the screw-rotation tensor S. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.\n\n               If the origin is omitted, it is assumed to be the centre of\n               reaction of the group, in which case S must be symmetric' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[3][2]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[3][2]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[3][2]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[3][2]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[3][2]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.S[3][2].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[3][2]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[3][2]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[3][2]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[3][2]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[3][3]',
        [ [ 'item_',
            '_item_description.description',
            '             The [3][3] element of the screw-rotation tensor S. This should\n              be given in the same coordinate frame as the\n              corresponding anisotropic displacement parameters.\n\n              The trace of S is indeterminate by crystallography, and should\n              be set to zero.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[3][3]' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[3][3]_esd', 'associated_esd' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'item_', '_item_type_conditions.code', 'esd' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[3][3]',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[3][3]',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls.S[3][3]_esd',
        [ [ 'item_',
            '_item_description.description',
            '              The estimated standard deviation of _pdbx_refine_tls.S[3][3].' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls.S[3][3]_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_default.value', '0.0' ],
          [ 'loop_',
            [ '_item_related.related_name', '_item_related.function_code' ],
            [ '_pdbx_refine_tls.S[3][3]', 'associated_value' ] ],
          [ 'item_', '_item_units.code', 'angstroms_degrees' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls.S[3][3]_esd',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls.S[3][3]_esd',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        'pdbx_refine_tls_group',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_REFINE_TLS_GROUP category record details about\n               a fragment of a TLS group.\n\n               Properties of the TLS group are recorded in PDBX_REFINE_TLS' ],
          [ 'item_', '_category.id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_refine_tls_group.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'refine_group', 'ccp4_group' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_refine_tls_group.id must uniquely identify \n               a record in the REFINE_TLS_GROUP list.\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls_group.id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', 'A' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.refine_tls_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _pdbx_refine_tls.id in the\n               REFINE_TLS category.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls_group.refine_tls_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.refine_tls_id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.refine_tls_id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.beg_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '             A component of the identifier for the residue at which the\n              TLS fragment range begins.\n\n              This data item is a pointer to _atom_site.label_asym_id in the\n              ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_tls_group.beg_label_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_pdbx_refine_tls_group.beg_label_asym_id',
              '_atom_site.label_asym_id' ] ],
          [ 'loop_', [ '_item_examples.case' ], [ 'O', '2B3' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.beg_label_asym_id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.beg_label_asym_id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.beg_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '             A component of the identifier for the residue at which the\n              TLS fragment range begins.\n\n              This data item is a pointer to _atom_site.label_seq_id in the\n              ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_tls_group.beg_label_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_pdbx_refine_tls_group.beg_label_seq_id',
              '_atom_site.label_seq_id' ] ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '303' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.beg_label_seq_id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.beg_label_seq_id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.beg_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '             A component of the identifier for the residue at which the\n              TLS fragment range begins.\n\n              This data item is a pointer to _atom_site.auth_asym_id in the\n              ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_tls_group.beg_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_pdbx_refine_tls_group.beg_auth_asym_id',
              '_atom_site.auth_asym_id' ] ],
          [ 'loop_', [ '_item_examples.case' ], [ 'O', '2B3' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.beg_auth_asym_id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.beg_auth_asym_id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.beg_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '             A component of the identifier for the residue at which the\n              TLS fragment range begins.\n\n              This data item is a pointer to _atom_site.auth_seq_id in the\n              ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_tls_group.beg_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_pdbx_refine_tls_group.beg_auth_seq_id',
              '_atom_site.auth_seq_id' ] ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '5A' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.beg_auth_asym_id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.beg_auth_asym_id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.end_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '             A component of the identifier for the residue at which the\n              TLS fragment range ends.\n\n              This data item is a pointer to _atom_site.label_asym_id in the\n              ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_tls_group.end_label_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_pdbx_refine_tls_group.end_label_asym_id',
              '_atom_site.label_asym_id' ] ],
          [ 'loop_', [ '_item_examples.case' ], [ 'O', '2B3' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.end_label_asym_id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.end_label_asym_id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.end_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '             A component of the identifier for the residue at which the\n              TLS fragment range ends.\n\n              This data item is a pointer to _atom_site.label_seq_id in the\n              ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_tls_group.end_label_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_pdbx_refine_tls_group.end_label_seq_id',
              '_atom_site.label_seq_id' ] ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '303' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.end_label_seq_id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.end_label_seq_id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.end_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '             A component of the identifier for the residue at which the\n              TLS fragment range ends.\n\n              This data item is a pointer to _atom_site.auth_asym_id in the\n              ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_tls_group.end_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_pdbx_refine_tls_group.end_auth_asym_id',
              '_atom_site.auth_asym_id' ] ],
          [ 'loop_', [ '_item_examples.case' ], [ 'O', '2B3' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.end_auth_asym_id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.end_auth_asym_id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.end_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '             A component of the identifier for the residue at which the\n              TLS fragment range ends.\n\n              This data item is a pointer to _atom_site.auth_seq_id in the\n              ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_tls_group.end_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_pdbx_refine_tls_group.end_auth_seq_id',
              '_atom_site.auth_seq_id' ] ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '5A' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.end_auth_seq_id',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.end_auth_seq_id',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_pdbx_refine_tls_group.selection',
        [ [ 'item_',
            '_item_description.description',
            '              A description of the subset of atoms in the specified\n               range included in the TLS fragment.' ],
          [ 'item_', '_item.name', '_pdbx_refine_tls_group.selection' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_tls_group' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'all',
              '                                 all atoms in specified range',
              'mnc',
              '                                 main chain atoms only',
              'sdc',
              '                                 side chain atoms only' ] ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_ccp4_refine_tls_group.selection',
              'cif_rcsb.dic',
              '1.1',
              '_ccp4_refine_tls_group.selection',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_refine.pdbx_solvent_vdw_probe_radii',
        [ [ 'item_',
            '_item_description.description',
            '              CCP4 solvent proble van der Waals radii ' ],
          [ 'item_', '_item.name', '_refine.pdbx_solvent_vdw_probe_radii' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine.ccp4_solvent_vdw_probe_radii',
              'cif_rcsb.dic',
              '1.1',
              '_refine.ccp4_solvent_vdw_probe_radii',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_refine.pdbx_solvent_ion_probe_radii',
        [ [ 'item_',
            '_item_description.description',
            '              CCP4 solvent ion proble radii ' ],
          [ 'item_', '_item.name', '_refine.pdbx_solvent_ion_probe_radii' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine.ccp4_solvent_ion_probe_radii',
              'cif_rcsb.dic',
              '1.1',
              '_refine.ccp4_solvent_ion_probe_radii',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ 'save_',
        '_refine.pdbx_solvent_shrinkage_radii',
        [ [ 'item_',
            '_item_description.description',
            '              CCP4 solvent shrinkage radii ' ],
          [ 'item_', '_item.name', '_refine.pdbx_solvent_shrinkage_radii' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_',
            [ '_item_aliases.alias_name',
              '_item_aliases.dictionary',
              '_item_aliases.version' ],
            [ '_refine.ccp4_solvent_shrinkage_radii',
              'cif_rcsb.dic',
              '1.1',
              '_refine.ccp4_solvent_shrinkage_radii',
              'cif_ccp4.dic',
              '1.0' ] ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# PDBX_CONTACT_AUTHOR ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'pdbx_contact_author',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_CONTACT_AUTHOR category record details\n               about the name and address of the author to be contacted\n               concerning the contents of this data block.  This category atomizes\n               information to a greater degree than the standard AUDIT_CONTACT_AUTHOR\n               category.' ],
          [ 'item_', '_category.id', 'pdbx_contact_author' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_contact_author.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'audit_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n    _pdbx_contact_author.id                1\n    _pdbx_contact_author.name_salutation   \'Dr.\'\n    _pdbx_contact_author.name_first        \'Paula\'\n    _pdbx_contact_author.name_last         \'Fitzgerald\'\n    _pdbx_contact_author.name_mi           \'M.D.\'\n    _pdbx_contact_author.address_1         \'Department of Biophysical Chemistry\'\n    _pdbx_contact_author.address_2         \'Merck Research Laboratories\'\n    _pdbx_contact_author.address_3         \'P. O. Box 2000, Ry80M203\'\n    _pdbx_contact_author.city              \'Rahway\'\n    _pdbx_contact_author.state_province    \'New Jersey\'\n    _pdbx_contact_author.postal_code       07065\n    _pdbx_contact_author.country           \'UNITED STATES\'\n    _pdbx_contact_author.phone             \'908 594 5510\'\n    _pdbx_contact_author.fax               \'908 594 6645\'\n    _pdbx_contact_author.email             \'paula_fitzgerald@merck.com\'\n    _pdbx_contact_author.role              \'principal investigator/group leader\'    \n    _pdbx_contact_author.organization_type \'commercial\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.id',
        [ [ 'item_',
            '_item_description.description',
            '          A unique integer identifier for this author    ' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.id' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_examples.case',
            '                            1 2 3' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ '1', '2', '3', '4', '5', '6', '7', '8', '9', '10' ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.address_1',
        [ [ 'item_',
            '_item_description.description',
            '              The mailing address of the author of the data block to whom\n               correspondence should be addressed, line 1 of 3.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.address_1' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                            610 Taylor Road' ] ] ],
      [ 'save_',
        '_pdbx_contact_author.address_2',
        [ [ 'item_',
            '_item_description.description',
            '              The mailing address of the author of the data block to whom\n               correspondence should be addressed, line 2 of 3.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.address_2' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                             Department of Chemistry and Chemical Biology' ] ] ],
      [ 'save_',
        '_pdbx_contact_author.address_3',
        [ [ 'item_',
            '_item_description.description',
            '              The mailing address of the author of the data block to whom\n               correspondence should be addressed, line 3 of 3.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.address_3' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                             Busch Campus' ] ] ],
      [ 'save_',
        '_pdbx_contact_author.city',
        [ [ 'item_',
            '_item_description.description',
            '              The mailing address of the author of the data block to whom\n               correspondence should be addressed, city.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.city' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_examples.case',
            '                             Piscataway' ] ] ],
      [ 'save_',
        '_pdbx_contact_author.state_province',
        [ [ 'item_',
            '_item_description.description',
            '              The mailing address of the author of the data block to whom\n               correspondence should be addressed, state or province.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.state_province' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_examples.case',
            '                             New Jersey' ] ] ],
      [ 'save_',
        '_pdbx_contact_author.postal_code',
        [ [ 'item_',
            '_item_description.description',
            '              The mailing address of the author of the data block to whom\n               correspondence should be addressed, zip code.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.postal_code' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_examples.case',
            '                             08854' ] ] ],
      [ 'save_',
        '_pdbx_contact_author.email',
        [ [ 'item_',
            '_item_description.description',
            '              The electronic mail address of the author of the data block to\n               whom correspondence should be addressed, in a form recognisable\n               to international networks.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.email' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'name@host.domain.country', 'bm@iucr.ac.uk' ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.fax',
        [ [ 'item_',
            '_item_description.description',
            '              The facsimile telephone number of the author of the data\n               block to whom correspondence should be addressed.\n\n               The recommended style includes the international dialing\n               prefix, the area code in parentheses, followed by the\n               local number with no spaces. ' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.fax' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '12(34) 947 7334', '732 445 0103' ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.name_first',
        [ [ 'item_',
            '_item_description.description',
            '              The first name of the author of the data block to whom correspondence\n               should be addressed.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.name_first' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Percival', 'Loyd', 'Susan' ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.name_last',
        [ [ 'item_',
            '_item_description.description',
            '              The last name of the author of the data block to whom correspondence\n               should be addressed.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.name_last' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'Samuels', 'Rodgers' ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.name_mi',
        [ [ 'item_',
            '_item_description.description',
            '              The middle initial(s) of the author of the data block to whom correspondence\n               should be addressed.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.name_mi' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'T.', 'M.F.' ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.name_salutation',
        [ [ 'item_',
            '_item_description.description',
            '              The salutation of the author of the data block to whom correspondence\n               should be addressed.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.name_salutation' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Dr.', 'Prof.', 'Mr.', 'Ms.', 'Mrs.' ] ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'Dr.', 'Prof.', 'Mr.', 'Ms.', 'Mrs.' ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.country',
        [ [ 'item_',
            '_item_description.description',
            '              The country of the author of the data block to whom\n               correspondence should be addressed.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.country' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'UNITED STATES', 'UNITED KINGDOM', 'AUSTRALIA' ] ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'UNITED STATES',
              'AFGHANISTAN',
              'ALBANIA',
              'ALGERIA',
              'AMERICAN SAMOA',
              'ANDORRA',
              'ANGOLA',
              'ANGUILLA',
              'ANTARCTICA',
              'ANTIGUA AND BARBUDA',
              'ARGENTINA',
              'ARMENIA',
              'ARUBA',
              'AUSTRALIA',
              'AUSTRIA',
              'AZERBAIJAN',
              'BAHAMAS',
              'BAHRAIN',
              'BANGLADESH',
              'BARBADOS',
              'BELARUS',
              'BELGIUM',
              'BELIZE',
              'BENIN',
              'BERMUDA',
              'BHUTAN',
              'BOLIVIA',
              'BOSNIA AND HERZEGOVINA',
              'BOTSWANA',
              'BOUVET ISLAND',
              'BRAZIL',
              'BRITISH INDIAN OCEAN TERRITORY',
              'BRUNEI DARUSSALAM',
              'BULGARIA',
              'BURKINA FASO',
              'BURUNDI',
              'CAMBODIA',
              'CAMEROON',
              'CANADA',
              'CAPE VERDE',
              'CAYMAN ISLANDS',
              'CENTRAL AFRICAN REPUBLIC',
              'CHAD',
              'CHILE',
              'CHINA',
              'CHRISTMAS ISLAND',
              'COCOS (KEELING) ISLANDS',
              'COLOMBIA',
              'COMOROS',
              'CONGO',
              'CONGO, THE DEMOCRATIC REPUBLIC OF THE',
              'COOK ISLANDS',
              'COSTA RICA',
              'COTE D\'IVOIRE',
              'CROATIA',
              'CUBA',
              'CYPRUS',
              'CZECH REPUBLIC',
              'DENMARK',
              'DJIBOUTI',
              'DOMINICA',
              'DOMINICAN REPUBLIC',
              'v\'ECUADOR\'',
              'EGYPT',
              'EL SALVADOR',
              'EQUATORIAL GUINEA',
              'ERITREA',
              'ESTONIA',
              'ETHIOPIA',
              'FALKLAND ISLANDS (MALVINAS)',
              'FAROE ISLANDS',
              'FIJI',
              'FINLAND',
              'FRANCE',
              'FRENCH GUIANA',
              'FRENCH POLYNESIA',
              'FRENCH SOUTHERN TERRITORIES',
              'GABON',
              'GAMBIA',
              'GEORGIA',
              'GERMANY',
              'GHANA',
              'GIBRALTAR',
              'GREECE',
              'GREENLAND',
              'GRENADA',
              'GUADELOUPE',
              'GUAM',
              'GUATEMALA',
              'GUINEA',
              'GUINEA-BISSAU',
              'GUYANA',
              'HAITI',
              'HEARD ISLAND AND MCDONALD ISLANDS',
              'HOLY SEE (VATICAN CITY STATE)',
              'HONDURAS',
              'HONG KONG',
              'HUNGARY',
              'ICELAND',
              'INDIA',
              'INDONESIA',
              'IRAN, ISLAMIC REPUBLIC OF',
              'IRAQ',
              'IRELAND',
              'ISRAEL',
              'ITALY',
              'JAMAICA',
              'JAPAN',
              'JORDAN',
              'KAZAKHSTAN',
              'KENYA',
              'KIRIBATI',
              'KOREA, DEMOCRATIC PEOPLE\'S REPUBLIC OF',
              'KOREA, REPUBLIC OF',
              'KUWAIT',
              'KYRGYZSTAN',
              'LAO PEOPLE\'S DEMOCRATIC REPUBLIC',
              'LATVIA',
              'LEBANON',
              'LESOTHO',
              'LIBERIA',
              'LIBYAN ARAB JAMAHIRIYA',
              'LIECHTENSTEIN',
              'LITHUANIA',
              'LUXEMBOURG',
              'MACAO',
              'MACEDONIA, THE FORMER YUGOSLAV REPUBLIC OF',
              'MADAGASCAR',
              'MALAWI',
              'MALAYSIA',
              'MALDIVES',
              'MALI',
              'MALTA',
              'MARSHALL ISLANDS',
              'MARTINIQUE',
              'MAURITANIA',
              'MAURITIUS',
              'MAYOTTE',
              'MEXICO',
              'MICRONESIA, FEDERATED STATES OF',
              'MOLDOVA, REPUBLIC OF',
              'MONACO',
              'MONGOLIA',
              'MONTSERRAT',
              'MOROCCO',
              'MOZAMBIQUE',
              'MYANMAR',
              'NAMIBIA',
              'NAURU',
              'NEPAL',
              'NETHERLANDS',
              'NETHERLANDS ANTILLES',
              'NEW CALEDONIA',
              'NEW ZEALAND',
              'NICARAGUA',
              'NIGER',
              'NIGERIA',
              'NIUE',
              'NORFOLK ISLAND',
              'NORTHERN MARIANA ISLANDS',
              'NORWAY',
              'OMAN',
              'PAKISTAN',
              'PALAU',
              'PALESTINIAN TERRITORY, OCCUPIED',
              'PANAMA',
              'PAPUA NEW GUINEA',
              'PARAGUAY',
              'PERU',
              'PHILIPPINES',
              'PITCAIRN',
              'POLAND',
              'PORTUGAL',
              'PUERTO RICO',
              'QATAR',
              'REUNION',
              'ROMANIA',
              'RUSSIAN FEDERATION',
              'RWANDA',
              'SAINT HELENA',
              'SAINT KITTS AND NEVIS',
              'SAINT LUCIA',
              'SAINT PIERRE AND MIQUELON',
              'SAINT VINCENT AND THE GRENADINES',
              'SAMOA',
              'SAN MARINO',
              'SAO TOME AND PRINCIPE',
              'SAUDI ARABIA',
              'SENEGAL',
              'SERBIA AND MONTENEGRO',
              'SEYCHELLES',
              'SIERRA LEONE',
              'SINGAPORE',
              'SLOVAKIA',
              'SLOVENIA',
              'SOLOMON ISLANDS',
              'SOMALIA',
              'SOUTH AFRICA',
              'SOUTH GEORGIA AND THE SOUTH SANDWICH ISLANDS',
              'SPAIN',
              'SRI LANKA',
              'SUDAN',
              'SURINAME',
              'SVALBARD AND JAN MAYEN',
              'SWAZILAND',
              'SWEDEN',
              'SWITZERLAND',
              'SYRIAN ARAB REPUBLIC',
              'TAIWAN, PROVINCE OF CHINA',
              'TAJIKISTAN',
              'TANZANIA, UNITED REPUBLIC OF',
              'THAILAND',
              'TIMOR-LESTE',
              'TOGO',
              'TOKELAU',
              'TONGA',
              'TRINIDAD AND TOBAGO',
              'TUNISIA',
              'TURKEY',
              'TURKMENISTAN',
              'TURKS AND CAICOS ISLANDS',
              'TUVALU',
              'UGANDA',
              'UKRAINE',
              'UNITED ARAB EMIRATES',
              'UNITED KINGDOM',
              'UNITED STATES MINOR OUTLYING ISLANDS',
              'URUGUAY',
              'UZBEKISTAN',
              'VANUATU',
              'VENEZUELA',
              'VIET NAM',
              'VIRGIN ISLANDS, BRITISH',
              'VIRGIN ISLANDS, U.S.',
              'WALLIS AND FUTUNA',
              'WESTERN SAHARA',
              'YEMEN',
              'ZAMBIA',
              'ZIMBABWE' ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.phone',
        [ [ 'item_',
            '_item_description.description',
            '              The telephone number of the author of the data block to whom\n               correspondence should be addressed.\n\n               The recommended style includes the international dialing\n               prefix, the area code in parentheses, followed by the\n               local number and any extension number prefixed by \'x\',\n               with no spaces. The earlier convention of including\n               the international dialing prefixes in parentheses is no longer\n               recommended.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.phone' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '12 (34) 947 7330', '947 732 0103 x8320' ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.role',
        [ [ 'item_',
            '_item_description.description',
            '        The role of this author in the project depositing this data.' ],
          [ 'item_', '_item.name', '_pdbx_contact_author.role' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'principal investigator/group leader',
              'responsible scientist',
              'investigator' ] ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'principal investigator/group leader',
              'responsible scientist',
              'investigator' ] ] ] ],
      [ 'save_',
        '_pdbx_contact_author.organization_type',
        [ [ 'item_',
            '_item_description.description',
            '        The organization type to which this author is affiliated.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_contact_author.organization_type' ],
          [ 'item_', '_item.category_id', 'pdbx_contact_author' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'academic', 'commercial', 'government', 'other' ] ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'academic', 'commercial', 'government', 'other' ] ] ] ],
      [ 'save_',
        '_struct_ref_seq_dif.pdbx_ordinal',
        [ [ 'item_',
            '_item_description.description',
            '       A synthetic integer primary key for this category.' ],
          [ 'item_', '_item.name', '_struct_ref_seq_dif.pdbx_ordinal' ],
          [ 'item_', '_item.category_id', 'struct_ref_seq_dif' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ref_seq_dif.rcsb_ordinal' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# PDBX_SG_PROJECT ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'pdbx_SG_project',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_CONTACT_AUTHOR category record details\n               about the Structural Genomics Project and name and initials\n               for each Center.' ],
          [ 'item_', '_category.id', 'pdbx_SG_project' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_SG_project.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'audit_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 -',
              '\n    _pdbx_SG_project.id                  1\n    _pdbx_SG_project.project_name        \'PSI, Protein Structure Initiative\'\n    _pdbx_SG_project.full_name_of_center \'Berkeley Structural Genomics Center\'\n    _pdbx_SG_project.initial_of_center   BSGC',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_SG_project.id',
        [ [ 'item_',
            '_item_description.description',
            '          A unique integer identifier for this center' ],
          [ 'item_', '_item.name', '_pdbx_SG_project.id' ],
          [ 'item_', '_item.category_id', 'pdbx_SG_project' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '                      1 2 3' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ '1', '2', '3', '4', '5', '6', '7', '8', '9', '10' ] ] ] ],
      [ 'save_',
        '_pdbx_SG_project.project_name',
        [ [ 'item_',
            '_item_description.description',
            '          The value identifies the Structural Genomics project.' ],
          [ 'item_', '_item.name', '_pdbx_SG_project.project_name' ],
          [ 'item_', '_item.category_id', 'pdbx_SG_project' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'PSI, Protein Structure Initiative' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'PSI, Protein Structure Initiative',
              'NPPSFA, National Project on Protein Structural and Functional Analyses' ] ] ] ],
      [ 'save_',
        '_pdbx_SG_project.full_name_of_center',
        [ [ 'item_',
            '_item_description.description',
            '          The value identifies the full name of center.' ],
          [ 'item_', '_item.name', '_pdbx_SG_project.full_name_of_center' ],
          [ 'item_', '_item.category_id', 'pdbx_SG_project' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'Berkeley Structural Genomics Center' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'Bacterial targets at IGS-CNRS France',
              'Berkeley Structural Genomics Center',
              'Center for Eukaryotic Structural Genomics',
              'Israel Structural Proteomics Center',
              'Joint Center for Structural Genomics',
              'Marseilles Structural Genomics Program @ AFMB',
              'Midwest Center for Structural Genomics',
              'Montreal-Kingston Bacterial Structural Genomics Initiative',
              'Mycobacterium Tuberculosis Structural Proteomics Project',
              'New York Structural Genomics Research Consortium',
              'Northeast Structural Genomics Consortium',
              'Oxford Protein Production Facility',
              'Paris-Sud Yeast Structural Genomics',
              'Protein Structure Factory',
              'RIKEN Structural Genomics/Proteomics Initiative',
              'Southeast Collaboratory for Structural Genomics',
              'Structural Genomics Consortium',
              'Structural Genomics of Pathogenic Protozoa Consortium',
              'Structural Proteomics in Europe',
              'Structure 2 Function Project',
              'TB Structural Genomics Consortium' ] ] ] ],
      [ 'save_',
        '_pdbx_SG_project.initial_of_center',
        [ [ 'item_',
            '_item_description.description',
            '          The value identifies the full name of center.' ],
          [ 'item_', '_item.name', '_pdbx_SG_project.initial_of_center' ],
          [ 'item_', '_item.category_id', 'pdbx_SG_project' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'BSGC' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'BIGS',
              'BSGC',
              'CESG',
              'ISPC',
              'JCSG',
              'MSGP',
              'MCSG',
              'BSGI',
              'XMTB',
              'NYSGRC',
              'NESG',
              'OPPF',
              'YSG',
              'PSF',
              'RSGI',
              'SECSG',
              'SGC',
              'SGPP',
              'SPINE',
              'S2F',
              'TBSGC' ] ] ] ],
      [ '@comment', '##  EOF mmcif_pdbx-def-1.dic' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment',
        ' File:  mmcif_pdbx-def-3.dic                                             ' ],
      [ '@comment', ' ' ],
      [ '@comment',
        '                      PDB Exchange Data Dictionary' ],
      [ '@comment',
        ' This data dictionary contains definitions used by wwPDB for data exchange' ],
      [ '@comment', ' and data processing.  ' ],
      [ '@comment',
        '                           Definition Section 3' ],
      [ '@comment', '                            ' ],
      [ '@comment',
        '                                                                        ' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment', '#' ],
      [ '@comment', '# ' ],
      [ 'save_',
        'pdbx_nmr_details',
        [ [ 'item_',
            '_category.description',
            '\nExperimental details of the NMR study that have not been \ndescribed elsewhere in this deposition.' ],
          [ 'item_', '_category.id', 'pdbx_nmr_details' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_nmr_details.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_details.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '  The entry ID for the structure determination.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_details.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_details' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_nmr_details.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_details.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_details.text',
        [ [ 'item_',
            '_item_description.description',
            '   Additional details describing the NMR experiment.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_details.text' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_details' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '\n This structure was determined using standard 2D homonuclear techniques.',
              '\n The structure was determined using triple-resonance NMR spectroscopy.' ] ],
          [ 'item_', '_item_aliases.alias_name', '_rcsb_nmr_details.text' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment',
        '# provided to hold a simple solution description.' ],
      [ 'save_',
        'pdbx_nmr_sample_details',
        [ [ 'item_',
            '_category.description',
            '   \nComplete description of each NMR sample, including the solvent\nsystem used.' ],
          [ 'item_', '_category.id', 'pdbx_nmr_sample_details' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_nmr_sample_details.solution_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ ' \nThis example was taken from the study of MCP-1 which is a dimer under the \nconditions studied.  Three solutions with different isotope compositions were \nstudied.\n ',
              '\n loop_\n _pdbx_nmr_sample_details.solution_id\n _pdbx_nmr_sample_details.solvent_system\n _pdbx_nmr_sample_details.contents\n 1 MCP-1 \'2 mM U-15N,13C, H2O 90 %, D2O 10 %\'\n 2 MCP-1 \'1 mM U-50% 15N, MCP-1 1 mM U-50% 13C, H2O 90 %, D2O 10 %\'\n 3 MCP-1 \'2 mM U-15N, H2O 90 %, D2O 10 %\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_sample_details.solution_id',
        [ [ 'item_',
            '_item_description.description',
            '   \nThe name (number) of the sample.     ' ],
          [ 'item_', '_item.name', '_pdbx_nmr_sample_details.solution_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_sample_details' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_sample_details.solution_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_sample_details.contents',
        [ [ 'item_',
            '_item_description.description',
            '   \nA complete description of each NMR sample. Include the concentration \nand concentration units for each component (include buffers, etc.). For each\ncomponent describe the isotopic composition, including the % labeling level,\nif known. \n\nFor example:\n\n1. Uniform (random) labeling with 15N: U-15N\n\n2. Uniform (random) labeling with 13C, 15N at known labeling\n   levels: U-95% 13C;U-98% 15N\n\n3. Residue selective labeling: U-95% 15N-Thymine\n\n4. Site specific labeling: 95% 13C-Ala18,\n\n5. Natural abundance labeling in an otherwise uniformly labeled\n   biomolecule is designated by NA: U-13C; NA-K,H\n  ' ],
          [ 'item_', '_item.name', '_pdbx_nmr_sample_details.contents' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_sample_details' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '  \n2mM Ribonuclease  U-15N,13C; 50mM phosphate buffer NA; 90% H2O, 10% D2O ' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_sample_details.contents' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_sample_details.solvent_system',
        [ [ 'item_',
            '_item_description.description',
            '   \nThe solvent system used for this sample.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_sample_details.solvent_system' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_sample_details' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', '  \n90% H2O, 10% D2O ' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_sample_details.solvent_system' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        'pdbx_nmr_exptl_sample',
        [ [ 'item_',
            '_category.description',
            '   The chemical constituents of\n    each NMR sample. Each sample is identified by a number and \n    each component in the sample is identified by name.   ' ],
          [ 'item_', '_category.id', 'pdbx_nmr_exptl_sample' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_nmr_exptl_sample.solution_id',
              '_pdbx_nmr_exptl_sample.component' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ ' \n  Example 1 \n\n  This example was taken from the study of MCP-1 which is a dimer under the \n  conditions studied. Three solutions with different isotope compositions were \n  studied.',
              ' \n loop_\n _pdbx_nmr_exptl_sample.solution_id\n _pdbx_nmr_exptl_sample.component\n _pdbx_nmr_exptl_sample.concentration\n _pdbx_nmr_exptl_sample.concentration_units\n _pdbx_nmr_exptl_sample.isotopic_labeling\n 1 MCP-1  2 \'mM\' \'U-15N,13C\'\n 1 H2O   90 \'%\'   .\n 1 D2O   10 \'%\'   .',
              ' \n  Example 2\n\n  This example was taken from the study of MCP-1 which is a dimer under the \n  conditions studied. Three solutions with different isotope compositions were \n  studied.',
              ' \n loop_\n _pdbx_nmr_exptl_sample.solution_id\n _pdbx_nmr_exptl_sample.component\n _pdbx_nmr_exptl_sample.concentration\n _pdbx_nmr_exptl_sample.concentration_units\n _pdbx_nmr_exptl_sample.isotopic_labeling\n 2 MCP-1 1 \'mM\' \'U-50% 15N\'\n 2 H2O  90 \'%\'  .\n 2 D2O  10 \'%\'  .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample.solution_id',
        [ [ 'item_',
            '_item_description.description',
            '   The name (number) of the sample.     ' ],
          [ 'item_', '_item.name', '_pdbx_nmr_exptl_sample.solution_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_exptl_sample' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample.solution_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample.component',
        [ [ 'item_',
            '_item_description.description',
            '   The name of each component in the sample   ' ],
          [ 'item_', '_item.name', '_pdbx_nmr_exptl_sample.component' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_exptl_sample' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'ribonuclease',
              'DNA strand 1',
              'TRIS buffer',
              'sodium chloride',
              'H2O',
              'D2O' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample.component' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample.concentration',
        [ [ 'item_',
            '_item_description.description',
            '   The concentration value of the component.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_exptl_sample.concentration' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_exptl_sample' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '2.0', '2.7', '0.01' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample.concentration' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample.concentration_units',
        [ [ 'item_',
            '_item_description.description',
            '   The concentration units of the component.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_exptl_sample.concentration_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_exptl_sample' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'mg/mL for mg per milliliter',
              'mM for millimolar',
              '% for percent by volume' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '%',
              'percent by volume',
              'mM',
              'millimolar',
              'mg/mL',
              'mg per milliliter',
              'M',
              'molar',
              'g/L',
              'grams per liter' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample.concentration_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample.isotopic_labeling',
        [ [ 'item_',
            '_item_description.description',
            '   The isotopic composition of each component, including\n    the % labeling level, if known. For example:\n\n    1. Uniform (random) labeling with 15N: U-15N\n\n    2. Uniform (random) labeling with 13C, 15N at known labeling\n     levels: U-95% 13C;U-98% 15N\n\n    3. Residue selective labeling: U-95% 15N-Thymine\n\n    4. Site specific labeling: 95% 13C-Ala18,\n\n    5. Natural abundance labeling in an otherwise uniformly labled\n      biomolecule is designated by NA: U-13C; NA-K,H ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_exptl_sample.isotopic_labeling' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_exptl_sample' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'U-13C,15N', 'U-2H' ] ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'U-15N', 'U-13C', 'U-15N,13C', 'U-2H', 'other' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample.isotopic_labeling' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment',
        '## ##########################################################################' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_nmr_exptl_sample_conditions',
        [ [ 'item_',
            '_category.description',
            '   \nThe experimental conditions used to for each sample.  Each set of conditions\nis identified by a numerical code.  ' ],
          [ 'item_', '_category.id', 'pdbx_nmr_exptl_sample_conditions' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_nmr_exptl_sample_conditions.conditions_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ ' This example was taken from a pH stability study. ',
              '\n loop_\n _pdbx_nmr_exptl_sample_conditions.conditions_id\n _pdbx_nmr_exptl_sample_conditions.temperature\n _pdbx_nmr_exptl_sample_conditions.pressure\n _pdbx_nmr_exptl_sample_conditions.pH\n _pdbx_nmr_exptl_sample_conditions.ionic_strength\n 1 298 ambient  7  \'25mM NaCl\'\n 2 298 ambient  3  \'25mM NaCl\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample_conditions.conditions_id',
        [ [ 'item_',
            '_item_description.description',
            '   The condition number as defined above.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_exptl_sample_conditions.conditions_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_nmr_exptl_sample_conditions' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample_conditions.conditions_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample_conditions.temperature',
        [ [ 'item_',
            '_item_description.description',
            '   The temperature (in Kelvin) at which NMR data were\n    collected.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_exptl_sample_conditions.temperature' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_nmr_exptl_sample_conditions' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float-range' ],
          [ 'loop_', [ '_item_examples.case' ], [ '298' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample_conditions.temperature' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample_conditions.pressure_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units of pressure at which NMR data were collected.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_exptl_sample_conditions.pressure_units' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_nmr_exptl_sample_conditions' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'Pa', 'atm', 'Torr' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample_conditions.pressure_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample_conditions.pressure',
        [ [ 'item_',
            '_item_description.description',
            '   The pressure at which NMR data were collected.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_exptl_sample_conditions.pressure' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_nmr_exptl_sample_conditions' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'ambient', '1atm' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample_conditions.pressure' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample_conditions.pH',
        [ [ 'item_',
            '_item_description.description',
            '   The pH at which the NMR data were collected.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_exptl_sample_conditions.pH' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_nmr_exptl_sample_conditions' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float-range' ],
          [ 'loop_', [ '_item_examples.case' ], [ '3.1', '7.0' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample_conditions.pH' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl_sample_conditions.ionic_strength',
        [ [ 'item_',
            '_item_description.description',
            '   The ionic strength at which the NMR data were collected -in lieu of\n    this enter the concentration and identity of the salt in the sample.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_exptl_sample_conditions.ionic_strength' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_nmr_exptl_sample_conditions' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ '@comment', '     loop_' ],
          [ '@comment', '    _item_examples.case          ' ],
          [ '@comment', '              \'3.1\'' ],
          [ '@comment', '              \'7.0\'  ' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl_sample_conditions.ionic_strength' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment',
        '## ##########################################################################' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_nmr_spectrometer',
        [ [ 'item_',
            '_category.description',
            '   \nThe details about each spectrometer used to collect data for this \ndeposition.' ],
          [ 'item_', '_category.id', 'pdbx_nmr_spectrometer' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_nmr_spectrometer.spectrometer_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ ' The instruments described here are the ones used for the MCP-1 studies.',
              '  \n loop_\n _pdbx_nmr_spectrometer.spectrometer_id\n _pdbx_nmr_spectrometer.type\n _pdbx_nmr_spectrometer.field_strength\n 1  \'Bruker AMX\'   600\n 2  \'Bruker AMX\'   500',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_spectrometer.spectrometer_id',
        [ [ 'item_',
            '_item_description.description',
            '   \nAssign a numerical ID to each instrument.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_spectrometer.spectrometer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_spectrometer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_spectrometer.spectrometer_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_spectrometer.model',
        [ [ 'item_',
            '_item_description.description',
            '   \nThe model of the NMR spectrometer.    ' ],
          [ 'item_', '_item.name', '_pdbx_nmr_spectrometer.model' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_spectrometer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'AVANCE',
              'WH',
              'WM',
              'AM',
              'AMX',
              'DMX',
              'DRX',
              'MSL',
              'OMEGA',
              'OMEGA PSG',
              'GX',
              'GSX',
              'A',
              'AL',
              'EC',
              'EX',
              'LA',
              'ECP',
              'VXRS',
              'UNITY',
              'UNITYPLUS',
              'INOVA' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_spectrometer.model' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_spectrometer.type',
        [ [ 'item_',
            '_item_description.description',
            '   \nSelect the instrument manufacturer(s) and the model(s) of the NMR(s)\nused for this work.    ' ],
          [ 'item_', '_item.name', '_pdbx_nmr_spectrometer.type' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_spectrometer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Bruker WH',
              'Bruker WM',
              'Bruker AM',
              'Bruker AMX',
              'Bruker DMX',
              'Bruker DRX',
              'Bruker MSL',
              'Bruker AVANCE',
              'GE Omega',
              'GE Omega PSG',
              'JEOL GX',
              'JEOL GSX',
              'JEOL A',
              'JEOL AL',
              'JEOL EC',
              'JEOL EX',
              'JEOL LA',
              'JEOL ECP',
              'Varian VXRS',
              'Varian UNITY',
              'Varian UNITYplus',
              'Varian INOVA',
              'other' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_spectrometer.type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_spectrometer.manufacturer',
        [ [ 'item_',
            '_item_description.description',
            '\nThe name of the manufacturer of the spectrometer. ' ],
          [ 'item_', '_item.name', '_pdbx_nmr_spectrometer.manufacturer' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_spectrometer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Varian', 'Bruker', 'JEOL', 'GE' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_spectrometer.manufacturer' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_spectrometer.field_strength',
        [ [ 'item_',
            '_item_description.description',
            '            \nSelect the field strength for protons in MHz.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_spectrometer.field_strength' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_spectrometer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '360',
              '400',
              '500',
              '600',
              '750',
              '800',
              '850',
              '900',
              '950',
              '1000' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_spectrometer.field_strength' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment',
        '## ##########################################################################' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_nmr_exptl',
        [ [ 'item_',
            '_category.description',
            '\nIn this section, enter information on those experiments that were \nused to generate constraint data. For each NMR experiment indicate \nwhich sample and which sample conditions were used for the experiment.' ],
          [ 'item_', '_category.id', 'pdbx_nmr_exptl' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_nmr_exptl.experiment_id',
              '_pdbx_nmr_exptl.conditions_id',
              '_pdbx_nmr_exptl.solution_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ ' This example was taken from the MCP-1 study.',
              '\n loop_ \n _pdbx_nmr_exptl.experiment_id\n _pdbx_nmr_exptl.solution_id\n _pdbx_nmr_exptl.conditions_id\n _pdbx_nmr_exptl.type\n 1 3 1 \'3D_15N-separated_NOESY\'\n 2 1 1 \'3D_13C-separated_NOESY\'\n 3 1 2 \'4D_13C/15N-separated_NOESY\'\n 4 1 1 \'4D_13C-separated_NOESY\'\n 5 1 1 \'3D_15N-separated_ROESY\'\n 6 3 1 \'3D_13C-separated_ROESY\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl.experiment_id',
        [ [ 'item_',
            '_item_description.description',
            '   A numerical ID for each experiment.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_exptl.experiment_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_exptl' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl.experiment_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl.conditions_id',
        [ [ 'item_',
            '_item_description.description',
            ' The number to identify the set of sample conditions.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_exptl.conditions_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_exptl' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl.conditions_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl.solution_id',
        [ [ 'item_',
            '_item_description.description',
            '   The solution_id from the Experimental Sample to identify the sample\n    that these conditions refer to. \n\n    [Remember to save the entries here before returning to the \n     Experimental Sample form]' ],
          [ 'item_', '_item.name', '_pdbx_nmr_exptl.solution_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_exptl' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1', '2', '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_exptl.solution_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_exptl.type',
        [ [ 'item_',
            '_item_description.description',
            '   The type of NMR experiment.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_exptl.type' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_exptl' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2D NOESY',
              '3D_15N-separated_NOESY',
              '3D_13C-separated_NOESY',
              '4D_13C-separated_NOESY',
              '4D_13C/15N-separated_NOESY',
              '3D_15N-separated_ROESY',
              '3D_13C-separated_ROESY',
              'HNCA-J',
              'HNHA',
              'DQF-COSY',
              'P-COSY',
              'PE-COSY',
              'E-COSY' ] ],
          [ 'item_', '_item_aliases.alias_name', '_rcsb_nmr_exptl.type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment',
        '## ##########################################################################' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_nmr_software',
        [ [ 'item_',
            '_category.description',
            ' \nDescription of the software that was used for data collection, data processing, \ndata analysis, structure calculations and refinement. The description should \ninclude the name of the software, the author of the software and the version used.' ],
          [ 'item_', '_category.id', 'pdbx_nmr_software' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_nmr_software.name',
              '_pdbx_nmr_software.version',
              '_pdbx_nmr_software.classification' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ ' This example describes the software used in the MCP-1 study.',
              ' \n  loop_\n  _pdbx_nmr_software.name\n  _pdbx_nmr_software.authors\n  _pdbx_nmr_software.version\n  _pdbx_nmr_software.classification\n  UXNMR\t Bruker\t  940501.3\t \'collection\'\n  FELIX\t Hare\t  1.1 \t\t \'processing\'\n  ANSIG\t Kraulis  3.0  \t         \'data analysis\'\n  X-PLOR Brunger  3.8\t\t \'structure calculation\'\n  X-PLOR Brunger  3.8            \'refinement\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_software.classification',
        [ [ 'item_',
            '_item_description.description',
            '  The purpose of the software.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_software.classification' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_software' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'collection',
              'processing',
              'data analysis',
              'structure solution',
              'refinement',
              'iterative matrix relaxation',
              [ '@comment', '     loop_' ],
              [ '@comment', '   _item_enumeration.value' ],
              [ '@comment', '              \'collection\' ' ],
              [ '@comment', '              \'processing\'' ],
              [ '@comment', '              \'data analysis\'' ],
              [ '@comment', '              \'structure solution\'      ' ],
              [ '@comment', '              \'refinement\'' ],
              [ '@comment',
                '              \'iterative matrix relaxation\'  ' ] ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_software.classification' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_software.name',
        [ [ 'item_',
            '_item_description.description',
            '   The name of the software used for the task.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_software.name' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_software' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'ANSIG',
              'AURELIA',
              'AZARA',
              'CHARMM',
              'CORMA',
              'DIANA',
              'DYANA',
              'DSPACE',
              'DISGEO',
              'DGII',
              'DISMAN',
              'DINOSAUR',
              'DISCOVER',
              'FELIX',
              'FT_NMR',
              'GROMOS',
              'IRMA',
              'MARDIGRAS',
              'NMRPipe',
              'SA',
              'UXNMR',
              'VNMR',
              'X-PLOR',
              'XWINNMR' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_software.name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_software.version',
        [ [ 'item_',
            '_item_description.description',
            '   The version of the software.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_software.version' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_software' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '940501.3', '2.1' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_software.version' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_software.authors',
        [ [ 'item_',
            '_item_description.description',
            '   The name of the authors of the software used in this\n    procedure.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_software.authors' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_software' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'Brunger', 'Guentert' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_software.authors' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment',
        '## ##########################################################################' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_nmr_constraints',
        [ [ 'item_',
            '_category.description',
            '   This section provides a tabulation of constraint data. ' ],
          [ 'item_', '_category.id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_nmr_constraints.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ ' This example uses the data from the MCP-1 structure determination. Remember\n  this is a dimer so there are intersubunit constraints as well as intrasubunit\n  constraints.',
              '  \n   loop_\n   _pdbx_nmr_constraints.entry_id\n   _pdbx_nmr_constraints.NOE_constraints_total\n   _pdbx_nmr_constraints.NOE_intraresidue_total_count\n   _pdbx_nmr_constraints.NOE_sequential_total_count\n   _pdbx_nmr_constraints.NOE_medium_range_total_count\n   _pdbx_nmr_constraints.NOE_long_range_total_count\n   _pdbx_nmr_constraints.protein_phi_angle_constraints_total_count\n   1ABC 4458  1144  272  1004   1356  96',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '   You can leave this blank as an ID will be assigned by the MSD \n    to the constraint file.     ' ],
          [ 'item_', '_item.name', '_pdbx_nmr_constraints.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_nmr_constraints.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NOE_constraints_total',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of all NOE constraints used in the final\n    structure calculation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NOE_constraints_total' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '4458' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NOE_constraints_total' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NOE_intraresidue_total_count',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of all intraresidue, [i-j]=0, NOE constraints \n    used in the final structure calculation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NOE_intraresidue_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1144' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NOE_intraresidue_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NOE_interentity_total_count',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of interentity, NOE constraints \n    used in the final structure calculation. This field should only be\n    if system is complex -i.e more than one entity e.g. a dimer or ligand-protein \n    complex' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NOE_interentity_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '272' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NOE_interentity_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NOE_sequential_total_count',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of sequential, [i-j]=1, NOE constraints used \n    in the final structure calculation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NOE_sequential_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1004' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NOE_sequential_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NOE_medium_range_total_count',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of medium range 1<[i-j]<=5 NOE constraints used \n    in the final structure calculation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NOE_medium_range_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '682' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NOE_medium_range_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NOE_long_range_total_count',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of long range [i-j]>5 NOE constraints used \n    in the final structure calculation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NOE_long_range_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1356' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NOE_long_range_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.protein_phi_angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of phi angle constraints used in the final structure\n    calculation' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.protein_phi_angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '96' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.protein_phi_angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.protein_psi_angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of psi angle constraints used in the final structure\n    calculation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.protein_psi_angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.protein_psi_angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.protein_chi_angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of chi angle constraints used in the final structure\n    calculation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.protein_chi_angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '66' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.protein_chi_angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.protein_other_angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of other angle constraints used in the final structure\n  calculation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.protein_other_angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.protein_other_angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NOE_interproton_distance_evaluation',
        [ [ 'item_',
            '_item_description.description',
            '   Describe the method used to quantify the NOE and ROE values.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NOE_interproton_distance_evaluation' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '\n  NOE buildup curves with 50, 75, 150 ms mixing times were\n  analyzed.',
              '\n   Noesy cross peak intensities were classified into three different \n   catagories with distances of 1.8-2.7 A, 1.8-3.5 A, 1.8- 5.0 A for strong,\n   medium and weak NOEs.' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NOE_interproton_distance_evaluation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NOE_pseudoatom_corrections',
        [ [ 'item_',
            '_item_description.description',
            '   Describe any corrections made for pseudoatoms' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NOE_pseudoatom_corrections' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '\n  Pseudo-atoms nomenclature and corrections according to Wuethrich, Billeter, \n  and Braun, J. Mol.Biol.(1983) 169, 949-961.',
              '\n  Pseudoatoms were not used.' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NOE_pseudoatom_corrections' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NOE_motional_averaging_correction',
        [ [ 'item_',
            '_item_description.description',
            'Describe any corrections that were made to the NOE data for motional averaging.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NOE_motional_averaging_correction' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Replace with item example text' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NOE_motional_averaging_correction' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_pdbx_nmr_constraints.hydrogen_bond_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '  The total number of hydrogen bond constraints used in the final\n   structure calculation. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.hydrogen_bond_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '6' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.hydrogen_bond_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.disulfide_bond_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '  The total number of disulfide bond constraints used in the final\n   structure calculation. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.disulfide_bond_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.disulfide_bond_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NA_alpha-angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '  The total number of nucleic acid alpha-angle constraints used in\n   the final structure calculation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NA_alpha-angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '18' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NA_alpha-angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NA_beta-angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '  The total number of nucleic acid beta-angle constraints used in\n   the final structure calculation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NA_beta-angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '24' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NA_beta-angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NA_gamma-angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '  The total number of nucleic acid gamma-angle constraints used in\n   the final structure calculation. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NA_gamma-angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '12' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NA_gamma-angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NA_delta-angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '  The total number of nucleic acid delta-angle constraints used in\n   the final structure calculation. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NA_delta-angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '15' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NA_delta-angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NA_epsilon-angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '  The total number of nucleic acid epsilon-angle constraints used\n   in the final structure calculation. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NA_epsilon-angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '31' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NA_epsilon-angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NA_chi-angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '  The total number of nucleic acid chi-angle constraints used\n   in the final structure calculation. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NA_chi-angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '15' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NA_chi-angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NA_other-angle_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '  The total number of nucleic acid other-angle constraints used\n   in the final structure calculation. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NA_other-angle_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '5' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NA_other-angle_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_constraints.NA_sugar_pucker_constraints_total_count',
        [ [ 'item_',
            '_item_description.description',
            '  The total number of nucleic acid sugar pucker constraints used\n   in the final structure calculation. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_constraints.NA_sugar_pucker_constraints_total_count' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_constraints' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '10' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_constraints.NA_sugar_pucker_constraints_total_count' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '##' ],
      [ '@comment',
        '## ##########################################################################' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_nmr_ensemble',
        [ [ 'item_',
            '_category.description',
            '   This category contains the information that describes the \n    ensemble of deposited structures. If only an average structure \n    has been deposited skip this section. ' ],
          [ 'item_', '_category.id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_nmr_ensemble.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ 'This example uses the data from the MCP-1 study.',
              '\n  loop_\n  _pdbx_nmr_ensemble.entry_id\n  _pdbx_nmr_ensemble.conformers_calculated_total_number\n  _pdbx_nmr_ensemble.conformers_submitted_total_number\n  _pdbx_nmr_ensemble.conformer_selection_criteria\n  _pdbx_nmr_ensemble.representative_conformer\n   1ABC 40  20  \'structures with the least restraint violations\'  1',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '   Leave this blank as the ID is provided by the MSD' ],
          [ 'item_', '_item.name', '_pdbx_nmr_ensemble.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_nmr_ensemble.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.conformers_calculated_total_number',
        [ [ 'item_',
            '_item_description.description',
            '   The total number of conformer (models) that were calculated in the final round.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.conformers_calculated_total_number' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '40' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.conformers_calculated_total_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.conformers_submitted_total_number',
        [ [ 'item_',
            '_item_description.description',
            '   The number of conformer (models) that are submitted for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.conformers_submitted_total_number' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '20' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.conformers_submitted_total_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.conformer_selection_criteria',
        [ [ 'item_',
            '_item_description.description',
            '   \nBy highlighting the appropriate choice(s), describe how the submitted \nconformer (models) were selected.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.conformer_selection_criteria' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'structures with the lowest energy',
              'structures with the least restraint violations',
              'structures with acceptable covalent geometry',
              'structures with favorable non-bond energy',
              'target function',
              'back calculated data agree with experimental NOESY spectrum',
              'all calculated structures submitted',
              '   The submitted conformer models are the 25 structures with the lowest \n    energy.',
              '   The submitted conformer models are those with the fewest number of \n    constraint violations.' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.conformer_selection_criteria' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.representative_conformer',
        [ [ 'item_',
            '_item_description.description',
            '   The number of the conformer identified as most representative.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.representative_conformer' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '20' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.representative_conformer' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.average_constraints_per_residue',
        [ [ 'item_',
            '_item_description.description',
            '   The average number of constraints per residue for the ensemble' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.average_constraints_per_residue' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '30.2' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.average_constraints_per_residue' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.average_constraint_violations_per_residue',
        [ [ 'item_',
            '_item_description.description',
            '   The average number of constraint violations on a per residue basis for\n    the ensemble. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.average_constraint_violations_per_residue' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.25' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.average_constraint_violations_per_residue' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.maximum_distance_constraint_violation',
        [ [ 'item_',
            '_item_description.description',
            '   The maximum distance constraint violation for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.maximum_distance_constraint_violation' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.4' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.maximum_distance_constraint_violation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.average_distance_constraint_violation',
        [ [ 'item_',
            '_item_description.description',
            '   The average distance restraint violation for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.average_distance_constraint_violation' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.11' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.average_distance_constraint_violation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.maximum_upper_distance_constraint_violation',
        [ [ 'item_',
            '_item_description.description',
            '   The maximum upper distance constraint violation for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.maximum_upper_distance_constraint_violation' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.4' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.maximum_upper_distance_constraint_violation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.maximum_lower_distance_constraint_violation',
        [ [ 'item_',
            '_item_description.description',
            '   The maximum lower distance constraint violation for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.maximum_lower_distance_constraint_violation' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.3' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.maximum_lower_distance_constraint_violation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.distance_constraint_violation_method',
        [ [ 'item_',
            '_item_description.description',
            '   Describe the method used to calculate the distance constraint violation statistics,\n    i.e. are they calculated over all the distance constraints or calculated for\n    violations only?' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.distance_constraint_violation_method' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Statistics were calculated over all of the distance constraints.',
              'Statistics were calculated for violations only' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.distance_constraint_violation_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.maximum_torsion_angle_constraint_violation',
        [ [ 'item_',
            '_item_description.description',
            '   The maximum torsion angle constraint violation for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.maximum_torsion_angle_constraint_violation' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'loop_', [ '_item_examples.case' ], [ '4' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.maximum_torsion_angle_constraint_violation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.average_torsion_angle_constraint_violation',
        [ [ 'item_',
            '_item_description.description',
            '   The average torsion angle constraint violation for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.average_torsion_angle_constraint_violation' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'loop_', [ '_item_examples.case' ], [ '2.4' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.average_torsion_angle_constraint_violation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble.torsion_angle_constraint_violation_method',
        [ [ 'item_',
            '_item_description.description',
            '   This item describes the method used to calculate the torsion angle constraint violation statistics.\n  i.e. are the entered values based on all torsion angle or calculated for violations only? ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble.torsion_angle_constraint_violation_method' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Statistics were calculated over all the torsion angle constraints.',
              'Statistics were calculated for torsion angle constraints violations only.' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble.torsion_angle_constraint_violation_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment',
        '## ##########################################################################' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_nmr_ensemble_rms',
        [ [ 'item_',
            '_category.description',
            ' Structural statistics are derived from molecular dynamics and simulated annealing \n  programs. ' ],
          [ 'item_', '_category.id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_nmr_ensemble_rms.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '  This example is derived from the MCP-1 structure calculation statistics. \n   For this  structure the statistics were calculated over residues 5-69 for \n   both the monomer and dimer .    ',
              '  \n loop_\n _pdbx_nmr_ensemble_rms.entry_id\n _pdbx_nmr_ensemble_rms.residue_range_begin\n _pdbx_nmr_ensemble_rms.chain_range_begin\n _pdbx_nmr_ensemble_rms.residue_range_end\n _pdbx_nmr_ensemble_rms.chain_range_end\n_pdbx_nmr_ensemble_rms.atom_type\n_pdbx_nmr_ensemble_rms.distance_rms_dev\n_pdbx_nmr_ensemble_rms.distance_rms_dev_error\n 1ABC 5  A  69  A  \'all heavy atoms\'  0.22  0.06',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '               \'?\'' ],
          [ 'item_', '_item.name', '_pdbx_nmr_ensemble_rms.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_nmr_ensemble_rms.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.residue_range_begin',
        [ [ 'item_',
            '_item_description.description',
            ' Structure statistics are often calculated only over the well-ordered region(s) \n  of the biopolymer. Portions of the macromolecule are often mobile and    \n  disordered,  hence they are excluded in calculating the statistics. To define\n  the range(s) over which the statistics are calculated, enter the beginning   \n  residue number(s): e.g. if the regions used were 5-32 and 41-69, enter 5,41      ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.residue_range_begin' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '5', '41' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.residue_range_begin' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.chain_range_begin',
        [ [ 'item_',
            '_item_description.description',
            '   The beginning chain id. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.chain_range_begin' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', 'A' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.chain_range_begin' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.residue_range_end',
        [ [ 'item_',
            '_item_description.description',
            '   The ending residue number: e.g. 32,69.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.residue_range_end' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'loop_', [ '_item_examples.case' ], [ '32', '69' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.residue_range_end' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.chain_range_end',
        [ [ 'item_',
            '_item_description.description',
            '   The ending chain id:' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.chain_range_end' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', 'A' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.chain_range_end' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.atom_type',
        [ [ 'item_',
            '_item_description.description',
            ' Statistics are often calculated over only some of the atoms, \n  e.g. backbone, or heavy atoms. Describe which type of atoms are used for \n  the statistical analysis.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_ensemble_rms.atom_type' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'backbone atoms', 'heavy atoms' ] ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'backbone heavy atoms',
              'side chain heavy atoms',
              'all heavy atoms',
              'all atoms',
              'all backbone atoms' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.atom_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.distance_rms_dev',
        [ [ 'item_',
            '_item_description.description',
            '  The distance rmsd to the mean structure for the ensemble of structures.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.distance_rms_dev' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.22' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.distance_rms_dev' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.distance_rms_dev_error',
        [ [ 'item_',
            '_item_description.description',
            ' The error in the distance rmsd.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.distance_rms_dev_error' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.07' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.distance_rms_dev_error' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.covalent_bond_rms_dev',
        [ [ 'item_',
            '_item_description.description',
            '   The covalent bond rmsd to the target value for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.covalent_bond_rms_dev' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.0066' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.covalent_bond_rms_dev' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.covalent_bond_rms_dev_error',
        [ [ 'item_',
            '_item_description.description',
            '   The error in the covalent bond rmsd.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.covalent_bond_rms_dev_error' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.0001' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.covalent_bond_rms_dev_error' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.bond_angle_rms_dev',
        [ [ 'item_',
            '_item_description.description',
            '   The bond angle rmsd to the target values for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.bond_angle_rms_dev' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.60' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.bond_angle_rms_dev' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.bond_angle_rms_dev_error',
        [ [ 'item_',
            '_item_description.description',
            '   The error in the bond angle rmsd.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.bond_angle_rms_dev_error' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.01' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.bond_angle_rms_dev_error' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.improper_torsion_angle_rms_dev',
        [ [ 'item_',
            '_item_description.description',
            '   The improper torsion angle rmsd to the target values for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.improper_torsion_angle_rms_dev' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.64' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.improper_torsion_angle_rms_dev' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.improper_torsion_angle_rms_dev_error',
        [ [ 'item_',
            '_item_description.description',
            '   The error in the improper torsion angle rmsd.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.improper_torsion_angle_rms_dev_error' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.04' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.improper_torsion_angle_rms_dev_error' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.peptide_planarity_rms_dev',
        [ [ 'item_',
            '_item_description.description',
            '   The peptide planarity rmsd.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.peptide_planarity_rms_dev' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.11' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.peptide_planarity_rms_dev' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.peptide_planarity_rms_dev_error',
        [ [ 'item_',
            '_item_description.description',
            '   The error in the peptide planarity rmsd.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.peptide_planarity_rms_dev_error' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.05' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.peptide_planarity_rms_dev_error' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.dihedral_angles_rms_dev',
        [ [ 'item_',
            '_item_description.description',
            '   The dihedral angle rmsd to the target values for the ensemble.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.dihedral_angles_rms_dev' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.66' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.dihedral_angles_rms_dev' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.dihedral_angles_rms_dev_error',
        [ [ 'item_',
            '_item_description.description',
            '   The error of the rmsd dihedral angles.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.dihedral_angles_rms_dev_error' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.07' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.dihedral_angles_rms_dev_error' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_ensemble_rms.coord_average_rmsd_method',
        [ [ 'item_',
            '_item_description.description',
            '   Describe the method for calculating the coordinate average rmsd.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_ensemble_rms.coord_average_rmsd_method' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_ensemble_rms' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Replace with item example text' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_ensemble_rms.coord_average_rmsd_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment',
        '## ##########################################################################' ],
      [ '@comment', '##' ],
      [ 'save_',
        'pdbx_nmr_representative',
        [ [ 'item_',
            '_category.description',
            ' An average structure is often calculated in addition to the ensemble, or one\n  of the ensemble is selected as a representative structure. This section\n  describes selection of the representative structure.' ],
          [ 'item_', '_category.id', 'pdbx_nmr_representative' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_nmr_representative.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '   This example is drawn from the MCP-1 structure.',
              '\n _pdbx_nmr_representative.entry_id              1ABC   \n _pdbx_nmr_representative.conformer_id          15   \n _pdbx_nmr_representative.selection_criteria    \'lowest energy\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_representative.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '    msd will assign the ID.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_representative.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_representative' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_nmr_representative.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_representative.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_representative.conformer_id',
        [ [ 'item_',
            '_item_description.description',
            '   If a member of the ensemble has been selected as a representative\n    structure, identify it by its model number.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_representative.conformer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_representative' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '15' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_representative.conformer_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_representative.selection_criteria',
        [ [ 'item_',
            '_item_description.description',
            '   \nBy highlighting the appropriate choice(s), describe the criteria used to \nselect this structure as a representative structure, or if an average \nstructure has been calculated describe how this was done.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_representative.selection_criteria' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_representative' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ '@comment', '     loop_' ],
          [ '@comment', '    _item_enumeration.value' ],
          [ '@comment', ' \'closest to the average\'' ],
          [ '@comment', ' \'lowest energy\'' ],
          [ '@comment', ' \'fewest violations\'' ],
          [ '@comment', ' \'minimized average structure\'' ],
          [ 'item_',
            '_item_examples.case',
            '                       \nThe structure closest to the average.\nThe structure with the lowest energy was selected.\nThe structure with the fewest number of violations was selected.\nA minimized average structure was calculated.  ' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_representative.selection_criteria' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        'pdbx_nmr_refine',
        [ [ 'item_',
            '_category.description',
            '   \nDescribe the method and details of the refinement of the deposited structure.' ],
          [ 'item_', '_category.id', 'pdbx_nmr_refine' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_nmr_refine.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '   This example is drawn from the MCP-1 structure.',
              '\n  loop_\n  _pdbx_nmr_refine.entry_id\n  _pdbx_nmr_refine.method\t\n  1ABC \'torsion angle dynamics\'' ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_refine.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '   You can leave this blank as an ID will be assigned by the RCSB\n    to the constraint file.     ' ],
          [ 'item_', '_item.name', '_pdbx_nmr_refine.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_refine' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_nmr_refine.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_refine.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_refine.method',
        [ [ 'item_',
            '_item_description.description',
            '   The method used to determine the structure.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_refine.method' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_refine' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '\n  distance geometry\n  simulated annealing\n  molecular dynamics\n  matrix relaxation\n  torsion angle dynamics' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_refine.method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_refine.details',
        [ [ 'item_',
            '_item_description.description',
            '   Additional details about the NMR refinement.' ],
          [ 'item_', '_item.name', '_pdbx_nmr_refine.details' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '\nAdditional comments about the NMR refinement can be placed here, e.g.\nthe structures are based on a total of 3344 restraints, 3167 are NOE-derived\ndistance constraints, 68 dihedral angle restraints,109 distance restraints \nfrom hydrogen bonds.' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_refine.details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        'pdbx_nmr_force_constants',
        [ [ 'item_',
            '_category.description',
            '\nThe final force constants, including units, employed for the various \nexperimental constraints, covalent geometry constraints, and the non-bonded \ninteraction terms in the target function used for simulated annealing.' ],
          [ 'item_', '_category.id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_nmr_force_constants.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - -  - - - - - -  -' ] ],
            [ ' \nThis example is taken from a study of BAF, a dimeric DNA binding protein. \nThe final force constants in the target function used for simulated \nannealing are:  Experimental Constraint terms: Distance (NOE,H-bonds), \nTorsion angles, J coupling, 13C shifts, 1H shifts, Dipolar coupling, \nD isotope shifts Covalent Geometry Constraint terms: Bond lengths, \nAngles, Impropers Non-bonded Interaction terms: van der Waals, \nType of van der Waals term, Conformational database potential, \nRadius of gyration.',
              '\n _pdbx_nmr_force_constants.entry_id                                      1ABC \n _pdbx_nmr_force_constants.exptl_distance_term                            30. \n _pdbx_nmr_force_constants.exptl_distance_term_units                     \'kcal/mol/A**2\' \n _pdbx_nmr_force_constants.exptl_torsion_angles_term                      200. \n _pdbx_nmr_force_constants.exptl_torsion_angles_term_units               \'kcal/mol/rad**2\' \n _pdbx_nmr_force_constants.exptl_J_coupling_term                          1. \n _pdbx_nmr_force_constants.exptl_J_coupling_term_units                   \'kcal/mol/Hz**2\'  \n _pdbx_nmr_force_constants.exptl_13C_shift_term                           0.5 \n _pdbx_nmr_force_constants.exptl_13C_shift_term_units                    \'kcal/mol/ppm**2\' \n _pdbx_nmr_force_constants.exptl_1H_shift_term                            7.5 \n _pdbx_nmr_force_constants.exptl_1H_shift_term_units                     \'kcal/mol/ppm**2\'\n _pdbx_nmr_force_constants.covalent_geom_bond_term                        1000. \n _pdbx_nmr_force_constants.covalent_geom_bond_term_units                  \'kcal/mol/A**2\'\n _pdbx_nmr_force_constants.non-bonded_inter_van_der_Waals_term_type       4.\n _pdbx_nmr_force_constants.non-bonded_inter_van_der_Waals_term_type_units \'kcal/mol/A**4\'' ] ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '   You can leave this blank as an ID will be assigned by the RCSB.          ' ],
          [ 'item_', '_item.name', '_pdbx_nmr_force_constants.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_nmr_force_constants.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_distance_term',
        [ [ 'item_',
            '_item_description.description',
            '  The final force constant for distance (NOEs) constraints \n   term employed in the target function used for simulated annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_distance_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '30' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_distance_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_distance_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the distance \n    constraints term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_distance_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/ A**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/A**2',
              'kilocalories per mole per square angstrom',
              'kJ/mol/nm**2',
              'kilojoules per mole per square nanometer',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_distance_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_torsion_angles_term',
        [ [ 'item_',
            '_item_description.description',
            '  The final force constant for the torsion angle term employed in \n   the target function used for simulated annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_torsion_angles_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '200' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_torsion_angles_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_torsion_angles_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the torsion angle \n    constraints term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_torsion_angles_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/rad**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/rad**2',
              'kilocalories per mole per square radian',
              'kJ/mol/rad**2',
              'kilojoules per mole per square radian',
              'author',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_torsion_angles_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_J_coupling_term',
        [ [ 'item_',
            '_item_description.description',
            '  The final force constant for J coupling term employed in the \n   target function used for simulated annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_J_coupling_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_J_coupling_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_J_coupling_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the J coupling term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_J_coupling_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/Hz**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/Hz**2',
              'kilocalories per mole per square angstrom',
              'kJ/mol/Hz**2',
              'kilojoules per mole per square nanometer',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_J_coupling_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_13C_shift_term',
        [ [ 'item_',
            '_item_description.description',
            '  The final force constant for 13C shift constraints term employed \n   in the target function used for simulated annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_13C_shift_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.5' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_13C_shift_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_13C_shift_term_units',
        [ [ 'item_',
            '_item_description.description',
            '  The units for the force constant for the 13C shift constraints\n   term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_13C_shift_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/ppm**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/ppm**2',
              'kilocalories per mole per part per million squared',
              'kJ/mol/ppm**2',
              'kilojoules per mole per part per million squared',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_13C_shift_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_1H_shift_term',
        [ [ 'item_',
            '_item_description.description',
            '  The final force constant for 1H shift constraints term employed \n   in the target function used for simulated annealing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_1H_shift_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '7.5' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_1H_shift_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_1H_shift_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the 1H shift constraints term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_1H_shift_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/ppm**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/ppm**2',
              'kilocalories per mole per part per million squared',
              'kJ/mol/ppm**2',
              'kilojoules per mole per part per million squared',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_1H_shift_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_dipolar_coupling_term',
        [ [ 'item_',
            '_item_description.description',
            '  The final force constant for dipolar coupling constraint term \n   employed in the target function used for simulated annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_dipolar_coupling_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_dipolar_coupling_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_dipolar_coupling_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the dipolar coupling \n    constraints term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_dipolar_coupling_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/Hz**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/Hz**2',
              'kilocalories per mole per Hertz squared',
              'kJ/mol/Hz**2',
              'kilojoules per mole per Hertz squared',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_dipolar_coupling_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_D_isotope_shift_term',
        [ [ 'item_',
            '_item_description.description',
            '  The final force constant for Deuterium isotope shift constraints \n   term employed in the target function used for simulated annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_D_isotope_shift_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '0.5' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_D_isotope_shift_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '##' ],
      [ '@comment', '## NEXT' ],
      [ 'save_',
        '_pdbx_nmr_force_constants.exptl_D_isotope_shift_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the Deuterium isotope \n    shift constraints term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.exptl_D_isotope_shift_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/ppb**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/ppb**2',
              'kilocalories per mole per part per billion squared',
              'kJ/mol/ppb**2',
              'kilojoules per mole per part per billion squared',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.exptl_D_isotope_shift_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.covalent_geom_bond_term',
        [ [ 'item_',
            '_item_description.description',
            '  The final force constant for the covalent geometry bond length \n   constraints term employed in the target function used for simulated \n   annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.covalent_geom_bond_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1000' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.covalent_geom_bond_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.covalent_geom_bond_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the covalent geometry bond \n    length constraints term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.covalent_geom_bond_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/A**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/A**2',
              'kilocalories per mole per square angstrom',
              'kJ/mol/nm**2',
              'kilojoules per mole per square nanometer',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.covalent_geom_bond_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.covalent_geom_angles_term',
        [ [ 'item_',
            '_item_description.description',
            '  The final force constant for covalent geometry angle constraints \n   term employed in the target function used for simulated annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.covalent_geom_angles_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '500' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.covalent_geom_angles_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.covalent_geom_angles_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the covalent geometry angle \n    constraints term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.covalent_geom_angles_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/rad**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/rad**2',
              'kilocalories per mole per radian squared',
              'kJ/mol/rad**2',
              'kilojoules per mole per radian squared',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.covalent_geom_angles_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.covalent_geom_impropers_term',
        [ [ 'item_',
            '_item_description.description',
            '  The final force constant for covalent geometry impropers \n   contstraints term employed in the target function used for simulated \n   annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.covalent_geom_impropers_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '500' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.covalent_geom_impropers_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.covalent_geom_impropers_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the covalent geometry \n    impropers constraints term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.covalent_geom_impropers_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/rad**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/rad**2',
              'kilocalories per mole per radian squared',
              'kJ/mol/rad**2',
              'kilojoules per mole per radian squared',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.covalent_geom_impropers_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.non-bonded_inter_van_der_Waals_term_type',
        [ [ 'item_',
            '_item_description.description',
            '  The type of van der Waals term employed in the target function \n   used for simulated annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.non-bonded_inter_van_der_Waals_term_type' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'quartic' ] ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'quartic', 'other' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.non-bonded_inter_van_der_Waals_term_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.non-bonded_inter_van_der_Waals_term',
        [ [ 'item_',
            '_item_description.description',
            '  The force constant used for the non-bonded interaction van der Waals \n   term employed in the target function used for simulated annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.non-bonded_inter_van_der_Waals_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '4' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.non-bonded_inter_van_der_Waals_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.non-bonded_inter_van_der_Waals_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the van der Waals term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.non-bonded_inter_van_der_Waals_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/ A**4' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/A**4',
              'kilocalories per mole per Angstrom to the 4th power',
              'kJ/mol/nm**4',
              'kilojoules per mole per nanometer to the 4th power',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.non-bonded_inter_van_der_Waals_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.non-bonded_inter_conf_db_potential_term',
        [ [ 'item_',
            '_item_description.description',
            '  The force constant used for the non-bonded interaction conformational \n   database potential term employed in the target function used for simulated \n   annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.non-bonded_inter_conf_db_potential_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '1.0' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.non-bonded_inter_conf_db_potential_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.non-bonded_inter_radius_of_gyration_term',
        [ [ 'item_',
            '_item_description.description',
            '  The force constant used for the non-bonded interaction radius of \n   gyration term employed in the target function used for simulated annealing. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.non-bonded_inter_radius_of_gyration_term' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_', [ '_item_examples.case' ], [ '100' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.non-bonded_inter_radius_of_gyration_term' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_nmr_force_constants.non-bonded_inter_radius_of_gyration_term_units',
        [ [ 'item_',
            '_item_description.description',
            '   The units for the force constant for the radius of gyration term.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_nmr_force_constants.non-bonded_inter_radius_of_gyration_term_units' ],
          [ 'item_', '_item.category_id', 'pdbx_nmr_force_constants' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'kcal/mol/ A**2' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'kcal/mol/ A**2',
              'kilocalories per mole per square Angstrom',
              'kJ/mol/ nm**4',
              'kilojoules per mole per square nanometer',
              'other',
              'author added units' ] ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_nmr_force_constants.non-bonded_inter_radius_of_gyration_term_units' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment',
        '## EOF  mmcif_pdbx-def-2.dic###########################################################################' ],
      [ '@comment',
        ' File:  mmcif_pdbx-def-4.dic                                             ' ],
      [ '@comment', ' ' ],
      [ '@comment',
        '                      PDB Exchange Data Dictionary' ],
      [ '@comment',
        ' This data dictionary contains definitions used by wwPDB for data exchange' ],
      [ '@comment', ' and data processing.  ' ],
      [ '@comment',
        '                           Definition Section 4' ],
      [ '@comment', '                            ' ],
      [ '@comment',
        '                                                                        ' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment', '#' ],
      [ '@comment',
        '#   Additional aliases required only for NDB/RCSB use:' ],
      [ '@comment', '#' ],
      [ 'save_',
        '_refine.overall_SU_B',
        [ [ '@comment',
            '    _item.name                  \'_refine.overall_SU_B\'' ],
          [ '@comment', '    _item.category_id             refine' ],
          [ '@comment', '    _item.mandatory_code          no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_overall_ESU_B' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ '@comment', '     _item_type.code             float' ] ] ],
      [ 'save_',
        '_refine.overall_SU_ML',
        [ [ '@comment',
            '    _item.name                  \'_refine.overall_SU_ML\'' ],
          [ '@comment', '    _item.category_id             refine' ],
          [ '@comment', '    _item.mandatory_code          no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine.ebi_overall_ESU_ML' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ '@comment', '     _item_type.code             float' ] ] ],
      [ 'save_',
        '_refine_hist.number_atoms_solvent',
        [ [ '@comment',
            '    _item.name                  \'_refine_hist.number_atoms_solvent\'' ],
          [ '@comment', '    _item.category_id             refine_hist' ],
          [ '@comment', '    _item.mandatory_code          no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_hist.rcsb_number_atoms_solvent' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ '@comment', '    _item_type.code             int' ] ] ],
      [ 'save_',
        '_refine_hist.number_atoms_total',
        [ [ '@comment',
            '    _item.name                  \'_refine_hist.number_atoms_total\'' ],
          [ '@comment', '    _item.category_id             refine_hist' ],
          [ '@comment', '    _item.mandatory_code          no' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_hist.rcsb_number_atoms_total' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ],
          [ '@comment', '    _item_type.code             int' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#  NDB Specific definitions' ],
      [ '@comment', '# ' ],
      [ '@comment', '#########################' ],
      [ '@comment', '# NDB_STRUCT_CONF_NA   ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'ndb_struct_conf_na',
        [ [ 'item_',
            '_category.description',
            '              Data items in the NDB_STRUCT_CONF_NA category \n               describes secondary structure features in this entry.' ],
          [ 'item_', '_category.id', 'ndb_struct_conf_na' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_ndb_struct_conf_na.entry_id',
              '_ndb_struct_conf_na.feature' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group', 'ndb_group' ] ],
          [ 'item_',
            '_category_examples.case',
            '\n loop_\n _ndb_struct_conf_na.entry_id    \n _ndb_struct_conf_na.feature\n _ndb_struct_conf_na.feature_count\n  rcsb000001  \'double helix\'           1\n  rcsb000001  \'b-form double helix\'    1\n  rcsb000001  \'quadruple helix\'        1' ] ] ],
      [ 'save_',
        '_ndb_struct_conf_na.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the \n               ENTRY category.' ],
          [ 'item_', '_item.name', '_ndb_struct_conf_na.entry_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_conf_na' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_conf_na.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_ndb_struct_conf_na.feature',
        [ [ 'item_',
            '_item_description.description',
            '              This data item identifies a secondary structure\n               feature of this entry.  ' ],
          [ 'item_', '_item.name', '_ndb_struct_conf_na.feature' ],
          [ 'item_', '_item.category_id', 'ndb_struct_conf_na' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'double helix',
              '.',
              'a-form double helix',
              '.',
              'b-form double helix',
              '.',
              'z-form double helix',
              '.',
              'other right-handed double helix',
              '.',
              'triple helix',
              '.',
              'quadruple helix',
              '.',
              'parallel strands',
              '.',
              'internal loop',
              '.',
              'bulge loop',
              '.',
              'tetraloop',
              '.',
              'hairpin loop',
              '.',
              'two-way junction',
              '.',
              'three-way junction',
              '.',
              'four-way junction',
              '.',
              'mismatched base pair',
              '.' ] ] ] ],
      [ 'save_',
        '_ndb_struct_conf_na.feature_count',
        [ [ 'item_',
            '_item_description.description',
            '              This data item counts the number of occurences of \n               this feature in this entry.  ' ],
          [ 'item_', '_item.name', '_ndb_struct_conf_na.feature_count' ],
          [ 'item_', '_item.category_id', 'ndb_struct_conf_na' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', '# NDB_STRUCT_FEATURE_NA   ##' ],
      [ '@comment', '############################' ],
      [ 'save_',
        'ndb_struct_feature_na',
        [ [ 'item_',
            '_category.description',
            '              Data items in the NDB_STRUCT_FEATURE_NA category \n               describes tertiary and other special structural \n               features in this entry.' ],
          [ 'item_', '_category.id', 'ndb_struct_feature_na' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_ndb_struct_feature_na.entry_id',
              '_ndb_struct_feature_na.feature' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group', 'ndb_group' ] ],
          [ 'item_',
            '_category_examples.case',
            '\n loop_\n _ndb_struct_feature_na.entry_id    \n _ndb_struct_feature_na.feature\n _ndb_struct_feature_na.feature_count\n  rcsb000001  \'intercalated base\'           1\n  rcsb000001  \'bent/kinked double helix\'    1' ] ] ],
      [ 'save_',
        '_ndb_struct_feature_na.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entry.id in the \n               ENTRY category.' ],
          [ 'item_', '_item.name', '_ndb_struct_feature_na.entry_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_feature_na' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_feature_na.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_ndb_struct_feature_na.feature',
        [ [ 'item_',
            '_item_description.description',
            '              This data item identifies a structural\n               feature of this entry.  ' ],
          [ 'item_', '_item.name', '_ndb_struct_feature_na.feature' ],
          [ 'item_', '_item.category_id', 'ndb_struct_feature_na' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'pseudoknot',
              '.',
              'intercalated base',
              '.',
              'backbone turn',
              '.',
              'intramolecular base triplet',
              '.',
              'ribose zipper',
              '.',
              'purine platform',
              '.',
              'bent/kinked double helix',
              '.' ] ] ] ],
      [ 'save_',
        '_ndb_struct_feature_na.feature_count',
        [ [ 'item_',
            '_item_description.description',
            '              This data item counts the number of occurences of \n               this feature in this entry.  ' ],
          [ 'item_', '_item.name', '_ndb_struct_feature_na.feature_count' ],
          [ 'item_', '_item.category_id', 'ndb_struct_feature_na' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', '# NDB_STRUCT_NA_BASE_PAIR ##' ],
      [ '@comment', '############################' ],
      [ 'save_',
        'ndb_struct_na_base_pair',
        [ [ 'item_',
            '_category.description',
            '              Data items in the NDB_STRUCT_NA_BASE_PAIR category record details \n               of base pairing interactions.' ],
          [ 'item_', '_category.id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_ndb_struct_na_base_pair.model_number',
              '_ndb_struct_na_base_pair.i_label_comp_id',
              '_ndb_struct_na_base_pair.i_label_asym_id',
              '_ndb_struct_na_base_pair.i_label_seq_id',
              '_ndb_struct_na_base_pair.i_symmetry',
              '_ndb_struct_na_base_pair.j_label_comp_id',
              '_ndb_struct_na_base_pair.j_label_asym_id',
              '_ndb_struct_na_base_pair.j_label_seq_id',
              '_ndb_struct_na_base_pair.j_symmetry' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group', 'ndb_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n     loop_\n    _ndb_struct_na_base_pair.model_number\n    _ndb_struct_na_base_pair.i_label_comp_id\n    _ndb_struct_na_base_pair.i_label_asym_id\n    _ndb_struct_na_base_pair.i_label_seq_id\n    _ndb_struct_na_base_pair.i_symmetry\n    _ndb_struct_na_base_pair.j_label_comp_id\n    _ndb_struct_na_base_pair.j_label_asym_id\n    _ndb_struct_na_base_pair.j_label_seq_id\n    _ndb_struct_na_base_pair.j_symmetry\n    _ndb_struct_na_base_pair.shear\n    _ndb_struct_na_base_pair.stretch\n    _ndb_struct_na_base_pair.stagger\n    _ndb_struct_na_base_pair.buckle\n    _ndb_struct_na_base_pair.propeller\n    _ndb_struct_na_base_pair.opening\n     1 G A 1  1_555 C A 8  7_555   -0.396  -0.156 -0.018  -5.523  -6.752  -3.291\n     1 G A 2  1_555 C A 7  7_555   -0.094  -0.220 -0.334  -4.727  -9.765   2.311\n     1 G A 3  1_555 C A 6  7_555   -0.285  -0.239  0.008  -6.454 -12.575  -1.181\n  #    ...',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.model_number',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the model number of the the base pair.\n\n               This data item is a pointer to _atom_site.ndb_model in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.model_number' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.model_number' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_model_num' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.pair_number',
        [ [ 'item_',
            '_item_description.description',
            '\n               Sequential number of pair in the pair sequence.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair.pair_number' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.pair_name',
        [ [ 'item_',
            '_item_description.description',
            '\n               Text label for this base pair.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair.pair_name' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.i_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the asym id of the i-th base in the base pair.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.i_label_asym_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.i_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.i_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              \n               Describes the component id of the i-th base in the base pair.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.i_label_comp_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.i_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.i_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the sequence number of the i-th base in the base pair.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.i_label_seq_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.i_label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.i_symmetry',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               coordinates of the i-th base to generate the first partner \n               in the base pair.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair.i_symmetry' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.j_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the asym id of the j-th base in the base pair.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.j_label_asym_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.j_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.j_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              \n               Describes the component id of the j-th base in the base pair.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.j_label_comp_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.j_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.j_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the sequence number of the j-th base in the base pair.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.j_label_seq_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.j_label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.j_symmetry',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               coordinates of the j-th base to generate the second partner \n               in the base pair.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair.j_symmetry' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.i_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the asym id of the i-th base in the base pair.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.i_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.i_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.i_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the sequence number of the i-th base in the base pair.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.i_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.i_auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.i_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the PDB insertion code of the i-th base in the base pair.\n\n               This data item is a pointer to _atom_site.pdbx_PDB_ins_code in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.i_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ '@comment',
            '    _item_linked.child_name\t\'_ndb_struct_na_base_pair.i_PDB_ins_code\'' ],
          [ '@comment',
            '    _item_linked.parent_name    \'_atom_site.pdbx_PDB_ins_code\'' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.j_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the asym id of the j-th base in the base pair.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.j_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.j_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.j_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the sequence number of the j-th base in the base pair.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.j_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair.j_auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.j_PDB_ins_code',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the PDB insertion code of the j-th base in the base pair.\n\n               This data item is a pointer to _atom_site.pdbx_PDB_ins_code in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.j_PDB_ins_code' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ '@comment',
            '    _item_linked.child_name\t\'_ndb_struct_na_base_pair.j_PDB_ins_code\'' ],
          [ '@comment',
            '    _item_linked.parent_name    \'_atom_site.pdbx_PDB_ins_code\'' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.shear',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair shear parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair.shear' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.stretch',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair stretch parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair.stretch' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.stagger',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair stagger parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair.stagger' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.buckle',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair buckle parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair.buckle' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.propeller',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair propeller parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair.propeller' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.opening',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair opening parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair.opening' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.hbond_type_12',
        [ [ 'item_',
            '_item_description.description',
            '\n               Base pair classification of Westhoff and Leontis.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.hbond_type_12' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair.hbond_type_28',
        [ [ 'item_',
            '_item_description.description',
            '\n               Base pair classification of Saenger' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair.hbond_type_28' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '#################################' ],
      [ '@comment', '# NDB_STRUCT_NA_BASE_PAIR_STEP ##' ],
      [ '@comment', '#################################' ],
      [ 'save_',
        'ndb_struct_na_base_pair_step',
        [ [ 'item_',
            '_category.description',
            '              Data items in the NDB_STRUCT_NA_BASE_PAIR_STEP category record details \n               of base pair step interactions.' ],
          [ 'item_', '_category.id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_ndb_struct_na_base_pair_step.model_number',
              '_ndb_struct_na_base_pair_step.i_label_comp_id_1',
              '_ndb_struct_na_base_pair_step.i_label_asym_id_1',
              '_ndb_struct_na_base_pair_step.i_label_seq_id_1',
              '_ndb_struct_na_base_pair_step.i_symmetry_1',
              '_ndb_struct_na_base_pair_step.j_label_comp_id_1',
              '_ndb_struct_na_base_pair_step.j_label_asym_id_1',
              '_ndb_struct_na_base_pair_step.j_label_seq_id_1',
              '_ndb_struct_na_base_pair_step.j_symmetry_1',
              '_ndb_struct_na_base_pair_step.i_label_comp_id_2',
              '_ndb_struct_na_base_pair_step.i_label_asym_id_2',
              '_ndb_struct_na_base_pair_step.i_label_seq_id_2',
              '_ndb_struct_na_base_pair_step.i_symmetry_2',
              '_ndb_struct_na_base_pair_step.j_label_comp_id_2',
              '_ndb_struct_na_base_pair_step.j_label_asym_id_2',
              '_ndb_struct_na_base_pair_step.j_label_seq_id_2',
              '_ndb_struct_na_base_pair_step.j_symmetry_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'struct_group', 'ndb_group' ] ],
          [ 'loop_',
            [ '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n     loop_\n    _ndb_struct_na_base_pair_step.model_number\n    _ndb_struct_na_base_pair_step.i_label_comp_id_1\n    _ndb_struct_na_base_pair_step.i_label_asym_id_1\n    _ndb_struct_na_base_pair_step.i_label_seq_id_1\n    _ndb_struct_na_base_pair_step.i_symmetry_1\n    _ndb_struct_na_base_pair_step.j_label_comp_id_1\n    _ndb_struct_na_base_pair_step.j_label_asym_id_1\n    _ndb_struct_na_base_pair_step.j_label_seq_id_1\n    _ndb_struct_na_base_pair_step.j_symmetry_1\n    _ndb_struct_na_base_pair_step.i_label_comp_id_2\n    _ndb_struct_na_base_pair_step.i_label_asym_id_2\n    _ndb_struct_na_base_pair_step.i_label_seq_id_2\n    _ndb_struct_na_base_pair_step.i_symmetry_2\n    _ndb_struct_na_base_pair_step.j_label_comp_id_2\n    _ndb_struct_na_base_pair_step.j_label_asym_id_2\n    _ndb_struct_na_base_pair_step.j_label_seq_id_2\n    _ndb_struct_na_base_pair_step.j_symmetry_2\n    _ndb_struct_na_base_pair_step.shift\n    _ndb_struct_na_base_pair_step.slide\n    _ndb_struct_na_base_pair_step.rise\n    _ndb_struct_na_base_pair_step.tilt\n    _ndb_struct_na_base_pair_step.roll\n    _ndb_struct_na_base_pair_step.twist\n     1 G A 1  1_555 C A 8  7_555 G A 2  1_555 C A 7  7_555  0.369 -1.414 3.339  3.056 9.755 33.530\n     1 G A 2  1_555 C A 7  7_555 G A 3  1_555 C A 6  7_555  0.176 -1.672 3.371 -1.176 6.725 30.004\n#    ...',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.model_number',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the model number of the the base pair step.\n\n               This data item is a pointer to _atom_site.ndb_model in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.model_number' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.model_number' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.pdbx_PDB_model_num' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.step_number',
        [ [ 'item_',
            '_item_description.description',
            '\n               The sequence number of this step in the step sequence.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.step_number' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.step_name',
        [ [ 'item_',
            '_item_description.description',
            '\n               The text name of this step.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.step_name' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_label_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the asym id of the i-th base in the first base pair \n               of the step.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_label_asym_id_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.i_label_asym_id_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_label_comp_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              \n               Describes the component id of the i-th base in the first base pair\n               of the step.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_label_comp_id_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.i_label_comp_id_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_label_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the sequence number of the i-th base in the first base pair\n               of the step.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_label_seq_id_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.i_label_seq_id_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_symmetry_1',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               coordinates of the i-th base to generate the first partner \n               in the first base pair of the step.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_symmetry_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_label_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the asym id of the j-th base in the first base pair\n               of the step.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_label_asym_id_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.j_label_asym_id_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_label_comp_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              \n               Describes the component id of the j-th base in the first base pair \n               of the step.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_label_comp_id_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.j_label_comp_id_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_label_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the sequence number of the j-th base in the first base pair\n               of the step.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_label_seq_id_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.j_label_seq_id_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_symmetry_1',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               coordinates of the j-th base to generate the second partner \n               in the first base pair of the step.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_symmetry_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_label_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the asym id of the i-th base in the second base pair \n               of the step.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_label_asym_id_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.i_label_asym_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_label_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              \n               Describes the component id of the i-th base in the second base pair\n               of the step.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_label_comp_id_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.i_label_comp_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_label_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the sequence number of the i-th base in the second base pair\n               of the step.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_label_seq_id_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.i_label_seq_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_symmetry_2',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               coordinates of the i-th base to generate the first partner \n               in the second base pair of the step.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_symmetry_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_label_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the asym id of the j-th base in the second base pair\n               of the step.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_label_asym_id_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.j_label_asym_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_label_comp_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              \n               Describes the component id of the j-th base in the second base pair \n               of the step.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_label_comp_id_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.j_label_comp_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_label_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the sequence number of the j-th base in the second base pair\n               of the step.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_label_seq_id_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.j_label_seq_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_symmetry_2',
        [ [ 'item_',
            '_item_description.description',
            '              Describes the symmetry operation that should be applied to the\n               coordinates of the j-th base to generate the second partner \n               in the second base pair of the step.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_symmetry_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_default.value', '1_555' ],
          [ 'item_', '_item_type.code', 'symop' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '.',
              'no symmetry or translation to site',
              '4',
              '4th symmetry operation applied',
              '7_645',
              '7th symm. posn.; +a on x; -b on y' ] ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_auth_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the author\'s asym id of the i-th base in the first \n               base pair of the step.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_auth_asym_id_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.i_auth_asym_id_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_auth_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the author\'s sequence number of the i-th base in the \n               first base pair of the step.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_auth_seq_id_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.i_auth_seq_id_1' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_PDB_ins_code_1',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the PDB insertion code of the i-th base in the \n               first base pair of the step.\n\n               This data item is a pointer to _atom_site.pdbx_PDB_ins_code in \n               the ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_PDB_ins_code_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ '@comment',
            '    _item_linked.child_name\t\'_ndb_struct_na_base_pair_step.i_PDB_ins_code_1\'' ],
          [ '@comment',
            '    _item_linked.parent_name    \'_atom_site.pdbx_PDB_ins_code\'' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_auth_asym_id_1',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the author\'s asym id of the j-th base in the first \n               base pair of the step.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_auth_asym_id_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.j_auth_asym_id_1' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_auth_seq_id_1',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the author\'s sequence number of the j-th base in the \n               first base pair of the step.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_auth_seq_id_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.j_auth_seq_id_1' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_PDB_ins_code_1',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the PDB insertion code of the j-th base in the \n               first base pair of the step.\n\n               This data item is a pointer to _atom_site.pdbx_PDB_ins_code in \n               the ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_PDB_ins_code_1' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ '@comment',
            '    _item_linked.child_name\t\'_ndb_struct_na_base_pair_step.j_PDB_ins_code_1\'' ],
          [ '@comment',
            '    _item_linked.parent_name    \'_atom_site.pdbx_PDB_ins_code\'' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_auth_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the author\'s asym id of the i-th base in the second \n               base pair of the step.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_auth_asym_id_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.i_auth_asym_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_auth_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the author\'s sequence number of the i-th base in the \n               second base pair of the step.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_auth_seq_id_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.i_auth_seq_id_2' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.i_PDB_ins_code_2',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the PDB insertion code of the i-th base in the \n               second base pair of the step.\n\n               This data item is a pointer to _atom_site.pdbx_PDB_ins_code in \n               the ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.i_PDB_ins_code_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ '@comment',
            '    _item_linked.child_name\t\'_ndb_struct_na_base_pair_step.i_PDB_ins_code_2\'' ],
          [ '@comment',
            '    _item_linked.parent_name    \'_atom_site.pdbx_PDB_ins_code\'' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_auth_asym_id_2',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the author\'s asym id of the j-th base in the second \n               base pair of the step.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_auth_asym_id_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.j_auth_asym_id_2' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_auth_seq_id_2',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the author\'s sequence number of the j-th base in the \n               second base pair of the step.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_auth_seq_id_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_ndb_struct_na_base_pair_step.j_auth_seq_id_2' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.j_PDB_ins_code_2',
        [ [ 'item_',
            '_item_description.description',
            '\n               Describes the PDB insertion code of the j-th base in the \n               second base pair of the step.\n\n               This data item is a pointer to _atom_site.pdbx_PDB_ins_code in \n               the ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.j_PDB_ins_code_2' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ '@comment',
            '    _item_linked.child_name\t\'_ndb_struct_na_base_pair_step.j_PDB_ins_code_2\'' ],
          [ '@comment',
            '    _item_linked.parent_name    \'_atom_site.pdbx_PDB_ins_code\'' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.shift',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step shift parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair_step.shift' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.slide',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step slide parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair_step.slide' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.rise',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step rise parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair_step.rise' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.tilt',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step tilt parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair_step.tilt' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.roll',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step roll parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair_step.roll' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.twist',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step twist parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair_step.twist' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.x_displacement',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step X displacement parameter.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.x_displacement' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.y_displacement',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step Y displacement parameter.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.y_displacement' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.helical_rise',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step helical rise parameter.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.helical_rise' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.inclination',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step inclination parameter.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.inclination' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.tip',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step twist parameter.' ],
          [ 'item_', '_item.name', '_ndb_struct_na_base_pair_step.tip' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_ndb_struct_na_base_pair_step.helical_twist',
        [ [ 'item_',
            '_item_description.description',
            '\n               The value of the base pair step helical twist parameter.' ],
          [ 'item_',
            '_item.name',
            '_ndb_struct_na_base_pair_step.helical_twist' ],
          [ 'item_', '_item.category_id', 'ndb_struct_na_base_pair_step' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        'ndb_original_ndb_coordinates',
        [ [ 'item_',
            '_category.description',
            ' Placeholder category for PDB coordinate data.' ],
          [ 'item_', '_category.id', 'ndb_original_ndb_coordinates' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_ndb_original_ndb_coordinates.coord_section' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'ndb_group' ] ] ] ],
      [ 'save_',
        '_ndb_original_ndb_coordinates.coord_section',
        [ [ 'item_', '_item_description.description', '' ],
          [ 'item_',
            '_item.name',
            '_ndb_original_ndb_coordinates.coord_section' ],
          [ 'item_', '_item.category_id', 'ndb_original_ndb_coordinates' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ 'save_',
        'pdbx_entity_nonpoly',
        [ [ 'item_', '_category.description', '             ' ],
          [ 'item_', '_category.id', 'pdbx_entity_nonpoly' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_entity_nonpoly.entity_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'entity_group', 'rcsb_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_entity_nonpoly.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_pdbx_entity_nonpoly.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_nonpoly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_nonpoly.entity_id' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_nonpoly.entity_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_entity_nonpoly.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP category.' ],
          [ 'item_', '_item.name', '_pdbx_entity_nonpoly.comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_nonpoly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_nonpoly.comp_id' ],
          [ 'item_', '_item_linked.parent_name', '_chem_comp.id' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_nonpoly.comp_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_entity_nonpoly.name',
        [ [ 'item_',
            '_item_description.description',
            '              A name for the non-polymer entity' ],
          [ 'item_', '_item.name', '_pdbx_entity_nonpoly.name' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_nonpoly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_rcsb_entity_nonpoly.name' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '## EOF mmcif_pdbx-def-4.dic' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment',
        ' File:  mmcif_pdbx-def-5.dic                                             ' ],
      [ '@comment', ' ' ],
      [ '@comment',
        '                      PDB Exchange Data Dictionary' ],
      [ '@comment',
        ' This data dictionary contains definitions used by wwPDB for data exchange' ],
      [ '@comment', ' and data processing.  ' ],
      [ '@comment',
        '                           Definition Section 5' ],
      [ '@comment',
        '                 This section primarily contains extensions for' ],
      [ '@comment',
        '                      data extraction by PDB_EXTRACT.        ' ],
      [ '@comment',
        '                                                                        ' ],
      [ '@comment',
        '##########################################################################' ],
      [ 'save_',
        'pdbx_phasing_dm',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_PHASING_DM category record details about \n               density modification ' ],
          [ 'item_', '_category.id', 'pdbx_phasing_dm' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_phasing_dm.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - density modification from resolve',
              '\n    _pdbx_phasing_dm.entry_id          ABC001 \n    _pdbx_phasing_dm.fom_acentric      0.85 \n    _pdbx_phasing_dm.fom_centric       0.79 \n    _pdbx_phasing_dm.fom               0.85 \n    _pdbx_phasing_dm.reflns_acentric   11351 \n    _pdbx_phasing_dm.reflns_centric    1135 \n    _pdbx_phasing_dm.reflns            12486 ',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.entry_id identifies the data block.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.method',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.method identifies the method used for \n       density modification' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.method' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm.method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.mask_type',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.mask_type  identifies the type of mask used for \n       density modification' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.mask_type' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm.mask_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.fom_acentric',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.fom_acentric  identifies the figure of merit \n       for acentric data' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.fom_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm.fom_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.fom_centric',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.fom_centric  identifies the figure of merit \n       for acentric data' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.fom_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm.fom_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.fom',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.fom identifies the figure of merit \n       for all the data' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.fom' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_aliases.alias_name', '_pdbx_phasing_dm.fom' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.reflns_acentric',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.reflns_acentric  identifies the number\n       of acentric reflections.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.reflns_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm.reflns_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.reflns_centric',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.reflns_centric  identifies the number\n       of centric reflections.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.reflns_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm.reflns_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.reflns',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.reflns  identifies the number\n       of centric and acentric reflections.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.reflns' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm.reflns' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.delta_phi_initial',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.delta_phi_initial  identifies phase difference \n       before density modification' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.delta_phi_initial' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm.delta_phi_initial' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm.delta_phi_final',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm.delta_phi_final  identifies phase difference \n       after density modification' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm.delta_phi_final' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm.delta_phi_final' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '## next category,  _pdbx_phasing_dm_shell ###' ],
      [ 'save_',
        'pdbx_phasing_dm_shell',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_PHASING_DM_SHELL category record details about \n               density modification in resolution shell.' ],
          [ 'item_', '_category.id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_phasing_dm_shell.d_res_low',
              '_pdbx_phasing_dm_shell.d_res_high' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - density modification with shells',
              '\n   loop_\n   _pdbx_phasing_dm_shell.d_res_low \n   _pdbx_phasing_dm_shell.d_res_high \n   _pdbx_phasing_dm_shell.reflns \n   _pdbx_phasing_dm_shell.fom \n   _pdbx_phasing_dm_shell.delta_phi_final \n   100.00 7.73 502 0.879 24.7 \n   7.73   6.24 506 0.857 29.2 \n   6.24   5.50 504 0.838 29.2 \n   5.50   5.02 502 0.851 25.3 \n   5.02   4.67 503 0.831 22.7 \n#   ....... ',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm_shell.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm_shell.d_res_high  identifies high resolution \n       ' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm_shell.d_res_high' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm_shell.d_res_high' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm_shell.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm_shell.d_res_low  identifies low resolution \n       ' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm_shell.d_res_low' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm_shell.d_res_low' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm_shell.fom_acentric',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm_shell.fom_acentric  identifies the figure of merit \n       for acentric data with resolution shells' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm_shell.fom_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm_shell.fom_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm_shell.fom_centric',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm_shell.fom_centric  identifies the figure of merit \n       for centric data with resolution shells.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm_shell.fom_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm_shell.fom_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm_shell.fom',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm_shell.fom identifies the figure of merit \n       for all the data with resolution shells.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm_shell.fom' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm_shell.fom' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm_shell.reflns_acentric',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm_shell.reflns_acentric  identifies the number\n       of acentric reflections with resolution shells.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_dm_shell.reflns_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm_shell.reflns_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm_shell.reflns_centric',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm_shell.reflns_centric  identifies the number\n       of centric reflections with resolution shells.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_dm_shell.reflns_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm_shell.reflns_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm_shell.reflns',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm_shell.reflns  identifies the number\n       of centric and acentric reflections with resolution shells.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_dm_shell.reflns' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm_shell.reflns' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm_shell.delta_phi_initial',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm_shell.delta_phi_initial  identifies phase difference \n       before density modification with resolution shells.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_dm_shell.delta_phi_initial' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm_shell.delta_phi_initial' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_dm_shell.delta_phi_final',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_dm_shell.delta_phi_final  identifies phase difference \n       after density modification with resolution shells.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_dm_shell.delta_phi_final' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_dm_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_dm_shell.delta_phi_final' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_d_res_low records the lowest resolution\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_d_res_low' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_d_res_low' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_d_res_high records the highest resolution\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_d_res_high' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_d_res_high' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_reflns_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_reflns_acentric records the number of \n               acentric reflections for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_reflns_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_reflns_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_reflns_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_reflns_centric records the number of \n               centric reflections for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_reflns_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_reflns_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_reflns',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_reflns records the number of \n               reflections used for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_reflns' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_reflns' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_fom_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_fom_acentric  records the figure of merit\n               using acentric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_fom_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_fom_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_fom_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_fom_centric  records the figure of merit\n               using centric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_fom_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_fom_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_fom',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_fom records the figure of merit\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_fom' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_aliases.alias_name', '_phasing_MAD.pdbx_fom' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_R_cullis_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_R_cullis_centric records R_cullis\n               using centric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_R_cullis_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_R_cullis_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_R_cullis_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_R_cullis_acentric records R_cullis\n               using acentric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_R_cullis_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_R_cullis_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_R_cullis',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_R_cullis records R_cullis\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_R_cullis' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_R_cullis' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_R_kraut_centric',
        [ [ 'item_',
            '_item_description.description',
            '             _phasing_MAD.pdbx_R_kraut_centric records R_kraut\n               using centric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_R_kraut_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_R_kraut_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_R_kraut_acentric',
        [ [ 'item_',
            '_item_description.description',
            '             _phasing_MAD.pdbx_R_kraut_acentric records R_kraut\n               using acentric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_R_kraut_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_R_kraut_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_R_kraut',
        [ [ 'item_',
            '_item_description.description',
            '             _phasing_MAD.pdbx_R_kraut records R_kraut\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_R_kraut' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_R_kraut' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_loc_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_loc_centric records lack of closure\n               using centric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_loc_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_loc_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_loc_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_loc_acentric records lack of closure\n               using acentric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_loc_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_loc_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_loc',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_loc records lack of closure\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_loc' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_aliases.alias_name', '_phasing_MAD.pdbx_loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_power_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_power_centric records phasing power\n               using centric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_power_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_power_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_power_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_power_acentric records phasing power\n               using acentric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_power_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_power_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_power',
        [ [ 'item_',
            '_item_description.description',
            '              _phasing_MAD.pdbx_power records phasing power\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_power' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_power' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_number_data_sets',
        [ [ 'item_',
            '_item_description.description',
            '             _phasing_MAD.pdbx_loc records the number \n               of data sets used for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_number_data_sets' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_number_data_sets' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD.pdbx_anom_scat_method',
        [ [ 'item_',
            '_item_description.description',
            '             _phasing_MAD.pdbx_anom_scat_method records the method\n               used to locate anomalous scatterers for MAD phasing.' ],
          [ 'item_', '_item.name', '_phasing_MAD.pdbx_anom_scat_method' ],
          [ 'item_', '_item.category_id', 'phasing_MAD' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD.pdbx_anom_scat_method' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        'pdbx_phasing_MAD_shell',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_PHASING_MAD_SHELL category record details about\n               the phasing of the structure, when methods involving multiple\n               anomalous dispersion techniques are involved (note: the\n               values are overall, but broken down into shells of resolution) \n' ],
          [ 'item_', '_category.id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_phasing_MAD_shell.d_res_low',
              '_pdbx_phasing_MAD_shell.d_res_high' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n    loop_\n    _pdbx_phasing_MAD_shell.d_res_low \n    _pdbx_phasing_MAD_shell.d_res_high \n    _pdbx_phasing_MAD_shell.reflns_acentric \n    _pdbx_phasing_MAD_shell.fom_acentric \n    _pdbx_phasing_MAD_shell.reflns_centric \n    _pdbx_phasing_MAD_shell.fom_centric \n    22.60 7.77 64  0.886 23 0.641 \n    7.77  5.67 132 0.863 32 0.642 \n    5.67  4.68 182 0.842 27 0.737 \n    4.68  4.07 209 0.789 24 0.682 \n    4.07  3.65 246 0.772 27 0.633 \n    3.65  3.34 260 0.752 31 0.700 ',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.d_res_low records the lower resolution\n               for the shell.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.d_res_low' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.d_res_low' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.d_res_high records the higher resolution\n               for the shell.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.d_res_high' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.d_res_high' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.reflns_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.reflns_acentric records the number of \n               acentric reflections for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_shell.reflns_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.reflns_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.reflns_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.reflns_centric records the number of \n               centric reflections for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_shell.reflns_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.reflns_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.reflns',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.reflns records the number of \n               reflections used for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.reflns' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.reflns' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.fom_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.fom_acentric  records the figure of merit\n               using acentric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.fom_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.fom_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.fom_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.fom_centric  records the figure of merit\n               using centric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.fom_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.fom_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.fom',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.fom records the figure of merit\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.fom' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.fom' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.R_cullis_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.R_cullis_centric records R_cullis\n               using centric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_shell.R_cullis_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.R_cullis_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.R_cullis_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.R_cullis_acentric records R_cullis\n               using acentric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_shell.R_cullis_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.R_cullis_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.R_cullis',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.R_cullis records R_cullis\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.R_cullis' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.R_cullis' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.R_kraut_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.R_kraut_centric records R_kraut\n               using centric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_shell.R_kraut_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.R_kraut_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.R_kraut_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.r_kraut_acentric records R_kraut\n               using acentric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_shell.R_kraut_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.R_kraut_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.R_kraut',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.R_kraut records R_kraut\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.R_kraut' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.R_kraut' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.loc_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.loc_centric records lack of closure\n               using centric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.loc_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.loc_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.loc_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.loc_acentric records lack of closure\n               using acentric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.loc_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.loc_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.loc',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.loc records lack of closure\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.loc' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.power_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.power_centric records phasing powe\n               using centric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_shell.power_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.power_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.power_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.power_acentric records phasing powe\n               using acentric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_shell.power_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.power_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_shell.power',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_shell.loc records phasing power\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_shell.power' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_shell.power' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '## next category,  _pdbx_phasing_MAD_set ###' ],
      [ 'save_',
        'pdbx_phasing_MAD_set',
        [ [ 'item_',
            '_category.description',
            '              Record details about each phasing set: (Note: the phasing \n               set is different from data set. for example: if there are \n               three data sets, the inflection point (IP), the peak (PK) \n               and the high remote (HR), the combination of the phasing \n               set will be IP_iso, PK_iso (the isomorphous repleacement \n               with HR as \'native\'), IP_ano, PK_ano and HR_ano (the \n               anomalous difference with itself). Therefore, there are \n               five set used for phasing.\n' ],
          [ 'item_', '_category.id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_phasing_MAD_set.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 -  three wavelengths ',
              '\n       \n    loop_\n    _pdbx_phasing_MAD_set.id \n    _pdbx_phasing_MAD_set.d_res_low \n    _pdbx_phasing_MAD_set.d_res_high \n    _pdbx_phasing_MAD_set.reflns_acentric \n    _pdbx_phasing_MAD_set.reflns_centric \n    _pdbx_phasing_MAD_set.R_cullis_acentric \n    _pdbx_phasing_MAD_set.R_cullis_centric \n    ISO_1 22.60 2.00 5387 471 0.000 0.000 \n    ISO_2 22.60 2.00 5365 469 0.803 0.718 \n    ISO_3 22.60 2.00 5317 460 0.658 0.500 \n    ANO_1 22.60 2.00 5278 0   0.841 0.000 \n    ANO_2 22.60 2.00 5083 0   0.649 0.000 \n    ANO_3 22.60 2.00 5329 0   0.829 0.000 \n',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.id records phase set name\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.id' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.d_res_low records the lowerest \n               resolution for phasing set.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.d_res_low' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.d_res_low' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.d_res_high records the highest resolution\n               for the phasing set.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.d_res_high' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.d_res_high' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.number_of_sites',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.number_of_sites records the number of site\n               refined for the phasing set.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set.number_of_sites' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.number_of_sites' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.reflns_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.reflns_acentric records the number of \n               acentric reflections for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set.reflns_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.reflns_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.reflns_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.reflns_centric records the number of \n               centric reflections for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.reflns_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.reflns_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.reflns',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.reflns records the number of \n               reflections used for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.reflns' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.reflns' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.fom_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.fom_acentric  records the figure of merit\n               using acentric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.fom_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.fom_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.fom_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.fom_centric  records the figure of merit\n               using centric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.fom_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.fom_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.fom',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.fom records the figure of merit\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.fom' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.fom' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.R_cullis_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.R_cullis_centric records R_cullis\n               using centric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set.R_cullis_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.R_cullis_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.R_cullis_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.R_cullis_acentric records R_cullis\n               using acentric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set.R_cullis_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.R_cullis_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.R_cullis',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.R_cullis records R_cullis\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.R_cullis' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.R_cullis' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.R_kraut_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.R_kraut_centric records r_kraut\n               using centric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set.R_kraut_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.R_kraut_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.R_kraut_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.r_kraut_acentric records r_kraut\n               using acentric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set.R_kraut_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.R_kraut_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.R_kraut',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.R_kraut records R_kraut\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.R_kraut' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.R_kraut' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.loc_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.loc_centric records lack of closure\n               using centric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.loc_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.loc_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.loc_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.loc_acentric records lack of closure\n               using acentric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.loc_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.loc_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.loc',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.loc records lack of closure\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.loc' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.power_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.power_centric records phasing powe\n               using centric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.power_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.power_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.power_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.power_acentric records phasing powe\n               using acentric data for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.power_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.power_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set.power',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set.power records phasing power\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set.power' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set.power' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment',
        '## next category,  _pdbx_phasing_MAD_set_shell ###' ],
      [ 'save_',
        'pdbx_phasing_MAD_set_shell',
        [ [ 'item_',
            '_category.description',
            '              The same as category pdbx_phasing_MAD_set, but\n               broken into shells.\n' ],
          [ 'item_', '_category.id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_phasing_MAD_set_shell.id',
              '_pdbx_phasing_MAD_set_shell.d_res_low',
              '_pdbx_phasing_MAD_set_shell.d_res_high' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 -  three wavelengths (SHARP example)',
              '\n       \n    loop_\n    _pdbx_phasing_MAD_set_shell.id \n    _pdbx_phasing_MAD_set_shell.d_res_low \n    _pdbx_phasing_MAD_set_shell.d_res_high \n    _pdbx_phasing_MAD_set_shell.reflns_acentric \n    _pdbx_phasing_MAD_set_shell.reflns_centric \n    _pdbx_phasing_MAD_set_shell.R_cullis_acentric \n    _pdbx_phasing_MAD_set_shell.R_cullis_centric \n    _pdbx_phasing_MAD_set_shell.power_acentric \n    _pdbx_phasing_MAD_set_shell.power_centric \n    ISO_1 22.60 7.77 64  23 0.000 0.000 0.000 0.000 \n    ISO_1 7.77  5.67 130 32 0.000 0.000 0.000 0.000 \n    ISO_1 5.67  4.68 182 27 0.000 0.000 0.000 0.000 \n    ISO_1 4.68  4.07 207 24 0.000 0.000 0.000 0.000 \n    ANO_1 22.60 7.77 62  0  0.610 0.000 1.804 0.000 \n    ANO_1 7.77  5.67 129 0  0.532 0.000 2.382 0.000 \n    ANO_1 5.67  4.68 178 0  0.673 0.000 1.858 0.000 \n    ANO_1 4.68  4.07 204 0  0.755 0.000 1.605 0.000 \n#    .........',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.id records phase set name\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_shell.id' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.d_res_low records the lowerest \n               resolution for phasing set.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.d_res_low' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.d_res_low' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.d_res_high records the highest resolution\n               for the phasing set.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.d_res_high' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.d_res_high' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.reflns_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.reflns_acentric records the number of \n               acentric reflections for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.reflns_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.reflns_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.reflns_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.reflns_centric records the number of \n               centric reflections for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.reflns_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.reflns_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.reflns',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.reflns records the number of \n               reflections used for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_shell.reflns' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.reflns' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.fom_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.fom_acentric  records the figure of merit\n               using acentric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.fom_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.fom_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.fom_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.fom_centric  records the figure of merit\n               using centric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.fom_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.fom_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.fom',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.fom records the figure of merit\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_shell.fom' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.fom' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.R_cullis_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.R_cullis_centric records R_cullis\n               using centric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.R_cullis_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.R_cullis_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.R_cullis_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.R_cullis_acentric records R_cullis\n               using acentric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.R_cullis_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.R_cullis_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.R_cullis',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.R_cullis records R_cullis\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_shell.R_cullis' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.R_cullis' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.R_kraut_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.R_kraut_centric records R_kraut\n               using centric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.R_kraut_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.R_kraut_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.R_kraut_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.R_kraut_acentric records R_kraut\n               using acentric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.R_kraut_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.R_kraut_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.R_kraut',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.R_kraut records R_kraut\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_shell.R_kraut' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.R_kraut' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.loc_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.loc_centric records lack of closure\n               using centric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.loc_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.loc_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.loc_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.loc_acentric records lack of closure\n               using acentric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.loc_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.loc_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.loc',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.loc records lack of closure\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_shell.loc' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.power_centric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.power_centric records phasing power\n               using centric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.power_centric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.power_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.power_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.power_acentric records phasing power\n               using acentric data for MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_shell.power_acentric' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.power_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_shell.power',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_shell.power records phasing power\n               for MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_shell.power' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_shell.power' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment',
        '## next category,  _pdbx_phasing_MAD_set_site ###' ],
      [ 'save_',
        'pdbx_phasing_MAD_set_site',
        [ [ 'item_',
            '_category.description',
            '              record the details (coordinates etc.) of anomalous scatters.\n               \n' ],
          [ 'item_', '_category.id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_phasing_MAD_set_site.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 -  anomalous scatters is Se',
              '\n       \n    loop_\n    _pdbx_phasing_MAD_set_site.id \n    _pdbx_phasing_MAD_set_site.atom_type_symbol \n    _pdbx_phasing_MAD_set_site.Cartn_x \n    _pdbx_phasing_MAD_set_site.Cartn_y \n    _pdbx_phasing_MAD_set_site.Cartn_z \n    _pdbx_phasing_MAD_set_site.B_iso \n    _pdbx_phasing_MAD_set_site.Occupancy \n    1  SE 25.9407  -0.103471 17.4094 15.2561 1 \n    2  SE 30.6534  6.62359   9.93063 12.9102 1 \n    3  SE -3.26506 15.5546   53.9529 30.5239 1 \n#    .......',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.id records the number of site\n               obtained from MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.id' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.atom_type_symbol',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.atom_type_symbol records the name of site\n               obtained from MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_site.atom_type_symbol' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.atom_type_symbol' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.Cartn_x',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.Cartn_x  records the X  Cartesian\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.Cartn_x' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.Cartn_x' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.Cartn_y',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.Cartn_y  records the Y  Cartesian\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.Cartn_y' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.Cartn_y' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.Cartn_z',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.Cartn_z  records the Z  Cartesian\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.Cartn_z' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.Cartn_z' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.Cartn_x_esd',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.Cartn_x_esd  records the estimated\n               standard deviation X  Cartesian\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_site.Cartn_x_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.Cartn_x_esd' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.Cartn_y_esd',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.Cartn_y_esd  records the estimated\n               standard deviation Y  Cartesian\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_site.Cartn_y_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.Cartn_y_esd' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.Cartn_z_esd',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.Cartn_z_esd  records the estimated \n               standard deviation Z  Cartesian\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_site.Cartn_z_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.Cartn_z_esd' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.fract_x',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.fract_x  records the X  fractional\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.fract_x' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.fract_x' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.fract_y',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.fract_y  records the Y  fractional\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.fract_y' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.fract_y' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.fract_z',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.fract_z  records the Z  fractional\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.fract_z' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.fract_z' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.fract_x_esd',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.fract_x_esd  records the estimated\n               standard deviation X  fractional\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_site.fract_x_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.fract_x_esd' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.fract_y_esd',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.fract_y_esd  records the estimated\n               standard deviation Y  fractional\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_site.fract_y_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.fract_y_esd' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.fract_z_esd',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.fract_z_esd  records the estimated \n               standard deviation Z  fractional\n                coordinate of site obtained from MAD phasing.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_site.fract_z_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.fract_z_esd' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.b_iso',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.b_iso  records isotropic \n               temperature factor parameterthe for the site\n               obtained from MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.b_iso' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.b_iso' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.b_iso_esd',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.b_iso_esd  records estimated \n               standard deviation of isotropic \n               temperature factor parameterthe for the site\n               obtained from MAD phasing.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.b_iso_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.b_iso_esd' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.occupancy',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.occupancy  records the fraction \n               of the atom type presented at this site.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.occupancy' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.occupancy' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.occupancy_esd',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_phasing_MAD_set_site.occupancy_esd  records estimated \n               standard deviation of the fraction \n               of the atom type presented at this site.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_site.occupancy_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.occupancy_esd' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '## new items for _phasing_MAD_set  ###' ],
      [ 'save_',
        '_phasing_MAD_set.pdbx_atom_type',
        [ [ 'item_',
            '_item_description.description',
            '              record the type of heavy atoms which produce anomolous singal.' ],
          [ 'item_', '_item.name', '_phasing_MAD_set.pdbx_atom_type' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD_set.pdbx_atom_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD_set.pdbx_f_prime_refined',
        [ [ 'item_',
            '_item_description.description',
            '              record the refined f_prime (not from experiment).' ],
          [ 'item_',
            '_item.name',
            '_phasing_MAD_set.pdbx_f_prime_refined' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD_set.pdbx_f_prime_refined' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MAD_set.pdbx_f_double_prime_refined',
        [ [ 'item_',
            '_item_description.description',
            '              record the refined f_double_prime (not from experiment).' ],
          [ 'item_',
            '_item.name',
            '_phasing_MAD_set.pdbx_f_double_prime_refined' ],
          [ 'item_', '_item.category_id', 'phasing_MAD_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MAD_set.pdbx_f_double_prime_refined' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment',
        '## new items for _pdbx_phasing_MAD_set_site  ###' ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.set_id',
        [ [ 'item_',
            '_item_description.description',
            '              record the phasing set.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MAD_set_site.set_id' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.set_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MAD_set_site.occupancy_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The relative real isotropic occupancy of the atom type\n               present at this heavy-atom site in a given atom site.              ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MAD_set_site.occupancy_iso' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MAD_set_site' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MAD_set_site.occupancy_iso' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '## new items for _refine_ls_restr_ncs  ###' ],
      [ 'save_',
        '_refine_ls_restr_ncs.pdbx_type',
        [ [ 'item_',
            '_item_description.description',
            '              record the type of NCS restraint. (for example: tight positional)' ],
          [ 'item_', '_item.name', '_refine_ls_restr_ncs.pdbx_type' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_restr_ncs.pdbx_type' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine_ls_restr_ncs.pdbx_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              record the chain ID.' ],
          [ 'item_', '_item.name', '_refine_ls_restr_ncs.pdbx_asym_id' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_restr_ncs.pdbx_asym_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine_ls_restr_ncs.pdbx_number',
        [ [ 'item_',
            '_item_description.description',
            '              record the residue number .' ],
          [ 'item_', '_item.name', '_refine_ls_restr_ncs.pdbx_number' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_restr_ncs.pdbx_number' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine_ls_restr_ncs.pdbx_rms',
        [ [ 'item_',
            '_item_description.description',
            '              record the standard divation between one segment to another' ],
          [ 'item_', '_item.name', '_refine_ls_restr_ncs.pdbx_rms' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_restr_ncs.pdbx_rms' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_refine_ls_restr_ncs.pdbx_weight',
        [ [ 'item_',
            '_item_description.description',
            '              record the weight used for NCS restraint.' ],
          [ 'item_', '_item.name', '_refine_ls_restr_ncs.pdbx_weight' ],
          [ 'item_', '_item.category_id', 'refine_ls_restr_ncs' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_ls_restr_ncs.pdbx_weight' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '## new items for _struct_ncs_dom_lim  ###' ],
      [ 'save_',
        '_struct_ncs_dom_lim.pdbx_component_id',
        [ [ 'item_',
            '_item_description.description',
            '              record the number of component used for NCS.' ],
          [ 'item_',
            '_item.name',
            '_struct_ncs_dom_lim.pdbx_component_id' ],
          [ 'item_', '_item.category_id', 'struct_ncs_dom_lim' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ncs_dom_lim.pdbx_component_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_struct_ncs_dom_lim.pdbx_refine_code',
        [ [ 'item_',
            '_item_description.description',
            '              record the refinement code number (from CCP4.)' ],
          [ 'item_', '_item.name', '_struct_ncs_dom_lim.pdbx_refine_code' ],
          [ 'item_', '_item.category_id', 'struct_ncs_dom_lim' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_struct_ncs_dom_lim.pdbx_refine_code' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '## new items for _refine_analyze  ###' ],
      [ 'save_',
        '_refine_analyze.pdbx_Luzzati_d_res_high_obs',
        [ [ 'item_',
            '_item_description.description',
            '              record the high resolution for calculating Luzzati statistics.' ],
          [ 'item_',
            '_item.name',
            '_refine_analyze.pdbx_Luzzati_d_res_high_obs' ],
          [ 'item_', '_item.category_id', 'refine_analyze' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_refine_analyze.pdbx_Luzzati_d_res_high_obs' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '-------------------------------' ],
      [ '@comment', '## new items for _phasing_MIR_shell  ###' ],
      [ 'save_',
        '_phasing_MIR.pdbx_number_derivatives',
        [ [ 'item_',
            '_item_description.description',
            '              The number of derivatives used in this phasing experiment.' ],
          [ 'item_', '_item.name', '_phasing_MIR.pdbx_number_derivatives' ],
          [ 'item_', '_item.category_id', 'phasing_MIR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.pdbx_loc_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record lack of closure from centric reflection for each shell.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.pdbx_loc_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.pdbx_loc_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.pdbx_loc_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record lack of closure from acentric reflection for each shell.' ],
          [ 'item_', '_item.name', '_phasing_MIR_shell.pdbx_loc_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.pdbx_loc_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.pdbx_power_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record phasing power from centric reflection for each shell.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_shell.pdbx_power_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.pdbx_power_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.pdbx_power_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record phasing power from acentric reflection for each shell.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_shell.pdbx_power_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.pdbx_power_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.pdbx_R_kraut_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record R_Kraut from from centric reflection for each shell.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_shell.pdbx_R_kraut_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.pdbx_R_kraut_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.pdbx_R_kraut_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record R_kraut from from acentric reflection for each shell.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_shell.pdbx_R_kraut_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.pdbx_R_kraut_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.pdbx_R_cullis_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record R_Cullis from from centric reflection for each shell.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_shell.pdbx_R_cullis_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.pdbx_R_cullis_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_shell.pdbx_R_cullis_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record R_Cullis from from acentric reflection for each shell.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_shell.pdbx_R_cullis_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_shell.pdbx_R_cullis_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '## new items for _phasing_MIR_der  ###' ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_R_kraut_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record R_kraut obtained from centric data for each derivative.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der.pdbx_R_kraut_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_R_kraut_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_R_kraut_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record R_kraut obtained from acentric data for each derivative.' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der.pdbx_R_kraut_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_R_kraut_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_R_kraut',
        [ [ 'item_',
            '_item_description.description',
            '              record R_kraut obtained from all data data for each derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.pdbx_R_kraut' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_R_kraut' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_loc_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record lack of closure obtained from centric data for \n               each derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.pdbx_loc_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_loc_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_loc_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record lack of closure obtained from acentric data for \n               each derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.pdbx_loc_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_loc_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_loc',
        [ [ 'item_',
            '_item_description.description',
            '              record lack of closure obtained from all data for \n               each derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.pdbx_loc' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_loc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_fom_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record figure of merit obtained from centric data for \n               each derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.pdbx_fom_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_fom_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_fom_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record figure of merit obtained from acentric data for \n               each derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.pdbx_fom_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_fom_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_fom',
        [ [ 'item_',
            '_item_description.description',
            '              record figure of merit obtained from all data for \n               each derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.pdbx_fom' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_fom' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_power',
        [ [ 'item_',
            '_item_description.description',
            '              record phasing power for each derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.pdbx_power' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_power' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_R_cullis',
        [ [ 'item_',
            '_item_description.description',
            '              record R_cullis for each derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.pdbx_R_cullis' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_R_cullis' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der.pdbx_reflns',
        [ [ 'item_',
            '_item_description.description',
            '              record number of reflections used for each derivative.' ],
          [ 'item_', '_item.name', '_phasing_MIR_der.pdbx_reflns' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der.pdbx_reflns' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '## new items for _phasing_MIR_der_shell  ###' ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_R_cullis_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record R Cullis obtained from centric data for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_R_cullis_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_R_cullis_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_R_cullis_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record R Cullis obtained from acentric data for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_R_cullis_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_R_cullis_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_R_kraut_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record R Kraut obtained from centric data for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_R_kraut_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_R_kraut_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_R_kraut_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record R Kraut obtained from acentric data for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_R_kraut_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_R_kraut_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_loc_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record lack of closure obtained from centric data for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_loc_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_loc_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_loc_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record lack of closure obtained from acentric data for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_loc_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_loc_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_power_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record phasing power obtained from centric data for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_power_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_power_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_power_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record phasing power obtained from acentric data for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_power_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_power_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_fom_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record figure of merit obtained from centric data for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_fom_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_fom_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_fom_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record figure of merit obtained from acentric data for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_fom_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_fom_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_reflns_centric',
        [ [ 'item_',
            '_item_description.description',
            '              record  number of centric reflections used for phasing for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_reflns_centric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_reflns_centric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_phasing_MIR_der_shell.pdbx_reflns_acentric',
        [ [ 'item_',
            '_item_description.description',
            '              record  number of acentric reflections used for phasing for each\n               derivative, but broken into resolution shells' ],
          [ 'item_',
            '_item.name',
            '_phasing_MIR_der_shell.pdbx_reflns_acentric' ],
          [ 'item_', '_item.category_id', 'phasing_MIR_der_shell' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_phasing_MIR_der_shell.pdbx_reflns_acentric' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '## category,  _pdbx_phasing_MR' ],
      [ 'save_',
        'pdbx_phasing_MR',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_PHASING_MR category record details about \n               molecular replacement. ' ],
          [ 'item_', '_category.id', 'pdbx_phasing_MR' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_phasing_MR.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'phasing_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - molecular replacement example from program CNS.',
              '\n    _pdbx_phasing_MR.entry_id           ABC001 \n    _pdbx_phasing_MR.method_rotation   \'real-space rotation search\'\n    _pdbx_phasing_MR.d_res_high_rotation 3.8\n    _pdbx_phasing_MR.d_res_low_rotation  13.0 \n    _pdbx_phasing_MR.sigma_F_rotation    1.0\n    _pdbx_phasing_MR.reflns_percent_rotation 97.8\n    _pdbx_phasing_MR.method_translation  \'gerneral using PC-refinement= e2e2\'\n    _pdbx_phasing_MR.d_res_high_translation 4.0\n    _pdbx_phasing_MR.d_res_low_translation  15.0\n    _pdbx_phasing_MR.sigma_F_translation  0\n    _pdbx_phasing_MR.reflns_percent_translation 97.7\n    _pdbx_phasing_MR.correlation_coeff_Fo_to_Fc 0.586\n    _pdbx_phasing_MR.packing  0.3086',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.entry_id identifies the data block.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.entry_id' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.method_rotation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.method_rotation identifies the method\n       used for rotation search. For example, the rotation method may be\n       realspace,  fastdirect, or direct.\n.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.method_rotation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.method_rotation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.d_res_high_rotation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.d_res_high_rotation identifies \n       the highest resolution used for rotation search.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.d_res_high_rotation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.d_res_high_rotation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.d_res_low_rotation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.d_res_low_rotation identifies \n       the lowest resolution used for rotation search.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.d_res_low_rotation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.d_res_low_rotation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.sigma_F_rotation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.sigma_F_rotation identifies the \n       sigma cut off of structure factor used for rotation search. ' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.sigma_F_rotation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.sigma_F_rotation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.sigma_I_rotation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.sigma_I_rotation identifies the \n       sigma cut off of intensity used for rotation search. ' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.sigma_I_rotation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.sigma_I_rotation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.reflns_percent_rotation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.reflns_percent_rotation identifies the \n       completness of data used for rotation search.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MR.reflns_percent_rotation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.reflns_percent_rotation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.method_translation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.method_translation identifies the method\n       used for translation search. For example in CNS, the translation method\n       may be "general" or "phased" with PC refinement target using\n       "fastf2f2" "e2e2" "e1e1" "f2f2" "f1f1" "residual" "vector".\n         \n.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.method_translation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.method_translation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.d_res_high_translation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.d_res_high_translation identifies \n       the highest resolution used for translation search.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MR.d_res_high_translation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.d_res_high_translation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.d_res_low_translation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.d_res_low_translation identifies \n       the lowest resolution used for translation search.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MR.d_res_low_translation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.d_res_low_translation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.sigma_F_translation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.sigma_F_translation identifies the \n       sigma cut off of structure factor used for translation search. ' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.sigma_F_translation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.sigma_F_translation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.sigma_I_translation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.sigma_I_translation identifies the \n       sigma cut off of intensity used for translation search. ' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.sigma_I_translation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.sigma_I_translation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.reflns_percent_translation',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.reflns_percent_translation identifies the \n       completness of data used for translation search.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MR.reflns_percent_translation' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.reflns_percent_translation' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.correlation_coeff_Io_to_Ic',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.correlation_coeff_Io_to_Ic identifies\n       the correlation between the observed and the calculated intensity\n       (~|F|^2) after rotation and translation. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MR.correlation_coeff_Io_to_Ic' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.correlation_coeff_Io_to_Ic' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.correlation_coeff_Fo_to_Fc',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.correlation_coeff_Fo_to_Fc identifies\n       the correlation between the observed and the calculated structure\n       factor after rotation and translation. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_phasing_MR.correlation_coeff_Fo_to_Fc' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.correlation_coeff_Fo_to_Fc' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.R_factor',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.R_factor identifies the R factor\n       (defined as uasual) after rotation and translation.\n' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.R_factor' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.R_factor' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.R_rigid_body',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.R_rigid_body identifies the R factor\n       for rigid body refinement after rotation and translation.(In general,\n       rigid body refinement has to be carried out after molecular\n       replacement.\n' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.R_rigid_body' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.R_rigid_body' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.packing',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.packing identifies the packing of\n       search model in the unit cell. Too many crystallographic contacts\n       may indicate a bad search.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.packing' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.packing' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.model_details',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _pdbx_phasing_MR.model_details records the details of\n       model used. For example, the original model can be truncated by\n       deleting side chains,  doubtful parts, using the monomer if the\n       original model was an oligomer. The search model may be one \n       domain of a large molecule. What is the pdb IDs.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.model_details' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pdbx_phasing_MR.model_details' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_rcsb.dic' ],
          [ 'item_', '_item_aliases.version', '1.1' ] ] ],
      [ 'save_',
        '_pdbx_phasing_MR.native_set_id',
        [ [ 'item_',
            '_item_description.description',
            '              The data set that was treated as the native in this\n               experiment.\n\n               This data item is a pointer to _phasing_set.id in the\n               PHASING_SET category.' ],
          [ 'item_', '_item.name', '_pdbx_phasing_MR.native_set_id' ],
          [ 'item_', '_item.category_id', 'pdbx_phasing_MR' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_phasing_MR.native_set_id' ],
          [ 'item_', '_item_linked.parent_name', '_phasing_set.id' ] ] ],
      [ 'save_',
        '_phasing_set.pdbx_temp_details',
        [ [ 'item_',
            '_item_description.description',
            '      The value of _phasing_set.pdbx_temp_details describes any\n       special details about the data collection temperature  \n       for this phasing data set. ' ],
          [ 'item_', '_item.name', '_phasing_set.pdbx_temp_details' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_phasing_set.pdbx_d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '              The smallest value in angstroms for the interplanar spacings\n               for the reflections in this shell. This is called the highest\n               resolution.' ],
          [ 'item_', '_item.name', '_phasing_set.pdbx_d_res_high' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_phasing_set.pdbx_d_res_low',
        [ [ 'item_',
            '_item_description.description',
            '              The highest value in angstroms for the interplanar spacings\n               for the reflections in this shell. This is called the lowest\n               resolution.' ],
          [ 'item_', '_item.name', '_phasing_set.pdbx_d_res_low' ],
          [ 'item_', '_item.category_id', 'phasing_set' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_refln.pdbx_anom_difference',
        [ [ 'item_',
            '_item_description.description',
            '              The amplitude difference of the Friedel pair,\n               D(hkl) = F(hkl) - F(-h-k-l).' ],
          [ 'item_', '_item.name', '_refln.pdbx_anom_difference' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_anom_difference_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The the standard deviation of the amplitude difference \n               of the Friedel pair,  D(hkl) = F(hkl) - F(-h-k-l).' ],
          [ 'item_', '_item.name', '_refln.pdbx_anom_difference_sigma' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_I_plus',
        [ [ 'item_',
            '_item_description.description',
            '              The intensity of the I(h,k,l) partner of \n               the Friedel pair.  ' ],
          [ 'item_', '_item.name', '_refln.pdbx_I_plus' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_I_minus',
        [ [ 'item_',
            '_item_description.description',
            '              The intensity of the I(-h,-k,-l) partner \n               of the Friedel pair.  ' ],
          [ 'item_', '_item.name', '_refln.pdbx_I_minus' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_F_plus',
        [ [ 'item_',
            '_item_description.description',
            '              The structure factor F(h,k,l) of the Friedel pair.  ' ],
          [ 'item_', '_item.name', '_refln.pdbx_F_plus' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_F_minus',
        [ [ 'item_',
            '_item_description.description',
            '              The structure factor F(-h,-k,-l) of the Friedel pair.  ' ],
          [ 'item_', '_item.name', '_refln.pdbx_F_minus' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_I_plus_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (derived from measurement) of the\n               intensity I(h,k,l) partner of the Friedel pair.  ' ],
          [ 'item_', '_item.name', '_refln.pdbx_I_plus_sigma' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_I_minus_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (derived from measurement) of the\n               intensity I(-h,-k,-l) partner of the Friedel pair.  ' ],
          [ 'item_', '_item.name', '_refln.pdbx_I_minus_sigma' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_F_minus_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (derived from measurement) of the\n               structure factor F(-h,-k,-l) of the Friedel pair.  ' ],
          [ 'item_', '_item.name', '_refln.pdbx_F_minus_sigma' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_F_plus_sigma',
        [ [ 'item_',
            '_item_description.description',
            '              The standard uncertainty (derived from measurement) of the\n               structure factor F(h,k,l) of the Friedel pair.  ' ],
          [ 'item_', '_item.name', '_refln.pdbx_F_plus_sigma' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_HL_A_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The isomorphous Hendrickson-Lattman coefficient A~iso~ for this\n               reflection.\n\n               Ref: Hendrickson, W. A. & Lattman, E. E. (1970). Acta\n                    Cryst. B26, 136-143.' ],
          [ 'item_', '_item.name', '_refln.pdbx_HL_A_iso' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_HL_B_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The isomorphous Hendrickson-Lattman coefficient B~iso~ for this\n               reflection.\n\n               Ref: Hendrickson, W. A. & Lattman, E. E. (1970). Acta\n                    Cryst. B26, 136-143.' ],
          [ 'item_', '_item.name', '_refln.pdbx_HL_B_iso' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_HL_C_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The isomorphous Hendrickson-Lattman coefficient C~iso~ for this\n               reflection.\n\n               Ref: Hendrickson, W. A. & Lattman, E. E. (1970). Acta\n                    Cryst. B26, 136-143.' ],
          [ 'item_', '_item.name', '_refln.pdbx_HL_C_iso' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refln.pdbx_HL_D_iso',
        [ [ 'item_',
            '_item_description.description',
            '              The isomorphous Hendrickson-Lattman coefficient D~iso~ for this\n               reflection.\n\n               Ref: Hendrickson, W. A. & Lattman, E. E. (1970). Acta\n                    Cryst. B26, 136-143.' ],
          [ 'item_', '_item.name', '_refln.pdbx_HL_D_iso' ],
          [ 'item_', '_item.category_id', 'refln' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', ' ' ],
      [ 'save_',
        'pdbx_refine_component',
        [ [ 'item_',
            '_category.description',
            '             Data items in the PDBX_REFINE_COMPONENT category record \n              statistics of the final model relative to the density map.' ],
          [ 'item_', '_category.id', 'pdbx_refine_component' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_refine_component.label_alt_id',
              '_pdbx_refine_component.label_asym_id',
              '_pdbx_refine_component.label_comp_id',
              '_pdbx_refine_component.label_seq_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'refine_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n#--',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_refine_component.label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the component.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.label_alt_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_refine_component.label_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_alt_id' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the component.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.label_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_refine_component.label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the component.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.label_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_refine_component.label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the component.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.label_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_refine_component.label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the component.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.auth_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_refine_component.auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the component.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.auth_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_refine_component.auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the component.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.auth_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_refine_component.auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.B_iso',
        [ [ 'item_',
            '_item_description.description',
            '\n      The average isotropic B factors for the group of atoms \n      (e.g. residue or ligand, side chain, main chain). \n      The B factors for each atom is given by _atom_site.B_iso_or_equiv' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.B_iso' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.B_iso_main_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      The average isotropic B factors for the group of atoms \n      (e.g. residue or ligand, side chain, main chain). \n      The B factors for each atom is given by _atom_site.B_iso_or_equiv' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.B_iso_main_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.B_iso_side_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      The average isotropic B factors for the group of atoms \n      (e.g. residue or ligand, side chain, main chain). \n      The B factors for each atom is given by _atom_site.B_iso_or_equiv' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.B_iso_side_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.shift',
        [ [ 'item_',
            '_item_description.description',
            '\n      The tendency of the group of atoms (e.g. residue\n      or ligand, side chain, main chain)  to move away from its current position.\n\n      Displacement of atoms from electron density is estimated from the\n      difference (Fobs - Fcal) map. The displacement vector is the ratio of\n      the gradient of difference density to the curvature. The amplitude of\n      the displacement vector is an indicator of the positional error.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.shift' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.shift_side_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      The tendency of the group of atoms (e.g. residue\n      or ligand, side chain, main chain)  to move away from its current position.\n\n      Displacement of atoms from electron density is estimated from the\n      difference (Fobs - Fcal) map. The displacement vector is the ratio of\n      the gradient of difference density to the curvature. The amplitude of\n      the displacement vector is an indicator of the positional error.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.shift_side_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.shift_main_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      The tendency of the group of atoms (e.g. residue\n      or ligand, side chain, main chain)  to move away from its current position.\n\n      Displacement of atoms from electron density is estimated from the\n      difference (Fobs - Fcal) map. The displacement vector is the ratio of\n      the gradient of difference density to the curvature. The amplitude of\n      the displacement vector is an indicator of the positional error.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.shift_main_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.correlation',
        [ [ 'item_',
            '_item_description.description',
            '\n      Correlation coefficient of electron density for each residue or ligand, \n      side chain, main chain\n \n      The density correlation coefficient is calculated for each component\n      from atomic densities of (2Fobs-Fcalc) map - "Robs" and the model\n      map (Fcalc) - "Rcalc" :\n \n      D_corr =  <Robs><Rcalc>/sqrt(<Robs**2><Rcalc**2>)\n \n          where <Robs> is the mean of "observed" densities of atoms of the\n           component (backbone or side chain).\n  \n                <Rcalc> is the mean of "calculated" densities of \n                 component atoms.\n \n          The value of density for some atom from map R(x) is:\n \n                   sum_i ( R(xi) * Ratom(xi - xa) )\n          Dens =  ---------------------------------- \n                       sum_i ( Ratom(xi - xa) ) \n \n            where\n\n                   Ratom(x) is atomic electron density for the x-th grid point.\n                   xa - vector of the centre of atom.\n                   xi - vector of the i-th point of grid.\n                   Sum is taken over all grid points which have distance\n                   from the center of the atom less than the Radius_limit.\n                   For all atoms Radius_limit = 2.5 A.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.correlation' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.correlation_side_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      Correlation coefficient of electron density for each residue or ligand, \n      side chain, main chain\n \n      The density correlation coefficient is calculated for each component\n      from atomic densities of (2Fobs-Fcalc) map - "Robs" and the model\n      map (Fcalc) - "Rcalc" :\n \n      D_corr =  <Robs><Rcalc>/sqrt(<Robs**2><Rcalc**2>)\n \n          where <Robs> is the mean of "observed" densities of atoms of the\n           component (backbone or side chain).\n  \n                <Rcalc> is the mean of "calculated" densities of \n                 component atoms.\n \n          The value of density for some atom from map R(x) is:\n \n                   sum_i ( R(xi) * Ratom(xi - xa) )\n          Dens =  ---------------------------------- \n                       sum_i ( Ratom(xi - xa) ) \n \n            where  Ratom(x) is atomic electron density for the x-th grid point.\n                   xa - vector of the centre of atom.\n                   xi - vector of the i-th point of grid.\n                   Sum is taken over all grid points which have distance\n                   from the center of the atom less than the Radius_limit.\n                   For all atoms Radius_limit = 2.5 A.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.correlation_side_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.correlation_main_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      Correlation coefficient of electron density for each residue or ligand, \n      side chain, main chain\n \n      The density correlation coefficient is calculated for each component\n      from atomic densities of (2Fobs-Fcalc) map - "Robs" and the model\n      map (Fcalc) - "Rcalc" :\n \n      D_corr =  <Robs><Rcalc>/sqrt(<Robs**2><Rcalc**2>)\n \n          where <Robs> is the mean of "observed" densities of atoms of the\n           component (backbone or side chain).\n  \n                <Rcalc> is the mean of "calculated" densities of \n                 component atoms.\n \n          The value of density for some atom from map R(x) is:\n \n                   sum_i ( R(xi) * Ratom(xi - xa) )\n          Dens =  ---------------------------------- \n                       sum_i ( Ratom(xi - xa) ) \n \n            where  Ratom(x) is atomic electron density for the x-th grid point.\n                   xa - vector of the centre of atom.\n                   xi - vector of the i-th point of grid.\n                   Sum is taken over all grid points which have distance\n                   from the center of the atom less than the Radius_limit.\n                   For all atoms Radius_limit = 2.5 A.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.correlation_main_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.real_space_R',
        [ [ 'item_',
            '_item_description.description',
            '\n      Real space R factor of electron density for each component,\n      residue side chain, or main chain.\n\n      The real space R factor is calculated by the equation\n\n      R_real = [Sum~i (|Dobs - Dcal|)]/[Sum~i (|Dobs + Dcal|)]\n\n      Where:\n         Dobs is the observed  electron density,\n         Dcal is the calculated  electron density,\n              summation is for all the grid points\n\n      Ref: Branden, C.I. & Jones, T.A. (1990).  Nature, 343, 687-689' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.real_space_R' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.real_space_R_side_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      Real space R factor of electron density for each component,\n      residue side chain, or main chain.\n\n      The real space R factor is calculated by the equation\n\n      R_real = [Sum~i (|Dobs - Dcal|)]/[Sum~i (|Dobs + Dcal|)]\n\n      Where:\n         Dobs is the observed  electron density,\n         Dcal is the calculated  electron density,\n              summation is for all the grid points\n\n      Ref: Branden, C.I. & Jones, T.A. (1990).  Nature, 343, 687-689' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.real_space_R_side_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.real_space_R_main_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      Real space R factor of electron density for each component,\n      residue side chain, or main chain.\n\n      The real space R factor is calculated by the equation\n\n      R_real = [Sum~i (|Dobs - Dcal|)]/[Sum~i (|Dobs + Dcal|)]\n\n      Where:\n         Dobs is the observed  electron density,\n         Dcal is the calculated  electron density,\n              summation is for all the grid points\n\n      Ref: Branden, C.I. & Jones, T.A. (1990).  Nature, 343, 687-689' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.real_space_R_main_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.connect',
        [ [ 'item_',
            '_item_description.description',
            '\n      The index of connectivity is the product of the (2Fobs-Fcal) electron \n      density values for the backbone atoms (N, CA and C) divided by the \n      average value for the structure. Low values (less than 1.0) of this \n      index indicate breaks in the backbone electron density which may be \n      due to flexibility of the chain or incorrect tracing.\n\n      connect = [(D(xi)...D(xi))^(1/N)] /<D>_all\n\n      Where:\n\n      D(xi) = (2*Fobs - Fcal)\n      <D>_all is the averaged value of density for the structure.\n      The product is for N atoms of group.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.connect' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.density_index',
        [ [ 'item_',
            '_item_description.description',
            '\n      The index of density is the product of the (2Fobs-Fcal) electron \n      density values for the group of atoms divided by the average value\n      for the structure. Low values (less than 1.0) may be problematic for \n      model fitting.\n\n      index = [(D(xi)...D(xi))^(1/N)] /<D>_all\n\n      Where :\n\n      D(xi) = (2*Fobs - Fcal)\n      <D>_all is the averaged value of density for the structure.\n      The product is for N atoms of group.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.density_index' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.density_index_main_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      The index of density is the product of the (2Fobs-Fcal) electron \n      density values for the group of atoms divided by the average value\n      for the structure. Low values (less than 1.0) may be problematic for \n      model fitting.\n\n      index = [(D(xi)...D(xi))^(1/N)] /<D>_all\n\n      Where :\n      D(xi) = (2*Fobs - Fcal)\n      <D>_all is the averaged value of density for the structure.\n      The product is for N atoms of group.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.density_index_main_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.density_index_side_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      The index of density is the product of the (2Fobs-Fcal) electron \n      density values for the group of atoms divided by the average value\n      for the structure. Low values (less than 1.0) may be problematic for \n      model fitting.\n\n      index = [(D(xi)...D(xi))^(1/N)] /<D>_all\n\n      Where :\n      D(xi) = (2*Fobs - Fcal)\n      <D>_all is the averaged value of density for the structure.\n      The product is for N atoms of group.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.density_index_side_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.density_ratio',
        [ [ 'item_',
            '_item_description.description',
            '\n      The density ratio is similar to the density index, but summation of the\n      density for the group is used for calculation.\n\n      The ratio of density is the summation of the (2Fobs-Fcal) electron \n      density values for the group of atoms divided by the average value\n      for the structure. Low values (less than 0.4) of this ratio may be \n      problematic for the group.\n\n\n      index = [Sum~i D(xi)]/<D>_all\n\n      Where:\n \n      D(xi) = (2*Fobs - Fcal)\n      <D>_all is the average value of density for the structure.\n             The summation is for all the atoms of group.' ],
          [ 'item_', '_item.name', '_pdbx_refine_component.density_ratio' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.density_ratio_main_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      The density ratio is similar to the density index, but summation of the\n      density for the group is used for calculation.\n\n      The ratio of density is the summation of the (2Fobs-Fcal) electron \n      density values for the group of atoms divided by the average value\n      for the structure. Low values (less than 0.4) of this ratio may be \n      problematic for the group.\n\n\n      index = [Sum~i D(xi)]/<D>_all\n\n      Where:\n \n      D(xi) = (2*Fobs - Fcal)\n      <D>_all is the average value of density for the structure.\n             The summation is for all the atoms of group.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.density_ratio_main_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_refine_component.density_ratio_side_chain',
        [ [ 'item_',
            '_item_description.description',
            '\n      The density ratio is similar to the density index, but summation of the\n      density for the group is used for calculation.\n\n      The ratio of density is the summation of the (2Fobs-Fcal) electron \n      density values for the group of atoms divided by the average value\n      for the structure. Low values (less than 0.4) of this ratio may be \n      problematic for the group.\n\n\n      index = [Sum~i D(xi)]/<D>_all\n\n      Where:\n \n      D(xi) = (2*Fobs - Fcal)\n      <D>_all is the average value of density for the structure.\n             The summation is for all the atoms of group.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_refine_component.density_ratio_side_chain' ],
          [ 'item_', '_item.category_id', 'pdbx_refine_component' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.pdbx_real_space_R',
        [ [ 'item_',
            '_item_description.description',
            '\n      Real space R factor of electron density for all atoms.\n\n      The real space R factor is calculated by the equation\n\n      R_real = [Sum~i (|Dobs - Dcal|)]/[Sum~i (|Dobs + Dcal|)]\n\n      Where:\n\n         Dobs is the observed  electron density,\n         Dcal is the calculated  electron density,\n              summation is for all the grid points\n\n      Ref: Branden, C.I. & Jones, T.A. (1990).  Nature, 343, 687-689' ],
          [ 'item_', '_item.name', '_refine.pdbx_real_space_R' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.pdbx_density_correlation',
        [ [ 'item_',
            '_item_description.description',
            '\n      The density correlation coefficient is calculated from atomic \n      densities of (2Fobs-Fcalc) map - "Robs" and the model\n      map (Fcalc) - "Rcalc" :\n \n      D_corr =  <Robs><Rcalc>/sqrt(<Robs**2><Rcalc**2>)\n \n          where <Robs> is the mean of "observed" densities of all atoms \n  \n                <Rcalc> is the mean of "calculated" densities of \n                 all atoms.\n \n          The value of density for some atom from map R(x) is:\n \n                   sum_i ( R(xi) * Ratom(xi - xa) )\n          Dens =  ---------------------------------- \n                       sum_i ( Ratom(xi - xa) ) \n \n            where  \n\n                   Ratom(x) is atomic electron density for the x-th grid point.\n                   xa - vector of the centre of atom.\n                   xi - vector of the i-th point of grid.\n                   Sum is taken over all grid points which have distance\n                   from the center of the atom less than the Radius_limit.\n                   For all atoms Radius_limit = 2.5 A.\n\n      Ref: Vaguine, A.A., Richelle, J. & Wodak, S.J. (1999). Acta Cryst. D55,199-205' ],
          [ 'item_', '_item.name', '_refine.pdbx_density_correlation' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment', '## EOF mmcif_pdbx-def-5.dic' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment',
        ' File:  mmcif_pdbx-def-6.dic                                             ' ],
      [ '@comment', ' ' ],
      [ '@comment',
        '                      PDB Exchange Data Dictionary' ],
      [ '@comment',
        ' This data dictionary contains definitions used by wwPDB for data exchange' ],
      [ '@comment', ' and data processing.  ' ],
      [ '@comment',
        '                           Definition Section 6' ],
      [ '@comment', '                            ' ],
      [ '@comment',
        '                                                                        ' ],
      [ '@comment',
        '##########################################################################' ],
      [ 'save_',
        '_entity.pdbx_target_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _entity.target_id points to a TARGETDB target idenitifier\n               from which this entity was generated.' ],
          [ 'item_', '_item.name', '_entity.pdbx_target_id' ],
          [ 'item_', '_item.category_id', 'entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        'pdbx_entity_prod_protocol',
        [ [ 'item_',
            '_category.description',
            '  This category contains descriptive protocols for the production\n   of this entity.' ],
          [ 'item_', '_category.id', 'pdbx_entity_prod_protocol' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_prod_protocol.entry_id',
              '_pdbx_entity_prod_protocol.entity_id',
              '_pdbx_entity_prod_protocol.protocol_type' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_prod_protocol.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_prod_protocol.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. ' ],
          [ 'item_', '_item.name', '_pdbx_entity_prod_protocol.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_prod_protocol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_prod_protocol.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_prod_protocol.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_prod_protocol.entity_id uniquely identifies\n  each protein contained in the project target protein complex whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category. ' ],
          [ 'item_', '_item.name', '_pdbx_entity_prod_protocol.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_prod_protocol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_prod_protocol.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_prod_protocol.protocol',
        [ [ 'item_',
            '_item_description.description',
            ' The protocol description associated with the protocol_type employed \n  in the production of this entity. ' ],
          [ 'item_', '_item.name', '_pdbx_entity_prod_protocol.protocol' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_prod_protocol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_prod_protocol.protocol_type',
        [ [ 'item_',
            '_item_description.description',
            ' The one of a set of protocol types associated with the production \n  of this entity. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_prod_protocol.protocol_type' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_prod_protocol' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'selection',
              'PCR',
              'cloning',
              'expression',
              'growth',
              'purification',
              'NMR',
              'other' ] ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# ENTITY_SRC_GEN ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        '_entity_src_gen.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _entity.id in the ENTITY category.' ],
          [ 'item_', '_item.name', '_entity_src_gen.entity_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_entity_src_gen.host_org_common_name',
        [ [ 'item_',
            '_item_description.description',
            '       The common name of the organism that served as host for the\n        production of the entity.  Where full details of the protein\n        production are available it would be expected that this item\n        be derived from _entity_src_gen_express.host_org_common_name\n        or via _entity_src_gen_express.host_org_tax_id' ],
          [ 'item_', '_item.name', '_entity_src_gen.host_org_common_name' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'yeast', 'bacteria' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.host_org_details',
        [ [ 'item_',
            '_item_description.description',
            '       A description of special aspects of the organism that served as\n        host for the production of the entity. Where full details of\n        the protein production are available it would be expected that\n        this item would derived from _entity_src_gen_express.host_org_details' ],
          [ 'item_', '_item.name', '_entity_src_gen.host_org_details' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity_src_gen.host_org_strain',
        [ [ 'item_',
            '_item_description.description',
            '       The strain of the organism in which the entity was expressed.\n        Where full details of the protein production are available\n        it would be expected that this item be derived from\n        _entity_src_gen_express.host_org_strain or via\n        _entity_src_gen_express.host_org_tax_id' ],
          [ 'item_', '_item.name', '_entity_src_gen.host_org_strain' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'DH5a', 'BMH 71-18' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.plasmid_details',
        [ [ 'item_',
            '_item_description.description',
            '       A description of special aspects of the plasmid that produced the\n        entity in the host organism. Where full details of the protein\n        production are available it would be expected that this item\n        would be derived from _pdbx_construct.details of the construct\n        pointed to from _entity_src_gen_express.plasmid_id.' ],
          [ 'item_', '_item.name', '_entity_src_gen.plasmid_details' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity_src_gen.plasmid_name',
        [ [ 'item_',
            '_item_description.description',
            '       The name of the plasmid that produced the entity in the host\n        organism. Where full details of the protein production are available\n        it would be expected that this item would be derived from\n        _pdbx_construct.name of the construct pointed to from\n        _entity_src_gen_express.plasmid_id.' ],
          [ 'item_', '_item.name', '_entity_src_gen.plasmid_name' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'pET3C', 'pT123sab' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_variant',
        [ [ 'item_',
            '_item_description.description',
            '       Variant of the organism used as the expression system. Where\n        full details of the protein production are available it would\n        be expected that this item be derived from\n        entity_src_gen_express.host_org_variant or via\n        _entity_src_gen_express.host_org_tax_id' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_variant' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'TRP-LAC', 'LAMBDA DE3' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_cell_line',
        [ [ 'item_',
            '_item_description.description',
            '       A specific line of cells used as the expression system. Where\n        full details of the protein production are available it would\n        be expected that this item would be derived from\n        entity_src_gen_express.host_org_cell_line' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_cell_line' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'HELA' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_atcc',
        [ [ 'item_',
            '_item_description.description',
            '       Americal Tissue Culture Collection of the expression system. Where\n        full details of the protein production are available it would\n        be expected that this item  would be derived from\n        _entity_src_gen_express.host_org_culture_collection' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_atcc' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_culture_collection',
        [ [ 'item_',
            '_item_description.description',
            '       Culture collection of the expression system. Where\n        full details of the protein production are available it would\n        be expected that this item  would be derived somehwere, but\n        exactly where is not clear.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_culture_collection' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_cell',
        [ [ 'item_',
            '_item_description.description',
            '       Cell type from which the gene is derived. Where\n        entity.target_id is provided this should be derived from\n        details of the target.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_cell' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'ENDOTHELIAL' ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_scientific_name',
        [ [ 'item_',
            '_item_description.description',
            '       The scientific name of the organism that served as host for the\n        production of the entity. Where full details of the protein\n        production are available it would be expected that this item\n        would be derived from _entity_src_gen_express.host_org_scientific_name\n        or via _entity_src_gen_express.host_org_tax_id' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_scientific_name' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'ESCHERICHIA COLI', 'SACCHAROMYCES CEREVISIAE' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_tissue',
        [ [ 'item_',
            '_item_description.description',
            '       The specific tissue which expressed the molecule. Where full details\n        of the protein production are available it would be expected that this\n        item would be derived from _entity_src_gen_express.host_org_tissue' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_tissue' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'heart', 'liver', 'eye lens' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_vector',
        [ [ 'item_',
            '_item_description.description',
            '       Identifies the vector used. Where full details of the protein\n        production are available it would be expected that this item\n        would be derived from _entity_src_gen_clone.vector_name.' ],
          [ 'item_', '_item.name', '_entity_src_gen.pdbx_host_org_vector' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'PBIT36', 'PET15B', 'PUC18' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.pdbx_host_org_vector_type',
        [ [ 'item_',
            '_item_description.description',
            ' Identifies the type of vector used (plasmid, virus, or cosmid).\n  Where full details of the protein production are available it\n  would be expected that this item would be derived from\n  _entity_src_gen_express.vector_type.' ],
          [ 'item_',
            '_item.name',
            '_entity_src_gen.pdbx_host_org_vector_type' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'COSMID', 'PLASMID' ] ] ] ],
      [ 'save_',
        '_entity_src_gen.expression_system_id',
        [ [ 'item_',
            '_item_description.description',
            '       A unique identifier for the expression system. This\n        should be extracted from a local list of expression\n        systems.' ],
          [ 'item_', '_item.name', '_entity_src_gen.expression_system_id' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'uline' ] ] ],
      [ 'save_',
        '_entity_src_gen.gene_src_dev_stage',
        [ [ 'item_',
            '_item_description.description',
            '       A string to indicate the life-cycle or cell development\n        cycle in which the gene is expressed and the mature\n        protein is active.' ],
          [ 'item_', '_item.name', '_entity_src_gen.gene_src_dev_stage' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_entity_src_gen.start_construct_id',
        [ [ 'item_',
            '_item_description.description',
            '       A pointer to _pdbx_construct.id in the PDBX_CONSTRUCT category.\n        The indentified sequence is the initial construct.' ],
          [ 'item_', '_item.name', '_entity_src_gen.start_construct_id' ],
          [ 'item_', '_item.category_id', 'entity_src_gen' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_entity_src_gen.start_construct_id' ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', ' ENTITY_SRC_GEN_PROD_OTHER #' ],
      [ '@comment', '############################' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_other.entry_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_other.entity_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_other.step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_other.next_step_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_other.end_construct_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_other.process_name' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_other.date' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_other.robot_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_other.details' ],
      [ 'save_',
        'pdbx_entity_src_gen_prod_other',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for process steps that are\n   not explicitly catered for elsewhere. It provides some basic\n   details as well as placeholders for a list of parameters and\n   values (the category PDBX_ENTITY_SRC_GEN_PROD_OTHER_PARAMETER).\n   Note that processes that have been modelled explicitly should\n   not be represented using this category.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_prod_other' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_prod_other.entry_id',
              '_pdbx_entity_src_gen_prod_other.entity_id',
              '_pdbx_entity_src_gen_prod_other.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_prod_other.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other.entry_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_other.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_prod_other.entity_id uniquely identifies\n  each protein contained in the project target protein complex whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category. This item may be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other.entity_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_other.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is the unique identifier for this process step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other.step_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other.next_step_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item unique identifier for the next production step.\n        This allows a workflow to have multiple entry points leading\n        to a single product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other.next_step_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other.end_construct_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT\n        category. The referenced nucleic acid sequence is that of the\n        product of the process step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other.end_construct_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_other.end_construct_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other.robot_id',
        [ [ 'item_',
            '_item_description.description',
            '       This data item is a pointer to pdbx_robot_system.id in the\n        PDBX_ROBOT_SYSTEM category. The referenced robot is the\n        robot responsible for the process step' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other.robot_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_other.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other.date',
        [ [ 'item_',
            '_item_description.description',
            '       The date of this process step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_prod_other.date' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other.process_name',
        [ [ 'item_',
            '_item_description.description',
            '       Name of this process step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other.process_name' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other.details',
        [ [ 'item_',
            '_item_description.description',
            '       Additional details of this process step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other.details' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '###########################################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_PROD_OTHER_PARAMETER #' ],
      [ '@comment', '###########################################' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_other_parameter.entry_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_other_parameter.entity_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_other_parameter.step_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_other_parameter.name' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_other_parameter.value' ],
      [ 'save_',
        'pdbx_entity_src_gen_prod_other_parameter',
        [ [ 'item_',
            '_category.description',
            '  This category contains parameters and values required to capture\n   information about a particular process step' ],
          [ 'item_',
            '_category.id',
            'pdbx_entity_src_gen_prod_other_parameter' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_prod_other_parameter.entry_id',
              '_pdbx_entity_src_gen_prod_other_parameter.entity_id',
              '_pdbx_entity_src_gen_prod_other_parameter.step_id',
              '_pdbx_entity_src_gen_prod_other_parameter.parameter' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other_parameter.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_prod_other_parameter.entry_id is a pointer\n  to _pdbx_entity_src_gen_prod_other.entry.id' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other_parameter.entry_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other_parameter' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_prod_other.entry_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_other_parameter.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other_parameter.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_prod_other_parameter.entity_id is a pointer\n  to _pdbx_entity_src_gen_prod_other.entity_id' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other_parameter.entity_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other_parameter' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_prod_other.entity_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_other_parameter.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other_parameter.step_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item is a pointer to _pdbx_entity_src_gen_prod_other.step_id' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other_parameter.step_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other_parameter' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_prod_other.step_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_other_parameter.step_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other_parameter.parameter',
        [ [ 'item_',
            '_item_description.description',
            '       The name of the parameter associated with the process step' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other_parameter.parameter' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other_parameter' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other_parameter.value',
        [ [ 'item_',
            '_item_description.description',
            '       The value of the parameter' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other_parameter.value' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other_parameter' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_other_parameter.details',
        [ [ 'item_',
            '_item_description.description',
            '       Additional details about the parameter' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_other_parameter.details' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_other_parameter' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '###############################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_PROD_PCR #' ],
      [ '@comment', '###############################' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_pcr.entry_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_pcr.entity_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_pcr.step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_pcr.next_step_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_pcr.end_construct_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_pcr.forward_primer_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_pcr.reverse_primer_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_pcr.reaction_details' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_prod_pcr.purification_details' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_pcr.date' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_pcr.robot_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_prod_pcr.summary' ],
      [ 'save_',
        'pdbx_entity_src_gen_prod_pcr',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the PCR steps used in\n   the overall protein production process. The PCR is assumed\n   to be applied to the result of the previous production step, or the\n   gene source if this is the first production step.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_prod_pcr.entry_id',
              '_pdbx_entity_src_gen_prod_pcr.entity_id',
              '_pdbx_entity_src_gen_prod_pcr.step_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - hypothetical example',
              '\n    _pdbx_entity_src_gen_prod_pcr.entry_id           111000111\n    _pdbx_entity_src_gen_prod_pcr.entity_id          222000111\n    _pdbx_entity_src_gen_prod_pcr.step_id            1\n    _pdbx_entity_src_gen_prod_pcr.next_step_id       2\n    _pdbx_entity_src_gen_prod_pcr.end_construct_id   440050000111\n    _pdbx_entity_src_gen_prod_pcr.robot_id           5\n    _pdbx_entity_src_gen_prod_pcr.date               \'2002-07-12:15:13\'\n    _pdbx_entity_src_gen_prod_pcr.forward_primer_id  2\n    _pdbx_entity_src_gen_prod_pcr.reverse_primer_id  3\n    _pdbx_entity_src_gen_prod_pcr.reaction_details\n  ;  Annealing temperature = 70 C\n     Annealing time = 60 s\n     Extending temperature = 74 C\n     Extending time = 120 s\n     Melting temperature = 95 C\n     Melting time = 120 s\n     Number of cycles = 40\n     Polymerase = KOD\n     Template = 10 pmol\n     Primer = 25 pmol\n     Total volume = 25 ul\n  ;\n    _pdbx_entity_src_gen_prod_pcr.purification_details\n  ;  No purification\n  ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_prod_pcr.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_pcr.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_prod_pcr.entity_id uniquely identifies\n  each protein contained in the project target protein complex whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category. This item may be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_pcr.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is the unique identifier for this PCR step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.next_step_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item unique identifier for the next production step.\n        This allows a workflow to have multiple entry points leading\n        to a single product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.next_step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.end_construct_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT\n        category. The referenced nucleic acid sequence is that of the\n        PCR product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.end_construct_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_pcr.end_construct_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.robot_id',
        [ [ 'item_',
            '_item_description.description',
            '       This data item is a pointer to pdbx_robot_system.id in the\n        PDBX_ROBOT_SYSTEM category. The referenced robot is the\n        robot responsible for the PCR reaction (normally the heat\n        cycler).' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.robot_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_pcr.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of this production step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_prod_pcr.date' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.forward_primer_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT category.\n  The referenced nucleic acid sequence is that of the forward primer.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.forward_primer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_pcr.forward_primer_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.reverse_primer_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT category.\n  The referenced nucleic acid sequence is that of the reverse primer.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.reverse_primer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_pcr.reverse_primer_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.reaction_details',
        [ [ 'item_',
            '_item_description.description',
            ' String value containing details of the PCR reaction.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.reaction_details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.purification_details',
        [ [ 'item_',
            '_item_description.description',
            ' String value containing details of any purification of the\n  product of the PCR reaction.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.purification_details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_pcr.summary',
        [ [ 'item_',
            '_item_description.description',
            ' Summary of the details of the PCR reaction any purification of the\n  product of the PCR reaction.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_pcr.summary' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_prod_pcr' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '##################################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_PROD_DIGEST #' ],
      [ '@comment', '##################################' ],
      [ '@comment', '   _pdbx_entity_src_gen_prod_digest.entry_id' ],
      [ '@comment', '   _pdbx_entity_src_gen_prod_digest.entity_id' ],
      [ '@comment', '   _pdbx_entity_src_gen_prod_digest.step_id' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_prod_digest.next_step_id' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_prod_digest.restriction_enzyme_1' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_prod_digest.restriction_enzyme_2' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_prod_digest.purification_details' ],
      [ '@comment', '   _pdbx_entity_src_gen_prod_digest.date' ],
      [ '@comment', '   _pdbx_entity_src_gen_prod_digest.robot_id' ],
      [ '@comment', '   _pdbx_entity_src_gen_prod_digest.summary' ],
      [ 'save_',
        'pdbx_entity_src_gen_prod_digest',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the DIGEST steps used in\n   the overall protein production process. The digestion is assumed\n   to be applied to the result of the previous production step, or the\n   gene source if this is the first production step.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_prod_digest.entry_id',
              '_pdbx_entity_src_gen_prod_digest.entity_id',
              '_pdbx_entity_src_gen_prod_digest.step_id' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - hypothetical example',
              '\n    _pdbx_entity_src_gen_prod_digest.entry_id              111000111\n    _pdbx_entity_src_gen_prod_digest.entity_id             222000111\n    _pdbx_entity_src_gen_prod_digest.step_id               2\n    _pdbx_entity_src_gen_prod_digest.next_step_id          3\n    _pdbx_entity_src_gen_prod_digest.end_construct_id      440050000123\n    _pdbx_entity_src_gen_prod_digest.robot_id              5\n    _pdbx_entity_src_gen_prod_digest.date                  \'2002-07-12:15:13\'\n    _pdbx_entity_src_gen_prod_digest.restriction_enzyme_1  \'NcoI\'\n    _pdbx_entity_src_gen_prod_digest.restriction_enzyme_2  \'BamII\'\n    _pdbx_entity_src_gen_prod_digest.purification_details\n  ;  No purification\n  ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_prod_digest.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.entry_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_digest.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_prod_digest.entity_id uniquely identifies\n  each protein contained in the project target protein complex whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category. This item may be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.entity_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_digest.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is the unique identifier for this digestion step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.step_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.next_step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item unique identifier for the next production step. This allows\n  a workflow to have multiple entry points leading to a single product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.next_step_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.end_construct_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_construct.id in the\n  PDBX_CONSTRUCT category. The referenced nucleic acid sequence\n  is that of the digest product' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.end_construct_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_digest.end_construct_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.robot_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to pdbx_robot_system.id\n  in the PDBX_ROBOT_SYSTEM category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.robot_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_prod_digest.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of this production step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.date' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'item_', '_item_examples.case', '2003-12-25' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.restriction_enzyme_1',
        [ [ 'item_',
            '_item_description.description',
            ' The first enzyme used in the restriction digestion. The sites at\n  which this cuts can be derived from the sequence.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.restriction_enzyme_1' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'BamIII' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.restriction_enzyme_2',
        [ [ 'item_',
            '_item_description.description',
            ' The second enzyme used in the restriction digestion. The sites at\n  which this cuts can be derived from the sequence.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.restriction_enzyme_2' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.purification_details',
        [ [ 'item_',
            '_item_description.description',
            ' String value containing details of any purification of the\n  product of the digestion.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.purification_details' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_prod_digest.summary',
        [ [ 'item_',
            '_item_description.description',
            ' Summary of the details of restriction digestion any purification of the\n  product of the digestion.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_prod_digest.summary' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_prod_digest' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', ' ENTITY_SRC_GEN_CLONE #' ],
      [ '@comment', '#######################' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.entry_id' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.entity_id' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.step_id' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.next_step_id' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.end_construct_id' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.robot_id' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.date' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.gene_insert_method' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.vector_name' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.vector_details' ],
      [ '@comment',
        ' _pdbx_entity_src_gen_clone.transformation_method' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.marker' ],
      [ '@comment',
        ' _pdbx_entity_src_gen_clone.verification_method' ],
      [ '@comment',
        ' _pdbx_entity_src_gen_clone.purification_details' ],
      [ '@comment', ' _pdbx_entity_src_gen_clone.summary' ],
      [ '@comment', ' ' ],
      [ 'save_',
        'pdbx_entity_src_gen_clone',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the cloning steps used in\n   the overall protein production process. Each row in PDBX_ENTITY_SRC_GEN_CLONE\n   should have an equivalent row in either PDBX_ENTITY_SRC_GEN_CLONE_LIGATION or\n   PDBX_ENTITY_SRC_GEN_CLONE_RECOMBINATION.  If only summary information is \n   provided data in the later two categories may be omitted.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_clone.entry_id',
              '_pdbx_entity_src_gen_clone.entity_id',
              '_pdbx_entity_src_gen_clone.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_clone.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_clone.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_clone.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_clone.entity_id uniquely identifies\n  each protein contained in the project target protein complex whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category. This item may be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_clone.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_clone.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is the unique identifier for this cloning step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_clone.step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.next_step_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item unique identifier for the next production step.\n        This allows a workflow to have multiple entry points leading\n        to a single product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone.next_step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.end_construct_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT\n        category. The referenced nucleic acid sequence is that of the\n        cloned product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone.end_construct_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_clone.end_construct_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.robot_id',
        [ [ 'item_',
            '_item_description.description',
            '       This data item is a pointer to pdbx_robot_system.id in the\n        PDBX_ROBOT_SYSTEM category.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_clone.robot_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_clone.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of this production step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_clone.date' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.gene_insert_method',
        [ [ 'item_',
            '_item_description.description',
            '  The method used to insert the gene into the vector. For \'Ligation\', an\n   PDBX_ENTITY_SRC_GEN_CLONE_LIGATION entry with matching .step_id is expected. For\n   \'Recombination\', an PDBX_ENTITY_SRC_GEN_CLONE_RECOMBINATION entry with matching\n   .step_id is expected.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone.gene_insert_method' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'L', 'Ligation', 'R', 'Recombination' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.vector_name',
        [ [ 'item_',
            '_item_description.description',
            '  The name of the vector used in this cloning step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone.vector_name' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.vector_details',
        [ [ 'item_',
            '_item_description.description',
            '  Details of any modifications made to the named vector.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone.vector_details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.transformation_method',
        [ [ 'item_',
            '_item_description.description',
            '  The method used to transform the expression cell line with the vector' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone.transformation_method' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'H', 'Heat shock', 'E', 'Electroporation', 'C', 'Calcium shock' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.marker',
        [ [ 'item_',
            '_item_description.description',
            ' The type of marker included to allow selection of transformed cells' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_clone.marker' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'A', 'Antibiotic resistance', 'F', 'Fluorescence' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.verification_method',
        [ [ 'item_',
            '_item_description.description',
            ' The method used to verify that the incorporated gene is correct' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone.verification_method' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'P',
              'PCR on plasmid',
              'R',
              'Restriction enzyme digestion',
              'D',
              'DNA sequencing' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.purification_details',
        [ [ 'item_',
            '_item_description.description',
            '  Details of any purification of the product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone.purification_details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone.summary',
        [ [ 'item_',
            '_item_description.description',
            '  Summary of ligation or recombionation cloning used, the associated \n   verification method and any purification of the product.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_clone.summary' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_clone' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#####################################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_CLONE_LIGATION #' ],
      [ '@comment', '#####################################' ],
      [ '@comment', '  _pdbx_entity_src_gen_clone_ligation.entry_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_clone_ligation.entity_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_clone_ligation.step_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_clone_ligation.cleavage_enzymes' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_clone_ligation.ligation_enzymes' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_clone_ligation.temperature' ],
      [ '@comment', '  _pdbx_entity_src_gen_clone_ligation.time' ],
      [ '@comment', '  _pdbx_entity_src_gen_clone_ligation.details' ],
      [ 'save_',
        'pdbx_entity_src_gen_clone_ligation',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the ligation-based cloning steps used in\n   the overall protein production process.\n   _pdbx_entity_src_gen_clone_ligation.clone_step_id in this category\n   must point at a defined _pdbx_entity_src_gen_clone.step_id. The details in\n   PDBX_ENTITY_SRC_GEN_CLONE_LIGATION extend the details in PDBX_ENTITY_SRC_GEN_CLONE\n   to cover ligation dependent cloning steps.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_clone_ligation' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_clone_ligation.entry_id',
              '_pdbx_entity_src_gen_clone_ligation.entity_id',
              '_pdbx_entity_src_gen_clone_ligation.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_ligation.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to _pdbx_entity_src_gen_clone.entry_id in the\n  PDBX_ENTITY_SRC_GEN_CLONE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_ligation.entry_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_ligation' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_clone.entry_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_clone_ligation.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_ligation.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to _pdbx_entity_src_gen_clone.entity_id in the\n  PDBX_ENTITY_SRC_GEN_CLONE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_ligation.entity_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_ligation' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_clone.entity_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_clone_ligation.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_ligation.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to _pdbx_entity_src_gen_clone.step_id in the\n  PDBX_ENTITY_SRC_GEN_CLONE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_ligation.step_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_ligation' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_clone.step_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_clone_ligation.step_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_ligation.cleavage_enzymes',
        [ [ 'item_',
            '_item_description.description',
            ' The names of the enzymes used to cleave the vector. In addition an enzyme\n  used to blunt the cut ends, etc., should be named here.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_ligation.cleavage_enzymes' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_ligation' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_ligation.ligation_enzymes',
        [ [ 'item_',
            '_item_description.description',
            ' The names of the enzymes used to ligate the gene into the cleaved vector.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_ligation.ligation_enzymes' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_ligation' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_ligation.temperature',
        [ [ 'item_',
            '_item_description.description',
            ' The temperature at which the ligation experiment was performed, in degrees\n  celsius.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_ligation.temperature' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_ligation' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'celsius' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_ligation.time',
        [ [ 'item_',
            '_item_description.description',
            ' The duration of the ligation reaction in minutes.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_ligation.time' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_ligation' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_units.code', 'minutes' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_ligation.details',
        [ [ 'item_',
            '_item_description.description',
            ' Any details to be associated with this ligation step, e.g. the protocol.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_ligation.details' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_ligation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '##########################################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_CLONE_RECOMBINATION #' ],
      [ '@comment', '##########################################' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_clone_recombination.entry__id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_clone_recombination.entity_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_clone_recombination.step_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_clone_recombination.system' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_clone_recombination.recombination_enzymes' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_clone_recombination.details' ],
      [ 'save_',
        'pdbx_entity_src_gen_clone_recombination',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the recombination-based cloning steps\n   used in the overall protein production process. It is assumed that these\n   reactions will use commercially available kits.\n   _pdbx_entity_src_gen_clone_recombination.clone_step_id in this category\n   must point at a defined _pdbx_entity_src_gen_clone.step_id. The details in\n   PDBX_ENTITY_SRC_GEN_CLONE_RECOMBINATION extend the details in PDBX_ENTITY_SRC_GEN_CLONE\n   to cover recombination dependent cloning steps.' ],
          [ 'item_',
            '_category.id',
            'pdbx_entity_src_gen_clone_recombination' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_clone_recombination.entry_id',
              '_pdbx_entity_src_gen_clone_recombination.entity_id',
              '_pdbx_entity_src_gen_clone_recombination.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_recombination.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to _pdbx_entity_src_gen_clone.entry_id in the\n  PDBX_ENTITY_SRC_GEN_CLONE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_recombination.entry_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_recombination' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_clone.entry_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_clone_recombination.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_recombination.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to _pdbx_entity_src_gen_clone.entity_id in the\n  PDBX_ENTITY_SRC_GEN_CLONE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_recombination.entity_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_recombination' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_clone.entity_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_clone_recombination.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_recombination.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to _pdbx_entity_src_gen_clone.step_id in the\n  PDBX_ENTITY_SRC_GEN_CLONE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_recombination.step_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_recombination' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_clone.step_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_clone_recombination.step_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_recombination.system',
        [ [ 'item_',
            '_item_description.description',
            ' The name of the recombination system.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_recombination.system' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_recombination' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'G',
              'Gateway Tm',
              'I',
              'Infusion/Creator Tm',
              'N',
              'Novagen\'s system - name?' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_recombination.recombination_enzymes',
        [ [ 'item_',
            '_item_description.description',
            ' The names of the enzymes used for this recombination step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_recombination.recombination_enzymes' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_recombination' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'B',
              'Gateway BP clonase',
              'L',
              'Gateway LR clonase',
              'T',
              'Topoisomerase',
              'C',
              'Cre recombinase',
              'I',
              'Infusion enzyme - name?' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_clone_recombination.details',
        [ [ 'item_',
            '_item_description.description',
            ' Any details to be associated with this recombination step, e.g. the protocol\n  or differences from the manufacturer\'s specified protocol.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_clone_recombination.details' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_clone_recombination' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '##############################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_EXPRESS #' ],
      [ '@comment', '##############################' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.entry_id' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.entity_id' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.step_id' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.next_step_id' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.end_construct_id' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.robot_id' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.date' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.promoter_type' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.plasmid_id' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.vector_type' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.N_terminal_seq_tag' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.C_terminal_seq_tag' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.host_org_scientific_name' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.host_org_common_name' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.host_org_variant' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.host_org_strain' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.host_org_tissue' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.host_org_tax_id' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.host_org_culture_collection' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.host_org_cell_line' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.host_org_details' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.culture_base_media' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.culture_additives' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.culture_volume' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.culture_time' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.culture_temperature' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.inducer' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.inducer_concentration' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.induction_details' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.multiplicity_of_infection' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.induction_growth_time' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.induction_temperature' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.harvesting_details' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express.storage_details' ],
      [ '@comment', '   _pdbx_entity_src_gen_express.summary' ],
      [ 'save_',
        'pdbx_entity_src_gen_express',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the EXPRESSION steps used in\n   the overall protein production process. It is hoped that this category\n   will cover all forms of cell-based expression by reading induction as\n   induction/transformation/transfection.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_express.entry_id',
              '_pdbx_entity_src_gen_express.entity_id',
              '_pdbx_entity_src_gen_express.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_express.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_express.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_express.entity_id uniquely identifies\n  each protein contained in the project target complex proteins whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category.\n  This item may be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_express.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is the unique identifier for this expression step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_express.step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.next_step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item unique identifier for the next production step. This allows\na workflow to have multiple entry points leading to a single product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.next_step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.end_construct_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT\n        category. The referenced sequence is expected to be the amino\n        acid sequence of the expressed product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.end_construct_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_express.end_construct_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.robot_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to pdbx_robot_system.id\n  in the PDBX_ROBOT_SYSTEM category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.robot_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_express.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of production step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_express.date' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.promoter_type',
        [ [ 'item_',
            '_item_description.description',
            '  The nature of the promoter controlling expression of the gene.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.promoter_type' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'T7 LacZ' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.plasmid_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to _pdbx_construct.id in the\n  PDBX_CONSTRUCT category. The referenced entry will contain\n  the nucleotide sequence that is to be expressed, including tags.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.plasmid_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_express.plasmid_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.vector_type',
        [ [ 'item_',
            '_item_description.description',
            ' Identifies the type of vector used (plasmid, virus, or cosmid)\n  in the expression system.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.vector_type' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.N_terminal_seq_tag',
        [ [ 'item_',
            '_item_description.description',
            ' Any N-terminal sequence tag as a string of one letter amino acid codes.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.N_terminal_seq_tag' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.C_terminal_seq_tag',
        [ [ 'item_',
            '_item_description.description',
            ' Any C-terminal sequence tag as a string of one letter amino acid codes' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.C_terminal_seq_tag' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.host_org_scientific_name',
        [ [ 'item_',
            '_item_description.description',
            ' The scientific name of the organism that served as host for the\n  expression system. It is expected that either this item or\n  _pdbx_entity_src_gen_express.host_org_tax_id should be populated.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.host_org_scientific_name' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.host_org_common_name',
        [ [ 'item_',
            '_item_description.description',
            ' The common name of the organism that served as host for the\n  expression system. Where _pdbx_entity_src_gen_express.host_org_tax_id\n  is populated it is expected that this item may be derived by\n  look up against the taxonomy database.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.host_org_common_name' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.host_org_variant',
        [ [ 'item_',
            '_item_description.description',
            ' The vairant of the organism that served as host for the expression\n  system. Where _pdbx_entity_src_gen_express.host_org_tax_id is\n  populated it is expected that this item may be derived by a\n  look up against the taxonomy database.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.host_org_variant' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.host_org_strain',
        [ [ 'item_',
            '_item_description.description',
            ' The strain of the organism that served as host for the expression\n  system. Where _pdbx_entity_src_gen_express.host_org_tax_id is\n  populated it is expected that this item may be derived by a\n  look up against the taxonomy database.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.host_org_strain' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.host_org_tissue',
        [ [ 'item_',
            '_item_description.description',
            '       The specific tissue which expressed the molecule.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.host_org_tissue' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.host_org_culture_collection',
        [ [ 'item_',
            '_item_description.description',
            '       Culture collection of the expression system' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.host_org_culture_collection' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.host_org_cell_line',
        [ [ 'item_',
            '_item_description.description',
            ' A specific line of cells used as the expression system' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.host_org_cell_line' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.host_org_tax_id',
        [ [ 'item_',
            '_item_description.description',
            ' The id for the NCBI taxonomy node corresponding to the organism that\n  served as host for the expression system.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.host_org_tax_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.host_org_details',
        [ [ 'item_',
            '_item_description.description',
            ' A description of special aspects of the organism that served as\n  host for the  expression system.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.host_org_details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.culture_base_media',
        [ [ 'item_',
            '_item_description.description',
            ' The name of the base media in which the expression host was grown.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.culture_base_media' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.culture_additives',
        [ [ 'item_',
            '_item_description.description',
            ' Any additives to the base media in which the expression host was grown.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.culture_additives' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.culture_volume',
        [ [ 'item_',
            '_item_description.description',
            ' The volume of media in milliliters in which the expression host was grown.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.culture_volume' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'milliliters' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.culture_time',
        [ [ 'item_',
            '_item_description.description',
            ' The time in hours for which the expression host was allowed to grow\n  prior to induction/transformation/transfection.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.culture_time' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'hours' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.culture_temperature',
        [ [ 'item_',
            '_item_description.description',
            ' The temperature in degrees celsius at which the expression host was allowed\n  to grow prior to induction/transformation/transfection.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.culture_temperature' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'celsius' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.inducer',
        [ [ 'item_',
            '_item_description.description',
            ' The chemical name of the inducing agent.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_express.inducer' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.inducer_concentration',
        [ [ 'item_',
            '_item_description.description',
            ' Concentration of the inducing agent.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.inducer_concentration' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'mg_per_ml' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.induction_details',
        [ [ 'item_',
            '_item_description.description',
            ' Details of induction/transformation/transfection.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.induction_details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.multiplicity_of_infection',
        [ [ 'item_',
            '_item_description.description',
            ' The multiplicity of infection for genes introduced by transfection, eg.\nfor baculovirus-based expression.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.multiplicity_of_infection' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'arbitrary' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.induction_timepoint',
        [ [ 'item_',
            '_item_description.description',
            ' The time in hours after induction/transformation/transfection at which\n  the optical density of the culture was measured.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.induction_timepoint' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'hours' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.induction_temperature',
        [ [ 'item_',
            '_item_description.description',
            ' The temperature in celsius at which the induced/transformed/transfected\n  cells were grown.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.induction_temperature' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'celsius' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.harvesting_details',
        [ [ 'item_',
            '_item_description.description',
            ' Details of the harvesting protocol.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.harvesting_details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.storage_details',
        [ [ 'item_',
            '_item_description.description',
            ' Details of how the harvested culture was stored.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express.storage_details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express.summary',
        [ [ 'item_',
            '_item_description.description',
            ' Summary of the details of the expression steps used in protein production.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_express.summary' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_express' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '########################################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_EXPRESS_TIMEPOINT #' ],
      [ '@comment', '########################################' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express_timepoint.entry_id' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express_timepoint.entity_id' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express_timepoint.step_id' ],
      [ '@comment',
        '   _pdbx_entity_src_gen_express_timepoint.serial' ],
      [ '@comment', '   _pdbx_entity_src_gen_express_timepoint.time' ],
      [ '@comment', '   _pdbx_entity_src_gen_express_timepoint.OD' ],
      [ 'save_',
        'pdbx_entity_src_gen_express_timepoint',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for OD time series used to monitor a\n   given EXPRESSION step used in the overall protein production process.' ],
          [ 'item_',
            '_category.id',
            'pdbx_entity_src_gen_express_timepoint' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_express_timepoint.entry_id',
              '_pdbx_entity_src_gen_express_timepoint.entity_id',
              '_pdbx_entity_src_gen_express_timepoint.step_id',
              '_pdbx_entity_src_gen_express_timepoint.serial' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express_timepoint.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_express_timepoint.entry_id is a pointer\n  to _pdbx_entity_src_gen_express.entry_id' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express_timepoint.entry_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_express_timepoint' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_express.entry_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_express_timepoint.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express_timepoint.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_express_timepoint.entity_id is a pointer\n  to _pdbx_entity_src_gen_express.entity_id' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express_timepoint.entity_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_express_timepoint' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_express.entity_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_express_timepoint.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express_timepoint.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to _pdbx_entity_src_gen_express.step_id' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express_timepoint.step_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_express_timepoint' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_entity_src_gen_express.step_id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_express_timepoint.step_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express_timepoint.serial',
        [ [ 'item_',
            '_item_description.description',
            ' This items uniquely defines a timepoint within a series.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express_timepoint.serial' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_express_timepoint' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express_timepoint.OD',
        [ [ 'item_',
            '_item_description.description',
            ' The optical density of the expression culture in arbitrary units at the\n  timepoint specified.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express_timepoint.OD' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_express_timepoint' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_express_timepoint.time',
        [ [ 'item_',
            '_item_description.description',
            ' The time in hours after induction/transformation/transfection at which\n  the optical density of the culture was measured.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_express_timepoint.time' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_express_timepoint' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_LYSIS #' ],
      [ '@comment', '############################' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.entry_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.entity_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.next_step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.end_construct_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.date' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.robot_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.method' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.buffer_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.buffer_volume' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.temperature' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.time' ],
      [ '@comment', '  _pdbx_entity_src_gen_lysis.details' ],
      [ 'save_',
        'pdbx_entity_src_gen_lysis',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the cell lysis steps used in\n   the overall protein production process.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_lysis.entry_id',
              '_pdbx_entity_src_gen_lysis.entity_id',
              '_pdbx_entity_src_gen_lysis.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_lysis.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_lysis.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_lysis.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_lysis.entity_id uniquely identifies\n  each protein contained in the project target protein complex whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category.  This item may be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_lysis.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_lysis.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is the unique identifier for this lysis step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_lysis.step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.next_step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item unique identifier for the next production step. This allows\na workflow to have multiple entry points leading to a single product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_lysis.next_step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.end_construct_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT\n        category. The referenced sequence is expected to be the amino\n        acid sequence of the expressed product after lysis.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_lysis.end_construct_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_lysis.end_construct_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.robot_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to pdbx_robot_system.id\n  in the PDBX_ROBOT_SYSTEM category.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_lysis.robot_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_lysis.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of this production step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_lysis.date' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.method',
        [ [ 'item_',
            '_item_description.description',
            ' The lysis method.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_lysis.method' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'S',
              'Sonication',
              'D',
              'Detergent',
              'B',
              'Sonication and detergent' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.buffer_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_buffer.id in the PDBX_BUFFER category.\n  The referenced buffer is that in which the lysis was performed.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_lysis.buffer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_buffer.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_lysis.buffer_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.buffer_volume',
        [ [ 'item_',
            '_item_description.description',
            ' The volume in milliliters of buffer in which the lysis was performed.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_lysis.buffer_volume' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'milliliters' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.temperature',
        [ [ 'item_',
            '_item_description.description',
            ' The temperature in degrees celsius at which the lysis was performed.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_lysis.temperature' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'celsius' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.time',
        [ [ 'item_',
            '_item_description.description',
            ' The time in seconds of the lysis experiment.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_lysis.time' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'minutes' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_lysis.details',
        [ [ 'item_',
            '_item_description.description',
            ' String value containing details of the lysis protocol.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_lysis.details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_lysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#############################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_REFOLD #' ],
      [ '@comment', '#############################' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.entry_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.entity_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.next_step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.end_construct_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.robot_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.date' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_refold.denature_buffer_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.refold_buffer_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.temperature' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.time' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_refold.storage_buffer_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_refold.details' ],
      [ 'save_',
        'pdbx_entity_src_gen_refold',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the refolding steps used in\n   the overall protein production process.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_refold.entry_id',
              '_pdbx_entity_src_gen_refold.entity_id',
              '_pdbx_entity_src_gen_refold.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_refold.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_refold.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_refold.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_refold.entity_id uniquely identifies\n  each protein contained in the project target protein complex whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category. This item may be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_refold.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_refold.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is the unique identifier for this refolding step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_refold.step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.next_step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item unique identifier for the next production step. This allows\na workflow to have multiple entry points leading to a single product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_refold.next_step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.end_construct_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT\n        category. The referenced sequence is expected to be the amino\n        acid sequence of the expressed product after the refolding\n        step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_refold.end_construct_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_refold.end_construct_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.robot_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to pdbx_robot_system.id\n  in the PDBX_ROBOT_SYSTEM category.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_refold.robot_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_refold.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of this production step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_refold.date' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.denature_buffer_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_buffer.id in the PDBX_BUFFER category.\n  The referenced buffer is that in which the protein was denatured.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_refold.denature_buffer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_buffer.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_refold.denature_buffer_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.refold_buffer_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_buffer.id in the PDBX_BUFFER category.\n  The referenced buffer is that in which the protein was refolded.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_refold.refold_buffer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_buffer.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_refold.refold_buffer_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.temperature',
        [ [ 'item_',
            '_item_description.description',
            ' The temperature in degrees celsius at which the protein was refolded.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_refold.temperature' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'celsius' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.time',
        [ [ 'item_',
            '_item_description.description',
            ' The time in hours over which the protein was refolded.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_refold.time' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'hours' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.storage_buffer_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_buffer.id in the PDBX_BUFFER category.\n  The referenced buffer is that in which the refolded protein was stored.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_refold.storage_buffer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_buffer.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_refold.storage_buffer_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_refold.details',
        [ [ 'item_',
            '_item_description.description',
            ' String value containing details of the refolding.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_refold.details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_refold' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '##################################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_PROTEOLYSIS #' ],
      [ '@comment', '##################################' ],
      [ '@comment', '  _pdbx_entity_src_gen_proteolysis.entry_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_proteolysis.entity_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_proteolysis.step_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_proteolysis.next_step_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_proteolysis.end_construct_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_proteolysis.date' ],
      [ '@comment', '  _pdbx_entity_src_gen_proteolysis.robot_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_proteolysis.details' ],
      [ '@comment', '  _pdbx_entity_src_gen_proteolysis.protease' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_proteolysis.protein_protease_ratio' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_proteolysis.cleavage_buffer' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_proteolysis.cleavage_temperature' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_proteolysis.cleavage_time' ],
      [ 'save_',
        'pdbx_entity_src_gen_proteolysis',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the protein purification\n   tag removal steps used in the overall protein production process' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_proteolysis.entry_id',
              '_pdbx_entity_src_gen_proteolysis.entity_id',
              '_pdbx_entity_src_gen_proteolysis.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_proteolysis.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.entry_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_proteolysis.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_proteolysis.entity_id uniquely identifies\n  each protein contained in the project target complex proteins whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category.  This item may be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.entity_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_proteolysis.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is the unique identifier for this tag removal step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.step_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.next_step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item unique identifier for the next production step. This allows\na workflow to have multiple entry points leading to a single product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.next_step_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.end_construct_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT\n        category. The referenced sequence is expected to be the amino\n        acid sequence of the expressed product after the proteolysis\n        step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.end_construct_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_proteolysis.end_construct_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.robot_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to pdbx_robot_system.id\n  in the PDBX_ROBOT_SYSTEM category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.robot_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_proteolysis.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of production step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.date' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.details',
        [ [ 'item_',
            '_item_description.description',
            ' Details of this tag removal step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.details' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.protease',
        [ [ 'item_',
            '_item_description.description',
            ' The name of the protease used for cleavage.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.protease' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.protein_protease_ratio',
        [ [ 'item_',
            '_item_description.description',
            ' The ratio of protein to protease used for the cleavage.\n    = mol protein / mol protease' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.protein_protease_ratio' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.cleavage_buffer_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_buffer.id in the PDBX_BUFFER category.\n  The referenced buffer is that in which the cleavage was performed.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.cleavage_buffer_id' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_buffer.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_proteolysis.cleavage_buffer_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.cleavage_temperature',
        [ [ 'item_',
            '_item_description.description',
            ' The temperature in degrees celsius at which the cleavage was performed.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.cleavage_temperature' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'celsius' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_proteolysis.cleavage_time',
        [ [ 'item_',
            '_item_description.description',
            ' The time in minutes for the cleavage reaction' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_proteolysis.cleavage_time' ],
          [ 'item_',
            '_item.category_id',
            'pdbx_entity_src_gen_proteolysis' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'minutes' ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_CHROM #' ],
      [ '@comment', '############################' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.entry_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.entity_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.next_step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.end_construct_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.robot_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.date' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.column_type' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.column_volume' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_chrom.column_temperature' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_chrom.equilibration_buffer' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.flow_rate' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.elution_protocol' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_chrom.sample_prep_details' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.sample_volume' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_chrom.protein_concentration' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.volume_fraction' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_chrom.volume_pooled_fractions' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_chrom.yield_pooled_fractions' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.yield_method' ],
      [ '@comment', '  _pdbx_entity_src_gen_chrom.post_treatment' ],
      [ 'save_',
        'pdbx_entity_src_gen_chrom',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the chromatographic steps used in the\n   purification of the protein.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_chrom.entry_id',
              '_pdbx_entity_src_gen_chrom.entity_id',
              '_pdbx_entity_src_gen_chrom.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_chrom.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_chrom.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_chrom.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_chrom.entity_id uniquely identifies\n  each protein contained in the project target complex proteins whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category. This item may be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_chrom.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_chrom.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.step_id',
        [ [ 'item_',
            '_item_description.description',
            '  This item is the unique identifier for this chromatography step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_chrom.step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.next_step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item unique identifier for the next production step. This allows\na workflow to have multiple entry points leading to a single product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.next_step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.end_construct_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT\n        category. The referenced sequence is expected to be the amino\n        acid sequence of the expressed product after the\n        chromatography step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.end_construct_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_chrom.end_construct_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.robot_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to pdbx_robot_system.id\n  in the PDBX_ROBOT_SYSTEM category.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_chrom.robot_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_chrom.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of production step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_chrom.date' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.column_type',
        [ [ 'item_',
            '_item_description.description',
            ' The type of column used in this step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.column_type' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.column_volume',
        [ [ 'item_',
            '_item_description.description',
            ' The volume of the column used in this step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.column_volume' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'milliliters' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.column_temperature',
        [ [ 'item_',
            '_item_description.description',
            ' The temperature in degrees celsius at which this column was run.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.column_temperature' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'celsius' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.equilibration_buffer_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_buffer.id in the PDBX_BUFFER category.\n  The referenced buffer is that in which the column was equilibrated.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.equilibration_buffer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_buffer.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_chrom.equilibration_buffer_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.flow_rate',
        [ [ 'item_',
            '_item_description.description',
            ' The rate at which the equilibration buffer flowed through the column.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_chrom.flow_rate' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'ml_per_min' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.elution_buffer_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_buffer.id in the PDBX_BUFFER category.\n  The referenced buffer is that with which the protein was eluted.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.elution_buffer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_buffer.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_chrom.elution_buffer_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.elution_protocol',
        [ [ 'item_',
            '_item_description.description',
            ' Details of the elution protocol.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.elution_protocol' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.sample_prep_details',
        [ [ 'item_',
            '_item_description.description',
            ' Details of the sample preparation prior to running the column.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.sample_prep_details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.sample_volume',
        [ [ 'item_',
            '_item_description.description',
            ' The volume of protein solution run on the column.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.sample_volume' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'milliliters' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.sample_concentration',
        [ [ 'item_',
            '_item_description.description',
            ' The concentration of the protein solution put onto the column.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.sample_concentration' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'mg_per_ml' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.sample_conc_method',
        [ [ 'item_',
            '_item_description.description',
            ' The method used to determine the concentration of the protein solution put\n  onto the column.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.sample_conc_method' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.volume_pooled_fractions',
        [ [ 'item_',
            '_item_description.description',
            ' The total volume of all the fractions pooled to give the purified protein\n  solution.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.volume_pooled_fractions' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'milliliters' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.yield_pooled_fractions',
        [ [ 'item_',
            '_item_description.description',
            ' The yield in milligrams of protein recovered in the pooled fractions.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.yield_pooled_fractions' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'milligrams' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.yield_method',
        [ [ 'item_',
            '_item_description.description',
            ' The method used to determine the yield' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.yield_method' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_chrom.post_treatment',
        [ [ 'item_',
            '_item_description.description',
            ' Details of any post-chromatographic treatment of the protein sample.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_chrom.post_treatment' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_chrom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_FRACT #' ],
      [ '@comment', '############################' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.entry_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.entity_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.next_step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.end_construct_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.robot_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.date' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.method' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.temperature' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.details' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.protein_location' ],
      [ '@comment', '  _pdbx_entity_src_gen_fract.protein_yield' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_fract.protein_yield_method' ],
      [ 'save_',
        'pdbx_entity_src_gen_fract',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the fraction steps used in\n   the overall protein production process. Examples of fractionation\n   steps are centrifugation and magnetic bead pull-down purification.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_fract.entry_id',
              '_pdbx_entity_src_gen_fract.entity_id',
              '_pdbx_entity_src_gen_fract.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_fract.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_fract.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_fract.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_fract.entity_id uniquely identifies\n  each protein contained in the project target protein complex whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category.  This item may be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_fract.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_fract.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is the unique identifier for this fractionation step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_fract.step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.next_step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item unique identifier for the next production step. This allows\na workflow to have multiple entry points leading to a single product.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_fract.next_step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.end_construct_id',
        [ [ 'item_',
            '_item_description.description',
            '       This item is a pointer to pdbx_construct.id in the PDBX_CONSTRUCT\n        category. The referenced sequence is expected to be the amino\n        acid sequence of the expressed product after the fractionation\n        step.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_fract.end_construct_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_fract.end_construct_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.robot_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to pdbx_robot_system.id\n  in the PDBX_ROBOT_SYSTEM category.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_fract.robot_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_fract.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of this production step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_fract.date' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.method',
        [ [ 'item_',
            '_item_description.description',
            ' This item describes the method of fractionation.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_fract.method' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'C',
              'Centrifugation',
              'M',
              'Magnetic beads',
              'P',
              'Phase separation' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.temperature',
        [ [ 'item_',
            '_item_description.description',
            ' The temperature in degrees celsius at which the fractionation was performed.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_fract.temperature' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'celsius' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.details',
        [ [ 'item_',
            '_item_description.description',
            ' String value containing details of the fractionation.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_fract.details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.protein_location',
        [ [ 'item_',
            '_item_description.description',
            ' The fraction containing the protein of interest.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_fract.protein_location' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'S',
              'Supernatant',
              'P',
              'Pellet',
              'B',
              'Band in gradient',
              'M',
              'Magnetic beads',
              'A',
              'Aqueous phase',
              'L',
              'Lipid phase' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.protein_volume',
        [ [ 'item_',
            '_item_description.description',
            ' The volume of the fraction containing the protein.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_fract.protein_volume' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'milliliters' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.protein_yield',
        [ [ 'item_',
            '_item_description.description',
            ' The yield in milligrams of protein from the fractionation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_fract.protein_yield' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'milligrams' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_fract.protein_yield_method',
        [ [ 'item_',
            '_item_description.description',
            ' The method used to determine the yield' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_fract.protein_yield_method' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_fract' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '###########################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_PURE #' ],
      [ '@comment', '###########################' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.entry_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.entity_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.next_step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.date' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.conc_device_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.conc_details' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.conc_assay_method' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_pure.protein_concentration' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.protein_yield' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.protein_purity' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_pure.protein_oligomeric_state' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.storage_buffer_id' ],
      [ '@comment',
        '  _pdbx_entity_src_gen_pure.storage_temperature' ],
      [ '@comment', '  _pdbx_entity_src_gen_pure.summary' ],
      [ 'save_',
        'pdbx_entity_src_gen_pure',
        [ [ 'item_',
            '_category.description',
            '  This category contains details for the final purified protein product. Note\n   that this category does not contain the amino acid sequence of the protein.\n   The sequence will be found in the ENTITY_POLY_SEQ entry with matching\n   entity_id.\n   Only one PDBX_ENTITY_SRC_GEN_PURE category is allowed per entity, hence there is\n   no step_id for this category.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_pure.entry_id',
              '_pdbx_entity_src_gen_pure.entity_id',
              '_pdbx_entity_src_gen_pure.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_pure.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_pure.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_pure.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_pure.entity_id uniquely identifies\n  each protein contained in the project target complex proteins whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category.  This item may be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_pure.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_pure.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item unique identifier the production step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_pure.step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.product_id',
        [ [ 'item_',
            '_item_description.description',
            ' When present, this item should be a globally unique identifier\n  that identifies the final product. It is envisaged that this\n  should be the same as and product code associated with the\n  sample and would provide the key by which information about\n  the production process may be extracted from the protein\n  production facility. For files describing the protein\n  production process (i.e. where _entity.type is \'P\' or \'E\')\n  this should have the same value as _entry.id' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_pure.product_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of production step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_pure.date' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.conc_device_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to pdbx_robot_system.id\n  in the PDBX_ROBOT_SYSTEM category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_pure.conc_device_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_pure.conc_device_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.conc_details',
        [ [ 'item_',
            '_item_description.description',
            ' Details of the protein concentration procedure' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_pure.conc_details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.conc_assay_method',
        [ [ 'item_',
            '_item_description.description',
            ' The method used to measure the protein concentration' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_pure.conc_assay_method' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.protein_concentration',
        [ [ 'item_',
            '_item_description.description',
            ' The final concentration of the protein.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_pure.protein_concentration' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'mg_per_ml' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.protein_yield',
        [ [ 'item_',
            '_item_description.description',
            ' The yield of protein in milligrams.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_pure.protein_yield' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'milligrams' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.protein_purity',
        [ [ 'item_',
            '_item_description.description',
            ' The purity of the protein (percent).' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_pure.protein_purity' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ '@comment', '   _item_units.code           percent' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.protein_oligomeric_state',
        [ [ 'item_',
            '_item_description.description',
            ' The oligomeric state of the protein. Monomeric is 1, dimeric 2, etc.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_pure.protein_oligomeric_state' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.storage_buffer_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is a pointer to pdbx_buffer.id in the PDBX_BUFFER category.\n  The referenced buffer is that in which the protein was stored.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_pure.storage_buffer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_buffer.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_pure.storage_buffer_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.storage_temperature',
        [ [ 'item_',
            '_item_description.description',
            ' The temperature in degrees celsius at which the protein was stored.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_pure.storage_temperature' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'celsius' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_pure.summary',
        [ [ 'item_',
            '_item_description.description',
            ' Summary of the details of protein purification method used to obtain the final\n  protein product.  This description should include any lysis, fractionation,\n  proteolysis, refolding, chromatography used as well as the method used the \n  characterize the final product.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_pure.summary' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_pure' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '################################' ],
      [ '@comment', ' PDBX_ENTITY_SRC_GEN_CHARACTER #' ],
      [ '@comment', '################################' ],
      [ '@comment', '  _pdbx_entity_src_gen_character.entry_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_character.entity_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_character.step_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_character.robot_id' ],
      [ '@comment', '  _pdbx_entity_src_gen_character.date' ],
      [ '@comment', '  _pdbx_entity_src_gen_character.method' ],
      [ '@comment', '  _pdbx_entity_src_gen_character.value' ],
      [ '@comment', '  _pdbx_entity_src_gen_character.details' ],
      [ 'save_',
        'pdbx_entity_src_gen_character',
        [ [ 'item_',
            '_category.description',
            '  This category contains details of protein characterisation. It\n   refers to the characteristion of the product of a specific\n   step.' ],
          [ 'item_', '_category.id', 'pdbx_entity_src_gen_character' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'protein_production_group',
              'entity_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_entity_src_gen_character.entry_id',
              '_pdbx_entity_src_gen_character.entity_id',
              '_pdbx_entity_src_gen_character.step_id' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_character.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_character.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_character.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_character' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_character.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_character.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_entity_src_gen_character.entity_id uniquely identifies\n  each protein contained in the project target complex proteins whose\n  structure is to be determined. This data item is a pointer to _entity.id\n  in the ENTITY category. This item may be a site dependent bar code.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_character.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_character' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_character.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_character.step_id',
        [ [ 'item_',
            '_item_description.description',
            ' This item is the unique identifier for the step whose product\n  has been characterised.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_character.step_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_character' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_character.robot_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to pdbx_robot_system.id\n  in the PDBX_ROBOT_SYSTEM category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_character.robot_id' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_character' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_entity_src_gen_character.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_character.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date of characterisation step.' ],
          [ 'item_', '_item.name', '_pdbx_entity_src_gen_character.date' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_character' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_character.method',
        [ [ 'item_',
            '_item_description.description',
            ' The method used for protein characterisation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_character.method' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_character' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'Dynamic light scattering', 'Mass spectrometry' ] ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_character.result',
        [ [ 'item_',
            '_item_description.description',
            ' The result from this method of protein characterisation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_character.result' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_character' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_entity_src_gen_character.details',
        [ [ 'item_',
            '_item_description.description',
            ' Any details associated with this method of protein characterisation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_entity_src_gen_character.details' ],
          [ 'item_', '_item.category_id', 'pdbx_entity_src_gen_character' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#################' ],
      [ '@comment', ' PDBX_CONSTRUCT #' ],
      [ '@comment', '#################' ],
      [ '@comment', '    _pdbx_construct.entry_id' ],
      [ '@comment', '    _pdbx_construct.id' ],
      [ '@comment', '    _pdbx_construct.name' ],
      [ '@comment', '    _pdbx_construct.class' ],
      [ '@comment', '    _pdbx_construct.type' ],
      [ '@comment', '    _pdbx_construct.seq' ],
      [ '@comment', '    _pdbx_construct.date' ],
      [ '@comment', '    _pdbx_construct.details' ],
      [ '@comment', '    _pdbx_construct.robot_id' ],
      [ '@comment', '    _pdbx_construct.ref_db_code' ],
      [ '@comment', '    _pdbx_construct.ref_db_name' ],
      [ '@comment', '    _pdbx_construct.ref_db_align_beg' ],
      [ '@comment', '    _pdbx_construct.ref_db_align_end' ],
      [ '@comment', '    _pdbx_construct.flag_align' ],
      [ '@comment', '    _pdbx_construct.flag_diff' ],
      [ 'save_',
        'pdbx_construct',
        [ [ 'item_',
            '_category.description',
            ' Data items in the PDBX_CONSTRUCT category specify a sequence of\n  nucleic acids or amino acids. It is a catch-all that may be used to\n  provide details of sequences known to be relevant to the project as well\n  as primers, plasmids, proteins and such like that are either used or\n  produced during the protein production process. Molecules described\n  here are not necessarily complete, so for instance it would be\n  possible to include either a complete plasmid or just its insert.\n  This category may be considered as an abbreviated form of _entity where\n  the molecules described are not required to appear in the final co-ordinates.\n\n  Note that the details provided here all pertain to a single entry as defined\n  at deposition. It is anticipated that _pdbx_construct.id would also be\n  composed of a sequence that is unique within a given site prefixed by a code\n  that identifies that site and would, therefore, be GLOBALLY unique. Thus\n  this category could also be used locally to store details about the different\n  constructs used during protein production without reference to the entry_id\n (which only becomes a  meaningful concept during deposition).' ],
          [ 'item_', '_category.id', 'pdbx_construct' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_', [ '_category_key.name' ], [ '_pdbx_construct.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'entity_group',
              'protein_production_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - hypothetical example',
              '\n    _pdbx_construct.entry_id       111000111\n    _pdbx_construct.id             1\n    _pdbx_construct.type           DNA\n    _pdbx_construct.entity_id      1\n    _pdbx_construct.seq\n  ;  gatgctgtag gcataggctt ggttatgccg gtactgccgg gcctcttgcg ggatatcgtc\n     gctcaaggcg cactcccgtt ctggataatg ttttttgcgc cgacatcata acggttctgg\n     caaatattct gaaatgagct gttgacaatt aatcatcgat aagcttcttg\n    # - - - - data truncated for brevity - - - -\n  ;',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_construct.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_construct.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_construct.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_construct.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_construct.id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_construct.id must uniquely identify a record\n  in the PDBX_CONSTRUCT list and should be arranged so that it is\n  composed of a site-speicific prefix combined with a value that is\n  unique within a given site.Note that this item need not be a\n  number; it can be any unique identifier.' ],
          [ 'loop_', [ '_item.name' ], [ '_pdbx_construct.id' ] ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_construct.name',
        [ [ 'item_',
            '_item_description.description',
            ' _pdbx_construct.name  provides a placeholder for the local name\n  of the construct, for example the plasmid name if this category\n  is used to list plasmids.' ],
          [ 'loop_', [ '_item.name' ], [ '_pdbx_construct.name' ] ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_construct.organisation',
        [ [ 'item_',
            '_item_description.description',
            ' _pdbx_construct.organisation describes the organisation in which\n  the _pdbx_construct.id is unique. This will normally be the lab\n  in which the constrcut originated. It is envisaged that this item\n  will permit a globally unique identifier to be constructed in cases\n  where this is not possible from the _pdbx_construct.id alone.' ],
          [ 'loop_', [ '_item.name' ], [ '_pdbx_construct.organisation' ] ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_construct.entity_id',
        [ [ 'item_',
            '_item_description.description',
            ' In cases where the construct IS found in the co-ordinates then this\n  item provides a  pointer to _entity.id in the ENTITY category for\n  the corresponding molecule.' ],
          [ 'item_', '_item.name', '_pdbx_construct.entity_id' ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_construct.entity_id' ] ] ],
      [ 'save_',
        '_pdbx_construct.robot_id',
        [ [ 'item_',
            '_item_description.description',
            ' In cases where the sequence has been determined by a robot this\n  data item provides a pointer to pdbx_robot_system.id in the\n  PDBX_ROBOT_SYSTEM category for the robot responsible' ],
          [ 'item_', '_item.name', '_pdbx_construct.robot_id' ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_robot_system.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_construct.robot_id' ] ] ],
      [ 'save_',
        '_pdbx_construct.date',
        [ [ 'item_',
            '_item_description.description',
            ' The date that the sequence was determined.' ],
          [ 'item_', '_item.name', '_pdbx_construct.date' ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd:hh:mm' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '2003-12-25', '2003-12-25:09:00' ] ] ] ],
      [ 'save_',
        '_pdbx_construct.details',
        [ [ 'item_',
            '_item_description.description',
            ' Additional details about the construct that cannot be\n  represented in the category _pdbx_construct_feature.' ],
          [ 'item_', '_item.name', '_pdbx_construct.details' ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_construct.class',
        [ [ 'item_',
            '_item_description.description',
            ' The primary function of the construct. This should be considered\n  as a guideline only.' ],
          [ 'item_', '_item.name', '_pdbx_construct.class' ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'plasmid', 'protein', 'insert', 'primer', 'transcript' ] ] ] ],
      [ 'save_',
        '_pdbx_construct.type',
        [ [ 'item_',
            '_item_description.description',
            ' The type of nucleic acid sequence in the construct. Note that\n  to find all the DNA molecules it is necessary to search for\n  DNA + cDNA and for RNA, RNA + mRNA + tRNA.' ],
          [ 'item_', '_item.name', '_pdbx_construct.type' ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'DNA', 'RNA', 'cDNA', 'mRNA', 'tRNA', 'protein' ] ] ] ],
      [ 'save_',
        '_pdbx_construct.seq',
        [ [ 'item_',
            '_item_description.description',
            ' sequence expressed as string of one-letter base codes or one\n  letter amino acid codes. Unusual residues may be represented\n  either using the appropriate one letter code wild cards or\n  by the three letter code in parentheses.' ],
          [ 'item_', '_item.name', '_pdbx_construct.seq' ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '    gatgctgtag gcataggctt ggttatgccg gtactgccgg gcctcttgcg ggatatcgtc\n     gctcaaggcg cactcccgtt ctggataatg ttttttgcgc cgacatcata acggttctgg\n     caaatattct gaaatgagct gttgacaatt aatcatcgat aagcttcttg' ] ] ],
      [ '@comment', '##########################' ],
      [ '@comment', ' PDBX_CONSTRUCT_FEATURE  #' ],
      [ '@comment', '##########################' ],
      [ '@comment', '        _pdbx_construct_feature.id' ],
      [ '@comment', '        _pdbx_construct_feature.entry_id' ],
      [ '@comment', '        _pdbx_construct_feature.entity_id' ],
      [ '@comment', '        _pdbx_construct_feature.construct_id' ],
      [ '@comment', '        _pdbx_construct_feature.start_seq' ],
      [ '@comment', '        _pdbx_construct_feature.end_seq' ],
      [ '@comment', '        _pdbx_construct_feature.type' ],
      [ '@comment', '        _pdbx_construct_feature.details' ],
      [ 'save_',
        'pdbx_construct_feature',
        [ [ 'item_',
            '_category.description',
            ' Data items in the PDBX_CONSTRUCT_FEATURE category may be used to\n  specify various properties of a nucleic acid sequence used during\n  protein production.' ],
          [ 'item_', '_category.id', 'pdbx_construct_feature' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_construct_feature.construct_id',
              '_pdbx_construct_feature.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group',
              'entity_group',
              'protein_production_group',
              'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - vector pUC28',
              '\n        loop_\n        _pdbx_construct_feature.id\n        _pdbx_construct_feature.entry_id\n        _pdbx_construct_feature.construct_id\n        _pdbx_construct_feature.start_seq\n        _pdbx_construct_feature.end_seq\n        _pdbx_construct_feature.type\n        _pdbx_construct_feature.details\n  1  111000111   1 . . .\n     "pKK84-1 ClaI 5260bp 5247..5247 ptac11 TaqI-TaqI 192bp, -35 trp promoter -> pEA300 5452bp"\n  2  111000111   1 . . .\n     "pKK84-1 1..5246 5246bp ClaI = AT^CGAT TaqI =  T^CGA cgact..."\n  3  111000111   1 5247  5436 .\n     "ptac11 190bp ...cat TaqI =  T^CGA ClaI = AT^CGAT"\n  4  111000111   1 5437  5450  .\n     "pKK84-1 5247..5260 14bp"\n  5  111000111   1 . . misc_binding\n     "SIT unique EcoRI-ClaI-HindIII-BamHI-PvuII"\n  6  111000111   1 . .   rep_origin\n     "ORI E. coli pMB1 (ColE1 and pBR322)"\n  7  111000111   1 . .  promoter\n     "PRO E. coli trp"\n  8  111000111   1 . . CDS\n     "ANT E. coli beta-lactamase gene (bla) ampicillin resistance gene (apr/amp)"',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_construct_feature.id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_construct_feature.id must uniquely\n  identify a record in the PDBX_CONSTRUCT_FEATURE list.\n  Note that this item need not be a number; it can be any unique\n  identifier.' ],
          [ 'loop_', [ '_item.name' ], [ '_pdbx_construct_feature.id' ] ],
          [ 'item_', '_item.category_id', 'pdbx_construct_feature' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_construct_feature.construct_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_construct_feature.construct_id uniquely\n  identifies the construct with which the feature is\n  associated. This is a pointer to _pdbx_construct.id\n  This item may be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_construct_feature.construct_id' ],
          [ 'item_', '_item.category_id', 'pdbx_construct_feature' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_construct.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_construct_feature.construct_id' ] ] ],
      [ 'save_',
        '_pdbx_construct_feature.entry_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _pdbx_construct_feature.entry_id uniquely identifies\n  a sample consisting of one or more proteins whose structure is\n  to be determined. This is a pointer to _entry.id. This item may\n  be a site dependent bar code.' ],
          [ 'item_', '_item.name', '_pdbx_construct_feature.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_construct' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_construct_feature.entry_id' ] ] ],
      [ 'save_',
        '_pdbx_construct_feature.start_seq',
        [ [ 'item_',
            '_item_description.description',
            ' The sequence position at which the feature begins' ],
          [ 'item_', '_item.name', '_pdbx_construct_feature.start_seq' ],
          [ 'item_', '_item.category_id', 'pdbx_construct_feature' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_construct_feature.end_seq',
        [ [ 'item_',
            '_item_description.description',
            ' The sequence position at which the feature ends' ],
          [ 'item_', '_item.name', '_pdbx_construct_feature.end_seq' ],
          [ 'item_', '_item.category_id', 'pdbx_construct_feature' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '1', '1', '1' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_construct_feature.type',
        [ [ 'item_',
            '_item_description.description',
            ' The type of the feature' ],
          [ 'item_', '_item.name', '_pdbx_construct_feature.type' ],
          [ 'item_', '_item.category_id', 'pdbx_construct_feature' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_construct_feature.details',
        [ [ 'item_',
            '_item_description.description',
            ' Details that describe the feature' ],
          [ 'item_', '_item.name', '_pdbx_construct_feature.details' ],
          [ 'item_', '_item.category_id', 'pdbx_construct_feature' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', ' PDBX_ROBOT_SYSTEM #' ],
      [ '@comment', '####################' ],
      [ '@comment', '    _pdbx_robot_system.id' ],
      [ '@comment', '    _pdbx_robot_system.model' ],
      [ '@comment', '    _pdbx_robot_system.type' ],
      [ '@comment', '    _pdbx_robot_system.manufacturer' ],
      [ 'save_',
        'pdbx_robot_system',
        [ [ 'item_',
            '_category.description',
            ' The details about each robotic system used to collect data for this\n  project.' ],
          [ 'item_', '_category.id', 'pdbx_robot_system' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_robot_system.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'protein_production_group', 'pdbx_group' ] ] ] ],
      [ 'save_',
        '_pdbx_robot_system.id',
        [ [ 'item_',
            '_item_description.description',
            ' Assign a numerical ID to each instrument.' ],
          [ 'item_', '_item.name', '_pdbx_robot_system.id' ],
          [ 'item_', '_item.category_id', 'pdbx_robot_system' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_robot_system.model',
        [ [ 'item_',
            '_item_description.description',
            ' The model of the robotic system.' ],
          [ 'item_', '_item.name', '_pdbx_robot_system.model' ],
          [ 'item_', '_item.category_id', 'pdbx_robot_system' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_robot_system.type',
        [ [ 'item_',
            '_item_description.description',
            ' The type of robotic system used for in the production pathway.' ],
          [ 'item_', '_item.name', '_pdbx_robot_system.type' ],
          [ 'item_', '_item.category_id', 'pdbx_robot_system' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_robot_system.manufacturer',
        [ [ 'item_',
            '_item_description.description',
            ' The name of the manufacturer of the robotic system.' ],
          [ 'item_', '_item.name', '_pdbx_robot_system.manufacturer' ],
          [ 'item_', '_item.category_id', 'pdbx_robot_system' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# PDB_BUFFER  ##' ],
      [ '@comment', '################' ],
      [ '@comment', '    _pdbx_buffer.id' ],
      [ '@comment', '    _pdbx_buffer.name' ],
      [ '@comment', '    _pdbx_buffer.details' ],
      [ '@comment', '    _pdbx_buffer_components.id' ],
      [ '@comment', '    _pdbx_buffer_components.buffer_id' ],
      [ '@comment', '    _pdbx_buffer_components.name' ],
      [ '@comment', '    _pdbx_buffer_components.volume' ],
      [ '@comment', '    _pdbx_buffer_components.conc' ],
      [ '@comment', '    _pdbx_buffer_components.details' ],
      [ '@comment', '    _pdbx_buffer_components.conc_units' ],
      [ '@comment', '    _pdbx_buffer_components.isotopic_labeling' ],
      [ 'save_',
        'pdbx_buffer',
        [ [ 'item_',
            '_category.description',
            '   Data items in the PDBX_BUFFER category\n    record details of the sample buffer. ' ],
          [ 'item_', '_category.id', 'pdbx_buffer' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_buffer.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'protein_production_group', 'pdbx_group' ] ] ] ],
      [ 'save_',
        '_pdbx_buffer.id',
        [ [ 'item_',
            '_item_description.description',
            '       The value of _pdbx_buffer.id must\n        uniquely identify the sample buffer.' ],
          [ 'item_', '_item.name', '_pdbx_buffer.id' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_buffer.name',
        [ [ 'item_',
            '_item_description.description',
            '       The name of each buffer. ' ],
          [ 'item_', '_item.name', '_pdbx_buffer.name' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'Acetic acid' ] ] ],
      [ 'save_',
        '_pdbx_buffer.details',
        [ [ 'item_',
            '_item_description.description',
            '   Any additional details to do with buffer. ' ],
          [ 'item_', '_item.name', '_pdbx_buffer.details' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'aerated' ] ] ],
      [ '@comment', '##############################' ],
      [ '@comment', '# PDBX_BUFFER_COMPONENTS    ##' ],
      [ '@comment', '##############################' ],
      [ '@comment', ' _pdbx_buffer_components.id' ],
      [ '@comment', ' _pdbx_buffer_components.buffer_id' ],
      [ '@comment', ' _pdbx_buffer_components.name' ],
      [ '@comment', ' _pdbx_buffer_components.volume' ],
      [ '@comment', ' _pdbx_buffer_components.conc' ],
      [ '@comment', ' _pdbx_buffer_components.details' ],
      [ '@comment', ' _pdbx_buffer_components.conc_units' ],
      [ '@comment', ' _pdbx_buffer_components.isotopic_labeling' ],
      [ 'save_',
        'pdbx_buffer_components',
        [ [ 'item_',
            '_category.description',
            '  Constituents of buffer in sample ' ],
          [ 'item_', '_category.id', 'pdbx_buffer_components' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_buffer_components.buffer_id',
              '_pdbx_buffer_components.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'protein_production_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '   Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              'loop_\n  _pdbx_buffer_components.buffer_id\n  _pdbx_buffer_components.id\n  _pdbx_buffer_components.name\n  _pdbx_buffer_components.volume\n  _pdbx_buffer_components.conc\n  _pdbx_buffer_components.details\n  1    1 \'NaCl\'            \'0.200 \'  \'4 \'     .\n  1    2 \'Acetic Acid\'     \'0.047 \'  \'100\'    .\n  1    3 \'water\'           \'0.700 \'  \'neat\'   .' ] ] ] ],
      [ 'save_',
        '_pdbx_buffer_components.id',
        [ [ 'item_',
            '_item_description.description',
            '   The value of _pdbx_buffer_components.id must \n    uniquely identify a component of the buffer.' ],
          [ 'item_', '_item.name', '_pdbx_buffer_components.id' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_buffer_components.buffer_id',
        [ [ 'item_',
            '_item_description.description',
            '      This data item is a pointer to _pdbx_buffer.id in the BUFFER category.' ],
          [ 'item_', '_item.name', '_pdbx_buffer_components.buffer_id' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_buffer.id' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_buffer_components.buffer_id' ] ] ],
      [ 'save_',
        '_pdbx_buffer_components.name',
        [ [ 'item_',
            '_item_description.description',
            '       The name of each buffer component. ' ],
          [ 'item_', '_item.name', '_pdbx_buffer_components.name' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'Acetic acid' ] ] ],
      [ 'save_',
        '_pdbx_buffer_components.volume',
        [ [ 'item_',
            '_item_description.description',
            '       The volume of buffer component. ' ],
          [ 'item_', '_item.name', '_pdbx_buffer_components.volume' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '0.200' ] ] ],
      [ 'save_',
        '_pdbx_buffer_components.conc',
        [ [ 'item_',
            '_item_description.description',
            '       The millimolar concentration of buffer component. ' ],
          [ 'item_', '_item.name', '_pdbx_buffer_components.conc' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '200' ] ] ],
      [ 'save_',
        '_pdbx_buffer_components.details',
        [ [ 'item_',
            '_item_description.description',
            '       Any additional details to do with buffer composition. ' ],
          [ 'item_', '_item.name', '_pdbx_buffer_components.details' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'pH adjusted with NaOH' ] ] ],
      [ 'save_',
        '_pdbx_buffer_components.conc_units',
        [ [ 'item_',
            '_item_description.description',
            '       The concentration units of the component.' ],
          [ 'item_', '_item.name', '_pdbx_buffer_components.conc_units' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'mg/mL for mg per milliliter',
              'mM for millimolar',
              '% for percent by volume' ] ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ '%',
              'percent by volume',
              'mM',
              'millimolar',
              'mg/mL',
              'mg per milliliter',
              'M',
              'molar',
              'g/L',
              'grams per liter' ] ] ] ],
      [ 'save_',
        '_pdbx_buffer_components.isotopic_labeling',
        [ [ 'item_',
            '_item_description.description',
            '       The isotopic composition of each component, including\n        the % labeling level, if known. For example:\n        1. Uniform (random) labeling with 15N: U-15N\n        2. Uniform (random) labeling with 13C, 15N at known labeling\n           levels: U-95% 13C;U-98% 15N\n        3. Residue selective labeling: U-95% 15N-Thymine\n        4. Site specific labeling: 95% 13C-Ala18,\n        5. Natural abundance labeling in an otherwise uniformly labled\n           biomolecule is designated by NA: U-13C; NA-K,H ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_buffer_components.isotopic_labeling' ],
          [ 'item_', '_item.category_id', 'pdbx_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'U-15N', 'U-13C', 'U-15N,13C', 'U-2H', 'other' ] ] ] ],
      [ '@comment', '## EOF  mmcif_pdbx-def-6.dic' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment',
        ' File:  mmcif_iims-def-2.dic                                             ' ],
      [ '@comment', ' ' ],
      [ '@comment',
        '               PDB Extension Dictionary for 3-Dimensional' ],
      [ '@comment',
        '                          Electron Microscopy                ' ],
      [ '@comment',
        '                           Definition Section 2' ],
      [ '@comment', '                            ' ],
      [ '@comment',
        '                                                                        ' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment', '#' ],
      [ '@comment', '##' ],
      [ '@comment',
        ' Please note the data items listed below are not for the PDB ' ],
      [ '@comment',
        '            but are intended solely for the EM database' ],
      [ '@comment', '  _em_entity_assembly.ebi_organism_scientific' ],
      [ '@comment', '  _em_entity_assembly.ebi_organism_common' ],
      [ '@comment', '  _em_entity_assembly.ebi_strain' ],
      [ '@comment', '  _em_entity_assembly.ebi_tissue' ],
      [ '@comment', '  _em_entity_assembly.ebi_cell' ],
      [ '@comment', '  _em_entity_assembly.ebi_organelle' ],
      [ '@comment', '  _em_entity_assembly.ebi_cellular_location' ],
      [ '@comment', '  _em_entity_assembly.ebi_engineered' ],
      [ '@comment', '  _em_entity_assembly.ebi_expression_system' ],
      [ '@comment',
        '  _em_entity_assembly.ebi_expression_system_plasmid' ],
      [ '@comment', '##' ],
      [ '@comment', ' categories' ],
      [ '@comment', '    ' ],
      [ '@comment', '   EM_ASSEMBLY' ],
      [ '@comment', '   _em_assembly.id' ],
      [ '@comment', '   _em_assembly.entry_id' ],
      [ '@comment', '   _em_assembly.name' ],
      [ '@comment', '   _em_assembly.aggregation_state' ],
      [ '@comment', '   _em_assembly.composition' ],
      [ '@comment', '   _em_assembly.num_components' ],
      [ '@comment', '   _em_assembly.mol_wt_exp' ],
      [ '@comment', '   _em_assembly.mol_wt_theo' ],
      [ '@comment', '   _em_assembly.mol_wt_method' ],
      [ '@comment', '   _em_assembly.details' ],
      [ '@comment', '   EM_ENTITY_ASSEMBLY' ],
      [ '@comment', '   _em_entity_assembly.id' ],
      [ '@comment', '   _em_entity_assembly.assembly_id' ],
      [ '@comment', '   _em_entity_assembly.type' ],
      [ '@comment', '   _em_entity_assembly.name' ],
      [ '@comment', '   _em_entity_assembly.details' ],
      [ '@comment', '   _em_entity_assembly.go_id' ],
      [ '@comment', '   _em_entity_assembly.ipr_id' ],
      [ '@comment', '   _em_entity_assembly.synonym' ],
      [ '@comment', '   EM_ENTITY_ASSEMBLY_LIST' ],
      [ '@comment', '   _em_entity_assembly_list.id ' ],
      [ '@comment', '   _em_entity_assembly_list.entity_id' ],
      [ '@comment', '   _em_entity_assembly_list.entity_assembly_id' ],
      [ '@comment', '   _em_entity_assembly_list.oligomeric_details' ],
      [ '@comment', '   _em_entity_assembly_list.number_of_copies' ],
      [ '@comment', '   EM_VIRUS_ENTITY' ],
      [ '@comment', '   _em_virus_entity.id' ],
      [ '@comment', '   _em_virus_entity.virus_host_category' ],
      [ '@comment', '   _em_virus_entity.virus_host_species' ],
      [ '@comment', '   _em_virus_entity.virus_host_growth_cell' ],
      [ '@comment', '   _em_virus_entity.virus_type' ],
      [ '@comment', '   _em_virus_entity.virus_isolate' ],
      [ '@comment', '   _em_virus_entity.ictvdb_id' ],
      [ '@comment', '   _em_virus_entity.entity_assembly_id' ],
      [ '@comment', '   _em_virus_entity.enveloped' ],
      [ '@comment', '   _em_virus_entity.empty' ],
      [ '@comment', '   EM_ICOS_VIRUS_SHELLS' ],
      [ '@comment', '   _em_icos_virus_shells.virus_entity_id' ],
      [ '@comment', '   _em_icos_virus_shells.id' ],
      [ '@comment', '   _em_icos_virus_shells.shell_diameter' ],
      [ '@comment', '   _em_icos_virus_shells.triangulation_num' ],
      [ '@comment', '   EM_SAMPLE_PREPARATION' ],
      [ '@comment', '   _em_sample_preparation.entry_id' ],
      [ '@comment', '   _em_sample_preparation.id' ],
      [ '@comment', '   _em_sample_preparation.ph' ],
      [ '@comment', '   _em_sample_preparation.buffer_id' ],
      [ '@comment', '   _em_sample_preparation.sample_concentration' ],
      [ '@comment', '   _em_sample_preparation.2d_crystal_grow_id' ],
      [ '@comment', '   _em_sample_preparation.support_id' ],
      [ '@comment', '   EM_SAMPLE_SUPPORT' ],
      [ '@comment', '   _em_sample_support.id' ],
      [ '@comment', '   _em_sample_support.film_material' ],
      [ '@comment', '   _em_sample_support.method' ],
      [ '@comment', '   _em_sample_support.grid_material' ],
      [ '@comment', '   _em_sample_support.grid_mesh_size' ],
      [ '@comment', '   _em_sample_support.grid_type' ],
      [ '@comment', '   _em_sample_support.pretreatment' ],
      [ '@comment', '   _em_sample_support.details' ],
      [ '@comment', '   _em_sample_support.citation_id' ],
      [ '@comment', '   EM_2D_CRYSTAL_GROW' ],
      [ '@comment', '   _em_2d_crystal_grow.id' ],
      [ '@comment', '   _em_2d_crystal_grow.method' ],
      [ '@comment', '   _em_2d_crystal_grow.apparatus' ],
      [ '@comment', '   _em_2d_crystal_grow.atmosphere' ],
      [ '@comment', '   _em_2d_crystal_grow.pH' ],
      [ '@comment', '   _em_2d_crystal_grow.temp' ],
      [ '@comment', '   _em_2d_crystal_grow.time' ],
      [ '@comment', '   _em_2d_crystal_grow.buffer_id' ],
      [ '@comment', '   _em_2d_crystal_grow.details' ],
      [ '@comment', '   _em_2d_crystal_grow.number_2d_crystals' ],
      [ '@comment', '   _em_2d_crystal_grow.mean_2d_crystal_size' ],
      [ '@comment', '   _em_2d_crystal_grow.citation_id' ],
      [ '@comment', '   EM_BUFFER' ],
      [ '@comment', '   _em_buffer.id' ],
      [ '@comment', '   _em_buffer.name' ],
      [ '@comment', '   _em_buffer.details' ],
      [ '@comment', '   EM_BUFFER_COMPONENTS' ],
      [ '@comment', '   _em_buffer_components.id' ],
      [ '@comment', '   _em_buffer_components.buffer_id' ],
      [ '@comment', '   _em_buffer_components.name' ],
      [ '@comment', '   _em_buffer_components.volume' ],
      [ '@comment', '   _em_buffer_components.conc' ],
      [ '@comment', '   _em_buffer_components.details' ],
      [ '@comment', '   EM_VITRIFICATION' ],
      [ '@comment', '   _em_vitrification.entry_id' ],
      [ '@comment', '   _em_vitrification.id' ],
      [ '@comment', '   _em_vitrification.sample_preparation_id' ],
      [ '@comment', '   _em_vitrification.cryogen_name' ],
      [ '@comment', '   _em_vitrification.humidity' ],
      [ '@comment', '   _em_vitrification.temp' ],
      [ '@comment', '   _em_vitrification.instrument' ],
      [ '@comment', '   _em_vitrification.method' ],
      [ '@comment', '   _em_vitrification.time_resolved_state' ],
      [ '@comment', '   _em_vitrification.citation_id' ],
      [ '@comment', '   _em_vitrification.details' ],
      [ '@comment', '   EM_IMAGING' ],
      [ '@comment', '   _em_imaging.entry_id' ],
      [ '@comment', '   _em_imaging.id' ],
      [ '@comment', '   _em_imaging.sample_support_id' ],
      [ '@comment', '   _em_imaging.detector_id' ],
      [ '@comment', '   _em_imaging.scans_id' ],
      [ '@comment', '   _em_imaging.microscope_model' ],
      [ '@comment', '   _em_imaging.specimen_holder_type' ],
      [ '@comment', '   _em_imaging.specimen_holder_model' ],
      [ '@comment', '   _em_imaging.details' ],
      [ '@comment', '   _em_imaging.date' ],
      [ '@comment', '   _em_imaging.accelerating_voltage' ],
      [ '@comment', '   _em_imaging.illumination_mode' ],
      [ '@comment', '   _em_imaging.mode' ],
      [ '@comment', '   _em_imaging.nominal_cs' ],
      [ '@comment', '   _em_imaging.nominal_defocus_min' ],
      [ '@comment', '   _em_imaging.nominal_defocus_max' ],
      [ '@comment', '   _em_imaging.tilt_angle_min' ],
      [ '@comment', '   _em_imaging.tilt_angle_max' ],
      [ '@comment', '   _em_imaging.nominal_magnification' ],
      [ '@comment', '   _em_imaging.calibrated_magnification' ],
      [ '@comment', '   _em_imaging.electron_source' ],
      [ '@comment', '   _em_imaging.electron_dose' ],
      [ '@comment', '   _em_imaging.energy_filter' ],
      [ '@comment', '   _em_imaging.energy_window' ],
      [ '@comment', '   _em_imaging.citation_id' ],
      [ '@comment', '   _em_imaging.temperature' ],
      [ '@comment', '   _em_imaging.detector_distance' ],
      [ '@comment', '   _em_imaging.recording_temperature_minimum' ],
      [ '@comment', '   _em_imaging.recording_temperature_maximum' ],
      [ '@comment', '   EM_DETECTOR' ],
      [ '@comment', '   _em_detector.entry_id' ],
      [ '@comment', '   _em_detector.id' ],
      [ '@comment', '   _em_detector.details' ],
      [ '@comment', '   _em_detector.type' ],
      [ '@comment', '   _em_detector.detective_quantum_efficiency' ],
      [ '@comment', '   EM_IMAGE_SCANS' ],
      [ '@comment', '   _em_image_scans.entry_id' ],
      [ '@comment', '   _em_image_scans.id' ],
      [ '@comment', '   _em_image_scans.number_digital_images' ],
      [ '@comment', '   _em_image_scans.details' ],
      [ '@comment', '   _em_image_scans.scanner_model' ],
      [ '@comment', '   _em_image_scans.sampling_size' ],
      [ '@comment', '   _em_image_scans.od_range' ],
      [ '@comment', '   _em_image_scans.quant_bit_size' ],
      [ '@comment', '   _em_image_scans.citation_id' ],
      [ '@comment', '   EM_2D_PROJECTION_SELECTION' ],
      [ '@comment', '   _em_2d_projection_selection.entry_id' ],
      [ '@comment', '   _em_2d_projection_selection.num_particles' ],
      [ '@comment', '   _em_2d_projection_selection.software_name' ],
      [ '@comment', '   _em_2d_projection_selection.method' ],
      [ '@comment', '   _em_2d_projection_selection.details' ],
      [ '@comment', '   _em_2d_projection_selection.citation_id' ],
      [ '@comment', '   EM_EULER_ANGLE_DISTRIBUTION' ],
      [ '@comment', '   _em_euler_angle_distribution.id' ],
      [ '@comment', '   _em_euler_angle_distribution.entry_id' ],
      [ '@comment', '   _em_euler_angle_distribution.details' ],
      [ '@comment', '   _em_euler_angle_distribution.alpha' ],
      [ '@comment', '   _em_euler_angle_distribution.beta' ],
      [ '@comment', '   _em_euler_angle_distribution.gamma' ],
      [ '@comment', '   EM_3D_RECONSTRUCTION' ],
      [ '@comment', '   _em_3d_reconstruction.entry_id' ],
      [ '@comment', '   _em_3d_reconstruction.id' ],
      [ '@comment', '   _em_3d_reconstruction.method' ],
      [ '@comment', '   _em_3d_reconstruction.citation_id' ],
      [ '@comment', '   _em_3d_reconstruction.details' ],
      [ '@comment', '   _em_3d_reconstruction.resolution' ],
      [ '@comment', '   _em_3d_reconstruction.resolution_method' ],
      [ '@comment',
        '   _em_3d_reconstruction.magnification_calibration' ],
      [ '@comment', '   _em_3d_reconstruction.ctf_correction_method' ],
      [ '@comment', '   _em_3d_reconstruction.nominal_pixel_size' ],
      [ '@comment', '   _em_3d_reconstruction.actual_pixel_size' ],
      [ '@comment', '   EM_3D_FITTING' ],
      [ '@comment', '   _em_3d_fitting.id' ],
      [ '@comment', '   _em_3d_fitting.entry_id' ],
      [ '@comment', '   _em_3d_fitting.method' ],
      [ '@comment', '   _em_3d_fitting.target_criteria' ],
      [ '@comment', '   _em_3d_fitting.software_name' ],
      [ '@comment', '   _em_3d_fitting.details' ],
      [ '@comment', '   _em_3d_fitting.overall_b_value' ],
      [ '@comment', '   _em_3d_fitting.ref_space' ],
      [ '@comment', '   _em_3d_fitting.ref_protocol' ],
      [ '@comment', '   EM_3D_FITTING_LIST' ],
      [ '@comment', '   _em_3d_fitting_list.id' ],
      [ '@comment', '   _em_3d_fitting_list.3d_fitting_id' ],
      [ '@comment', '   _em_3d_fitting_list.pdb_entry_id' ],
      [ '@comment', '   _em_3d_fitting_list.pdb_chain_id' ],
      [ '@comment', '################' ],
      [ '@comment', '# EM_ASSEMBLY ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'em_assembly',
        [ [ 'item_',
            '_category.description',
            '   Data items in the EM_ASSEMBLY category record details\n    about the type of complex assembly that describes the \n    nature of the sample studied.' ],
          [ 'item_', '_category.id', 'em_assembly' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_assembly.id', '_em_assembly.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ ' Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              ' _em_assembly.id                   1 \n  _em_assembly.entry_id             1DYL\n  _em_assembly.name                 virus\n  _em_assembly.aggregation_state    icosahedral\n  _em_assembly.composition          virus\n  _em_assembly.num_components       1  \n  _em_assembly.mol_wt_exp           .\n  _em_assembly.mol_wt_theo          .\n  _em_assembly.mol_wt_method        .\n  _em_assembly.details              .' ] ] ] ],
      [ 'save_',
        '_em_assembly.id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_assembly.id must uniquely identify\n     a collection of observed complexes.' ],
          [ 'item_', '_item.name', '_em_assembly.id' ],
          [ 'item_', '_item.category_id', 'em_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_entity_assembly.assembly_id' ],
          [ 'item_', '_item_linked.parent_name', '_em_assembly.id' ] ] ],
      [ 'save_',
        '_em_assembly.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_em_assembly.entry_id' ],
          [ 'item_', '_item.category_id', 'em_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.child_name', '_em_assembly.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_assembly.name',
        [ [ 'item_',
            '_item_description.description',
            '    The name of the assembly of observed complexes.' ],
          [ 'item_', '_item.name', '_em_assembly.name' ],
          [ 'item_', '_item.category_id', 'em_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_em_assembly.aggregation_state',
        [ [ 'item_',
            '_item_description.description',
            '    A description of the aggregation state of the assembly.' ],
          [ 'item_', '_item.name', '_em_assembly.aggregation_state' ],
          [ 'item_', '_item.category_id', 'em_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'SINGLE PARTICLE',
              'INDIVIDUAL STRUCTURE',
              '2D-CRYSTAL',
              'ICOSAHEDRAL',
              'HELICAL',
              'FILAMENT',
              'HELICAL FILAMENTS',
              'TISSUE' ] ] ] ],
      [ 'save_',
        '_em_assembly.composition',
        [ [ 'item_',
            '_item_description.description',
            '    The known composition of the assembly.' ],
          [ 'item_', '_item.name', '_em_assembly.composition' ],
          [ 'item_', '_item.category_id', 'em_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_em_assembly.num_components',
        [ [ 'item_',
            '_item_description.description',
            '    The number of components of the biological assembly.' ],
          [ 'item_', '_item.name', '_em_assembly.num_components' ],
          [ 'item_', '_item.category_id', 'em_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_em_assembly.mol_wt_exp',
        [ [ 'item_',
            '_item_description.description',
            '    The value (in megadaltons) of the experimentally\n     determined molecular weight of the assembly.' ],
          [ 'item_', '_item.name', '_em_assembly.mol_wt_exp' ],
          [ 'item_', '_item.category_id', 'em_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'megadaltons' ] ] ],
      [ 'save_',
        '_em_assembly.mol_wt_theo',
        [ [ 'item_',
            '_item_description.description',
            '    The value (in megadaltons) of the theoretically\n     determined molecular weight of the assembly.' ],
          [ 'item_', '_item.name', '_em_assembly.mol_wt_theo' ],
          [ 'item_', '_item.category_id', 'em_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'megadaltons' ] ] ],
      [ 'save_',
        '_em_assembly.mol_wt_method',
        [ [ 'item_',
            '_item_description.description',
            '    The method used in determining\n     the molecular weight.' ],
          [ 'item_', '_item.name', '_em_assembly.mol_wt_method' ],
          [ 'item_', '_item.category_id', 'em_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_assembly.details',
        [ [ 'item_',
            '_item_description.description',
            '    A description of any additional details \n     describing the observed sample.' ],
          [ 'item_', '_item.name', '_em_assembly.details' ],
          [ 'item_', '_item.category_id', 'em_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ ' This structure was preferentially oriented (end-on)on the grid.',
              ' The structure was monodisperse.' ] ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# EM_ENTITY_ASSEMBLY ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'em_entity_assembly',
        [ [ 'item_',
            '_category.description',
            '    Data items in the EM_ENTITY_ASSEMBLY category\n     record details about each component of\n     the complex.' ],
          [ 'item_', '_category.id', 'em_entity_assembly' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_entity_assembly.id', '_em_entity_assembly.assembly_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ ' Example 1 - based on PDB entry 1DYL and laboratory records for the\n              structure corresponding to PDB entry 1DYL',
              ' _em_entity_assembly.id                  1\n  _em_entity_assembly.assembly_id         1\n  _em_entity_assembly.type                VIRUS' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_entity_assembly.id must uniquely identify\n     each of the components of the complex.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.id' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_virus_entity.entity_assembly_id' ],
          [ 'item_', '_item_linked.parent_name', '_em_entity_assembly.id' ] ] ],
      [ 'save_',
        '_em_entity_assembly.assembly_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to _em_assembly.id in the\n     ASSEMBLY category.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.assembly_id' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_entity_assembly.type',
        [ [ 'item_',
            '_item_description.description',
            '    A description of types of components of the\n     assembly of the biological structure.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.type' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'VIRUS',
              'PROTEIN',
              'NUCLEIC ACID',
              'LIGAND',
              'LABEL',
              'CELLULAR COMPONENT' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.name',
        [ [ 'item_',
            '_item_description.description',
            '   The name of the component of the observed assembly.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.name' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_entity_assembly.details',
        [ [ 'item_',
            '_item_description.description',
            '   Additional details about the component.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.details' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_entity_assembly.ebi_organism_scientific',
        [ [ 'item_',
            '_item_description.description',
            '   The species of the natural organism from which the component\n    was obtained.' ],
          [ 'item_',
            '_item.name',
            '_em_entity_assembly.ebi_organism_scientific' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_entity_assembly.ebi_organism_common',
        [ [ 'item_',
            '_item_description.description',
            '   The common name of the species of the natural organism from which \n    the component was obtained.' ],
          [ 'item_',
            '_item.name',
            '_em_entity_assembly.ebi_organism_common' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_entity_assembly.ebi_strain',
        [ [ 'item_',
            '_item_description.description',
            '    The strain of the natural organism from which the component was\n      obtained, if relevant.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.ebi_strain' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'DH5a', 'BMH 71-18' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.ebi_tissue',
        [ [ 'item_',
            '_item_description.description',
            '  The tissue of the natural organism from which the component was\n   obtained.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.ebi_tissue' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'heart', 'liver', 'eye lens' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.ebi_cell',
        [ [ 'item_',
            '_item_description.description',
            '              The cell from which the component was\n               obtained.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.ebi_cell' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'CHO', 'HELA', '3T3' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.ebi_organelle',
        [ [ 'item_',
            '_item_description.description',
            '   The organelle from which the component was\n    obtained.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.ebi_organelle' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'golgi', 'mitochondrion', 'cytoskeleton' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.ebi_cellular_location',
        [ [ 'item_',
            '_item_description.description',
            '   The cellular location of the component.' ],
          [ 'item_',
            '_item.name',
            '_em_entity_assembly.ebi_cellular_location' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'cytoplasm', 'endoplasmic reticulum', 'plasma membrane' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.ebi_engineered',
        [ [ 'item_',
            '_item_description.description',
            '    A flag to indicate whether the component is engineered.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.ebi_engineered' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_enumeration.value' ], [ 'YES', 'NO' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.ebi_expression_system',
        [ [ 'item_',
            '_item_description.description',
            '   The expression system used to produce the component.' ],
          [ 'item_',
            '_item.name',
            '_em_entity_assembly.ebi_expression_system' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'eschericia coli', 'saccharomyces cerevisiae' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.ebi_expression_system_plasmid',
        [ [ 'item_',
            '_item_description.description',
            '   The plasmid used in the expression system used to produce the component.' ],
          [ 'item_',
            '_item.name',
            '_em_entity_assembly.ebi_expression_system_plasmid' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'pBR322', 'pMB9' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.go_id',
        [ [ 'item_',
            '_item_description.description',
            ' The Gene Ontology (GO) identifier for the component.\n  The GO id is the appropriate identifier used by the Gene Ontology\n  Consortium. Reference: Nature Genetics vol 25:25-29 (2000).' ],
          [ 'item_', '_item.name', '_em_entity_assembly.go_id' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'GO:0005876', 'GO:0015630' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.ipr_id',
        [ [ 'item_',
            '_item_description.description',
            '  The InterPro (IPR) identifier for the component.\n   The IPR id is the appropriate identifier used by the Interpro Resource.\n   Reference: Nucleic Acid Research vol 29(1):37-40(2001).' ],
          [ 'item_', '_item.name', '_em_entity_assembly.ipr_id' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '001304', '002353' ] ] ] ],
      [ 'save_',
        '_em_entity_assembly.synonym',
        [ [ 'item_',
            '_item_description.description',
            '             Alternative name of the component.' ],
          [ 'item_', '_item.name', '_em_entity_assembly.synonym' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'FADV-1' ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', '# EM_ENTITY_ASSEMBLY_LIST ##' ],
      [ '@comment', '############################' ],
      [ 'save_',
        'em_entity_assembly_list',
        [ [ 'item_',
            '_category.description',
            '    Data items in the  EM_ENTITY_ASSEMBLY_LIST category record details\n     of the structural elements in each component.' ],
          [ 'item_', '_category.id', 'em_entity_assembly_list' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_entity_assembly_list.entity_assembly_id',
              '_em_entity_assembly_list.id',
              '_em_entity_assembly_list.entity_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - microtubule',
              ' \n   loop_\n   _em_entity_assembly_list.entity_assembly_id\n   _em_entity_assembly_list.id\n   _em_entity_assembly_list.entity_id\n   _em_entity_assembly_list.oligomeric_details\n   _em_entity_assembly_list.number_of_copies\n    1 1 1  \'DIMER\' 2',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_em_entity_assembly_list.id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_entity_assembly_list.id must uniquely identify\n     the component.' ],
          [ 'item_', '_item.name', '_em_entity_assembly_list.id' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly_list' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_entity_assembly_list.entity_id',
        [ [ 'item_',
            '_item_description.description',
            '    A pointer to entity id.' ],
          [ 'item_', '_item.name', '_em_entity_assembly_list.entity_id' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly_list' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_entity_assembly_list.entity_id' ],
          [ 'item_', '_item_linked.parent_name', '_entity.id' ] ] ],
      [ 'save_',
        '_em_entity_assembly_list.entity_assembly_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to _em_entity_assembly.id in\n     the ENTITY_ASSEMBLY category.' ],
          [ 'item_',
            '_item.name',
            '_em_entity_assembly_list.entity_assembly_id' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly_list' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_entity_assembly_list.oligomeric_details',
        [ [ 'item_',
            '_item_description.description',
            '    The oligomeric state of the entity.' ],
          [ 'item_',
            '_item.name',
            '_em_entity_assembly_list.oligomeric_details' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly_list' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_em_entity_assembly_list.number_of_copies',
        [ [ 'item_',
            '_item_description.description',
            '    The number of copies of the entity.' ],
          [ 'item_',
            '_item.name',
            '_em_entity_assembly_list.number_of_copies' ],
          [ 'item_', '_item.category_id', 'em_entity_assembly_list' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# EM_VIRUS_ENTITY ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'em_virus_entity',
        [ [ 'item_',
            '_category.description',
            '   Data items in the EM_VIRUS_ENTITY category record details\n    of the icosahedral virus.' ],
          [ 'item_', '_category.id', 'em_virus_entity' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_virus_entity.id', '_em_virus_entity.entity_assembly_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ ' Example 1 - based on PDB entry 1DYL and laboratory records for the\n             structure corresponding to PDB entry 1DYL',
              '  loop_\n   _em_virus_entity.id\n   _em_virus_entity.virus_host_category\n   _em_virus_entity.virus_host_species\n   _em_virus_entity.virus_type\n   _em_virus_entity.virus_isolate\n   _em_virus_entity.ictvdb_id\n   _em_virus_entity.entity_assembly_id\n   _em_virus_entity.enveloped\n   _em_virus_entity.empty\n   1  \'VERTERBRATES\' \'HOMO SAPIENS\' \'VIRUS\' \'STRAIN\'  \'00.073.0.01.023\' 1 \'YES\' \'NO\'' ] ] ] ],
      [ 'save_',
        '_em_virus_entity.id',
        [ [ 'item_',
            '_item_description.description',
            '   Is the unique identifier for VIRUS_ENTITY category.' ],
          [ 'item_', '_item.name', '_em_virus_entity.id' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_virus_entity.virus_host_category',
        [ [ 'item_',
            '_item_description.description',
            '   The host category description for the virus.' ],
          [ 'item_', '_item.name', '_em_virus_entity.virus_host_category' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'ALGAE',
              'ARCHAEA',
              'BACTERIA(EUBACTERIA)',
              'FUNGI',
              'INVERTEBRATES',
              'PLANTAE (HIGHER PLANTS)',
              'PROTOZOA',
              'VERTEBRATES' ] ] ] ],
      [ 'save_',
        '_em_virus_entity.virus_host_species',
        [ [ 'item_',
            '_item_description.description',
            '  The host species from which the virus was isolated.' ],
          [ 'item_', '_item.name', '_em_virus_entity.virus_host_species' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'homo sapiens', 'gallus gallus' ] ] ] ],
      [ 'save_',
        '_em_virus_entity.virus_host_growth_cell',
        [ [ 'item_',
            '_item_description.description',
            '   The host cell from which the virus was isolated.' ],
          [ 'item_',
            '_item.name',
            '_em_virus_entity.virus_host_growth_cell' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'HELA', 'CHO' ] ] ] ],
      [ 'save_',
        '_em_virus_entity.virus_type',
        [ [ 'item_',
            '_item_description.description',
            '   The type of virus.' ],
          [ 'item_', '_item.name', '_em_virus_entity.virus_type' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'VIRION', 'SATELLITE', 'PRION', 'VIROID', 'VIRUS-LIKE PARTICLE' ] ] ] ],
      [ 'save_',
        '_em_virus_entity.virus_isolate',
        [ [ 'item_',
            '_item_description.description',
            '   The isolate from which the virus was obtained.' ],
          [ 'item_', '_item.name', '_em_virus_entity.virus_isolate' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'STRAIN', 'SEROTYPE', 'SEROCOMPLEX', 'SUBSPECIES', 'SPECIES' ] ] ] ],
      [ 'save_',
        '_em_virus_entity.ictvdb_id',
        [ [ 'item_',
            '_item_description.description',
            '     The International Committee on Taxonomy of Viruses\n      (ICTV) Taxon Identifier is the Virus Code used throughout the\n      ICTV database (ICTVdb). The ICTVdb id is the appropriate\n      identifier used by the International Committee on Taxonomy of Viruses\n      Resource. Reference: Virus Taxonomy, Academic Press (1999).\n      ISBN:0123702003.' ],
          [ 'item_', '_item.name', '_em_virus_entity.ictvdb_id' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ '01.0.2.0.001', '01.0.2.0.002' ] ] ] ],
      [ 'save_',
        '_em_virus_entity.entity_assembly_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to _em_virus_entity.id in the \n     ENTITY_ASSEMBLY category.' ],
          [ 'item_', '_item.name', '_em_virus_entity.entity_assembly_id' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_virus_entity.enveloped',
        [ [ 'item_',
            '_item_description.description',
            '    Flag to indicate if the virus is enveloped or not.' ],
          [ 'item_', '_item.name', '_em_virus_entity.enveloped' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_enumeration.value' ], [ 'YES', 'NO' ] ] ] ],
      [ 'save_',
        '_em_virus_entity.empty',
        [ [ 'item_',
            '_item_description.description',
            '  Flag to indicate if the virus is empty or not.' ],
          [ 'item_', '_item.name', '_em_virus_entity.empty' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_enumeration.value' ], [ 'YES', 'NO' ] ] ] ],
      [ 'save_',
        '_em_virus_entity.details',
        [ [ 'item_',
            '_item_description.description',
            '    Additional details about this virus entity' ],
          [ 'item_', '_item.name', '_em_virus_entity.details' ],
          [ 'item_', '_item.category_id', 'em_virus_entity' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# EM_ICOS_VIRUS_SHELLS ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'em_icos_virus_shells',
        [ [ 'item_',
            '_category.description',
            '   Data items in the EM_ICOS_VIRUS_SHELLS category record details\n    of the viral shell number, diameter of each shell and triangulation number.' ],
          [ 'item_', '_category.id', 'em_icos_virus_shells' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_icos_virus_shells.virus_entity_id',
              '_em_icos_virus_shells.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ ' Example 1 - based on PDB entry 1DYL and laboratory records for the\n             structure corresponding to PDB entry 1DYL',
              '  loop_\n   _em_icos_virus_shells.virus_entity_id\n   _em_icos_virus_shells.id\n   _em_icos_virus_shells.shell_diameter\n   _em_icos_virus_shells.triangulation_num\n     1 1 400 4' ] ] ] ],
      [ 'save_',
        '_em_icos_virus_shells.virus_entity_id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_icos_virus_shells.virus_entity_id is\n     a pointer to _em_virus_entity.id in the VIRUS_ENTITY\n     category.' ],
          [ 'item_',
            '_item.name',
            '_em_icos_virus_shells.virus_entity_id' ],
          [ 'item_', '_item.category_id', 'em_icos_virus_shells' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_icos_virus_shells.virus_entity_id' ],
          [ 'item_', '_item_linked.parent_name', '_em_virus_entity.id' ] ] ],
      [ 'save_',
        '_em_icos_virus_shells.id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_em_icos_virus_shells.id must uniquely identify\n     the number and diameter of each virus protein shell and its\n     triangulation number.' ],
          [ 'item_', '_item.name', '_em_icos_virus_shells.id' ],
          [ 'item_', '_item.category_id', 'em_icos_virus_shells' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_icos_virus_shells.shell_diameter',
        [ [ 'item_',
            '_item_description.description',
            '   The value of the diameter (in angstroms) for each\n    protein shell of the virus.     ' ],
          [ 'item_', '_item.name', '_em_icos_virus_shells.shell_diameter' ],
          [ 'item_', '_item.category_id', 'em_icos_virus_shells' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_em_icos_virus_shells.triangulation_num',
        [ [ 'item_',
            '_item_description.description',
            '   The triangulation number (T number) is a geometric and abstract\n    concept that does not correspond to the structural components of\n    an individul virus.\n    It refers to the organisation of the geometric figure.\n    The triangulation number, T is given by the following relationship:\n    T= h*2 + hk +k*2,  where h and k are positive integers which define the \n    position of the five-fold vertex on the original hexagonal net.' ],
          [ 'item_',
            '_item.name',
            '_em_icos_virus_shells.triangulation_num' ],
          [ 'item_', '_item.category_id', 'em_icos_virus_shells' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '4' ] ] ],
      [ '@comment', '##########################' ],
      [ '@comment', '# EM_SAMPLE_PREPARATION ##' ],
      [ '@comment', '##########################' ],
      [ 'save_',
        'em_sample_preparation',
        [ [ 'item_',
            '_category.description',
            '   Data items in the EM_SAMPLE_PREPARATION category\n    record details of sample conditions prior to loading\n    onto grid support.' ],
          [ 'item_', '_category.id', 'em_sample_preparation' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_sample_preparation.id',
              '_em_sample_preparation.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              ' \n  _em_sample_preparation.entry_id                   1DYL\n  _em_sample_preparation.id                         1\n  _em_sample_preparation.ph                         7.6\n  _em_sample_preparation.buffer_id                  1\n  _em_sample_preparation.sample_concentration       5\n  _em_sample_preparation.2d_crystal_grow_id         .\n  _em_sample_preparation.support.id                 1' ] ] ] ],
      [ 'save_',
        '_em_sample_preparation.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '  This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_em_sample_preparation.entry_id' ],
          [ 'item_', '_item.category_id', 'em_sample_preparation' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_sample_preparation.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_sample_preparation.id',
        [ [ 'item_',
            '_item_description.description',
            '   The value of _em_sample_preparation.id must\n    uniquely identify the sample preparation.' ],
          [ 'item_', '_item.name', '_em_sample_preparation.id' ],
          [ 'item_', '_item.category_id', 'em_sample_preparation' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_sample_preparation.ph',
        [ [ 'item_',
            '_item_description.description',
            '   The pH value of the observed sample buffer.' ],
          [ 'item_', '_item.name', '_em_sample_preparation.ph' ],
          [ 'item_', '_item.category_id', 'em_sample_preparation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_em_sample_preparation.buffer_id',
        [ [ 'item_',
            '_item_description.description',
            '   This data item is a pointer to _em_buffer.id in the\n    BUFFER category.' ],
          [ 'item_', '_item.name', '_em_sample_preparation.buffer_id' ],
          [ 'item_', '_item.category_id', 'em_sample_preparation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_sample_preparation.sample_concentration',
        [ [ 'item_',
            '_item_description.description',
            '   The value of the concentration (mg/mL for mg per milliliter)\n    of the complex in the sample.' ],
          [ 'item_',
            '_item.name',
            '_em_sample_preparation.sample_concentration' ],
          [ 'item_', '_item.category_id', 'em_sample_preparation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'mg_per_ml' ] ] ],
      [ 'save_',
        '_em_sample_preparation.2d_crystal_grow_id',
        [ [ 'item_',
            '_item_description.description',
            '  This data item is a pointer to _em_2d_crystal_grow.id\n   in the 2D_CRYSTAL_GROW category.' ],
          [ 'item_',
            '_item.name',
            '_em_sample_preparation.2d_crystal_grow_id' ],
          [ 'item_', '_item.category_id', 'em_sample_preparation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_sample_preparation.support_id',
        [ [ 'item_',
            '_item_description.description',
            '   This data item is a pointer to _em_sample_support.id\n    in the EM_SAMPLE_SUPPORT category.' ],
          [ 'item_', '_item.name', '_em_sample_preparation.support_id' ],
          [ 'item_', '_item.category_id', 'em_sample_preparation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ '@comment', '######################' ],
      [ '@comment', '# EM_SAMPLE_SUPPORT ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'em_sample_support',
        [ [ 'item_',
            '_category.description',
            '   Data items in the EM_SAMPLE_SUPPORT category record details\n    of the electron microscope grid type, grid support film and pretreatment\n    of whole before sample is applied' ],
          [ 'item_', '_category.id', 'em_sample_support' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_em_sample_support.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              ' \n   _em_sample_support.id               1\n   _em_sample_support.film_material    \'HOLEY CARBON\'\n   _em_sample_support.method           .\n   _em_sample_support.grid_material    COPPER\n   _em_sample_support.grid_mesh_size   400\n   _em_sample_support.grid_type        MESH\n   _em_sample_support.pretreatment     \'GLOW DISCHARGE\'\n   _em_sample_support.details          .\n   _em_sample_support.citation_id      2',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_em_sample_support.id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_sample_support.id must uniquely identify\n     the sample support.' ],
          [ 'item_', '_item.name', '_em_sample_support.id' ],
          [ 'item_', '_item.category_id', 'em_sample_support' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_em_imaging.sample_support_id',
              '_em_sample_support.id',
              '_em_sample_preparation.support_id',
              '_em_sample_support.id' ] ] ] ],
      [ 'save_',
        '_em_sample_support.film_material',
        [ [ 'item_',
            '_item_description.description',
            '   The support material covering the em grid.' ],
          [ 'item_', '_item.name', '_em_sample_support.film_material' ],
          [ 'item_', '_item.category_id', 'em_sample_support' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'CARBON',
              'FORMVAR PLUS CARBON',
              'CELLULOSE ACETATE PLUS CARBON',
              'PARLODION PLUS CARBON',
              'HOLEY CARBON' ] ] ] ],
      [ 'save_',
        '_em_sample_support.method',
        [ [ 'item_',
            '_item_description.description',
            '   A description of the method used to produce the support film.' ],
          [ 'item_', '_item.name', '_em_sample_support.method' ],
          [ 'item_', '_item.category_id', 'em_sample_support' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '1%formvar in chloroform cast on distilled water' ] ] ],
      [ 'save_',
        '_em_sample_support.grid_material',
        [ [ 'item_',
            '_item_description.description',
            '   The name of the material from which the grid is made.' ],
          [ 'item_', '_item.name', '_em_sample_support.grid_material' ],
          [ 'item_', '_item.category_id', 'em_sample_support' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'COPPER',
              'COPPER/PALLADIUM',
              'COPPER/RHODIUM',
              'GOLD',
              'NICKEL',
              'PLATINUM',
              'TUNGSTEN',
              'TITANIUM',
              'MOLYBDENUM' ] ] ] ],
      [ 'save_',
        '_em_sample_support.grid_mesh_size',
        [ [ 'item_',
            '_item_description.description',
            '   The value of the mesh size (per inch) of the em grid.' ],
          [ 'item_', '_item.name', '_em_sample_support.grid_mesh_size' ],
          [ 'item_', '_item.category_id', 'em_sample_support' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '400' ] ] ],
      [ 'save_',
        '_em_sample_support.grid_type',
        [ [ 'item_',
            '_item_description.description',
            '   A description of the grid type.' ],
          [ 'item_', '_item.name', '_em_sample_support.grid_type' ],
          [ 'item_', '_item.category_id', 'em_sample_support' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'SLOT', 'APERTURE', 'DIAMOND', 'HEXAGONAL' ] ] ] ],
      [ 'save_',
        '_em_sample_support.pretreatment',
        [ [ 'item_',
            '_item_description.description',
            '   A description of the grid plus support film pretreatment. ' ],
          [ 'item_', '_item.name', '_em_sample_support.pretreatment' ],
          [ 'item_', '_item.category_id', 'em_sample_support' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'glow-discharged for 30 sec in argon' ] ] ],
      [ 'save_',
        '_em_sample_support.details',
        [ [ 'item_',
            '_item_description.description',
            '   A description of any additional details concerning the sample support. ' ],
          [ 'item_', '_item.name', '_em_sample_support.details' ],
          [ 'item_', '_item.category_id', 'em_sample_support' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'This grid plus sample was kept at -170 deg C for a month before use' ] ] ],
      [ 'save_',
        '_em_sample_support.citation_id',
        [ [ 'item_',
            '_item_description.description',
            '   This data item is a pointer to _citation.id\n    in the CITATION category.' ],
          [ 'item_', '_item.name', '_em_sample_support.citation_id' ],
          [ 'item_', '_item.category_id', 'em_sample_support' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_sample_support.citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# EM_2D_CRYSTAL_GROW ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'em_2d_crystal_grow',
        [ [ 'item_',
            '_category.description',
            '   Data items in the EM_2D_CRYSTAL_GROW category\n    record details of growth conditions for 2d crystal samples.' ],
          [ 'item_', '_category.id', 'em_2d_crystal_grow' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_em_2d_crystal_grow.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - based on PDB entry 1AT9 and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              ' _em_2d_crystal_grow.id              1\n  _em_2d_crystal_grow.method          .\n  _em_2d_crystal_grow.apparatus       .\n  _em_2d_crystal_grow.atmosphere      \'room air\'\n  _em_2d_crystal_grow.pH              5.2\n  _em_2d_crystal_grow.temp            18\n  _em_2d_crystal_grow.time            .\n  _em_2d_crystal_grow.buffer_id       2\n  _em_2d_crystal_grow.details         \'on grid\'\n  _em_2d_crystal_grow.number_2d_crystals    129\n  _em_2d_crystal_grow.mean_2d_crystal_size  .\n  _em_2d_crystal_grow.citation_id       2',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.id',
        [ [ 'item_',
            '_item_description.description',
            '  The value of _em_2d_crystal_grow.crystal_id\n   must uniquely identify the sample 2d crystal.' ],
          [ 'item_', '_item.name', '_em_2d_crystal_grow.id' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.method',
        [ [ 'item_',
            '_item_description.description',
            '   The method used for growing the crystals. ' ],
          [ 'item_', '_item.name', '_em_2d_crystal_grow.method' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'lipid monolayer' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.apparatus',
        [ [ 'item_',
            '_item_description.description',
            '  The type of the apparatus used for growing the crystals. ' ],
          [ 'item_', '_item.name', '_em_2d_crystal_grow.apparatus' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'Langmuir trough' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.atmosphere',
        [ [ 'item_',
            '_item_description.description',
            '  The type of atmosphere in which crystals were grown. ' ],
          [ 'item_', '_item.name', '_em_2d_crystal_grow.atmosphere' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'room air' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.pH',
        [ [ 'item_',
            '_item_description.description',
            '   the pH value used for growing the crystals. ' ],
          [ 'item_', '_item.name', '_em_2d_crystal_grow.pH' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '4.7' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.temp',
        [ [ 'item_',
            '_item_description.description',
            '  The value of the temperature in degrees Kelvin used for\n   growing the crystals. ' ],
          [ 'item_', '_item.name', '_em_2d_crystal_grow.temp' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_', '_item_examples.case', '293' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.time',
        [ [ 'item_',
            '_item_description.description',
            '   The length of time required to grow the crystals. ' ],
          [ 'item_', '_item.name', '_em_2d_crystal_grow.time' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'approximately 2 days' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.buffer_id',
        [ [ 'item_',
            '_item_description.description',
            '   This data item is a pointer to _em_buffer.id in the\n    BUFFER category.' ],
          [ 'item_', '_item.name', '_em_2d_crystal_grow.buffer_id' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.details',
        [ [ 'item_',
            '_item_description.description',
            '   Any additional items concerning 2d crystal growth. ' ],
          [ 'item_', '_item.name', '_em_2d_crystal_grow.details' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.number_2d_crystals',
        [ [ 'item_',
            '_item_description.description',
            ' The number of 2d crystals imaged. ' ],
          [ 'item_',
            '_item.name',
            '_em_2d_crystal_grow.number_2d_crystals' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.mean_2d_crystal_size',
        [ [ 'item_',
            '_item_description.description',
            ' The approximate size (microns squared) of 2d crystals imaged. ' ],
          [ 'item_',
            '_item.name',
            '_em_2d_crystal_grow.mean_2d_crystal_size' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'microns_squared' ] ] ],
      [ 'save_',
        '_em_2d_crystal_grow.citation_id',
        [ [ 'item_',
            '_item_description.description',
            '   This data item is a pointer to _citation.id\n    in the CITATION category.' ],
          [ 'item_', '_item.name', '_em_2d_crystal_grow.citation_id' ],
          [ 'item_', '_item.category_id', 'em_2d_crystal_grow' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_2d_crystal_grow.citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ '@comment', '##############' ],
      [ '@comment', '# EM_BUFFER ##' ],
      [ '@comment', '##############' ],
      [ 'save_',
        'em_buffer',
        [ [ 'item_',
            '_category.description',
            '   Data items in the BUFFER category\n    record details of the sample buffer. ' ],
          [ 'item_', '_category.id', 'em_buffer' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_em_buffer.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ] ] ],
      [ 'save_',
        '_em_buffer.id',
        [ [ 'item_',
            '_item_description.description',
            '  The value of _em_buffer.id must\n   uniquely identify the sample buffer.' ],
          [ 'item_', '_item.name', '_em_buffer.id' ],
          [ 'item_', '_item.category_id', 'em_buffer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_em_2d_crystal_grow.buffer_id',
              '_em_buffer.id',
              '_em_sample_preparation.buffer_id',
              '_em_buffer.id',
              '_em_buffer_components.buffer_id',
              '_em_buffer.id' ] ] ] ],
      [ 'save_',
        '_em_buffer.name',
        [ [ 'item_',
            '_item_description.description',
            '  The name of the buffer. ' ],
          [ 'item_', '_item.name', '_em_buffer.name' ],
          [ 'item_', '_item.category_id', 'em_buffer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'Acetic acid' ] ] ],
      [ 'save_',
        '_em_buffer.details',
        [ [ 'item_',
            '_item_description.description',
            '   Any additional details to do with buffer. ' ],
          [ 'item_', '_item.name', '_em_buffer.details' ],
          [ 'item_', '_item.category_id', 'em_buffer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'aerated' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# EM_BUFFER_COMPONENTS ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'em_buffer_components',
        [ [ 'item_',
            '_category.description',
            '  Constituents of buffer in sample ' ],
          [ 'item_', '_category.id', 'em_buffer_components' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_buffer_components.buffer_id',
              '_em_buffer_components.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '   Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              'loop_\n  _em_buffer_components.buffer_id\n  _em_buffer_components.id\n  _em_buffer_components.name\n  _em_buffer_components.volume\n  _em_buffer_components.conc\n  _em_buffer_components.details\n  1    1 \'NaCl\'            \'0.200 \'  \'4 \'     .\n  1    2 \'Acetic Acid\'     \'0.047 \'  \'100\'    .\n  1    3 \'water\'           \'0.700 \'  \'neat\'   .' ] ] ] ],
      [ 'save_',
        '_em_buffer_components.id',
        [ [ 'item_',
            '_item_description.description',
            '   The value of _em_buffer_components.id must \n    uniquely identify a component of the buffer.' ],
          [ 'item_', '_item.name', '_em_buffer_components.id' ],
          [ 'item_', '_item.category_id', 'em_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_buffer_components.buffer_id',
        [ [ 'item_',
            '_item_description.description',
            '  This data item is a pointer to _em_buffer.id in the BUFFER category.' ],
          [ 'item_', '_item.name', '_em_buffer_components.buffer_id' ],
          [ 'item_', '_item.category_id', 'em_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_buffer_components.name',
        [ [ 'item_',
            '_item_description.description',
            '   The name of each buffer component. ' ],
          [ 'item_', '_item.name', '_em_buffer_components.name' ],
          [ 'item_', '_item.category_id', 'em_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'Acetic acid' ] ] ],
      [ 'save_',
        '_em_buffer_components.volume',
        [ [ 'item_',
            '_item_description.description',
            '  The volume of buffer component. ' ],
          [ 'item_', '_item.name', '_em_buffer_components.volume' ],
          [ 'item_', '_item.category_id', 'em_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '0.200' ] ] ],
      [ 'save_',
        '_em_buffer_components.conc',
        [ [ 'item_',
            '_item_description.description',
            '  The millimolar concentration of buffer component. ' ],
          [ 'item_', '_item.name', '_em_buffer_components.conc' ],
          [ 'item_', '_item.category_id', 'em_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_examples.case', '200' ] ] ],
      [ 'save_',
        '_em_buffer_components.details',
        [ [ 'item_',
            '_item_description.description',
            '   Any additional details to do with buffer composition. ' ],
          [ 'item_', '_item.name', '_em_buffer_components.details' ],
          [ 'item_', '_item.category_id', 'em_buffer_components' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'pH adjusted with NaOH' ] ] ],
      [ '@comment', '#####################' ],
      [ '@comment', '# EM_VITRIFICATION ##' ],
      [ '@comment', '#####################' ],
      [ 'save_',
        'em_vitrification',
        [ [ 'item_',
            '_category.description',
            ' Data items in the EM_VITRIFICATION category\n  record details about the method and cryogen used in\n  rapid freezing of the sample on the grid prior to its\n  insertion in the electron microscope' ],
          [ 'item_', '_category.id', 'em_vitrification' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_vitrification.id', '_em_vitrification.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ ' Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              ' \n  _em_vitrification.entry_id               1DYL\n  _em_vitrification.id                     1\n  _em_vitrification.sample_preparation_id  1\n  _em_vitrification.cryogen_name           "ETHANE"\n  _em_vitrification.humidity               90\n  _em_vitrification.temp                   95\n  _em_vitrification.instrument             .\n  _em_vitrification.method                 "PLUNGE VITRIFICATION"\n  _em_vitrification.time_resolved_state    .\n  _em_vitrification.citation_id            1\n  _em_vitrification.details                \n  ; SAMPLES WERE PREPARED AS THIN             \n    LAYERS OF VITREOUS ICE AND                \n    MAINTAINED AT NEAR LIQUID NITROGEN        \n    TEMPERATURE IN THE ELECTRON MICROSCOPE    \n    WITH A GATAN 626-0300 CRYOTRANSFER        \n    HOLDER.                            \n  ;' ] ] ] ],
      [ 'save_',
        '_em_vitrification.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '  This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_em_vitrification.entry_id' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_vitrification.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_vitrification.id',
        [ [ 'item_',
            '_item_description.description',
            '  The value of _em_vitrification.id must uniquely identify\n   the vitrification procedure. ' ],
          [ 'item_', '_item.name', '_em_vitrification.id' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_vitrification.sample_preparation_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to _em_sample_preparation.id in the\n  EM_SAMPLE_PREPARATION category.' ],
          [ 'item_',
            '_item.name',
            '_em_vitrification.sample_preparation_id' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_vitrification.sample_preparation_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_em_sample_preparation.id' ] ] ],
      [ 'save_',
        '_em_vitrification.cryogen_name',
        [ [ 'item_',
            '_item_description.description',
            ' This is the name of the cryogen.' ],
          [ 'item_', '_item.name', '_em_vitrification.cryogen_name' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'HELIUM',
              'NITROGEN',
              'PROPANE',
              'ETHANE',
              'METHANE',
              'FREON 22',
              'FREON 12' ] ] ] ],
      [ 'save_',
        '_em_vitrification.humidity',
        [ [ 'item_',
            '_item_description.description',
            ' The humidity (%) in the vicinity of the vitrification process. ' ],
          [ 'item_', '_item.name', '_em_vitrification.humidity' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', '90' ] ] ],
      [ 'save_',
        '_em_vitrification.temp',
        [ [ 'item_',
            '_item_description.description',
            '  The temperature (in degrees Kelvin) at which vitrification took place. ' ],
          [ 'item_', '_item.name', '_em_vitrification.temp' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ],
          [ 'item_', '_item_examples.case', '4.2' ] ] ],
      [ 'save_',
        '_em_vitrification.instrument',
        [ [ 'item_',
            '_item_description.description',
            ' The type of instrument used in the vitrification process. ' ],
          [ 'item_', '_item.name', '_em_vitrification.instrument' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'Reichert plunger' ] ] ],
      [ 'save_',
        '_em_vitrification.method',
        [ [ 'item_',
            '_item_description.description',
            ' The procedure for vitrification. ' ],
          [ 'item_', '_item.name', '_em_vitrification.method' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'blot for 2 seconds before plunging' ] ] ],
      [ 'save_',
        '_em_vitrification.time_resolved_state',
        [ [ 'item_',
            '_item_description.description',
            ' The length of time after an event effecting the sample that\n  vitrification was induced and a description of the event. ' ],
          [ 'item_',
            '_item.name',
            '_em_vitrification.time_resolved_state' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '30 msec after spraying with effector\'' ] ] ],
      [ 'save_',
        '_em_vitrification.citation_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to _citation.id in the\n  CITATION category.' ],
          [ 'item_', '_item.name', '_em_vitrification.citation_id' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_vitrification.citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ] ] ],
      [ 'save_',
        '_em_vitrification.details',
        [ [ 'item_',
            '_item_description.description',
            '  Any additional details relating to vitrification.' ],
          [ 'item_', '_item.name', '_em_vitrification.details' ],
          [ 'item_', '_item.category_id', 'em_vitrification' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'argon atmosphere' ] ] ],
      [ '@comment', '###############' ],
      [ '@comment', '# EM_IMAGING ##' ],
      [ '@comment', '###############' ],
      [ 'save_',
        'em_imaging',
        [ [ 'item_',
            '_category.description',
            '  Data items in the EM_IMAGING category record details about \n   the parameters used in imaging the sample in the electron microscope.' ],
          [ 'item_', '_category.id', 'em_imaging' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_imaging.entry_id', '_em_imaging.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              '  \n   _em_imaging.entry_id               1DYL\n   _em_imaging.id                     1\n   _em_imaging.sample_support_id      1\n   _em_imaging.microscope_model       \'FEI/PHILIPS CM200 FEG\'\n   _em_imaging.specimen_holder_type   \'cryotransfer\'\n   _em_imaging.specimen_holder_model  \'gatan 626-0300\'\n   _em_imaging.details                .\n   _em_imaging.date                   1998-15-06\n   _em_imaging.accelerating_voltage   200\n   _em_imaging.illumination_mode      \'bright field\'\n   _em_imaging.mode                   \'low dose\'\n   _em_imaging.nominal_cs             2.0\n   _em_imaging.nominal_defocus_min    975\n   _em_imaging.nominal_defocus_max    7600\n   _em_imaging.tilt_angle_min         0\n   _em_imaging.tilt_angle_max         0\n   _em_imaging.nominal_magnification  50000\n   _em_imaging.calibrated_magnification .\n   _em_imaging.electron_source        \'FEG\'\n   _em_imaging.electron_dose          .\n   _em_imaging.energy_filter          .\n   _em_imaging.energy_window          .\n   _em_imaging.citation_id            1\n   _em_imaging.temperature            95\n   _em_imaging.detector_distance      .\n   _em_imaging.recording_temp_range   .' ] ] ] ],
      [ 'save_',
        '_em_imaging.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '  This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_em_imaging.entry_id' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.child_name', '_em_imaging.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_imaging.id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _em_imaging.id must uniquely identify\n  each imaging experiment. ' ],
          [ 'item_', '_item.name', '_em_imaging.id' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_imaging.sample_support_id',
        [ [ 'item_',
            '_item_description.description',
            ' This data item is a pointer to _em_sample_support.id in\n  the EM_SAMPLE_SUPPORT category.' ],
          [ 'item_', '_item.name', '_em_imaging.sample_support_id' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_imaging.detector_id',
        [ [ 'item_',
            '_item_description.description',
            '   The value of _em_imaging.detector_id must uniquely identify\n    the type of detector used in the experiment. ' ],
          [ 'item_', '_item.name', '_em_imaging.detector_id' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_imaging.scans_id',
        [ [ 'item_',
            '_item_description.description',
            ' The value of _em_imaging.scans_id must uniquely identify\n  the image_scans used in the experiment. ' ],
          [ 'item_', '_item.name', '_em_imaging.scans_id' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_imaging.microscope_model',
        [ [ 'item_',
            '_item_description.description',
            '  The name of the model of microscope. ' ],
          [ 'item_', '_item.name', '_em_imaging.microscope_model' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'HITACHI H8100',
              'HITACHI HF2000',
              'HITACHI HF2000-UHR',
              'HITACHI H9000-UHR',
              'HITACHI H9000-NAR',
              'HITACHI 300KEV FEG',
              'HITACHI HU1250',
              'HITACHI H-1500',
              'JEOL 2000EX',
              'JEOL 2010HT',
              'JEOL 2010UHR',
              'JEOL 2010F',
              'JEOL 3010HT',
              'JEOL 3010UHR',
              'JEOL KYOTO-3000SFF',
              'JEOL 4000EX',
              'JEOL HAREM',
              'JEOL ARM-1000',
              'JEOL KYOTO-1000',
              'JEOL ARM-1250',
              'FEI/PHILIPS CM120T',
              'FEI/PHILIPS CM200T',
              'FEI/PHILIPS CM20/ST',
              'FEI/PHILIPS CM20/SOPHIE',
              'FEI/PHILIPS CM200FEG/ST',
              'FEI/PHILIPS CM20/UT',
              'FEI/PHILIPS CM200FEG/UT',
              'FEI/PHILIPS CM30/T',
              'FEI/PHILIPS CM300FEG/T',
              'FEI/PHILIPS CM300FEG/HE',
              'FEI/PHILIPS CM30/ST',
              'FEI/PHILIPS CM300FEG/ST',
              'FEI/PHILIPS CM300FEG/UT',
              'FEI TECNAI 12',
              'FEI TECNAI 20',
              'FEI TECNAI F20',
              'FEI TECNAI F30',
              'FEI MORGAGNI' ] ] ] ],
      [ 'save_',
        '_em_imaging.specimen_holder_type',
        [ [ 'item_',
            '_item_description.description',
            ' The type of specimen holder used during imaging. ' ],
          [ 'item_', '_item.name', '_em_imaging.specimen_holder_type' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'cryo' ] ] ],
      [ 'save_',
        '_em_imaging.specimen_holder_model',
        [ [ 'item_',
            '_item_description.description',
            ' The name of the model of specimen holder used during imaging. ' ],
          [ 'item_', '_item.name', '_em_imaging.specimen_holder_model' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'GATAN HELIUM', 'GATAN LIQUID NITROGEN', 'OTHER' ] ] ] ],
      [ 'save_',
        '_em_imaging.details',
        [ [ 'item_',
            '_item_description.description',
            ' Any additional imaging details. ' ],
          [ 'item_', '_item.name', '_em_imaging.details' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'weak beam illumination' ] ] ],
      [ 'save_',
        '_em_imaging.date',
        [ [ 'item_',
            '_item_description.description',
            ' Date (YYYY-MM-DD) of imaging experiment or the date at which\n  a series of experiments began. ' ],
          [ 'item_', '_item.name', '_em_imaging.date' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ '@comment', '    _item_type.code               yyyy-mm-dd' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', '2001-05-08' ] ] ],
      [ 'save_',
        '_em_imaging.accelerating_voltage',
        [ [ 'item_',
            '_item_description.description',
            '  A value of accelerating voltage (in kV) used for imaging. ' ],
          [ 'item_', '_item.name', '_em_imaging.accelerating_voltage' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_units.code', 'kilovolts' ],
          [ 'item_', '_item_examples.case', '300' ] ] ],
      [ 'save_',
        '_em_imaging.illumination_mode',
        [ [ 'item_',
            '_item_description.description',
            ' The mode of illumination. ' ],
          [ 'item_', '_item.name', '_em_imaging.illumination_mode' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'FLOOD BEAM', 'FLOOD BEAM LOW DOSE', 'SPOT SCAN', 'OTHER' ] ] ] ],
      [ 'save_',
        '_em_imaging.mode',
        [ [ 'item_',
            '_item_description.description',
            '  The mode of imaging. ' ],
          [ 'item_', '_item.name', '_em_imaging.mode' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'BRIGHT FIELD', 'DARK FIELD', 'DIFFRACTION', 'OTHER' ] ] ] ],
      [ 'save_',
        '_em_imaging.nominal_cs',
        [ [ 'item_',
            '_item_description.description',
            ' The spherical aberration coefficient (Cs) in millimetres,\n  of the objective lens. ' ],
          [ 'item_', '_item.name', '_em_imaging.nominal_cs' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'millimetres' ],
          [ 'item_', '_item_examples.case', '1.4' ] ] ],
      [ 'save_',
        '_em_imaging.nominal_defocus_min',
        [ [ 'item_',
            '_item_description.description',
            '   The minimum defocus value of the objective lens (in nanometres) used\n    to obtain the recorded images. ' ],
          [ 'item_', '_item.name', '_em_imaging.nominal_defocus_min' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'nanometres' ],
          [ 'item_', '_item_examples.case', '975' ] ] ],
      [ 'save_',
        '_em_imaging.nominal_defocus_max',
        [ [ 'item_',
            '_item_description.description',
            '   The maximum defocus value of the objective lens (in nanometres) used\n    to obtain the recorded images. ' ],
          [ 'item_', '_item.name', '_em_imaging.nominal_defocus_max' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'nanometres' ],
          [ 'item_', '_item_examples.case', '7600' ] ] ],
      [ 'save_',
        '_em_imaging.tilt_angle_min',
        [ [ 'item_',
            '_item_description.description',
            '  The minimum angle at which the specimen was tilted to obtain\n   recorded images.' ],
          [ 'item_', '_item.name', '_em_imaging.tilt_angle_min' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_', '_item_examples.case', '0' ] ] ],
      [ 'save_',
        '_em_imaging.tilt_angle_max',
        [ [ 'item_',
            '_item_description.description',
            '  The maximum angle at which the specimen was tilted to obtain\n   recorded images.' ],
          [ 'item_', '_item.name', '_em_imaging.tilt_angle_max' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_', '_item_examples.case', '60' ] ] ],
      [ 'save_',
        '_em_imaging.nominal_magnification',
        [ [ 'item_',
            '_item_description.description',
            '  The magnification indicated by the microscope readout. ' ],
          [ 'item_', '_item.name', '_em_imaging.nominal_magnification' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '60000' ] ] ],
      [ 'save_',
        '_em_imaging.calibrated_magnification',
        [ [ 'item_',
            '_item_description.description',
            '  The magnification value obtained for a known standard just\n   prior to, during or just after the imaging experiment. ' ],
          [ 'item_', '_item.name', '_em_imaging.calibrated_magnification' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '61200' ] ] ],
      [ 'save_',
        '_em_imaging.electron_source',
        [ [ 'item_',
            '_item_description.description',
            ' The source of electrons. The electron gun.' ],
          [ 'item_', '_item.name', '_em_imaging.electron_source' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'FIELD EMISSION GUN',
              'LAB6',
              'TUNGSTEN HAIRPIN',
              'SCHOTTKY FIELD EMISSION GUN',
              'OTHER' ] ] ] ],
      [ 'save_',
        '_em_imaging.electron_dose',
        [ [ 'item_',
            '_item_description.description',
            ' The electron dose received by the specimen (electrons per square angstrom). ' ],
          [ 'item_', '_item.name', '_em_imaging.electron_dose' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'electrons_angstrom_squared' ],
          [ 'item_', '_item_examples.case', '0.9' ] ] ],
      [ 'save_',
        '_em_imaging.energy_filter',
        [ [ 'item_',
            '_item_description.description',
            '   The type of energy filter spectrometer apparatus. ' ],
          [ 'item_', '_item.name', '_em_imaging.energy_filter' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'FEI' ] ] ],
      [ 'save_',
        '_em_imaging.energy_window',
        [ [ 'item_',
            '_item_description.description',
            '   The energy filter range in electron volts (eV)set by spectrometer. ' ],
          [ 'item_', '_item.name', '_em_imaging.energy_window' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_units.code', 'electron_volts' ],
          [ 'item_', '_item_examples.case', '0 - 15' ] ] ],
      [ 'save_',
        '_em_imaging.citation_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to _citation.id in\n     the CITATION category.' ],
          [ 'item_', '_item.name', '_em_imaging.citation_id' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_linked.child_name', '_em_imaging.citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_imaging.temperature',
        [ [ 'item_',
            '_item_description.description',
            '    The mean specimen stage temperature (degrees Kelvin) during imaging\n     in the microscope. ' ],
          [ 'item_', '_item.name', '_em_imaging.temperature' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ 'save_',
        '_em_imaging.detector_distance',
        [ [ 'item_',
            '_item_description.description',
            '    The camera length (in millimetres). The camera length is the \n     product of the objective focal length and the combined magnification\n     of the intermediate and projector lenses when the microscope is\n     operated in the diffraction mode. ' ],
          [ 'item_', '_item.name', '_em_imaging.detector_distance' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'millimetres' ] ] ],
      [ 'save_',
        '_em_imaging.recording_temperature_minimum',
        [ [ 'item_',
            '_item_description.description',
            '    The specimen temperature minimum (degrees Kelvin) for the duration\n     of imaging. ' ],
          [ 'item_',
            '_item.name',
            '_em_imaging.recording_temperature_minimum' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ 'save_',
        '_em_imaging.recording_temperature_maximum',
        [ [ 'item_',
            '_item_description.description',
            '    The specimen temperature maximum (degrees Kelvin) for the duration\n     of imaging. ' ],
          [ 'item_',
            '_item.name',
            '_em_imaging.recording_temperature_maximum' ],
          [ 'item_', '_item.category_id', 'em_imaging' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ '@comment', '################' ],
      [ '@comment', '# EM_DETECTOR ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'em_detector',
        [ [ 'item_',
            '_category.description',
            '  Data items in the EM_DETECTOR category record details\n   of the image detector type.' ],
          [ 'item_', '_category.id', 'em_detector' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_detector.entry_id', '_em_detector.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              ' \n  _em_detector.entry_id                        1DYL\n  _em_detector.id                             1\n  _em_detector.details                        .\n  _em_detector.type                           \'KODAK SO163 FILM\'\n  _em_detector.detective_quantum_efficiency   .' ] ] ] ],
      [ 'save_',
        '_em_detector.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '   This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_em_detector.entry_id' ],
          [ 'item_', '_item.category_id', 'em_detector' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.child_name', '_em_detector.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_detector.id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_detector.id must uniquely identify\n     the detector used for imaging.     ' ],
          [ 'item_', '_item.name', '_em_detector.id' ],
          [ 'item_', '_item.category_id', 'em_detector' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_em_imaging.detector_id', '_em_detector.id' ] ] ] ],
      [ 'save_',
        '_em_detector.details',
        [ [ 'item_',
            '_item_description.description',
            '    Any additional information about the detection system. ' ],
          [ 'item_', '_item.name', '_em_detector.details' ],
          [ 'item_', '_item.category_id', 'em_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_detector.type',
        [ [ 'item_',
            '_item_description.description',
            '    The detector type used for recording images. \n     Usually film or CCD camera.  ' ],
          [ 'item_', '_item.name', '_em_detector.type' ],
          [ 'item_', '_item.category_id', 'em_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'KODAK SO163 FILM',
              'GATAN 673',
              'GATAN 676',
              'GATAN 692',
              'GATAN 794',
              'GATAN 1000',
              'GATAN 4000',
              'TVIPS BIOCAM',
              'TVIPS TEMCAM F214',
              'TVIPS TEMCAM F224',
              'TVIPS FASTSCAN F114',
              'PROSCAN',
              'AMT' ] ] ] ],
      [ 'save_',
        '_em_detector.detective_quantum_efficiency',
        [ [ 'item_',
            '_item_description.description',
            '    The detective_quantum_efficiency (DQE)is defined as the \n     square of the signal-to-noise ratio in the recording device\n     divided by the square of the signal-to-ratio in the electron beam:\n            (SIGNAL/NOISE)2 recording device\n   DQE =    -------------------------------\n            (SIGNAL/NOISE)2 electron beam\n   A DQE value of 1 indicates a perfect recorder. "DQE = 0.25" menas\n   that the signal-to-noise ratio is reduced by half in the \n   recording step. \n        (0.5)**2  \n  DQE = --------- = 0.25.\n         (1.0)**2' ],
          [ 'item_',
            '_item.name',
            '_em_detector.detective_quantum_efficiency' ],
          [ 'item_', '_item.category_id', 'em_detector' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '0.25' ] ] ],
      [ '@comment', '###################' ],
      [ '@comment', '# EM_IMAGE_SCANS ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'em_image_scans',
        [ [ 'item_',
            '_category.description',
            '  Data items in the EM_IMAGE_SCANS category record details\n   of the image scanning device (microdensitometer)\n   and parameters for digitization of the image.' ],
          [ 'item_', '_category.id', 'em_image_scans' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_image_scans.entry_id', '_em_image_scans.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              '  \n   _em_image_scans.entry_id                      1DYL\n   _em_image_scans.id                            2\n   _em_image_scans.number_digital_images         48\n   _em_image_scans.details                       .\n   _em_image_scans.scanner_model                 .\n   _em_image_scans.sampling_size                 .\n   _em_image_scans.od_range                      .\n   _em_image_scans.quant_bit_number              .\n   _em_image_scans.citation_id                   1' ] ] ] ],
      [ 'save_',
        '_em_image_scans.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to _entry.id in the\n     ENTRY category.' ],
          [ 'item_', '_item.name', '_em_image_scans.entry_id' ],
          [ 'item_', '_item.category_id', 'em_image_scans' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_image_scans.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_image_scans.id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_image_scans.id must uniquely identify\n     the images scanned.     ' ],
          [ 'item_', '_item.name', '_em_image_scans.id' ],
          [ 'item_', '_item.category_id', 'em_image_scans' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_em_imaging.scans_id', '_em_image_scans.id' ] ] ] ],
      [ 'save_',
        '_em_image_scans.number_digital_images',
        [ [ 'item_',
            '_item_description.description',
            '    The number of images scanned and digitised.     ' ],
          [ 'item_',
            '_item.name',
            '_em_image_scans.number_digital_images' ],
          [ 'item_', '_item.category_id', 'em_image_scans' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_em_image_scans.details',
        [ [ 'item_',
            '_item_description.description',
            '    Any additional details about scanning images.     ' ],
          [ 'item_', '_item.name', '_em_image_scans.details' ],
          [ 'item_', '_item.category_id', 'em_image_scans' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_image_scans.scanner_model',
        [ [ 'item_',
            '_item_description.description',
            '    The scanner model.     ' ],
          [ 'item_', '_item.name', '_em_image_scans.scanner_model' ],
          [ 'item_', '_item.category_id', 'em_image_scans' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'ZEISS SCAI',
              'EMIL 10',
              'OPTRONICS',
              'PERKIN ELMER',
              'TEMSCAN',
              'OTHER' ] ] ] ],
      [ 'save_',
        '_em_image_scans.sampling_size',
        [ [ 'item_',
            '_item_description.description',
            '    The sampling step size (microns) set on the scanner.     ' ],
          [ 'item_', '_item.name', '_em_image_scans.sampling_size' ],
          [ 'item_', '_item.category_id', 'em_image_scans' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'microns' ] ] ],
      [ 'save_',
        '_em_image_scans.od_range',
        [ [ 'item_',
            '_item_description.description',
            '    The optical density range (OD=-log 10 transmission).     \n     To the eye OD=1 appears light grey and OD=3 is opaque.' ],
          [ 'item_', '_item.name', '_em_image_scans.od_range' ],
          [ 'item_', '_item.category_id', 'em_image_scans' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_examples.case', '1.4' ] ] ],
      [ 'save_',
        '_em_image_scans.quant_bit_size',
        [ [ 'item_',
            '_item_description.description',
            '    The number of bits per pixel.' ],
          [ 'item_', '_item.name', '_em_image_scans.quant_bit_size' ],
          [ 'item_', '_item.category_id', 'em_image_scans' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '8' ] ] ],
      [ 'save_',
        '_em_image_scans.citation_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to _citation.id\n     in the CITATION category.' ],
          [ 'item_', '_item.name', '_em_image_scans.citation_id' ],
          [ 'item_', '_item.category_id', 'em_image_scans' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_image_scans.citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ] ] ],
      [ '@comment', '###############################' ],
      [ '@comment', '# EM_2D_PROJECTION_SELECTION ##' ],
      [ '@comment', '###############################' ],
      [ 'save_',
        'em_2d_projection_selection',
        [ [ 'item_',
            '_category.description',
            '   Data items in the EM_2D_PROJECTION_SELECTION category\n    record details of images from scanned micrographs and the \n    number of particles selected from a scanned set of micrographs.' ],
          [ 'item_', '_category.id', 'em_2d_projection_selection' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_em_2d_projection_selection.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              ' \n   _em_2d_projection_selection.entry_id            1\n   _em_2d_projection_selection.number_particles    5267\n   _em_2d_projection_selection.software_name         1\n   _em_2d_projection_selection.method              \'INTERACTIVE\'\n   _em_2d_projection_selection.details             .\n   _em_2d_projection_selection.citation_id         1' ] ] ] ],
      [ 'save_',
        '_em_2d_projection_selection.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '  The value of _em_2d_projection_selection.entry_id points to\n   the ENTRY category.' ],
          [ 'item_', '_item.name', '_em_2d_projection_selection.entry_id' ],
          [ 'item_', '_item.category_id', 'em_2d_projection_selection' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_2d_projection_selection.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_2d_projection_selection.num_particles',
        [ [ 'item_',
            '_item_description.description',
            '    The number of particles selected from the projection set of images.' ],
          [ 'item_',
            '_item.name',
            '_em_2d_projection_selection.num_particles' ],
          [ 'item_', '_item.category_id', 'em_2d_projection_selection' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '840' ] ] ],
      [ 'save_',
        '_em_2d_projection_selection.software_name',
        [ [ 'item_',
            '_item_description.description',
            '   This data item is a pointer to _software.name in the\n    SOFTWARE category.' ],
          [ 'item_',
            '_item.name',
            '_em_2d_projection_selection.software_name' ],
          [ 'item_', '_item.category_id', 'em_2d_projection_selection' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ '@comment',
            '    _item_linked.child_name      \'_em_2d_projection_selection.software_name\'' ],
          [ '@comment',
            '    _item_linked.parent_name     \'_software.name\'' ] ] ],
      [ 'save_',
        '_em_2d_projection_selection.method',
        [ [ 'item_',
            '_item_description.description',
            '    The method used for selecting observed assemblies.' ],
          [ 'item_', '_item.name', '_em_2d_projection_selection.method' ],
          [ 'item_', '_item.category_id', 'em_2d_projection_selection' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'particles picked interactively from monitor' ] ] ],
      [ 'save_',
        '_em_2d_projection_selection.details',
        [ [ 'item_',
            '_item_description.description',
            '    Any additional details used for selecting observed assemblies.' ],
          [ 'item_', '_item.name', '_em_2d_projection_selection.details' ],
          [ 'item_', '_item.category_id', 'em_2d_projection_selection' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'negative monitor contrast facilitated particle picking' ] ] ],
      [ 'save_',
        '_em_2d_projection_selection.citation_id',
        [ [ 'item_',
            '_item_description.description',
            '   This data item is a pointer to _citation.id in the\n    CITATION category.' ],
          [ 'item_',
            '_item.name',
            '_em_2d_projection_selection.citation_id' ],
          [ 'item_', '_item.category_id', 'em_2d_projection_selection' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_2d_projection_selection.citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ] ] ],
      [ '@comment', '################################' ],
      [ '@comment', '# EM_EULER_ANGLE_DISTRIBUTION ##' ],
      [ '@comment', '################################' ],
      [ 'save_',
        'em_euler_angle_distribution',
        [ [ 'item_',
            '_category.description',
            '  Data items in the EM_EULER_ANGLE_DISTRIBUTION category\n   record details of assignment of Euler angles for projection\n   sets of particles.' ],
          [ 'item_', '_category.id', 'em_euler_angle_distribution' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_euler_angle_distribution.id',
              '_em_euler_angle_distribution.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ ' Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              ' \n  _em_euler_angle_distribution.entry_id                          1DYL\n  _em_euler_angle_distribution.id                                1\n  _em_euler_angle_distribution.details                           .\n  _em_euler_angle_distribution.alpha                             .\n  _em_euler_angle_distribution.beta                              .\n  _em_euler_angle_distribution.gamma                             .' ] ] ] ],
      [ 'save_',
        '_em_euler_angle_distribution.id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_euler_angle_distribution.id must\n     uniquely identify the  euler angle assignments of\n     the projection set used in the final reconstruction.' ],
          [ 'item_', '_item.name', '_em_euler_angle_distribution.id' ],
          [ 'item_', '_item.category_id', 'em_euler_angle_distribution' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_euler_angle_distribution.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_euler_angle_distribution.entry_id is a pointer\n     to the ENTRY category.' ],
          [ 'item_',
            '_item.name',
            '_em_euler_angle_distribution.entry_id' ],
          [ 'item_', '_item.category_id', 'em_euler_angle_distribution' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_euler_angle_distribution.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_euler_angle_distribution.details',
        [ [ 'item_',
            '_item_description.description',
            '    Any additional details of the euler angles distribution and assignment.' ],
          [ 'item_', '_item.name', '_em_euler_angle_distribution.details' ],
          [ 'item_', '_item.category_id', 'em_euler_angle_distribution' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_euler_angle_distribution.alpha',
        [ [ 'item_',
            '_item_description.description',
            '    The euler-alpha angle assignment.' ],
          [ 'item_', '_item.name', '_em_euler_angle_distribution.alpha' ],
          [ 'item_', '_item.category_id', 'em_euler_angle_distribution' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_', '_item_examples.case', '90' ] ] ],
      [ 'save_',
        '_em_euler_angle_distribution.beta',
        [ [ 'item_',
            '_item_description.description',
            '    The euler-beta angle assignment.' ],
          [ 'item_', '_item.name', '_em_euler_angle_distribution.beta' ],
          [ 'item_', '_item.category_id', 'em_euler_angle_distribution' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_', '_item_examples.case', '90' ] ] ],
      [ 'save_',
        '_em_euler_angle_distribution.gamma',
        [ [ 'item_',
            '_item_description.description',
            '    The euler-gamma angle assignment.' ],
          [ 'item_', '_item.name', '_em_euler_angle_distribution.gamma' ],
          [ 'item_', '_item.category_id', 'em_euler_angle_distribution' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'degrees' ],
          [ 'item_', '_item_examples.case', '0' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# EM_3D_RECONSTRUCTION ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'em_3d_reconstruction',
        [ [ 'item_',
            '_category.description',
            '  Data items in the EM_3D_RECONSTRUCTION category\n   record details of the 3D reconstruction procedure from 2D projections.' ],
          [ 'item_', '_category.id', 'em_3d_reconstruction' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_3d_reconstruction.entry_id', '_em_3d_reconstruction.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ ' Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              '\n  _em_3d_reconstruction.entry_id                 1DYL\n  _em_3d_reconstruction.id                       1\n  _em_3d_reconstruction.method                   \'CROSS-COMMON LINES\'\n  _em_3d_reconstruction.citation_id              1\n  _em_3d_reconstruction.details                  .\n  _em_3d_reconstruction.resolution               9\n  _em_3d_reconstruction.resolution_method        .\n  _em_3d_reconstruction.ctf_correction_method    .\n  _em_3d_reconstruction.nominal_pixel_size       2.64\n  _em_3d_reconstruction.actual_pixel_size        2.52' ] ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '  This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_em_3d_reconstruction.entry_id' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_3d_reconstruction.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_3d_reconstruction.id must\n     uniquely identify the 3d reconstruction.' ],
          [ 'item_', '_item.name', '_em_3d_reconstruction.id' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.method',
        [ [ 'item_',
            '_item_description.description',
            '    The algorithm method used for the 3d-reconstruction.' ],
          [ 'item_', '_item.name', '_em_3d_reconstruction.method' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'cross-common lines' ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.citation_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to _citation.id in the\n     CITATION category.' ],
          [ 'item_', '_item.name', '_em_3d_reconstruction.citation_id' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_3d_reconstruction.citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.details',
        [ [ 'item_',
            '_item_description.description',
            '    Any additional details used in the 3d reconstruction.' ],
          [ 'item_', '_item.name', '_em_3d_reconstruction.details' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.resolution',
        [ [ 'item_',
            '_item_description.description',
            '    The final resolution (in angstroms)of the 3d reconstruction.' ],
          [ 'item_', '_item.name', '_em_3d_reconstruction.resolution' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.resolution_method',
        [ [ 'item_',
            '_item_description.description',
            '    The  method used to determine the final resolution \n     of the 3d reconstruction.\n     The Fourier Shell Correlation criterion as a measure of\n     resolution is based on the concept of splitting the (2D)\n     data set into two halves; averaging each and comparing them\n     using the Fourier Ring Correlation (FRC) technique.' ],
          [ 'item_',
            '_item.name',
            '_em_3d_reconstruction.resolution_method' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'FSC at 0.5 cut-off' ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.magnification_calibration',
        [ [ 'item_',
            '_item_description.description',
            '    The  magnification calibration method for the 3d reconstruction.' ],
          [ 'item_',
            '_item.name',
            '_em_3d_reconstruction.magnification_calibration' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.ctf_correction_method',
        [ [ 'item_',
            '_item_description.description',
            '    The  CTF-correction method. \n     The Contrast Transfer Function CTF compensation for low contrast\n     specimens (e.g. frozen-hydrated), for which phase contrast is the only\n     significant mechanism, then higher defocus levels must be used to\n     achieve any significant transfer, and several images at different\n     focus levels must be combined to complete the information lost from\n     the transfer gaps of any one image.  The CTF correction can be applied\n     to each extracted particle separately or to the whole micrograph after\n     digitisation. The simplest level of compensation is to reverse phases\n     at the negative lobes of the CTF.' ],
          [ 'item_',
            '_item.name',
            '_em_3d_reconstruction.ctf_correction_method' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'CTF correction of each particle' ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.nominal_pixel_size',
        [ [ 'item_',
            '_item_description.description',
            '    The nominal pixel size of the projection set of images.' ],
          [ 'item_',
            '_item.name',
            '_em_3d_reconstruction.nominal_pixel_size' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_em_3d_reconstruction.actual_pixel_size',
        [ [ 'item_',
            '_item_description.description',
            '    The actual pixel size of projection set of images.' ],
          [ 'item_',
            '_item.name',
            '_em_3d_reconstruction.actual_pixel_size' ],
          [ 'item_', '_item.category_id', 'em_3d_reconstruction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ '@comment', '##################' ],
      [ '@comment', '# EM_3D_FITTING ##' ],
      [ '@comment', '##################' ],
      [ 'save_',
        'em_3d_fitting',
        [ [ 'item_',
            '_category.description',
            '    Data items in the 3D_FITTING category\n     record details of the method of fitting atomic\n     coordinates from a PDB file into a 3d-em\n     volume map file ' ],
          [ 'item_', '_category.id', 'em_3d_fitting' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_3d_fitting.id', '_em_3d_fitting.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'em_group', 'inclusive_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              ' \n   _em_3d_fitting.id                      1\n   _em_3d_fitting.entry_id                1DYL\n   _em_3d_fitting.method                  AUTOMATIC\n   _em_3d_fitting.target_criteria         R-FACTOR\n   _em_3d_fitting.software_name             1\n   _em_3d_fitting.over_all_b_value        .\n   _em_3d_fitting.ref_space               REAL\n   _em_3d_fitting.ref_protocol            \'RIGID BODY REFINEMENT\'\n   _em_3d_fitting.details                 \n   ;   THE CRYSTAL STRUCTURE OF THE CAPSID              \n       PROTEIN FROM CHOI ET AL (1997) PROTEINS 3 27:345-359                \n       (SUBUNIT A OF PDB FILE 1VCQ) WAS PLACED INTO THE CRYO-EM            \n       DENSITY MAP. THE CAPSID PROTEIN WAS FIRST MANUALLY POSITIONED       \n       INTO THE CRYO-EM DENSITY CORRESPONDING TO POSITIONS OF THE          \n       FOUR INDEPENDENT MONOMER DENSITIES BETWEEN THE INNER LEAFLET        \n       OF THE BILAYER AND THE RNA. THESE POSITIONS WERE THEN REFINED       \n       BY RIGID BODY REFINEMENT IN REAL SPACE WITH THE PROGRAM EMFIT       \n       (CHENG ET AL. 1995, CELL 80, 621-630).  THE QUALITY OF THE FIT      \n       CAN BE SEEN FROM THE MAP DENSITY WITHIN THE PROTEIN. ALL 4563       \n       ATOMS ARE IN DENSITY OF AT LEAST 4 SIGMA (96.73) ABOVE THE          \n       AVERAGE (512.04), 1167 ATOMS ARE IN DENSITY BETWEEN 4 AND 5         \n       SIGMA, 3174 ATOMS ARE IN DENSITY BETWEEN 5 AND 6 SIGMA, AND 222     \n       ATOMS ARE IN DENSTY OF 6 SIGMA OR ABOVE. THE VARIATION IN           \n       DENSITY OVER THE FITTED PROTEIN CAN BE VISUALIZED WITH THE          \n       PSEUDO TEMPERATURE FACTOR. THE DENSITY VALUE AT EACH ATOM IS        \n       GIVEN IN THE 8TH COLUM (USUALLY THE OCCUPANCY) AS THE NUMBER        \n       OF STANDARD DEVIATION ABOVE BACKGROUND. COLUMN NINE (USUALLY        \n       THE TEMPERATURE FACTOR) CONTAINS THE VALUE OF THE RELATIVE          \n       DENSITY WITHIN THE FITTED PROTEIN SCALED LINEARLY SO THAT THE       \n       MINIMUM DENSITY IS 100.0 AND THE MAXIMUM DENSITY IS 1.0.  THE       \n       ATOMS THAT LIE IN THE LOWER DENSITY REGIONS WILL HAVE THE           \n       HIGHEST PSEUDO TEMPERATURE FACTORS.                                 \n   ;' ] ] ] ],
      [ 'save_',
        '_em_3d_fitting.id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_3d_fitting.id must uniquely identify\n     a fitting procedure of atomic coordinates\n     into 3dem reconstructed volume map.' ],
          [ 'loop_',
            [ '_item.name', '_item.category_id', '_item.mandatory_code' ],
            [ '_em_3d_fitting.id',
              'em_3d_fitting',
              'yes',
              '_em_3d_fitting_list.3d_fitting_id',
              'em_3d_fitting_list',
              'yes' ] ],
          [ 'loop_',
            [ '_item_linked.child_name', '_item_linked.parent_name' ],
            [ '_em_3d_fitting_list.3d_fitting_id', '_em_3d_fitting.id' ] ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_3d_fitting.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to _entry_id in\n     the ENTRY category.' ],
          [ 'item_', '_item.name', '_em_3d_fitting.entry_id' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.child_name', '_em_3d_fitting.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_3d_fitting.method',
        [ [ 'item_',
            '_item_description.description',
            '    The method used to fit atomic coordinates\n     into the 3dem reconstructed map.' ],
          [ 'item_', '_item.name', '_em_3d_fitting.method' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_em_3d_fitting.target_criteria',
        [ [ 'item_',
            '_item_description.description',
            '    The quality of fit of the atomic coordinates into the \n     3dem volume map.' ],
          [ 'item_', '_item.name', '_em_3d_fitting.target_criteria' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'best visual fit using the program O' ] ] ],
      [ 'save_',
        '_em_3d_fitting.software_name',
        [ [ 'item_',
            '_item_description.description',
            '  This data item is a pointer to _software.name\n   in the  category.' ],
          [ 'item_', '_item.name', '_em_3d_fitting.software_name' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ '@comment',
            '    _item_linked.child_name      \'_em_3d_fitting.software_name\'' ],
          [ '@comment',
            '    _item_linked.parent_name     \'_software.name\'' ] ] ],
      [ 'save_',
        '_em_3d_fitting.details',
        [ [ 'item_',
            '_item_description.description',
            '    Any additional details regarding fitting of atomic\n     coordinates into the 3d-em volume. ' ],
          [ 'item_', '_item.name', '_em_3d_fitting.details' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'partial' ] ] ],
      [ 'save_',
        '_em_3d_fitting.overall_b_value',
        [ [ 'item_',
            '_item_description.description',
            '     The overall B (temperature factor) value for the 3d-em volume. ' ],
          [ 'item_', '_item.name', '_em_3d_fitting.overall_b_value' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_em_3d_fitting.ref_space',
        [ [ 'item_',
            '_item_description.description',
            '    A flag to indicate whether fitting was carried out in real\n     or reciprocal refinement space.' ],
          [ 'item_', '_item.name', '_em_3d_fitting.ref_space' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'REAL', 'RECIPROCAL' ] ] ] ],
      [ 'save_',
        '_em_3d_fitting.ref_protocol',
        [ [ 'item_',
            '_item_description.description',
            '  The type of protocol used in the refinement. ' ],
          [ 'item_', '_item.name', '_em_3d_fitting.ref_protocol' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'rigid body' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# EM_3D_FITTING_LIST ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'em_3d_fitting_list',
        [ [ 'item_',
            '_category.description',
            '  Data items in the 3D_FITTING_LIST category\n   lists the methods of fitting atomic coordinates from a PDB file\n   into a 3d-em volume map file ' ],
          [ 'item_', '_category.id', 'em_3d_fitting_list' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_3d_fitting_list.id',
              '_em_3d_fitting_list.3d_fitting_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - based on PDB entry 1DYL and laboratory records for the\n                structure corresponding to PDB entry 1DYL',
              '  _em_3d_fitting_list.id                       1\n   _em_3d_fitting_list.fitting_id               l\n   _em_3d_fitting_list.pdb_entry_id             1VCQ\n   _em_3d_fitting_list.pdb_chain_id             .' ] ] ] ],
      [ 'save_',
        '_em_3d_fitting_list.id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a unique identifier.' ],
          [ 'item_', '_item.name', '_em_3d_fitting_list.id' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting_list' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_3d_fitting_list.3d_fitting_id',
        [ [ 'item_',
            '_item_description.description',
            '    The value of _em_3d_fitting_list.3d_fitting_id is a pointer\n     to  _em_3d_fitting.id in the 3d_fitting category' ],
          [ 'item_', '_item.name', '_em_3d_fitting_list.3d_fitting_id' ],
          [ 'item_', '_item.mandatory_code', 'yes' ] ] ],
      [ 'save_',
        '_em_3d_fitting_list.pdb_entry_id',
        [ [ 'item_',
            '_item_description.description',
            '  The PDB code for the entry used in fitting. ' ],
          [ 'item_', '_item.name', '_em_3d_fitting_list.pdb_entry_id' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting_list' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_em_3d_fitting_list.pdb_chain_id',
        [ [ 'item_',
            '_item_description.description',
            '  The chain id for the entry used in fitting. ' ],
          [ 'item_', '_item.name', '_em_3d_fitting_list.pdb_chain_id' ],
          [ 'item_', '_item.category_id', 'em_3d_fitting_list' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', '# EM_ELECTRON_DIFFRACTION ##' ],
      [ '@comment', '############################' ],
      [ 'save_',
        'em_electron_diffraction',
        [ [ 'item_',
            '_category.description',
            '   Data items in the EM_ELECTRON_DIFFRACTION category\n    record details about the electron diffraction data\n    from the electron crystallography experiment.' ],
          [ 'item_', '_category.id', 'em_electron_diffraction' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_electron_diffraction.id',
              '_em_electron_diffraction.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  Example 1 - based on PDB entry 1TUB and laboratory records for the\n                structure corresponding to PDB entry 1TUB',
              ' \n  _em_electron_diffraction.entry_id                   1TUB\n  _em_electron_diffraction.id                         1\n  _em_electron_diffraction.num_structure_factors  12000\n  _em_electron_diffraction.num_diff_patterns          .\n  _em_electron_diffraction.details                    .' ] ] ] ],
      [ 'save_',
        '_em_electron_diffraction.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '  This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_em_electron_diffraction.entry_id' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_electron_diffraction.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_electron_diffraction.id',
        [ [ 'item_',
            '_item_description.description',
            '   The value of _electron_diffraction.id must\n    uniquely identify the electron diffraction experiment.' ],
          [ 'item_', '_item.name', '_em_electron_diffraction.id' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_electron_diffraction.num_structure_factors',
        [ [ 'item_',
            '_item_description.description',
            '    The number of structure factors from the electron diffraction experiment.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction.num_structure_factors' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '12000' ] ] ],
      [ 'save_',
        '_em_electron_diffraction.num_diff_patterns',
        [ [ 'item_',
            '_item_description.description',
            '    The number of diffraction patterns used from the electron \n     diffraction experiment.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction.num_diff_patterns' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_em_electron_diffraction.details',
        [ [ 'item_',
            '_item_description.description',
            '   Details of the electron diffraction experiment' ],
          [ 'item_', '_item.name', '_em_electron_diffraction.details' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case', '_item_examples.detail' ],
            [ '1',
              ' THE MODEL WAS DERIVED USING ELECTRON DIFFRACTION\n  AND IMAGE DATA FROM TWO DIMENSIONAL CRYSTALS OF TUBULIN\n  INDUCED BY THE PRESENCE OF ZN++ IONS.\n  WHAT FOLLOWS ARE THE COORDINATES FOR THE AB-TUBULIN DIMER\n  BOUND TO TAXOL AS OBTAINED BY ELECTRON CRYSTALLOGRAPHY OF\n  ZINC-INDUCED SHEETS.  THIS IS THE UNREFINED MODEL, BUILT\n  INTO A RAW DENSITY MAP WHERE THE RESOLUTION IN THE PLANE\n  OF THE SHEET WAS 3.7 ANGSTROMS AND THAT PERPENDICULAR TO\n  THE SHEET ABOUT 4.8 ANGSTROMS.  THE MODEL DOES NOT CONTAIN\n  MOST OF THE C-TERMINAL RESIDUES OF EITHER MONOMER WHICH\n  WERE DISORDERED IN THE MAP.  THE LOOP BETWEEN HELIX H1 AND\n  STRAND S2, AND THAT BETWEEN H2 AND S3 ARE PRESENT FOR\n  COMPLETENESS BUT WERE BUILT INTO VERY WEAK DENSITY.\n  GIVEN THE LIMITED RESOLUTION OF THE MAP, THE CONFORMATION\n  OF THE SIDE CHAINS, ESPECIALLY THOSE CORRESPONDING TO\n  RESIDUES ON THE SURFACE OF THE DIMER, MUST BE TAKEN\n  CAUTIOUSLY.  IN ADDITION, BECAUSE THIS IS AN UNREFINED\n  MODEL, CERTAIN GEOMETRY ERRORS MAY STILL BE PRESENT IN THE\n  STRUCTURE.  PLEASE TAKE THIS INTO ACCOUNT WHEN\n  INTERPRETING YOUR OWN DATA BASED ON THE PRESENT TUBULIN\n  STRUCTURE.  ALTHOUGH THE POSITION OF RESIDUES (WITH THE\n  EXCEPTION OF THOSE IN THE LOOPS MENTIONED ABOVE) SHOULD\n  NOT CHANGE SIGNIFICANTLY UPON REFINEMENT, DRAWING\n  INFORMATION AT THE LEVEL OF SIDE CHAIN CONFORMATION IS\n  CLEARLY NOT ADVISED.  FINALLY, PLEASE NOTICE THAT THE\n  TAXOID IN THE MODEL IS THE TAXOL DERIVATIVE TAXOTERE.' ] ] ] ],
      [ '@comment', '##################################' ],
      [ '@comment', '# EM_ELECTRON_DIFFRACTION_PHASE ##' ],
      [ '@comment', '##################################' ],
      [ 'save_',
        'em_electron_diffraction_phase',
        [ [ 'item_',
            '_category.description',
            '   data items in the em_electron_diffraction_phase category\n    record details about the phase information\n    from the electron diffraction experiment.' ],
          [ 'item_', '_category.id', 'em_electron_diffraction_phase' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_electron_diffraction_phase.id',
              '_em_electron_diffraction_phase.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  example 1 - based on pdb entry 1tub and laboratory records for the\n                structure corresponding to pdb entry 1tub',
              ' \n  _em_electron_diffraction_phase.entry_id                       1TUB\n  _em_electron_diffraction_phase.id                             1\n  _em_electron_diffraction_phase.d_res_low                      5.0\n  _em_electron_diffraction_phase.d_res_high                     4.0\n  _em_electron_diffraction_phase.residual                        .\n  _em_electron_diffraction_phase.highest_resolution_shell_error  .\n  _em_electron_diffraction_phase.overall_error                   .\n  _em_electron_diffraction_phase.rejection_criteria_error        .' ] ] ] ],
      [ 'save_',
        '_em_electron_diffraction_phase.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '  this data item is a pointer to _entry.id in the entry category.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction_phase.entry_id' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction_phase' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_electron_diffraction_phase.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_electron_diffraction_phase.id',
        [ [ 'item_',
            '_item_description.description',
            '   the value of _electron_diffraction_phase.id must\n    uniquely identify the electron diffraction phase experiment.' ],
          [ 'item_', '_item.name', '_em_electron_diffraction_phase.id' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction_phase' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_electron_diffraction_phase.d_res_high',
        [ [ 'item_',
            '_item_description.description',
            '    the highest resolution d-value for the electron diffraction experiment.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction_phase.d_res_high' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction_phase' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '5.0' ] ] ],
      [ 'save_',
        '_em_electron_diffraction_phase.residual',
        [ [ 'item_',
            '_item_description.description',
            '    the phase residual value for the electron diffraction experiment.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction_phase.residual' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction_phase' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_em_electron_diffraction_phase.highest_resolution_shell_error',
        [ [ 'item_',
            '_item_description.description',
            '    the highest resolution shell error in degrees.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction_phase.highest_resolution_shell_error' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction_phase' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_em_electron_diffraction_phase.overall_error',
        [ [ 'item_',
            '_item_description.description',
            '    the overall phase error in degrees.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction_phase.overall_error' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction_phase' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_em_electron_diffraction_phase.rejection_criteria_error',
        [ [ 'item_',
            '_item_description.description',
            '    the rejection criteria (phase error) in degrees.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction_phase.rejection_criteria_error' ],
          [ 'item_', '_item.category_id', 'em_electron_diffraction_phase' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '####################################' ],
      [ '@comment', '# EM_ELECTRON_DIFFRACTION_PATTERN ##' ],
      [ '@comment', '####################################' ],
      [ 'save_',
        'em_electron_diffraction_pattern',
        [ [ 'item_',
            '_category.description',
            '   data items in the em_electron_diffraction_pattern category\n    record details about the pattern information\n    from the electron diffraction experiment.' ],
          [ 'item_', '_category.id', 'em_electron_diffraction_pattern' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_em_electron_diffraction_pattern.id',
              '_em_electron_diffraction_pattern.entry_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'em_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ '  example 1 - based on pdb entry 1tub and laboratory records for the\n                structure corresponding to pdb entry 1tub',
              ' \n  _em_electron_diffraction_pattern.entry_id                   1TUB\n  _em_electron_diffraction_pattern.id                         1\n  _em_electron_diffraction_pattern.num_patterns_by_tilt_angle 1\n  _em_electron_diffraction_pattern.num_images_by_tilt_angle   4\n  _em_electron_diffraction_pattern.tilt_angle                 .' ] ] ] ],
      [ 'save_',
        '_em_electron_diffraction_pattern.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '  this data item is a pointer to _entry.id in the entry category.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction_pattern.entry_id' ],
          [ 'item_',
            '_item.category_id',
            'em_electron_diffraction_pattern' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_em_electron_diffraction_pattern.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_em_electron_diffraction_pattern.id',
        [ [ 'item_',
            '_item_description.description',
            '   the value of _electron_diffraction_pattern.id must\n    uniquely identify the electron diffraction pattern experiment.' ],
          [ 'item_', '_item.name', '_em_electron_diffraction_pattern.id' ],
          [ 'item_',
            '_item.category_id',
            'em_electron_diffraction_pattern' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_em_electron_diffraction_pattern.num_patterns_by_tilt_angle',
        [ [ 'item_',
            '_item_description.description',
            '    the number of diffraction patterns by tilt angle.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction_pattern.num_patterns_by_tilt_angle' ],
          [ 'item_',
            '_item.category_id',
            'em_electron_diffraction_pattern' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '1' ] ] ],
      [ 'save_',
        '_em_electron_diffraction_pattern.num_images_by_tilt_angle',
        [ [ 'item_',
            '_item_description.description',
            '    the number of images by tilt angle.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction_pattern.num_images_by_tilt_angle' ],
          [ 'item_',
            '_item.category_id',
            'em_electron_diffraction_pattern' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '4' ] ] ],
      [ 'save_',
        '_em_electron_diffraction_pattern.tilt_angle',
        [ [ 'item_',
            '_item_description.description',
            '    the tilt angle at which the diffraction pattern was obtained.' ],
          [ 'item_',
            '_item.name',
            '_em_electron_diffraction_pattern.tilt_angle' ],
          [ 'item_',
            '_item.category_id',
            'em_electron_diffraction_pattern' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '## EOF  mmcif_iims-def-2.dic' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment',
        ' File:  mmcif_pdbx-def-8.dic                                             ' ],
      [ '@comment', ' ' ],
      [ '@comment',
        '                      PDB Exchange Data Dictionary' ],
      [ '@comment',
        ' This data dictionary contains definitions used by wwPDB for data exchange' ],
      [ '@comment', ' and data processing.  ' ],
      [ '@comment',
        '                           Definition Section 8' ],
      [ '@comment', '                            ' ],
      [ '@comment',
        '                                                                        ' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment', '###################' ],
      [ '@comment', '# RCSB_TABLEINFO ##' ],
      [ '@comment', '###################' ],
      [ 'save_',
        'rcsb_tableinfo',
        [ [ 'item_', '_category.description', '              ' ],
          [ 'item_', '_category.id', 'rcsb_tableinfo' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_rcsb_tableinfo.tablename' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'rcsb_group' ] ],
          [ 'item_',
            '_category_examples.case',
            '\n loop_\n _rcsb_tableinfo.tablename\n _rcsb_tableinfo.description\n _rcsb_tableinfo.type\n _rcsb_tableinfo.table_serial_no\n _rcsb_tableinfo.group_name\n _rcsb_tableinfo.WWW_Selection_Criteria\n _rcsb_tableinfo.WWW_Report_Criteria\n  summary \'summary data\' 1 1 STRUCTURE 1 1\n#   ...' ] ] ],
      [ 'save_',
        '_rcsb_tableinfo.tablename',
        [ [ 'item_',
            '_item_description.description',
            '              SQL table name.' ],
          [ 'item_', '_item.name', '_rcsb_tableinfo.tablename' ],
          [ 'item_', '_item.category_id', 'rcsb_tableinfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code30' ],
          [ 'item_', '_item_examples.case', 'structure_summary' ] ] ],
      [ 'save_',
        '_rcsb_tableinfo.description',
        [ [ 'item_',
            '_item_description.description',
            '              SQL table description.' ],
          [ 'item_', '_item.name', '_rcsb_tableinfo.description' ],
          [ 'item_', '_item.category_id', 'rcsb_tableinfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'Table of solvent coordinates' ] ] ],
      [ 'save_',
        '_rcsb_tableinfo.type',
        [ [ 'item_',
            '_item_description.description',
            '              SQL table type.' ],
          [ 'item_', '_item.name', '_rcsb_tableinfo.type' ],
          [ 'item_', '_item.category_id', 'rcsb_tableinfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_examples.case',
            '0=general, 1=coordinate, 2=derived, 3=schema' ] ] ],
      [ 'save_',
        '_rcsb_tableinfo.table_serial_no',
        [ [ 'item_',
            '_item_description.description',
            '              SQL table serial number.' ],
          [ 'item_', '_item.name', '_rcsb_tableinfo.table_serial_no' ],
          [ 'item_', '_item.category_id', 'rcsb_tableinfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '1,2,3,4,...' ] ] ],
      [ 'save_',
        '_rcsb_tableinfo.column_serial_no',
        [ [ 'item_',
            '_item_description.description',
            '              SQL column serial number.' ],
          [ 'item_', '_item.name', '_rcsb_tableinfo.column_serial_no' ],
          [ 'item_', '_item.category_id', 'rcsb_tableinfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '1,2,3,4,...' ] ] ],
      [ 'save_',
        '_rcsb_tableinfo.group_name',
        [ [ 'item_',
            '_item_description.description',
            '              SQL table group name..' ],
          [ 'item_', '_item.name', '_rcsb_tableinfo.group_name' ],
          [ 'item_', '_item.category_id', 'rcsb_tableinfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_',
            '_item_examples.case',
            'STRUCTURE FEATURES, SOURCE ORGANISM' ] ] ],
      [ 'save_',
        '_rcsb_tableinfo.WWW_Selection_Criteria',
        [ [ 'item_',
            '_item_description.description',
            '              SQL table visibility in WWW selection querires.' ],
          [ 'item_',
            '_item.name',
            '_rcsb_tableinfo.WWW_Selection_Criteria' ],
          [ 'item_', '_item.category_id', 'rcsb_tableinfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '0=no, 1=yes' ] ] ],
      [ 'save_',
        '_rcsb_tableinfo.WWW_Report_Criteria',
        [ [ 'item_',
            '_item_description.description',
            '              SQL table visibility in WWW reports queries.' ],
          [ 'item_', '_item.name', '_rcsb_tableinfo.WWW_Report_Criteria' ],
          [ 'item_', '_item.category_id', 'rcsb_tableinfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '0=no, 1=yes' ] ] ],
      [ '@comment', '####################' ],
      [ '@comment', '# RCSB_COLUMNINFO ##' ],
      [ '@comment', '####################' ],
      [ 'save_',
        'rcsb_columninfo',
        [ [ 'item_', '_category.description', '              ' ],
          [ 'item_', '_category.id', 'rcsb_columninfo' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_rcsb_columninfo.tablename', '_rcsb_columninfo.columnname' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'rcsb_group' ] ],
          [ 'item_',
            '_category_examples.case',
            '\n loop_\n _rcsb_columninfo.tablename\n _rcsb_columninfo.columnname\n _rcsb_columninfo.description\n _rcsb_columninfo.example\n _rcsb_columninfo.type\n _rcsb_columninfo.table_serial_no\n _rcsb_columninfo.column_serial_no\n _rcsb_columninfo.WWW_Selection_Criteria\n _rcsb_columninfo.WWW_Report_Criteria\n  summary  id \'id code\' \'id1, id2\' 1 1 1 1 1\n#   ...' ] ] ],
      [ 'save_',
        '_rcsb_columninfo.columnname',
        [ [ 'item_',
            '_item_description.description',
            '              SQL column name.' ],
          [ 'item_', '_item.name', '_rcsb_columninfo.columnname' ],
          [ 'item_', '_item.category_id', 'rcsb_columninfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code30' ],
          [ 'item_', '_item_examples.case', 'id' ] ] ],
      [ 'save_',
        '_rcsb_columninfo.tablename',
        [ [ 'item_',
            '_item_description.description',
            '              SQL table name.' ],
          [ 'item_', '_item.name', '_rcsb_columninfo.tablename' ],
          [ 'item_', '_item.category_id', 'rcsb_columninfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code30' ],
          [ 'item_', '_item_examples.case', 'structure_summary' ] ] ],
      [ 'save_',
        '_rcsb_columninfo.description',
        [ [ 'item_',
            '_item_description.description',
            '              SQL column description.' ],
          [ 'item_', '_item.name', '_rcsb_columninfo.description' ],
          [ 'item_', '_item.category_id', 'rcsb_columninfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'Table of solvent coordinates' ] ] ],
      [ 'save_',
        '_rcsb_columninfo.example',
        [ [ 'item_',
            '_item_description.description',
            '              SQL column example.' ],
          [ 'item_', '_item.name', '_rcsb_columninfo.example' ],
          [ 'item_', '_item.category_id', 'rcsb_columninfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            'Table of solvent coordinates' ] ] ],
      [ 'save_',
        '_rcsb_columninfo.type',
        [ [ 'item_',
            '_item_description.description',
            '              SQL column type.' ],
          [ 'item_', '_item.name', '_rcsb_columninfo.type' ],
          [ 'item_', '_item.category_id', 'rcsb_columninfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_examples.case',
            '1:integer, 2:float, 3:string-single-left, 4:string-single-right, 5:string-multi-left, \n 6:string-multi-right, 7:angle, 8:boolean, 9:single character, \n 10:author or atom name column, 11: Date' ] ] ],
      [ 'save_',
        '_rcsb_columninfo.table_serial_no',
        [ [ 'item_',
            '_item_description.description',
            '              SQL table serial number.' ],
          [ 'item_', '_item.name', '_rcsb_columninfo.table_serial_no' ],
          [ 'item_', '_item.category_id', 'rcsb_columninfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '1,2,3,4,...' ] ] ],
      [ 'save_',
        '_rcsb_columninfo.column_serial_no',
        [ [ 'item_',
            '_item_description.description',
            '              SQL column serial number.' ],
          [ 'item_', '_item.name', '_rcsb_columninfo.column_serial_no' ],
          [ 'item_', '_item.category_id', 'rcsb_columninfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '1,2,3,4,...' ] ] ],
      [ 'save_',
        '_rcsb_columninfo.WWW_Selection_Criteria',
        [ [ 'item_',
            '_item_description.description',
            '              SQL column visibility in WWW selection querires.' ],
          [ 'item_',
            '_item.name',
            '_rcsb_columninfo.WWW_Selection_Criteria' ],
          [ 'item_', '_item.category_id', 'rcsb_columninfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '0=no, 1=yes' ] ] ],
      [ 'save_',
        '_rcsb_columninfo.WWW_Report_Criteria',
        [ [ 'item_',
            '_item_description.description',
            '              SQL column visibility in WWW reports queries.' ],
          [ 'item_', '_item.name', '_rcsb_columninfo.WWW_Report_Criteria' ],
          [ 'item_', '_item.category_id', 'rcsb_columninfo' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_examples.case', '0=no, 1=yes' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '## EOF mmcif_pdbx-def-8.dic' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment',
        ' File:  mmcif_pdbx-def-9.dic                                             ' ],
      [ '@comment', ' ' ],
      [ '@comment',
        '                      PDB Exchange Data Dictionary' ],
      [ '@comment',
        ' This data dictionary contains definitions used by RCSB for describing' ],
      [ '@comment', ' derived data and computation.' ],
      [ '@comment',
        '                           Definition Section 9' ],
      [ '@comment', '                            ' ],
      [ '@comment',
        '                                                                        ' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment', '################' ],
      [ '@comment', '# PDBX_DOMAIN ##' ],
      [ '@comment', '################' ],
      [ 'save_',
        'pdbx_domain',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_DOMAIN category record information\n               about domain definitions.\n\n               A domain need not correspond to a completely polypeptide chain;\n               it can be composed of one or more segments in a single chain,\n               or by segments from more than one chain.' ],
          [ 'item_', '_category.id', 'pdbx_domain' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_domain.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'pdbx_erf_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n     loop_\n    _pdbx_domain.id   \n    _pdbx_domain.details\n     d1  \'Chains A, B\'\n     d2  \'Asym_id D Residues 1-134\'',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_domain.details',
        [ [ 'item_',
            '_item_description.description',
            '              A description of special aspects of the structural elements that\n               comprise a domain.' ],
          [ 'item_', '_item.name', '_pdbx_domain.details' ],
          [ 'item_', '_item.category_id', 'pdbx_domain' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '                                 The loop between residues 18 and 23.' ] ] ],
      [ 'save_',
        '_pdbx_domain.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_domain.id must uniquely identify a\n               record in the PDBX_DOMAIN list.\n\n               Note that this item need not be a number; it can be any unique\n               identifier.' ],
          [ 'item_', '_item.name', '_pdbx_domain.id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ '@comment', '######################' ],
      [ '@comment', '# PDBX_DOMAIN_RANGE ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'pdbx_domain_range',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_DOMAIN_RANGE category identify the\n               beginning and ending points of polypeptide chain segments\n               that form all or part of a domain.' ],
          [ 'item_', '_category.id', 'pdbx_domain_range' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_domain_range.domain_id',
              '_pdbx_domain_range.beg_label_alt_id',
              '_pdbx_domain_range.beg_label_asym_id',
              '_pdbx_domain_range.beg_label_comp_id',
              '_pdbx_domain_range.beg_label_seq_id',
              '_pdbx_domain_range.end_label_alt_id',
              '_pdbx_domain_range.end_label_asym_id',
              '_pdbx_domain_range.end_label_comp_id',
              '_pdbx_domain_range.end_label_seq_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'pdbx_erf_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n     loop_\n    _pdbx_domain_range.domain_id                \n    _pdbx_domain_range.beg_label_alt_id      \n    _pdbx_domain_range.beg_label_asym_id     \n    _pdbx_domain_range.beg_label_comp_id     \n    _pdbx_domain_range.beg_label_seq_id      \n    _pdbx_domain_range.end_label_alt_id      \n    _pdbx_domain_range.end_label_asym_id\n    _pdbx_domain_range.end_label_comp_id\n    _pdbx_domain_range.end_label_seq_id \n     d1   .  A PRO  1  . A GLY  29 \n     d1   .  B PRO  31 . B GLY  59 \n     d1   .  C PRO  61 . B GLY  89\n     d2   .  D PRO  91 . D GLY 119 \n     d2   .  E PRO 121 . E GLY 149\n     d2   .  F PRO 151 . F GLY 179',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_domain_range.beg_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.beg_label_alt_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.beg_label_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_alt_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.beg_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.beg_label_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.beg_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.beg_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.beg_label_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.beg_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.beg_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.beg_label_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.beg_label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.beg_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.beg_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.beg_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.beg_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.beg_auth_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.beg_auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.beg_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain begins.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.beg_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.beg_auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.domain_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _pdbx_domain.id in the\n               PDBX_DOMAIN category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.domain_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.domain_id' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_domain.id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.end_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.end_label_alt_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.end_label_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_alt_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.end_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.end_label_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.end_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.end_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.end_label_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.end_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.end_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.end_label_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.end_label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.end_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.end_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.end_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.end_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.end_auth_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.end_auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_domain_range.end_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the domain ends.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_domain_range.end_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_domain_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_domain_range.end_auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# PDBX_SEQUENCE_RANGE ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'pdbx_sequence_range',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_SEQUENCE_RANGE category identify the\n               beginning and ending points of polypeptide sequence segments.' ],
          [ 'item_', '_category.id', 'pdbx_sequence_range' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_sequence_range.seq_range_id',
              '_pdbx_sequence_range.beg_label_alt_id',
              '_pdbx_sequence_range.beg_label_asym_id',
              '_pdbx_sequence_range.beg_label_comp_id',
              '_pdbx_sequence_range.beg_label_seq_id',
              '_pdbx_sequence_range.end_label_alt_id',
              '_pdbx_sequence_range.end_label_asym_id',
              '_pdbx_sequence_range.end_label_comp_id',
              '_pdbx_sequence_range.end_label_seq_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'pdbx_erf_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n     loop_\n    _pdbx_sequence_range.seq_range_id                \n    _pdbx_sequence_range.beg_label_alt_id      \n    _pdbx_sequence_range.beg_label_asym_id     \n    _pdbx_sequence_range.beg_label_comp_id     \n    _pdbx_sequence_range.beg_label_seq_id      \n    _pdbx_sequence_range.end_label_alt_id      \n    _pdbx_sequence_range.end_label_asym_id\n    _pdbx_sequence_range.end_label_comp_id\n    _pdbx_sequence_range.end_label_seq_id \n     s1   .  A PRO  1  . A GLY  29 \n     s2   .  D PRO  91 . D GLY 119 \n',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.beg_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range begins.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.beg_label_alt_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.beg_label_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_alt_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.beg_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range begins.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.beg_label_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.beg_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.beg_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range begins.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.beg_label_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.beg_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.beg_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range begins.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.beg_label_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.beg_label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.beg_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range begins.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.beg_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.beg_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.beg_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range begins.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.beg_auth_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.beg_auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.beg_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range begins.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_sequence_range.beg_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.beg_auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.seq_range_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is an identifier for a sequence range.' ],
          [ 'item_', '_item.name', '_pdbx_sequence_range.seq_range_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.end_label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range ends.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.end_label_alt_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.end_label_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_alt_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.end_label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range ends.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.end_label_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.end_label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.end_label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range ends.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.end_label_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.end_label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.end_label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range ends.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.end_label_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.end_label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.end_auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range ends.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.end_auth_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.end_auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.end_auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range ends.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_sequence_range.end_auth_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.end_auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_sequence_range.end_auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer at which this\n               segment of the sequence range ends.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_sequence_range.end_auth_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_sequence_range.end_auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ '@comment', '#######################' ],
      [ '@comment', '# PDBX_FEATURE_ENTRY ##' ],
      [ '@comment', '#######################' ],
      [ 'save_',
        'pdbx_feature_entry',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_FEATURE_ENTRY category records \n               information about properties pertaining to this \n               structure entry.' ],
          [ 'item_', '_category.id', 'pdbx_feature_entry' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_feature_entry.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'pdbx_erf_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - Gene Ontology data by entry ',
              '\n     loop_\n    _pdbx_feature_entry.id   \n    _pdbx_feature_entry.feature_name\n    _pdbx_feature_entry.feature\n    _pdbx_feature_entry.feature_type\n    _pdbx_feature_entry.feature_assigned_by\n    _pdbx_feature_entry.feature_citation_id\n     1  \'molecular function\'  \'DNA binding activity\' \n          value   GO  GO\n     2  \'biological process\'  \'regulation of transcription, DNA-dependent\' \n          value   GO  GO',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_feature_entry.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_entry.id uniquely identifies a \n               feature in the PDBX_FEATURE_ENTRY category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_entry.id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_entry' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_feature_entry.feature_name',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_entry.feature_name identifies a feature\n                by name.' ],
          [ 'item_', '_item.name', '_pdbx_feature_entry.feature_name' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_entry' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_entry.feature_type',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_entry.feature_type identifies the\n                type of feature.' ],
          [ 'item_', '_item.name', '_pdbx_feature_entry.feature_type' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_entry' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_enumeration.value' ], [ 'value', 'uri' ] ] ] ],
      [ 'save_',
        '_pdbx_feature_entry.feature',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_entry.feature_name.' ],
          [ 'item_', '_item.name', '_pdbx_feature_entry.feature' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_entry' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_entry.feature_identifier',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_entry.feature_identifier is an \n               additional identifier used to identify or \n               accession this feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_entry.feature_identifier' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_entry' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_entry.feature_assigned_by',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_entry.feature_assigned_by identifies \n                the individual, organization or program that \n                assigned the feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_entry.feature_assigned_by' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_entry' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_entry.feature_citation_id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_entry.feature_citation_id is a \n                reference to a citation in the CITATION category' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_entry.feature_citation_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_entry' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_entry.feature_citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ] ] ],
      [ 'save_',
        '_pdbx_feature_entry.feature_software_id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_entry.feature_software_id is a \n                reference to an application described in the \n                SOFTWARE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_entry.feature_software_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_entry' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_entry.feature_software_id' ],
          [ 'item_', '_item_linked.parent_name', '_software.name' ] ] ],
      [ '@comment', '########################' ],
      [ '@comment', '# PDBX_FEATURE_DOMAIN ##' ],
      [ '@comment', '########################' ],
      [ 'save_',
        'pdbx_feature_domain',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_FEATURE_DOMAIN category records \n               information about properties pertaining to this structure \n               domain.' ],
          [ 'item_', '_category.id', 'pdbx_feature_domain' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_feature_domain.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'pdbx_erf_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - SCOP data for PDB Entry 1KIP domain d1kipa_',
              '\n     loop_\n    _pdbx_feature_domain.id   \n    _pdbx_feature_domain.domain_id\n    _pdbx_feature_domain.feature_name\n    _pdbx_feature_domain.feature\n    _pdbx_feature_domain.feature_type\n    _pdbx_feature_domain.feature_assigned_by\n    _pdbx_feature_domain.feature_citation_id\n     1  \'d1kipa_\'  class \'All beta proteins\' \n                   value SCOP  scop\n     2  \'d1kipa_\'  fold  \'Immunoglobulin-like beta-sandwich\' \n                   value SCOP  scop\n\n     3  \'d1kipa_\'  superfamily  \'Immunoglobulin\' \n                   value SCOP  scop\n\n     4  \'d1kipa_\'  family  \'V set domains (antibody variable domain-like)\' \n                   value SCOP  scop\n\n     5  \'d1kipa_\'  domain  \'Immunoglobulin light chain kappa variable domain\'\n                   value SCOP  scop\n\n     6  \'d1kipa_\'  species  \'Mouse (Mus musculus), cluster 4\'\n                   value SCOP  scop\n\n',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_feature_domain.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_domain.id uniquely identifies \n               a feature in the PDBX_FEATURE_DOMAIN category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_domain.id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_domain' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_feature_domain.domain_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_domain.id references a domain \n               definition in category PDBX_DOMAIN.' ],
          [ 'item_', '_item.name', '_pdbx_feature_domain.domain_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_domain' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_domain.domain_id' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_domain.id' ] ] ],
      [ 'save_',
        '_pdbx_feature_domain.feature_name',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_domain.feature_name identifies a feature\n                by name.' ],
          [ 'item_', '_item.name', '_pdbx_feature_domain.feature_name' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_domain' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_domain.feature_type',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_domain.feature_type identifies the\n                type of feature.' ],
          [ 'item_', '_item.name', '_pdbx_feature_domain.feature_type' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_domain' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_enumeration.value' ], [ 'value', 'uri' ] ] ] ],
      [ 'save_',
        '_pdbx_feature_domain.feature',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_domain.feature_name.' ],
          [ 'item_', '_item.name', '_pdbx_feature_domain.feature' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_domain' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_domain.feature_identifier',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_domain.feature_identifier is an \n               additional identifier used to identify or \n               accession this feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_domain.feature_identifier' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_domain' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_domain.feature_assigned_by',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_domain.feature_assigned_by identifies \n                the individual, organization or program that \n                assigned the feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_domain.feature_assigned_by' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_domain' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_domain.feature_citation_id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_domain.feature_citation_id is a \n                reference to a citation in the CITATION category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_domain.feature_citation_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_domain' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_domain.feature_citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ] ] ],
      [ 'save_',
        '_pdbx_feature_domain.feature_software_id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_domain.feature_software_id is a \n                reference to an application described in the \n                SOFTWARE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_domain.feature_software_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_domain' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_domain.feature_software_id' ],
          [ 'item_', '_item_linked.parent_name', '_software.name' ] ] ],
      [ '@comment', '################################' ],
      [ '@comment', '# PDBX_FEATURE_SEQUENCE_RANGE ##' ],
      [ '@comment', '################################' ],
      [ 'save_',
        'pdbx_feature_sequence_range',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_FEATURE_SEQUENCE_RANGE category \n               records information about properties pertaining to \n               this structure sequence_range.' ],
          [ 'item_', '_category.id', 'pdbx_feature_sequence_range' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_',
            '_category_key.name',
            '_pdbx_feature_sequence_range.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'pdbx_erf_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - Secondary structure computed by program DSSP.',
              '\n     loop_\n    _pdbx_feature_sequence_range.id   \n    _pdbx_feature_sequence_range.seq_range_id\n    _pdbx_feature_sequence_range.feature_name\n    _pdbx_feature_sequence_range.feature\n    _pdbx_feature_sequence_range.feature_type\n    _pdbx_feature_sequence_range.feature_assigned_by\n    _pdbx_feature_sequence_range.feature_software_id\n      1  H1 \'secondary structure\' \'4-helix (alpha-helix)\'               \n                     value  DSSP DSSP\n      2  T1 \'secondary structure\' \'hydrogen-bonded turn in beta-ladder\' \n                     value  DSSP DSSP',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_feature_sequence_range.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_sequence_range.id uniquely identifies \n               a feature in the PDBX_FEATURE_SEQUENCE_RANGE category' ],
          [ 'item_', '_item.name', '_pdbx_feature_sequence_range.id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_feature_sequence_range.seq_range_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_sequence_range.seq_range_id\n               references a sequence_range definition in category \n               PDBX_SEQUENCE_RANGE.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_sequence_range.seq_range_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_sequence_range.seq_range_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_sequence_range.seq_range_id' ] ] ],
      [ 'save_',
        '_pdbx_feature_sequence_range.feature_name',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_sequence_range.feature_name identifies a feature\n                by name.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_sequence_range.feature_name' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_sequence_range.feature_type',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_sequence_range.feature_type identifies the\n                type of feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_sequence_range.feature_type' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_enumeration.value' ], [ 'value', 'uri' ] ] ] ],
      [ 'save_',
        '_pdbx_feature_sequence_range.feature',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_sequence_range.feature_name.' ],
          [ 'item_', '_item.name', '_pdbx_feature_sequence_range.feature' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_sequence_range.feature_identifier',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_sequence_range.feature_identifier is an \n               additional identifier used to identify or \n               accession this feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_sequence_range.feature_identifier' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_sequence_range.feature_assigned_by',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_sequence_range.feature_assigned_by identifies \n                the individual, organization or program that assigned \n                the feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_sequence_range.feature_assigned_by' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_sequence_range.feature_citation_id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_sequence_range.feature_citation_id is a \n                reference to a citation in the CITATION category' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_sequence_range.feature_citation_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_sequence_range.feature_citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ] ] ],
      [ 'save_',
        '_pdbx_feature_sequence_range.feature_software_id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_sequence_range.feature_software_id is a \n                reference to an application descripted in the \n                SOFTWARE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_sequence_range.feature_software_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_sequence_range' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_sequence_range.feature_software_id' ],
          [ 'item_', '_item_linked.parent_name', '_software.name' ] ] ],
      [ '@comment', '##########################' ],
      [ '@comment', '# PDBX_FEATURE_ASSEMBLY ##' ],
      [ '@comment', '##########################' ],
      [ 'save_',
        'pdbx_feature_assembly',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_FEATURE_ASSEMBLY category records \n               information about properties pertaining to this \n               structural assembly.' ],
          [ 'item_', '_category.id', 'pdbx_feature_assembly' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_feature_assembly.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'pdbx_erf_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - Biological process for functional assembly  ',
              '\n     loop_\n    _pdbx_feature_assembly.id   \n    _pdbx_feature_assembly.assembly_id\n    _pdbx_feature_assembly.feature_name\n    _pdbx_feature_assembly.feature\n    _pdbx_feature_assembly.feature_type\n    _pdbx_feature_assembly.feature_assigned_by\n    _pdbx_feature_assembly.feature_citation_id\n        1 b1  \'biological process\'  \'nitrogen metabolism\' value GO GO',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_feature_assembly.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_assembly.id uniquely identifies a \n               feature in the PDBX_FEATURE_ASSEMBLY category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_assembly.id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_feature_assembly.assembly_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_assembly.assembly_id references an\n               assembly definition in category STRUCT_BIOL' ],
          [ 'item_', '_item.name', '_pdbx_feature_assembly.assembly_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_assembly.assembly_id' ],
          [ 'item_', '_item_linked.parent_name', '_struct_biol.id' ] ] ],
      [ 'save_',
        '_pdbx_feature_assembly.feature_name',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_assembly.feature_name identifies a feature\n                by name.' ],
          [ 'item_', '_item.name', '_pdbx_feature_assembly.feature_name' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_assembly.feature_type',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_assembly.feature_type identifies the\n                type of feature.' ],
          [ 'item_', '_item.name', '_pdbx_feature_assembly.feature_type' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_enumeration.value' ], [ 'value', 'uri' ] ] ] ],
      [ 'save_',
        '_pdbx_feature_assembly.feature',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_assembly.feature_name.' ],
          [ 'item_', '_item.name', '_pdbx_feature_assembly.feature' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_assembly.feature_identifier',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_assembly_range.feature_identifier is an \n               additional identifier used to identify or \n               accession this feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_assembly.feature_identifier' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_assembly.feature_assigned_by',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_assembly.feature_assigned_by identifies \n                the individual, organization or program that assigned \n                the feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_assembly.feature_assigned_by' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_assembly' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_assembly.feature_citation_id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_assembly.feature_citation_id is a \n                reference to a citation in the CITATION category' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_assembly.feature_citation_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_assembly.feature_citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ] ] ],
      [ 'save_',
        '_pdbx_feature_assembly.feature_software_id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_assembly.feature_software_id is a \n                reference to an application described in the \n                SOFTWARE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_assembly.feature_software_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_assembly' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_assembly.feature_software_id' ],
          [ 'item_', '_item_linked.parent_name', '_software.name' ] ] ],
      [ '@comment', '#########################' ],
      [ '@comment', '# PDBX_FEATURE_MONOMER ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'pdbx_feature_monomer',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_FEATURE_MONOMER category records \n               information about properties pertaining to particular\n               monomers in this structure.' ],
          [ 'item_', '_category.id', 'pdbx_feature_monomer' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_feature_monomer.id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group', 'pdbx_erf_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n     loop_\n    _pdbx_feature_monomer.id   \n    _pdbx_feature_monomer.label_alt_id\n    _pdbx_feature_monomer.label_asym_id\n    _pdbx_feature_monomer.label_comp_id\n    _pdbx_feature_monomer.label_seq_id\n    _pdbx_feature_monomer.feature_name\n    _pdbx_feature_monomer.feature\n    _pdbx_feature_monomer.feature_type\n    _pdbx_feature_monomer.feature_assigned_by\n    _pdbx_feature_monomer.feature_citation_id\n     1 . A ASP 1 \'SASA\'  129.4  value POPS  pops\n     1 . A ILE 2 \'SASA\'   35.5  value POPS  pops\n     1 . A VAL 3 \'SASA\'   87.2  value POPS  pops',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_monomer.id uniquely identifies \n               a feature in the PDBX_FEATURE_MONOMER category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.feature_name',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_monomer.feature_name identifies a feature\n                by name.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.feature_name' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.feature_type',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_monomer.feature_type identifies the\n                type of feature.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.feature_type' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_', [ '_item_enumeration.value' ], [ 'value', 'uri' ] ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.feature',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_feature_monomer.feature_name.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.feature' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.feature_identifier',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_monomer.feature_identifier is an \n               additional identifier used to identify or \n               accession this feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_monomer.feature_identifier' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.feature_assigned_by',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_monomer.feature_assigned_by identifies \n                the individual, organization or program that \n                assigned the feature.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_monomer.feature_assigned_by' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.feature_citation_id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_monomer.feature_citation_id is a \n                reference to a citation in the CITATION category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_monomer.feature_citation_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_monomer.feature_citation_id' ],
          [ 'item_', '_item_linked.parent_name', '_citation.id' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.feature_software_id',
        [ [ 'item_',
            '_item_description.description',
            '              _pdbx_feature_monomer.feature_software_id is a \n                reference to an application described in the \n                SOFTWARE category.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_feature_monomer.feature_software_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_monomer.feature_software_id' ],
          [ 'item_', '_item_linked.parent_name', '_software.name' ] ] ],
      [ '@comment', '#' ],
      [ 'save_',
        '_pdbx_feature_monomer.label_alt_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_sites_alt.id in the\n               ATOM_SITES_ALT category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.label_alt_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_monomer.label_alt_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_alt_id' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.label_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.label_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.label_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_monomer.label_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.label_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.label_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.label_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_monomer.label_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.label_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.label_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.label_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_monomer.label_seq_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.label_seq_id' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.auth_asym_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.auth_asym_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.auth_asym_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_monomer.auth_asym_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_asym_id' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.auth_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.auth_comp_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.auth_comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_monomer.auth_comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_atom_site.auth_comp_id' ] ] ],
      [ 'save_',
        '_pdbx_feature_monomer.auth_seq_id',
        [ [ 'item_',
            '_item_description.description',
            '              A component of the identifier for the monomer.\n\n               This data item is a pointer to _atom_site.auth_seq_id in the\n               ATOM_SITE category.' ],
          [ 'item_', '_item.name', '_pdbx_feature_monomer.auth_seq_id' ],
          [ 'item_', '_item.category_id', 'pdbx_feature_monomer' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_feature_monomer.auth_seq_id' ],
          [ 'item_', '_item_linked.parent_name', '_atom_site.auth_seq_id' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '# End of file:  mmcif_pdbx-def-9.dic' ],
      [ '@comment', '#' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment',
        ' File:  mmcif_pdbx-def-10.dic                                             ' ],
      [ '@comment', ' ' ],
      [ '@comment',
        '                      PDB Exchange Data Dictionary' ],
      [ '@comment',
        ' This data dictionary contains preliminary extensions for describing ' ],
      [ '@comment',
        ' macromolecular powder diffraction experiments.' ],
      [ '@comment',
        '                           Definition Section 10' ],
      [ '@comment', '                            ' ],
      [ '@comment',
        '                                                                        ' ],
      [ '@comment',
        '##########################################################################' ],
      [ 'save_',
        '_diffrn_radiation.pdbx_analyzer',
        [ [ 'item_',
            '_item_description.description',
            ' Indicates the method used to obtain monochromatic radiation.\n  _diffrn_radiation.monochromator describes the primary beam\n  monochromator (pre-specimen monochromation). \n  _diffrn_radiation.pdbx_analyzer specifies the\n  post-diffraction analyser (post-specimen) monochromation.\n  Note that monochromators may have either \'parallel\' or\n  \'antiparallel\' orientation. It is assumed that the\n  geometry is parallel unless specified otherwise.\n  In a parallel geometry, the position of the monochromator\n  allows the incident beam and the final post-specimen\n  and post-monochromator beam to be as close to parallel\n  as possible. In a parallel geometry, the diffracting\n  planes in the specimen and monochromator will be parallel\n  when 2*theta(monochromator) is equal to 2*theta (specimen).\n  For further discussion see R. Jenkins and R. Snyder,\n  Introduction to X-ray Powder Diffraction, Wiley (1996),\n  pp. 164-5.' ],
          [ 'item_', '_item.name', '_diffrn_radiation.pdbx_analyzer' ],
          [ 'item_', '_item.category_id', 'diffrn_radiation' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'GE(111)',
              'Zr filter',
              'Ge 220',
              'none',
              'equatorial mounted graphite (0001)',
              'Si (111), antiparallel' ] ],
          [ 'item_', '_item_aliases.dictionary', 'cif_pd.dic' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pd_instr_monochr_post_spec' ] ] ],
      [ 'save_',
        'pdbx_exptl_pd',
        [ [ 'item_',
            '_category.description',
            '              Data items in the pdbx_exptl_pd record information about\n               powder sample preparations.' ],
          [ 'item_', '_category.id', 'pdbx_exptl_pd' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'item_', '_category_key.name', '_pdbx_exptl_pd.entry_id' ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'pdbx_group' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_pd.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '              The value of _pdbx_exptl_pd.entry_id uniquely identifies a \n               record in the PDBX_EXPTL_PD category.' ],
          [ 'item_', '_item.name', '_pdbx_exptl_pd.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_pd' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_', '_item_linked.child_name', '_pdbx_exptl_pd.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_pdbx_exptl_pd.spec_preparation_pH',
        [ [ 'item_',
            '_item_description.description',
            '  The pH at which the powder sample was prepared.' ],
          [ 'item_', '_item.name', '_pdbx_exptl_pd.spec_preparation_pH' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_pd' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_exptl_pd.spec_preparation_pH_range',
        [ [ 'item_',
            '_item_description.description',
            ' The range of pH values at which the sample was prepared.   Used when \n  a point estimate of pH is not appropriate.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_exptl_pd.spec_preparation_pH_range' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_pd' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_', [ '_item_examples.case' ], [ '5.6 - 6.4' ] ] ] ],
      [ 'save_',
        '_pdbx_exptl_pd.spec_preparation',
        [ [ 'item_',
            '_item_description.description',
            ' A description of preparation steps for producing the\n  diffraction specimen from the sample. Include any procedures\n  related to grinding, sieving, spray drying, etc.' ],
          [ 'item_', '_item.name', '_pdbx_exptl_pd.spec_preparation' ],
          [ 'item_', '_item.category_id', 'pdbx_exptl_pd' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'POLYCRYSTAL SLURRY',
              'wet grinding in acetone',
              'sieved through a 44 micron (325 mesh/inch) sieve',
              'spray dried in water with 1% clay' ] ],
          [ 'item_', '_item_aliases.dictionary', 'cif_pd.dic' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_', '_item_aliases.alias_name', '_pd_spec_preparation' ] ] ],
      [ 'save_',
        '_refine.pdbx_pd_number_of_powder_patterns',
        [ [ 'item_',
            '_item_description.description',
            '     The total number of powder patterns used.' ],
          [ 'item_',
            '_item.name',
            '_refine.pdbx_pd_number_of_powder_patterns' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_refine.pdbx_pd_number_of_points',
        [ [ 'item_',
            '_item_description.description',
            '     The total number of data points in the processed diffractogram.' ],
          [ 'item_', '_item.name', '_refine.pdbx_pd_number_of_points' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_pd.dic' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pd_proc_number_of_points' ] ] ],
      [ 'save_',
        '_refine.pdbx_pd_meas_number_of_points',
        [ [ 'item_',
            '_item_description.description',
            '     The total number of points in the measured\n      diffractogram.' ],
          [ 'item_',
            '_item.name',
            '_refine.pdbx_pd_meas_number_of_points' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_pd.dic' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pd_meas_number_of_points' ] ] ],
      [ 'save_',
        '_refine.pdbx_pd_proc_ls_prof_R_factor',
        [ [ 'item_',
            '_item_description.description',
            '      Rietveld/Profile fit R factors.\n       Note that the R factor computed for Rietveld refinements\n       using the extracted reflection intensity values (often\n       called the Rietveld or Bragg R factor, R~B~) is not properly\n       a profile R factor.  \n       _refine.pdbx_pd_proc_ls_prof_R_factor, often called R~p~, is an\n       unweighted fitness metric for the agreement between the\n       observed and computed diffraction patterns\n\n        R~p~ = sum~i~ | I~obs~(i) - I~calc~(i) |\n               / sum~i~ ( I~obs~(i) )\n\n   Note that in the above equations,\n\n   w(i) is the weight for the ith data point \n   I~obs~(i) is the observed intensity for the ith data\n             point, sometimes referred to as y~i~(obs) or\n             y~oi~.\n   I~calc~(i) is the computed intensity for the ith data\n              point with background and other corrections\n              applied to match the scale of the observed dataset,\n              sometimes referred to as y~i~(calc) or\n              y~ci~.\n   n is the total number of data points (see _refine.pdbx_pd_number_of_points)\n     less the number of data points excluded from the refinement.\n   p is the total number of refined parameters.' ],
          [ 'item_',
            '_item.name',
            '_refine.pdbx_pd_proc_ls_prof_R_factor' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_pd.dic' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pd_proc_ls_prof_R_factor' ] ] ],
      [ 'save_',
        '_refine.pdbx_pd_proc_ls_prof_wR_factor',
        [ [ 'item_',
            '_item_description.description',
            '      Rietveld/Profile fit R factors.\n       Note that the R factor computed for Rietveld refinements\n       using the extracted reflection intensity values (often\n       called the Rietveld or Bragg R factor, R~B~) is not properly\n       a profile R factor.\n       pdbx_pd_proc_ls_prof_wR_factor often called R~wp~, is a\n       weighted fitness metric for the agreement between the\n       observed and computed diffraction patterns\n       R~wp~ = SQRT {\n             sum~i~ ( w(i) [ I~obs~(i) - I~calc~(i) ]^2^ )\n           / sum~i~ ( w(i) [I~obs~(i)]^2^ ) }\n   Note that in the above equations,\n   w(i) is the weight for the ith data point \n   I~obs~(i) is the observed intensity for the ith data\n             point, sometimes referred to as y~i~(obs) or\n             y~oi~.\n   I~calc~(i) is the computed intensity for the ith data\n              point with background and other corrections\n              applied to match the scale of the observed dataset,\n              sometimes referred to as y~i~(calc) or\n              y~ci~.\n   n is the total number of data points (see _refine.pdbx_pd_number_of_points)\n     less the number of data points excluded from the refinement.\n   p is the total number of refined parameters.' ],
          [ 'item_',
            '_item.name',
            '_refine.pdbx_pd_proc_ls_prof_wR_factor' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_aliases.dictionary', 'cif_pd.dic' ],
          [ 'item_', '_item_aliases.version', '1.0' ],
          [ 'item_',
            '_item_aliases.alias_name',
            '_pd_proc_ls_prof_wR_factor' ] ] ],
      [ 'save_',
        '_refine.pdbx_pd_Marquardt_correlation_coeff',
        [ [ 'item_',
            '_item_description.description',
            '  The correlation coefficient between the observed and\n   calculated structure factors for reflections included in\n   the refinement. This correlation factor is found in the \n   fitting using the Levenberg-Marquardt algorithm to search \n   for the minimum value of chisquare. Almost all computer \n   codes for Rietveld refinement employ the Gauss-Newton algorithm \n   to find parameters which minimize the weighted sum of squares \n   of the residuals.\n   A description of the equations is given on\n    http://www.water.hut.fi/~tkarvone/fr_org_s.htm' ],
          [ 'item_',
            '_item.name',
            '_refine.pdbx_pd_Marquardt_correlation_coeff' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.pdbx_pd_Fsqrd_R_factor',
        [ [ 'item_',
            '_item_description.description',
            '   Residual factor R for reflections that satisfy the resolution\n    limits established by _refine.ls_d_res_high and \n    _refine.ls_d_res_low and the observation limit established by\n    _reflns.observed_criterion.\n\n                   sum|F~obs~**2 - F~calc~**2|\n               R = ---------------------\n                        sum|F~obs~**2|\n\n               F~obs~  = the observed structure-factor amplitudes\n               F~calc~ = the calculated structure-factor amplitudes\n               sum is taken over the specified reflections' ],
          [ 'item_', '_item.name', '_refine.pdbx_pd_Fsqrd_R_factor' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_refine.pdbx_pd_ls_matrix_band_width',
        [ [ 'item_',
            '_item_description.description',
            '      The least squares refinement "band matrix" approximation to the full matrix.' ],
          [ 'item_', '_item.name', '_refine.pdbx_pd_ls_matrix_band_width' ],
          [ 'item_', '_item.category_id', 'refine' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0', '0', '0' ] ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ '@comment', '## EOF mmcif_pdbx-def-10.dic' ],
      [ '@comment', '######################' ],
      [ '@comment', '# PDBX_SOLN_SCATTER ##' ],
      [ '@comment', '######################' ],
      [ 'save_',
        'pdbx_soln_scatter',
        [ [ 'item_',
            '_category.description',
            '   Data items in the PDBX_SOLN_SCATTER category record details about a \n    solution scattering experiment' ],
          [ 'item_', '_category.id', 'pdbx_soln_scatter' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_soln_scatter.entry_id', '_pdbx_soln_scatter.id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'solution_scattering_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail', '_category_examples.case' ],
            [ ' Example 1 - based on PDB entry 1HAQ',
              '  loop_\n   _pdbx_soln_scatter.entry_id\n   _pdbx_soln_scatter.id\n   _pdbx_soln_scatter.type\n   _pdbx_soln_scatter.source_type\n   _pdbx_soln_scatter.source_class\n   _pdbx_soln_scatter.source_beamline\n   _pdbx_soln_scatter.source_beamline_instrument\n   _pdbx_soln_scatter.detector_specific\n   _pdbx_soln_scatter.detector_type\n   _pdbx_soln_scatter.temperature   \n   _pdbx_soln_scatter.pH\n   _pdbx_soln_scatter.num_time_frames\n   _pdbx_soln_scatter.concentration_range\n   _pdbx_soln_scatter.buffer_name\n   _pdbx_soln_scatter.mean_guiner_radius\n   _pdbx_soln_scatter.mean_guiner_radius_esd\n   _pdbx_soln_scatter.min_mean_cross_sectional_radii_gyration \n   _pdbx_soln_scatter.min_mean_cross_sectional_radii_gyration_esd\n   _pdbx_soln_scatter.max_mean_cross_sectional_radii_gyration \n   _pdbx_soln_scatter.max_mean_cross_sectional_radii_gyration_eds\n   _pdbx_soln_scatter.protein_length\n  1HAQ 1 x-ray     \'SRS BEAMLINE 2.1\' \'synchrotron\' \'2.1\' . . \'500-channel quadrant\'\n         288 . 10 \'0.7 - 14\' tris \n         11.1 0.4 4.4 0.2 1.7 0.1 40\n  1HAQ 2 neutron  \'ILL\'               \'neutron source\' . \'D11, D22\' . \'area\' \n         .  .  .  \'0.4 - 9.6\'  \'PBS in 99.9% D2O\' \n         11.3 0.4 3.9 0.2 1.51 0.06 \'37.0 - 39.0\'\n  1HAQ 3 neutron  \'ISIS\' \'neutron source\' \'Pulsed Neutron\' \'LOQ\' . \'AREA (TIME-OF-FLIGHT)\'\n         .  .  .  \'3.7, 6.1\'     \'PBS in 99.9% D2O\' \n         11.7 0.5 . . . . 40.0' ] ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.entry_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to _entry.id in the ENTRY category.' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.entry_id' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_soln_scatter.entry_id' ],
          [ 'item_', '_item_linked.parent_name', '_entry.id' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.id',
        [ [ 'item_',
            '_item_description.description',
            '   The value of _pdbx_soln_scatter.id must\n    uniquely identify the sample in the category PDBX_SOLN_SCATTER' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.id' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.type',
        [ [ 'item_',
            '_item_description.description',
            '   The type of solution scattering experiment carried out' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.type' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value' ],
            [ 'x-ray', 'neutron', 'modelling' ] ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.source_beamline',
        [ [ 'item_',
            '_item_description.description',
            '   The beamline name used for the experiment' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.source_beamline' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.source_beamline_instrument',
        [ [ 'item_',
            '_item_description.description',
            '   The instrumentation used on the beamline' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter.source_beamline_instrument' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.detector_type',
        [ [ 'item_',
            '_item_description.description',
            '    The general class of the radiation detector.' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.detector_type' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.detector_specific',
        [ [ 'item_',
            '_item_description.description',
            ' The particular radiation detector. In general this will be a\n   manufacturer, description, model number or some combination of\n   these.' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.detector_specific' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.source_type',
        [ [ 'item_',
            '_item_description.description',
            ' The make, model, name or beamline of the source of radiation.' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.source_type' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.source_class',
        [ [ 'item_',
            '_item_description.description',
            '  The general class of the radiation source.' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.source_class' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'loop_',
            [ '_item_examples.case' ],
            [ 'neutron source', 'synchrotron' ] ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.num_time_frames',
        [ [ 'item_',
            '_item_description.description',
            '    The number of time frame solution scattering images used.' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.num_time_frames' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.sample_pH',
        [ [ 'item_',
            '_item_description.description',
            '   The pH value of the buffered sample.' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.sample_pH' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.temperature',
        [ [ 'item_',
            '_item_description.description',
            '    The temperature in kelvins at which the experiment\n     was conducted' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.temperature' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_item_range.maximum', '_item_range.minimum' ],
            [ '.', '0.0', '0.0', '0.0' ] ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'kelvins' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.concentration_range',
        [ [ 'item_',
            '_item_description.description',
            '   The concentration range (mg/mL) of the complex in the \n    sample used in the solution scattering experiment to\n    determine the mean radius of structural elongation.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter.concentration_range' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_units.code', 'mg_per_ml' ],
          [ 'item_', '_item_examples.case', '0.7 - 14' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.buffer_name',
        [ [ 'item_',
            '_item_description.description',
            '  The name of the buffer used for the sample in the solution scattering\n   experiment. ' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.buffer_name' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'item_', '_item_examples.case', 'acetic acid' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.mean_guiner_radius',
        [ [ 'item_',
            '_item_description.description',
            '    The mean radius of structural elongation of the sample.\n     In a given solute-solvent contrast, the radius of gyration\n     R_G is a measure of structural elongation if the internal\n     inhomogeneity of scattering densities has no effect. Guiner \n     analysis at low Q gives the R_G and the forward scattering at \n     zero angle I(0).\n\n         lnl(Q) = lnl(0) - R_G^2Q^2/3\n\n     where \n           Q = 4(pi)sin(theta/lamda)\n           2theta = scattering angle\n           lamda = wavelength\n\n     The above expression is valid in a QR_G range for extended \n     rod-like particles. The relative I(0)/c values ( where\n      c = sample concentration) for sample measurements in a\n     constant buffer for a single sample data session, gives the\n     relative masses of the protein(s) studied when referenced\n     against a standard.\n\n     see: O.Glatter & O.Kratky, (1982). Editors of "Small angle\n          X-ray Scattering, Academic Press, New York.\n          O.Kratky. (1963). X-ray small angle scattering with\n          substances of biological interest in diluted solutions.\n          Prog. Biophys. Chem., 13, 105-173.\n\n          G.D.Wignall & F.S.Bates, (1987). The small-angle approximation\n          of X-ray and neutron scatter from rigid rods of non-uniform\n          cross section and finite length. J.Appl. Crystallog., 18, 452-460.\n\n     If the structure is elongated, the mean radius of gyration\n     of the cross-sectional structure R_XS  and the mean cross sectional\n     intensity at zero angle [I(Q).Q]_Q->0 is obtained from\n\n        ln[I(Q).Q] = ln[l(Q).(Q)]_Q->0 - ((R_XS)^2Q^2)/2' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter.mean_guiner_radius' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_units.code', 'nanometres' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_',
            '_item_related.related_name',
            '_pdbx_soln_scatter.mean_guiner_radius_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.mean_guiner_radius_esd',
        [ [ 'item_',
            '_item_description.description',
            '    The estimated standard deviation for the\n     mean radius of structural elongation of the sample.\n     In a given solute-solvent contrast, the radius of gyration\n     R_G is a measure of structural elongation if the internal\n     inhomogeneity of scattering densities has no effect. Guiner \n     analysis at low Q give the R_G and the forward scattering at \n     zero angle I(0).\n\n         lnl(Q) = lnl(0) - R_G^2Q^2/3\n\n     where \n           Q = 4(pi)sin(theta/lamda)\n           2theta = scattering angle\n           lamda = wavelength\n\n     The above expression is valid in a QR_G range for extended \n     rod-like particles. The relative I(0)/c values ( where\n      c = sample concentration) for sample measurements in a\n     constant buffer for a single sample data session, gives the\n     relative masses of the protein(s) studied when referenced\n     against a standard.\n\n     see: \n         O.Glatter & O.Kratky, (1982). Editors of "Small angle\n          X-ray Scattering, Academic Press, New York.\n         O.Kratky. (1963). X-ray small angle scattering with\n          substances of biological interest in diluted solutions.\n          Prog. Biophys. Chem., 13, 105-173.\n         G.D.Wignall & F.S.Bates, (1987). The small-angle approximation\n          of X-ray and neutron scatter from rigid rods of non-uniform\n          cross section and finite length. J.Appl. Crystallog., 18, 452-460.\n\n     If the structure is elongated, the mean radius of gyration\n     of the cross-sectional structure R_XS  and the mean cross sectional\n     intensity at zero angle [I(Q).Q]_Q->0 is obtained from\n        ln[I(Q).Q] = ln[l(Q).(Q)]_Q->0 - ((R_XS)^2Q^2)/2' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter.mean_guiner_radius_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'nanometres' ],
          [ 'item_',
            '_item_related.related_name',
            '_pdbx_soln_scatter.mean_guiner_radius' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.min_mean_cross_sectional_radii_gyration',
        [ [ 'item_',
            '_item_description.description',
            '    \n     The minimum mean radius of structural elongation of the sample.\n     In a given solute-solvent contrast, the radius of gyration\n     R_G is a measure of structural elongation if the internal\n     inhomogeneity of scattering densities has no effect. Guiner \n     analysis at low Q give the R_G and the forward scattering at \n     zero angle I(0).\n\n         lnl(Q) = lnl(0) - R_G^2Q^2/3\n\n     where \n           Q = 4(pi)sin(theta/lamda)\n           2theta = scattering angle\n           lamda = wavelength\n\n     The above expression is valid in a QR_G range for extended \n     rod-like particles. The relative I(0)/c values ( where\n      c = sample concentration) for sample measurements in a\n     constant buffer for a single sample data session, gives the\n     relative masses of the protein(s) studied when referenced\n     against a standard.\n\n     see: \n         O.Glatter & O.Kratky, (1982). Editors of "Small angle\n          X-ray Scattering, Academic Press, New York.\n         O.Kratky. (1963). X-ray small angle scattering with\n          substances of biological interest in diluted solutions.\n          Prog. Biophys. Chem., 13, 105-173.\n         G.D.Wignall & F.S.Bates, (1987). The small-angle approximation\n          of X-ray and neutron scatter from rigid rods of non-uniform\n          cross section and finite length. J.Appl. Crystallog., 18, 452-460.\n\n     If the structure is elongated, the mean radius of gyration\n     of the cross-sectional structure R_XS  and the mean cross sectional\n     intensity at zero angle [I(Q).Q]_Q->0 is obtained from\n        ln[I(Q).Q] = ln[l(Q).(Q)]_Q->0 - ((R_XS)^2Q^2)/2' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter.min_mean_cross_sectional_radii_gyration' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'nanometres' ],
          [ 'item_',
            '_item_related.related_name',
            '_pdbx_soln_scatter.min_mean_cross_sectional_radii_gyration_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.min_mean_cross_sectional_radii_gyration_esd',
        [ [ 'item_',
            '_item_description.description',
            '\n     The estimated standard deviation for the\n     minimum mean radius of structural elongation of the sample.\n     In a given solute-solvent contrast, the radius of gyration\n     R_G is a measure of structural elongation if the internal\n     inhomogeneity of scattering densities has no effect. Guiner \n     analysis at low Q give the R_G and the forward scattering at \n     zero angle I(0).\n \n        lnl(Q) = lnl(0) - R_G^2Q^2/3\n\n     where \n           Q = 4(pi)sin(theta/lamda)\n           2theta = scattering angle\n           lamda = wavelength\n\n     The above expression is valid in a QR_G range for extended \n     rod-like particles. The relative I(0)/c values ( where\n      c = sample concentration) for sample measurements in a\n     constant buffer for a single sample data session, gives the\n     relative masses of the protein(s) studied when referenced\n     against a standard.\n\n     see: \n         O.Glatter & O.Kratky, (1982). Editors of "Small angle\n          X-ray Scattering, Academic Press, New York.\n         O.Kratky. (1963). X-ray small angle scattering with\n          substances of biological interest in diluted solutions.\n          Prog. Biophys. Chem., 13, 105-173.\n         G.D.Wignall & F.S.Bates, (1987). The small-angle approximation\n          of X-ray and neutron scatter from rigid rods of non-uniform\n          cross section and finite length. J.Appl. Crystallog., 18, 452-460.\n\n     If the structure is elongated, the mean radius of gyration\n     of the cross-sectional structure R_XS  and the mean cross sectional\n     intensity at zero angle [I(Q).Q]_Q->0 is obtained from\n\n        ln[I(Q).Q] = ln[l(Q).(Q)]_Q->0 - ((R_XS)^2Q^2)/2' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter.min_mean_cross_sectional_radii_gyration_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'nanometres' ],
          [ 'item_',
            '_item_related.related_name',
            '_pdbx_soln_scatter.min_mean_cross_sectional_radii_gyration' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.max_mean_cross_sectional_radii_gyration',
        [ [ 'item_',
            '_item_description.description',
            '    The maximum mean radius of structural elongation of the sample.\n     In a given solute-solvent contrast, the radius of gyration\n     R_G is a measure of structural elongation if the internal\n     inhomogeneity of scattering densities has no effect. Guiner \n     analysis at low Q give the R_G and the forward scattering at \n     zero angle I(0).\n \n        lnl(Q) = lnl(0) - R_G^2Q^2/3\n\n     where \n           Q = 4(pi)sin(theta/lamda)\n           2theta = scattering angle\n           lamda = wavelength\n\n     The above expression is valid in a QR_G range for extended \n     rod-like particles. The relative I(0)/c values ( where\n      c = sample concentration) for sample measurements in a\n     constant buffer for a single sample data session, gives the\n     relative masses of the protein(s) studied when referenced\n     against a standard.\n\n     see:\n         O.Glatter & O.Kratky, (1982). Editors of "Small angle\n          X-ray Scattering, Academic Press, New York.\n         O.Kratky. (1963). X-ray small angle scattering with\n          substances of biological interest in diluted solutions.\n          Prog. Biophys. Chem., 13, 105-173.\n         G.D.Wignall & F.S.Bates, (1987). The small-angle approximation\n          of X-ray and neutron scatter from rigid rods of non-uniform\n          cross section and finite length. J.Appl. Crystallog., 18, 452-460.\n\n     If the structure is elongated, the mean radius of gyration\n     of the cross-sectional structure R_XS  and the mean cross sectional\n     intensity at zero angle [I(Q).Q]_Q->0 is obtained from\n        ln[I(Q).Q] = ln[l(Q).(Q)]_Q->0 - ((R_XS)^2Q^2)/2' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter.max_mean_cross_sectional_radii_gyration' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'nanometres' ],
          [ 'item_',
            '_item_related.related_name',
            '_pdbx_soln_scatter.max_mean_cross_sectional_radii_gyration_esd' ],
          [ 'item_', '_item_related.function_code', 'associated_esd' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.max_mean_cross_sectional_radii_gyration_esd',
        [ [ 'item_',
            '_item_description.description',
            '    \n     The estimated standard deviation for the\n     minimum mean radius of structural elongation of the sample.\n     In a given solute-solvent contrast, the radius of gyration\n     R_G is a measure of structural elongation if the internal\n     inhomogeneity of scattering densities has no effect. Guiner \n     analysis at low Q give the R_G and the forward scattering at \n     zero angle I(0).\n\n         lnl(Q) = lnl(0) - R_G^2Q^2/3\n\n     where \n           Q = 4(pi)sin(theta/lamda)\n           2theta = scattering angle\n           lamda = wavelength\n\n     The above expression is valid in a QR_G range for extended \n     rod-like particles. The relative I(0)/c values ( where\n      c = sample concentration) for sample measurements in a\n     constant buffer for a single sample data session, gives the\n     relative masses of the protein(s) studied when referenced\n     against a standard.\n\n     see:  \n         O.Glatter & O.Kratky, (1982). Editors of "Small angle\n          X-ray Scattering, Academic Press, New York.\n         O.Kratky. (1963). X-ray small angle scattering with\n          substances of biological interest in diluted solutions.\n          Prog. Biophys. Chem., 13, 105-173.\n         G.D.Wignall & F.S.Bates, (1987). The small-angle approximation\n          of X-ray and neutron scatter from rigid rods of non-uniform\n          cross section and finite length. J.Appl. Crystallog., 18, 452-460.\n\n     If the structure is elongated, the mean radius of gyration\n     of the cross-sectional structure R_XS  and the mean cross sectional\n     intensity at zero angle [I(Q).Q]_Q->0 is obtained from\n        ln[I(Q).Q] = ln[l(Q).(Q)]_Q->0 - ((R_XS)^2Q^2)/2' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter.max_mean_cross_sectional_radii_gyration_esd' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'item_', '_item_units.code', 'nanometres' ],
          [ 'item_',
            '_item_related.related_name',
            '_pdbx_soln_scatter.max_mean_cross_sectional_radii_gyration' ],
          [ 'item_', '_item_related.function_code', 'associated_value' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.protein_length',
        [ [ 'item_',
            '_item_description.description',
            '  \n   The length (or range) of the protein sample under study.\n   If the solution structure is approximated as an elongated elliptical\n   cyclinder the the length L is determined from,\n\n     L = sqrt [12( (R_G)^2  -  (R_XS)^2 ) ]   \n\n   The length should also be given by\n\n     L = pi I(0) / [ I(Q).Q]_Q->0' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter.protein_length' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.data_reduction_software_list',
        [ [ 'item_',
            '_item_description.description',
            '   A list of the software used in the data reduction' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter.data_reduction_software_list' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'OTOKO' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter.data_analysis_software_list',
        [ [ 'item_',
            '_item_description.description',
            '   A list of the software used in the data analysis' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter.data_analysis_software_list' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', 'SCTPL5 GNOM' ] ] ],
      [ '@comment', '############################' ],
      [ '@comment', '# PDBX_SOLN_SCATTER_MODEL ##' ],
      [ '@comment', '############################' ],
      [ 'save_',
        'pdbx_soln_scatter_model',
        [ [ 'item_',
            '_category.description',
            '   Data items in the PDBX_SOLN_SCATTER_MODEL category record details about the\n    homology model fitting to the solution scatter data.' ],
          [ 'item_', '_category.id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_soln_scatter_model.id',
              '_pdbx_soln_scatter_model.scatter_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'solution_scattering_group' ] ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter_model.scatter_id',
        [ [ 'item_',
            '_item_description.description',
            '    This data item is a pointer to  _pdbx_soln_scatter.id in the  PDBX_SOLN_SCATTER category.' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter_model.scatter_id' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_soln_scatter_model.scatter_id' ],
          [ 'item_', '_item_linked.parent_name', '_pdbx_soln_scatter.id' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter_model.id',
        [ [ 'item_',
            '_item_description.description',
            '   The value of _pdbx_soln_scatter_model.id must\n    uniquely identify the sample in the category PDBX_SOLN_SCATTER_MODEL' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter_model.id' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter_model.details',
        [ [ 'item_',
            '_item_description.description',
            '   A description of any additional details concerning the experiment. ' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter_model.details' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '   Homology models were built for\n    the 17 SCR domains and energy minimisations were \n    performed to improve the connectivity in the fh model.\n    triantennary complex-type carbohydrate structures\n    (MAN3GLCNAC6GAL3FUC3NEUNAC1) were added to each of the\n    N-linked glycosylation sites. a library of linker peptide\n    conformations was used in domain modelling constrained\n    by the solution scattering fits. modelling with the\n    scattering data was also carried out by rotational \n    search methods. the x-ray and neutron scattering curve \n    I(Q) was calculated assuming a uniform scattering density \n    for the spheres using the debye equation as adapted to \n    spheres. x-ray curves were calculated from the hydrated \n    sphere models without corrections for wavelength spread or\n    beam divergence, while these corrections were applied for \n    the neutron curves but now using unhydrated models.' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter_model.method',
        [ [ 'item_',
            '_item_description.description',
            '   A description of the methods used in the modelling' ],
          [ 'item_', '_item.name', '_pdbx_soln_scatter_model.method' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '  Constrained scattering fitting of homology models' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter_model.software_list',
        [ [ 'item_',
            '_item_description.description',
            '   A list of the software used in the modeeling' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter_model.software_list' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '     INSIGHT II, HOMOLOGY, DISCOVERY, BIOPOLYMER, DELPHI ' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter_model.software_author_list',
        [ [ 'item_',
            '_item_description.description',
            '   A list of the software authors' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter_model.software_author_list' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_', '_item_examples.case', ' MSI' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter_model.entry_fitting_list',
        [ [ 'item_',
            '_item_description.description',
            '   A list of the entries used to fit the model\n    to the scattering data' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter_model.entry_fitting_list' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            ' PDB CODE 1HFI, 1HCC, 1HFH, 1VCC' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter_model.num_conformers_calculated',
        [ [ 'item_',
            '_item_description.description',
            '   The number of model conformers calculated.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter_model.num_conformers_calculated' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter_model.num_conformers_submitted',
        [ [ 'item_',
            '_item_description.description',
            '   The number of model conformers submitted in the entry' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter_model.num_conformers_submitted' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'int' ] ] ],
      [ 'save_',
        '_pdbx_soln_scatter_model.conformer_selection_criteria',
        [ [ 'item_',
            '_item_description.description',
            '   A description of the conformer selection criteria\n    used.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_soln_scatter_model.conformer_selection_criteria' ],
          [ 'item_', '_item.category_id', 'pdbx_soln_scatter_model' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'text' ],
          [ 'item_',
            '_item_examples.case',
            '  The modelled scattering curves were assessed by calculation of the \n   RG, RSX-1 and RXS-2 values in the same Q ranges \n   used in the experimental Guinier fits. models were \n   then ranked using a goodness-of-fit R-factor \n   defined by analogy with protein crystallography \n   and based on the experimental curves in the Q range \n   extending to 1.4 nm-1.' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment',
        '##########################################################################' ],
      [ '@comment',
        ' File:  mmcif_pdbx-def-13.dic                                             ' ],
      [ '@comment', ' ' ],
      [ '@comment',
        '                      PDB Exchange Data Dictionary' ],
      [ '@comment',
        ' This data dictionary contains extensions used by RCSB for describing' ],
      [ '@comment', ' chemical components.' ],
      [ '@comment',
        '                           Definition Section 13' ],
      [ '@comment', '                            ' ],
      [ '@comment',
        '                                                                        ' ],
      [ '@comment',
        '##########################################################################' ],
      [ 'save_',
        '_chem_comp_atom.pdbx_alt_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              An alternative identifier for the atom. This data item would be\n               used in cases where alternative nomenclatures exist for labelling\n               atoms in a group.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.pdbx_alt_atom_id' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'atcode' ] ] ],
      [ 'save_',
        '_chem_comp_atom.pdbx_alt_comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              An alternative identifier for the atom. This data item would be\n               used in cases where alternative nomenclatures exist for labelling\n               atoms in a group.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.pdbx_alt_comp_id' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ] ] ],
      [ 'save_',
        '_chem_comp_atom.pdbx_model_Cartn_x_2',
        [ [ 'item_',
            '_item_description.description',
            '              An alternative x component of the coordinates for this atom in this\n               component specified as orthogonal angstroms. ' ],
          [ 'item_', '_item.name', '_chem_comp_atom.pdbx_model_Cartn_x_2' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_atom.pdbx_model_Cartn_y_2',
              '_chem_comp_atom.pdbx_model_Cartn_z_2' ] ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_atom.pdbx_model_Cartn_y_2',
        [ [ 'item_',
            '_item_description.description',
            '              An alternative y component of the coordinates for this atom in this\n               component specified as orthogonal angstroms. ' ],
          [ 'item_', '_item.name', '_chem_comp_atom.pdbx_model_Cartn_y_2' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_atom.pdbx_model_Cartn_x_2',
              '_chem_comp_atom.pdbx_model_Cartn_z_2' ] ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_atom.pdbx_model_Cartn_z_2',
        [ [ 'item_',
            '_item_description.description',
            '              An alternative z component of the coordinates for this atom in this\n               component specified as orthogonal angstroms. ' ],
          [ 'item_', '_item.name', '_chem_comp_atom.pdbx_model_Cartn_z_2' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'float' ],
          [ 'loop_',
            [ '_item_dependent.dependent_name' ],
            [ '_chem_comp_atom.pdbx_model_Cartn_x_2',
              '_chem_comp_atom.pdbx_model_Cartn_y_2' ] ],
          [ 'item_', '_item_sub_category.id', 'cartesian_coordinate' ],
          [ 'item_', '_item_units.code', 'angstroms' ] ] ],
      [ 'save_',
        '_chem_comp_atom.pdbx_stereo_config',
        [ [ 'item_',
            '_item_description.description',
            '              The chiral configuration of the atom that is a chiral center.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.pdbx_stereo_config' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'R',
              'rectus - right handed configuration',
              'S',
              'sinister - left handed configuration' ] ] ] ],
      [ 'save_',
        '_chem_comp_atom.pdbx_aromatic_flag',
        [ [ 'item_',
            '_item_description.description',
            '              A flag indicating an aromatic atom.' ],
          [ 'item_', '_item.name', '_chem_comp_atom.pdbx_aromatic_flag' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes - an aromatic atom' ] ] ] ],
      [ 'save_',
        '_chem_comp_atom.pdbx_leaving_atom_flag',
        [ [ 'item_',
            '_item_description.description',
            '              A flag indicating a leaving atom.' ],
          [ 'item_',
            '_item.name',
            '_chem_comp_atom.pdbx_leaving_atom_flag' ],
          [ 'item_', '_item.category_id', 'chem_comp_atom' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes - a leaving atom' ] ] ] ],
      [ 'save_',
        '_chem_comp_bond.pdbx_stereo_config',
        [ [ 'item_',
            '_item_description.description',
            '              Stereochemical configuration across a double bond.' ],
          [ 'item_', '_item.name', '_chem_comp_bond.pdbx_stereo_config' ],
          [ 'item_', '_item.category_id', 'chem_comp_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'E', 'zusammen for together', 'Z', 'entgegen for opposite' ] ] ] ],
      [ 'save_',
        '_chem_comp_bond.pdbx_aromatic_flag',
        [ [ 'item_',
            '_item_description.description',
            '              A flag indicating an aromatic bond.' ],
          [ 'item_', '_item.name', '_chem_comp_bond.pdbx_aromatic_flag' ],
          [ 'item_', '_item.category_id', 'chem_comp_bond' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'Y', 'Yes - an aromatic bond' ] ] ] ],
      [ '@comment', '##############################' ],
      [ '@comment', '# PDBX_CHEM_COMP_DESCRIPTOR ##' ],
      [ '@comment', '##############################' ],
      [ 'save_',
        'pdbx_chem_comp_descriptor',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_DESCRIPTOR category provide\n               string descriptors of component chemical structure.' ],
          [ 'item_', '_category.id', 'pdbx_chem_comp_descriptor' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_chem_comp_descriptor.comp_id',
              '_pdbx_chem_comp_descriptor.descriptor',
              '_pdbx_chem_comp_descriptor.type',
              '_pdbx_chem_comp_descriptor.program' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n    loop_\n    _pdbx_chem_comp_descriptor.comp_id\n    _pdbx_chem_comp_descriptor.descriptor\n    _pdbx_chem_comp_descriptor.type\n    _pdbx_chem_comp_descriptor.program\n    _pdbx_chem_comp_descriptor.program_version\n     . . . . .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_descriptor.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_descriptor.comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_descriptor' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_chem_comp_descriptor.comp_id' ],
          [ 'item_', '_item_linked.parent_name', '_chem_comp.id' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_descriptor.descriptor',
        [ [ 'item_',
            '_item_description.description',
            '              This data item contains the descriptor value for this \n               component.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_chem_comp_descriptor.descriptor' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_descriptor' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_descriptor.type',
        [ [ 'item_',
            '_item_description.description',
            '              This data item contains the descriptor type. ' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_descriptor.type' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_descriptor' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'SMILES_CANNONICAL',
              'Smiles descriptor',
              'SMILES',
              'Cannonical Smiles descriptor',
              'INCHI',
              'InChi descriptor',
              'INCHI_MAIN',
              'InChi descriptor- main layer',
              'INCHI_MAIN_FORMULA',
              'InChi descriptor- main layer - chemical formula sub-layer',
              'INCHI_MAIN_CONNECT',
              'InChi descriptor- main layer - atom connection sub-layer',
              'INCHI_MAIN_HATOM',
              'InChi descriptor- main layer - hydrogen atom sub-layer',
              'INCHI_CHARGE',
              'InChi descriptor- charge layer',
              'INCHI_STEREO',
              'InChi descriptor- stereochemical layer',
              'INCHI_ISOTOPE',
              'InChi descriptor- isotopic layer',
              'INCHI_FIXEDH',
              'InChi descriptor- fixed hydrogren layer',
              'INCHI_RECONNECT',
              'InChi descriptor- reconnected layer' ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_descriptor.program',
        [ [ 'item_',
            '_item_description.description',
            '              This data item contains the name of the program\n               or library used to compute the descriptor. ' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_descriptor.program' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_descriptor' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'OECHEM',
              'OpenEye OECHEM library',
              'CACTVS',
              'CACTVS  program library',
              'DAYLIGHT',
              'Daylight program library',
              'OTHER',
              'Other program or library' ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_descriptor.program_version',
        [ [ 'item_',
            '_item_description.description',
            '              This data item contains the version of the program\n               or library used to compute the descriptor. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_chem_comp_descriptor.program_version' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_descriptor' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '##############################' ],
      [ '@comment', '# PDBX_CHEM_COMP_IDENTIFIER ##' ],
      [ '@comment', '##############################' ],
      [ 'save_',
        'pdbx_chem_comp_identifier',
        [ [ 'item_',
            '_category.description',
            '              Data items in the CHEM_COMP_IDENTIFIER category provide\n               identifiers for chemical components.' ],
          [ 'item_', '_category.id', 'pdbx_chem_comp_identifier' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_chem_comp_identifier.comp_id',
              '_pdbx_chem_comp_identifier.identifier',
              '_pdbx_chem_comp_identifier.type' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n    loop_\n    _pdbx_chem_comp_identifier.comp_id\n    _pdbx_chem_comp_identifier.identifier\n    _pdbx_chem_comp_identifier.type\n    _pdbx_chem_comp_identifier.program\n    _pdbx_chem_comp_identifier.program_version\n     . . . . .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_identifier.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_identifier.comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_identifier' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_chem_comp_identifier.comp_id' ],
          [ 'item_', '_item_linked.parent_name', '_chem_comp.id' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_identifier.identifier',
        [ [ 'item_',
            '_item_description.description',
            '              This data item contains the identifier value for this \n               component.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_chem_comp_identifier.identifier' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_identifier' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'text' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_identifier.type',
        [ [ 'item_',
            '_item_description.description',
            '              This data item contains the identifier type. ' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_identifier.type' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_identifier' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'COMMON_NAME',
              'Common chemical name',
              'SYSTEMATIC NAME',
              'Systematic chemical name',
              'CAS REGISTRY NUMBER',
              'Chemical Abstracts Registry Number',
              'PUBCHEM Identifier',
              'PubChem accession number',
              'MDL Identifier',
              'Molecular Design Limited Identifier' ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_identifier.program',
        [ [ 'item_',
            '_item_description.description',
            '              This data item contains the name of the program\n               or library used to compute the identifier. ' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_identifier.program' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_identifier' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'OECHEM',
              'OpenEye OECHEM program',
              'DAYLIGHT',
              'Daylight program library',
              'ACD',
              'Advanced Chemistry Development Naming Program',
              'AUTONOM',
              'Belstein AutoNom Naming Program',
              'OTHER',
              'Other program or library' ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_identifier.program_version',
        [ [ 'item_',
            '_item_description.description',
            '              This data item contains the version of the program\n               or library used to compute the identifier. ' ],
          [ 'item_',
            '_item.name',
            '_pdbx_chem_comp_identifier.program_version' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_identifier' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '##########################' ],
      [ '@comment', '# PDBX_CHEM_COMP_IMPORT ##' ],
      [ '@comment', '##########################' ],
      [ 'save_',
        'pdbx_chem_comp_import',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_CHEM_COMP_IMPORT category identifies\n               existing chemical components to be imported into the \n               current component definition.  Components in this list\n               can be edited by instructions in categories \n               pdbx_chem_comp_atom_edit and pdbx_chem_comp_bond_edit.' ],
          [ 'item_', '_category.id', 'pdbx_chem_comp_import' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_chem_comp_import.comp_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n    loop_\n    _pdbx_chem_comp_import.comp_id\n     ATP',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_import.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_import.comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_import' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_chem_comp_import.comp_id' ],
          [ 'item_', '_item_linked.parent_name', '_chem_comp.id' ] ] ],
      [ '@comment', '#############################' ],
      [ '@comment', '# PDBX_CHEM_COMP_ATOM_EDIT ##' ],
      [ '@comment', '#############################' ],
      [ 'save_',
        'pdbx_chem_comp_atom_edit',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_CHEM_COMP_ATOM_EDIT category provide\n               atom level editing instructions to be applied to imported\n               chemical components. ' ],
          [ 'item_', '_category.id', 'pdbx_chem_comp_atom_edit' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_chem_comp_atom_edit.comp_id',
              '_pdbx_chem_comp_atom_edit.edit_op',
              '_pdbx_chem_comp_atom_edit.atom_id' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n    loop_\n    _pdbx_chem_comp_atom_edit.comp_id\n    _pdbx_chem_comp_atom_edit.edit_op\n    _pdbx_chem_comp_atom_edit.atom_id\n    . . .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_atom_edit.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _pdbx_chem_comp_import.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_atom_edit.comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_atom_edit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_chem_comp_atom_edit.comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_chem_comp_import.comp_id' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_atom_edit.edit_op',
        [ [ 'item_',
            '_item_description.description',
            '              The operation applied to the named imported component.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_atom_edit.edit_op' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_atom_edit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'RENAME',
              'Rename the target atom to ',
              'DELETE',
              'Remove the target atom from the component',
              'ADD',
              'Add the edit_atom_id to the component',
              'CHARGE',
              'Assign charge to edit_atom_id',
              'PARTIAL_CHARGE',
              'Assign partial charge to edit_atom_id',
              'MODEL_CARTN_X',
              'Assign model X Cartesian coordinate to edit_atom_id',
              'MODEL_CARTN_Y',
              'Assign model Y Cartesian coordinate to edit_atom_id',
              'MODEL_CARTN_Z',
              'Assign model Z Cartesian coordinate to edit_atom_id',
              'STEREO_CONFIG',
              'Stereochemical configuration',
              'AROMATIC_FLAG',
              'Aromatic flag' ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_atom_edit.atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier for the target atom in imported component to be edited.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_atom_edit.atom_id' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_atom_edit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'atcode' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_atom_edit.edit_atom_id',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier for the edited atom in the generated component.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_chem_comp_atom_edit.edit_atom_id' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_atom_edit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'atcode' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_atom_edit.edit_atom_value',
        [ [ 'item_',
            '_item_description.description',
            '              The value for the edited atomic property value in the generated component.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_chem_comp_atom_edit.edit_atom_value' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_atom_edit' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '#############################' ],
      [ '@comment', '# PDBX_CHEM_COMP_BOND_EDIT ##' ],
      [ '@comment', '#############################' ],
      [ 'save_',
        'pdbx_chem_comp_bond_edit',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_CHEM_COMP_BOND_EDIT category provide\n               bond level editing instructions to be applied to imported\n               chemical components. ' ],
          [ 'item_', '_category.id', 'pdbx_chem_comp_bond_edit' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_chem_comp_bond_edit.comp_id',
              '_pdbx_chem_comp_bond_edit.edit_op',
              '_pdbx_chem_comp_bond_edit.atom_id_1',
              '_pdbx_chem_comp_bond_edit.atom_id_2' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n    loop_\n    _pdbx_chem_comp_bond_edit.comp_id\n    _pdbx_chem_comp_bond_edit.edit_op\n    _pdbx_chem_comp_bond_edit.atom_id_1\n    _pdbx_chem_comp_bond_edit.atom_id_2\n    . . . .',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_bond_edit.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _pdbx_chem_comp_import.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_bond_edit.comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_bond_edit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_chem_comp_bond_edit.comp_id' ],
          [ 'item_',
            '_item_linked.parent_name',
            '_pdbx_chem_comp_import.comp_id' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_bond_edit.edit_op',
        [ [ 'item_',
            '_item_description.description',
            '              The operation or assignment applied to the named imported component.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_bond_edit.edit_op' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_bond_edit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'DELETE',
              'Remove the target atom from the component',
              'ADD',
              'Add the edit_atom_id to the component',
              'VALUE_ORDER',
              'Bond order',
              'VALUE_DIST',
              'Bond distance',
              'STEREO_CONFIG',
              'Stereochemical configuration',
              'AROMATIC_FLAG',
              'Aromatic flag' ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_bond_edit.atom_id_1',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier for the first atom in the target bond \n               in imported component.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_bond_edit.atom_id_1' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_bond_edit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'atcode' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_bond_edit.atom_id_2',
        [ [ 'item_',
            '_item_description.description',
            '              The identifier for the second atom in the target bond \n               in imported component.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_bond_edit.atom_id_2' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_bond_edit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'atcode' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_bond_edit.edit_bond_value',
        [ [ 'item_',
            '_item_description.description',
            '              The value for the edited bond property value in the generated component.' ],
          [ 'item_',
            '_item.name',
            '_pdbx_chem_comp_bond_edit.edit_bond_value' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_bond_edit' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ] ] ],
      [ '@comment', '#' ],
      [ '@comment', '#' ],
      [ '@comment', '#########################' ],
      [ '@comment', '# PDBX_CHEM_COMP_AUDIT ##' ],
      [ '@comment', '#########################' ],
      [ 'save_',
        'pdbx_chem_comp_audit',
        [ [ 'item_',
            '_category.description',
            '              Data items in the PDBX_CHEM_COMP_AUDIT category records\n               the status and tracking information for this component.' ],
          [ 'item_', '_category.id', 'pdbx_chem_comp_audit' ],
          [ 'item_', '_category.mandatory_code', 'no' ],
          [ 'loop_',
            [ '_category_key.name' ],
            [ '_pdbx_chem_comp_audit.comp_id',
              '_pdbx_chem_comp_audit.date',
              '_pdbx_chem_comp_audit.action_type' ] ],
          [ 'loop_',
            [ '_category_group.id' ],
            [ 'inclusive_group', 'chem_comp_group', 'pdbx_group' ] ],
          [ 'loop_',
            [ '_category_examples.detail',
              '_category_examples.case',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ],
            [ '\n    Example 1 - ',
              '\n    loop_\n    _pdbx_chem_comp_audit.comp_id\n    _pdbx_chem_comp_audit.date\n    _pdbx_chem_comp_audit.action_type\n    _pdbx_chem_comp_audit.annotator\n     ATP . . . ',
              [ '@comment',
                ' - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -' ] ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_audit.comp_id',
        [ [ 'item_',
            '_item_description.description',
            '              This data item is a pointer to _chem_comp.id in the CHEM_COMP\n               category.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_audit.comp_id' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_audit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'ucode' ],
          [ 'item_',
            '_item_linked.child_name',
            '_pdbx_chem_comp_audit.comp_id' ],
          [ 'item_', '_item_linked.parent_name', '_chem_comp.id' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_audit.date',
        [ [ 'item_',
            '_item_description.description',
            '              The date associated with this audit record.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_audit.date' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_audit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'yyyy-mm-dd' ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_audit.annotator',
        [ [ 'item_',
            '_item_description.description',
            '              The initials of the annotator creating of modifying the component.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_audit.annotator' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_audit' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_', [ '_item_examples.case' ], [ 'JO', 'SJ', 'KB' ] ] ] ],
      [ 'save_',
        '_pdbx_chem_comp_audit.action_type',
        [ [ 'item_',
            '_item_description.description',
            '              The action associated with this audit record.' ],
          [ 'item_', '_item.name', '_pdbx_chem_comp_audit.action_type' ],
          [ 'item_', '_item.category_id', 'pdbx_chem_comp_audit' ],
          [ 'item_', '_item.mandatory_code', 'yes' ],
          [ 'item_', '_item_type.code', 'code' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'CREATE',
              'Created new component',
              'CHANGE_STATUS',
              'Change status',
              'CHANGE_ATOM_NOMENCLATURE',
              'Change atomic nomenclature',
              'CHANGE_MODEL',
              'Change model coordinates',
              'CHANGE_NOMENCLATURE',
              'Change component nomenclature',
              'CHANGE_BOND',
              'Change topology',
              'CHANGE_STEREO',
              'Change stereochemistry',
              'OTHER',
              'Miscellaneous change' ] ] ] ],
      [ 'save_',
        '_chem_comp.pdbx_release_status',
        [ [ 'item_',
            '_item_description.description',
            '              This data item holds the current release status for the component.' ],
          [ 'item_', '_item.name', '_chem_comp.pdbx_release_status' ],
          [ 'item_', '_item.category_id', 'chem_comp' ],
          [ 'item_', '_item.mandatory_code', 'no' ],
          [ 'item_', '_item_type.code', 'line' ],
          [ 'loop_',
            [ '_item_enumeration.value', '_item_enumeration.detail' ],
            [ 'REL',
              'Released',
              'HOLD',
              'On hold until yyyy-mm-dd',
              'HPUB',
              'On hold until publication',
              'OBS',
              'Entry has been obsoleted and replaced by another entry',
              'DEL',
              'Entry has been deleted' ] ] ] ],
      [ '@comment', '#' ] ] ] ]
 ;
}

if (typeof require !== 'undefined' && typeof exports !== 'undefined') {
  exports.cif_dict = cif_dict;

}
